<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit at IEO" />


<title>LFQ_Analysis</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.18/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.9.4.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Template_LFQ</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LFQ_Analysis</h1>
<h4 class="author">Fabio Bedin | MS-Unit at IEO</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-29
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>SKAP2_HaloTag/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20211221code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20211221)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20211221code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20211221)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong2865ca5"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 2865ca5 </a>
</p>
</div>
<div id="strongRepositoryversionstrong2865ca5" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 2865ca5. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  Counts.pdf
    Untracked:  Intensity_Distributions.pdf
    Untracked:  code/custom_functions.R
    Untracked:  data/expdesign.tsv
    Untracked:  data/proteinGroups.txt
    Untracked:  output/enrichmentmap.txt
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_vs_Halo.png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_vs_Halo.png
    Untracked:  output/output.zip
    Untracked:  output/results_table_Halo._vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_C._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_C._vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_C_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_C..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_N_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_N_vs_Halo.xlsx

Unstaged changes:
    Modified:   analysis/_site.yml
    Deleted:    analysis/about.Rmd
    Deleted:    analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/CSCAP2_vs_CSCAP2_act.Rmd</code>) and HTML (<code>docs/CSCAP2_vs_CSCAP2_act.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
2865ca5
</td>
<td>
FabioBedin
</td>
<td>
2021-12-29
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>## Proteomics
library(&quot;MBQN&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;enrichR&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)

## Plots
library(&quot;DT&quot;)
library(&quot;patchwork&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)

## Networks
library(&quot;STRINGdb&quot;)
library(&quot;RCy3&quot;)
library(&quot;RColorBrewer&quot;)

## Other
library(&quot;here&quot;)
library(&quot;openxlsx&quot;)
library(&quot;tidyverse&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(paste0(&quot;data/&quot;, params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot; &amp; data$Peptides &gt;= params$n_peptides,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<pre class="r"><code>cond_1 &lt;- params$test_1

cond_2 &lt;- params$test_2

conditions&lt;-c(cond_1,cond_2)

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)

expdesign &lt;- read.table(here::here(paste0(&quot;data/&quot;, params$expdesign)), header = T, stringsAsFactors = F)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
if(max(expdesign$replicate)&lt;=3){
       threshold&lt;-0
     } else if(max(expdesign$replicate)&lt;6){
       threshold&lt;-1
     } else if (max(expdesign$replicate)&gt;=6){
       threshold&lt;-trunc(max(expdesign$replicate)*0.25) ## 0.25 serve per avere sempre il 75% di valid vales
     }

data_filt &lt;- filter_missval(data_se, thr = threshold)</code></pre>
<div id="skap2_c.-vs-skap2_c_act" class="section level1">
<h1><strong>SKAP2_C.</strong> vs <strong>SKAP2_C_act</strong></h1>
<hr />
</div>
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>First, we need to wrangling the original dataset. From the <strong>proteinGroups</strong> obtained by <a href="https://www.maxquant.org/">MaxQuant</a> software, I remove proteins that are marked form the software as potential contaminant, only identify by site and reverse.<br />
We che also filter the initial datasets by excluding all proteins identified by less than 2 peptides.<br />
Now, we generate some quality control plots to investigate the structure of the dataset and observe the effect of filters.</p>
<div id="normalization-strategy" class="section level4">
<h4><strong>Normalization strategy:</strong></h4>
<pre class="r"><code>if(params$Normalization == &quot;VSN&quot;){
  data_filt &lt;- normalize_vsn(data_filt)
  print(&quot;VSN normalization is used&quot;)
} else if(params$Normalization == &quot;MBQN&quot;){
  data_filt &lt;- normalize_MBQN(data_filt)
  print(&quot;MBQN normalization is used&quot;)
} else {
  print(&quot;Max-LFQ normalization is used&quot;)
}</code></pre>
<p>[1] “Max-LFQ normalization is used”</p>
</div>
<div id="quality-control-plots" class="section level2 tabset tabset-fade">
<h2><strong>1. Quality control plots</strong></h2>
<div id="proteins-per-samples" class="section level3 tabset">
<h3>Proteins per samples</h3>
<p>This plot show the number of proteins identify in each samples <em>after and before User define</em> filters:</p>
<div id="filtered" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_filt)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/proteins-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_se)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/proteins-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="distributions" class="section level3 tabset">
<h3>Distributions</h3>
<p>This plot is useful for checking the distributions of the samples <em>after and before User define</em> filters:</p>
<div id="filtered-1" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_normalization(data_filt)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/normalization-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-1" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_normalization(data_se)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/normalization-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="missing-data" class="section level3 tabset">
<h3>Missing data</h3>
<p>This plot we can explore the missing data pattern <em>after and before User define</em> filters. Notice that the percentages of missingness are provided in the data. These are accurate to 1 decimal place. Also the dataset is arranged by columns with most missingness.</p>
<div id="filtered-2" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/missing-data-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-2" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>assay(data_se) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/missing-data-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample-cvs" class="section level3 tabset">
<h3>Sample CVs</h3>
<p>The coefficient of variation (CV) is a statistical measure of the dispersion of data points in a data series around the mean. The coefficient of variation represents the ratio of the standard deviation to the mean, and it is a useful statistic for comparing the degree of variation from one data series to another, even if the means are drastically different from one another.</p>
<div id="filtered-3" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_cvs(data_filt)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/CVs-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-3" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_cvs(data_se)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/CVs-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="upset-plot" class="section level2 tabset tabset-fade">
<h2><strong>2. Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within the 2 conditions or in common between all condition.</p>
<div id="skap2_c." class="section level3">
<h3>SKAP2_C.</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_1)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_1)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/upset-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="skap2_c_act" class="section level3">
<h3>SKAP2_C_act</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_2)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_2)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/upset-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="both-conditions" class="section level3">
<h3>Both conditions</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()
n_sets &lt;- length(define_set)
n_each &lt;- n_sets / 2

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;red3&quot;, &quot;royalblue1&quot;), each = n_each),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/upset-both-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="imputation" class="section level2">
<h2><strong>3. Imputation</strong></h2>
<p>An idealized version of a label-free discovery mass spectrometry proteomics experiment would provide absolute abundance measurements for a whole proteome, across varying conditions. Unfortunately, this ideal is not realized. Measurements are made on peptides requiring an inferential step to obtain protein level estimates. The inference is complicated by experimental factors that necessitate relative abundance estimation and result in widespread non-ignorable missing data. Relative abundance on the log scale takes the form of parameter contrasts. In a complete-case analysis, contrast estimates may be biased by missing data and a substantial amount of useful information will often go unused.</p>
<pre class="r"><code>data_imputed &lt;- mix_imputation_mean(data_filt, cond1 = cond_1, cond2 = cond_2)

plot_imputation(data_filt, data_imputed)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/imputation-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>data_filt &lt;- data_imputed</code></pre>
</div>
<div id="differential-enrichment-analysis" class="section level2">
<h2><strong>4. Differential enrichment analysis</strong></h2>
<p>Protein-wise linear models combined with empirical Bayes statistics are used for the differential enrichment analysis (or differential expression analysis) and False discovery rate (FDR) is ajusted with <em>Benjamini-Hochberg</em> procedure. Significant proteins are are define with <strong>FDR = 0.05</strong> and <strong>Fold change = 1</strong></p>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results&lt;-get_results(dep)</code></pre>
<p>There are <strong>619</strong> significant proteins in <strong>SKAP2_C.</strong> vs <strong>SKAP2_C_act</strong> comparison.</p>
<p>The results from the previous analysis can be easily visualized by a number of functions. These visualizations assist in the determination of the optimal cutoffs to be used, highlight the most interesting samples and contrasts, and pinpoint differentially enriched/expressed proteins.</p>
<div id="visualization-of-the-results" class="section level3 tabset tabset-fade">
<h3><strong>4.1. Visualization of the results</strong></h3>
<div id="pca" class="section level4">
<h4>PCA</h4>
<p>The PCA plot can be used to get a high-level overview of the data. This can be very useful to observe batch effects, such as clear differences between replicates.</p>
<pre class="r"><code>if(params$Imputation){
  if(nrow(dep) &gt;= 500){
    n_value &lt;- 500
  }else {
    n_value &lt;- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
} else {
  var &lt;- apply(assay(dep), 1, sd)
  df &lt;- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value &lt;- df %&gt;% as.data.frame() %&gt;% drop_na() %&gt;% nrow()
  if(n_value &gt;= 500){
    n_value &lt;- 500
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
}</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/PCA-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano" class="section level4">
<h4>Volcano</h4>
<p>Volcano plots allows to inspect the enrichment of proteins between the two samples (x axis) and their corresponding adjusted p value (y axis).</p>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T, label_size=5, adjusted = F)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/volcano-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-and-unique" class="section level4">
<h4>Volcano and unique</h4>
<pre class="r"><code>plot_volcano_2(dep, contrast=test, add_names=T, label_size=5, adjusted = F, unique_vec = &quot;SKAP2&quot;)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/volcano-unique-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="correlation" class="section level4">
<h4>Correlation</h4>
<pre class="r"><code>plot_cor(dep, significant = TRUE, lower = 0, upper = 1, pal = &quot;Reds&quot;)</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/correlation-matrix-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-heatmap" class="section level4">
<h4>Cluster heatmap</h4>
<p>The heatmap representation gives an overview of all significant proteins (rows) in all samples (columns). This allows to see general trends, for example if one sample or replicate is really different compared to the others. Additionally, the clustering of samples (columns) can indicate closer related samples and clustering of proteins (rows) indicates similarly behaving proteins.</p>
<pre class="r"><code>if(results %&gt;% filter(significant) %&gt;% nrow() &lt;= 100){
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = T, indicate = &quot;condition&quot;, col_limit = 5)
}else {
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = F, indicate = &quot;condition&quot;, col_limit = 5)
}</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/cluster-heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="result-table" class="section level3">
<h3><strong>4.2. Result table</strong></h3>
<p>In this table are summarized the results of DEP analysis, sorted by p.value. Proteins that are UP regulated are colored with a red box in Fold change column, in the orhter hand proteins that are DOWN regulated are colored with blue.</p>
<pre class="r"><code>results %&gt;% 
  select(starts_with(&quot;name&quot;) | starts_with(test)) %&gt;% 
  filter(across(ends_with(&quot;significant&quot;))) %&gt;% 
  arrange(across(ends_with(&quot;p.adj&quot;))) %&gt;%
  mutate(across(2:3, format, scientific = T, digits = 2)) %&gt;%
  dplyr::rename_with(~ tolower(gsub(paste0(test,&quot;_&quot;), &quot;&quot;, .x)), .cols = starts_with(test)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(Fold.Change = ratio, Gene.name = name) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:5)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;Fold.Change&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-d1ba32e0541469b5ed49" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-d1ba32e0541469b5ed49">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546","547","548","549","550","551","552","553","554","555","556","557","558","559","560","561","562","563","564","565","566","567","568","569","570","571","572","573","574","575","576","577","578","579","580","581","582","583","584","585","586","587","588","589","590","591","592","593","594","595","596","597","598","599","600","601","602","603","604","605","606","607","608","609","610","611","612","613","614","615","616","617","618","619"],["DLG3","KRI1","MRPL3","AAK1","ERLIN2","HELLS","PYCRL","SQLE","TBC1D15","UBE2H","FPGT-TNNI3K","TFIP11","RCBTB2","MRPL19","MRPS18B","PGRMC1","UTP11L","WDR82","ANAPC7","TSFM","WDR70","MFGE8","CKAP2","KDM1A","TYMS","MORF4L1","CCDC47","NCOA4","ANKRD39","REL","CDSN","EMG1","INCENP","PRPF38A","SFSWAP","KIAA1429","ARL8B","STXBP3","NDUFA10","TRA2A","ZMYM3","MRPS21","RIOK1","ZNHIT6","EOMES","STRBP","FANCD2","ERAP1","RRP8","DRAP1","MLEC","MORC3","TRAFD1","SERPINB12","A0A191URJ7","NSMCE2","DNM1L","MRPS17","NOL7","PTP4A3","RBM5","KDM3A","ZGPAT","EIF2A","RNF214","TAB2","SH2D1A","ING5","BRD1","GABRA2","MRPS7","CCDC22","CCDC59","S100A8","GAK","MAK16","CHAMP1","PDCD4","PHIP","PFKM","REXO4","WAPAL","RPRD1B","PTGS2","GRAMD1C","NGDN","PPIG","VDAC2","SERPINB9","TRIM24","NSA2","CALCOCO1","LRRC8C","SMS","JUNB","FAM107B","ASNA1","DUSP11","ECD","SMC6","IMP3","ESD","UPP1","GPX1","MRPS15","PRKRA","COX7A2","CDK11B","RCL1","DHX37","LGALS7","CD97","MRPL47","NOP9","VDAC3","DDX52","AQR","LIG3","HIST2H2AC","TRAF1","STAG2","AATF","WDR46.1","PWP2","IMMT","STOM","RSL1D1","WDR36","NUDT4","GNL3","SUCLG2","S100A9","HNRNPC","H2AFY","BANF1","ATAD3B","IK","RAI1","PNN","CDV3","SUPT6H","CUL4A","BOP1","hCG_2043597","SRFBP1","TBL3","CENPV","DSP","KIAA0101","HDAC10","HIST1H4A","WDR43","HNRNPH3","RRP1","POLR2G","EFHD2","MPHOSPH10","RGS1","HIST1H2BN","GTPBP4","EBNA1BP2","DCAF13","MFAP1","C17orf85","DNTTIP2","ANXA1","IMMT.1","POLR2E","GLTSCR2","SPEN","LIMA1","ADD3","EPDR1","SAP18","PNPO","ARMC10","NOB1","HSPA5","WDR12","VIM","PFKFB3","NOC3L","POLD1","MAGOHB","ANXA2","KIF22","RP9","BAZ1B","NAT10","SYNCRIP","PALLD","RALY","RPL24","UTP3","GBP5","RNPS1","SH3GLB1","DCPS","SNRPD3","TMSB4X","NHP2L1","NOP58","HNRNPUL1","UTP15","HNRNPUL2","SAMM50","NOP56","TOP2A","MKI67","NUP210","FTSJ3","SF3B6","TSN","YBX3","ERH","NME1","TPM3","SON","CHD3","CSNK2B-LY6G5B-1181","IRF2BP2","NVL","THRAP3","ACIN1","WDR75","DSC1","USP36","RPL10","RPS16","SLC3A2","GANAB","SNRPD2","PDIA6","IGF2BP1","NOC2L","FAM208A","SLC2A1","SRSF9","FAM120A","SNAP29","RRP12","KDM2A","TMEM43","DEF6","BYSL","C12orf75","LMNB2","CDC42","RPS11","LMNB1","DDX39B","MPHOSPH8","ALKBH4","CDC20","RBMX","TNS4","MYBBP1A","CHCHD3","HNRNPDL","HNRNPAB","CWC22","EIF4EBP1","GSR","KIF20A","NOP2","XRCC5","NAMPT","RBPJ","SNRPB2","NCAPG","YME1L1","H2AFV","HNRNPM","TSR1","DHX9","PRSS56","UBXN1","CHEK1","CEBPZ","DDX51","PKM","POLRMT","SUPT16H","HNRNPM.1","NHP2","SSRP1","TPX2","IBA57","RBM19","FBL","SLC1A4","DDX27","BLMH","HADH","MATR3","ILF2","UBE2C","CARS","PTBP3","PRPF8","POLR2A","S100A4","MRTO4","PARN","CDC2","EIF3H","LGALS1","RPA2","STAT5A","RPL7L1","DKC1","RRP1B","SRSF7","HNRNPU","RBM28","CHUK","TNFAIP3","DDX56","DDX18","MAP2K3","ATXN10","RACGAP1","HDHD3","SRSF1","HSCB","CFL1","CCNB1","DDX39A.1","SNRNP200","ELP3","SMARCA5","SNRPF","STK10","DIMT1","MCM7","IL16","DHX33","RRM1","HSPB1","SP140L","MCM6","NONO","HCLS1","HEATR3","PRDX2","PTPN7","ETFA","HIST1H2AA","IGF2BP3","MYH14","PDHB","CAPZB","CDC123","SKAP2","EMB","S100A6","BCLAF1","RPS24","CSTB","GNL2","SLBP","ANLN","GOPC","TAF15","UBTF","SF3A2","ATAD3A","CHCHD4","KRR1","FYB","RPS4X","SF3B1","CDC5L","PRKAR2A","RPS3A","BRK1","HIST1H1B","PABPN1","AIMP2","GZMA","XIRP1","FDFT1","STAU1","C21orf33","DDX39A","RASA4B","GRAPL","XRCC6","STMN1","SLC7A5","BOLA1","SNRPA","UTP18","MAP7","SMC1A","EFTUD2","TTN","ACAP1","TOP2B","DNAJB6","GLYR1","SRRM2","ETFB","ITM2A","PDCD11","ZNF326","POLR2L","RPS18","SRSF10","NUSAP1","NIFK","MYL12A","PDCD6","DDX46","DUT.1","VAV1","EXOSC4","UBC","WDR77","PTMA","CPOX","HNRNPA1","MCM4","NOL11","SNRNP40","RPL23","BMS1","STOML2","TRMT10C","PHF5A","KIFC1","POLR2B","SEC16A","EIF4A3","HIST1H1C","GCN1L1","AURKB","PDIA3","TRIM28","CTPS1","ILF3","HEATR1","HNRNPD","FADS2","CBX3","GMFG","THOC2","PES1","TRIM14","NIP7","SF3A1","EEF1D","MCM2","NOL10","TCAF2","HNRNPR","LRPPRC","LAS1L","NUP205","BZW1","EIF4A1","MRPS22","SAFB2","SOD1","VDAC1","HNRNPA0","TRIM65","SUZ12","RRP9","CBLL1","HNRNPA2B1","PELO","CBX5","UTP14A","TRMT61A","SF3B2","UTP20","CIRH1A","RAB18","CRMP1","DDX24","BAG3","PRAM1","RRS1","DHX30","RAD21","AGK","ANP32E","SAFB","CLPP","SF3B3","CSNK2A2","HNRNPA3","NOP16","KIAA0020","ZYX","WTAP","FUBP1","WDR74","PTER","LSM2","GOT2","BRIX1","RPA1","C12orf10","STMN2","CCAR1","CCDC86","SAE1","SNRPD1","AARS","GK","CAPG","JUP","SAR1A","NCOA5","SEC23A","EIF4G2","FKBP1A","HIST2H3PS2","DDX10","MRPS9","B2M","DCTN2","PA2G4","CS","DDX54","AKR1A1","DPP4","HSPE1","POLR2J3","HNRNPK.1","NOLC1","SMC3","ADH5","TRA2B","WDR3","USP14","RPN2","NAPA","VRK1","ATAD2","RRP15","SH3BGRL3","BPNT1","EPB41L2","UBE2L3","SNRNP70","ACSL1","KIR3DL2","PRKCSH","IDH2","RPF2","SNRPA1","OXCT1","GSTK1","SHMT2","UTP6","PRPF19","ZC3H14","SCAF8","SNRPN","ALDOA","EIF4B","TIMM44","FANCI","RPRC1","SF3A3","ATIC","LANCL1","G3BP1","NARFL","EHD1","KIAA1524","SSBP1","RBM3","FERMT3","LSM14A","BIN2","DAAM1","PCBP2.1","PSMD13","YY1","NOP14","OPA1","BST2","TRIP6","ZC3H15","PDS5B","APOL2","RPS7","SARS","LONP1","PYCR1","FEN1","MRPL11","HMMR","NDNL2","PSMD11","SSB","TPR","CLIC1","TMED10","MSN","TPT1","SLC29A1","YARS","PPAN-P2RY11","SYMPK","PPIP5K2","CHD1","DLAT","S100A11","LYAR","HMGA1","EMC8","FKBP4","SEC22B","PTPRC"],["1.8e-13","1.6e-12","1.8e-12","4.0e-12","4.9e-12","6.2e-12","5.1e-12","5.3e-12","3.7e-12","6.5e-12","8.1e-12","8.3e-12","1.1e-11","1.3e-11","1.3e-11","1.9e-11","2.0e-11","2.8e-11","3.0e-11","3.3e-11","3.4e-11","3.7e-11","4.9e-11","6.1e-11","6.4e-11","7.4e-11","9.1e-11","1.0e-10","1.3e-10","1.3e-10","1.5e-10","1.5e-10","1.6e-10","1.8e-10","1.8e-10","2.0e-10","2.2e-10","2.2e-10","2.3e-10","2.6e-10","2.6e-10","2.7e-10","2.8e-10","2.9e-10","3.0e-10","3.1e-10","3.3e-10","3.4e-10","3.6e-10","4.1e-10","4.5e-10","4.7e-10","5.2e-10","6.3e-10","6.7e-10","6.7e-10","7.1e-10","8.1e-10","8.9e-10","8.8e-10","8.8e-10","9.3e-10","1.0e-09","1.1e-09","1.2e-09","1.5e-09","1.6e-09","1.8e-09","2.0e-09","1.9e-09","2.0e-09","2.1e-09","2.1e-09","2.7e-09","2.8e-09","3.0e-09","3.5e-09","3.6e-09","4.0e-09","7.0e-09","7.0e-09","7.5e-09","9.3e-09","1.1e-08","1.5e-08","1.5e-08","1.6e-08","1.8e-08","2.3e-08","2.6e-08","2.8e-08","2.9e-08","2.9e-08","3.0e-08","3.1e-08","3.3e-08","3.5e-08","4.4e-08","4.3e-08","4.3e-08","4.5e-08","5.6e-08","5.7e-08","6.3e-08","6.8e-08","6.8e-08","6.9e-08","7.2e-08","7.5e-08","7.8e-08","1.9e-07","2.0e-07","2.0e-07","2.1e-07","2.2e-07","2.4e-07","2.5e-07","2.6e-07","2.7e-07","2.9e-07","3.0e-07","3.2e-07","3.3e-07","3.6e-07","4.4e-07","4.7e-07","4.9e-07","5.1e-07","5.3e-07","5.5e-07","5.5e-07","5.8e-07","5.9e-07","6.1e-07","6.2e-07","6.4e-07","6.5e-07","6.5e-07","7.4e-07","8.0e-07","8.3e-07","8.4e-07","8.5e-07","8.7e-07","9.2e-07","9.1e-07","9.6e-07","9.5e-07","9.5e-07","1.0e-06","1.0e-06","1.1e-06","1.1e-06","1.2e-06","1.2e-06","1.3e-06","1.3e-06","1.3e-06","1.4e-06","1.4e-06","1.5e-06","1.7e-06","1.8e-06","2.1e-06","2.1e-06","2.1e-06","2.2e-06","2.2e-06","2.2e-06","2.4e-06","2.6e-06","2.8e-06","2.8e-06","2.9e-06","3.0e-06","3.1e-06","3.1e-06","3.1e-06","3.2e-06","3.4e-06","4.3e-06","4.6e-06","4.7e-06","4.9e-06","5.0e-06","5.2e-06","5.3e-06","5.5e-06","5.6e-06","6.4e-06","6.5e-06","6.8e-06","6.9e-06","7.0e-06","7.3e-06","7.3e-06","7.3e-06","7.4e-06","7.9e-06","8.0e-06","8.3e-06","8.6e-06","8.8e-06","9.3e-06","9.5e-06","9.9e-06","1.0e-05","1.1e-05","1.1e-05","1.1e-05","1.1e-05","1.1e-05","1.2e-05","1.2e-05","1.2e-05","1.2e-05","1.2e-05","1.2e-05","1.3e-05","1.3e-05","1.3e-05","1.3e-05","1.4e-05","1.5e-05","1.5e-05","1.5e-05","1.5e-05","1.5e-05","1.6e-05","1.6e-05","1.6e-05","1.6e-05","1.7e-05","1.7e-05","1.7e-05","1.7e-05","1.8e-05","1.8e-05","1.8e-05","1.8e-05","1.8e-05","1.9e-05","1.9e-05","1.9e-05","2.0e-05","2.0e-05","2.0e-05","2.0e-05","2.1e-05","2.1e-05","2.1e-05","2.2e-05","2.2e-05","2.2e-05","2.3e-05","2.5e-05","2.5e-05","2.6e-05","2.6e-05","2.6e-05","2.8e-05","2.9e-05","2.9e-05","2.9e-05","3.2e-05","3.2e-05","3.3e-05","3.4e-05","3.4e-05","3.6e-05","3.6e-05","3.6e-05","3.7e-05","3.8e-05","3.8e-05","3.8e-05","3.9e-05","3.9e-05","4.0e-05","4.0e-05","4.1e-05","4.2e-05","4.4e-05","4.4e-05","4.4e-05","4.4e-05","4.4e-05","4.5e-05","4.5e-05","4.6e-05","4.6e-05","4.9e-05","5.0e-05","5.3e-05","5.3e-05","5.4e-05","5.4e-05","5.4e-05","5.4e-05","5.6e-05","5.7e-05","5.7e-05","5.8e-05","5.8e-05","5.9e-05","6.0e-05","6.0e-05","6.0e-05","6.3e-05","6.3e-05","6.3e-05","6.4e-05","6.4e-05","6.6e-05","6.8e-05","6.8e-05","6.9e-05","7.1e-05","7.2e-05","7.1e-05","7.2e-05","8.1e-05","8.3e-05","8.6e-05","8.8e-05","9.1e-05","9.2e-05","9.2e-05","9.3e-05","1.0e-04","1.0e-04","1.0e-04","1.0e-04","1.1e-04","1.1e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.3e-04","1.3e-04","1.3e-04","1.4e-04","1.4e-04","1.4e-04","1.4e-04","1.5e-04","1.4e-04","1.5e-04","1.5e-04","1.5e-04","1.5e-04","1.5e-04","1.5e-04","1.6e-04","1.6e-04","1.7e-04","1.7e-04","1.7e-04","1.8e-04","1.8e-04","1.8e-04","1.8e-04","1.8e-04","1.8e-04","1.8e-04","1.9e-04","1.9e-04","2.0e-04","2.0e-04","2.0e-04","2.0e-04","2.0e-04","2.0e-04","2.0e-04","2.1e-04","2.1e-04","2.1e-04","2.1e-04","2.1e-04","2.1e-04","2.2e-04","2.2e-04","2.2e-04","2.2e-04","2.2e-04","2.3e-04","2.3e-04","2.5e-04","2.5e-04","2.6e-04","2.6e-04","2.6e-04","2.6e-04","2.6e-04","2.8e-04","2.8e-04","2.9e-04","2.9e-04","3.0e-04","3.1e-04","3.2e-04","3.3e-04","3.3e-04","3.3e-04","3.4e-04","3.4e-04","3.5e-04","3.5e-04","3.6e-04","3.7e-04","3.8e-04","3.9e-04","3.9e-04","3.9e-04","4.0e-04","4.1e-04","4.1e-04","4.2e-04","4.3e-04","4.3e-04","4.4e-04","4.4e-04","4.4e-04","4.5e-04","4.5e-04","4.5e-04","4.6e-04","4.6e-04","4.7e-04","4.7e-04","4.7e-04","4.7e-04","4.8e-04","4.8e-04","4.9e-04","5.0e-04","5.0e-04","5.0e-04","5.0e-04","5.2e-04","5.5e-04","5.6e-04","5.8e-04","5.9e-04","6.0e-04","6.0e-04","6.2e-04","6.3e-04","6.5e-04","6.5e-04","6.7e-04","6.7e-04","6.9e-04","7.0e-04","7.0e-04","7.2e-04","7.4e-04","7.4e-04","7.4e-04","7.5e-04","7.6e-04","7.7e-04","7.8e-04","8.0e-04","8.0e-04","8.2e-04","8.3e-04","8.6e-04","8.8e-04","8.8e-04","8.8e-04","8.9e-04","9.3e-04","9.6e-04","9.7e-04","9.9e-04","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.8e-03","1.8e-03","1.9e-03","1.9e-03","2.0e-03","2.1e-03","2.1e-03","2.1e-03","2.2e-03","2.3e-03","2.3e-03","2.4e-03","2.4e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.7e-03","2.7e-03","2.7e-03","2.7e-03","2.7e-03","2.8e-03","2.9e-03","2.9e-03","2.9e-03","3.0e-03","3.0e-03","3.0e-03","3.0e-03","3.2e-03","3.2e-03","3.2e-03","3.3e-03","3.3e-03","3.3e-03","3.5e-03","3.5e-03","3.7e-03","3.7e-03","3.9e-03","3.9e-03","4.0e-03","4.0e-03","4.0e-03","4.0e-03","4.0e-03","4.1e-03","4.2e-03","4.3e-03","5.1e-03","5.1e-03","5.2e-03","5.2e-03","5.3e-03","5.4e-03","5.3e-03","5.5e-03","5.9e-03","5.9e-03","5.9e-03","6.0e-03","6.2e-03","6.2e-03","6.2e-03","6.4e-03","6.8e-03","6.9e-03","7.4e-03","7.5e-03","7.5e-03","8.2e-03","8.4e-03","8.4e-03","8.7e-03","8.8e-03","8.8e-03","8.9e-03","8.9e-03","9.0e-03","9.1e-03","9.4e-03","9.5e-03","9.5e-03","9.6e-03","9.7e-03","9.9e-03","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02","1.8e-02","1.9e-02","2.0e-02","2.0e-02","2.1e-02","2.1e-02","2.2e-02","2.2e-02"],["4.2e-10","1.4e-09","1.4e-09","1.5e-09","1.5e-09","1.5e-09","1.5e-09","1.5e-09","1.5e-09","1.5e-09","1.6e-09","1.6e-09","2.0e-09","2.1e-09","2.1e-09","2.7e-09","2.7e-09","3.7e-09","3.7e-09","3.9e-09","3.9e-09","3.9e-09","5.0e-09","6.0e-09","6.1e-09","6.8e-09","8.0e-09","8.6e-09","1.1e-08","1.1e-08","1.1e-08","1.1e-08","1.2e-08","1.2e-08","1.2e-08","1.3e-08","1.4e-08","1.4e-08","1.4e-08","1.5e-08","1.5e-08","1.5e-08","1.5e-08","1.5e-08","1.6e-08","1.6e-08","1.7e-08","1.7e-08","1.7e-08","1.9e-08","2.1e-08","2.2e-08","2.3e-08","2.8e-08","2.8e-08","2.8e-08","2.9e-08","3.3e-08","3.5e-08","3.5e-08","3.5e-08","3.6e-08","3.9e-08","4.2e-08","4.3e-08","5.4e-08","5.8e-08","6.3e-08","6.6e-08","6.6e-08","6.6e-08","6.8e-08","6.8e-08","8.7e-08","8.9e-08","9.5e-08","1.1e-07","1.1e-07","1.2e-07","2.1e-07","2.1e-07","2.2e-07","2.7e-07","3.1e-07","4.2e-07","4.2e-07","4.5e-07","4.9e-07","6.2e-07","7.0e-07","7.2e-07","7.5e-07","7.5e-07","7.6e-07","7.8e-07","8.2e-07","8.7e-07","1.0e-06","1.0e-06","1.0e-06","1.1e-06","1.3e-06","1.3e-06","1.4e-06","1.5e-06","1.5e-06","1.5e-06","1.6e-06","1.6e-06","1.7e-06","4.1e-06","4.2e-06","4.3e-06","4.3e-06","4.5e-06","5.0e-06","5.1e-06","5.3e-06","5.3e-06","5.8e-06","5.8e-06","6.3e-06","6.4e-06","6.9e-06","8.3e-06","8.9e-06","9.1e-06","9.5e-06","9.7e-06","1.0e-05","1.0e-05","1.0e-05","1.1e-05","1.1e-05","1.1e-05","1.1e-05","1.1e-05","1.1e-05","1.3e-05","1.4e-05","1.4e-05","1.4e-05","1.4e-05","1.4e-05","1.5e-05","1.5e-05","1.5e-05","1.5e-05","1.5e-05","1.6e-05","1.6e-05","1.7e-05","1.7e-05","1.8e-05","1.9e-05","1.9e-05","1.9e-05","2.0e-05","2.1e-05","2.1e-05","2.2e-05","2.4e-05","2.6e-05","3.0e-05","3.0e-05","3.0e-05","3.1e-05","3.1e-05","3.1e-05","3.3e-05","3.6e-05","3.8e-05","3.8e-05","3.9e-05","4.0e-05","4.1e-05","4.1e-05","4.2e-05","4.3e-05","4.5e-05","5.6e-05","6.0e-05","6.0e-05","6.3e-05","6.4e-05","6.6e-05","6.7e-05","7.0e-05","7.0e-05","8.0e-05","8.0e-05","8.3e-05","8.4e-05","8.6e-05","8.8e-05","8.8e-05","8.8e-05","8.9e-05","9.3e-05","9.5e-05","9.8e-05","1.0e-04","1.0e-04","1.1e-04","1.1e-04","1.1e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.4e-04","1.4e-04","1.4e-04","1.4e-04","1.5e-04","1.5e-04","1.5e-04","1.6e-04","1.6e-04","1.6e-04","1.6e-04","1.6e-04","1.6e-04","1.6e-04","1.7e-04","1.7e-04","1.7e-04","1.7e-04","1.7e-04","1.7e-04","1.8e-04","1.8e-04","1.8e-04","1.8e-04","1.8e-04","1.9e-04","1.9e-04","1.9e-04","1.9e-04","1.9e-04","2.0e-04","2.0e-04","2.0e-04","2.0e-04","2.1e-04","2.1e-04","2.2e-04","2.3e-04","2.3e-04","2.3e-04","2.4e-04","2.4e-04","2.5e-04","2.6e-04","2.6e-04","2.6e-04","2.8e-04","2.8e-04","2.8e-04","3.0e-04","3.0e-04","3.1e-04","3.1e-04","3.1e-04","3.1e-04","3.2e-04","3.2e-04","3.2e-04","3.2e-04","3.3e-04","3.3e-04","3.3e-04","3.4e-04","3.4e-04","3.6e-04","3.6e-04","3.6e-04","3.6e-04","3.6e-04","3.6e-04","3.6e-04","3.6e-04","3.6e-04","3.9e-04","4.0e-04","4.1e-04","4.1e-04","4.2e-04","4.2e-04","4.2e-04","4.2e-04","4.3e-04","4.4e-04","4.4e-04","4.4e-04","4.4e-04","4.4e-04","4.5e-04","4.5e-04","4.5e-04","4.7e-04","4.7e-04","4.7e-04","4.7e-04","4.7e-04","4.8e-04","5.0e-04","5.0e-04","5.0e-04","5.2e-04","5.2e-04","5.2e-04","5.2e-04","5.8e-04","5.9e-04","6.1e-04","6.2e-04","6.4e-04","6.5e-04","6.5e-04","6.5e-04","6.9e-04","7.0e-04","7.0e-04","7.0e-04","7.2e-04","7.8e-04","7.8e-04","7.9e-04","7.9e-04","8.1e-04","8.5e-04","8.7e-04","8.7e-04","9.1e-04","9.2e-04","9.2e-04","9.4e-04","9.4e-04","9.4e-04","9.4e-04","9.4e-04","9.4e-04","9.4e-04","9.8e-04","9.9e-04","9.9e-04","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.6e-03","1.6e-03","1.7e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.9e-03","1.9e-03","2.0e-03","2.0e-03","2.0e-03","2.0e-03","2.0e-03","2.1e-03","2.1e-03","2.1e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.4e-03","2.4e-03","2.4e-03","2.4e-03","2.4e-03","2.5e-03","2.6e-03","2.6e-03","2.7e-03","2.8e-03","2.8e-03","2.8e-03","2.9e-03","2.9e-03","3.0e-03","3.0e-03","3.1e-03","3.1e-03","3.1e-03","3.1e-03","3.2e-03","3.2e-03","3.3e-03","3.3e-03","3.3e-03","3.3e-03","3.4e-03","3.4e-03","3.4e-03","3.5e-03","3.5e-03","3.5e-03","3.6e-03","3.7e-03","3.8e-03","3.8e-03","3.8e-03","3.8e-03","3.9e-03","4.0e-03","4.0e-03","4.1e-03","4.3e-03","4.3e-03","4.3e-03","4.3e-03","4.4e-03","4.6e-03","4.6e-03","4.8e-03","4.9e-03","4.9e-03","4.9e-03","5.0e-03","5.4e-03","5.5e-03","5.6e-03","5.8e-03","5.8e-03","5.9e-03","6.1e-03","6.3e-03","6.4e-03","6.4e-03","6.4e-03","6.5e-03","6.8e-03","6.9e-03","7.0e-03","7.3e-03","7.4e-03","7.6e-03","7.7e-03","7.9e-03","8.1e-03","8.2e-03","8.3e-03","8.6e-03","8.7e-03","8.9e-03","9.0e-03","9.0e-03","9.0e-03","9.4e-03","9.4e-03","9.5e-03","9.5e-03","9.6e-03","9.6e-03","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","2.0e-02","2.0e-02","2.1e-02","2.1e-02","2.1e-02","2.3e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.9e-02","2.9e-02","3.0e-02","3.2e-02","3.2e-02","3.2e-02","3.4e-02","3.5e-02","3.5e-02","3.6e-02","3.6e-02","3.8e-02","3.9e-02","3.9e-02","4.0e-02","4.1e-02","4.1e-02","4.3e-02","4.4e-02","4.5e-02","4.7e-02","4.8e-02","4.9e-02","4.9e-02","5.0e-02"],["True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True"],[-5.27,5.02,4.13,-3.58,4.06,3.61,-4.41,-4.42,3.74,-5.69,3.74,-3.93,-4.1,3.68,3.31,3.93,5.23,3.05,3.49,3.61,3.11,-2.88,3.29,3.45,4.5,3.49,3.17,3.09,4.16,-3.48,4.93,3.99,4.87,4.63,-2.74,-2.53,4.55,3.41,4.01,5.18,3.33,3.11,2.6,3.45,-4.49,3.03,3.44,3.22,4.62,2.87,3.48,-3.15,-3.1,4.85,-2.5,4.22,-3.69,4.4,6.14,4.32,2.42,-3.32,-3.16,3.6,-2.34,-3.61,4.66,3.18,3.56,-6.15,3.54,-2.72,3.25,4.89,2.74,6.3,2.1,2.56,3.11,-3.01,2.39,2.61,4.26,-4.28,5.53,4.87,2.76,5.46,-3.32,3.21,4.58,-3.09,-4.91,4.76,-4.36,-2.76,4.07,3.68,2.2,3.95,5.83,4.36,-3.27,2.07,3.35,5,3.62,3.34,5.55,2.66,4.14,-2.64,3.82,3.37,5.21,3.13,2.92,4.07,3.69,-2.22,3.49,5.01,3.98,4.58,3.37,2.87,4.13,4.55,-1.6,3.14,3.31,2.52,3.68,5.3,7.57,1.68,1.76,2.85,4.3,-2.04,3.55,2.96,5.3,-1.76,3.31,5.79,5.63,4.77,2.82,-1.79,3.14,2.47,5.79,4.86,3.22,-1.27,5.72,-2.11,3.23,4.91,3.45,4.87,3.72,2.82,5.72,-1.59,6.15,1.42,1.94,3.43,-1.42,3.07,-1.86,3.09,-2.07,2.15,1.28,-1.18,4.52,3.25,-3.28,2.43,1.46,2.17,-1.56,2.41,2.41,5.25,2.78,2.28,-1.67,5.48,1.43,4.74,-1.53,3.47,-1.38,-1.1,1.52,-5.24,2.78,3.04,3.65,5.34,4.48,4.26,3.17,6.67,2.68,4.01,4.32,1.14,-1.89,2.14,2.39,-2.73,-1.49,4.86,2.62,-1.43,-1.24,3.49,2.77,3.8,4.28,4.29,3.3,1.27,1.11,-1.2,3.49,1.68,-1.48,3.18,4.3,3.54,-1.9,3.63,1.49,3.46,2.73,2.19,2.32,-1.4,3.83,-2.45,3.88,1.41,1.38,2.67,1.78,3.9,-2.58,1.38,3.07,-1.12,1.53,3.51,3.47,4.58,3.37,-1.44,-1.28,1.94,3.08,1.27,-1.35,-1.28,2.76,1.01,-6.3,3.77,1.54,1.1,3.01,-1.25,-1.68,1.51,3.57,3.64,-1.01,1.32,4.07,1.46,2.17,4.74,1.06,-1.7,3.19,2.39,2.4,7.97,1.44,2.48,3.07,1.71,1.19,-1,2.19,1.55,4.32,-1.13,2.81,2.51,1.49,1.62,-1.05,2.04,-1.06,4.39,3.05,2.1,2.71,2.24,2.55,-2.61,-2.66,5.66,3.54,-1.07,-1.07,1.01,-1.72,2.64,2.08,-1.09,2.31,1.54,1.9,-1.2,3.85,1.01,-1.32,1.44,1.13,-1.07,1.46,1.18,1.69,1.76,1.53,1.12,-1.09,1.06,-1.22,-1.01,2.76,4.52,1.8,-1.3,1.49,-1.07,1.03,-1.23,-1.24,-1.01,3.28,1.05,-1.25,1.54,1.12,1.85,1.35,2.38,1.61,2,1.66,-1.04,3.91,-1.38,1.27,1.31,2.62,-1.1,1.05,-1.12,1.51,1.9,-1.82,-1.47,-4.79,1.1,1.74,-1.31,-3.96,1.35,1.05,1.26,-2.17,-1.08,-1.06,1.37,3.22,1.43,1.73,1.58,-1.59,-1.35,2.1,1.45,1.22,1.52,1.97,-1.34,4.35,2.8,2.1,1.47,3.08,1.44,6.06,-1.32,-1.67,1.77,-1.18,-1.21,-1.08,1.35,-1.19,-3.49,-1.08,2.19,2.56,2.17,1.13,1.15,2.9,2.73,-1.04,1.47,1.57,3.36,-1.02,2.54,1.16,2.12,1.62,-1.1,1.81,-1.01,1.73,6.21,2.22,1.61,2.13,1.13,-1.31,2.71,1.02,1.82,2.35,-1.04,1.51,3.37,-1.24,1.73,1.08,1.32,1.36,1.2,1.21,1.1,1.47,-1.37,2.9,1.96,-1.38,1.37,3.12,2.4,2.31,-1.22,4.53,5.19,-1.12,2.19,4.67,2.95,-1.11,-1,1.61,-1,1.11,4.16,2.75,2.41,1.11,3.11,2.24,1.06,1.04,-1.05,1.73,4.06,3.88,-1.36,1.48,-1.38,1.91,-1.01,1.23,-2.26,2.31,1.87,1.55,-3.02,1.41,1.8,1.56,3.03,1.26,1.41,1.71,2.73,1.77,-1.38,-1,1.5,-1.27,2.12,1.81,-1.01,-1.36,-1.02,1.12,1.68,2.59,1.16,-1.09,-3.11,1.52,1.38,2.27,2.32,1.83,1.67,3.14,2.27,2.33,1.39,1.39,3.46,3.24,-2.83,-1.01,1.45,-1.01,1.3,1.93,1.24,-2.06,1.3,2.92,1.49,1.09,1.45,1.03,2.43,1.38,1.84,1.02,1.73,-1.32,-1.02,2.03,1.48,1.31,2.23,1.96,-1,2.41,-1.24,3.22,1.3,1.75,1.57,2.36,-1.39,-1.12,-1.91,2.01,1.63,1.78,3.19,-1.21,1.34,1.17,-1.09,2.9,-1.11,2.44,2.46,2.1,1.15,1.67,1.7,1.59,1.24,1.06,1.33,1.33,1.39,1.3,-1.06,1.01,1.92,1.8,1.48,1.21,-4.47,2.01,1.06,1.63,1.07,1.74,-1.56,1.51,1.35,-1.35]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene.name<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>Fold.Change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[5]; $(this.api().cell(row, 5).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[4]; $(this.api().cell(row, 4).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
<div id="gene-ontology" class="section level2">
<h2><strong>5. Gene Ontology</strong></h2>
<p>Now I perfom a gene onology analysis (GO or by branch <a href="GO:MF" class="uri">GO:MF</a>, <a href="GO:BP" class="uri">GO:BP</a>, <a href="GO:CC" class="uri">GO:CC</a>) and a REAC ontology.<br />
The enrichment results are visualized with an interactive <strong>Manhattan-like-plot</strong> named “Gostplot”.</p>
<div id="skap2_c.-1" class="section level3 tabset tabset-fade">
<h3><strong>5.1. SKAP2_C.</strong></h3>
<div id="gostplot" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in SKAP2_C. were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &gt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_1)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-06484ed42468d86de4a4" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-06484ed42468d86de4a4">{"x":{"data":[{"x":[81.4008725542289,101.03850178107,116.213716409311,81.3032102818689,93.7551500077641,109.813081328489,110.049724526899,80.5482057917015,152.668037688289,93.7513937665195,85.9872431139025,81.4046287954735,125.739544205652,72.964354718826,82.5465261338361,110.098555663079,73.0169420962506,72.9718672013152,93.7476375252749,164.924652869464,89.893734008301,116.266303786735,119.969957653925,116.262547545491,102.037661952138,82.8132192622037,120.184063404868,146.81581382918,118.302186541316,120.187819646112,146.53033949459,73.1897291935028,73.1596792635459,131.625574235962,85.6980125380672,171.370362845222,101.057282987294,129.901459504684,146.766982693,81.4271662429412,80.9726610523429,118.959528759124,73.1521667810567,81.4196537604519,81.4234100016966,73.2197791234597,73.2235353647043,73.1634355047905,73.1859729522582,81.4572161728981,127.01666622882,158.726854815851,171.374119086466,144.806224763312,119.537989910794,130.86681350455,73.5240346642735,93.6424627704257,117.990418518013,119.237490611225,116.217472650555,81.7952778849133,129.833847162281,129.837603403525,81.0477858772352,131.509130757379,83.598273682328,110.087286939345,131.29878124768,72.8366425165091,73.1747042285244,155.654249477757,96.6474557661168,72.8028363453076,105.298079352463,126.284199186121,83.4029491376081,120.262944471005,96.6587244898506,81.4346787254304,145.670160249573,119.23373436998,144.851299658247,83.5644675111265,97.5414411823349,107.07102521992,72.43848094458,93.4133320545042,109.997137149475,105.579797445809,127.628933551693,83.151280974219,72.5060932869831,83.3428492776943,164.928409110709,107.236299834683,72.6676116605015,72.983135925049,166.408368161087,142.477355191651,127.020422470065,80.9764172935875,110.064749491878,81.4609724141427,86.1900801411116,73.3512475670212,120.266700712249,73.1033356448767,131.295025006436,80.9989547410552,81.438434966675,145.673916490818,72.6413179717892,81.0552983597244,73.6329656603673,148.457291253077,182.939585878633,168.425469709444,103.172046808011,96.7751679684337,129.908971987173,90.29189558023,123.823861170899,104.704593235814,90.3670204051223,182.947098361122,118.978309965347,142.646386047659,118.970797482857,119.301346712383,142.650142288904,81.2769165931566,120.251675747271,81.1529606320844,131.557961893559,100.933327026221,72.8291300340199,118.974553724102,83.695935954688,94.4312734317946,146.958550996476,73.2272916059489,104.719618200792,130.941938329442,170.547746012651,103.175803049256,139.295818857463,165.672144877142,81.1229107021274,81.1867668032859,153.231473874981,120.214113334825,81.4684848966319,183.112372975885,81.4722411378765,131.696942819609,73.3550038082658,105.666190994435,133.560038476938,73.3587600495105,132.906452500375,86.186323899867,186.549333714707,186.553089955951,81.4647286553873,129.905215745929,93.6650002178934,127.02417871131,127.557564968045,123.793811240942,168.192582752278,80.9951984998106,86.1825676586224,81.0778358071921,144.498212981254,81.2694041106674,91.0919749653328,103.517621002516,132.996602290246,123.384380945279,75.6125047962788,152.679306412023,73.674284314058,73.197241675992,133.578819683161,152.671793929534,106.935800535114,107.240056075928,165.638338705941,83.6057861648172,81.1792543207966,89.8974902495456,174.957573233828,109.283451312998,103.243415391659,168.440494674423,88.2672815493831,131.730748990811,123.301743637897,108.659915266392,83.4292428263204,158.801979640743,90.1454021716901,151.83039589074,107.074781461165,86.0623679387948,81.0365171535013,104.727130683281,81.2881853168904,82.148364561907,157.836625640878,131.561718134803,84.5486027172153,101.376563493086,73.3662725319997,171.152500853034,83.6696422659757,104.74215564826,83.7072046784218,131.708211543343,82.1446083206624,167.497678122025,101.015964333603,72.5173620107169,107.521774169274,186.515527543505,91.527698949708,72.9868921662937,157.840381882122,121.329716984475,81.0064672235444,131.704455302099,153.227717633736,133.575063441916,131.715724025832,124.706577863383,72.9906484075383,103.795582854617,129.766234819878,171.377875327711,103.307271492817,90.4045828175685,107.799736021376,123.46701825266,121.337229466964,126.798804236633,121.720366073915,91.4675990897942,109.891962394626,108.24297248824,101.695843998878,129.785016026101,130.115565255627,109.888206153381,170.682970697457,150.801185789716],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,14.3770754326031,14.0065777084671,13.9360656330616,11.5489503278206,11.1549483394275,10.4395822213601,10.3787197019,10.3764529065897,9.78987457606212,9.76427193599354,9.76427193599354,9.49886898321582,9.42979406782741,9.05846681067635,9.01107991683775,8.93026132971091,8.86290420054828,8.83962428298511,8.75981173814648,8.69720170914374,8.56765195448358,8.51414280461934,8.50578348486126,8.43021815713718,8.43014303712434,8.06129849418387,7.85042063849779,7.85042063849779,7.80471426216166,7.70215899241123,7.69776219656667,7.51390642394754,7.08823158027186,7.03419230072612,6.94073057388392,6.90237770390114,6.79192139782604,6.64655501878345,6.51036295367938,6.4821786308882,6.45405460155067,6.35690230308798,6.33114395923089,6.27545111840454,6.2189552597769,6.16920318834158,6.16299176292541,6.15042800273879,6.11523747430013,6.09693151181401,6.05537207955939,5.97973658846896,5.91001706296339,5.9042149464378,5.8494210815582,5.84208000100786,5.81020396340064,5.66610066171112,5.61729347072117,5.46875026160541,5.39600668283115,5.36435661601438,5.24908617809652,5.14020120781729,4.88248092399764,4.85279791063807,4.78618137089037,4.77935865288401,4.71398051403147,4.65309947050602,4.64361327409739,4.62910566845916,4.61286388507773,4.55388809381378,4.50475101642944,4.50475101642944,4.46748321521979,4.46149369089422,4.41857357904636,4.41620038435872,4.34867726933582,4.3193892697325,4.25474159522864,4.21296807755756,4.20896142192142,4.20287557747644,4.17995465319508,4.15864692390445,4.15291323717767,4.14956573564389,4.07710975704846,3.98569994537911,3.96370247231527,3.95979131095164,3.93741848571528,3.89211046682991,3.88610782187324,3.86400039029716,3.79364483138405,3.73085095257635,3.71164812922059,3.68986225812371,3.68962256321009,3.64121181592825,3.58729713290237,3.55782215092824,3.55269716847532,3.53037435792323,3.52576907726015,3.47585979184078,3.44548890269924,3.38285312222661,3.36330407835132,3.33755745306357,3.28365688006795,3.27809608288505,3.24097464376588,3.21483291133691,3.17690295913346,3.1537765818209,3.14218514693188,3.11568610941632,3.0740282775754,3.0703619934576,3.01466457429206,2.93283998027027,2.9051195646686,2.90189839121097,2.86317731574871,2.85053792900138,2.78503963365511,2.78133585194801,2.76806420749349,2.75181448739293,2.73180633805862,2.73180633805862,2.67973966615333,2.65739530600245,2.64558465258052,2.62245763120091,2.61770736970332,2.61031536901155,2.60855376659421,2.59643106770733,2.57205504500135,2.5320792671677,2.50913206029347,2.50913206029347,2.50105990033662,2.48775551192409,2.48585700410413,2.46513513297946,2.45163009212555,2.43516774165996,2.42552007667306,2.42069514108297,2.39791765685119,2.34967448152012,2.33131074023365,2.28534902107279,2.27122405661043,2.26445786204111,2.23189659748171,2.22886111432428,2.22545414018019,2.2210967843809,2.21547136815749,2.20077934994342,2.17890040999003,2.14025759361505,2.12834464105089,2.11184399871168,2.09370331365675,2.00336369354209,2.00281078895342,1.98799889024551,1.97464288742472,1.91453893779909,1.89598423971598,1.82774741283075,1.79832387018977,1.794361782869,1.79109548624202,1.76673204445222,1.76440670153107,1.73120948232444,1.73104858793844,1.70753242214608,1.67741761970776,1.67676949021138,1.67249100655896,1.65151861582001,1.63074450973384,1.61490052774376,1.58350756221595,1.58310837104319,1.5778180629563,1.57152737371256,1.55840048949403,1.55827375250077,1.54916726870555,1.52135984568659,1.51362715794577,1.50680673559807,1.5062213931238,1.50120457033969,1.49586317694747,1.49450774742946,1.48468432051143,1.43706759972584,1.43631060210294,1.41538366273193,1.39541755886773,1.3916019704233,1.36358831936481,1.34822581212628,1.34444017556398,1.33323688278632,1.33323688278632,1.30986391835533],"text":["GO:0006396 (1010) <br> RNA processing <br> 2.568e-111","GO:0022613 (473) <br> ribonucleoprotein complex biogenesis <br> 9.790e-84","GO:0042254 (335) <br> ribosome biogenesis <br> 9.287e-78","GO:0006364 (265) <br> rRNA processing <br> 7.160e-72","GO:0016072 (275) <br> rRNA metabolic process <br> 2.725e-70","GO:0034470 (439) <br> ncRNA processing <br> 5.066e-56","GO:0034641 (6576) <br> cellular nitrogen compound metabolic process <br> 3.293e-52","GO:0006139 (5752) <br> nucleobase-containing compound metabolic process <br> 8.286e-52","GO:0090304 (5234) <br> nucleic acid metabolic process <br> 1.357e-51","GO:0016071 (880) <br> mRNA metabolic process <br> 1.666e-51","GO:0008380 (459) <br> RNA splicing <br> 2.009e-50","GO:0006397 (532) <br> mRNA processing <br> 2.141e-50","GO:0046483 (5929) <br> heterocycle metabolic process <br> 4.394e-50","GO:0000375 (363) <br> RNA splicing, via transesterification reactions <br> 1.654e-49","GO:0006725 (5973) <br> cellular aromatic compound metabolic process <br> 2.485e-49","GO:0034660 (535) <br> ncRNA metabolic process <br> 2.743e-49","GO:0000398 (360) <br> mRNA splicing, via spliceosome <br> 1.295e-48","GO:0000377 (360) <br> RNA splicing, via transesterification reactions with bulged adenosine as nucleophile <br> 1.295e-48","GO:0016070 (4714) <br> RNA metabolic process <br> 3.145e-47","GO:1901360 (6227) <br> organic cyclic compound metabolic process <br> 4.289e-47","GO:0010467 (6192) <br> gene expression <br> 6.034e-36","GO:0042274 (74) <br> ribosomal small subunit biogenesis <br> 9.468e-30","GO:0044085 (3377) <br> cellular component biogenesis <br> 1.060e-29","GO:0042273 (72) <br> ribosomal large subunit biogenesis <br> 2.063e-28","GO:0030490 (50) <br> maturation of SSU-rRNA <br> 2.270e-28","GO:0006807 (10148) <br> nitrogen compound metabolic process <br> 8.807e-26","GO:0044237 (10941) <br> cellular metabolic process <br> 2.843e-25","GO:0071840 (6880) <br> cellular component organization or biogenesis <br> 1.053e-24","GO:0043170 (9867) <br> macromolecule metabolic process <br> 2.254e-22","GO:0044238 (10642) <br> primary metabolic process <br> 2.803e-22","GO:0071704 (11397) <br> organic substance metabolic process <br> 1.364e-19","GO:0000470 (29) <br> maturation of LSU-rRNA <br> 2.149e-19","GO:0000462 (37) <br> maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 7.256e-19","GO:0051276 (1270) <br> chromosome organization <br> 1.092e-17","GO:0008152 (11915) <br> metabolic process <br> 8.130e-17","GO:1903311 (384) <br> regulation of mRNA metabolic process <br> 9.990e-17","GO:0022618 (196) <br> ribonucleoprotein complex assembly <br> 4.197e-15","GO:0050684 (145) <br> regulation of mRNA processing <br> 9.850e-15","GO:0071826 (203) <br> ribonucleoprotein complex subunit organization <br> 1.159e-14","GO:0006403 (222) <br> RNA localization <br> 2.825e-12","GO:0006259 (976) <br> DNA metabolic process <br> 6.999e-12","GO:0043484 (152) <br> regulation of RNA splicing <br> 3.634e-11","GO:0000460 (36) <br> maturation of 5.8S rRNA <br> 4.181e-11","GO:0006401 (431) <br> RNA catabolic process <br> 4.203e-11","GO:0006402 (388) <br> mRNA catabolic process <br> 1.622e-10","GO:0000478 (15) <br> endonucleolytic cleavage involved in rRNA processing <br> 1.721e-10","GO:0000479 (15) <br> endonucleolytic cleavage of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 1.721e-10","GO:0000463 (15) <br> maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 3.171e-10","GO:0000469 (27) <br> cleavage involved in rRNA processing <br> 3.717e-10","GO:0006412 (804) <br> translation <br> 8.740e-10","GO:0048024 (107) <br> regulation of mRNA splicing, via spliceosome <br> 9.748e-10","GO:0140014 (346) <br> mitotic nuclear division <br> 1.174e-09","GO:1903312 (95) <br> negative regulation of mRNA metabolic process <br> 1.371e-09","GO:0071103 (359) <br> DNA conformation change <br> 1.447e-09","GO:0043933 (2017) <br> protein-containing complex subunit organization <br> 1.739e-09","GO:0051028 (151) <br> mRNA transport <br> 2.008e-09","GO:0000819 (214) <br> sister chromatid segregation <br> 2.706e-09","GO:0016032 (1011) <br> viral process <br> 3.061e-09","GO:0043043 (834) <br> peptide biosynthetic process <br> 3.120e-09","GO:0043604 (975) <br> amide biosynthetic process <br> 3.713e-09","GO:0042255 (62) <br> ribosome assembly <br> 3.714e-09","GO:0006518 (998) <br> peptide metabolic process <br> 8.684e-09","GO:0050657 (183) <br> nucleic acid transport <br> 1.411e-08","GO:0050658 (183) <br> RNA transport <br> 1.411e-08","GO:0006281 (586) <br> DNA repair <br> 1.568e-08","GO:0051236 (186) <br> establishment of RNA localization <br> 1.985e-08","GO:0007059 (353) <br> chromosome segregation <br> 2.006e-08","GO:0034655 (573) <br> nucleobase-containing compound catabolic process <br> 3.063e-08","GO:0051169 (356) <br> nuclear transport <br> 8.161e-08","GO:0000280 (495) <br> nuclear division <br> 9.243e-08","GO:0000466 (24) <br> maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 1.146e-07","GO:0098813 (293) <br> nuclear chromosome segregation <br> 1.252e-07","GO:0019080 (237) <br> viral gene expression <br> 1.615e-07","GO:0000245 (59) <br> spliceosomal complex assembly <br> 2.257e-07","GO:0032392 (114) <br> DNA geometric change <br> 3.088e-07","GO:0046700 (617) <br> heterocycle catabolic process <br> 3.295e-07","GO:0006996 (4136) <br> organelle organization <br> 3.515e-07","GO:0044270 (623) <br> cellular nitrogen compound catabolic process <br> 4.396e-07","GO:0019083 (181) <br> viral transcription <br> 4.665e-07","GO:0006405 (129) <br> RNA export from nucleus <br> 5.303e-07","GO:0071426 (130) <br> ribonucleoprotein complex export from nucleus <br> 6.040e-07","GO:0043603 (1284) <br> cellular amide metabolic process <br> 6.773e-07","GO:0071166 (131) <br> ribonucleoprotein complex localization <br> 6.871e-07","GO:0007049 (1867) <br> cell cycle <br> 7.072e-07","GO:0019439 (636) <br> aromatic compound catabolic process <br> 7.669e-07","GO:0033044 (299) <br> regulation of chromosome organization <br> 8.000e-07","GO:0000027 (27) <br> ribosomal large subunit assembly <br> 8.803e-07","GO:0015931 (246) <br> nucleobase-containing compound transport <br> 1.048e-06","GO:0034622 (1150) <br> cellular protein-containing complex assembly <br> 1.230e-06","GO:0032508 (109) <br> DNA duplex unwinding <br> 1.247e-06","GO:0048285 (544) <br> organelle fission <br> 1.414e-06","GO:0006913 (353) <br> nucleocytoplasmic transport <br> 1.439e-06","GO:0000070 (185) <br> mitotic sister chromatid segregation <br> 1.548e-06","GO:0006974 (894) <br> cellular response to DNA damage stimulus <br> 2.157e-06","GO:1901361 (664) <br> organic cyclic compound catabolic process <br> 2.414e-06","GO:0033119 (27) <br> negative regulation of RNA splicing <br> 3.398e-06","GO:0000184 (123) <br> nuclear-transcribed mRNA catabolic process, nonsense-mediated decay <br> 4.018e-06","GO:0000380 (76) <br> alternative mRNA splicing, via spliceosome <br> 4.322e-06","GO:1901796 (202) <br> regulation of signal transduction by p53 class mediator <br> 5.635e-06","GO:0065003 (1724) <br> protein-containing complex assembly <br> 7.241e-06","GO:0048025 (22) <br> negative regulation of mRNA splicing, via spliceosome <br> 1.311e-05","GO:0006260 (338) <br> DNA replication <br> 1.403e-05","GO:0034645 (5081) <br> cellular macromolecule biosynthetic process <br> 1.636e-05","GO:0006413 (203) <br> translational initiation <br> 1.662e-05","GO:0009059 (5145) <br> macromolecule biosynthetic process <br> 1.932e-05","GO:0000723 (169) <br> telomere maintenance <br> 2.223e-05","GO:0044271 (5054) <br> cellular nitrogen compound biosynthetic process <br> 2.272e-05","GO:0000447 (9) <br> endonucleolytic cleavage in ITS1 to separate SSU-rRNA from 5.8S rRNA and LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 2.349e-05","GO:0051168 (202) <br> nuclear export <br> 2.439e-05","GO:0006268 (16) <br> DNA unwinding involved in DNA replication <br> 2.793e-05","GO:0006406 (112) <br> mRNA export from nucleus <br> 3.128e-05","GO:0071427 (112) <br> mRNA-containing ribonucleoprotein complex export from nucleus <br> 3.128e-05","GO:0000154 (37) <br> rRNA modification <br> 3.408e-05","GO:0006283 (72) <br> transcription-coupled nucleotide-excision repair <br> 3.455e-05","GO:0000956 (214) <br> nuclear-transcribed mRNA catabolic process <br> 3.814e-05","GO:0072331 (290) <br> signal transduction by p53 class mediator <br> 3.835e-05","GO:2000232 (15) <br> regulation of rRNA processing <br> 4.480e-05","GO:1902369 (76) <br> negative regulation of RNA catabolic process <br> 4.793e-05","GO:0031123 (157) <br> RNA 3'-end processing <br> 5.562e-05","GO:0019219 (4123) <br> regulation of nucleobase-containing compound metabolic process <br> 6.124e-05","GO:0050686 (31) <br> negative regulation of mRNA processing <br> 6.181e-05","GO:0010608 (928) <br> posttranscriptional regulation of gene expression <br> 6.268e-05","GO:0045943 (23) <br> positive regulation of transcription by RNA polymerase I <br> 6.608e-05","GO:0032200 (183) <br> telomere organization <br> 6.940e-05","GO:0010629 (1470) <br> negative regulation of gene expression <br> 7.032e-05","GO:2000234 (10) <br> positive regulation of rRNA processing <br> 7.087e-05","GO:0043489 (66) <br> RNA stabilization <br> 8.373e-05","GO:0070125 (87) <br> mitochondrial translational elongation <br> 1.033e-04","GO:0043487 (259) <br> regulation of RNA stability <br> 1.087e-04","GO:0043624 (228) <br> cellular protein complex disassembly <br> 1.097e-04","GO:0070126 (88) <br> mitochondrial translational termination <br> 1.155e-04","GO:0006356 (34) <br> regulation of transcription by RNA polymerase I <br> 1.282e-04","GO:0044265 (1244) <br> cellular macromolecule catabolic process <br> 1.300e-04","GO:0006310 (328) <br> DNA recombination <br> 1.368e-04","GO:0051252 (3847) <br> regulation of RNA metabolic process <br> 1.608e-04","GO:0022402 (1388) <br> cell cycle process <br> 1.858e-04","GO:0000278 (1098) <br> mitotic cell cycle <br> 1.942e-04","GO:0043488 (247) <br> regulation of mRNA stability <br> 2.042e-04","GO:0007088 (110) <br> regulation of mitotic nuclear division <br> 2.044e-04","GO:0016572 (38) <br> histone phosphorylation <br> 2.284e-04","GO:0071897 (201) <br> DNA biosynthetic process <br> 2.586e-04","GO:0000480 (7) <br> endonucleolytic cleavage in 5'-ETS of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 2.768e-04","GO:0032204 (82) <br> regulation of telomere maintenance <br> 2.801e-04","GO:0051052 (389) <br> regulation of DNA metabolic process <br> 2.949e-04","GO:1903047 (956) <br> mitotic cell cycle process <br> 2.980e-04","GO:0031124 (104) <br> mRNA 3'-end processing <br> 3.343e-04","GO:0061013 (255) <br> regulation of mRNA catabolic process <br> 3.585e-04","GO:1901576 (6297) <br> organic substance biosynthetic process <br> 4.141e-04","GO:0006302 (280) <br> double-strand break repair <br> 4.332e-04","GO:0006325 (839) <br> chromatin organization <br> 4.597e-04","GO:0090502 (81) <br> RNA phosphodiester bond hydrolysis, endonucleolytic <br> 5.204e-04","GO:0044249 (6210) <br> cellular biosynthetic process <br> 5.271e-04","GO:0006415 (104) <br> translational termination <br> 5.741e-04","GO:2000278 (112) <br> regulation of DNA biosynthetic process <br> 6.098e-04","GO:0006417 (504) <br> regulation of translation <br> 6.654e-04","GO:0051301 (630) <br> cell division <br> 7.018e-04","GO:0000724 (140) <br> double-strand break repair via homologous recombination <br> 7.208e-04","GO:0032543 (135) <br> mitochondrial translation <br> 7.662e-04","GO:0051983 (86) <br> regulation of chromosome segregation <br> 8.433e-04","GO:0000725 (142) <br> recombinational repair <br> 8.504e-04","GO:0051726 (1120) <br> regulation of cell cycle <br> 9.668e-04","GO:0009058 (6391) <br> biosynthetic process <br> 1.167e-03","GO:2001251 (92) <br> negative regulation of chromosome organization <br> 1.244e-03","GO:2001252 (192) <br> positive regulation of chromosome organization <br> 1.253e-03","GO:0006414 (143) <br> translational elongation <br> 1.370e-03","GO:0050685 (35) <br> positive regulation of mRNA processing <br> 1.411e-03","GO:0016043 (6661) <br> cellular component organization <br> 1.640e-03","GO:0048026 (25) <br> positive regulation of mRNA splicing, via spliceosome <br> 1.654e-03","GO:0048255 (57) <br> mRNA stabilization <br> 1.706e-03","GO:0045934 (1569) <br> negative regulation of nucleobase-containing compound metabolic process <br> 1.771e-03","GO:1902299 (8) <br> pre-replicative complex assembly involved in cell cycle DNA replication <br> 1.854e-03","GO:0006267 (8) <br> pre-replicative complex assembly involved in nuclear cell cycle DNA replication <br> 1.854e-03","GO:0009057 (1479) <br> macromolecule catabolic process <br> 2.091e-03","GO:0006289 (107) <br> nucleotide-excision repair <br> 2.201e-03","GO:0070934 (5) <br> CRD-mediated mRNA stabilization <br> 2.262e-03","GO:0006354 (104) <br> DNA-templated transcription, elongation <br> 2.385e-03","GO:0010833 (87) <br> telomere maintenance via telomere lengthening <br> 2.412e-03","GO:0031330 (264) <br> negative regulation of cellular catabolic process <br> 2.453e-03","GO:0051783 (138) <br> regulation of nuclear division <br> 2.463e-03","GO:0045815 (58) <br> positive regulation of gene expression, epigenetic <br> 2.533e-03","GO:0002181 (149) <br> cytoplasmic translation <br> 2.679e-03","GO:0090307 (66) <br> mitotic spindle assembly <br> 2.937e-03","GO:0000967 (5) <br> rRNA 5'-end processing <br> 3.096e-03","GO:0000472 (5) <br> endonucleolytic cleavage to generate mature 5'-end of SSU-rRNA from (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 3.096e-03","GO:0051988 (14) <br> regulation of attachment of spindle microtubules to kinetochore <br> 3.155e-03","GO:0090305 (306) <br> nucleic acid phosphodiester bond hydrolysis <br> 3.253e-03","GO:0032984 (337) <br> protein-containing complex disassembly <br> 3.267e-03","GO:0033120 (39) <br> positive regulation of RNA splicing <br> 3.427e-03","GO:1901566 (1889) <br> organonitrogen compound biosynthetic process <br> 3.535e-03","GO:0007062 (56) <br> sister chromatid cohesion <br> 3.671e-03","GO:0006323 (246) <br> DNA packaging <br> 3.754e-03","GO:0010468 (5218) <br> regulation of gene expression <br> 3.796e-03","GO:1904356 (63) <br> regulation of telomere maintenance via telomere lengthening <br> 4.000e-03","GO:0034248 (577) <br> regulation of cellular amide metabolic process <br> 4.470e-03","GO:0031145 (87) <br> anaphase-promoting complex-dependent catabolic process <br> 4.663e-03","GO:1902373 (64) <br> negative regulation of mRNA catabolic process <br> 5.184e-03","GO:0009895 (324) <br> negative regulation of catabolic process <br> 5.355e-03","GO:0051310 (65) <br> metaphase plate congression <br> 5.439e-03","GO:0045787 (317) <br> positive regulation of cell cycle <br> 5.863e-03","GO:0033979 (3) <br> box H/ACA RNA metabolic process <br> 5.904e-03","GO:0007004 (75) <br> telomere maintenance via telomerase <br> 5.950e-03","GO:0140053 (166) <br> mitochondrial gene expression <br> 6.010e-03","GO:0010564 (815) <br> regulation of cell cycle process <br> 6.089e-03","GO:0090068 (237) <br> positive regulation of cell cycle process <br> 6.298e-03","GO:0033045 (70) <br> regulation of sister chromatid segregation <br> 6.624e-03","GO:0008608 (36) <br> attachment of spindle microtubules to kinetochore <br> 7.240e-03","GO:0006278 (77) <br> RNA-dependent DNA biosynthetic process <br> 7.441e-03","GO:0032206 (52) <br> positive regulation of telomere maintenance <br> 7.730e-03","GO:0006360 (68) <br> transcription by RNA polymerase I <br> 8.059e-03","GO:0006614 (96) <br> SRP-dependent cotranslational protein targeting to membrane <br> 9.923e-03","GO:0110095 (8) <br> cellular detoxification of aldehyde <br> 9.935e-03","GO:0051253 (1452) <br> negative regulation of RNA metabolic process <br> 1.028e-02","GO:0007346 (612) <br> regulation of mitotic cell cycle <br> 1.060e-02","GO:0030071 (59) <br> regulation of mitotic metaphase/anaphase transition <br> 1.217e-02","GO:0000727 (12) <br> double-strand break repair via break-induced replication <br> 1.271e-02","GO:1903241 (3) <br> U2-type prespliceosome assembly <br> 1.487e-02","GO:0007080 (50) <br> mitotic metaphase plate congression <br> 1.591e-02","GO:0032210 (54) <br> regulation of telomere maintenance via telomerase <br> 1.606e-02","GO:0007091 (61) <br> metaphase/anaphase transition of mitotic cell cycle <br> 1.618e-02","GO:0051304 (95) <br> chromosome separation <br> 1.711e-02","GO:0006613 (102) <br> cotranslational protein targeting to membrane <br> 1.720e-02","GO:1902099 (62) <br> regulation of metaphase/anaphase transition of cell cycle <br> 1.857e-02","GO:0022607 (3104) <br> cellular component assembly <br> 1.858e-02","GO:0000075 (196) <br> cell cycle checkpoint signaling <br> 1.961e-02","GO:0033260 (66) <br> nuclear DNA replication <br> 2.102e-02","GO:2001242 (167) <br> regulation of intrinsic apoptotic signaling pathway <br> 2.105e-02","GO:0010965 (63) <br> regulation of mitotic sister chromatid separation <br> 2.126e-02","GO:0000381 (60) <br> regulation of alternative mRNA splicing, via spliceosome <br> 2.231e-02","GO:0110096 (12) <br> cellular response to aldehyde <br> 2.340e-02","GO:0044784 (64) <br> metaphase/anaphase transition of cell cycle <br> 2.427e-02","GO:0006270 (49) <br> DNA replication initiation <br> 2.609e-02","GO:0051303 (82) <br> establishment of chromosome localization <br> 2.612e-02","GO:0090501 (157) <br> RNA phosphodiester bond hydrolysis <br> 2.644e-02","GO:0051987 (6) <br> positive regulation of attachment of spindle microtubules to kinetochore <br> 2.682e-02","GO:0051306 (65) <br> mitotic sister chromatid separation <br> 2.764e-02","GO:0046185 (13) <br> aldehyde catabolic process <br> 2.765e-02","GO:0000387 (40) <br> spliceosomal snRNP assembly <br> 2.824e-02","GO:0031503 (289) <br> protein-containing complex localization <br> 3.011e-02","GO:0050000 (84) <br> chromosome localization <br> 3.065e-02","GO:1903313 (144) <br> positive regulation of mRNA metabolic process <br> 3.113e-02","GO:0031167 (25) <br> rRNA methylation <br> 3.117e-02","GO:0010639 (361) <br> negative regulation of organelle organization <br> 3.154e-02","GO:0033365 (974) <br> protein localization to organelle <br> 3.193e-02","GO:0045839 (48) <br> negative regulation of mitotic nuclear division <br> 3.203e-02","GO:0044786 (70) <br> cell cycle DNA replication <br> 3.276e-02","GO:0046907 (1723) <br> intracellular transport <br> 3.655e-02","GO:0045047 (111) <br> protein targeting to ER <br> 3.662e-02","GO:0010948 (400) <br> negative regulation of cell cycle process <br> 3.843e-02","GO:0034502 (94) <br> protein localization to chromosome <br> 4.023e-02","GO:0033554 (2142) <br> cellular response to stress <br> 4.059e-02","GO:0030261 (67) <br> chromosome condensation <br> 4.329e-02","GO:0050434 (24) <br> positive regulation of viral transcription <br> 4.485e-02","GO:0050792 (193) <br> regulation of viral process <br> 4.524e-02","GO:0034501 (19) <br> protein localization to kinetochore <br> 4.643e-02","GO:1903083 (19) <br> protein localization to condensed chromosome <br> 4.643e-02","GO:0075713 (8) <br> establishment of integrated proviral latency <br> 4.899e-02"],"key":["GO:0006396","GO:0022613","GO:0042254","GO:0006364","GO:0016072","GO:0034470","GO:0034641","GO:0006139","GO:0090304","GO:0016071","GO:0008380","GO:0006397","GO:0046483","GO:0000375","GO:0006725","GO:0034660","GO:0000398","GO:0000377","GO:0016070","GO:1901360","GO:0010467","GO:0042274","GO:0044085","GO:0042273","GO:0030490","GO:0006807","GO:0044237","GO:0071840","GO:0043170","GO:0044238","GO:0071704","GO:0000470","GO:0000462","GO:0051276","GO:0008152","GO:1903311","GO:0022618","GO:0050684","GO:0071826","GO:0006403","GO:0006259","GO:0043484","GO:0000460","GO:0006401","GO:0006402","GO:0000478","GO:0000479","GO:0000463","GO:0000469","GO:0006412","GO:0048024","GO:0140014","GO:1903312","GO:0071103","GO:0043933","GO:0051028","GO:0000819","GO:0016032","GO:0043043","GO:0043604","GO:0042255","GO:0006518","GO:0050657","GO:0050658","GO:0006281","GO:0051236","GO:0007059","GO:0034655","GO:0051169","GO:0000280","GO:0000466","GO:0098813","GO:0019080","GO:0000245","GO:0032392","GO:0046700","GO:0006996","GO:0044270","GO:0019083","GO:0006405","GO:0071426","GO:0043603","GO:0071166","GO:0007049","GO:0019439","GO:0033044","GO:0000027","GO:0015931","GO:0034622","GO:0032508","GO:0048285","GO:0006913","GO:0000070","GO:0006974","GO:1901361","GO:0033119","GO:0000184","GO:0000380","GO:1901796","GO:0065003","GO:0048025","GO:0006260","GO:0034645","GO:0006413","GO:0009059","GO:0000723","GO:0044271","GO:0000447","GO:0051168","GO:0006268","GO:0006406","GO:0071427","GO:0000154","GO:0006283","GO:0000956","GO:0072331","GO:2000232","GO:1902369","GO:0031123","GO:0019219","GO:0050686","GO:0010608","GO:0045943","GO:0032200","GO:0010629","GO:2000234","GO:0043489","GO:0070125","GO:0043487","GO:0043624","GO:0070126","GO:0006356","GO:0044265","GO:0006310","GO:0051252","GO:0022402","GO:0000278","GO:0043488","GO:0007088","GO:0016572","GO:0071897","GO:0000480","GO:0032204","GO:0051052","GO:1903047","GO:0031124","GO:0061013","GO:1901576","GO:0006302","GO:0006325","GO:0090502","GO:0044249","GO:0006415","GO:2000278","GO:0006417","GO:0051301","GO:0000724","GO:0032543","GO:0051983","GO:0000725","GO:0051726","GO:0009058","GO:2001251","GO:2001252","GO:0006414","GO:0050685","GO:0016043","GO:0048026","GO:0048255","GO:0045934","GO:1902299","GO:0006267","GO:0009057","GO:0006289","GO:0070934","GO:0006354","GO:0010833","GO:0031330","GO:0051783","GO:0045815","GO:0002181","GO:0090307","GO:0000967","GO:0000472","GO:0051988","GO:0090305","GO:0032984","GO:0033120","GO:1901566","GO:0007062","GO:0006323","GO:0010468","GO:1904356","GO:0034248","GO:0031145","GO:1902373","GO:0009895","GO:0051310","GO:0045787","GO:0033979","GO:0007004","GO:0140053","GO:0010564","GO:0090068","GO:0033045","GO:0008608","GO:0006278","GO:0032206","GO:0006360","GO:0006614","GO:0110095","GO:0051253","GO:0007346","GO:0030071","GO:0000727","GO:1903241","GO:0007080","GO:0032210","GO:0007091","GO:0051304","GO:0006613","GO:1902099","GO:0022607","GO:0000075","GO:0033260","GO:2001242","GO:0010965","GO:0000381","GO:0110096","GO:0044784","GO:0006270","GO:0051303","GO:0090501","GO:0051987","GO:0051306","GO:0046185","GO:0000387","GO:0031503","GO:0050000","GO:1903313","GO:0031167","GO:0010639","GO:0033365","GO:0045839","GO:0044786","GO:0046907","GO:0045047","GO:0010948","GO:0034502","GO:0033554","GO:0030261","GO:0050434","GO:0050792","GO:0034501","GO:1903083","GO:0075713"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[19.4503282144806,18.4643827493734,17.9934386332075,17.6643246110046,17.7168500269786,18.3638334737416,21.6537323834277,21.5053658734699,21.4000311246093,19.2759171540558,18.4239673394241,18.6214455917298,21.5390620738563,18.1044082182465,21.5472727056056,18.6289175161382,18.0929757080159,18.0929757080159,21.2824953778867,21.5934484800124,21.5872057726815,15.7151677983179,20.9024525251713,15.6697988614665,15.0486221338682,22.1262884949501,22.2070086720384,21.7035374999992,22.0960714860976,22.1773231312919,22.2506677513959,14.0509285588584,14.5086078145015,19.73606939506,22.298075013352,18.181645451538,17.2289935746564,16.7794477421486,17.2803636129636,17.4104643922402,19.4071640563441,16.8507929922937,14.4581139285914,18.3389367682336,18.195832112632,12.6954028668369,12.6954028668369,12.6954028668369,13.9127623976552,19.1605106996624,16.309816295931,18.0382058611615,16.121131665261,18.0891416461647,20.2980535736863,16.840827913987,17.3572475673179,19.451573873278,19.2074226898582,19.4058700354974,15.4190791173213,19.4352755293199,17.1279572175344,17.1279572175344,18.7493854888517,17.1519653138816,18.0658885682371,18.7197942104249,18.0775688230189,18.5253260577865,13.6808214449962,17.8062984506393,17.5047579060072,15.3347186146137,16.4091839862407,18.8171599959616,21.1344462088703,18.8298494843551,17.1117076782815,16.6008025798044,16.6126778615783,19.7496169904031,16.624451291171,20.2055094409634,18.8568934935132,17.8347771324352,13.9127623976552,17.5582210891217,19.6128903566504,16.3389366763927,18.6510623608111,18.0658885682371,17.1440106914419,19.2959963103746,18.9131562034715,13.9127623976552,16.5273136649716,15.7591617451593,17.2731463069687,20.1095332573674,13.5059251363754,18.0058060899366,21.3667866523318,17.2803636129636,21.3808206783456,17.0098206791988,21.3608095525535,11.4827422948031,17.2731463069687,12.8370677538266,16.3815087539172,16.3815087539172,14.5086078145015,15.6697988614665,17.3572475673179,17.7918175061017,12.6954028668369,15.7591617451593,16.8995454455618,21.1308676162256,14.1782205386673,19.3433767765611,13.5956642813554,17.1279572175344,19.915852066474,11.747980750738,15.5245580341656,15.9797164726717,17.631749719545,17.4490032718266,15.9981813072674,14.3519997480295,19.710477230295,17.9641020663474,21.0519171992385,19.8455306846866,19.5551369029075,17.5640277699944,16.3532722717548,14.5575278577922,17.265889291748,10.8097872173079,15.8836369590203,18.1993537615243,19.3810073827861,16.2649666785139,17.6095673525824,21.6058228470476,17.7423292661052,19.2150634219582,15.8636236908885,21.5904209757188,16.2649666785139,16.3815087539172,18.5494106764228,18.8444870340137,16.7260902965966,16.6705591144528,15.9610096906275,16.747684911647,19.5799237913336,21.622211971863,16.0697362528688,17.1987173630513,16.7583551431596,14.405947329161,21.6679004416094,13.7618192670139,15.2756954716879,19.9953323018734,11.1749781597015,11.1749781597015,19.9233120507587,16.309816295931,9.79200637188604,16.2649666785139,15.9797164726717,17.6589522750826,16.7041480940405,15.3054992444299,16.820675004284,15.5245580341656,9.79200637188604,9.79200637188604,12.5414277899272,17.8672182940543,18.0016970365285,14.6049645781774,20.2195718163645,15.2452846314144,17.5582210891217,21.3966032993525,15.4461643032416,18.7289763933658,15.9797164726717,15.4727616697066,17.9470280058521,15.498887690643,17.9165837938067,7.7791008157754,15.7373316539529,16.9830886832557,19.1779286245093,17.5047579060072,15.6229705518591,14.4581139285914,15.7806675338474,15.1170689685676,15.5745906356112,16.1378571469232,11.1749781597015,19.9007835736055,18.8064825080221,15.3347186146137,12.187219027138,7.7791008157754,15.0486221338682,15.1825444005545,15.3914887766185,16.121131665261,16.2342495236679,15.4190791173213,20.8050634202662,17.2289935746564,15.5245580341656,16.9920586807013,15.4461643032416,15.3633750239743,12.187219027138,15.4727616697066,15.0132022257977,15.8836369590203,16.8995454455618,10.3630624020686,15.498887690643,12.3729643352122,14.651001049555,17.7869538601274,15.9228455355486,16.7689423433419,13.7618192670139,18.0967981119871,19.4045745794367,14.976936429373,15.6229705518591,20.1088323582115,16.3674620265117,18.2374501939285,16.1042061318618,20.3697031428378,15.5497876137808,13.6808214449962,17.206351575782,13.2039079443259,13.2039079443259,11.1749781597015],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[68.5696635609685,59.898598782194,57.3138099746564,62.1001776211723,57.3363517375128,53.0534167947906,53.3013761862114,56.1529091875501,52.9895484666974,66.7625655719778,59.8910848612419,59.894841821718,59.8798139798137,57.7796730736894,53.1398268857403,59.8835709402897,62.9041671630517,59.8760570193376,59.8723000588615,57.3814352632257,52.9482219014606,53.1510977671685,62.8966532420996,56.1604231085023,62.8741114792432,55.258752594245,64.6473968239492,56.0740130175526,56.1641800689783,66.458251773416,55.2700234756732,66.9128439910207,53.1586116881206,65.4513863658286,53.1886673719292,62.8816254001953,60.6499908774085,56.1679370294544,53.3088901071635,53.6845861547707,52.0390374662511,53.3990571585892,52.2719690157676,55.0370919261567,52.2982677391001,53.6132039057254,58.3056475403395,56.878002559432,58.7639967184202,55.1422868194867,52.2606981343394,61.2736463164364,64.6511537844253,56.0589851756483,53.1698825695489,62.8778684397192,62.9004102025757,53.1548547276446,58.3018905798634,53.4065710795414,52.1179336362487,53.33143187002,55.6945600094693,65.5152546939219,53.1999382533574,63.7457263096919,55.0145501633003,54.2819428704662,59.6393685093451,52.5049005652841,53.1623686485967,61.4877930635725,53.4140850004935,52.1216905967247,55.6983169699454,53.1435838462163,55.5668233532829,53.2675635419267,63.2422936058982,60.0601480826651,62.8327849140064,52.294510778624,56.6525849308677,59.5078748926825,64.1289362782512,52.3020246995762,62.8703545187671,62.8928962816235,62.554769838777,57.2912682118,57.3175669351325,58.5536069317602,55.0295780052046,60.0789328850455,63.7344554282637,56.8817595199081,57.7871869946415,61.7507802968976,53.0646876762188,56.250590159928,54.1880188585644,53.075958557647,53.3464597119242,52.3997056719541,53.0872294390753,58.5573638922363,64.4633057606217,55.0333349656806,58.5611208527123,57.2837542908478,52.5086575257602,51.978926098634,62.186587712122,53.5906621428689],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,14.986271035308,14.5945769363538,14.4074123862089,12.1988686727867,11.041424376302,10.8888304648531,10.4317811867174,10.4235237509272,9.90318818089537,9.62449923206817,9.55897060354972,8.49094412374271,8.34709779219887,7.6415034055226,7.46432807697708,7.46406764464861,6.17565010324709,6.05133281136729,5.94439751895962,5.92613699610068,5.92613699610068,5.31599422222393,5.11654590651616,5.1033249421265,5.1033249421265,5.08948809314869,5.06747146759192,5.04960612868297,5.04960612868297,4.83219001696907,4.51898423685624,4.36442037917466,4.34286401202202,4.3011487854968,3.9743139425071,3.94268107478118,3.94268107478118,3.93116322798929,3.84246579605153,3.81912904070563,3.81642058365076,3.81642058365076,3.80922548247665,3.77150973790905,3.53958276742672,3.44893873594668,3.26884018825035,3.20099384954836,3.18307369902085,3.00911634988012,2.90299081954665,2.90299081954665,2.82671457012823,2.80559944049508,2.64247192389544,2.64247192389544,2.50030112594974,2.49299771460761,2.49299771460761,2.38724645205463,2.38292317727373,2.3314903545139,2.32347233885153,2.32347233885153,2.27314348629076,2.2486254676521,2.19015712150653,2.13404415437091,2.11556247733378,2.01549631719274,1.95220514227877,1.94995276213306,1.93743160289264,1.82404705690688,1.78002308368751,1.60102081133115,1.4377265322396,1.4236499075728,1.40752473775624,1.35749152196307,1.3192797584104,1.30103229423037],"text":["GO:1990904 (694) <br> ribonucleoprotein complex <br> 1.375e-101","GO:0043233 (5579) <br> organelle lumen <br> 1.155e-93","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 1.155e-93","GO:0070013 (5579) <br> intracellular organelle lumen <br> 1.155e-93","GO:0031981 (4453) <br> nuclear lumen <br> 2.149e-85","GO:0005654 (4112) <br> nucleoplasm <br> 3.680e-71","GO:0005730 (955) <br> nucleolus <br> 4.893e-66","GO:0030684 (79) <br> preribosome <br> 3.064e-63","GO:0005634 (7607) <br> nucleus <br> 3.679e-62","GO:0140513 (1236) <br> nuclear protein-containing complex <br> 1.195e-51","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 4.601e-48","GO:0043232 (5094) <br> intracellular non-membrane-bounded organelle <br> 1.787e-45","GO:0043228 (5102) <br> non-membrane-bounded organelle <br> 2.441e-45","GO:0032991 (5541) <br> protein-containing complex <br> 7.970e-45","GO:0005681 (192) <br> spliceosomal complex <br> 1.514e-41","GO:0043229 (13117) <br> intracellular organelle <br> 1.896e-40","GO:0071013 (85) <br> catalytic step 2 spliceosome <br> 1.524e-38","GO:0043227 (13201) <br> membrane-bounded organelle <br> 1.791e-38","GO:0043226 (14027) <br> organelle <br> 1.264e-33","GO:0032040 (39) <br> small-subunit processome <br> 6.117e-32","GO:0005622 (14808) <br> intracellular anatomical structure <br> 6.346e-31","GO:0005684 (93) <br> U2-type spliceosomal complex <br> 2.040e-28","GO:0071011 (52) <br> precatalytic spliceosome <br> 8.396e-26","GO:0030686 (28) <br> 90S preribosome <br> 2.345e-25","GO:0071005 (50) <br> U2-type precatalytic spliceosome <br> 1.260e-24","GO:0016604 (816) <br> nuclear body <br> 7.678e-23","GO:0097525 (61) <br> spliceosomal snRNP complex <br> 2.611e-22","GO:0030532 (70) <br> small nuclear ribonucleoprotein complex <br> 1.076e-20","GO:0030687 (24) <br> preribosome, large subunit precursor <br> 3.498e-19","GO:0120114 (82) <br> Sm-like protein family complex <br> 6.544e-19","GO:0016607 (413) <br> nuclear speck <br> 9.714e-19","GO:1902494 (1377) <br> catalytic complex <br> 1.426e-17","GO:0005686 (21) <br> U2 snRNP <br> 1.032e-15","GO:0098687 (348) <br> chromosomal region <br> 2.543e-15","GO:0005694 (1795) <br> chromosome <br> 3.914e-15","GO:0071007 (30) <br> U2-type catalytic step 2 spliceosome <br> 6.326e-13","GO:0044391 (191) <br> ribosomal subunit <br> 9.090e-12","GO:0030688 (15) <br> preribosome, small subunit precursor <br> 1.292e-11","GO:0005732 (24) <br> sno(s)RNA-containing ribonucleoprotein complex <br> 3.700e-11","GO:0005840 (247) <br> ribosome <br> 3.771e-11","GO:0000228 (237) <br> nuclear chromosome <br> 1.250e-10","GO:0005759 (482) <br> mitochondrial matrix <br> 2.374e-10","GO:0000781 (160) <br> chromosome, telomeric region <br> 2.761e-10","GO:0015935 (76) <br> small ribosomal subunit <br> 3.229e-09","GO:0000793 (239) <br> condensed chromosome <br> 4.497e-09","GO:0005819 (390) <br> spindle <br> 2.283e-08","GO:0034399 (132) <br> nuclear periphery <br> 3.433e-08","GO:0031428 (7) <br> box C/D RNP complex <br> 3.435e-08","GO:0035145 (20) <br> exon-exon junction complex <br> 6.673e-07","GO:0016363 (110) <br> nuclear matrix <br> 8.885e-07","GO:0000775 (200) <br> chromosome, centromeric region <br> 1.137e-06","GO:0046540 (32) <br> U4/U6 x U5 tri-snRNP complex <br> 1.185e-06","GO:0097526 (32) <br> spliceosomal tri-snRNP complex <br> 1.185e-06","GO:0030496 (197) <br> midbody <br> 4.831e-06","GO:0005689 (28) <br> U12-type spliceosomal complex <br> 7.646e-06","GO:0071006 (12) <br> U2-type catalytic step 1 spliceosome <br> 7.883e-06","GO:0071012 (12) <br> catalytic step 1 spliceosome <br> 7.883e-06","GO:0005685 (19) <br> U1 snRNP <br> 8.138e-06","GO:0034388 (6) <br> Pwp2p-containing subcomplex of 90S preribosome <br> 8.561e-06","GO:0005761 (88) <br> mitochondrial ribosome <br> 8.921e-06","GO:0000313 (88) <br> organellar ribosome <br> 8.921e-06","GO:0005739 (1678) <br> mitochondrion <br> 1.472e-05","GO:0022626 (106) <br> cytosolic ribosome <br> 3.027e-05","GO:0098798 (272) <br> mitochondrial protein-containing complex <br> 4.321e-05","GO:0005697 (21) <br> telomerase holoenzyme complex <br> 4.541e-05","GO:0072686 (161) <br> mitotic spindle <br> 4.999e-05","GO:0015030 (57) <br> Cajal body <br> 1.061e-04","GO:0009295 (45) <br> nucleoid <br> 1.141e-04","GO:0042645 (45) <br> mitochondrial nucleoid <br> 1.141e-04","GO:0001650 (140) <br> fibrillar center <br> 1.172e-04","GO:0005687 (10) <br> U4 snRNP <br> 1.437e-04","GO:0051233 (36) <br> spindle midzone <br> 1.517e-04","GO:0005763 (29) <br> mitochondrial small ribosomal subunit <br> 1.526e-04","GO:0000314 (29) <br> organellar small ribosomal subunit <br> 1.526e-04","GO:0022627 (46) <br> cytosolic small ribosomal subunit <br> 1.552e-04","GO:0005682 (18) <br> U5 snRNP <br> 1.692e-04","GO:0019866 (564) <br> organelle inner membrane <br> 2.887e-04","GO:0005721 (23) <br> pericentric heterochromatin <br> 3.557e-04","GO:0071162 (10) <br> CMG complex <br> 5.385e-04","GO:0043596 (35) <br> nuclear replication fork <br> 6.295e-04","GO:0070937 (6) <br> CRD-mediated mRNA stability complex <br> 6.560e-04","GO:0000792 (76) <br> heterochromatin <br> 9.792e-04","GO:0031261 (12) <br> DNA replication preinitiation complex <br> 1.250e-03","GO:0042555 (12) <br> MCM complex <br> 1.250e-03","GO:0097125 (2) <br> cyclin B1-CDK1 complex <br> 1.490e-03","GO:0000794 (73) <br> condensed nuclear chromosome <br> 1.565e-03","GO:0071004 (16) <br> U2-type prespliceosome <br> 2.278e-03","GO:0071010 (16) <br> prespliceosome <br> 2.278e-03","GO:0070545 (3) <br> PeBoW complex <br> 3.160e-03","GO:0031967 (1244) <br> organelle envelope <br> 3.214e-03","GO:0031975 (1244) <br> envelope <br> 3.214e-03","GO:0034708 (101) <br> methyltransferase complex <br> 4.100e-03","GO:0015630 (1313) <br> microtubule cytoskeleton <br> 4.141e-03","GO:0043601 (22) <br> nuclear replisome <br> 4.661e-03","GO:0072588 (5) <br> box H/ACA RNP complex <br> 4.748e-03","GO:0031429 (5) <br> box H/ACA snoRNP complex <br> 4.748e-03","GO:0032993 (212) <br> protein-DNA complex <br> 5.332e-03","GO:0061574 (4) <br> ASAP complex <br> 5.641e-03","GO:0005657 (67) <br> replication fork <br> 6.454e-03","GO:0030894 (24) <br> replisome <br> 7.344e-03","GO:0008278 (10) <br> cohesin complex <br> 7.664e-03","GO:0005662 (13) <br> DNA replication factor A complex <br> 9.649e-03","GO:0005743 (502) <br> mitochondrial inner membrane <br> 1.116e-02","GO:0000922 (168) <br> spindle pole <br> 1.122e-02","GO:0005665 (15) <br> RNA polymerase II, core complex <br> 1.155e-02","GO:0034709 (12) <br> methylosome <br> 1.500e-02","GO:0097431 (36) <br> mitotic spindle pole <br> 1.659e-02","GO:0015934 (119) <br> large ribosomal subunit <br> 2.506e-02","GO:0034715 (6) <br> pICln-Sm protein complex <br> 3.650e-02","GO:0031965 (301) <br> nuclear membrane <br> 3.770e-02","GO:0001651 (2) <br> dense fibrillar component <br> 3.913e-02","GO:0000152 (45) <br> nuclear ubiquitin ligase complex <br> 4.390e-02","GO:0070062 (2178) <br> extracellular exosome <br> 4.794e-02","GO:0005813 (620) <br> centrosome <br> 5.000e-02"],"key":["GO:1990904","GO:0043233","GO:0031974","GO:0070013","GO:0031981","GO:0005654","GO:0005730","GO:0030684","GO:0005634","GO:0140513","GO:0043231","GO:0043232","GO:0043228","GO:0032991","GO:0005681","GO:0043229","GO:0071013","GO:0043227","GO:0043226","GO:0032040","GO:0005622","GO:0005684","GO:0071011","GO:0030686","GO:0071005","GO:0016604","GO:0097525","GO:0030532","GO:0030687","GO:0120114","GO:0016607","GO:1902494","GO:0005686","GO:0098687","GO:0005694","GO:0071007","GO:0044391","GO:0030688","GO:0005732","GO:0005840","GO:0000228","GO:0005759","GO:0000781","GO:0015935","GO:0000793","GO:0005819","GO:0034399","GO:0031428","GO:0035145","GO:0016363","GO:0000775","GO:0046540","GO:0097526","GO:0030496","GO:0005689","GO:0071006","GO:0071012","GO:0005685","GO:0034388","GO:0005761","GO:0000313","GO:0005739","GO:0022626","GO:0098798","GO:0005697","GO:0072686","GO:0015030","GO:0009295","GO:0042645","GO:0001650","GO:0005687","GO:0051233","GO:0005763","GO:0000314","GO:0022627","GO:0005682","GO:0019866","GO:0005721","GO:0071162","GO:0043596","GO:0070937","GO:0000792","GO:0031261","GO:0042555","GO:0097125","GO:0000794","GO:0071004","GO:0071010","GO:0070545","GO:0031967","GO:0031975","GO:0034708","GO:0015630","GO:0043601","GO:0072588","GO:0031429","GO:0032993","GO:0061574","GO:0005657","GO:0030894","GO:0008278","GO:0005662","GO:0005743","GO:0000922","GO:0005665","GO:0034709","GO:0097431","GO:0015934","GO:0034715","GO:0031965","GO:0001651","GO:0000152","GO:0070062","GO:0005813"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[18.9706470582724,21.4713489077623,21.4713489077623,21.4713489077623,21.2181834681373,21.1278301730825,19.3796840599657,15.8227421236023,21.8137488730278,19.7024866025059,22.3097189832527,21.3696524759062,21.3714121953266,21.4637267954854,17.1987173630513,22.4001723500672,15.9420548318458,22.4069336180039,22.4710947059109,14.6049645781774,22.5281903038104,16.0870759222918,15.1170689685676,13.9833131530912,15.0486221338682,19.17949942902,15.3914887766185,15.6229705518591,13.6808214449962,15.8836369590203,18.2810218264132,19.835758359278,13.4111152730112,18.0461774501309,20.1582119259402,14.1158314099108,17.1910389050914,12.6954028668369,13.6808214449962,17.5640277699944,17.5047579060072,18.4896805188233,16.9279733014706,15.7591617451593,17.5168301728758,18.2028655123974,16.6361245292714,10.8097872173079,13.3106655321065,16.3532722717548,17.2585921453936,14.2382740983495,14.2382740983495,17.2364555998064,13.9833131530912,12.187219027138,12.187219027138,13.2039079443259,10.3630624020686,15.9981813072674,15.9981813072674,20.076831390311,16.2950254596216,17.7013163294157,13.4111152730112,16.9373174877114,15.2756954716879,14.862657240381,14.862657240381,16.7260902965966,11.747980750738,14.4581139285914,14.0509285588584,14.0509285588584,14.9017074238205,13.0900518224006,18.6988766650139,13.5956642813554,11.747980750738,14.405947329161,10.3630624020686,15.7591617451593,12.187219027138,12.187219027138,3.77952755905512,15.692660304218,12.8370677538266,12.8370677538266,7.7791008157754,19.710477230295,19.710477230295,16.2186353293393,19.7771805912925,13.5059251363754,9.79200637188604,9.79200637188604,17.3435989423967,9.00890082870231,15.5497876137808,13.6808214449962,11.747980750738,12.3729643352122,18.5440992955177,17.0009690943617,12.6954028668369,12.187219027138,14.4581139285914,16.4760514553463,10.3630624020686,17.8441305674686,3.77952755905512,14.862657240381,20.3895089474417,18.8235214283333],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[3.64155719716214,3.49505666926895,40.706190754139,46.8817514684057,3.64531362095428,5.77144948730158,20.2787581725194,9.36259063258078,46.2469158475352,16.4659880235044,46.2506722713273,3.66033931612281,2.12771840893252,46.8704821970293,3.68287785887561,25.8194832659157,9.08837169575506,22.3072270202713,16.8679253692626,22.299714172687,22.3222527154398,20.3388609531935,25.9960351841459,27.4460147679093,19.2607673248514,3.57018514511161,24.4108243438658,40.8301527392793,16.8491432503019,3.51383878822962,27.3746427158588,3.91953255777999,9.05456388162587,33.45253641153,8.35962548008125,25.5114565149608,3.65658289233068,2.77006687738728,2.76631045359514,16.1128841870439,3.50256951685322,3.64907004474641,26.9727053701005,3.53637733098242,2.74001548705021,8.99070467715961,47.7457289405963,19.3434086482783,10.5946976364004,46.2581851189116,38.873055943578,20.4853614810867,40.9916789623411,26.8149355708309,24.41833719145,3.66409573991494,27.4347454965329,4.1599436804765,27.4009376824037,24.1253361356637,40.9841661147568,16.9655923878581,38.985748657342,41.1907694233241,38.392233698185,28.7757887903244,25.9547145224325,24.3845293773208,13.5885673987305],"y":[17,17,17,17,17,17,17,17,17,15.4705042817354,13.2243869563033,11.0100729277942,10.2243446624929,10.2156308440163,9.78724560115699,9.74993279571527,9.58485731976256,9.10379385789054,8.9716349448872,8.9541249432066,8.93530998127792,8.80785690908813,8.57428774664203,7.79784873294528,7.57080556722909,7.20088954699733,6.87025120791112,6.74590057963339,6.55741150336081,6.4629062409358,5.47238167415237,5.29358362898048,5.23171181722633,4.80408917689116,4.4714680449129,4.3115517185103,4.24767899689518,3.67007782952898,3.67007782952898,3.45898842961807,3.44929173066438,3.38009206115563,3.21660887927229,2.93738148299292,2.82997242132262,2.82407308083524,2.76426109151052,2.65203040344854,2.52517213919466,2.52517213919466,2.50652101800985,2.47439966799257,2.34308123288986,2.23759417914007,2.14035328871977,2.12054331423481,2.04818486479786,1.98387613270511,1.98095775912442,1.86776887757699,1.86776887757699,1.84300140015952,1.7650270259613,1.5558060008803,1.45938618092619,1.37327444832493,1.34783718198768,1.34472496585717,1.34080292964132],"text":["GO:0003723 (1951) <br> RNA binding <br> 9.505e-125","GO:0003676 (4252) <br> nucleic acid binding <br> 5.017e-75","GO:0097159 (6291) <br> organic cyclic compound binding <br> 3.641e-60","GO:1901363 (6210) <br> heterocyclic compound binding <br> 2.229e-59","GO:0003724 (76) <br> RNA helicase activity <br> 1.080e-21","GO:0004386 (163) <br> helicase activity <br> 2.725e-20","GO:0030515 (32) <br> snoRNA binding <br> 1.950e-19","GO:0008094 (185) <br> ATPase, acting on DNA <br> 6.871e-19","GO:0140097 (299) <br> catalytic activity, acting on DNA <br> 4.323e-18","GO:0016887 (553) <br> ATPase <br> 3.385e-16","GO:0140098 (394) <br> catalytic activity, acting on RNA <br> 5.965e-14","GO:0003729 (553) <br> mRNA binding <br> 9.771e-12","GO:0000166 (2176) <br> nucleotide binding <br> 5.966e-11","GO:1901265 (2177) <br> nucleoside phosphate binding <br> 6.087e-11","GO:0003735 (171) <br> structural constituent of ribosome <br> 1.632e-10","GO:0035639 (1846) <br> purine ribonucleoside triphosphate binding <br> 1.779e-10","GO:0005524 (1500) <br> ATP binding <br> 2.601e-10","GO:0032555 (1917) <br> purine ribonucleotide binding <br> 7.874e-10","GO:0017076 (1932) <br> purine nucleotide binding <br> 1.067e-09","GO:0032553 (1934) <br> ribonucleotide binding <br> 1.111e-09","GO:0032559 (1564) <br> adenyl ribonucleotide binding <br> 1.161e-09","GO:0030554 (1577) <br> adenyl nucleotide binding <br> 1.556e-09","GO:0036094 (2516) <br> small molecule binding <br> 2.665e-09","GO:0043168 (2413) <br> anion binding <br> 1.593e-08","GO:0019843 (68) <br> rRNA binding <br> 2.687e-08","GO:0003697 (121) <br> single-stranded DNA binding <br> 6.297e-08","GO:0034511 (10) <br> U3 snoRNA binding <br> 1.348e-07","GO:0097367 (2282) <br> carbohydrate derivative binding <br> 1.795e-07","GO:0017069 (47) <br> snRNA binding <br> 2.771e-07","GO:0003682 (573) <br> chromatin binding <br> 3.444e-07","GO:0043021 (150) <br> ribonucleoprotein complex binding <br> 3.370e-06","GO:0003824 (5682) <br> catalytic activity <br> 5.086e-06","GO:0005515 (14767) <br> protein binding <br> 5.865e-06","GO:0048027 (27) <br> mRNA 5'-UTR binding <br> 1.570e-05","GO:0005198 (701) <br> structural molecule activity <br> 3.377e-05","GO:0035173 (16) <br> histone kinase activity <br> 4.880e-05","GO:0003727 (91) <br> single-stranded RNA binding <br> 5.654e-05","GO:0001099 (70) <br> basal RNA polymerase II transcription machinery binding <br> 2.138e-04","GO:0001098 (70) <br> basal transcription machinery binding <br> 2.138e-04","GO:0016787 (2357) <br> hydrolase activity <br> 3.475e-04","GO:0003678 (76) <br> DNA helicase activity <br> 3.554e-04","GO:0003725 (78) <br> double-stranded RNA binding <br> 4.168e-04","GO:0042393 (240) <br> histone binding <br> 6.073e-04","GO:0003688 (23) <br> DNA replication origin binding <br> 1.155e-03","GO:0001091 (20) <br> RNA polymerase II general transcription initiation factor binding <br> 1.479e-03","GO:0005488 (17070) <br> binding <br> 1.499e-03","GO:1990518 (4) <br> single-stranded 3'-5' DNA helicase activity <br> 1.721e-03","GO:0019899 (2048) <br> enzyme binding <br> 2.228e-03","GO:0008649 (23) <br> rRNA methyltransferase activity <br> 2.984e-03","GO:0140102 (23) <br> catalytic activity, acting on a rRNA <br> 2.984e-03","GO:0070034 (22) <br> telomerase RNA binding <br> 3.115e-03","GO:0030619 (8) <br> U1 snRNA binding <br> 3.354e-03","GO:0098505 (10) <br> G-rich strand telomeric DNA binding <br> 4.539e-03","GO:0042162 (36) <br> telomeric DNA binding <br> 5.786e-03","GO:0034513 (5) <br> box H/ACA snoRNA binding <br> 7.238e-03","GO:0003730 (192) <br> mRNA 3'-UTR binding <br> 7.576e-03","GO:0043138 (17) <br> 3'-5' DNA helicase activity <br> 8.950e-03","GO:0003899 (43) <br> DNA-directed 5'-3' RNA polymerase activity <br> 1.038e-02","GO:0043047 (12) <br> single-stranded telomeric DNA binding <br> 1.045e-02","GO:0034062 (45) <br> 5'-3' RNA polymerase activity <br> 1.356e-02","GO:0097747 (45) <br> RNA polymerase activity <br> 1.356e-02","GO:0017116 (19) <br> single-stranded DNA helicase activity <br> 1.435e-02","GO:0070182 (18) <br> DNA polymerase binding <br> 1.718e-02","GO:0098847 (15) <br> sequence-specific single stranded DNA binding <br> 2.781e-02","GO:0061752 (2) <br> telomeric repeat-containing RNA binding <br> 3.472e-02","GO:0044877 (1304) <br> protein-containing complex binding <br> 4.234e-02","GO:0036002 (37) <br> pre-mRNA binding <br> 4.489e-02","GO:0034458 (4) <br> 3'-5' RNA helicase activity <br> 4.521e-02","GO:0015288 (5) <br> porin activity <br> 4.562e-02"],"key":["GO:0003723","GO:0003676","GO:0097159","GO:1901363","GO:0003724","GO:0004386","GO:0030515","GO:0008094","GO:0140097","GO:0016887","GO:0140098","GO:0003729","GO:0000166","GO:1901265","GO:0003735","GO:0035639","GO:0005524","GO:0032555","GO:0017076","GO:0032553","GO:0032559","GO:0030554","GO:0036094","GO:0043168","GO:0019843","GO:0003697","GO:0034511","GO:0097367","GO:0017069","GO:0003682","GO:0043021","GO:0003824","GO:0005515","GO:0048027","GO:0005198","GO:0035173","GO:0003727","GO:0001099","GO:0001098","GO:0016787","GO:0003678","GO:0003725","GO:0042393","GO:0003688","GO:0001091","GO:0005488","GO:1990518","GO:0019899","GO:0008649","GO:0140102","GO:0070034","GO:0030619","GO:0098505","GO:0042162","GO:0034513","GO:0003730","GO:0043138","GO:0003899","GO:0043047","GO:0034062","GO:0097747","GO:0017116","GO:0070182","GO:0098847","GO:0061752","GO:0044877","GO:0036002","GO:0034458","GO:0015288"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[20.258276019327,21.1658574902378,21.6047679255868,21.5904209757188,15.7591617451593,16.9558134205769,14.2382740983495,17.1440106914419,17.8347771324352,18.6728117067331,18.2168145994675,18.6728117067331,20.38841785752,20.3889635367376,17.0273503124218,20.1919193878134,19.9405304557382,20.2372173548861,20.2465569336091,20.2477963305085,19.9914490344426,20.001517942691,20.5599632179123,20.5107536570554,15.5745906356112,16.5019202141693,11.747980750738,20.4448133217894,14.9397854919929,18.7197942104249,16.8307889373762,21.4917345087043,22.525272898967,13.9127623976552,18.9836734866874,12.8370677538266,16.0521821733416,15.6229705518591,15.6229705518591,20.4830462803831,15.7591617451593,15.801858092204,17.5228245316566,13.5956642813554,13.3106655321065,22.6771653543307,9.00890082870231,20.3162578263421,13.5956642813554,13.5956642813554,13.5059251363754,11.1749781597015,11.747980750738,14.4581139285914,9.79200637188604,17.1987173630513,12.968152232712,14.781443852525,12.187219027138,14.862657240381,14.862657240381,13.2039079443259,13.0900518224006,12.6954028668369,3.77952755905512,19.7686971273735,14.5086078145015,9.00890082870231,9.79200637188604],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[195.411955705664,199.83466363138,199.830906020568,199.936119123303,196.734634711485,199.928603901679,199.924846290867,195.321773046176,199.838421242192,197.940827782135,194.10430714309,194.611584602709,197.925797338887,194.607826991897,196.565542224946,192.112773412734,195.652442797631,190.624759531185,195.611109078699,195.614866689511,195.607351467887,199.274779620393,193.333996926632,199.349931836633,195.603593857075,191.868528709954,197.099122960249,193.518119856419,196.058264765327,196.065779986951,193.657151456463,199.413811220436,197.557551479312,190.621001920373,197.775492906407,194.667948764889,199.353689447445,197.756704852347,191.763315607219,190.91785317452,199.560358042104,190.80512485016,196.576815057382,195.002376127157,194.06673103497,190.549607314945,197.508702538756,193.875092883559],"y":[17,17,17,17,17,17,17,17,12.3894775042977,3.75583758416169,3.51906222471571,3.42647826994669,3.40235696636987,3.32010106464881,3.25259660384691,3.21402102316722,3.02843772074586,2.98357645058463,2.6430932920422,2.6430932920422,2.6430932920422,2.62521419099113,2.54927498182239,2.52145362332443,2.37994064020102,2.35442683789542,2.23886637922022,2.1032727325261,2.045585770792,2.045585770792,2.03019025066309,1.95102654951172,1.94311131179047,1.94311131179047,1.87975510678122,1.84919339209676,1.81226180170001,1.81226180170001,1.79926361298674,1.75220280608367,1.74055851543301,1.68744945545977,1.5451380589623,1.5293150546992,1.49124541701354,1.44517727170341,1.41490944674541,1.39737026229581],"text":["REAC:R-HSA-8953854 (662) <br> Metabolism of RNA <br> 1.882e-55","REAC:R-HSA-72163 (178) <br> mRNA Splicing - Major Pathway <br> 1.321e-36","REAC:R-HSA-72172 (186) <br> mRNA Splicing <br> 2.063e-35","REAC:R-HSA-8868773 (193) <br> rRNA processing in the nucleus and cytosol <br> 9.438e-35","REAC:R-HSA-72203 (237) <br> Processing of Capped Intron-Containing Pre-mRNA <br> 3.110e-34","REAC:R-HSA-72312 (203) <br> rRNA processing <br> 2.295e-33","REAC:R-HSA-6790901 (60) <br> rRNA modification in the nucleus and cytosol <br> 2.379e-33","REAC:R-HSA-6791226 (183) <br> Major pathway of rRNA processing in the nucleolus and cytosol <br> 1.053e-30","REAC:R-HSA-72165 (51) <br> mRNA Splicing - Minor Pathway <br> 4.079e-13","REAC:R-HSA-2990846 (176) <br> SUMOylation <br> 1.755e-04","REAC:R-HSA-6782210 (63) <br> Gap-filling DNA repair synthesis and ligation in TC-NER <br> 3.026e-04","REAC:R-HSA-168273 (133) <br> Influenza Viral RNA Transcription and Replication <br> 3.746e-04","REAC:R-HSA-3108232 (170) <br> SUMO E3 ligases SUMOylate target proteins <br> 3.960e-04","REAC:R-HSA-168255 (152) <br> Influenza Infection <br> 4.785e-04","REAC:R-HSA-176417 (6) <br> Phosphorylation of Emi1 <br> 5.590e-04","REAC:R-HSA-69190 (32) <br> DNA strand elongation <br> 6.109e-04","REAC:R-HSA-68884 (13) <br> Mitotic Telophase/Cytokinesis <br> 9.366e-04","REAC:R-HSA-174048 (24) <br> APC/C:Cdc20 mediated degradation of Cyclin B <br> 1.039e-03","REAC:R-HSA-5368286 (87) <br> Mitochondrial translation initiation <br> 2.275e-03","REAC:R-HSA-5419276 (87) <br> Mitochondrial translation termination <br> 2.275e-03","REAC:R-HSA-5389840 (87) <br> Mitochondrial translation elongation <br> 2.275e-03","REAC:R-HSA-6781827 (77) <br> Transcription-Coupled Nucleotide Excision Repair (TC-NER) <br> 2.370e-03","REAC:R-HSA-6782135 (64) <br> Dual incision in TC-NER <br> 2.823e-03","REAC:R-HSA-72766 (292) <br> Translation <br> 3.010e-03","REAC:R-HSA-5368287 (93) <br> Mitochondrial translation <br> 4.169e-03","REAC:R-HSA-2470946 (10) <br> Cohesin Loading onto Chromatin <br> 4.422e-03","REAC:R-HSA-73856 (65) <br> RNA Polymerase II Transcription Termination <br> 5.769e-03","REAC:R-HSA-2468052 (11) <br> Establishment of Sister Chromatid Cohesion <br> 7.884e-03","REAC:R-HSA-975957 (116) <br> Nonsense Mediated Decay (NMD) enhanced by the Exon Junction Complex (EJC) <br> 9.004e-03","REAC:R-HSA-927802 (116) <br> Nonsense-Mediated Decay (NMD) <br> 9.004e-03","REAC:R-HSA-6803529 (25) <br> FGFR2 alternative splicing <br> 9.328e-03","REAC:R-HSA-72202 (80) <br> Transport of Mature Transcript to Cytoplasm <br> 1.119e-02","REAC:R-HSA-453276 (88) <br> Regulation of mitotic cell cycle <br> 1.140e-02","REAC:R-HSA-174143 (88) <br> APC/C-mediated degradation of cell cycle proteins <br> 1.140e-02","REAC:R-HSA-69242 (162) <br> S Phase <br> 1.319e-02","REAC:R-HSA-2995383 (16) <br> Initiation of Nuclear Envelope (NE) Reformation <br> 1.415e-02","REAC:R-HSA-72649 (59) <br> Translation initiation complex formation <br> 1.541e-02","REAC:R-HSA-72702 (59) <br> Ribosomal scanning and start codon recognition <br> 1.541e-02","REAC:R-HSA-75035 (13) <br> Chk1/Chk2(Cds1) mediated inactivation of Cyclin B:Cdk1 complex <br> 1.588e-02","REAC:R-HSA-72662 (60) <br> Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S <br> 1.769e-02","REAC:R-HSA-176974 (12) <br> Unwinding of DNA <br> 1.817e-02","REAC:R-HSA-176187 (37) <br> Activation of ATR in response to replication stress <br> 2.054e-02","REAC:R-HSA-176412 (20) <br> Phosphorylation of the APC/C <br> 2.850e-02","REAC:R-HSA-156827 (112) <br> L13a-mediated translational silencing of Ceruloplasmin expression <br> 2.956e-02","REAC:R-HSA-72706 (113) <br> GTP hydrolysis and joining of the 60S ribosomal subunit <br> 3.227e-02","REAC:R-HSA-164843 (7) <br> 2-LTR circle formation <br> 3.588e-02","REAC:R-HSA-9010553 (171) <br> Regulation of expression of SLITs and ROBOs <br> 3.847e-02","REAC:R-HSA-72695 (52) <br> Formation of the ternary complex, and subsequently, the 43S complex <br> 4.005e-02"],"key":["REAC:R-HSA-8953854","REAC:R-HSA-72163","REAC:R-HSA-72172","REAC:R-HSA-8868773","REAC:R-HSA-72203","REAC:R-HSA-72312","REAC:R-HSA-6790901","REAC:R-HSA-6791226","REAC:R-HSA-72165","REAC:R-HSA-2990846","REAC:R-HSA-6782210","REAC:R-HSA-168273","REAC:R-HSA-3108232","REAC:R-HSA-168255","REAC:R-HSA-176417","REAC:R-HSA-69190","REAC:R-HSA-68884","REAC:R-HSA-174048","REAC:R-HSA-5368286","REAC:R-HSA-5419276","REAC:R-HSA-5389840","REAC:R-HSA-6781827","REAC:R-HSA-6782135","REAC:R-HSA-72766","REAC:R-HSA-5368287","REAC:R-HSA-2470946","REAC:R-HSA-73856","REAC:R-HSA-2468052","REAC:R-HSA-975957","REAC:R-HSA-927802","REAC:R-HSA-6803529","REAC:R-HSA-72202","REAC:R-HSA-453276","REAC:R-HSA-174143","REAC:R-HSA-69242","REAC:R-HSA-2995383","REAC:R-HSA-72649","REAC:R-HSA-72702","REAC:R-HSA-75035","REAC:R-HSA-72662","REAC:R-HSA-176974","REAC:R-HSA-176187","REAC:R-HSA-176412","REAC:R-HSA-156827","REAC:R-HSA-72706","REAC:R-HSA-164843","REAC:R-HSA-9010553","REAC:R-HSA-72695"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,102,204,1)","opacity":0.8,"size":[18.90922365592,17.0869555296552,17.1519653138816,17.206351575782,17.5047579060072,17.2803636129636,15.3633750239743,17.1279572175344,15.0832328994516,17.0701951198665,15.4461643032416,16.6476991969387,17.01861417614,16.8507929922937,10.3630624020686,14.2382740983495,12.3729643352122,13.6808214449962,15.9797164726717,15.9797164726717,15.9797164726717,15.7806675338474,15.4727616697066,17.8014896682064,16.0870759222918,11.747980750738,15.498887690643,11.980531463383,16.4363189409718,16.4363189409718,13.7618192670139,15.8433279821367,15.9981813072674,15.9981813072674,16.9465972450821,12.8370677538266,15.3347186146137,15.3347186146137,12.3729643352122,15.3633750239743,12.187219027138,14.5086078145015,13.3106655321065,16.3815087539172,16.3954151879507,10.8097872173079,17.0273503124218,15.1170689685676],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,102,204,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"REAC","legendgroup":"REAC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"3280763258e3":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"32806f0b438f":{"x":{},"xend":{},"y":{},"yend":{}},"32802bfd6791":{"x":{},"xend":{},"y":{},"yend":{}},"32801ef437ed":{"x":{},"xend":{},"y":{},"yend":{}},"328063e4608":{"x":{},"xend":{},"y":{},"yend":{}},"328057bc774":{"x":{},"y":{}},"328046965ed8":{"yintercept":{}}},"cur_data":"3280763258e3","visdat":{"3280763258e3":["function (y) ","x"],"32806f0b438f":["function (y) ","x"],"32802bfd6791":["function (y) ","x"],"32801ef437ed":["function (y) ","x"],"328063e4608":["function (y) ","x"],"328057bc774":["function (y) ","x"],"328046965ed8":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedDatad1ba32e0"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;%
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-21697d58abce2053a245" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-21697d58abce2053a245">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487"],["GO:MF","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:MF","GO:MF","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","REAC","GO:BP","REAC","REAC","REAC","GO:CC","REAC","REAC","GO:CC","GO:CC","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:MF","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","REAC","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:MF","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:MF","GO:MF","GO:CC","GO:BP","REAC","REAC","GO:BP","GO:MF","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:MF","GO:BP","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:CC","REAC","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","REAC","REAC","REAC","GO:CC","GO:CC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:MF","GO:MF","REAC","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","REAC","REAC","GO:BP","GO:MF","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","REAC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP","REAC","GO:BP","GO:MF","REAC","REAC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:CC","REAC","GO:CC","REAC","REAC","GO:CC","GO:BP","GO:BP","REAC","GO:MF","GO:MF","REAC","GO:MF","GO:BP","GO:CC","REAC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:MF","REAC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","REAC","GO:CC","REAC","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC"],[438,2400,4442,2134,1446,2720,1452,7628,11668,399,2374,312,5689,378,1137,295,10305,11949,9964,1295,10027,2173,3961,21373,5688,3621,2401,14204,154,2705,10040,168,156,2132,5687,24636,2133,2129,1570,335,2130,2934,2128,2472,4661,2471,2499,1647,2127,2497,2496,1464,284,1271,11682,12668,338,11681,7894,2932,2776,1139,12725,19815,2926,899,12224,3398,12726,439,1116,1005,19739,4867,214,1140,3880,1961,206,902,11780,15771,4001,3544,26352,3852,340,3612,348,7633,15312,19802,11781,2473,2928,2407,2286,2334,443,1141,12399,380,480,204,2405,35,11946,42,2406,449,222,223,6342,404,1888,104,207,213,5407,2415,14544,3959,5405,5411,22986,26353,19280,4883,12553,15569,6389,303,5659,12141,840,12473,11669,111,2505,15294,15295,2306,6775,15740,2985,461,4596,10037,1710,1330,419,15684,120,210,22168,5967,6459,10338,111,3954,8762,14349,404,2933,12746,6462,2409,19510,1832,12472,19292,2976,6697,9234,14,868,5598,101,2500,3399,10013,8837,14707,2866,32,2917,24637,6756,9278,75,159,1112,512,25031,1879,18660,343,2927,2933,339,1709,406,63,14545,2287,386,8374,10031,2416,3675,257,12747,191,15683,2293,1015,2410,19511,1694,68,2308,332,20252,3629,29432,351,25568,6260,3158,8196,442,6493,15314,4767,13694,8604,4787,29434,12404,18705,834,12402,12490,639,2065,18706,166,2367,12743,2334,341,2557,408,64,1016,15753,336,1968,7600,118,12403,3011,206,205,5869,19853,224,8608,981,15589,26133,947,8197,3758,401,369,17813,1082,1964,24835,440,2326,2343,1081,21523,12733,3024,1602,2418,6649,29478,417,2177,2915,2419,15790,258,8860,16286,259,1359,16112,110,21,410,3674,30393,1270,2030,30394,2417,15313,198,3260,1862,112,5665,14546,14688,12179,13686,25506,2292,3673,2314,4618,19198,1348,1349,1347,2925,2931,2323,2365,4980,8288,16136,13577,859,742,21376,2289,11783,2343,343,216,9817,16291,2841,1440,1447,21374,9198,4922,9279,24826,2987,2341,4662,27307,1776,838,1346,352,9823,10381,2182,8215,3155,1331,25572,1572,4228,15799,2627,1744,6607,13555,9657,2940,23006,4728,21150,315,9235,5969,3641,1163,2303,444,614,8610,791,2370,6772,1467,1469,828,318,2599,22749,15754,576,6763,3238,390,2360,138,1866,20,321,7718,261,1924,5891,10379,1097,3985,26294,1777,2344,1919,324,3004,8614,3014,3349,15793,9847,2598,99,2399,25321,7622,35,69,9354,30384,5096,160,22750,13030,839,2295,15792,21522,16290,15795,13929,10434,161,1605,1186,8362,15276,26354,8232,4797,9428,13599,937,13032,9689,1,1778,14486,13134,1438,5080,1853,167,886,9985,9546,7129,7803,26,15281,6378,5960,15369,3086,9984,26169,2743,20876,455],["RNA binding","RNA processing","ribonucleoprotein complex","organelle lumen","membrane-enclosed lumen","intracellular organelle lumen","nuclear lumen","ribonucleoprotein complex biogenesis","ribosome biogenesis","nucleic acid binding","rRNA processing","nucleoplasm","rRNA metabolic process","nucleolus","preribosome","nucleus","organic cyclic compound binding","heterocyclic compound binding","ncRNA processing","Metabolism of RNA","cellular nitrogen compound metabolic process","nucleobase-containing compound metabolic process","nuclear protein-containing complex","nucleic acid metabolic process","mRNA metabolic process","RNA splicing","mRNA processing","heterocycle metabolic process","RNA splicing, via transesterification reactions","cellular aromatic compound metabolic process","ncRNA metabolic process","mRNA splicing, via spliceosome","RNA splicing, via transesterification reactions with bulged adenosine as nucleophile","intracellular membrane-bounded organelle","RNA metabolic process","organic cyclic compound metabolic process","intracellular non-membrane-bounded organelle","non-membrane-bounded organelle","protein-containing complex","spliceosomal complex","intracellular organelle","catalytic step 2 spliceosome","membrane-bounded organelle","mRNA Splicing - Major Pathway","gene expression","mRNA Splicing","rRNA processing in the nucleus and cytosol","Processing of Capped Intron-Containing Pre-mRNA","organelle","rRNA processing","rRNA modification in the nucleus and cytosol","small-subunit processome","intracellular anatomical structure","Major pathway of rRNA processing in the nucleolus and cytosol","ribosomal small subunit biogenesis","cellular component biogenesis","U2-type spliceosomal complex","ribosomal large subunit biogenesis","maturation of SSU-rRNA","precatalytic spliceosome","nitrogen compound metabolic process","90S preribosome","cellular metabolic process","cellular component organization or biogenesis","U2-type precatalytic spliceosome","nuclear body","macromolecule metabolic process","spliceosomal snRNP complex","primary metabolic process","RNA helicase activity","small nuclear ribonucleoprotein complex","helicase activity","organic substance metabolic process","snoRNA binding","maturation of LSU-rRNA","preribosome, large subunit precursor","Sm-like protein family complex","ATPase, acting on DNA","maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","nuclear speck","catalytic activity, acting on DNA","chromosome organization","catalytic complex","metabolic process","regulation of mRNA metabolic process","ATPase","U2 snRNP","chromosomal region","chromosome","ribonucleoprotein complex assembly","regulation of mRNA processing","ribonucleoprotein complex subunit organization","catalytic activity, acting on RNA","mRNA Splicing - Minor Pathway","U2-type catalytic step 2 spliceosome","RNA localization","DNA metabolic process","ribosomal subunit","mRNA binding","preribosome, small subunit precursor","regulation of RNA splicing","sno(s)RNA-containing ribonucleoprotein complex","ribosome","maturation of 5.8S rRNA","RNA catabolic process","nucleotide binding","nucleoside phosphate binding","nuclear chromosome","mRNA catabolic process","structural constituent of ribosome","endonucleolytic cleavage involved in rRNA processing","endonucleolytic cleavage of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","purine ribonucleoside triphosphate binding","mitochondrial matrix","ATP binding","chromosome, telomeric region","maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","cleavage involved in rRNA processing","purine ribonucleotide binding","translation","regulation of mRNA splicing, via spliceosome","purine nucleotide binding","ribonucleotide binding","adenyl ribonucleotide binding","mitotic nuclear division","negative regulation of mRNA metabolic process","DNA conformation change","adenyl nucleotide binding","protein-containing complex subunit organization","mRNA transport","small molecule binding","sister chromatid segregation","viral process","peptide biosynthetic process","small ribosomal subunit","amide biosynthetic process","ribosome assembly","condensed chromosome","peptide metabolic process","nucleic acid transport","RNA transport","DNA repair","anion binding","establishment of RNA localization","chromosome segregation","spindle","rRNA binding","nucleobase-containing compound catabolic process","nuclear periphery","box C/D RNP complex","single-stranded DNA binding","nuclear transport","nuclear division","maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","nuclear chromosome segregation","U3 snoRNA binding","viral gene expression","carbohydrate derivative binding","spliceosomal complex assembly","snRNA binding","DNA geometric change","heterocycle catabolic process","chromatin binding","organelle organization","cellular nitrogen compound catabolic process","viral transcription","RNA export from nucleus","ribonucleoprotein complex export from nucleus","exon-exon junction complex","cellular amide metabolic process","ribonucleoprotein complex localization","cell cycle","aromatic compound catabolic process","regulation of chromosome organization","ribosomal large subunit assembly","nuclear matrix","nucleobase-containing compound transport","chromosome, centromeric region","U4/U6 x U5 tri-snRNP complex","spliceosomal tri-snRNP complex","cellular protein-containing complex assembly","DNA duplex unwinding","organelle fission","nucleocytoplasmic transport","mitotic sister chromatid segregation","cellular response to DNA damage stimulus","organic cyclic compound catabolic process","ribonucleoprotein complex binding","negative regulation of RNA splicing","nuclear-transcribed mRNA catabolic process, nonsense-mediated decay","alternative mRNA splicing, via spliceosome","midbody","catalytic activity","regulation of signal transduction by p53 class mediator","protein binding","protein-containing complex assembly","U12-type spliceosomal complex","U2-type catalytic step 1 spliceosome","catalytic step 1 spliceosome","U1 snRNP","Pwp2p-containing subcomplex of 90S preribosome","mitochondrial ribosome","organellar ribosome","negative regulation of mRNA splicing, via spliceosome","DNA replication","mitochondrion","mRNA 5'-UTR binding","cellular macromolecule biosynthetic process","translational initiation","macromolecule biosynthetic process","telomere maintenance","cellular nitrogen compound biosynthetic process","endonucleolytic cleavage in ITS1 to separate SSU-rRNA from 5.8S rRNA and LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","nuclear export","DNA unwinding involved in DNA replication","cytosolic ribosome","mRNA export from nucleus","mRNA-containing ribonucleoprotein complex export from nucleus","structural molecule activity","rRNA modification","transcription-coupled nucleotide-excision repair","nuclear-transcribed mRNA catabolic process","signal transduction by p53 class mediator","mitochondrial protein-containing complex","regulation of rRNA processing","telomerase holoenzyme complex","negative regulation of RNA catabolic process","histone kinase activity","mitotic spindle","RNA 3'-end processing","single-stranded RNA binding","regulation of nucleobase-containing compound metabolic process","negative regulation of mRNA processing","posttranscriptional regulation of gene expression","positive regulation of transcription by RNA polymerase I","telomere organization","negative regulation of gene expression","positive regulation of rRNA processing","RNA stabilization","mitochondrial translational elongation","Cajal body","regulation of RNA stability","cellular protein complex disassembly","nucleoid","mitochondrial nucleoid","mitochondrial translational termination","fibrillar center","regulation of transcription by RNA polymerase I","cellular macromolecule catabolic process","DNA recombination","U4 snRNP","spindle midzone","mitochondrial small ribosomal subunit","organellar small ribosomal subunit","cytosolic small ribosomal subunit","regulation of RNA metabolic process","U5 snRNP","SUMOylation","cell cycle process","mitotic cell cycle","regulation of mRNA stability","regulation of mitotic nuclear division","basal RNA polymerase II transcription machinery binding","basal transcription machinery binding","histone phosphorylation","DNA biosynthetic process","endonucleolytic cleavage in 5'-ETS of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","regulation of telomere maintenance","organelle inner membrane","regulation of DNA metabolic process","mitotic cell cycle process","Gap-filling DNA repair synthesis and ligation in TC-NER","mRNA 3'-end processing","hydrolase activity","DNA helicase activity","pericentric heterochromatin","regulation of mRNA catabolic process","Influenza Viral RNA Transcription and Replication","SUMO E3 ligases SUMOylate target proteins","organic substance biosynthetic process","double-stranded RNA binding","double-strand break repair","chromatin organization","Influenza Infection","RNA phosphodiester bond hydrolysis, endonucleolytic","cellular biosynthetic process","CMG complex","Phosphorylation of Emi1","translational termination","histone binding","regulation of DNA biosynthetic process","DNA strand elongation","nuclear replication fork","CRD-mediated mRNA stability complex","regulation of translation","cell division","double-strand break repair via homologous recombination","mitochondrial translation","regulation of chromosome segregation","recombinational repair","Mitotic Telophase/Cytokinesis","regulation of cell cycle","heterochromatin","APC/C:Cdc20 mediated degradation of Cyclin B","DNA replication origin binding","biosynthetic process","negative regulation of chromosome organization","DNA replication preinitiation complex","MCM complex","positive regulation of chromosome organization","translational elongation","positive regulation of mRNA processing","RNA polymerase II general transcription initiation factor binding","cyclin B1-CDK1 complex","binding","condensed nuclear chromosome","cellular component organization","positive regulation of mRNA splicing, via spliceosome","mRNA stabilization","single-stranded 3'-5' DNA helicase activity","negative regulation of nucleobase-containing compound metabolic process","pre-replicative complex assembly involved in cell cycle DNA replication","pre-replicative complex assembly involved in nuclear cell cycle DNA replication","macromolecule catabolic process","nucleotide-excision repair","enzyme binding","CRD-mediated mRNA stabilization","Mitochondrial translation initiation","Mitochondrial translation termination","Mitochondrial translation elongation","U2-type prespliceosome","prespliceosome","Transcription-Coupled Nucleotide Excision Repair (TC-NER)","DNA-templated transcription, elongation","telomere maintenance via telomere lengthening","negative regulation of cellular catabolic process","regulation of nuclear division","positive regulation of gene expression, epigenetic","cytoplasmic translation","Dual incision in TC-NER","mitotic spindle assembly","rRNA methyltransferase activity","catalytic activity, acting on a rRNA","Translation","rRNA 5'-end processing","endonucleolytic cleavage to generate mature 5'-end of SSU-rRNA from (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","telomerase RNA binding","regulation of attachment of spindle microtubules to kinetochore","PeBoW complex","organelle envelope","envelope","nucleic acid phosphodiester bond hydrolysis","protein-containing complex disassembly","U1 snRNA binding","positive regulation of RNA splicing","organonitrogen compound biosynthetic process","sister chromatid cohesion","DNA packaging","regulation of gene expression","regulation of telomere maintenance via telomere lengthening","methyltransferase complex","microtubule cytoskeleton","Mitochondrial translation","Cohesin Loading onto Chromatin","regulation of cellular amide metabolic process","G-rich strand telomeric DNA binding","nuclear replisome","anaphase-promoting complex-dependent catabolic process","box H/ACA RNP complex","box H/ACA snoRNP complex","negative regulation of mRNA catabolic process","protein-DNA complex","negative regulation of catabolic process","metaphase plate congression","ASAP complex","RNA Polymerase II Transcription Termination","telomeric DNA binding","positive regulation of cell cycle","box H/ACA RNA metabolic process","telomere maintenance via telomerase","mitochondrial gene expression","regulation of cell cycle process","positive regulation of cell cycle process","replication fork","regulation of sister chromatid segregation","box H/ACA snoRNA binding","attachment of spindle microtubules to kinetochore","replisome","RNA-dependent DNA biosynthetic process","mRNA 3'-UTR binding","cohesin complex","positive regulation of telomere maintenance","Establishment of Sister Chromatid Cohesion","transcription by RNA polymerase I","3'-5' DNA helicase activity","Nonsense Mediated Decay (NMD) enhanced by the Exon Junction Complex (EJC)","Nonsense-Mediated Decay (NMD)","FGFR2 alternative splicing","DNA replication factor A complex","SRP-dependent cotranslational protein targeting to membrane","cellular detoxification of aldehyde","negative regulation of RNA metabolic process","DNA-directed 5'-3' RNA polymerase activity","single-stranded telomeric DNA binding","regulation of mitotic cell cycle","mitochondrial inner membrane","Transport of Mature Transcript to Cytoplasm","spindle pole","Regulation of mitotic cell cycle","APC/C-mediated degradation of cell cycle proteins","RNA polymerase II, core complex","regulation of mitotic metaphase/anaphase transition","double-strand break repair via break-induced replication","S Phase","5'-3' RNA polymerase activity","RNA polymerase activity","Initiation of Nuclear Envelope (NE) Reformation","single-stranded DNA helicase activity","U2-type prespliceosome assembly","methylosome","Translation initiation complex formation","Ribosomal scanning and start codon recognition","Chk1/Chk2(Cds1) mediated inactivation of Cyclin B:Cdk1 complex","mitotic metaphase plate congression","regulation of telomere maintenance via telomerase","metaphase/anaphase transition of mitotic cell cycle","mitotic spindle pole","chromosome separation","DNA polymerase binding","cotranslational protein targeting to membrane","Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S","Unwinding of DNA","regulation of metaphase/anaphase transition of cell cycle","cellular component assembly","cell cycle checkpoint signaling","Activation of ATR in response to replication stress","nuclear DNA replication","regulation of intrinsic apoptotic signaling pathway","regulation of mitotic sister chromatid separation","regulation of alternative mRNA splicing, via spliceosome","cellular response to aldehyde","metaphase/anaphase transition of cell cycle","large ribosomal subunit","DNA replication initiation","establishment of chromosome localization","RNA phosphodiester bond hydrolysis","positive regulation of attachment of spindle microtubules to kinetochore","mitotic sister chromatid separation","aldehyde catabolic process","sequence-specific single stranded DNA binding","spliceosomal snRNP assembly","Phosphorylation of the APC/C","L13a-mediated translational silencing of Ceruloplasmin expression","protein-containing complex localization","chromosome localization","positive regulation of mRNA metabolic process","rRNA methylation","negative regulation of organelle organization","protein localization to organelle","negative regulation of mitotic nuclear division","GTP hydrolysis and joining of the 60S ribosomal subunit","cell cycle DNA replication","telomeric repeat-containing RNA binding","2-LTR circle formation","pICln-Sm protein complex","intracellular transport","protein targeting to ER","nuclear membrane","negative regulation of cell cycle process","Regulation of expression of SLITs and ROBOs","dense fibrillar component","Formation of the ternary complex, and subsequently, the 43S complex","protein localization to chromosome","cellular response to stress","protein-containing complex binding","chromosome condensation","nuclear ubiquitin ligase complex","positive regulation of viral transcription","pre-mRNA binding","3'-5' RNA helicase activity","regulation of viral process","porin activity","protein localization to kinetochore","protein localization to condensed chromosome","extracellular exosome","establishment of integrated proviral latency","centrosome"],["9.5e-125","2.6e-111","1.4e-101"," 1.2e-93"," 1.2e-93"," 1.2e-93"," 2.1e-85"," 9.8e-84"," 9.3e-78"," 5.0e-75"," 7.2e-72"," 3.7e-71"," 2.7e-70"," 4.9e-66"," 3.1e-63"," 3.7e-62"," 3.6e-60"," 2.2e-59"," 5.1e-56"," 1.9e-55"," 3.3e-52"," 8.3e-52"," 1.2e-51"," 1.4e-51"," 1.7e-51"," 2.0e-50"," 2.1e-50"," 4.4e-50"," 1.7e-49"," 2.5e-49"," 2.7e-49"," 1.3e-48"," 1.3e-48"," 4.6e-48"," 3.1e-47"," 4.3e-47"," 1.8e-45"," 2.4e-45"," 8.0e-45"," 1.5e-41"," 1.9e-40"," 1.5e-38"," 1.8e-38"," 1.3e-36"," 6.0e-36"," 2.1e-35"," 9.4e-35"," 3.1e-34"," 1.3e-33"," 2.3e-33"," 2.4e-33"," 6.1e-32"," 6.3e-31"," 1.1e-30"," 9.5e-30"," 1.1e-29"," 2.0e-28"," 2.1e-28"," 2.3e-28"," 8.4e-26"," 8.8e-26"," 2.3e-25"," 2.8e-25"," 1.1e-24"," 1.3e-24"," 7.7e-23"," 2.3e-22"," 2.6e-22"," 2.8e-22"," 1.1e-21"," 1.1e-20"," 2.7e-20"," 1.4e-19"," 2.0e-19"," 2.1e-19"," 3.5e-19"," 6.5e-19"," 6.9e-19"," 7.3e-19"," 9.7e-19"," 4.3e-18"," 1.1e-17"," 1.4e-17"," 8.1e-17"," 1.0e-16"," 3.4e-16"," 1.0e-15"," 2.5e-15"," 3.9e-15"," 4.2e-15"," 9.8e-15"," 1.2e-14"," 6.0e-14"," 4.1e-13"," 6.3e-13"," 2.8e-12"," 7.0e-12"," 9.1e-12"," 9.8e-12"," 1.3e-11"," 3.6e-11"," 3.7e-11"," 3.8e-11"," 4.2e-11"," 4.2e-11"," 6.0e-11"," 6.1e-11"," 1.2e-10"," 1.6e-10"," 1.6e-10"," 1.7e-10"," 1.7e-10"," 1.8e-10"," 2.4e-10"," 2.6e-10"," 2.8e-10"," 3.2e-10"," 3.7e-10"," 7.9e-10"," 8.7e-10"," 9.7e-10"," 1.1e-09"," 1.1e-09"," 1.2e-09"," 1.2e-09"," 1.4e-09"," 1.4e-09"," 1.6e-09"," 1.7e-09"," 2.0e-09"," 2.7e-09"," 2.7e-09"," 3.1e-09"," 3.1e-09"," 3.2e-09"," 3.7e-09"," 3.7e-09"," 4.5e-09"," 8.7e-09"," 1.4e-08"," 1.4e-08"," 1.6e-08"," 1.6e-08"," 2.0e-08"," 2.0e-08"," 2.3e-08"," 2.7e-08"," 3.1e-08"," 3.4e-08"," 3.4e-08"," 6.3e-08"," 8.2e-08"," 9.2e-08"," 1.1e-07"," 1.3e-07"," 1.3e-07"," 1.6e-07"," 1.8e-07"," 2.3e-07"," 2.8e-07"," 3.1e-07"," 3.3e-07"," 3.4e-07"," 3.5e-07"," 4.4e-07"," 4.7e-07"," 5.3e-07"," 6.0e-07"," 6.7e-07"," 6.8e-07"," 6.9e-07"," 7.1e-07"," 7.7e-07"," 8.0e-07"," 8.8e-07"," 8.9e-07"," 1.0e-06"," 1.1e-06"," 1.2e-06"," 1.2e-06"," 1.2e-06"," 1.2e-06"," 1.4e-06"," 1.4e-06"," 1.5e-06"," 2.2e-06"," 2.4e-06"," 3.4e-06"," 3.4e-06"," 4.0e-06"," 4.3e-06"," 4.8e-06"," 5.1e-06"," 5.6e-06"," 5.9e-06"," 7.2e-06"," 7.6e-06"," 7.9e-06"," 7.9e-06"," 8.1e-06"," 8.6e-06"," 8.9e-06"," 8.9e-06"," 1.3e-05"," 1.4e-05"," 1.5e-05"," 1.6e-05"," 1.6e-05"," 1.7e-05"," 1.9e-05"," 2.2e-05"," 2.3e-05"," 2.3e-05"," 2.4e-05"," 2.8e-05"," 3.0e-05"," 3.1e-05"," 3.1e-05"," 3.4e-05"," 3.4e-05"," 3.5e-05"," 3.8e-05"," 3.8e-05"," 4.3e-05"," 4.5e-05"," 4.5e-05"," 4.8e-05"," 4.9e-05"," 5.0e-05"," 5.6e-05"," 5.7e-05"," 6.1e-05"," 6.2e-05"," 6.3e-05"," 6.6e-05"," 6.9e-05"," 7.0e-05"," 7.1e-05"," 8.4e-05"," 1.0e-04"," 1.1e-04"," 1.1e-04"," 1.1e-04"," 1.1e-04"," 1.1e-04"," 1.2e-04"," 1.2e-04"," 1.3e-04"," 1.3e-04"," 1.4e-04"," 1.4e-04"," 1.5e-04"," 1.5e-04"," 1.5e-04"," 1.6e-04"," 1.6e-04"," 1.7e-04"," 1.8e-04"," 1.9e-04"," 1.9e-04"," 2.0e-04"," 2.0e-04"," 2.1e-04"," 2.1e-04"," 2.3e-04"," 2.6e-04"," 2.8e-04"," 2.8e-04"," 2.9e-04"," 2.9e-04"," 3.0e-04"," 3.0e-04"," 3.3e-04"," 3.5e-04"," 3.6e-04"," 3.6e-04"," 3.6e-04"," 3.7e-04"," 4.0e-04"," 4.1e-04"," 4.2e-04"," 4.3e-04"," 4.6e-04"," 4.8e-04"," 5.2e-04"," 5.3e-04"," 5.4e-04"," 5.6e-04"," 5.7e-04"," 6.1e-04"," 6.1e-04"," 6.1e-04"," 6.3e-04"," 6.6e-04"," 6.7e-04"," 7.0e-04"," 7.2e-04"," 7.7e-04"," 8.4e-04"," 8.5e-04"," 9.4e-04"," 9.7e-04"," 9.8e-04"," 1.0e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.4e-03"," 1.4e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.6e-03"," 1.6e-03"," 1.7e-03"," 1.7e-03"," 1.7e-03"," 1.8e-03"," 1.9e-03"," 1.9e-03"," 2.1e-03"," 2.2e-03"," 2.2e-03"," 2.3e-03"," 2.3e-03"," 2.3e-03"," 2.3e-03"," 2.3e-03"," 2.3e-03"," 2.4e-03"," 2.4e-03"," 2.4e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.7e-03"," 2.8e-03"," 2.9e-03"," 3.0e-03"," 3.0e-03"," 3.0e-03"," 3.1e-03"," 3.1e-03"," 3.1e-03"," 3.2e-03"," 3.2e-03"," 3.2e-03"," 3.2e-03"," 3.3e-03"," 3.3e-03"," 3.4e-03"," 3.4e-03"," 3.5e-03"," 3.7e-03"," 3.8e-03"," 3.8e-03"," 4.0e-03"," 4.1e-03"," 4.1e-03"," 4.2e-03"," 4.4e-03"," 4.5e-03"," 4.5e-03"," 4.7e-03"," 4.7e-03"," 4.7e-03"," 4.7e-03"," 5.2e-03"," 5.3e-03"," 5.4e-03"," 5.4e-03"," 5.6e-03"," 5.8e-03"," 5.8e-03"," 5.9e-03"," 5.9e-03"," 6.0e-03"," 6.0e-03"," 6.1e-03"," 6.3e-03"," 6.5e-03"," 6.6e-03"," 7.2e-03"," 7.2e-03"," 7.3e-03"," 7.4e-03"," 7.6e-03"," 7.7e-03"," 7.7e-03"," 7.9e-03"," 8.1e-03"," 8.9e-03"," 9.0e-03"," 9.0e-03"," 9.3e-03"," 9.6e-03"," 9.9e-03"," 9.9e-03"," 1.0e-02"," 1.0e-02"," 1.0e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.2e-02"," 1.2e-02"," 1.3e-02"," 1.3e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.8e-02"," 1.8e-02"," 1.9e-02"," 1.9e-02"," 2.0e-02"," 2.1e-02"," 2.1e-02"," 2.1e-02"," 2.1e-02"," 2.2e-02"," 2.3e-02"," 2.4e-02"," 2.5e-02"," 2.6e-02"," 2.6e-02"," 2.6e-02"," 2.7e-02"," 2.8e-02"," 2.8e-02"," 2.8e-02"," 2.8e-02"," 2.9e-02"," 3.0e-02"," 3.0e-02"," 3.1e-02"," 3.1e-02"," 3.1e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.3e-02"," 3.5e-02"," 3.6e-02"," 3.6e-02"," 3.7e-02"," 3.7e-02"," 3.8e-02"," 3.8e-02"," 3.8e-02"," 3.9e-02"," 4.0e-02"," 4.0e-02"," 4.1e-02"," 4.2e-02"," 4.3e-02"," 4.4e-02"," 4.5e-02"," 4.5e-02"," 4.5e-02"," 4.5e-02"," 4.6e-02"," 4.6e-02"," 4.6e-02"," 4.8e-02"," 4.9e-02"," 5.0e-02"],[1951,1010,694,5579,5579,5579,4453,473,335,4252,265,4112,275,955,79,7607,6291,6210,439,662,6576,5752,1236,5234,880,459,532,5929,363,5973,535,360,360,12046,4714,6227,5094,5102,5541,192,13117,85,13201,178,6192,186,193,237,14027,203,60,39,14808,183,74,3377,93,72,50,52,10148,28,10941,6880,50,816,9867,61,10642,76,70,163,11397,32,29,24,82,185,37,413,299,1270,1377,11915,384,553,21,348,1795,196,145,203,394,51,30,222,976,191,553,15,152,24,247,36,431,2176,2177,237,388,171,15,15,1846,482,1500,160,15,27,1917,804,107,1932,1934,1564,346,95,359,1577,2017,151,2516,214,1011,834,76,975,62,239,998,183,183,586,2413,186,353,390,68,573,132,7,121,356,495,24,293,10,237,2282,59,47,114,617,573,4136,623,181,129,130,20,1284,131,1867,636,299,27,110,246,200,32,32,1150,109,544,353,185,894,664,150,27,123,76,197,5682,202,14767,1724,28,12,12,19,6,88,88,22,338,1678,27,5081,203,5145,169,5054,9,202,16,106,112,112,701,37,72,214,290,272,15,21,76,16,161,157,91,4123,31,928,23,183,1470,10,66,87,57,259,228,45,45,88,140,34,1244,328,10,36,29,29,46,3847,18,176,1388,1098,247,110,70,70,38,201,7,82,564,389,956,63,104,2357,76,23,255,133,170,6297,78,280,839,152,81,6210,10,6,104,240,112,32,35,6,504,630,140,135,86,142,13,1120,76,24,23,6391,92,12,12,192,143,35,20,2,17070,73,6661,25,57,4,1569,8,8,1479,107,2048,5,87,87,87,16,16,77,104,87,264,138,58,149,64,66,23,23,292,5,5,22,14,3,1244,1244,306,337,8,39,1889,56,246,5218,63,101,1313,93,10,577,10,22,87,5,5,64,212,324,65,4,65,36,317,3,75,166,815,237,67,70,5,36,24,77,192,10,52,11,68,17,116,116,25,13,96,8,1452,43,12,612,502,80,168,88,88,15,59,12,162,45,45,16,19,3,12,59,59,13,50,54,61,36,95,18,102,60,12,62,3104,196,37,66,167,63,60,12,64,119,49,82,157,6,65,13,15,40,20,112,289,84,144,25,361,974,48,113,70,2,7,6,1723,111,301,400,171,2,52,94,2142,1304,67,45,24,37,4,193,5,19,19,2178,8,620],[241,180,146,328,328,328,289,116,98,273,86,262,86,131,51,338,308,305,90,129,310,289,130,275,115,83,88,291,75,291,91,74,74,397,254,294,253,253,265,52,403,37,402,57,275,57,59,62,407,59,36,26,413,54,32,182,31,31,27,24,342,20,356,267,23,69,330,23,345,18,23,25,354,18,18,16,23,25,19,48,29,88,88,358,46,35,13,40,98,30,28,30,23,19,13,31,66,23,44,10,25,11,25,14,41,59,59,27,38,21,10,10,53,39,47,23,10,12,53,45,20,53,53,47,34,19,35,47,93,22,61,26,62,45,14,49,14,25,49,23,23,44,58,23,32,31,13,43,19,6,17,31,38,10,28,7,25,54,13,11,18,43,39,153,35,19,18,18,8,53,18,66,35,28,9,16,24,20,9,9,60,17,38,29,21,52,35,18,9,15,14,13,69,21,377,75,8,6,6,7,5,10,10,8,27,45,8,135,18,136,16,134,6,20,7,12,15,15,32,8,11,18,24,17,7,6,13,4,16,17,13,145,9,50,8,16,68,6,12,10,8,23,15,9,9,10,15,9,47,26,5,6,6,6,8,136,6,22,61,52,22,12,11,11,5,16,5,10,21,21,47,11,14,35,8,5,22,16,21,153,11,23,27,17,12,151,4,3,10,19,11,8,7,4,32,12,13,11,12,13,6,17,10,4,5,153,12,4,4,18,11,8,5,2,409,9,113,7,10,3,67,4,4,64,11,77,4,10,10,10,5,5,11,13,10,21,12,9,13,10,10,5,5,23,4,4,5,5,3,51,51,22,16,4,8,58,9,19,166,8,11,15,10,5,33,4,5,5,3,3,10,13,23,7,3,11,7,9,3,9,11,32,15,8,10,3,4,5,9,13,4,6,5,10,4,13,13,6,4,10,2,61,6,4,11,17,12,13,5,5,4,9,4,18,6,6,3,4,3,4,9,9,3,6,7,9,6,11,4,10,9,4,9,104,7,7,8,15,9,9,2,9,9,7,7,14,3,9,2,4,7,3,12,19,7,13,5,13,36,4,12,8,2,4,3,64,10,18,9,15,2,8,9,79,51,4,3,5,6,2,15,3,4,4,33,4,9]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/BP-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/MF-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/CC-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/scatter-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="skap2_c_act-1" class="section level3 tabset tabset-fade">
<h3><strong>5.2. SKAP2_C_act</strong></h3>
<div id="gostplot-1" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in SKAP2_C_act were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &lt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_2)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-4929a56e939b13f92b78" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-4929a56e939b13f92b78">{"x":{"data":[{"x":[144.381769502671,81.5886846164596,88.6128557438876,165.630826223452,145.245704988932,145.369660950004,76.2848719790647,108.806408674932,116.17239775562,103.551427173717,160.435944582151,142.477355191651,81.5924408577042,126.329274081056,101.004695609869,105.294323111218,119.537989910794,118.189499303978,76.7994770295768],"y":[3.33363299893327,3.14405553341062,3.02180858310735,2.71557643258503,2.57565514998125,2.22394400620862,2.18453834391752,2.0931460914831,2.01817686363668,1.88593710638968,1.84819648086571,1.79048774889464,1.77463118343558,1.71689621855009,1.6274486867131,1.61737329739083,1.61372811666438,1.4735489490225,1.40442433330242],"text":["GO:0070887 (3443) <br> cellular response to chemical stimulus <br> 4.638e-04","GO:0006457 (230) <br> protein folding <br> 7.177e-04","GO:0010033 (3457) <br> response to organic substance <br> 9.510e-04","GO:1901564 (6904) <br> organonitrogen compound metabolic process <br> 1.925e-03","GO:0071310 (2821) <br> cellular response to organic substance <br> 2.657e-03","GO:0071345 (1148) <br> cellular response to cytokine stimulus <br> 5.971e-03","GO:0002376 (3438) <br> immune system process <br> 6.538e-03","GO:0034097 (1242) <br> response to cytokine <br> 8.070e-03","GO:0042221 (4829) <br> response to chemical <br> 9.590e-03","GO:0031340 (11) <br> positive regulation of vesicle fusion <br> 1.300e-02","GO:1900122 (8) <br> positive regulation of receptor binding <br> 1.418e-02","GO:0065003 (1724) <br> protein-containing complex assembly <br> 1.620e-02","GO:0006458 (40) <br> 'de novo' protein folding <br> 1.680e-02","GO:0046716 (21) <br> muscle cell cellular homeostasis <br> 1.919e-02","GO:0022604 (312) <br> regulation of cell morphogenesis <br> 2.358e-02","GO:0032388 (227) <br> positive regulation of intracellular transport <br> 2.413e-02","GO:0043933 (2017) <br> protein-containing complex subunit organization <br> 2.434e-02","GO:0043122 (250) <br> regulation of I-kappaB kinase/NF-kappaB signaling <br> 3.361e-02","GO:0002521 (566) <br> leukocyte differentiation <br> 3.941e-02"],"key":["GO:0070887","GO:0006457","GO:0010033","GO:1901564","GO:0071310","GO:0071345","GO:0002376","GO:0034097","GO:0042221","GO:0031340","GO:1900122","GO:0065003","GO:0006458","GO:0046716","GO:0022604","GO:0032388","GO:0043933","GO:0043122","GO:0002521"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[21.0609703433762,17.5703173351792,21.0656756582917,21.8498224161939,20.8283362100316,19.7365147296747,21.0592849240276,19.835054721058,21.4489310346629,12.0456958427095,11.2337416908317,20.2392898942552,14.7373847204614,13.4876469321386,18.006537444193,17.5512244674609,20.4293081746705,17.6909608332441,18.82214113588],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedDatafb95dc9b","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[53.64701655001,53.3276749095439,62.186587712122,64.6210981006167,52.5725258538534,67.1194768172047,62.0964206606962,59.8873279007658,52.9482219014606,57.6218807336943,59.6243406674408,57.3401086979889,55.0258210447285,53.9475733880958,62.1001776211723,57.3138099746564,59.898598782194,55.7396435351822,59.7445634026751,66.3831125638945,59.8835709402897,66.77007949293,52.9256801386042,59.8760570193376,56.6187722865831,55.574337274235,55.7246156932779,59.8723000588615,58.647530943662,61.6493423640436,57.343865658465,67.5928538371898,62.2692408425955,61.4690082611922,59.5041179322065,55.7208587328018,67.187102105774,59.9587101498112,56.8479468756235,57.18607331847,65.0193359110803,54.8680287047335,55.5780942347111,52.5988245771859],"y":[14.3280142873523,9.05116980588413,6.23009914278928,5.88746092576277,5.88746092576277,5.74025774132847,5.72905811770875,5.72905811770875,5.47928501789688,5.46329158172863,5.41371641555828,5.10529625670382,5.01568232886942,3.37896228650259,3.35554632481727,3.35554632481727,3.35554632481727,3.29028665643699,3.19723863105258,3.1866172090753,3.18546863691839,2.9582953476124,2.43702362347858,2.36701902111538,2.28527398243072,2.10925620259951,2.08893630587289,2.02659595144777,2.00255057481687,1.98146122644125,1.97101721803704,1.95255878222706,1.92495559478395,1.74294224975137,1.74294224975137,1.73953823350252,1.6951241643316,1.62759947829704,1.60430454689107,1.60090973289814,1.5869055575054,1.53905293254249,1.52820831665151,1.44522364321249],"text":["GO:0005829 (5361) <br> cytosol <br> 4.699e-15","GO:0005737 (11951) <br> cytoplasm <br> 8.889e-10","GO:0070062 (2178) <br> extracellular exosome <br> 5.887e-07","GO:0097517 (70) <br> contractile actin filament bundle <br> 1.296e-06","GO:0001725 (70) <br> stress fiber <br> 1.296e-06","GO:1903561 (2263) <br> extracellular vesicle <br> 1.819e-06","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 1.866e-06","GO:0043230 (2265) <br> extracellular organelle <br> 1.866e-06","GO:0005622 (14808) <br> intracellular anatomical structure <br> 3.317e-06","GO:0032432 (78) <br> actin filament bundle <br> 3.441e-06","GO:0042641 (79) <br> actomyosin <br> 3.857e-06","GO:0031982 (4058) <br> vesicle <br> 7.847e-06","GO:0015629 (517) <br> actin cytoskeleton <br> 9.645e-06","GO:0005925 (420) <br> focal adhesion <br> 4.179e-04","GO:0070013 (5579) <br> intracellular organelle lumen <br> 4.410e-04","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 4.410e-04","GO:0043233 (5579) <br> organelle lumen <br> 4.410e-04","GO:0030055 (427) <br> cell-substrate junction <br> 5.125e-04","GO:0042995 (2337) <br> cell projection <br> 6.350e-04","GO:0120025 (2234) <br> plasma membrane bounded cell projection <br> 6.507e-04","GO:0043229 (13117) <br> intracellular organelle <br> 6.524e-04","GO:0140534 (128) <br> endoplasmic reticulum protein-containing complex <br> 1.101e-03","GO:0005615 (3595) <br> extracellular space <br> 3.656e-03","GO:0043227 (13201) <br> membrane-bounded organelle <br> 4.295e-03","GO:0031252 (426) <br> cell leading edge <br> 5.185e-03","GO:0019897 (178) <br> extrinsic component of plasma membrane <br> 7.776e-03","GO:0030017 (211) <br> sarcomere <br> 8.148e-03","GO:0043226 (14027) <br> organelle <br> 9.406e-03","GO:0034774 (320) <br> secretory granule lumen <br> 9.941e-03","GO:0060205 (324) <br> cytoplasmic vesicle lumen <br> 1.044e-02","GO:0031983 (326) <br> vesicle lumen <br> 1.069e-02","GO:1990184 (2) <br> amino acid transport complex <br> 1.115e-02","GO:0070161 (838) <br> anchoring junction <br> 1.189e-02","GO:0048770 (109) <br> pigment granule <br> 1.807e-02","GO:0042470 (109) <br> melanosome <br> 1.807e-02","GO:0030016 (234) <br> myofibril <br> 1.822e-02","GO:1904724 (55) <br> tertiary granule lumen <br> 2.018e-02","GO:0043292 (242) <br> contractile fiber <br> 2.357e-02","GO:0031410 (2433) <br> cytoplasmic vesicle <br> 2.487e-02","GO:0031674 (141) <br> I band <br> 2.507e-02","GO:0097708 (2438) <br> intracellular vesicle <br> 2.589e-02","GO:0010008 (520) <br> endosome membrane <br> 2.890e-02","GO:0019898 (315) <br> extrinsic component of membrane <br> 2.963e-02","GO:0001891 (27) <br> phagocytic cup <br> 3.587e-02"],"key":["GO:0005829","GO:0005737","GO:0070062","GO:0097517","GO:0001725","GO:1903561","GO:0065010","GO:0043230","GO:0005622","GO:0032432","GO:0042641","GO:0031982","GO:0015629","GO:0005925","GO:0070013","GO:0031974","GO:0043233","GO:0030055","GO:0042995","GO:0120025","GO:0043229","GO:0140534","GO:0005615","GO:0043227","GO:0031252","GO:0019897","GO:0030017","GO:0043226","GO:0034774","GO:0060205","GO:0031983","GO:1990184","GO:0070161","GO:0048770","GO:0042470","GO:0030016","GO:1904724","GO:0043292","GO:0031410","GO:0031674","GO:0097708","GO:0010008","GO:0019898","GO:0001891"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[21.5670744132122,22.4484604312869,20.5214902439301,15.717077397889,15.717077397889,20.5672559227953,20.5683104706776,20.5683104706776,22.6771653543307,15.8973863612294,15.9184363351098,21.250520850314,18.7010140522786,18.41927534827,21.6119263993467,21.6119263993467,21.6119263993467,18.4418855248262,20.6056241461277,20.5518518073364,22.5481301823792,16.6906051657902,21.1109970585917,22.5549451746932,18.4386804137039,17.1926950579195,17.4444452076508,22.6196160803639,18.042156911785,18.0596015863544,18.0682354374322,3.77952755905512,19.3361763823889,16.4387325196073,16.4387325196073,17.5953506241288,15.3051018762464,17.644028433943,20.6535046726708,16.8398886539314,20.6559427724165,18.7087830831335,18.0200111191535,13.9932800960407],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedDatafb95dc9b","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[27.1229623217858,9.35507778499651,9.05456388162587,36.1083280325681,3.75424991092613,3.64155719716214,19.2419852058907,28.9448278609704,28.7757887903244,7.31533966586827,37.997809200011,41.1269102188579,33.5464470063334,3.91953255777999,46.8817514684057],"y":[5.17889589283441,4.21564332685985,3.77183612711724,3.5539365463746,3.35849177219721,3.28128996119869,3.03361060901617,2.95059272421093,2.32196786218939,2.21460934772671,1.90100204082707,1.84992705571003,1.79931088767728,1.54597410871657,1.37575556968322],"text":["GO:0042802 (2063) <br> identical protein binding <br> 6.624e-06","GO:0008092 (1001) <br> cytoskeletal protein binding <br> 6.086e-05","GO:0005515 (14767) <br> protein binding <br> 1.691e-04","GO:0050839 (544) <br> cell adhesion molecule binding <br> 2.793e-04","GO:0003779 (449) <br> actin binding <br> 4.380e-04","GO:0003723 (1951) <br> RNA binding <br> 5.233e-04","GO:0019834 (5) <br> phospholipase A2 inhibitor activity <br> 9.255e-04","GO:0045296 (333) <br> cadherin binding <br> 1.120e-03","GO:0044877 (1304) <br> protein-containing complex binding <br> 4.765e-03","GO:0004859 (12) <br> phospholipase inhibitor activity <br> 6.101e-03","GO:0055102 (17) <br> lipase inhibitor activity <br> 1.256e-02","GO:0098641 (18) <br> cadherin binding involved in cell-cell adhesion <br> 1.413e-02","GO:0048306 (83) <br> calcium-dependent protein binding <br> 1.587e-02","GO:0003824 (5682) <br> catalytic activity <br> 2.845e-02","GO:1901363 (6210) <br> heterocyclic compound binding <br> 4.210e-02"],"key":["GO:0042802","GO:0008092","GO:0005515","GO:0050839","GO:0003779","GO:0003723","GO:0019834","GO:0045296","GO:0044877","GO:0004859","GO:0055102","GO:0098641","GO:0048306","GO:0003824","GO:1901363"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[20.4564289599961,19.5634860592025,22.6742247680722,18.7692301298177,18.5103934714585,20.3892145517334,9.83978094370953,18.0980022367888,19.8957455135249,12.254025725742,13.0411641506074,13.1640323425788,15.9997101118596,21.6324739822909,21.7319446023732],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedDatafb95dc9b","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"3280223177c6":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"328031004d5d":{"x":{},"xend":{},"y":{},"yend":{}},"32801e7c6958":{"x":{},"xend":{},"y":{},"yend":{}},"3280606bcbb":{"x":{},"xend":{},"y":{},"yend":{}},"3280118773bf":{"x":{},"xend":{},"y":{},"yend":{}},"32806f316814":{"x":{},"y":{}},"328075e12a3d":{"yintercept":{}}},"cur_data":"3280223177c6","visdat":{"3280223177c6":["function (y) ","x"],"328031004d5d":["function (y) ","x"],"32801e7c6958":["function (y) ","x"],"3280606bcbb":["function (y) ","x"],"3280118773bf":["function (y) ","x"],"32806f316814":["function (y) ","x"],"328075e12a3d":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedDatafb95dc9b"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results-1" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-56457827ab1876f7c579" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-56457827ab1876f7c579">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78"],["GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:MF","GO:MF","GO:MF","GO:CC","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","GO:MF","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:CC","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF"],[470,385,2743,3391,184,4056,2719,2131,284,1528,2061,6689,1453,837,1959,1879,9081,550,468,2720,1446,2134,19167,1027,438,2093,3860,2130,2450,4591,4320,3963,7174,24824,19397,278,2128,7129,1261,19430,1416,1038,983,9696,1023,2127,11657,1801,2600,1454,4182,2765,9584,8297,10417,23441,8399,18660,2451,2552,2029,1022,14361,4074,2150,7619,8761,12553,1322,1412,3497,512,795,984,12194,191,1175,11949],["cytosol","cytoplasm","extracellular exosome","contractile actin filament bundle","stress fiber","extracellular vesicle","extracellular membrane-bounded organelle","extracellular organelle","intracellular anatomical structure","actin filament bundle","actomyosin","identical protein binding","vesicle","actin cytoskeleton","cytoskeletal protein binding","protein binding","cell adhesion molecule binding","focal adhesion","actin binding","intracellular organelle lumen","membrane-enclosed lumen","organelle lumen","cellular response to chemical stimulus","cell-substrate junction","RNA binding","cell projection","plasma membrane bounded cell projection","intracellular organelle","protein folding","phospholipase A2 inhibitor activity","response to organic substance","endoplasmic reticulum protein-containing complex","cadherin binding","organonitrogen compound metabolic process","cellular response to organic substance","extracellular space","membrane-bounded organelle","protein-containing complex binding","cell leading edge","cellular response to cytokine stimulus","phospholipase inhibitor activity","immune system process","extrinsic component of plasma membrane","response to cytokine","sarcomere","organelle","response to chemical","secretory granule lumen","cytoplasmic vesicle lumen","vesicle lumen","amino acid transport complex","anchoring junction","lipase inhibitor activity","positive regulation of vesicle fusion","cadherin binding involved in cell-cell adhesion","positive regulation of receptor binding","calcium-dependent protein binding","protein-containing complex assembly","'de novo' protein folding","pigment granule","melanosome","myofibril","muscle cell cellular homeostasis","tertiary granule lumen","contractile fiber","regulation of cell morphogenesis","positive regulation of intracellular transport","protein-containing complex subunit organization","cytoplasmic vesicle","I band","intracellular vesicle","catalytic activity","endosome membrane","extrinsic component of membrane","regulation of I-kappaB kinase/NF-kappaB signaling","phagocytic cup","leukocyte differentiation","heterocyclic compound binding"],["4.7e-15","8.9e-10","5.9e-07","1.3e-06","1.3e-06","1.8e-06","1.9e-06","1.9e-06","3.3e-06","3.4e-06","3.9e-06","6.6e-06","7.8e-06","9.6e-06","6.1e-05","1.7e-04","2.8e-04","4.2e-04","4.4e-04","4.4e-04","4.4e-04","4.4e-04","4.6e-04","5.1e-04","5.2e-04","6.3e-04","6.5e-04","6.5e-04","7.2e-04","9.3e-04","9.5e-04","1.1e-03","1.1e-03","1.9e-03","2.7e-03","3.7e-03","4.3e-03","4.8e-03","5.2e-03","6.0e-03","6.1e-03","6.5e-03","7.8e-03","8.1e-03","8.1e-03","9.4e-03","9.6e-03","9.9e-03","1.0e-02","1.1e-02","1.1e-02","1.2e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.6e-02","1.6e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","2.0e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.5e-02","2.6e-02","2.8e-02","2.9e-02","3.0e-02","3.4e-02","3.6e-02","3.9e-02","4.2e-02"],[5361,11951,2178,70,70,2263,2265,2265,14808,78,79,2063,4058,517,1001,14767,544,420,449,5579,5579,5579,3443,427,1951,2337,2234,13117,230,5,3457,128,333,6904,2821,3595,13201,1304,426,1148,12,3438,178,1242,211,14027,4829,320,324,326,2,838,17,11,18,8,83,1724,40,109,109,234,21,55,242,312,227,2017,2433,141,2438,5682,520,315,250,27,566,6210],[91,130,41,9,9,41,41,41,144,9,9,41,57,19,26,143,16,15,16,72,72,72,51,15,37,35,34,130,10,2,55,7,12,63,47,44,113,21,10,26,2,46,3,27,9,133,64,4,4,4,2,19,2,2,2,2,6,32,4,6,6,9,4,3,9,9,10,35,33,7,33,51,4,9,10,2,13,74]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap-1" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp-1" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/BP-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf-1" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/MF-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc-1" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/CC-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot-1" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/scatter-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="network-analysis" class="section level2">
<h2><strong>6. Network analysis</strong></h2>
<p>Overview from <strong>STRING Database</strong>: In STRING, each protein-protein interaction is annotated with one or more ‘scores’. Importantly, these scores do <strong>not</strong> indicate the strength or the specificity of the interaction. Instead, they are indicators of <strong>confidence</strong>, i.e. how likely STRING judges an interaction to be true, given the available evidence. All scores rank from 0 to 1, with 1 being the highest possible confidence. A score of 0.5 would indicate that roughly every second interaction might be erroneous (i.e., a false positive).</p>
<pre class="r"><code>if(leng_dep != 0){
  
  string_db &lt;- STRINGdb$new(version=&quot;11&quot;, species=9606, score_threshold=400)

  string_mapped &lt;- string_db$map(string_res, &quot;Gene.name&quot;, removeUnmappedRows = TRUE, quiet = T)

  hits &lt;- string_mapped$STRING_id[1:leng_dep]

  string_db$plot_network(hits, add_link = T)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/CSCAP2_vs_CSCAP2_act.Rmd/stringdb-1.png" width="1440" style="display: block; margin: auto;" /></p>
<div id="cytoscape-network" class="section level3">
<h3><strong>6.1. Cytoscape Network</strong></h3>
<p>Cytoscape is an open source software platform for visualizing complex networks and integrating these with any type of attribute data. In this presentation all proteins showing at least one confidence level from the STRING database are shown. The <strong>color</strong> code is relative to <strong>fold change</strong> and the <strong>size</strong> is directly proportional to the <strong>significance (p-value)</strong>.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>if(n_edge != 0){
  knitr::include_graphics(paste0(&quot;assets/network_FC_pvalue&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_FC_pvalueSKAP2_C._vs_SKAP2_C_act.png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
<div id="cytoscape-enrichmentmap" class="section level3">
<h3><strong>6.2. Cytoscape EnrichmentMap</strong></h3>
<p>An <strong>enrichment map</strong> is a different sort of network. Instead of nodes representing genes, nodes represent pathways or functions. Edges between these pathways or functions represent shared genes or pathway crosstalk. An enrichment map is a way to visualize your enrichment results to help reduce redundancy and uncover main themes.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant) %&gt;% pull(name)

if(!params$Imputation){
  
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  pull(name) %&gt;% 
  unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  
  print(&quot;NOT enough data.&quot;)
  
}else{
  
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  
  if(is.null(GO)){
    
    print(&quot;NOT enough data.&quot;)

    enrich_map = F
    
  }else{
    
    enrich_map = T
    
    gem &lt;- GO$result[,c(&quot;term_id&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;intersection&quot;)]
    colnames(gem) &lt;- c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;Genes&quot;)
    gem$FDR &lt;- gem$p.Val
    gem$Phenotype = &quot;+1&quot;
    gem &lt;- gem[,c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;FDR&quot;, &quot;Phenotype&quot;, &quot;Genes&quot;)]
    head(gem)
  
    write.table(gem, here::here(&quot;output/enrichmentmap.txt&quot;),col.name=TRUE,sep=&quot;\t&quot;,row.names=FALSE,quote=FALSE)
    
  }
}</code></pre>
<pre class="r"><code>if(enrich_map){
  knitr::include_graphics(paste0(&quot;assets/network_enrichmentmap&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_enrichmentmapSKAP2_C._vs_SKAP2_C_act.png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="section" class="section level2 unnumbered">
<h2></h2>
<pre class="r"><code>saveWorkbook(excel, here::here(paste0(&quot;output/results_table_&quot;, test, &quot;.xlsx&quot;)), overwrite = T)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] forcats_0.5.1               stringr_1.4.0              
 [3] dplyr_1.0.7                 purrr_0.3.4                
 [5] readr_1.4.0                 tidyr_1.1.3                
 [7] tibble_3.1.2                tidyverse_1.3.1            
 [9] openxlsx_4.2.4              here_1.0.1                 
[11] RColorBrewer_1.1-2          RCy3_2.10.2                
[13] STRINGdb_2.2.2              naniar_0.6.1               
[15] visdat_0.5.3                ggrepel_0.9.1              
[17] ggplot2_3.3.5               UpSetR_1.4.0               
[19] ggstatsplot_0.8.0           datapasta_3.1.0            
[21] patchwork_1.1.1             DT_0.18                    
[23] rrvgo_1.2.0                 org.Hs.eg.db_3.12.0        
[25] AnnotationDbi_1.52.0        gprofiler2_0.2.0           
[27] enrichR_3.0                 preprocessCore_1.52.1      
[29] SummarizedExperiment_1.20.0 Biobase_2.50.0             
[31] GenomicRanges_1.42.0        GenomeInfoDb_1.26.2        
[33] IRanges_2.24.1              S4Vectors_0.28.1           
[35] BiocGenerics_0.36.0         MatrixGenerics_1.2.1       
[37] matrixStats_0.59.0          DEP_1.12.0                 
[39] MBQN_2.2.0                  workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.3            R.methodsS3_1.8.1        
  [3] coda_0.19-4               bit64_4.0.5              
  [5] knitr_1.33                DelayedArray_0.16.2      
  [7] R.utils_2.10.1            data.table_1.14.0        
  [9] RCurl_1.98-1.3            doParallel_1.0.16        
 [11] generics_0.1.0            RSQLite_2.2.7            
 [13] correlation_0.6.1         chron_2.3-56             
 [15] bit_4.0.4                 lubridate_1.7.10         
 [17] xml2_1.3.2                httpuv_1.6.1             
 [19] assertthat_0.2.1          WRS2_1.1-2               
 [21] xfun_0.24                 hms_1.1.0                
 [23] jquerylib_0.1.4           evaluate_0.14            
 [25] promises_1.2.0.1          fansi_0.5.0              
 [27] readxl_1.3.1              caTools_1.18.2           
 [29] dbplyr_2.1.1              igraph_1.2.6             
 [31] DBI_1.1.1                 htmlwidgets_1.5.3        
 [33] reshape_0.8.8             kSamples_1.2-9           
 [35] Rmpfr_0.8-4               paletteer_1.3.0          
 [37] hash_2.2.6.1              ellipsis_0.3.2           
 [39] crosstalk_1.1.1           backports_1.2.1          
 [41] insight_0.14.2            gridBase_0.4-7           
 [43] vctrs_0.3.8               imputeLCMD_2.0           
 [45] Cairo_1.5-12.2            cachem_1.0.5             
 [47] withr_2.4.2               treemap_2.4-2            
 [49] cluster_2.1.0             lazyeval_0.2.2           
 [51] crayon_1.4.1              labeling_0.4.2           
 [53] pkgconfig_2.0.3           SuppDists_1.1-9.5        
 [55] slam_0.1-48               statsExpressions_1.1.0   
 [57] ProtGenerics_1.22.0       wordcloud_2.6            
 [59] rlang_0.4.11              RJSONIO_1.3-1.4          
 [61] lifecycle_1.0.0           MatrixModels_0.5-0       
 [63] sandwich_3.0-1            affyio_1.60.0            
 [65] BiocFileCache_1.14.0      modelr_0.1.8             
 [67] cellranger_1.1.0          rprojroot_2.0.2          
 [69] graph_1.68.0              Matrix_1.2-18            
 [71] mc2d_0.1-21               zoo_1.8-9                
 [73] reprex_2.0.0              whisker_0.4              
 [75] GlobalOptions_0.1.2       pheatmap_1.0.12          
 [77] png_0.1-7                 viridisLite_0.4.0        
 [79] rjson_0.2.20              mzR_2.24.1               
 [81] PMCMRplus_1.9.0           parameters_0.14.0        
 [83] bitops_1.0-7              shinydashboard_0.7.1     
 [85] R.oo_1.24.0               KernSmooth_2.23-17       
 [87] blob_1.2.1                shape_1.4.6              
 [89] multcompView_0.1-8        tmvtnorm_1.4-10          
 [91] ggsignif_0.6.2            scales_1.1.1             
 [93] memoise_2.0.0             magrittr_2.0.1           
 [95] plyr_1.8.6                gplots_3.1.1             
 [97] zlibbioc_1.36.0           compiler_4.0.3           
 [99] plotrix_3.8-1             pcaMethods_1.82.0        
[101] clue_0.3-59               cli_3.0.0                
[103] affy_1.68.0               XVector_0.30.0           
[105] pbapply_1.4-3             MASS_7.3-53              
[107] tidyselect_1.1.1          vsn_3.58.0               
[109] stringi_1.6.2             highr_0.9                
[111] yaml_2.2.1                GOSemSim_2.16.1          
[113] norm_1.0-9.5              MALDIquant_1.19.3        
[115] grid_4.0.3                sass_0.4.0               
[117] tools_4.0.3               rstudioapi_0.13          
[119] circlize_0.4.13           foreach_1.5.1            
[121] git2r_0.28.0              gridExtra_2.3            
[123] ipmisc_6.0.2              pairwiseComparisons_3.1.6
[125] farver_2.1.0              mzID_1.28.0              
[127] digest_0.6.27             BiocManager_1.30.16      
[129] shiny_1.6.0               proto_1.0.0              
[131] BWStest_0.2.2             Rcpp_1.0.6               
[133] broom_0.7.8               BayesFactor_0.9.12-4.2   
[135] performance_0.7.2         later_1.2.0              
[137] ncdf4_1.17                httr_1.4.2               
[139] MSnbase_2.15.7            ComplexHeatmap_2.6.2     
[141] effectsize_0.4.5          colorspace_2.0-2         
[143] rvest_1.0.0               XML_3.99-0.6             
[145] fs_1.5.0                  rematch2_2.1.2           
[147] gmm_1.6-6                 plotly_4.9.4.1           
[149] xtable_1.8-4              gmp_0.6-2                
[151] jsonlite_1.7.2            zeallot_0.1.0            
[153] NLP_0.2-1                 R6_2.5.0                 
[155] tm_0.7-8                  gsubfn_0.7               
[157] pillar_1.6.1              htmltools_0.5.1.1        
[159] mime_0.11                 glue_1.4.2               
[161] fastmap_1.1.0             BiocParallel_1.24.1      
[163] codetools_0.2-16          mvtnorm_1.1-2            
[165] utf8_1.2.1                lattice_0.20-41          
[167] bslib_0.2.5.1             sqldf_0.4-11             
[169] curl_4.3.2                gtools_3.9.2             
[171] zip_2.2.0                 GO.db_3.12.1             
[173] limma_3.46.0              rmarkdown_2.9            
[175] munsell_0.5.0             GetoptLong_1.0.5         
[177] GenomeInfoDbData_1.2.4    iterators_1.0.13         
[179] impute_1.64.0             haven_2.4.1              
[181] gtable_0.3.0              bayestestR_0.10.0        </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
