<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit at IEO" />


<title>LFQ_Analysis</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.18/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.9.4.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Template_LFQ</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LFQ_Analysis</h1>
<h4 class="author">Fabio Bedin | MS-Unit at IEO</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-29
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>SKAP2_HaloTag/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20211221code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20211221)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20211221code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20211221)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong2865ca5"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 2865ca5 </a>
</p>
</div>
<div id="strongRepositoryversionstrong2865ca5" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 2865ca5. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  Counts.pdf
    Untracked:  Intensity_Distributions.pdf
    Untracked:  code/custom_functions.R
    Untracked:  data/expdesign.tsv
    Untracked:  data/proteinGroups.txt
    Untracked:  output/enrichmentmap.txt
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_vs_Halo.png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_vs_Halo.png
    Untracked:  output/output.zip
    Untracked:  output/results_table_Halo._vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_C._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_C._vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_C_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_C..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_N_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_N_vs_Halo.xlsx

Unstaged changes:
    Modified:   analysis/_site.yml
    Deleted:    analysis/about.Rmd
    Deleted:    analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/CSCAP2_act_vs_Halo_act.Rmd</code>) and HTML (<code>docs/CSCAP2_act_vs_Halo_act.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
2865ca5
</td>
<td>
FabioBedin
</td>
<td>
2021-12-29
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>## Proteomics
library(&quot;MBQN&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;enrichR&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)

## Plots
library(&quot;DT&quot;)
library(&quot;patchwork&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)

## Networks
library(&quot;STRINGdb&quot;)
library(&quot;RCy3&quot;)
library(&quot;RColorBrewer&quot;)

## Other
library(&quot;here&quot;)
library(&quot;openxlsx&quot;)
library(&quot;tidyverse&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(paste0(&quot;data/&quot;, params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot; &amp; data$Peptides &gt;= params$n_peptides,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<pre class="r"><code>cond_1 &lt;- params$test_1

cond_2 &lt;- params$test_2

conditions&lt;-c(cond_1,cond_2)

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)

expdesign &lt;- read.table(here::here(paste0(&quot;data/&quot;, params$expdesign)), header = T, stringsAsFactors = F)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
if(max(expdesign$replicate)&lt;=3){
       threshold&lt;-0
     } else if(max(expdesign$replicate)&lt;6){
       threshold&lt;-1
     } else if (max(expdesign$replicate)&gt;=6){
       threshold&lt;-trunc(max(expdesign$replicate)*0.25) ## 0.25 serve per avere sempre il 75% di valid vales
     }

data_filt &lt;- filter_missval(data_se, thr = threshold)</code></pre>
<div id="skap2_c_act-vs-halo_act" class="section level1">
<h1><strong>SKAP2_C_act</strong> vs <strong>Halo_act</strong></h1>
<hr />
</div>
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>First, we need to wrangling the original dataset. From the <strong>proteinGroups</strong> obtained by <a href="https://www.maxquant.org/">MaxQuant</a> software, I remove proteins that are marked form the software as potential contaminant, only identify by site and reverse.<br />
We che also filter the initial datasets by excluding all proteins identified by less than 2 peptides.<br />
Now, we generate some quality control plots to investigate the structure of the dataset and observe the effect of filters.</p>
<div id="normalization-strategy" class="section level4">
<h4><strong>Normalization strategy:</strong></h4>
<pre class="r"><code>if(params$Normalization == &quot;VSN&quot;){
  data_filt &lt;- normalize_vsn(data_filt)
  print(&quot;VSN normalization is used&quot;)
} else if(params$Normalization == &quot;MBQN&quot;){
  data_filt &lt;- normalize_MBQN(data_filt)
  print(&quot;MBQN normalization is used&quot;)
} else {
  print(&quot;Max-LFQ normalization is used&quot;)
}</code></pre>
<p>[1] “Max-LFQ normalization is used”</p>
</div>
<div id="quality-control-plots" class="section level2 tabset tabset-fade">
<h2><strong>1. Quality control plots</strong></h2>
<div id="proteins-per-samples" class="section level3 tabset">
<h3>Proteins per samples</h3>
<p>This plot show the number of proteins identify in each samples <em>after and before User define</em> filters:</p>
<div id="filtered" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_filt)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/proteins-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_se)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/proteins-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="distributions" class="section level3 tabset">
<h3>Distributions</h3>
<p>This plot is useful for checking the distributions of the samples <em>after and before User define</em> filters:</p>
<div id="filtered-1" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_normalization(data_filt)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/normalization-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-1" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_normalization(data_se)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/normalization-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="missing-data" class="section level3 tabset">
<h3>Missing data</h3>
<p>This plot we can explore the missing data pattern <em>after and before User define</em> filters. Notice that the percentages of missingness are provided in the data. These are accurate to 1 decimal place. Also the dataset is arranged by columns with most missingness.</p>
<div id="filtered-2" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/missing-data-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-2" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>assay(data_se) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/missing-data-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample-cvs" class="section level3 tabset">
<h3>Sample CVs</h3>
<p>The coefficient of variation (CV) is a statistical measure of the dispersion of data points in a data series around the mean. The coefficient of variation represents the ratio of the standard deviation to the mean, and it is a useful statistic for comparing the degree of variation from one data series to another, even if the means are drastically different from one another.</p>
<div id="filtered-3" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_cvs(data_filt)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/CVs-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-3" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_cvs(data_se)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/CVs-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="upset-plot" class="section level2 tabset tabset-fade">
<h2><strong>2. Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within the 2 conditions or in common between all condition.</p>
<div id="skap2_c_act" class="section level3">
<h3>SKAP2_C_act</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_1)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_1)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/upset-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="halo_act" class="section level3">
<h3>Halo_act</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_2)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_2)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/upset-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="both-conditions" class="section level3">
<h3>Both conditions</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()
n_sets &lt;- length(define_set)
n_each &lt;- n_sets / 2

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;red3&quot;, &quot;royalblue1&quot;), each = n_each),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/upset-both-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="imputation" class="section level2">
<h2><strong>3. Imputation</strong></h2>
<p>An idealized version of a label-free discovery mass spectrometry proteomics experiment would provide absolute abundance measurements for a whole proteome, across varying conditions. Unfortunately, this ideal is not realized. Measurements are made on peptides requiring an inferential step to obtain protein level estimates. The inference is complicated by experimental factors that necessitate relative abundance estimation and result in widespread non-ignorable missing data. Relative abundance on the log scale takes the form of parameter contrasts. In a complete-case analysis, contrast estimates may be biased by missing data and a substantial amount of useful information will often go unused.</p>
<pre class="r"><code>data_imputed &lt;- mix_imputation_mean(data_filt, cond1 = cond_1, cond2 = cond_2)

plot_imputation(data_filt, data_imputed)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/imputation-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>data_filt &lt;- data_imputed</code></pre>
</div>
<div id="differential-enrichment-analysis" class="section level2">
<h2><strong>4. Differential enrichment analysis</strong></h2>
<p>Protein-wise linear models combined with empirical Bayes statistics are used for the differential enrichment analysis (or differential expression analysis) and False discovery rate (FDR) is ajusted with <em>Benjamini-Hochberg</em> procedure. Significant proteins are are define with <strong>FDR = 0.05</strong> and <strong>Fold change = 1</strong></p>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results&lt;-get_results(dep)</code></pre>
<p>There are <strong>104</strong> significant proteins in <strong>SKAP2_C_act</strong> vs <strong>Halo_act</strong> comparison.</p>
<p>The results from the previous analysis can be easily visualized by a number of functions. These visualizations assist in the determination of the optimal cutoffs to be used, highlight the most interesting samples and contrasts, and pinpoint differentially enriched/expressed proteins.</p>
<div id="visualization-of-the-results" class="section level3 tabset tabset-fade">
<h3><strong>4.1. Visualization of the results</strong></h3>
<div id="pca" class="section level4">
<h4>PCA</h4>
<p>The PCA plot can be used to get a high-level overview of the data. This can be very useful to observe batch effects, such as clear differences between replicates.</p>
<pre class="r"><code>if(params$Imputation){
  if(nrow(dep) &gt;= 500){
    n_value &lt;- 500
  }else {
    n_value &lt;- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
} else {
  var &lt;- apply(assay(dep), 1, sd)
  df &lt;- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value &lt;- df %&gt;% as.data.frame() %&gt;% drop_na() %&gt;% nrow()
  if(n_value &gt;= 500){
    n_value &lt;- 500
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
}</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/PCA-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano" class="section level4">
<h4>Volcano</h4>
<p>Volcano plots allows to inspect the enrichment of proteins between the two samples (x axis) and their corresponding adjusted p value (y axis).</p>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T, label_size=5, adjusted = F)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/volcano-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-and-unique" class="section level4">
<h4>Volcano and unique</h4>
<pre class="r"><code>plot_volcano_2(dep, contrast=test, add_names=T, label_size=5, adjusted = F, unique_vec = &quot;SKAP2&quot;)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/volcano-unique-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="correlation" class="section level4">
<h4>Correlation</h4>
<pre class="r"><code>plot_cor(dep, significant = TRUE, lower = 0, upper = 1, pal = &quot;Reds&quot;)</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/correlation-matrix-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-heatmap" class="section level4">
<h4>Cluster heatmap</h4>
<p>The heatmap representation gives an overview of all significant proteins (rows) in all samples (columns). This allows to see general trends, for example if one sample or replicate is really different compared to the others. Additionally, the clustering of samples (columns) can indicate closer related samples and clustering of proteins (rows) indicates similarly behaving proteins.</p>
<pre class="r"><code>if(results %&gt;% filter(significant) %&gt;% nrow() &lt;= 100){
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = T, indicate = &quot;condition&quot;, col_limit = 5)
}else {
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = F, indicate = &quot;condition&quot;, col_limit = 5)
}</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/cluster-heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="result-table" class="section level3">
<h3><strong>4.2. Result table</strong></h3>
<p>In this table are summarized the results of DEP analysis, sorted by p.value. Proteins that are UP regulated are colored with a red box in Fold change column, in the orhter hand proteins that are DOWN regulated are colored with blue.</p>
<pre class="r"><code>results %&gt;% 
  select(starts_with(&quot;name&quot;) | starts_with(test)) %&gt;% 
  filter(across(ends_with(&quot;significant&quot;))) %&gt;% 
  arrange(across(ends_with(&quot;p.adj&quot;))) %&gt;%
  mutate(across(2:3, format, scientific = T, digits = 2)) %&gt;%
  dplyr::rename_with(~ tolower(gsub(paste0(test,&quot;_&quot;), &quot;&quot;, .x)), .cols = starts_with(test)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(Fold.Change = ratio, Gene.name = name) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:5)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;Fold.Change&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-2a0dd1ba32e0541469b5" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2a0dd1ba32e0541469b5">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104"],["ACOT9","VPS28","CBLL1","INF2","IRF3","SNAP29","A0A3B3ITJ4","ZC3H15","ZNF112","TNPO3","HIST1H2AA","ARAP1","MRPS23","GNG4","HNRNPH3","PSPH","SNUPN","SH2D1A","WDR82","CSNK1G2","APIP","MRPS17","TYMS","KCTD10","G3BP2","MRPS14","ASNA1","TMED5","DOCK11","IGF2BP1","UBC","C1orf21","LRRC8C","FYB","HRAS","EEFSEC","HLA-C.1","RALA","POLR2B","SKAP2","FYN","INPP5A","CCM2","CNN2","PPA2","DSC1","UTP3","GNAI3","STAU1","RNPS1","CYBRD1","PITRM1","LCK","LAMTOR1","ULBP2","LAT","GNAI2","GLIPR2","ISOC2","GNB2","ATP1B3","LGALS8","MYO1G","BST2","CD48","LAMTOR3","GNA13","TMEM43","HMGA1","RFTN1","GNG5","CDC5L","PPP1CC","RAP1A","BANF1","RRAS2","GNG2","STXBP2","DSP","SF3B2","UPP1","COX5B","TMED10","AP2B1","HMGCL","KIFC1","B2M","RAP2C","GNB1","PAG1","MCM2","UBXN1","EPB41L2","IMMT.1","VCL","EFR3A","GNAS","ACTA2","YES1","SNAP23","LGALS9","VRK1","STOM","PHB"],["5.5e-12","1.5e-11","3.4e-11","3.2e-11","5.7e-11","9.3e-11","1.8e-10","2.9e-10","2.6e-10","4.4e-10","5.7e-10","9.0e-10","1.6e-09","1.9e-09","2.4e-09","4.7e-09","9.8e-09","1.1e-08","1.6e-08","1.9e-08","3.1e-08","1.0e-07","1.2e-07","1.3e-07","6.3e-07","7.7e-07","1.2e-06","1.4e-06","2.1e-06","3.6e-06","1.3e-05","1.6e-05","1.6e-05","1.9e-05","2.1e-05","2.4e-05","2.7e-05","2.9e-05","3.1e-05","7.4e-05","7.7e-05","8.0e-05","9.5e-05","9.6e-05","1.1e-04","1.3e-04","1.9e-04","2.0e-04","2.3e-04","2.5e-04","2.8e-04","2.8e-04","3.3e-04","3.9e-04","3.9e-04","4.2e-04","4.3e-04","4.7e-04","4.6e-04","4.8e-04","5.5e-04","5.9e-04","6.3e-04","6.9e-04","6.8e-04","6.9e-04","7.5e-04","7.7e-04","8.2e-04","9.3e-04","1.0e-03","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.2e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.7e-03","1.9e-03","1.9e-03","2.1e-03","2.2e-03","2.2e-03","2.3e-03","2.5e-03","2.6e-03","2.6e-03","2.7e-03","2.7e-03","2.7e-03","2.8e-03","2.8e-03","2.9e-03","2.9e-03","3.2e-03","3.3e-03","3.4e-03","3.6e-03","3.7e-03","3.8e-03","3.8e-03"],["1.2e-08","1.6e-08","1.8e-08","1.8e-08","2.4e-08","3.4e-08","5.5e-08","7.0e-08","7.0e-08","9.5e-08","1.1e-07","1.6e-07","2.6e-07","2.9e-07","3.5e-07","6.4e-07","1.2e-06","1.3e-06","1.8e-06","2.1e-06","3.2e-06","1.0e-05","1.2e-05","1.2e-05","5.4e-05","6.4e-05","9.3e-05","1.1e-04","1.6e-04","2.5e-04","8.6e-04","1.0e-03","1.0e-03","1.2e-03","1.2e-03","1.4e-03","1.6e-03","1.6e-03","1.7e-03","3.7e-03","3.8e-03","3.8e-03","4.4e-03","4.4e-03","4.8e-03","5.6e-03","7.9e-03","8.0e-03","8.9e-03","9.7e-03","1.0e-02","1.0e-02","1.1e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.6e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","2.1e-02","2.2e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.4e-02","2.5e-02","2.5e-02","2.6e-02","2.7e-02","3.0e-02","3.2e-02","3.2e-02","3.6e-02","3.6e-02","3.6e-02","3.6e-02","3.9e-02","4.0e-02","4.0e-02","4.0e-02","4.0e-02","4.1e-02","4.1e-02","4.1e-02","4.2e-02","4.2e-02","4.5e-02","4.6e-02","4.6e-02","4.8e-02","4.8e-02","4.9e-02","5.0e-02"],["True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True"],[4.05,-4.85,6.05,-3.01,-5.1,-2.95,3.78,3.79,-7.76,-3.53,3.92,-2.47,-2.6,-5.96,-3.65,-3.77,3.12,-4.02,-2.08,-5.18,3.22,-2.95,-3.81,-2.04,-2.91,-3.56,-3.1,-4.03,-4.5,-3.29,-1.07,-3.08,1.8,11.4,-2.95,3.3,-6.08,-4.02,-1.4,8.73,-3.59,-2.83,-1.77,-1.12,1.05,-3.22,-1.58,-4.12,-1.15,-1.64,-1.96,1.03,-5.16,-2.01,-4.34,-3.07,-3.39,-4.07,1.51,-3.01,-3.05,-3.74,-3.35,-4.11,-3.91,-2.11,-4.06,-1.51,-1.19,-4.53,-4.86,-1.35,-1.17,-1.84,-2.79,-5.25,-3.81,-4.55,-3.2,-1.59,1.18,-1.76,-2.3,-1.24,1.39,1.04,-2.12,-2.69,-2.34,-4.1,-1.15,1.07,-5.04,-3.33,-3.34,-3.9,-1.52,-1.24,-3.37,-3.54,-1.67,-1.04,-3.91,-2.91]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene.name<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>Fold.Change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[5]; $(this.api().cell(row, 5).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[4]; $(this.api().cell(row, 4).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
<div id="gene-ontology" class="section level2">
<h2><strong>5. Gene Ontology</strong></h2>
<p>Now I perfom a gene onology analysis (GO or by branch <a href="GO:MF" class="uri">GO:MF</a>, <a href="GO:BP" class="uri">GO:BP</a>, <a href="GO:CC" class="uri">GO:CC</a>) and a REAC ontology.<br />
The enrichment results are visualized with an interactive <strong>Manhattan-like-plot</strong> named “Gostplot”.</p>
<div id="skap2_c_act-1" class="section level3 tabset tabset-fade">
<h3><strong>5.1. SKAP2_C_act</strong></h3>
<div id="gostplot" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in SKAP2_C_act were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &gt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_1)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-ed4906484ed42468d86d" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-ed4906484ed42468d86d">{"x":{"data":[{"x":[53.3990571585892],"y":[1.74499045167017],"text":"GO:0005759 (482) <br> mitochondrial matrix <br> 1.799e-02","key":["GO:0005759"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":22.6771653543307,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isSimpleKey":true,"_isNestedKey":false,"frame":null},{"x":[29.3993551398185],"y":[1.36942021231003],"text":"GO:0046570 (1) <br> methylthioribulose 1-phosphate dehydratase activity <br> 4.271e-02","key":["GO:0046570"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":3.77952755905512,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isSimpleKey":true,"_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"225c35c1ca5":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"225c9053894":{"x":{},"xend":{},"y":{},"yend":{}},"225c61972d8":{"x":{},"xend":{},"y":{},"yend":{}},"225c19e777a0":{"x":{},"xend":{},"y":{},"yend":{}},"225c15bf545f":{"x":{},"xend":{},"y":{},"yend":{}},"225c6a374136":{"x":{},"y":{}},"225c1fae3e74":{"yintercept":{}}},"cur_data":"225c35c1ca5","visdat":{"225c35c1ca5":["function (y) ","x"],"225c9053894":["function (y) ","x"],"225c61972d8":["function (y) ","x"],"225c19e777a0":["function (y) ","x"],"225c15bf545f":["function (y) ","x"],"225c6a374136":["function (y) ","x"],"225c1fae3e74":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData2a0dd1ba"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;%
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-e4a421697d58abce2053" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-e4a421697d58abce2053">{"x":{"filter":"none","data":[["1","2"],["GO:CC","GO:MF"],[404,7295],["mitochondrial matrix","methylthioribulose 1-phosphate dehydratase activity"],["1.8e-02","4.3e-02"],[482,1],[4,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<pre><code>[1] &quot;NOT enough data.&quot;</code></pre>
</div>
<div id="mf" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<pre><code>[1] &quot;NOT enough data.&quot;</code></pre>
</div>
<div id="cc" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<pre><code>[1] &quot;NOT enough data.&quot;</code></pre>
</div>
</div>
<div id="scatterplot" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<pre><code>[1] &quot;NOT enough data.&quot;</code></pre>
</div>
</div>
<div id="halo_act-1" class="section level3 tabset tabset-fade">
<h3><strong>5.2. Halo_act</strong></h3>
<div id="gostplot-1" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in Halo_act were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &lt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_2)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-a2454929a56e939b13f9" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-a2454929a56e939b13f9">{"x":{"data":[{"x":[74.4931449053838,75.8266105472218,83.2827494177805,76.5102464537415,83.0648874255928,118.467461156079,75.9130040958479,76.5140026949861,115.759211218712,121.750416003872,101.48549448918,76.2548220491078,75.8679292009126,132.688590508187,122.189896229492,118.516292292259,75.9430540258048,75.9092478546033,76.5215151774754,115.87941093854,132.658540578231,113.629422433016,87.3958335806327,94.0293556186209,159.185116247694,93.6424627704257,76.2848719790647,85.4200506859658,85.4313194096996,130.186933839275,130.442358243908,106.793063367819,84.2856658300924,130.078002843181,111.807645429379,84.289422071337,82.8244879859375,84.0377539079478,131.336343660126,126.795047995388,84.1166349740847,129.923996952152,83.267724452802,128.297544493234,131.501618274889,130.382258383994,101.200020154589,81.5886846164596,130.08551532567,84.026485184214,145.501129393566,84.0302414254586,106.601495064344,166.051525242848,83.9062854643864,145.497373152321,130.295864835368,110.121093110547,83.9476041180771,87.4071023043665,128.507894002932,171.516856253762,171.513100012517,105.346910488643,77.3892069049812,145.640110319616,165.923813040531,110.117336869302],"y":[7.8457164211234,6.61479471501932,5.90557643242774,5.7870286631977,5.67349517574875,5.66136964473701,5.50040149189901,5.43283915355031,4.98865748090829,4.93974545307565,4.93741525828741,4.68114983322312,4.64566056659311,4.47301247043442,4.43382880141204,4.41212294586389,4.35594654793903,4.29765251407728,4.22377842543205,4.20210549140983,4.13635051669171,4.12702680553625,4.07797348152853,4.0121252935412,3.98403300013637,3.95158414597778,3.92049380530874,3.75537091538199,3.71750463622624,3.68940098189547,3.67845996998786,3.5931987692207,3.41742708168943,3.29400866932314,3.26795797417682,3.15357125135561,2.9404800966175,2.93540814542878,2.91303444082914,2.89726727931841,2.85096044183491,2.78657537414278,2.71971733654622,2.59843948587571,2.56647759057639,2.46515200310924,2.43840718298539,2.36765501944505,2.33438809952696,2.30279486447684,2.18547875751581,2.01018766342726,1.96470083745431,1.95019176630424,1.87980741263843,1.83289384190437,1.77884646593408,1.66014268155118,1.61313423392058,1.61137962890038,1.60029830653322,1.55182232889804,1.53187990094895,1.52597242806578,1.39682589453292,1.35700366921299,1.34468861636613,1.32978553936905],"text":["GO:0001775 (1507) <br> cell activation <br> 1.427e-08","GO:0002252 (1135) <br> immune effector process <br> 2.428e-07","GO:0006955 (2544) <br> immune response <br> 1.243e-06","GO:0002443 (888) <br> leukocyte mediated immunity <br> 1.633e-06","GO:0006887 (906) <br> exocytosis <br> 2.121e-06","GO:0043299 (538) <br> leukocyte degranulation <br> 2.181e-06","GO:0002275 (552) <br> myeloid cell activation involved in immune response <br> 3.159e-06","GO:0002444 (558) <br> myeloid leukocyte mediated immunity <br> 3.691e-06","GO:0042060 (555) <br> wound healing <br> 1.026e-05","GO:0045055 (790) <br> regulated exocytosis <br> 1.149e-05","GO:0030168 (163) <br> platelet activation <br> 1.155e-05","GO:0002366 (723) <br> leukocyte activation involved in immune response <br> 2.084e-05","GO:0002263 (727) <br> cell activation involved in immune response <br> 2.261e-05","GO:0051649 (2778) <br> establishment of localization in cell <br> 3.365e-05","GO:0045321 (1341) <br> leukocyte activation <br> 3.683e-05","GO:0043312 (483) <br> neutrophil degranulation <br> 3.871e-05","GO:0002283 (488) <br> neutrophil activation involved in immune response <br> 4.406e-05","GO:0002274 (671) <br> myeloid leukocyte activation <br> 5.039e-05","GO:0002446 (500) <br> neutrophil mediated immunity <br> 5.973e-05","GO:0042119 (502) <br> neutrophil activation <br> 6.279e-05","GO:0051641 (3519) <br> cellular localization <br> 7.305e-05","GO:0036230 (509) <br> granulocyte activation <br> 7.464e-05","GO:0009611 (680) <br> response to wounding <br> 8.357e-05","GO:0016192 (2193) <br> vesicle-mediated transport <br> 9.725e-05","GO:0140352 (1413) <br> export from cell <br> 1.037e-04","GO:0016032 (1011) <br> viral process <br> 1.118e-04","GO:0002376 (3438) <br> immune system process <br> 1.201e-04","GO:0007596 (351) <br> blood coagulation <br> 1.756e-04","GO:0007599 (355) <br> hemostasis <br> 1.916e-04","GO:0050817 (358) <br> coagulation <br> 2.045e-04","GO:0050896 (9487) <br> response to stimulus <br> 2.097e-04","GO:0032940 (1351) <br> secretion by cell <br> 2.552e-04","GO:0007264 (520) <br> small GTPase mediated signal transduction <br> 3.824e-04","GO:0050776 (1159) <br> regulation of immune response <br> 5.081e-04","GO:0035556 (2886) <br> intracellular signal transduction <br> 5.396e-04","GO:0007265 (352) <br> Ras protein signal transduction <br> 7.021e-04","GO:0006810 (5214) <br> transport <br> 1.147e-03","GO:0007191 (10) <br> adenylate cyclase-activating dopamine receptor signaling pathway <br> 1.160e-03","GO:0051179 (6938) <br> localization <br> 1.222e-03","GO:0046903 (1492) <br> secretion <br> 1.267e-03","GO:0007212 (41) <br> dopamine receptor signaling pathway <br> 1.409e-03","GO:0050690 (29) <br> regulation of defense response to virus by virus <br> 1.635e-03","GO:0006950 (4321) <br> response to stress <br> 1.907e-03","GO:0048518 (6474) <br> positive regulation of biological process <br> 2.521e-03","GO:0051234 (5364) <br> establishment of localization <br> 2.713e-03","GO:0050878 (519) <br> regulation of body fluid levels <br> 3.426e-03","GO:0023052 (6937) <br> signaling <br> 3.644e-03","GO:0006457 (230) <br> protein folding <br> 4.289e-03","GO:0050778 (757) <br> positive regulation of immune response <br> 4.630e-03","GO:0007188 (236) <br> adenylate cyclase-modulating G protein-coupled receptor signaling pathway <br> 4.980e-03","GO:0071380 (17) <br> cellular response to prostaglandin E stimulus <br> 6.524e-03","GO:0007189 (148) <br> adenylate cyclase-activating G protein-coupled receptor signaling pathway <br> 9.768e-03","GO:0032879 (2906) <br> regulation of localization <br> 1.085e-02","GO:1901699 (728) <br> cellular response to nitrogen compound <br> 1.122e-02","GO:0007154 (6961) <br> cell communication <br> 1.319e-02","GO:0071379 (22) <br> cellular response to prostaglandin stimulus <br> 1.469e-02","GO:0050852 (202) <br> T cell receptor signaling pathway <br> 1.664e-02","GO:0034695 (25) <br> response to prostaglandin E <br> 2.187e-02","GO:0007165 (6447) <br> signal transduction <br> 2.437e-02","GO:0009615 (366) <br> response to virus <br> 2.447e-02","GO:0048584 (2368) <br> positive regulation of response to stimulus <br> 2.510e-02","GO:1903351 (86) <br> cellular response to dopamine <br> 2.807e-02","GO:1903350 (87) <br> response to dopamine <br> 2.938e-02","GO:0032418 (79) <br> lysosome localization <br> 2.979e-02","GO:0002682 (1734) <br> regulation of immune system process <br> 4.010e-02","GO:0071417 (671) <br> cellular response to organonitrogen compound <br> 4.395e-02","GO:1901655 (97) <br> cellular response to ketone <br> 4.522e-02","GO:0034694 (32) <br> response to prostaglandin <br> 4.680e-02"],"key":["GO:0001775","GO:0002252","GO:0006955","GO:0002443","GO:0006887","GO:0043299","GO:0002275","GO:0002444","GO:0042060","GO:0045055","GO:0030168","GO:0002366","GO:0002263","GO:0051649","GO:0045321","GO:0043312","GO:0002283","GO:0002274","GO:0002446","GO:0042119","GO:0051641","GO:0036230","GO:0009611","GO:0016192","GO:0140352","GO:0016032","GO:0002376","GO:0007596","GO:0007599","GO:0050817","GO:0050896","GO:0032940","GO:0007264","GO:0050776","GO:0035556","GO:0007265","GO:0006810","GO:0007191","GO:0051179","GO:0046903","GO:0007212","GO:0050690","GO:0006950","GO:0048518","GO:0051234","GO:0050878","GO:0023052","GO:0006457","GO:0050778","GO:0007188","GO:0071380","GO:0007189","GO:0032879","GO:1901699","GO:0007154","GO:0071379","GO:0050852","GO:0034695","GO:0007165","GO:0009615","GO:0048584","GO:1903351","GO:1903350","GO:0032418","GO:0002682","GO:0071417","GO:1901655","GO:0034694"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[20.0746712736595,19.7222049058623,20.706407382327,19.4106572227896,19.4363650611617,18.7543947502514,18.788735757365,18.8031640286352,18.7959711236385,19.2600018955208,17.0605109063983,19.1448417015984,19.1520361624938,20.8102706711732,19.9305008488025,18.609367044494,18.6232781866,19.0471784086146,18.6560402520018,18.6614171464117,21.0862719030328,18.6800539816722,19.0646575907216,20.5297041084802,19.9952668691908,19.5761024288984,21.0592849240276,18.171496771787,18.1872674403198,18.1989681011075,22.1989315786353,19.9397170643226,18.7087830831335,19.7484871750406,20.8550932356453,18.1754578485376,21.5357213433502,11.8112973047217,21.8552699322811,20.0623636974978,14.7824526398553,14.1325441121217,21.3224068576028,21.7783613472339,21.5677047149222,18.7061988463933,21.8551101159863,17.5703173351792,19.2046630030309,17.6076905446195,13.0411641506074,16.9140277546203,20.8631967480583,19.1538280687713,21.8589389746917,13.5832101454551,17.380365289039,13.8411375859653,21.773707326355,18.2296517587411,20.6213122635352,16.0578025627736,16.0766579870883,15.9184363351098,20.2463304681304,19.0471784086146,16.2527164787856,14.3213782810468],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData32e05414","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[67.2359425919629,53.6620443919143,56.5661748399181,62.186587712122,67.1194768172047,59.8873279007658,62.0964206606962,54.8304590999727,55.574337274235,65.259781381549,57.3401086979889,55.5780942347111,65.624206547728,61.0369578064439,62.7576457044849,65.5114977334458,52.9256801386042,65.2222117767882,56.1078256618373,59.5492014579193,53.9475733880958,53.3276749095439,55.7396435351822,55.7358865747061,53.8160797714333,56.8479468756235,65.0193359110803,62.2692408425955,63.648045337314,52.7866726009895,65.3386775515465,55.938762440414,56.0815269385048,65.5077407729697,56.4534660256359,57.1935872394221,65.6091787058237,53.4215989214457,57.7796730736894,55.0746615309174,66.0825557258088,53.4516546052543,53.4178419609696,52.1329614781529,54.950681835207,52.9482219014606,58.8842194536546,59.5529584183954,53.4253558819218,59.8760570193376,54.9544387956831,53.4478976447782,58.8767055327024,66.9128439910207,53.64701655001,56.0514712546962,62.761402664961,55.7471574561343,59.8723000588615],"y":[15.0977447282841,15.0977447282841,12.899992477924,11.2772068297669,10.7528870047927,10.7408398848099,10.7408398848099,10.4845946772709,10.2355611368606,9.82115089322416,8.12770455588233,7.75473124265858,7.58809870264857,7.58809870264857,7.00166230793699,6.76972561604096,6.59818159128586,6.0911732594956,5.36109566409726,5.33088032407414,5.10930182950231,5.02987985674859,5.02505479097463,4.98727547905639,4.89080328494219,4.80276411817895,4.78301747373916,4.62960412654921,4.28492565557438,4.11703816085824,4.05286804994948,4.05277413597258,3.99020026283341,3.8550836231825,3.66692032242153,3.58799887892011,3.43240109933365,3.43240109933365,3.19015554224688,3.13784552224946,3.02088926506615,2.9272695053546,2.55233884739254,2.55233884739254,2.43075917443266,2.35879950089709,2.31730621209442,2.25517131578822,2.25517131578822,2.22799288984774,2.17632948843689,2.0215981412715,1.85190836239282,1.80322070460494,1.75555262708707,1.54274776965646,1.45372950882594,1.40678182318932,1.33402314479374],"text":["GO:1905360 (36) <br> GTPase complex <br> 7.985e-16","GO:0005834 (36) <br> heterotrimeric G-protein complex <br> 7.985e-16","GO:0031234 (103) <br> extrinsic component of cytoplasmic side of plasma membrane <br> 1.259e-13","GO:0070062 (2178) <br> extracellular exosome <br> 5.282e-12","GO:1903561 (2263) <br> extracellular vesicle <br> 1.766e-11","GO:0043230 (2265) <br> extracellular organelle <br> 1.816e-11","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 1.816e-11","GO:0009898 (169) <br> cytoplasmic side of plasma membrane <br> 3.276e-11","GO:0019897 (178) <br> extrinsic component of plasma membrane <br> 5.814e-11","GO:0098562 (195) <br> cytoplasmic side of membrane <br> 1.510e-10","GO:0031982 (4058) <br> vesicle <br> 7.452e-09","GO:0019898 (315) <br> extrinsic component of membrane <br> 1.759e-08","GO:0098857 (339) <br> membrane microdomain <br> 2.582e-08","GO:0045121 (339) <br> membrane raft <br> 2.582e-08","GO:0070820 (163) <br> tertiary granule <br> 9.962e-08","GO:0098797 (696) <br> plasma membrane protein complex <br> 1.699e-07","GO:0005615 (3595) <br> extracellular space <br> 2.522e-07","GO:0098552 (622) <br> side of membrane <br> 8.106e-07","GO:0030667 (308) <br> secretory granule membrane <br> 4.354e-06","GO:0042581 (158) <br> specific granule <br> 4.668e-06","GO:0005925 (420) <br> focal adhesion <br> 7.775e-06","GO:0005737 (11951) <br> cytoplasm <br> 9.335e-06","GO:0030055 (427) <br> cell-substrate junction <br> 9.439e-06","GO:0030054 (2107) <br> cell junction <br> 1.030e-05","GO:0005886 (5680) <br> plasma membrane <br> 1.286e-05","GO:0031410 (2433) <br> cytoplasmic vesicle <br> 1.575e-05","GO:0097708 (2438) <br> intracellular vesicle <br> 1.648e-05","GO:0070161 (838) <br> anchoring junction <br> 2.346e-05","GO:0071944 (6178) <br> cell periphery <br> 5.189e-05","GO:0005576 (4564) <br> extracellular region <br> 7.638e-05","GO:0098588 (2165) <br> bounding membrane of organelle <br> 8.854e-05","GO:0030141 (857) <br> secretory granule <br> 8.856e-05","GO:0030659 (789) <br> cytoplasmic vesicle membrane <br> 1.023e-04","GO:0098796 (1326) <br> membrane protein complex <br> 1.396e-04","GO:0031090 (3644) <br> organelle membrane <br> 2.153e-04","GO:0031680 (12) <br> G-protein beta/gamma-subunit complex <br> 2.582e-04","GO:0098852 (395) <br> lytic vacuole membrane <br> 3.695e-04","GO:0005765 (395) <br> lysosomal membrane <br> 3.695e-04","GO:0032991 (5541) <br> protein-containing complex <br> 6.454e-04","GO:0016020 (9829) <br> membrane <br> 7.280e-04","GO:0099503 (1027) <br> secretory vesicle <br> 9.530e-04","GO:0005774 (450) <br> vacuolar membrane <br> 1.182e-03","GO:0005764 (723) <br> lysosome <br> 2.803e-03","GO:0000323 (723) <br> lytic vacuole <br> 2.803e-03","GO:0012505 (4651) <br> endomembrane system <br> 3.709e-03","GO:0005622 (14808) <br> intracellular anatomical structure <br> 4.377e-03","GO:0035579 (90) <br> specific granule membrane <br> 4.816e-03","GO:0042582 (154) <br> azurophil granule <br> 5.557e-03","GO:0005766 (154) <br> primary lysosome <br> 5.557e-03","GO:0043227 (13201) <br> membrane-bounded organelle <br> 5.916e-03","GO:0012506 (1183) <br> vesicle membrane <br> 6.663e-03","GO:0005773 (818) <br> vacuole <br> 9.515e-03","GO:0035577 (57) <br> azurophil granule membrane <br> 1.406e-02","GO:1902494 (1377) <br> catalytic complex <br> 1.573e-02","GO:0005829 (5361) <br> cytosol <br> 1.756e-02","GO:0030485 (2) <br> smooth muscle contractile fiber <br> 2.866e-02","GO:0070821 (72) <br> tertiary granule membrane <br> 3.518e-02","GO:0030057 (25) <br> desmosome <br> 3.919e-02","GO:0043226 (14027) <br> organelle <br> 4.634e-02"],"key":["GO:1905360","GO:0005834","GO:0031234","GO:0070062","GO:1903561","GO:0043230","GO:0065010","GO:0009898","GO:0019897","GO:0098562","GO:0031982","GO:0019898","GO:0098857","GO:0045121","GO:0070820","GO:0098797","GO:0005615","GO:0098552","GO:0030667","GO:0042581","GO:0005925","GO:0005737","GO:0030055","GO:0030054","GO:0005886","GO:0031410","GO:0097708","GO:0070161","GO:0071944","GO:0005576","GO:0098588","GO:0030141","GO:0030659","GO:0098796","GO:0031090","GO:0031680","GO:0098852","GO:0005765","GO:0032991","GO:0016020","GO:0099503","GO:0005774","GO:0005764","GO:0000323","GO:0012505","GO:0005622","GO:0035579","GO:0042582","GO:0005766","GO:0043227","GO:0012506","GO:0005773","GO:0035577","GO:1902494","GO:0005829","GO:0030485","GO:0070821","GO:0030057","GO:0043226"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[14.5429649371964,14.5429649371964,16.3487788308752,20.5214902439301,20.5672559227953,20.5683104706776,20.5683104706776,17.114947301444,17.1926950579195,17.3282977259613,21.250520850314,18.0200111191535,18.1229749928138,18.1229749928138,17.0605109063983,19.0951202425796,21.1109970585917,18.9473154826326,17.9883493723974,17.0134143816997,18.41927534827,22.4484604312869,18.4418855248262,20.4817706635075,21.6320787809657,20.6535046726708,20.6559427724165,19.3361763823889,21.7261763418418,21.384799437161,20.5143223808468,19.3650320892764,19.2583620424432,19.9165368378714,21.1266444165659,12.254025725742,18.3350224500158,18.3350224500158,21.604243722507,22.2374238416628,19.5960108346389,18.5134197986091,19.1448417015984,19.1448417015984,21.4062815734808,22.6771653543307,16.1317846515619,16.9745252158235,16.9745252158235,22.5549451746932,19.774188875036,19.3050263840062,15.3670429037096,19.9633396074552,21.5670744132122,3.77952755905512,15.7642778006896,13.8411375859653,22.6196160803639],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData32e05414","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[4.24258500390343,16.9543231164817,15.3202787669038,16.2143076294315,16.2105512056393,21.169030611255,19.3434086482783,18.7010601798235,9.05456388162587,28.7757887903244,3.14946568039271,38.0353734379323,4.24634142769556,9.0921281195472,19.362190767239,18.6935473322393,22.3297655630241,21.1615177636707,27.0929109314488,27.0515902697353,21.1652741874628,20.5266821428002],"y":[6.91659864370096,6.28402251954227,5.61284572316924,5.49571484172274,5.485211655212,5.09620999425307,4.39157291179348,4.13945010400836,3.29110751336065,3.05254615629031,2.87364929661844,2.83628022035909,2.83364298724054,2.67594276816802,2.57709352785328,2.48756962381478,2.48756962381478,2.4763170227671,1.93153143213745,1.80278187454154,1.77257873929302,1.63193550889991],"text":["GO:0003924 (300) <br> GTPase activity <br> 1.212e-07","GO:0017111 (424) <br> nucleoside-triphosphatase activity <br> 5.200e-07","GO:0016462 (482) <br> pyrophosphatase activity <br> 2.439e-06","GO:0016818 (493) <br> hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides <br> 3.194e-06","GO:0016817 (494) <br> hydrolase activity, acting on acid anhydrides <br> 3.272e-06","GO:0031683 (20) <br> G-protein beta/gamma-subunit complex binding <br> 8.013e-06","GO:0019899 (2048) <br> enzyme binding <br> 4.059e-05","GO:0019003 (73) <br> GDP binding <br> 7.254e-05","GO:0005515 (14767) <br> protein binding <br> 5.116e-04","GO:0044877 (1304) <br> protein-containing complex binding <br> 8.860e-04","GO:0001664 (292) <br> G protein-coupled receptor binding <br> 1.338e-03","GO:0060090 (341) <br> molecular adaptor activity <br> 1.458e-03","GO:0003925 (32) <br> G protein activity <br> 1.467e-03","GO:0005525 (381) <br> GTP binding <br> 2.109e-03","GO:0019904 (706) <br> protein domain specific binding <br> 2.648e-03","GO:0019001 (402) <br> guanyl nucleotide binding <br> 3.254e-03","GO:0032561 (402) <br> guanyl ribonucleotide binding <br> 3.254e-03","GO:0031681 (22) <br> G-protein beta-subunit binding <br> 3.340e-03","GO:0042731 (4) <br> PH domain binding <br> 1.171e-02","GO:0042610 (4) <br> CD8 receptor binding <br> 1.575e-02","GO:0031682 (6) <br> G-protein gamma-subunit binding <br> 1.688e-02","GO:0030674 (272) <br> protein-macromolecule adaptor activity <br> 2.334e-02"],"key":["GO:0003924","GO:0017111","GO:0016462","GO:0016818","GO:0016817","GO:0031683","GO:0019899","GO:0019003","GO:0005515","GO:0044877","GO:0001664","GO:0060090","GO:0003925","GO:0005525","GO:0019904","GO:0019001","GO:0032561","GO:0031681","GO:0042731","GO:0042610","GO:0031682","GO:0030674"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[17.9511812971334,18.4322454425848,18.6065659634108,18.6370346939429,18.6397677301037,13.3863990273633,20.4476570765472,15.7873208982415,22.6742247680722,19.8957455135249,17.912906815234,18.1311916030541,14.3213782810468,18.2852512758846,19.1137758499688,18.3591888413143,18.3591888413143,13.5832101454551,9.05045292005097,9.05045292005097,10.4153745295623,17.811937508837],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData32e05414","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[198.094889825427,190.583425812253,193.935214656551,193.957760321422,196.783483652041,191.962468980254,190.835185736656,194.01036687279,199.147020852785,193.597029683471,190.985490169136,199.143263241973,194.014124483602,194.14188325121,194.134368029586,193.938972267363,194.138125640398,197.542521036064,199.624237425908,193.942729878175,193.950245099799,196.715846657425,190.575910590629,194.382370343178,196.61063355469,196.170993089686,190.579668201441,191.128279379992,193.946487488987,191.755800385595,195.06625551096,191.075672828624,196.790998873665,194.619099824333,190.831428125844,193.95400271061,194.713040094633,190.906580342084,193.961517932234,194.930981521729,193.920184213303,196.621906387126,195.062497900148,194.600311770273,196.013173435583,191.827194991022,193.416664364495,191.582950288243,199.398780777188,192.052651639742,198.361680193078,199.135748020349,191.301129477343,195.998142992335,198.489438960686,198.267739922779,192.04889402893,194.431219283734,196.678270549305,193.923941824115,194.720555316257,191.222219650292,195.907960332847,196.024446268019,194.322248570186,198.444347630942,190.647305196057,193.927699434927,196.249902916738,193.138601164408,193.931457045739,193.702242786207,197.399731825208],"y":[10.9849124116597,10.1394998094369,9.44412144484697,9.30335007641712,8.32529081604476,7.95826157263128,7.88199311173166,7.88199311173166,7.63430637367034,7.57163936727731,7.19095014399308,6.90268866218858,6.81861836300453,6.72718894869579,6.72718894869579,6.23136687455307,6.05997241028499,5.99969811292566,5.99561958845668,5.97564576954763,5.97564576954763,5.85846808289009,5.80340395012584,5.75824704906446,5.75396475827093,5.5590066333188,5.54134633553274,5.48017742317669,5.44535801949138,5.39354031499992,5.34444620737705,5.34444620737705,5.17681076042707,5.09973289688383,5.09973289688383,5.09973289688383,5.0314668908518,5.02147343642889,4.8731425004057,4.66852445059616,4.33669084380505,4.17827930397686,4.16209542881583,4.15339963372903,4.15220350462863,3.94434103136716,3.81329292479787,3.50252155356123,3.23110415245668,2.92188290261651,2.85302156058651,2.85064172349738,2.84372542265781,2.79060421710213,2.54306408467669,2.54306408467669,2.42251185825665,2.34310933571841,2.30042942424609,2.16774903943571,1.95726621608286,1.95726621608286,1.87889473857571,1.82370307160465,1.77115143113658,1.7670614689785,1.65708204024088,1.63448068699397,1.55987653904282,1.5003434930982,1.42381322244476,1.34868286317358,1.32584194563715],"text":["REAC:R-HSA-392518 (32) <br> Signal amplification <br> 1.035e-11","REAC:R-HSA-392170 (22) <br> ADP signalling through P2Y purinoceptor 12 <br> 7.253e-11","REAC:R-HSA-418597 (48) <br> G alpha (z) signalling events <br> 3.596e-10","REAC:R-HSA-202040 (28) <br> G-protein activation <br> 4.973e-10","REAC:R-HSA-392851 (19) <br> Prostacyclin signalling through prostacyclin receptor <br> 4.728e-09","REAC:R-HSA-6814122 (42) <br> Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding <br> 1.101e-08","REAC:R-HSA-991365 (43) <br> Activation of GABAB receptors <br> 1.312e-08","REAC:R-HSA-977444 (43) <br> GABA B receptor activation <br> 1.312e-08","REAC:R-HSA-428930 (24) <br> Thromboxane signalling through TP receptor <br> 2.321e-08","REAC:R-HSA-9009391 (73) <br> Extra-nuclear estrogen signaling <br> 2.681e-08","REAC:R-HSA-400042 (28) <br> Adrenaline,noradrenaline inhibits insulin secretion <br> 6.442e-08","REAC:R-HSA-456926 (31) <br> Thrombin signalling through proteinase activated receptors (PARs) <br> 1.251e-07","REAC:R-HSA-977443 (60) <br> GABA receptor activation <br> 1.518e-07","REAC:R-HSA-420092 (33) <br> Glucagon-type ligand receptors <br> 1.874e-07","REAC:R-HSA-163359 (33) <br> Glucagon signaling in metabolic regulation <br> 1.874e-07","REAC:R-HSA-8964315 (18) <br> G beta:gamma signalling through BTK <br> 5.870e-07","REAC:R-HSA-381676 (42) <br> Glucagon-like Peptide-1 (GLP1) regulates insulin secretion <br> 8.710e-07","REAC:R-HSA-422356 (78) <br> Regulation of insulin secretion <br> 1.001e-06","REAC:R-HSA-432040 (43) <br> Vasopressin regulates renal water homeostasis via Aquaporins <br> 1.010e-06","REAC:R-HSA-8964616 (20) <br> G beta:gamma signalling through CDC42 <br> 1.058e-06","REAC:R-HSA-418217 (20) <br> G beta:gamma signalling through PLC beta <br> 1.058e-06","REAC:R-HSA-500657 (21) <br> Presynaptic function of Kainate receptors <br> 1.385e-06","REAC:R-HSA-9660821 (133) <br> ADORA2B mediated anti-inflammatory cytokines production <br> 1.573e-06","REAC:R-HSA-109582 (673) <br> Hemostasis <br> 1.745e-06","REAC:R-HSA-76002 (258) <br> Platelet activation, signaling and aggregation <br> 1.762e-06","REAC:R-HSA-111885 (90) <br> Opioid Signalling <br> 2.761e-06","REAC:R-HSA-418592 (24) <br> ADP signalling through P2Y purinoceptor 1 <br> 2.875e-06","REAC:R-HSA-445717 (52) <br> Aquaporin-mediated transport <br> 3.310e-06","REAC:R-HSA-392451 (25) <br> G beta:gamma signalling through PI3Kgamma <br> 3.586e-06","REAC:R-HSA-390466 (95) <br> Chaperonin-mediated protein folding <br> 4.041e-06","REAC:R-HSA-9664433 (221) <br> Leishmania parasite growth and survival <br> 4.524e-06","REAC:R-HSA-9662851 (221) <br> Anti-inflammatory response favouring Leishmania parasite infection <br> 4.524e-06","REAC:R-HSA-391251 (102) <br> Protein folding <br> 6.656e-06","REAC:R-HSA-997272 (29) <br> Inhibition  of voltage gated Ca2+ channels via Gbeta/gamma subunits <br> 7.948e-06","REAC:R-HSA-1296041 (29) <br> Activation of G protein gated Potassium channels <br> 7.948e-06","REAC:R-HSA-1296059 (29) <br> G protein gated Potassium channels <br> 7.948e-06","REAC:R-HSA-163685 (107) <br> Integration of energy metabolism <br> 9.301e-06","REAC:R-HSA-451326 (30) <br> Activation of kainate receptors upon glutamate binding <br> 9.518e-06","REAC:R-HSA-397795 (32) <br> G-protein beta:gamma signalling <br> 1.339e-05","REAC:R-HSA-1296065 (35) <br> Inwardly rectifying K+ channels <br> 2.145e-05","REAC:R-HSA-416482 (80) <br> G alpha (12/13) signalling events <br> 4.606e-05","REAC:R-HSA-418346 (85) <br> Platelet homeostasis <br> 6.633e-05","REAC:R-HSA-9658195 (303) <br> Leishmania infection <br> 6.885e-05","REAC:R-HSA-5663205 (909) <br> Infectious disease <br> 7.024e-05","REAC:R-HSA-112314 (196) <br> Neurotransmitter receptors and postsynaptic signal transmission <br> 7.044e-05","REAC:R-HSA-373080 (93) <br> Class B/2 (Secretin family receptors) <br> 1.137e-04","REAC:R-HSA-8939211 (217) <br> ESR-mediated signaling <br> 1.537e-04","REAC:R-HSA-4086398 (59) <br> Ca2+ pathway <br> 3.144e-04","REAC:R-HSA-112315 (259) <br> Transmission across Chemical Synapses <br> 5.873e-04","REAC:R-HSA-2424491 (25) <br> DAP12 signaling <br> 1.197e-03","REAC:R-HSA-9006931 (291) <br> Signaling by Nuclear Receptors <br> 1.403e-03","REAC:R-HSA-164952 (26) <br> The role of Nef in HIV-1 replication and disease pathogenesis <br> 1.410e-03","REAC:R-HSA-3858494 (143) <br> Beta-catenin independent WNT signaling <br> 1.433e-03","REAC:R-HSA-112316 (400) <br> Neuronal System <br> 1.620e-03","REAC:R-HSA-9670439 (17) <br> Signaling by phosphorylated juxtamembrane, extracellular and kinase domain KIT mutants <br> 2.864e-03","REAC:R-HSA-9669938 (17) <br> Signaling by KIT in disease <br> 2.864e-03","REAC:R-HSA-2172127 (33) <br> DAP12 interactions <br> 3.780e-03","REAC:R-HSA-162909 (125) <br> Host Interactions of HIV factors <br> 4.538e-03","REAC:R-HSA-1296071 (103) <br> Potassium Channels <br> 5.007e-03","REAC:R-HSA-418594 (398) <br> G alpha (i) signalling events <br> 6.796e-03","REAC:R-HSA-175567 (3) <br> Integration of viral DNA into host genomic DNA <br> 1.103e-02","REAC:R-HSA-177539 (3) <br> Autointegration results in viral DNA circles <br> 1.103e-02","REAC:R-HSA-164938 (20) <br> Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters <br> 1.322e-02","REAC:R-HSA-6798695 (476) <br> Neutrophil degranulation <br> 1.501e-02","REAC:R-HSA-162906 (226) <br> HIV Infection <br> 1.694e-02","REAC:R-HSA-195721 (328) <br> Signaling by WNT <br> 1.710e-02","REAC:R-HSA-180689 (4) <br> APOBEC3G mediated resistance to HIV-1 infection <br> 2.203e-02","REAC:R-HSA-416476 (212) <br> G alpha (q) signalling events <br> 2.320e-02","REAC:R-HSA-210990 (11) <br> PECAM1 interactions <br> 2.755e-02","REAC:R-HSA-1643685 (1616) <br> Disease <br> 3.160e-02","REAC:R-HSA-418555 (506) <br> G alpha (s) signalling events <br> 3.769e-02","REAC:R-HSA-9607240 (36) <br> FLT3 Signaling <br> 4.480e-02","REAC:R-HSA-1433559 (13) <br> Regulation of KIT signaling <br> 4.722e-02"],"key":["REAC:R-HSA-392518","REAC:R-HSA-392170","REAC:R-HSA-418597","REAC:R-HSA-202040","REAC:R-HSA-392851","REAC:R-HSA-6814122","REAC:R-HSA-991365","REAC:R-HSA-977444","REAC:R-HSA-428930","REAC:R-HSA-9009391","REAC:R-HSA-400042","REAC:R-HSA-456926","REAC:R-HSA-977443","REAC:R-HSA-420092","REAC:R-HSA-163359","REAC:R-HSA-8964315","REAC:R-HSA-381676","REAC:R-HSA-422356","REAC:R-HSA-432040","REAC:R-HSA-8964616","REAC:R-HSA-418217","REAC:R-HSA-500657","REAC:R-HSA-9660821","REAC:R-HSA-109582","REAC:R-HSA-76002","REAC:R-HSA-111885","REAC:R-HSA-418592","REAC:R-HSA-445717","REAC:R-HSA-392451","REAC:R-HSA-390466","REAC:R-HSA-9664433","REAC:R-HSA-9662851","REAC:R-HSA-391251","REAC:R-HSA-997272","REAC:R-HSA-1296041","REAC:R-HSA-1296059","REAC:R-HSA-163685","REAC:R-HSA-451326","REAC:R-HSA-397795","REAC:R-HSA-1296065","REAC:R-HSA-416482","REAC:R-HSA-418346","REAC:R-HSA-9658195","REAC:R-HSA-5663205","REAC:R-HSA-112314","REAC:R-HSA-373080","REAC:R-HSA-8939211","REAC:R-HSA-4086398","REAC:R-HSA-112315","REAC:R-HSA-2424491","REAC:R-HSA-9006931","REAC:R-HSA-164952","REAC:R-HSA-3858494","REAC:R-HSA-112316","REAC:R-HSA-9670439","REAC:R-HSA-9669938","REAC:R-HSA-2172127","REAC:R-HSA-162909","REAC:R-HSA-1296071","REAC:R-HSA-418594","REAC:R-HSA-175567","REAC:R-HSA-177539","REAC:R-HSA-164938","REAC:R-HSA-6798695","REAC:R-HSA-162906","REAC:R-HSA-195721","REAC:R-HSA-180689","REAC:R-HSA-416476","REAC:R-HSA-210990","REAC:R-HSA-1643685","REAC:R-HSA-418555","REAC:R-HSA-9607240","REAC:R-HSA-1433559"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,102,204,1)","opacity":0.8,"size":[14.3213782810468,13.5832101454551,15.0659099511043,14.064391440917,13.2787931541743,14.8262572073822,14.8688640092558,14.8688640092558,13.7594961631324,15.7873208982415,14.064391440917,14.2608475416207,15.45541914919,14.3797163833018,14.3797163833018,13.1640323425788,14.8262572073822,15.8973863612294,14.8688640092558,13.3863990273633,13.3863990273633,13.4876469321386,16.7499484371089,19.0510845822719,17.7362643582832,16.1317846515619,13.7594961631324,15.2071559698249,13.8411375859653,16.2191968510427,17.5121875085122,17.5121875085122,16.3332135329498,14.1325441121217,14.1325441121217,14.1325441121217,16.409380751435,14.1979626749156,14.3213782810468,14.4903838270411,15.9391857668738,16.0386970905277,17.9652456519876,19.4405959398567,17.3358617234104,16.1848705044524,17.4855057979174,15.426535038788,17.7418181294509,13.8411375859653,17.9080415234916,13.9189573720891,16.8614836447151,18.3523314408885,13.0411641506074,13.0411641506074,14.3797163833018,16.6537390526195,16.3487788308752,18.3454364147149,7.81088103584182,7.81088103584182,13.3863990273633,18.5896251814072,17.5447981132505,18.0768113156737,9.05045292005097,17.4513777342553,12.0456958427095,20.160331856797,18.6721012001934,14.5429649371964,12.4412469479671],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,102,204,1)"}},"hoveron":"points","set":"SharedData32e05414","name":"REAC","legendgroup":"REAC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"225c33013680":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"225c1ad74c4":{"x":{},"xend":{},"y":{},"yend":{}},"225c263352f9":{"x":{},"xend":{},"y":{},"yend":{}},"225c18683996":{"x":{},"xend":{},"y":{},"yend":{}},"225cc6d3076":{"x":{},"xend":{},"y":{},"yend":{}},"225c3ab6712":{"x":{},"y":{}},"225c3299784":{"yintercept":{}}},"cur_data":"225c33013680","visdat":{"225c33013680":["function (y) ","x"],"225c1ad74c4":["function (y) ","x"],"225c263352f9":["function (y) ","x"],"225c18683996":["function (y) ","x"],"225cc6d3076":["function (y) ","x"],"225c3ab6712":["function (y) ","x"],"225c3299784":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData32e05414"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results-1" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-2b7856457827ab1876f7" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2b7856457827ab1876f7">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222"],["GO:CC","GO:CC","GO:CC","GO:CC","REAC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","REAC","GO:CC","REAC","REAC","REAC","GO:CC","REAC","REAC","REAC","GO:BP","GO:CC","REAC","GO:CC","GO:CC","REAC","REAC","GO:CC","GO:MF","REAC","REAC","GO:CC","REAC","REAC","GO:BP","GO:CC","GO:MF","REAC","GO:CC","REAC","REAC","REAC","REAC","REAC","GO:BP","REAC","REAC","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:MF","REAC","REAC","GO:BP","GO:MF","GO:MF","REAC","REAC","GO:BP","REAC","GO:CC","REAC","REAC","GO:CC","REAC","GO:CC","REAC","REAC","REAC","GO:MF","REAC","GO:CC","GO:CC","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","REAC","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","REAC","REAC","REAC","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:CC","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:BP","GO:MF","REAC","GO:BP","REAC","REAC","GO:MF","GO:MF","REAC","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","REAC","REAC","GO:MF","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:CC","REAC","GO:BP","GO:CC","GO:BP","REAC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","REAC","GO:CC","GO:BP","GO:BP","REAC","REAC","GO:BP","GO:MF","GO:BP","REAC","GO:CC","GO:BP","REAC","GO:CC","GO:MF","GO:BP","GO:MF","REAC","REAC","GO:CC","GO:BP","REAC","REAC","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:CC","REAC","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:BP","REAC"],[4087,474,1247,2743,2009,4056,2131,2719,785,983,10,3561,902,908,1660,1453,377,77,922,561,984,2289,3658,2437,812,117,2895,598,2288,923,3628,957,955,916,278,3982,903,3551,956,1862,2416,904,906,2901,1642,8,1098,1021,1614,2843,12268,3547,1497,9,939,3785,3784,155,905,1099,322,1125,1203,141,2041,1662,550,1084,76,907,5104,1109,385,1027,96,11547,1026,13142,7747,515,909,1322,3497,1030,1167,927,2765,16054,13259,12281,4618,947,898,938,3132,1101,11579,1617,1202,1079,1455,4447,16046,10980,241,3996,3582,1080,5762,1118,23108,5659,341,1038,3627,764,3470,3473,15388,15456,1217,9160,1414,276,3654,410,3168,15359,1879,10495,2356,1570,3169,850,7129,3780,2779,3102,418,401,15694,14485,307,2080,3123,2286,201,9594,599,1451,15318,2897,1889,14885,4623,15738,409,67,2114,2055,4445,5413,5102,15440,7671,817,400,2450,284,1034,15361,1864,3099,1632,2042,411,2128,19465,818,899,417,3100,9109,1111,180,24936,6681,3067,1427,1862,19464,1458,4001,6670,15417,5103,1005,2102,470,10046,27,900,4933,3078,3999,14941,1518,26391,1110,26390,8775,690,2896,901,1029,1332,19502,840,24902,2127,10045,1824],["GTPase complex","heterotrimeric G-protein complex","extrinsic component of cytoplasmic side of plasma membrane","extracellular exosome","Signal amplification","extracellular vesicle","extracellular organelle","extracellular membrane-bounded organelle","cytoplasmic side of plasma membrane","extrinsic component of plasma membrane","ADP signalling through P2Y purinoceptor 12","cytoplasmic side of membrane","G alpha (z) signalling events","G-protein activation","Prostacyclin signalling through prostacyclin receptor","vesicle","Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding","Activation of GABAB receptors","GABA B receptor activation","cell activation","extrinsic component of membrane","Thromboxane signalling through TP receptor","membrane microdomain","membrane raft","Extra-nuclear estrogen signaling","Adrenaline,noradrenaline inhibits insulin secretion","tertiary granule","GTPase activity","Thrombin signalling through proteinase activated receptors (PARs)","GABA receptor activation","plasma membrane protein complex","Glucagon-type ligand receptors","Glucagon signaling in metabolic regulation","immune effector process","extracellular space","nucleoside-triphosphatase activity","G beta:gamma signalling through BTK","side of membrane","Glucagon-like Peptide-1 (GLP1) regulates insulin secretion","Regulation of insulin secretion","Vasopressin regulates renal water homeostasis via Aquaporins","G beta:gamma signalling through CDC42","G beta:gamma signalling through PLC beta","immune response","Presynaptic function of Kainate receptors","ADORA2B mediated anti-inflammatory cytokines production","leukocyte mediated immunity","Hemostasis","Platelet activation, signaling and aggregation","exocytosis","leukocyte degranulation","pyrophosphatase activity","Opioid Signalling","ADP signalling through P2Y purinoceptor 1","myeloid cell activation involved in immune response","hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides","hydrolase activity, acting on acid anhydrides","Aquaporin-mediated transport","G beta:gamma signalling through PI3Kgamma","myeloid leukocyte mediated immunity","Chaperonin-mediated protein folding","secretory granule membrane","Leishmania parasite growth and survival","Anti-inflammatory response favouring Leishmania parasite infection","specific granule","Protein folding","focal adhesion","Inhibition  of voltage gated Ca2+ channels via Gbeta/gamma subunits","Activation of G protein gated Potassium channels","G protein gated Potassium channels","G-protein beta/gamma-subunit complex binding","Integration of energy metabolism","cytoplasm","cell-substrate junction","Activation of kainate receptors upon glutamate binding","wound healing","cell junction","regulated exocytosis","platelet activation","plasma membrane","G-protein beta:gamma signalling","cytoplasmic vesicle","intracellular vesicle","leukocyte activation involved in immune response","Inwardly rectifying K+ channels","cell activation involved in immune response","anchoring junction","establishment of localization in cell","leukocyte activation","neutrophil degranulation","enzyme binding","neutrophil activation involved in immune response","G alpha (12/13) signalling events","myeloid leukocyte activation","cell periphery","neutrophil mediated immunity","neutrophil activation","Platelet homeostasis","Leishmania infection","Infectious disease","Neurotransmitter receptors and postsynaptic signal transmission","GDP binding","cellular localization","granulocyte activation","extracellular region","response to wounding","bounding membrane of organelle","secretory granule","vesicle-mediated transport","cytoplasmic vesicle membrane","export from cell","viral process","Class B/2 (Secretin family receptors)","immune system process","membrane protein complex","ESR-mediated signaling","blood coagulation","hemostasis","coagulation","response to stimulus","organelle membrane","secretion by cell","G-protein beta/gamma-subunit complex","Ca2+ pathway","lytic vacuole membrane","lysosomal membrane","small GTPase mediated signal transduction","regulation of immune response","protein binding","intracellular signal transduction","Transmission across Chemical Synapses","protein-containing complex","Ras protein signal transduction","membrane","protein-containing complex binding","secretory vesicle","transport","adenylate cyclase-activating dopamine receptor signaling pathway","vacuolar membrane","DAP12 signaling","localization","secretion","G protein-coupled receptor binding","Signaling by Nuclear Receptors","dopamine receptor signaling pathway","The role of Nef in HIV-1 replication and disease pathogenesis","Beta-catenin independent WNT signaling","molecular adaptor activity","G protein activity","Neuronal System","regulation of defense response to virus by virus","response to stress","GTP binding","positive regulation of biological process","protein domain specific binding","establishment of localization","lysosome","lytic vacuole","Signaling by phosphorylated juxtamembrane, extracellular and kinase domain KIT mutants","Signaling by KIT in disease","guanyl nucleotide binding","guanyl ribonucleotide binding","G-protein beta-subunit binding","regulation of body fluid levels","signaling","endomembrane system","DAP12 interactions","protein folding","intracellular anatomical structure","Host Interactions of HIV factors","positive regulation of immune response","specific granule membrane","adenylate cyclase-modulating G protein-coupled receptor signaling pathway","Potassium Channels","azurophil granule","primary lysosome","membrane-bounded organelle","cellular response to prostaglandin E stimulus","vesicle membrane","G alpha (i) signalling events","vacuole","adenylate cyclase-activating G protein-coupled receptor signaling pathway","regulation of localization","Integration of viral DNA into host genomic DNA","Autointegration results in viral DNA circles","cellular response to nitrogen compound","PH domain binding","cell communication","Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters","azurophil granule membrane","cellular response to prostaglandin stimulus","Neutrophil degranulation","catalytic complex","CD8 receptor binding","T cell receptor signaling pathway","G-protein gamma-subunit binding","HIV Infection","Signaling by WNT","cytosol","response to prostaglandin E","APOBEC3G mediated resistance to HIV-1 infection","G alpha (q) signalling events","protein-macromolecule adaptor activity","signal transduction","response to virus","positive regulation of response to stimulus","PECAM1 interactions","cellular response to dopamine","smooth muscle contractile fiber","response to dopamine","lysosome localization","Disease","tertiary granule membrane","G alpha (s) signalling events","desmosome","regulation of immune system process","cellular response to organonitrogen compound","FLT3 Signaling","cellular response to ketone","organelle","response to prostaglandin","Regulation of KIT signaling"],["8.0e-16","8.0e-16","1.3e-13","5.3e-12","1.0e-11","1.8e-11","1.8e-11","1.8e-11","3.3e-11","5.8e-11","7.3e-11","1.5e-10","3.6e-10","5.0e-10","4.7e-09","7.5e-09","1.1e-08","1.3e-08","1.3e-08","1.4e-08","1.8e-08","2.3e-08","2.6e-08","2.6e-08","2.7e-08","6.4e-08","1.0e-07","1.2e-07","1.3e-07","1.5e-07","1.7e-07","1.9e-07","1.9e-07","2.4e-07","2.5e-07","5.2e-07","5.9e-07","8.1e-07","8.7e-07","1.0e-06","1.0e-06","1.1e-06","1.1e-06","1.2e-06","1.4e-06","1.6e-06","1.6e-06","1.7e-06","1.8e-06","2.1e-06","2.2e-06","2.4e-06","2.8e-06","2.9e-06","3.2e-06","3.2e-06","3.3e-06","3.3e-06","3.6e-06","3.7e-06","4.0e-06","4.4e-06","4.5e-06","4.5e-06","4.7e-06","6.7e-06","7.8e-06","7.9e-06","7.9e-06","7.9e-06","8.0e-06","9.3e-06","9.3e-06","9.4e-06","9.5e-06","1.0e-05","1.0e-05","1.1e-05","1.2e-05","1.3e-05","1.3e-05","1.6e-05","1.6e-05","2.1e-05","2.1e-05","2.3e-05","2.3e-05","3.4e-05","3.7e-05","3.9e-05","4.1e-05","4.4e-05","4.6e-05","5.0e-05","5.2e-05","6.0e-05","6.3e-05","6.6e-05","6.9e-05","7.0e-05","7.0e-05","7.3e-05","7.3e-05","7.5e-05","7.6e-05","8.4e-05","8.9e-05","8.9e-05","9.7e-05","1.0e-04","1.0e-04","1.1e-04","1.1e-04","1.2e-04","1.4e-04","1.5e-04","1.8e-04","1.9e-04","2.0e-04","2.1e-04","2.2e-04","2.6e-04","2.6e-04","3.1e-04","3.7e-04","3.7e-04","3.8e-04","5.1e-04","5.1e-04","5.4e-04","5.9e-04","6.5e-04","7.0e-04","7.3e-04","8.9e-04","9.5e-04","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.9e-03","2.1e-03","2.5e-03","2.6e-03","2.7e-03","2.8e-03","2.8e-03","2.9e-03","2.9e-03","3.3e-03","3.3e-03","3.3e-03","3.4e-03","3.6e-03","3.7e-03","3.8e-03","4.3e-03","4.4e-03","4.5e-03","4.6e-03","4.8e-03","5.0e-03","5.0e-03","5.6e-03","5.6e-03","5.9e-03","6.5e-03","6.7e-03","6.8e-03","9.5e-03","9.8e-03","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.3e-02","1.3e-02","1.4e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.8e-02","2.2e-02","2.2e-02","2.3e-02","2.3e-02","2.4e-02","2.4e-02","2.5e-02","2.8e-02","2.8e-02","2.9e-02","2.9e-02","3.0e-02","3.2e-02","3.5e-02","3.8e-02","3.9e-02","4.0e-02","4.4e-02","4.5e-02","4.5e-02","4.6e-02","4.7e-02","4.7e-02"],[36,36,103,2178,32,2263,2265,2265,169,178,22,195,48,28,19,4058,42,43,43,1507,315,24,339,339,73,28,163,300,31,60,696,33,33,1135,3595,424,18,622,42,78,43,20,20,2544,21,133,888,673,258,906,538,482,90,24,552,493,494,52,25,558,95,308,221,221,158,102,420,29,29,29,20,107,11951,427,30,555,2107,790,163,5680,32,2433,2438,723,35,727,838,2778,1341,483,2048,488,80,671,6178,500,502,85,303,909,196,73,3519,509,4564,680,2165,857,2193,789,1413,1011,93,3438,1326,217,351,355,358,9487,3644,1351,12,59,395,395,520,1159,14767,2886,259,5541,352,9829,1304,1027,5214,10,450,25,6938,1492,292,291,41,26,143,341,32,400,29,4321,381,6474,706,5364,723,723,17,17,402,402,22,519,6937,4651,33,230,14808,125,757,90,236,103,154,154,13201,17,1183,398,818,148,2906,3,3,728,4,6961,20,57,22,476,1377,4,202,6,226,328,5361,25,4,212,272,6447,366,2368,11,86,2,87,79,1616,72,506,25,1734,671,36,97,14027,32,13],[9,9,11,37,8,37,37,37,11,11,7,10,8,7,6,41,7,7,7,29,10,6,14,14,8,6,10,12,6,7,12,6,6,24,40,13,5,11,6,7,6,5,5,31,5,8,20,16,14,19,16,13,7,5,16,13,13,6,5,16,7,11,9,9,9,7,13,5,5,5,4,7,69,13,5,11,25,18,8,43,5,29,29,17,5,17,17,33,23,14,27,14,6,16,44,14,14,6,9,14,8,6,37,14,40,11,26,16,29,15,21,19,6,33,14,8,8,8,8,58,33,20,3,5,10,10,12,18,80,27,8,38,10,63,19,16,44,3,10,4,48,20,8,8,4,4,6,9,4,9,4,27,9,45,11,44,12,12,4,4,9,9,3,8,45,38,4,6,79,6,13,5,6,5,6,6,74,3,16,8,12,5,27,2,2,10,2,44,2,4,3,13,10,2,7,2,9,7,40,3,2,6,7,40,8,22,3,4,1,4,5,15,4,8,3,19,9,4,4,75,3,3]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap-1" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp-1" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/BP-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf-1" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/MF-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc-1" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/CC-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot-1" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/scatter-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="network-analysis" class="section level2">
<h2><strong>6. Network analysis</strong></h2>
<p>Overview from <strong>STRING Database</strong>: In STRING, each protein-protein interaction is annotated with one or more ‘scores’. Importantly, these scores do <strong>not</strong> indicate the strength or the specificity of the interaction. Instead, they are indicators of <strong>confidence</strong>, i.e. how likely STRING judges an interaction to be true, given the available evidence. All scores rank from 0 to 1, with 1 being the highest possible confidence. A score of 0.5 would indicate that roughly every second interaction might be erroneous (i.e., a false positive).</p>
<pre class="r"><code>if(leng_dep != 0){
  
  string_db &lt;- STRINGdb$new(version=&quot;11&quot;, species=9606, score_threshold=400)

  string_mapped &lt;- string_db$map(string_res, &quot;Gene.name&quot;, removeUnmappedRows = TRUE, quiet = T)

  hits &lt;- string_mapped$STRING_id[1:leng_dep]

  string_db$plot_network(hits, add_link = T)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/CSCAP2_act_vs_Halo_act.Rmd/stringdb-1.png" width="1440" style="display: block; margin: auto;" /></p>
<div id="cytoscape-network" class="section level3">
<h3><strong>6.1. Cytoscape Network</strong></h3>
<p>Cytoscape is an open source software platform for visualizing complex networks and integrating these with any type of attribute data. In this presentation all proteins showing at least one confidence level from the STRING database are shown. The <strong>color</strong> code is relative to <strong>fold change</strong> and the <strong>size</strong> is directly proportional to the <strong>significance (p-value)</strong>.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>if(n_edge != 0){
  knitr::include_graphics(paste0(&quot;assets/network_FC_pvalue&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_FC_pvalueSKAP2_C_act_vs_Halo_act.png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
<div id="cytoscape-enrichmentmap" class="section level3">
<h3><strong>6.2. Cytoscape EnrichmentMap</strong></h3>
<p>An <strong>enrichment map</strong> is a different sort of network. Instead of nodes representing genes, nodes represent pathways or functions. Edges between these pathways or functions represent shared genes or pathway crosstalk. An enrichment map is a way to visualize your enrichment results to help reduce redundancy and uncover main themes.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant) %&gt;% pull(name)

if(!params$Imputation){
  
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  pull(name) %&gt;% 
  unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  
  print(&quot;NOT enough data.&quot;)
  
}else{
  
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  
  if(is.null(GO)){
    
    print(&quot;NOT enough data.&quot;)

    enrich_map = F
    
  }else{
    
    enrich_map = T
    
    gem &lt;- GO$result[,c(&quot;term_id&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;intersection&quot;)]
    colnames(gem) &lt;- c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;Genes&quot;)
    gem$FDR &lt;- gem$p.Val
    gem$Phenotype = &quot;+1&quot;
    gem &lt;- gem[,c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;FDR&quot;, &quot;Phenotype&quot;, &quot;Genes&quot;)]
    head(gem)
  
    write.table(gem, here::here(&quot;output/enrichmentmap.txt&quot;),col.name=TRUE,sep=&quot;\t&quot;,row.names=FALSE,quote=FALSE)
    
  }
}</code></pre>
<pre class="r"><code>if(enrich_map){
  knitr::include_graphics(paste0(&quot;assets/network_enrichmentmap&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_enrichmentmapSKAP2_C_act_vs_Halo_act.png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="section" class="section level2 unnumbered">
<h2></h2>
<pre class="r"><code>saveWorkbook(excel, here::here(paste0(&quot;output/results_table_&quot;, test, &quot;.xlsx&quot;)), overwrite = T)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] forcats_0.5.1               stringr_1.4.0              
 [3] dplyr_1.0.7                 purrr_0.3.4                
 [5] readr_1.4.0                 tidyr_1.1.3                
 [7] tibble_3.1.2                tidyverse_1.3.1            
 [9] openxlsx_4.2.4              here_1.0.1                 
[11] RColorBrewer_1.1-2          RCy3_2.10.2                
[13] STRINGdb_2.2.2              naniar_0.6.1               
[15] visdat_0.5.3                ggrepel_0.9.1              
[17] ggplot2_3.3.5               UpSetR_1.4.0               
[19] ggstatsplot_0.8.0           datapasta_3.1.0            
[21] patchwork_1.1.1             DT_0.18                    
[23] rrvgo_1.2.0                 org.Hs.eg.db_3.12.0        
[25] AnnotationDbi_1.52.0        gprofiler2_0.2.0           
[27] enrichR_3.0                 preprocessCore_1.52.1      
[29] SummarizedExperiment_1.20.0 Biobase_2.50.0             
[31] GenomicRanges_1.42.0        GenomeInfoDb_1.26.2        
[33] IRanges_2.24.1              S4Vectors_0.28.1           
[35] BiocGenerics_0.36.0         MatrixGenerics_1.2.1       
[37] matrixStats_0.59.0          DEP_1.12.0                 
[39] MBQN_2.2.0                  workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.3            R.methodsS3_1.8.1        
  [3] coda_0.19-4               bit64_4.0.5              
  [5] knitr_1.33                DelayedArray_0.16.2      
  [7] R.utils_2.10.1            data.table_1.14.0        
  [9] RCurl_1.98-1.3            doParallel_1.0.16        
 [11] generics_0.1.0            RSQLite_2.2.7            
 [13] correlation_0.6.1         chron_2.3-56             
 [15] bit_4.0.4                 lubridate_1.7.10         
 [17] xml2_1.3.2                httpuv_1.6.1             
 [19] assertthat_0.2.1          WRS2_1.1-2               
 [21] xfun_0.24                 hms_1.1.0                
 [23] jquerylib_0.1.4           evaluate_0.14            
 [25] promises_1.2.0.1          fansi_0.5.0              
 [27] readxl_1.3.1              caTools_1.18.2           
 [29] dbplyr_2.1.1              igraph_1.2.6             
 [31] DBI_1.1.1                 htmlwidgets_1.5.3        
 [33] reshape_0.8.8             kSamples_1.2-9           
 [35] Rmpfr_0.8-4               paletteer_1.3.0          
 [37] hash_2.2.6.1              ellipsis_0.3.2           
 [39] crosstalk_1.1.1           backports_1.2.1          
 [41] insight_0.14.2            gridBase_0.4-7           
 [43] vctrs_0.3.8               imputeLCMD_2.0           
 [45] Cairo_1.5-12.2            cachem_1.0.5             
 [47] withr_2.4.2               treemap_2.4-2            
 [49] cluster_2.1.0             lazyeval_0.2.2           
 [51] crayon_1.4.1              labeling_0.4.2           
 [53] pkgconfig_2.0.3           SuppDists_1.1-9.5        
 [55] slam_0.1-48               statsExpressions_1.1.0   
 [57] ProtGenerics_1.22.0       wordcloud_2.6            
 [59] rlang_0.4.11              RJSONIO_1.3-1.4          
 [61] lifecycle_1.0.0           MatrixModels_0.5-0       
 [63] sandwich_3.0-1            affyio_1.60.0            
 [65] BiocFileCache_1.14.0      modelr_0.1.8             
 [67] cellranger_1.1.0          rprojroot_2.0.2          
 [69] graph_1.68.0              Matrix_1.2-18            
 [71] mc2d_0.1-21               zoo_1.8-9                
 [73] reprex_2.0.0              whisker_0.4              
 [75] GlobalOptions_0.1.2       pheatmap_1.0.12          
 [77] png_0.1-7                 viridisLite_0.4.0        
 [79] rjson_0.2.20              mzR_2.24.1               
 [81] PMCMRplus_1.9.0           parameters_0.14.0        
 [83] bitops_1.0-7              shinydashboard_0.7.1     
 [85] R.oo_1.24.0               KernSmooth_2.23-17       
 [87] blob_1.2.1                shape_1.4.6              
 [89] multcompView_0.1-8        tmvtnorm_1.4-10          
 [91] ggsignif_0.6.2            scales_1.1.1             
 [93] memoise_2.0.0             magrittr_2.0.1           
 [95] plyr_1.8.6                gplots_3.1.1             
 [97] zlibbioc_1.36.0           compiler_4.0.3           
 [99] plotrix_3.8-1             pcaMethods_1.82.0        
[101] clue_0.3-59               cli_3.0.0                
[103] affy_1.68.0               XVector_0.30.0           
[105] pbapply_1.4-3             MASS_7.3-53              
[107] tidyselect_1.1.1          vsn_3.58.0               
[109] stringi_1.6.2             highr_0.9                
[111] yaml_2.2.1                GOSemSim_2.16.1          
[113] norm_1.0-9.5              MALDIquant_1.19.3        
[115] grid_4.0.3                sass_0.4.0               
[117] tools_4.0.3               rstudioapi_0.13          
[119] circlize_0.4.13           foreach_1.5.1            
[121] git2r_0.28.0              gridExtra_2.3            
[123] ipmisc_6.0.2              pairwiseComparisons_3.1.6
[125] farver_2.1.0              mzID_1.28.0              
[127] digest_0.6.27             BiocManager_1.30.16      
[129] shiny_1.6.0               proto_1.0.0              
[131] BWStest_0.2.2             Rcpp_1.0.6               
[133] broom_0.7.8               BayesFactor_0.9.12-4.2   
[135] performance_0.7.2         later_1.2.0              
[137] ncdf4_1.17                httr_1.4.2               
[139] MSnbase_2.15.7            ComplexHeatmap_2.6.2     
[141] effectsize_0.4.5          colorspace_2.0-2         
[143] rvest_1.0.0               XML_3.99-0.6             
[145] fs_1.5.0                  rematch2_2.1.2           
[147] gmm_1.6-6                 plotly_4.9.4.1           
[149] xtable_1.8-4              gmp_0.6-2                
[151] jsonlite_1.7.2            zeallot_0.1.0            
[153] NLP_0.2-1                 R6_2.5.0                 
[155] tm_0.7-8                  gsubfn_0.7               
[157] pillar_1.6.1              htmltools_0.5.1.1        
[159] mime_0.11                 glue_1.4.2               
[161] fastmap_1.1.0             BiocParallel_1.24.1      
[163] codetools_0.2-16          mvtnorm_1.1-2            
[165] utf8_1.2.1                lattice_0.20-41          
[167] bslib_0.2.5.1             sqldf_0.4-11             
[169] curl_4.3.2                gtools_3.9.2             
[171] zip_2.2.0                 GO.db_3.12.1             
[173] limma_3.46.0              rmarkdown_2.9            
[175] munsell_0.5.0             GetoptLong_1.0.5         
[177] GenomeInfoDbData_1.2.4    iterators_1.0.13         
[179] impute_1.64.0             haven_2.4.1              
[181] gtable_0.3.0              bayestestR_0.10.0        </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
