<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit at IEO" />


<title>LFQ_Analysis</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.18/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.9.4.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Template_LFQ</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LFQ_Analysis</h1>
<h4 class="author">Fabio Bedin | MS-Unit at IEO</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-29
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>SKAP2_HaloTag/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20211221code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20211221)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20211221code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20211221)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong2865ca5"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 2865ca5 </a>
</p>
</div>
<div id="strongRepositoryversionstrong2865ca5" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 2865ca5. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  Counts.pdf
    Untracked:  Intensity_Distributions.pdf
    Untracked:  code/custom_functions.R
    Untracked:  data/expdesign.tsv
    Untracked:  data/proteinGroups.txt
    Untracked:  output/enrichmentmap.txt
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_vs_Halo.png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_vs_Halo.png
    Untracked:  output/output.zip
    Untracked:  output/results_table_Halo._vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_C._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_C._vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_C_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_C..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_N_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_N_vs_Halo.xlsx

Unstaged changes:
    Modified:   analysis/_site.yml
    Deleted:    analysis/about.Rmd
    Deleted:    analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Halo_nt_vs_Halo_act.Rmd</code>) and HTML (<code>docs/Halo_nt_vs_Halo_act.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
2865ca5
</td>
<td>
FabioBedin
</td>
<td>
2021-12-29
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>## Proteomics
library(&quot;MBQN&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;enrichR&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)

## Plots
library(&quot;DT&quot;)
library(&quot;patchwork&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)

## Networks
library(&quot;STRINGdb&quot;)
library(&quot;RCy3&quot;)
library(&quot;RColorBrewer&quot;)

## Other
library(&quot;here&quot;)
library(&quot;openxlsx&quot;)
library(&quot;tidyverse&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(paste0(&quot;data/&quot;, params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot; &amp; data$Peptides &gt;= params$n_peptides,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<pre class="r"><code>cond_1 &lt;- params$test_1

cond_2 &lt;- params$test_2

conditions&lt;-c(cond_1,cond_2)

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)

expdesign &lt;- read.table(here::here(paste0(&quot;data/&quot;, params$expdesign)), header = T, stringsAsFactors = F)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
if(max(expdesign$replicate)&lt;=3){
       threshold&lt;-0
     } else if(max(expdesign$replicate)&lt;6){
       threshold&lt;-1
     } else if (max(expdesign$replicate)&gt;=6){
       threshold&lt;-trunc(max(expdesign$replicate)*0.25) ## 0.25 serve per avere sempre il 75% di valid vales
     }

data_filt &lt;- filter_missval(data_se, thr = threshold)</code></pre>
<div id="halo.-vs-halo_act" class="section level1">
<h1><strong>Halo.</strong> vs <strong>Halo_act</strong></h1>
<hr />
</div>
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>First, we need to wrangling the original dataset. From the <strong>proteinGroups</strong> obtained by <a href="https://www.maxquant.org/">MaxQuant</a> software, I remove proteins that are marked form the software as potential contaminant, only identify by site and reverse.<br />
We che also filter the initial datasets by excluding all proteins identified by less than 2 peptides.<br />
Now, we generate some quality control plots to investigate the structure of the dataset and observe the effect of filters.</p>
<div id="normalization-strategy" class="section level4">
<h4><strong>Normalization strategy:</strong></h4>
<pre class="r"><code>if(params$Normalization == &quot;VSN&quot;){
  data_filt &lt;- normalize_vsn(data_filt)
  print(&quot;VSN normalization is used&quot;)
} else if(params$Normalization == &quot;MBQN&quot;){
  data_filt &lt;- normalize_MBQN(data_filt)
  print(&quot;MBQN normalization is used&quot;)
} else {
  print(&quot;Max-LFQ normalization is used&quot;)
}</code></pre>
<p>[1] “Max-LFQ normalization is used”</p>
</div>
<div id="quality-control-plots" class="section level2 tabset tabset-fade">
<h2><strong>1. Quality control plots</strong></h2>
<div id="proteins-per-samples" class="section level3 tabset">
<h3>Proteins per samples</h3>
<p>This plot show the number of proteins identify in each samples <em>after and before User define</em> filters:</p>
<div id="filtered" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_filt)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/proteins-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_se)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/proteins-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="distributions" class="section level3 tabset">
<h3>Distributions</h3>
<p>This plot is useful for checking the distributions of the samples <em>after and before User define</em> filters:</p>
<div id="filtered-1" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_normalization(data_filt)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/normalization-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-1" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_normalization(data_se)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/normalization-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="missing-data" class="section level3 tabset">
<h3>Missing data</h3>
<p>This plot we can explore the missing data pattern <em>after and before User define</em> filters. Notice that the percentages of missingness are provided in the data. These are accurate to 1 decimal place. Also the dataset is arranged by columns with most missingness.</p>
<div id="filtered-2" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/missing-data-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-2" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>assay(data_se) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/missing-data-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample-cvs" class="section level3 tabset">
<h3>Sample CVs</h3>
<p>The coefficient of variation (CV) is a statistical measure of the dispersion of data points in a data series around the mean. The coefficient of variation represents the ratio of the standard deviation to the mean, and it is a useful statistic for comparing the degree of variation from one data series to another, even if the means are drastically different from one another.</p>
<div id="filtered-3" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_cvs(data_filt)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/CVs-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-3" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_cvs(data_se)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/CVs-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="upset-plot" class="section level2 tabset tabset-fade">
<h2><strong>2. Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within the 2 conditions or in common between all condition.</p>
<div id="halo." class="section level3">
<h3>Halo.</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_1)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_1)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/upset-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="halo_act" class="section level3">
<h3>Halo_act</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_2)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_2)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/upset-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="both-conditions" class="section level3">
<h3>Both conditions</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()
n_sets &lt;- length(define_set)
n_each &lt;- n_sets / 2

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;red3&quot;, &quot;royalblue1&quot;), each = n_each),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/upset-both-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="imputation" class="section level2">
<h2><strong>3. Imputation</strong></h2>
<p>An idealized version of a label-free discovery mass spectrometry proteomics experiment would provide absolute abundance measurements for a whole proteome, across varying conditions. Unfortunately, this ideal is not realized. Measurements are made on peptides requiring an inferential step to obtain protein level estimates. The inference is complicated by experimental factors that necessitate relative abundance estimation and result in widespread non-ignorable missing data. Relative abundance on the log scale takes the form of parameter contrasts. In a complete-case analysis, contrast estimates may be biased by missing data and a substantial amount of useful information will often go unused.</p>
<pre class="r"><code>data_imputed &lt;- mix_imputation_mean(data_filt, cond1 = cond_1, cond2 = cond_2)

plot_imputation(data_filt, data_imputed)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/imputation-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>data_filt &lt;- data_imputed</code></pre>
</div>
<div id="differential-enrichment-analysis" class="section level2">
<h2><strong>4. Differential enrichment analysis</strong></h2>
<p>Protein-wise linear models combined with empirical Bayes statistics are used for the differential enrichment analysis (or differential expression analysis) and False discovery rate (FDR) is ajusted with <em>Benjamini-Hochberg</em> procedure. Significant proteins are are define with <strong>FDR = 0.05</strong> and <strong>Fold change = 1</strong></p>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results&lt;-get_results(dep)</code></pre>
<p>There are <strong>499</strong> significant proteins in <strong>Halo.</strong> vs <strong>Halo_act</strong> comparison.</p>
<p>The results from the previous analysis can be easily visualized by a number of functions. These visualizations assist in the determination of the optimal cutoffs to be used, highlight the most interesting samples and contrasts, and pinpoint differentially enriched/expressed proteins.</p>
<div id="visualization-of-the-results" class="section level3 tabset tabset-fade">
<h3><strong>4.1. Visualization of the results</strong></h3>
<div id="pca" class="section level4">
<h4>PCA</h4>
<p>The PCA plot can be used to get a high-level overview of the data. This can be very useful to observe batch effects, such as clear differences between replicates.</p>
<pre class="r"><code>if(params$Imputation){
  if(nrow(dep) &gt;= 500){
    n_value &lt;- 500
  }else {
    n_value &lt;- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
} else {
  var &lt;- apply(assay(dep), 1, sd)
  df &lt;- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value &lt;- df %&gt;% as.data.frame() %&gt;% drop_na() %&gt;% nrow()
  if(n_value &gt;= 500){
    n_value &lt;- 500
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
}</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/PCA-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano" class="section level4">
<h4>Volcano</h4>
<p>Volcano plots allows to inspect the enrichment of proteins between the two samples (x axis) and their corresponding adjusted p value (y axis).</p>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T, label_size=5, adjusted = F)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/volcano-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-and-unique" class="section level4">
<h4>Volcano and unique</h4>
<pre class="r"><code>plot_volcano_2(dep, contrast=test, add_names=T, label_size=5, adjusted = F, unique_vec = &quot;SKAP2&quot;)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/volcano-unique-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="correlation" class="section level4">
<h4>Correlation</h4>
<pre class="r"><code>plot_cor(dep, significant = TRUE, lower = 0, upper = 1, pal = &quot;Reds&quot;)</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/correlation-matrix-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-heatmap" class="section level4">
<h4>Cluster heatmap</h4>
<p>The heatmap representation gives an overview of all significant proteins (rows) in all samples (columns). This allows to see general trends, for example if one sample or replicate is really different compared to the others. Additionally, the clustering of samples (columns) can indicate closer related samples and clustering of proteins (rows) indicates similarly behaving proteins.</p>
<pre class="r"><code>if(results %&gt;% filter(significant) %&gt;% nrow() &lt;= 100){
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = T, indicate = &quot;condition&quot;, col_limit = 5)
}else {
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = F, indicate = &quot;condition&quot;, col_limit = 5)
}</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/cluster-heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="result-table" class="section level3">
<h3><strong>4.2. Result table</strong></h3>
<p>In this table are summarized the results of DEP analysis, sorted by p.value. Proteins that are UP regulated are colored with a red box in Fold change column, in the orhter hand proteins that are DOWN regulated are colored with blue.</p>
<pre class="r"><code>results %&gt;% 
  select(starts_with(&quot;name&quot;) | starts_with(test)) %&gt;% 
  filter(across(ends_with(&quot;significant&quot;))) %&gt;% 
  arrange(across(ends_with(&quot;p.adj&quot;))) %&gt;%
  mutate(across(2:3, format, scientific = T, digits = 2)) %&gt;%
  dplyr::rename_with(~ tolower(gsub(paste0(test,&quot;_&quot;), &quot;&quot;, .x)), .cols = starts_with(test)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(Fold.Change = ratio, Gene.name = name) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:5)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;Fold.Change&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-2a0dd1ba32e0541469b5" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2a0dd1ba32e0541469b5">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499"],["CSNK2B-LY6G5B-1181","FLAD1","ATP6V1B2","RAB21","RCN2","WDR5","CROCC","NDUFA10","INF2","AATF","COMMD2","ORC4","SNUPN","STRBP","UBR2","IMP3","HELLS","LAMP2","UBXN4","ANKRD39","H1FX","MRPS9","PWP2","CAST","NCOA4","MORF4L1","CBLL1","PYCRL","PDIA3","CLTA","POLR2J3","DDX39A","TNPO3","ZNHIT6","SIT1","ANKRD44","ARAP1","DLG3","NOM1","EGR2","UTP11L","GSPT2","BUD31","MRPS23","TRAFD1","CDSN","CCM2","CCDC22","MFGE8","CKAP2","MELK","MTOR","PTCD3","KIAA1429","RNASEH2A","SFSWAP","KDM3A","SMC6","CPSF6","GZMA","DNM1L","GNG4","PSME4","PIK3CD","PSPC1","SQLE","OSBP","ICAM1","A0A3B3ITJ4","TRA2A","GBP4","TBCE","GABRA2","FHOD1","IGBP1","NRBP1","SSR1","MKRN2","RIOK1","PARG","DDX52","LAMTOR3","POLD2","FAM107B","PREP","TRIM24","PPAT","PCM1","NUMB","GPX1","CSNK1G2","TMOD3","TMEM43","RSAD1","CYBRD1","NOL8","HMGN1","KPNA3","MRPS17","SNX20","TAB2","CHAMP1","RAP1A","TUBB3","RPTOR","SVIP","MRPS35","YME1L1","LGALS3","OPTN","CYCS","ACOT9","G3BP2","FOXRED1","WDR43","SYTL1","PPAN-P2RY11","MRPS14","API5","MRPS15","ISOC2","TCF25","ZGPAT","KIFC1","PPP1R16B","LAT","THY1","KIF22","DOCK11","MAEA","CDV3","LGALS1","RRM2","UBA2","RAP2B","CHEK1","RHOA","SERPINB9","COPB2","GNB2L1.1","SVIL","IGF2R","PDCD4","TUFM","C1orf21","CWC22","KIAA0101","DDX49","HRAS","UQCRC2","RAP2C","UBE2C","RPS20","GRK6","MYL12A","IK","WDR74","IL16","FAM162A","RPS16","STAT1","CCNB1","PDCD6","POLR2C","ECH1","CHUK","LCK","CDC20","LMNA","INPP5A","ACSL1","HLA-C.1","NIFK","DDX39A.1","RPS12","UHRF1","APEH","C12orf75","FYN","HARS","PTGS2","LIMA1","GNL3","RRP12","TPM4","HIST2H2AC","TPM3","WDR36","KIN","EIF3G","GBP5","NHP2L1","RFC3","MYH9","SRPR","STK10","KIAA0020","RPL11","MYH14","CALCOCO1","FAM120A","PDE12","RAB8B","GATAD2B","DIAPH1","PPP3CA","MKI67","UPP1","SF3B6","SDHB","PNPO","PARN","ULBP2","RAD21","H2AFV","PCK2","MAP7","THUMPD1","CAPNS1","ZMPSTE24","LRRC8C","GIGYF2","C20orf27","TSN","B2M","HNRNPM.1","MYH10","DNTTIP2","MYL6","RRM1","RPL7L1","PLA2G4A","GTPBP4","RPS5","TCAF2","SMARCC2","PTMA","SLC2A1","STMN2","TOP2A","BANF1","LGALS8","YTHDF1","ESYT1","TIPRL","NOP56","TRAF1","DKC1","PBK","SLFN5","PRPF31","VIM","KDM2A","RCL1","MCM7","XIRP1","HNRNPC","ICAM3","ATP6V1A","PHF5A","ARF5","GNAI3","PKP2","EPDR1","PFKFB3","SAFB","PRKCSH","HIST1H4A","RAB6A","TNFAIP3","CBX3","RALA","C9orf142","HEATR3","JUNB","LMNB2","TOP2B","RPL21","RRP1","SDHA","NOP58","PSMG2","IVNS1ABP","GNB2","BAZ1B","TPD52L2","TMSB4X","PDIA6","DHX36","TTN","LSM14A","AURKB","NME1","GNAI2","MRTO4","BYSL","Pe1Fe10","EMB","ACTN4","CALCOCO2","YBX3","RP9","BLMH","ATP1B3","GNG2","ADRM1","RRAS2","VCP","FLOT1","HSP90AB2P","EZR","DNAJB11","GOT2","CBFB","DDX56","GNB2L1","GNA13","PSMA6","CALM2","PSMD7","NUSAP1","PNN","PPA2","MSN","NOSIP","RBM22","ACADVL","DAAM1","GNG5","ANLN","SRP9","GSTK1","NOL11","PKM","RRP1B","CD44","TMA16","PBRM1","UTP3","MARCKSL1","GRAPL","PRKAR2A","CD97","CHCHD3","DDX18","PGAM1","POLR2L","TSR1","DDX39B","NCOA5","GNB1","ARL6IP4","RPL38","PSMA5","PGRMC2","HLA-B","UBAP2","LAMTOR1","SOD1","DLST","SH3BGRL3","EFR3A","RACGAP1","FBL","MYO1G","RFTN1","EIF5A","FUBP1","HDHD3","RASA4B","EPB41L2","TMED10","YES1","EIF3B","CD3EAP","H2AFY","HEATR1","ITGB2","RALY","CORO2A","S100A8","CCDC84","CD5","ALKBH4","IGF2BP1","UTP6","WDR3","ACAT1","NOP2","EBNA1BP2","HSPE1","RP2","TRMT10C","BST2","POLR2A","POLRMT","RRS1","WDR12","ALDOA","APOL2","HIST1H1B","ACIN1","PRNP","SET","RBMX","UBE2S","UTP14A","PDCD11","SEMA7A","LMNB1","PTBP3","C12orf10","HIST2H3PS2","STMN1","HDAC10","PAG1","RFK","RSL1D1","SYNE1","SNRPD1","SNAP23","CHAC2","LLPH","IQGAP1","UBE2L3","ENO1","ESF1","COX5A","STXBP2","KIAA1524","SON","ZW10","KPNA4","EIF4H","RAP1B.1","GNAO1","POLR2G","SPTBN1","CYB5R3","PDIA4","NRD1","PDCD2L","CD247","NAP1L1","OPA1","COTL1","GDI2","KTN1","DDOST","GNAS","LMAN2","EIF3L","DDX19A","AP1M1","ADRBK2","CD48","FTSJ3","JUP","NGDN","PTPRC","DHX30","FUS","MDH2","SLC3A2","FLOT2","ACAP1","HLA-A","RAB7A","HIST1H2BN","MMRN2","SRSF7","AFG3L2","BCLAF1","NOL10","BOP1","ITM2A","AP1B1","CANX","IMMT.1","NHP2","IMMT","ERBB2IP","DHCR7","OGDH","NARF","UGP2","TPI1","FMNL1.1","GLIPR2","PPP1R9B","RBM28","DSP","KRR1","LAMP1","RAB1B","DYNC1H1","SSRP1","WDR75","CALR"],["7.7e-14","2.3e-13","8.1e-13","1.3e-12","1.1e-12","2.1e-12","4.1e-12","4.1e-12","5.1e-12","7.2e-12","7.6e-12","7.0e-12","1.4e-11","1.2e-11","1.4e-11","1.5e-11","1.8e-11","1.8e-11","1.7e-11","1.9e-11","2.3e-11","2.2e-11","2.1e-11","2.5e-11","2.4e-11","2.7e-11","3.3e-11","3.3e-11","4.3e-11","4.6e-11","4.7e-11","4.9e-11","5.6e-11","5.9e-11","6.1e-11","6.9e-11","6.9e-11","7.1e-11","7.0e-11","8.0e-11","1.0e-10","1.1e-10","1.2e-10","1.2e-10","1.3e-10","1.3e-10","1.4e-10","1.4e-10","1.5e-10","1.7e-10","1.6e-10","1.7e-10","1.6e-10","1.8e-10","1.7e-10","2.2e-10","2.8e-10","2.8e-10","3.5e-10","3.7e-10","4.0e-10","4.1e-10","4.1e-10","4.3e-10","4.3e-10","4.6e-10","6.0e-10","6.3e-10","7.3e-10","7.3e-10","9.0e-10","8.9e-10","9.8e-10","1.0e-09","1.1e-09","1.2e-09","1.4e-09","1.4e-09","1.4e-09","1.5e-09","1.7e-09","1.7e-09","1.7e-09","1.9e-09","1.9e-09","1.9e-09","2.2e-09","2.3e-09","2.7e-09","2.9e-09","3.4e-09","3.7e-09","5.2e-09","5.6e-09","6.0e-09","6.3e-09","6.8e-09","7.2e-09","7.3e-09","9.8e-09","1.0e-08","1.0e-08","1.1e-08","1.3e-08","1.5e-08","1.5e-08","1.7e-08","2.0e-08","2.0e-08","2.5e-08","3.2e-08","4.1e-08","4.1e-08","4.9e-08","6.2e-08","7.1e-08","7.2e-08","7.6e-08","1.1e-07","1.3e-07","1.4e-07","1.4e-07","1.5e-07","1.6e-07","1.6e-07","2.3e-07","2.3e-07","2.6e-07","3.2e-07","3.4e-07","4.4e-07","4.7e-07","4.9e-07","5.0e-07","5.9e-07","6.3e-07","6.2e-07","6.5e-07","6.8e-07","6.8e-07","7.0e-07","7.5e-07","8.1e-07","8.1e-07","1.3e-06","1.3e-06","1.4e-06","1.4e-06","1.5e-06","1.7e-06","1.8e-06","2.1e-06","2.2e-06","2.3e-06","2.4e-06","2.4e-06","2.7e-06","2.8e-06","3.4e-06","3.5e-06","3.5e-06","3.6e-06","3.6e-06","3.9e-06","4.9e-06","5.1e-06","5.1e-06","5.8e-06","5.9e-06","6.0e-06","6.6e-06","6.9e-06","6.9e-06","7.7e-06","8.0e-06","8.1e-06","8.5e-06","8.8e-06","9.3e-06","9.5e-06","9.4e-06","1.0e-05","1.1e-05","1.1e-05","1.2e-05","1.6e-05","1.7e-05","1.8e-05","1.9e-05","1.9e-05","2.0e-05","2.0e-05","2.1e-05","2.1e-05","2.2e-05","2.3e-05","2.3e-05","2.4e-05","2.6e-05","2.6e-05","2.7e-05","2.9e-05","2.9e-05","3.0e-05","3.3e-05","3.6e-05","3.9e-05","4.0e-05","4.0e-05","4.1e-05","4.4e-05","4.5e-05","4.6e-05","4.7e-05","4.8e-05","4.9e-05","5.1e-05","5.1e-05","5.4e-05","5.4e-05","5.5e-05","5.6e-05","5.7e-05","5.8e-05","6.3e-05","6.5e-05","6.6e-05","6.7e-05","6.7e-05","6.8e-05","7.3e-05","7.5e-05","7.7e-05","8.1e-05","8.6e-05","8.9e-05","9.0e-05","9.1e-05","9.1e-05","9.7e-05","1.0e-04","1.1e-04","1.1e-04","1.1e-04","1.1e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.3e-04","1.3e-04","1.5e-04","1.5e-04","1.5e-04","1.5e-04","1.5e-04","1.5e-04","1.6e-04","1.6e-04","1.7e-04","1.7e-04","1.7e-04","2.0e-04","2.0e-04","2.0e-04","2.1e-04","2.3e-04","2.3e-04","2.3e-04","2.5e-04","2.5e-04","2.6e-04","2.6e-04","2.6e-04","2.7e-04","2.8e-04","2.8e-04","2.8e-04","2.8e-04","2.9e-04","2.9e-04","3.0e-04","3.1e-04","3.2e-04","3.2e-04","3.3e-04","3.4e-04","3.6e-04","3.6e-04","3.6e-04","3.7e-04","3.8e-04","3.9e-04","4.0e-04","4.2e-04","4.2e-04","4.3e-04","4.3e-04","4.4e-04","4.5e-04","4.7e-04","4.9e-04","5.1e-04","5.2e-04","5.4e-04","5.6e-04","5.7e-04","5.8e-04","5.8e-04","6.1e-04","6.3e-04","6.6e-04","6.7e-04","6.7e-04","6.8e-04","6.9e-04","7.0e-04","7.2e-04","7.3e-04","7.4e-04","7.7e-04","7.9e-04","8.3e-04","8.3e-04","8.4e-04","8.9e-04","9.0e-04","9.5e-04","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","2.0e-03","2.0e-03","2.1e-03","2.1e-03","2.1e-03","2.1e-03","2.1e-03","2.2e-03","2.2e-03","2.2e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.4e-03","2.4e-03","2.4e-03","2.5e-03","2.6e-03","2.6e-03","2.6e-03","2.8e-03","2.8e-03","2.9e-03","3.0e-03","3.0e-03","3.1e-03","3.2e-03","3.2e-03","3.2e-03","3.3e-03","3.3e-03","3.5e-03","3.6e-03","3.6e-03","3.6e-03","3.6e-03","3.6e-03","3.7e-03","3.8e-03","3.9e-03","3.9e-03","3.9e-03","4.1e-03","4.1e-03","4.1e-03","4.2e-03","4.4e-03","4.4e-03","4.5e-03","4.7e-03","4.9e-03","5.0e-03","5.2e-03","5.3e-03","5.3e-03","5.4e-03","5.4e-03","5.6e-03","5.9e-03","6.1e-03","6.2e-03","6.4e-03","6.5e-03","6.7e-03","6.8e-03","6.9e-03","6.9e-03","7.0e-03","7.0e-03","7.0e-03","7.2e-03","7.3e-03","7.4e-03","7.5e-03","7.6e-03","7.5e-03","7.6e-03","7.7e-03","7.7e-03","7.8e-03","7.7e-03","7.9e-03","8.1e-03","8.4e-03","8.5e-03","8.8e-03","8.8e-03","9.1e-03","9.0e-03","9.1e-03","9.2e-03","9.5e-03","9.5e-03","9.9e-03","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02"],["1.7e-10","2.6e-10","5.7e-10","5.7e-10","5.7e-10","7.8e-10","1.2e-09","1.2e-09","1.3e-09","1.4e-09","1.4e-09","1.4e-09","2.1e-09","2.1e-09","2.1e-09","2.2e-09","2.2e-09","2.2e-09","2.2e-09","2.2e-09","2.2e-09","2.2e-09","2.2e-09","2.3e-09","2.3e-09","2.3e-09","2.7e-09","2.7e-09","3.4e-09","3.4e-09","3.4e-09","3.4e-09","3.8e-09","3.9e-09","3.9e-09","4.1e-09","4.1e-09","4.1e-09","4.1e-09","4.5e-09","5.6e-09","5.7e-09","6.2e-09","6.2e-09","6.3e-09","6.3e-09","6.6e-09","6.7e-09","6.9e-09","7.2e-09","7.2e-09","7.2e-09","7.2e-09","7.3e-09","7.3e-09","8.8e-09","1.1e-08","1.1e-08","1.4e-08","1.4e-08","1.5e-08","1.5e-08","1.5e-08","1.5e-08","1.5e-08","1.6e-08","2.0e-08","2.1e-08","2.4e-08","2.4e-08","2.8e-08","2.8e-08","3.0e-08","3.1e-08","3.4e-08","3.6e-08","4.0e-08","4.2e-08","4.2e-08","4.2e-08","4.7e-08","4.7e-08","4.7e-08","5.0e-08","5.0e-08","5.0e-08","5.7e-08","5.9e-08","6.8e-08","7.3e-08","8.4e-08","9.1e-08","1.3e-07","1.3e-07","1.4e-07","1.5e-07","1.6e-07","1.7e-07","1.7e-07","2.2e-07","2.2e-07","2.3e-07","2.4e-07","2.8e-07","3.2e-07","3.2e-07","3.6e-07","4.1e-07","4.2e-07","5.1e-07","6.6e-07","8.3e-07","8.3e-07","9.7e-07","1.2e-06","1.4e-06","1.4e-06","1.5e-06","2.1e-06","2.4e-06","2.6e-06","2.6e-06","2.8e-06","3.0e-06","3.0e-06","4.1e-06","4.1e-06","4.5e-06","5.6e-06","5.8e-06","7.6e-06","8.1e-06","8.4e-06","8.4e-06","9.8e-06","1.0e-05","1.0e-05","1.1e-05","1.1e-05","1.1e-05","1.1e-05","1.2e-05","1.3e-05","1.3e-05","2.0e-05","2.1e-05","2.1e-05","2.2e-05","2.3e-05","2.6e-05","2.7e-05","3.1e-05","3.3e-05","3.4e-05","3.5e-05","3.6e-05","3.9e-05","4.0e-05","4.8e-05","5.0e-05","5.0e-05","5.0e-05","5.0e-05","5.4e-05","6.7e-05","6.9e-05","7.0e-05","7.8e-05","7.9e-05","8.0e-05","8.8e-05","9.1e-05","9.1e-05","1.0e-04","1.0e-04","1.0e-04","1.1e-04","1.1e-04","1.2e-04","1.2e-04","1.2e-04","1.3e-04","1.4e-04","1.4e-04","1.4e-04","1.9e-04","2.0e-04","2.1e-04","2.3e-04","2.3e-04","2.3e-04","2.4e-04","2.4e-04","2.5e-04","2.5e-04","2.6e-04","2.6e-04","2.8e-04","2.9e-04","2.9e-04","3.1e-04","3.2e-04","3.2e-04","3.3e-04","3.6e-04","3.9e-04","4.3e-04","4.3e-04","4.4e-04","4.4e-04","4.7e-04","4.8e-04","4.9e-04","4.9e-04","5.1e-04","5.1e-04","5.3e-04","5.3e-04","5.5e-04","5.5e-04","5.6e-04","5.7e-04","5.7e-04","5.8e-04","6.2e-04","6.4e-04","6.5e-04","6.5e-04","6.5e-04","6.6e-04","7.0e-04","7.2e-04","7.4e-04","7.7e-04","8.1e-04","8.4e-04","8.5e-04","8.5e-04","8.5e-04","9.0e-04","9.3e-04","9.7e-04","9.7e-04","9.9e-04","1.0e-03","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.7e-03","1.7e-03","1.7e-03","1.8e-03","1.9e-03","1.9e-03","1.9e-03","2.0e-03","2.0e-03","2.1e-03","2.1e-03","2.1e-03","2.1e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.3e-03","2.4e-03","2.4e-03","2.4e-03","2.5e-03","2.6e-03","2.7e-03","2.7e-03","2.7e-03","2.8e-03","2.8e-03","2.8e-03","2.9e-03","3.0e-03","3.0e-03","3.1e-03","3.1e-03","3.1e-03","3.2e-03","3.3e-03","3.4e-03","3.6e-03","3.6e-03","3.7e-03","3.9e-03","3.9e-03","3.9e-03","3.9e-03","4.1e-03","4.2e-03","4.4e-03","4.4e-03","4.5e-03","4.5e-03","4.5e-03","4.6e-03","4.7e-03","4.7e-03","4.8e-03","4.9e-03","5.0e-03","5.3e-03","5.3e-03","5.3e-03","5.6e-03","5.6e-03","5.9e-03","6.3e-03","6.3e-03","6.4e-03","6.4e-03","6.4e-03","6.4e-03","6.5e-03","6.5e-03","6.6e-03","6.6e-03","6.8e-03","6.9e-03","7.0e-03","7.1e-03","7.1e-03","7.2e-03","7.2e-03","7.3e-03","7.4e-03","7.4e-03","7.5e-03","7.5e-03","7.7e-03","7.8e-03","7.9e-03","7.9e-03","8.0e-03","8.0e-03","8.2e-03","8.4e-03","8.9e-03","8.9e-03","9.0e-03","9.0e-03","9.1e-03","9.1e-03","9.1e-03","9.3e-03","9.3e-03","9.3e-03","9.3e-03","9.5e-03","9.7e-03","9.7e-03","9.8e-03","9.9e-03","9.9e-03","9.9e-03","9.9e-03","9.9e-03","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","2.0e-02","2.1e-02","2.1e-02","2.1e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.5e-02","2.6e-02","2.6e-02","2.6e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.9e-02","2.9e-02","2.9e-02","2.9e-02","2.9e-02","3.0e-02","3.0e-02","3.1e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.3e-02","3.4e-02","3.4e-02","3.5e-02","3.5e-02","3.7e-02","3.7e-02","3.7e-02","3.7e-02","3.8e-02","3.8e-02","4.0e-02","4.0e-02","4.1e-02","4.2e-02","4.2e-02","4.2e-02","4.4e-02","4.4e-02","4.4e-02","4.5e-02","4.6e-02","4.7e-02","4.8e-02","4.9e-02","4.9e-02","4.9e-02","5.0e-02"],["True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True"],[-6.97,5.27,-4.9,-4.48,5.81,-4.86,-7.04,4.25,-3.97,4.25,-3.85,3.84,5.05,3.54,-4.78,5.81,3.72,-4.81,-3.88,4.36,-5.3,-5.07,3.98,-4.45,3.4,4.23,8.56,-4.81,-6.06,-4.33,3.47,-7.12,-4.49,3.75,-3.64,-5.02,-3.43,-5.67,4.42,-4.18,4.74,-5.35,4.69,-3.56,-3.12,5.58,-5.26,-3.22,-3.76,3.69,5.51,-3.33,3.11,-2.98,4,-3.02,-3.66,3.58,-4.57,-5.84,-4.1,-6.93,-3.16,-4.17,-5.14,-4.34,-3.47,-4.93,5.29,5.3,-3.58,-4.09,-6.21,-3.29,-2.45,-4.1,-5.07,-3.98,3.19,-3.97,3.25,-5.73,3.38,-2.97,-3.57,3.03,-3.06,-2.44,-3.67,2.81,-6.14,-6.82,-4.01,3.43,-5.87,4,-3.87,-4.1,-3.92,3.46,-3.51,2.34,-5.75,-4.73,-3.78,-5.83,-3.84,-11.3,-3.53,-2.22,-4.08,4.76,-3.87,-4.4,3.07,-2.78,3.63,-4.52,-4.2,3.87,2.85,-2.09,-3.89,2.76,-5.01,-7.07,-7.64,3.03,-5.47,-3.21,-4.26,-2.07,2.01,-3.49,-5.37,1.82,4.96,-4.3,-3.14,1.79,-3.38,-3.52,2.92,1.65,-4.05,2.61,3.61,1.73,-3.91,-3.97,-5.49,1.75,1.77,-3.95,-1.75,1.65,2.79,-1.92,1.81,1.55,-1.41,2.57,-2.48,1.99,-4.64,-1.69,-6.58,2.16,-2.82,-3.79,2.57,-7.05,4.82,1.56,1.52,1.18,-1.46,-5.75,-4.55,-3.77,-4.43,-1.35,2.83,2.45,-1.17,2.93,-1.26,4.07,1.24,-1.14,-1.5,2.25,1.18,-1.05,2.25,-1.15,4.02,1.29,-1.65,-3.13,1.43,-1.15,-3.28,-1.48,-1.06,-1.77,1.71,-2.16,1.08,2.01,-1.53,2.07,-4.17,1.41,3.04,1.36,1.19,1.2,1.25,-2.76,-1.97,-1.75,1.08,-1.83,-3.26,1.14,-3.18,3.61,-1.06,1.63,3.21,-2.56,3.13,1.68,-1.78,-1.5,-4.76,-2.1,-3.6,4.35,3.42,-4.5,1.29,-1.49,1.03,2.25,-2,2.09,1.96,-1.34,1.07,2.58,1.65,3.2,1.21,-4.62,2.34,-3.87,-1.35,1.18,1.3,-4.53,1.56,-1.68,-1.94,1.65,-2.85,2.2,-1.52,-1.2,1.38,-3.62,1.27,1.21,-2.2,2.38,1.6,-1.26,3.82,1.29,1.87,1.09,1.49,-5.14,3.98,-2.01,-3,-1.07,1.16,-1.42,-1.59,1.18,-1.98,-3.24,1.38,1.97,1.31,-1.63,-1.87,-5.09,1.48,2.78,1.55,-2.48,-4.43,1.29,-5.59,-2.64,-3.79,-2.33,-3.42,-1.03,-2.72,-1,3.84,1.14,-4.71,-2.7,-2.49,-1.29,1.34,2.15,1.23,-2.25,1.09,1.04,-1.63,-3.69,-4.45,1.49,1.1,1.22,1.44,-1.09,1.05,-3.5,-2.02,1.22,1.76,-2.9,1.31,-1.26,-1.71,2,2.04,-2.44,1.32,1.15,1.29,-1.24,-2.34,1.28,1.17,-1.79,1.3,-2.32,1.03,-1.6,-1.46,-2.27,-3.44,-3.99,1.1,1.35,-3.86,-3.67,-1.21,-1.75,-1.13,1.13,-5.54,-1.92,-3.71,-2.31,1.03,3.08,3.79,-2.44,3.96,1.12,3.3,1.35,-3.17,-1.66,-1.59,2.12,1.71,-1.1,2.26,1.88,-2.26,-3.84,-1.01,-3.8,2.03,1.04,3.01,2.57,-2.08,-1.21,1.1,1.85,-2.73,-2.72,1.92,1.01,3.23,4.71,-4.57,1.27,1.5,1.03,1.68,-1.43,-1.03,-3.68,1.14,2.42,1.01,1.45,-3.8,1.22,1.37,-1.42,-1.28,1.84,1.25,-1.04,-3.97,1.73,2.18,1.85,-1.41,-1.12,-3.25,-2.98,1.16,-2.12,-1.13,-2.08,-1.04,1.22,1.31,-1.07,-1.06,-1.15,-1.91,-1.59,-1.51,-1.26,-2,-2.24,-1.06,-1.01,1.06,-2.63,2.4,2.39,2.05,-3.32,1.62,-3.34,-2.03,-1.3,-3.22,-1.01,-1.04,-1.3,1.73,1.65,1.05,1.01,1.58,1.63,2.47,-1.04,-1.18,-1,2.36,1.12,1.87,-2.62,1.47,-3.26,1.04,-2.02,-2.73,-3.15,-2.52,-1.94,1.15,1.99,1.71,-1.42,-1.05,-1.24,2.77,1.53,-3.29]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene.name<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>Fold.Change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[5]; $(this.api().cell(row, 5).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[4]; $(this.api().cell(row, 4).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
<div id="gene-ontology" class="section level2">
<h2><strong>5. Gene Ontology</strong></h2>
<p>Now I perfom a gene onology analysis (GO or by branch <a href="GO:MF" class="uri">GO:MF</a>, <a href="GO:BP" class="uri">GO:BP</a>, <a href="GO:CC" class="uri">GO:CC</a>) and a REAC ontology.<br />
The enrichment results are visualized with an interactive <strong>Manhattan-like-plot</strong> named “Gostplot”.</p>
<div id="halo.-1" class="section level3 tabset tabset-fade">
<h3><strong>5.1. Halo.</strong></h3>
<div id="gostplot" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in Halo. were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &gt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_1)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-ed4906484ed42468d86d" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-ed4906484ed42468d86d">{"x":{"data":[{"x":[81.4008725542289,116.213716409311,81.3032102818689,93.7551500077641,101.03850178107,109.813081328489,110.098555663079,152.668037688289,93.7476375252749,80.5482057917015,125.739544205652,82.5465261338361,116.262547545491,110.049724526899,164.924652869464,116.266303786735,102.037661952138,93.7513937665195,81.4046287954735,73.1897291935028,89.893734008301,85.9872431139025,119.969957653925,146.81581382918,73.1596792635459,72.9718672013152,73.0169420962506,73.1634355047905,72.964354718826,82.8132192622037,131.625574235962,118.302186541316,120.184063404868,120.187819646112,158.726854815851,73.5240346642735,72.43848094458,116.217472650555,101.057282987294,146.766982693,83.598273682328,146.53033949459,72.5060932869831,72.8366425165091,133.560038476938,123.46701825266,131.708211543343,83.695935954688,127.628933551693,133.00035853149,155.654249477757,80.9726610523429,123.474530735149,101.376563493086,83.7072046784218,185.058105940595,107.07853770241,167.501434363269,107.086050184899,167.497678122025,91.527698949708,85.6980125380672,133.567550959427,179.990936501611,121.329716984475,131.715724025832,107.082293943654,94.4312734317946,132.996602290246,153.546998139529,186.549333714707,107.074781461165,179.987180260366,81.4196537604519,83.6696422659757,83.5644675111265,72.6413179717892,73.6329656603673,129.901459504684,118.959528759124,182.947098361122,73.3512475670212,83.7147171609111,144.87759334696,144.873837105715,96.6587244898506,103.949588745646,73.1747042285244,171.370362845222,151.514871626192,107.07102521992,73.1859729522582,104.704593235814,131.730748990811,104.719618200792,72.6676116605015,176.020589506054,151.428478077566,91.0919749653328,73.2235353647043,73.2197791234597,104.727130683281,127.5538087268,83.3428492776943,110.087286939345,72.8291300340199,182.939585878633,131.704455302099,109.891962394626,83.4029491376081,81.4234100016966,129.766234819878,100.933327026221],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.4172104716197,13.5143457892957,12.7480503162647,12.6134499861046,12.5855606354317,12.431805928768,10.8643992464009,9.82944338179527,8.69517283188485,8.69517283188485,8.6729777357822,8.62044742013199,7.43794472312714,7.14712731428642,6.39719803545814,5.88339133457856,5.78620016642139,5.52520025052269,5.43607229834336,5.32955265077288,5.25448810761506,5.1694550557414,4.97262202644449,4.6006522931315,4.51013008462864,4.47227706253142,4.26568399745962,4.15956043934646,4.02422424848644,3.74109506441023,3.72840597972035,3.54696713778882,3.54643007203659,3.51147872838696,3.41714740930927,3.32034799047594,3.29653233809607,3.18143558131848,3.16788398342729,3.16788398342729,3.16788398342729,3.16788398342729,3.12548972490725,3.07056361404067,3.02537628758634,3.02348750364154,3.02348750364154,3.0166230910671,2.96363568809316,2.95406814058111,2.92810659287391,2.88438061396364,2.81714824496342,2.81507671703447,2.71196425406818,2.71196425406818,2.64293948174719,2.63147245768584,2.6117061272839,2.58026536781208,2.47748997969706,2.47176105848166,2.2860123470739,2.27603987122644,2.19317764532165,2.16899411976494,2.16899411976494,2.16899411976494,2.14776311124595,2.10038760202253,2.05458070157167,2.02236537033443,2.01315416628356,1.94045944245223,1.90194241090374,1.88505874367824,1.88083823575099,1.8006607891628,1.7770721063901,1.77330177966864,1.77209641026807,1.6517039790099,1.61912965546297,1.61912965546297,1.60753358751984,1.5861037129529,1.56488207691346,1.49353496559625,1.48292114957429,1.48137435885965,1.38493997988573,1.34315304644136,1.34299944873058,1.33765199803187,1.33401563453487,1.31879197067433],"text":["GO:0006396 (1010) <br> RNA processing <br> 9.134e-51","GO:0042254 (335) <br> ribosome biogenesis <br> 5.046e-49","GO:0006364 (265) <br> rRNA processing <br> 1.135e-47","GO:0016072 (275) <br> rRNA metabolic process <br> 8.860e-47","GO:0022613 (473) <br> ribonucleoprotein complex biogenesis <br> 2.551e-45","GO:0034470 (439) <br> ncRNA processing <br> 3.896e-37","GO:0034660 (535) <br> ncRNA metabolic process <br> 8.676e-34","GO:0090304 (5234) <br> nucleic acid metabolic process <br> 2.355e-21","GO:0016070 (4714) <br> RNA metabolic process <br> 3.482e-21","GO:0006139 (5752) <br> nucleobase-containing compound metabolic process <br> 4.242e-21","GO:0046483 (5929) <br> heterocycle metabolic process <br> 2.229e-20","GO:0006725 (5973) <br> cellular aromatic compound metabolic process <br> 4.793e-20","GO:0042273 (72) <br> ribosomal large subunit biogenesis <br> 7.896e-20","GO:0034641 (6576) <br> cellular nitrogen compound metabolic process <br> 1.956e-19","GO:1901360 (6227) <br> organic cyclic compound metabolic process <br> 8.317e-19","GO:0042274 (74) <br> ribosomal small subunit biogenesis <br> 2.654e-18","GO:0030490 (50) <br> maturation of SSU-rRNA <br> 8.434e-17","GO:0016071 (880) <br> mRNA metabolic process <br> 3.826e-16","GO:0006397 (532) <br> mRNA processing <br> 3.060e-14","GO:0000470 (29) <br> maturation of LSU-rRNA <br> 1.786e-13","GO:0010467 (6192) <br> gene expression <br> 2.435e-13","GO:0008380 (459) <br> RNA splicing <br> 2.597e-13","GO:0044085 (3377) <br> cellular component biogenesis <br> 3.700e-13","GO:0071840 (6880) <br> cellular component organization or biogenesis <br> 1.366e-11","GO:0000462 (37) <br> maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 1.481e-10","GO:0000377 (360) <br> RNA splicing, via transesterification reactions with bulged adenosine as nucleophile <br> 2.018e-09","GO:0000398 (360) <br> mRNA splicing, via spliceosome <br> 2.018e-09","GO:0000463 (15) <br> maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 2.123e-09","GO:0000375 (363) <br> RNA splicing, via transesterification reactions <br> 2.396e-09","GO:0006807 (10148) <br> nitrogen compound metabolic process <br> 3.648e-08","GO:0051276 (1270) <br> chromosome organization <br> 7.126e-08","GO:0043170 (9867) <br> macromolecule metabolic process <br> 4.007e-07","GO:0044237 (10941) <br> cellular metabolic process <br> 1.308e-06","GO:0044238 (10642) <br> primary metabolic process <br> 1.636e-06","GO:0140014 (346) <br> mitotic nuclear division <br> 2.984e-06","GO:0000819 (214) <br> sister chromatid segregation <br> 3.664e-06","GO:0000027 (27) <br> ribosomal large subunit assembly <br> 4.682e-06","GO:0042255 (62) <br> ribosome assembly <br> 5.566e-06","GO:0022618 (196) <br> ribonucleoprotein complex assembly <br> 6.769e-06","GO:0071826 (203) <br> ribonucleoprotein complex subunit organization <br> 1.065e-05","GO:0007059 (353) <br> chromosome segregation <br> 2.508e-05","GO:0071704 (11397) <br> organic substance metabolic process <br> 3.089e-05","GO:0000070 (185) <br> mitotic sister chromatid segregation <br> 3.371e-05","GO:0000280 (495) <br> nuclear division <br> 5.424e-05","GO:0051983 (86) <br> regulation of chromosome segregation <br> 6.925e-05","GO:0045839 (48) <br> negative regulation of mitotic nuclear division <br> 9.457e-05","GO:0051304 (95) <br> chromosome separation <br> 1.815e-04","GO:0007088 (110) <br> regulation of mitotic nuclear division <br> 1.869e-04","GO:0048285 (544) <br> organelle fission <br> 2.838e-04","GO:0051784 (56) <br> negative regulation of nuclear division <br> 2.842e-04","GO:0098813 (293) <br> nuclear chromosome segregation <br> 3.080e-04","GO:0006259 (976) <br> DNA metabolic process <br> 3.827e-04","GO:0045841 (40) <br> negative regulation of mitotic metaphase/anaphase transition <br> 4.782e-04","GO:0030071 (59) <br> regulation of mitotic metaphase/anaphase transition <br> 5.052e-04","GO:0007091 (61) <br> metaphase/anaphase transition of mitotic cell cycle <br> 6.585e-04","GO:2000816 (42) <br> negative regulation of mitotic sister chromatid separation <br> 6.794e-04","GO:0033046 (42) <br> negative regulation of sister chromatid segregation <br> 6.794e-04","GO:1902100 (42) <br> negative regulation of metaphase/anaphase transition of cell cycle <br> 6.794e-04","GO:0033048 (42) <br> negative regulation of mitotic sister chromatid segregation <br> 6.794e-04","GO:1902099 (62) <br> regulation of metaphase/anaphase transition of cell cycle <br> 7.490e-04","GO:0010965 (63) <br> regulation of mitotic sister chromatid separation <br> 8.500e-04","GO:0008152 (11915) <br> metabolic process <br> 9.432e-04","GO:0051985 (44) <br> negative regulation of chromosome segregation <br> 9.474e-04","GO:1905819 (44) <br> negative regulation of chromosome separation <br> 9.474e-04","GO:0044784 (64) <br> metaphase/anaphase transition of cell cycle <br> 9.624e-04","GO:0051306 (65) <br> mitotic sister chromatid separation <br> 1.087e-03","GO:0033047 (45) <br> regulation of mitotic sister chromatid segregation <br> 1.112e-03","GO:0016572 (38) <br> histone phosphorylation <br> 1.180e-03","GO:0051783 (138) <br> regulation of nuclear division <br> 1.305e-03","GO:0090669 (4) <br> telomerase RNA stabilization <br> 1.524e-03","GO:2001251 (92) <br> negative regulation of chromosome organization <br> 1.531e-03","GO:0033045 (70) <br> regulation of sister chromatid segregation <br> 1.941e-03","GO:1905818 (70) <br> regulation of chromosome separation <br> 1.941e-03","GO:0006401 (431) <br> RNA catabolic process <br> 2.275e-03","GO:0007080 (50) <br> mitotic metaphase plate congression <br> 2.336e-03","GO:0007049 (1867) <br> cell cycle <br> 2.445e-03","GO:0000154 (37) <br> rRNA modification <br> 2.629e-03","GO:0000956 (214) <br> nuclear-transcribed mRNA catabolic process <br> 3.331e-03","GO:0050684 (145) <br> regulation of mRNA processing <br> 3.375e-03","GO:0043484 (152) <br> regulation of RNA splicing <br> 5.176e-03","GO:2000234 (10) <br> positive regulation of rRNA processing <br> 5.296e-03","GO:0000723 (169) <br> telomere maintenance <br> 6.409e-03","GO:0007094 (38) <br> mitotic spindle assembly checkpoint signaling <br> 6.777e-03","GO:0071174 (38) <br> mitotic spindle checkpoint signaling <br> 6.777e-03","GO:0071173 (38) <br> spindle assembly checkpoint signaling <br> 6.777e-03","GO:0019083 (181) <br> viral transcription <br> 7.116e-03","GO:0031577 (39) <br> spindle checkpoint signaling <br> 7.936e-03","GO:0000466 (24) <br> maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 8.819e-03","GO:1903311 (384) <br> regulation of mRNA metabolic process <br> 9.498e-03","GO:0086073 (6) <br> bundle of His cell-Purkinje myocyte adhesion involved in cell communication <br> 9.702e-03","GO:0033044 (299) <br> regulation of chromosome organization <br> 1.147e-02","GO:0000469 (27) <br> cleavage involved in rRNA processing <br> 1.253e-02","GO:0032200 (183) <br> telomere organization <br> 1.303e-02","GO:0051310 (65) <br> metaphase plate congression <br> 1.316e-02","GO:0032204 (82) <br> regulation of telomere maintenance <br> 1.582e-02","GO:0000184 (123) <br> nuclear-transcribed mRNA catabolic process, nonsense-mediated decay <br> 1.671e-02","GO:1904666 (28) <br> regulation of ubiquitin protein ligase activity <br> 1.685e-02","GO:0086042 (7) <br> cardiac muscle cell-cardiac muscle cell adhesion <br> 1.690e-02","GO:0010833 (87) <br> telomere maintenance via telomere lengthening <br> 2.230e-02","GO:0000479 (15) <br> endonucleolytic cleavage of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 2.404e-02","GO:0000478 (15) <br> endonucleolytic cleavage involved in rRNA processing <br> 2.404e-02","GO:0032206 (52) <br> positive regulation of telomere maintenance <br> 2.469e-02","GO:0048254 (5) <br> snoRNA localization <br> 2.594e-02","GO:0006974 (894) <br> cellular response to DNA damage stimulus <br> 2.723e-02","GO:0034655 (573) <br> nucleobase-containing compound catabolic process <br> 3.210e-02","GO:0000278 (1098) <br> mitotic cell cycle <br> 3.289e-02","GO:2000232 (15) <br> regulation of rRNA processing <br> 3.301e-02","GO:0051303 (82) <br> establishment of chromosome localization <br> 4.122e-02","GO:0034502 (94) <br> protein localization to chromosome <br> 4.538e-02","GO:0006996 (4136) <br> organelle organization <br> 4.539e-02","GO:0006402 (388) <br> mRNA catabolic process <br> 4.596e-02","GO:0050000 (84) <br> chromosome localization <br> 4.634e-02","GO:0022402 (1388) <br> cell cycle process <br> 4.800e-02"],"key":["GO:0006396","GO:0042254","GO:0006364","GO:0016072","GO:0022613","GO:0034470","GO:0034660","GO:0090304","GO:0016070","GO:0006139","GO:0046483","GO:0006725","GO:0042273","GO:0034641","GO:1901360","GO:0042274","GO:0030490","GO:0016071","GO:0006397","GO:0000470","GO:0010467","GO:0008380","GO:0044085","GO:0071840","GO:0000462","GO:0000377","GO:0000398","GO:0000463","GO:0000375","GO:0006807","GO:0051276","GO:0043170","GO:0044237","GO:0044238","GO:0140014","GO:0000819","GO:0000027","GO:0042255","GO:0022618","GO:0071826","GO:0007059","GO:0071704","GO:0000070","GO:0000280","GO:0051983","GO:0045839","GO:0051304","GO:0007088","GO:0048285","GO:0051784","GO:0098813","GO:0006259","GO:0045841","GO:0030071","GO:0007091","GO:2000816","GO:0033046","GO:1902100","GO:0033048","GO:1902099","GO:0010965","GO:0008152","GO:0051985","GO:1905819","GO:0044784","GO:0051306","GO:0033047","GO:0016572","GO:0051783","GO:0090669","GO:2001251","GO:0033045","GO:1905818","GO:0006401","GO:0007080","GO:0007049","GO:0000154","GO:0000956","GO:0050684","GO:0043484","GO:2000234","GO:0000723","GO:0007094","GO:0071174","GO:0071173","GO:0019083","GO:0031577","GO:0000466","GO:1903311","GO:0086073","GO:0033044","GO:0000469","GO:0032200","GO:0051310","GO:0032204","GO:0000184","GO:1904666","GO:0086042","GO:0010833","GO:0000479","GO:0000478","GO:0032206","GO:0048254","GO:0006974","GO:0034655","GO:0000278","GO:2000232","GO:0051303","GO:0034502","GO:0006996","GO:0006402","GO:0050000","GO:0022402"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[19.5748468722174,18.1063809880406,17.7746518578295,17.8275946356498,18.5810671803456,18.4797189499499,18.7469093243631,21.5400419320257,21.4215722576922,21.646213660555,21.6801776076722,21.6884534803027,15.7642778006896,21.7957590763696,21.7349961629215,15.8100072347003,15.1381652632347,19.399049958471,18.7393780287701,14.1325441121217,21.7287038516454,18.5403306334799,21.0385096214352,21.8459599395529,14.5938600426124,18.2067089747635,18.2067089747635,12.766247544801,18.2182323266084,22.2720700727894,19.862858524424,22.2416129625238,22.3534316445534,22.3235102253876,18.1515039320334,17.4651348098225,13.9932800960407,15.5115658618445,17.3358617234104,17.3876399430454,18.1794066032068,22.3974376347088,17.250203574472,18.6424947383594,16.0578025627736,15.0659099511043,16.2191968510427,16.4531820242158,18.7692301298177,15.3363904218573,17.9177538077844,19.5313397358788,14.7373847204614,15.426535038788,15.4837562909781,14.8262572073822,14.8262572073822,14.8262572073822,14.8262572073822,15.5115658618445,15.5388662640176,22.4452215901572,14.9103337480098,14.9103337480098,15.5656749705695,15.592008586327,14.9507227107922,14.6431687998088,16.8068458716563,9.05045292005097,16.1673930557028,15.717077397889,15.717077397889,18.4546244176415,15.1381652632347,20.3360286952706,14.5938600426124,17.4651348098225,16.882743843444,16.9546559960072,11.8112973047217,17.114947301444,14.6431687998088,14.6431687998088,14.6431687998088,17.2176438847111,14.69098244776,13.7594961631324,18.296083279443,10.4153745295623,17.9464587784159,13.9932800960407,17.2340225412223,15.592008586327,15.9798150349669,16.6286063334144,14.064391440917,10.8656489767313,16.0766579870883,12.766247544801,12.766247544801,15.2071559698249,9.83978094370953,19.4192886618129,18.8385081160232,19.6804883602879,12.766247544801,15.9798150349669,16.2021368290015,21.2723467043792,18.3103826663661,16.0193351590306,19.973189582811],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[68.5696635609685,53.3013761862114,57.3363517375128,62.1001776211723,59.898598782194,57.3138099746564,56.1529091875501,52.9895484666974,53.0534167947906,59.894841821718,59.8798139798137,57.3814352632257,59.8910848612419,59.8835709402897,66.7625655719778,57.7796730736894,59.8760570193376,52.9482219014606,59.8723000588615,56.1641800689783,53.1398268857403,56.1604231085023,53.1886673719292,53.3088901071635,56.1679370294544,55.258752594245,56.878002559432,62.9041671630517,53.1510977671685,58.3018905798634,55.2700234756732,66.9128439910207,62.8966532420996,52.0390374662511,65.4513863658286,52.2982677391001,62.8741114792432,58.3056475403395,52.2606981343394,55.7471574561343,64.6473968239492,53.6132039057254,55.6945600094693,52.5086575257602,56.0740130175526,55.0145501633003,52.290753818148,62.7050482578199,53.1736395300249,53.2675635419267,54.0978518071387,53.1623686485967,58.3394601846241,66.458251773416,53.0872294390753,55.1422868194867,62.8816254001953,53.0008193481256],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,13.843194513196,13.5881873700791,12.6154450289472,12.2520517821154,11.1558119888832,10.8196414590847,9.37175817994379,8.93065221857213,8.70495100546711,8.10335949225572,6.05615945534888,5.93646531703038,5.32637228457328,5.01747016908013,4.71120068264236,4.3683151005879,4.25026539404687,4.16634861587293,4.04719190511588,3.93129203106541,3.82716168542363,3.80226465943774,3.26066194355648,2.77218523339642,2.58625323001178,2.48944611952432,2.2656327612216,2.26290153612251,2.25587428834198,2.10814972898617,2.05130906334744,1.91757571848858,1.88914454765519,1.86346860637294,1.84456981685475,1.79787334089994,1.66820327630961,1.61907007666524,1.55569416856299,1.54758790747284,1.52803337553655,1.40232844737571,1.36404353796391,1.32493734895047,1.32456849869072],"text":["GO:1990904 (694) <br> ribonucleoprotein complex <br> 3.629e-37","GO:0005730 (955) <br> nucleolus <br> 1.172e-36","GO:0031981 (4453) <br> nuclear lumen <br> 2.097e-36","GO:0070013 (5579) <br> intracellular organelle lumen <br> 1.101e-35","GO:0043233 (5579) <br> organelle lumen <br> 1.101e-35","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 1.101e-35","GO:0030684 (79) <br> preribosome <br> 1.546e-35","GO:0005634 (7607) <br> nucleus <br> 4.745e-31","GO:0005654 (4112) <br> nucleoplasm <br> 5.379e-26","GO:0043232 (5094) <br> intracellular non-membrane-bounded organelle <br> 3.006e-24","GO:0043228 (5102) <br> non-membrane-bounded organelle <br> 3.517e-24","GO:0032040 (39) <br> small-subunit processome <br> 6.963e-20","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 1.829e-17","GO:0043229 (13117) <br> intracellular organelle <br> 1.435e-14","GO:0140513 (1236) <br> nuclear protein-containing complex <br> 2.581e-14","GO:0032991 (5541) <br> protein-containing complex <br> 2.424e-13","GO:0043227 (13201) <br> membrane-bounded organelle <br> 5.597e-13","GO:0005622 (14808) <br> intracellular anatomical structure <br> 6.985e-12","GO:0043226 (14027) <br> organelle <br> 1.515e-11","GO:0030687 (24) <br> preribosome, large subunit precursor <br> 4.249e-10","GO:0005681 (192) <br> spliceosomal complex <br> 1.173e-09","GO:0030686 (28) <br> 90S preribosome <br> 1.973e-09","GO:0005694 (1795) <br> chromosome <br> 7.882e-09","GO:0005732 (24) <br> sno(s)RNA-containing ribonucleoprotein complex <br> 8.787e-07","GO:0030688 (15) <br> preribosome, small subunit precursor <br> 1.158e-06","GO:0016604 (816) <br> nuclear body <br> 4.717e-06","GO:0031428 (7) <br> box C/D RNP complex <br> 9.606e-06","GO:0071013 (85) <br> catalytic step 2 spliceosome <br> 1.944e-05","GO:0005684 (93) <br> U2-type spliceosomal complex <br> 4.282e-05","GO:0034388 (6) <br> Pwp2p-containing subcomplex of 90S preribosome <br> 5.620e-05","GO:0016607 (413) <br> nuclear speck <br> 6.818e-05","GO:1902494 (1377) <br> catalytic complex <br> 8.970e-05","GO:0071011 (52) <br> precatalytic spliceosome <br> 1.171e-04","GO:0000228 (237) <br> nuclear chromosome <br> 1.489e-04","GO:0098687 (348) <br> chromosomal region <br> 1.577e-04","GO:0000793 (239) <br> condensed chromosome <br> 5.487e-04","GO:0071005 (50) <br> U2-type precatalytic spliceosome <br> 1.690e-03","GO:0034399 (132) <br> nuclear periphery <br> 2.593e-03","GO:0000775 (200) <br> chromosome, centromeric region <br> 3.240e-03","GO:0030057 (25) <br> desmosome <br> 5.425e-03","GO:0097525 (61) <br> spliceosomal snRNP complex <br> 5.459e-03","GO:0005819 (390) <br> spindle <br> 5.548e-03","GO:0022626 (106) <br> cytosolic ribosome <br> 7.796e-03","GO:0001651 (2) <br> dense fibrillar component <br> 8.886e-03","GO:0030532 (70) <br> small nuclear ribonucleoprotein complex <br> 1.209e-02","GO:0015030 (57) <br> Cajal body <br> 1.291e-02","GO:0000791 (39) <br> euchromatin <br> 1.369e-02","GO:0070761 (7) <br> pre-snoRNP complex <br> 1.430e-02","GO:0005690 (2) <br> U4atac snRNP <br> 1.593e-02","GO:0005721 (23) <br> pericentric heterochromatin <br> 2.147e-02","GO:0005971 (2) <br> ribonucleoside-diphosphate reductase complex <br> 2.404e-02","GO:0005687 (10) <br> U4 snRNP <br> 2.782e-02","GO:0034455 (3) <br> t-UTP complex <br> 2.834e-02","GO:0120114 (82) <br> Sm-like protein family complex <br> 2.965e-02","GO:0005665 (15) <br> RNA polymerase II, core complex <br> 3.960e-02","GO:0016363 (110) <br> nuclear matrix <br> 4.325e-02","GO:0071007 (30) <br> U2-type catalytic step 2 spliceosome <br> 4.732e-02","GO:0005638 (4) <br> lamin filament <br> 4.736e-02"],"key":["GO:1990904","GO:0005730","GO:0031981","GO:0070013","GO:0043233","GO:0031974","GO:0030684","GO:0005634","GO:0005654","GO:0043232","GO:0043228","GO:0032040","GO:0043231","GO:0043229","GO:0140513","GO:0032991","GO:0043227","GO:0005622","GO:0043226","GO:0030687","GO:0005681","GO:0030686","GO:0005694","GO:0005732","GO:0030688","GO:0016604","GO:0031428","GO:0071013","GO:0005684","GO:0034388","GO:0016607","GO:1902494","GO:0071011","GO:0000228","GO:0098687","GO:0000793","GO:0071005","GO:0034399","GO:0000775","GO:0030057","GO:0097525","GO:0005819","GO:0022626","GO:0001651","GO:0030532","GO:0015030","GO:0000791","GO:0070761","GO:0005690","GO:0005721","GO:0005971","GO:0005687","GO:0034455","GO:0120114","GO:0005665","GO:0016363","GO:0071007","GO:0005638"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[19.0913542161997,19.5036413861221,21.3567493317634,21.6119263993467,21.6119263993467,21.6119263993467,15.9184363351098,21.9570470414316,21.2656780982146,21.5094218975349,21.5111955995147,14.69098244776,22.4569580819102,22.5481301823792,19.8290088862669,21.604243722507,22.5549451746932,22.6771653543307,22.6196160803639,13.7594961631324,17.3053449399728,14.064391440917,20.2883553587936,13.7594961631324,12.766247544801,19.3018661075705,10.8656489767313,16.0386970905277,16.1848705044524,10.4153745295623,18.3962492893267,19.9633396074552,15.2071559698249,17.613817251315,18.1595388624734,17.6259854432419,15.1381652632347,16.7382817982581,17.3656954815117,13.8411375859653,15.4837562909781,18.3174719528419,16.394472388579,3.77952755905512,15.717077397889,15.3670429037096,14.69098244776,10.8656489767313,3.77952755905512,13.6736623489524,3.77952755905512,11.8112973047217,7.81088103584182,15.9798150349669,12.766247544801,16.4531820242158,14.1979626749156,9.05045292005097],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[3.64155719716214,3.49505666926895,46.8817514684057,40.706190754139,20.2787581725194,3.64531362095428,46.2506722713273,9.36259063258078,5.77144948730158,16.4659880235044,38.873055943578,46.2469158475352,40.2854712894201,25.5114565149608,25.8194832659157,22.3072270202713,9.08837169575506,20.492874328671,9.05456388162587,16.8679253692626,22.299714172687,4.1599436804765,24.1253361356637,40.9841661147568,22.3222527154398,20.3388609531935,2.12771840893252,46.8704821970293,19.2607673248514,16.0865892204989,40.2742020180437,3.68287785887561,9.58421963631663,40.8301527392793],"y":[17,17,17,17,11.3576765523259,6.7417421981436,6.31699122763649,4.99061524399066,4.83593443319818,4.4157308032906,4.26831763529854,3.71730713361659,3.01263929454258,2.94117039753944,2.89872933394197,2.62145704955488,2.60941043604736,2.60262551427948,2.5906103760146,2.56472899306012,2.55721256498977,2.44100685133569,2.34191714527779,2.34191714527779,2.33819825880126,2.28497760917323,2.22636321122378,2.2229842912788,2.01573474651621,1.93852406951249,1.80667845073966,1.49900466117975,1.47416081530308,1.4003456736635],"text":["GO:0003723 (1951) <br> RNA binding <br> 9.303e-49","GO:0003676 (4252) <br> nucleic acid binding <br> 4.818e-30","GO:1901363 (6210) <br> heterocyclic compound binding <br> 5.174e-29","GO:0097159 (6291) <br> organic cyclic compound binding <br> 2.492e-28","GO:0030515 (32) <br> snoRNA binding <br> 4.389e-12","GO:0003724 (76) <br> RNA helicase activity <br> 1.812e-07","GO:0140098 (394) <br> catalytic activity, acting on RNA <br> 4.820e-07","GO:0008094 (185) <br> ATPase, acting on DNA <br> 1.022e-05","GO:0004386 (163) <br> helicase activity <br> 1.459e-05","GO:0016887 (553) <br> ATPase <br> 3.839e-05","GO:0070034 (22) <br> telomerase RNA binding <br> 5.391e-05","GO:0140097 (299) <br> catalytic activity, acting on DNA <br> 1.917e-04","GO:0086083 (5) <br> cell adhesive protein binding involved in bundle of His cell-Purkinje myocyte communication <br> 9.713e-04","GO:0035173 (16) <br> histone kinase activity <br> 1.145e-03","GO:0035639 (1846) <br> purine ribonucleoside triphosphate binding <br> 1.263e-03","GO:0032555 (1917) <br> purine ribonucleotide binding <br> 2.391e-03","GO:0005524 (1500) <br> ATP binding <br> 2.458e-03","GO:0030621 (6) <br> U4 snRNA binding <br> 2.497e-03","GO:0005515 (14767) <br> protein binding <br> 2.567e-03","GO:0017076 (1932) <br> purine nucleotide binding <br> 2.724e-03","GO:0032553 (1934) <br> ribonucleotide binding <br> 2.772e-03","GO:0003899 (43) <br> DNA-directed 5'-3' RNA polymerase activity <br> 3.622e-03","GO:0034062 (45) <br> 5'-3' RNA polymerase activity <br> 4.551e-03","GO:0097747 (45) <br> RNA polymerase activity <br> 4.551e-03","GO:0032559 (1564) <br> adenyl ribonucleotide binding <br> 4.590e-03","GO:0030554 (1577) <br> adenyl nucleotide binding <br> 5.188e-03","GO:0000166 (2176) <br> nucleotide binding <br> 5.938e-03","GO:1901265 (2177) <br> nucleoside phosphate binding <br> 5.984e-03","GO:0019843 (68) <br> rRNA binding <br> 9.644e-03","GO:0016779 (134) <br> nucleotidyltransferase activity <br> 1.152e-02","GO:0086080 (11) <br> protein binding involved in heterotypic cell-cell adhesion <br> 1.561e-02","GO:0003735 (171) <br> structural constituent of ribosome <br> 3.170e-02","GO:0008177 (2) <br> succinate dehydrogenase (ubiquinone) activity <br> 3.356e-02","GO:0097367 (2282) <br> carbohydrate derivative binding <br> 3.978e-02"],"key":["GO:0003723","GO:0003676","GO:1901363","GO:0097159","GO:0030515","GO:0003724","GO:0140098","GO:0008094","GO:0004386","GO:0016887","GO:0070034","GO:0140097","GO:0086083","GO:0035173","GO:0035639","GO:0032555","GO:0005524","GO:0030621","GO:0005515","GO:0017076","GO:0032553","GO:0003899","GO:0034062","GO:0097747","GO:0032559","GO:0030554","GO:0000166","GO:1901265","GO:0019843","GO:0016779","GO:0086080","GO:0003735","GO:0008177","GO:0097367"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[20.3892145517334,21.3040075767332,21.7319446023732,21.7464055517094,14.3213782810468,15.8543507531641,18.331531878001,17.250203574472,17.0605109063983,18.791152293927,13.5832101454551,17.9464587784159,9.83978094370953,12.9090380872033,20.3223306568802,20.3679885571932,20.0689442128028,10.4153745295623,22.6742247680722,20.3774023473002,20.3786515923268,14.8688640092558,14.9507227107922,14.9507227107922,20.1202673855808,20.1304163003948,20.5203904843139,20.5209404994456,15.6683130595324,16.7615173175404,12.0456958427095,17.1326162234276,3.77952755905512,20.577234061457],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[199.936119123303,199.928603901679,195.321773046176,199.924846290867,195.411955705664,199.83466363138,199.830906020568,196.734634711485,199.838421242192,196.565542224946,199.071868636545,194.611584602709],"y":[17,17,17,17,17,4.27264000259352,4.02242728078939,3.38124044775485,2.26268784078366,1.46080602218395,1.44518564144737,1.4173636095736],"text":["REAC:R-HSA-8868773 (193) <br> rRNA processing in the nucleus and cytosol <br> 7.156e-29","REAC:R-HSA-72312 (203) <br> rRNA processing <br> 5.879e-28","REAC:R-HSA-6791226 (183) <br> Major pathway of rRNA processing in the nucleolus and cytosol <br> 7.009e-25","REAC:R-HSA-6790901 (60) <br> rRNA modification in the nucleus and cytosol <br> 3.396e-24","REAC:R-HSA-8953854 (662) <br> Metabolism of RNA <br> 1.606e-21","REAC:R-HSA-72163 (178) <br> mRNA Splicing - Major Pathway <br> 5.338e-05","REAC:R-HSA-72172 (186) <br> mRNA Splicing <br> 9.497e-05","REAC:R-HSA-72203 (237) <br> Processing of Capped Intron-Containing Pre-mRNA <br> 4.157e-04","REAC:R-HSA-72165 (51) <br> mRNA Splicing - Minor Pathway <br> 5.462e-03","REAC:R-HSA-176417 (6) <br> Phosphorylation of Emi1 <br> 3.461e-02","REAC:R-HSA-157579 (112) <br> Telomere Maintenance <br> 3.588e-02","REAC:R-HSA-168273 (133) <br> Influenza Viral RNA Transcription and Replication <br> 3.825e-02"],"key":["REAC:R-HSA-8868773","REAC:R-HSA-72312","REAC:R-HSA-6791226","REAC:R-HSA-6790901","REAC:R-HSA-8953854","REAC:R-HSA-72163","REAC:R-HSA-72172","REAC:R-HSA-72203","REAC:R-HSA-72165","REAC:R-HSA-176417","REAC:R-HSA-157579","REAC:R-HSA-168273"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,102,204,1)","opacity":0.8,"size":[17.3130398134153,17.3876399430454,17.2340225412223,15.45541914919,19.0294427494667,17.1926950579195,17.2582214035679,17.613817251315,15.1730510425948,10.4153745295623,16.481642870719,16.7499484371089],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,102,204,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"REAC","legendgroup":"REAC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"17241a1b7101":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"17243f2a713f":{"x":{},"xend":{},"y":{},"yend":{}},"172435493b53":{"x":{},"xend":{},"y":{},"yend":{}},"17244e285b86":{"x":{},"xend":{},"y":{},"yend":{}},"172456bc2123":{"x":{},"xend":{},"y":{},"yend":{}},"17247ee55cc4":{"x":{},"y":{}},"17246b7f41c7":{"yintercept":{}}},"cur_data":"17241a1b7101","visdat":{"17241a1b7101":["function (y) ","x"],"17243f2a713f":["function (y) ","x"],"172435493b53":["function (y) ","x"],"17244e285b86":["function (y) ","x"],"172456bc2123":["function (y) ","x"],"17247ee55cc4":["function (y) ","x"],"17246b7f41c7":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData2a0dd1ba"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;%
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-e4a421697d58abce2053" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-e4a421697d58abce2053">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217"],["GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:MF","GO:MF","REAC","GO:MF","REAC","GO:CC","REAC","GO:CC","REAC","GO:CC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","REAC","GO:MF","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","REAC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:MF","GO:MF","GO:CC","GO:BP","GO:MF","GO:MF","GO:CC","GO:BP","GO:BP","GO:MF","GO:MF","GO:MF","GO:MF","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","REAC","GO:CC","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","REAC","REAC","REAC","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP"],[2400,11668,438,2374,5689,7628,4442,9964,378,1452,2720,2134,1446,1137,10040,295,399,11949,2499,10305,2497,312,1271,2133,2496,2129,1295,21373,5687,2173,14204,2705,1464,11681,10027,24636,11682,2132,7894,5688,2130,3961,2401,214,1570,4661,3621,12668,2128,4867,284,19815,2127,206,1140,335,1139,156,168,207,154,348,2776,15771,439,12224,11781,380,1141,12725,12726,22986,303,14,899,11669,7633,1330,1961,19802,1005,2934,2985,19739,32,3852,338,2472,9817,120,1709,902,16286,4001,13599,2471,2932,42,3612,15793,3011,11780,14707,16137,22168,2286,1647,13601,7718,111,3014,29996,9236,25322,9238,25321,5096,3544,16288,28647,13030,10193,15795,9237,6260,5869,6342,16136,21607,30393,2926,9235,28646,2405,3004,5407,2976,1888,4924,1879,1710,68,3959,5405,101,332,15312,576,5891,10379,5411,12399,4883,29434,1029,3398,2473,461,35,11946,257,3016,19299,19298,6462,1015,8403,210,167,26352,4596,21066,9234,3751,1116,213,834,8604,15799,109,2881,10190,8608,344,75,27590,21043,369,4980,223,222,590,8610,14687,2917,341,1719,3880,449,10037,118,29432,2020,1602,2269,1082,321,10338,15792,868,9985,2933,2406,15276,2928,298,7600],["RNA processing","ribosome biogenesis","RNA binding","rRNA processing","rRNA metabolic process","ribonucleoprotein complex biogenesis","ribonucleoprotein complex","ncRNA processing","nucleolus","nuclear lumen","intracellular organelle lumen","organelle lumen","membrane-enclosed lumen","preribosome","ncRNA metabolic process","nucleus","nucleic acid binding","heterocyclic compound binding","rRNA processing in the nucleus and cytosol","organic cyclic compound binding","rRNA processing","nucleoplasm","Major pathway of rRNA processing in the nucleolus and cytosol","intracellular non-membrane-bounded organelle","rRNA modification in the nucleus and cytosol","non-membrane-bounded organelle","Metabolism of RNA","nucleic acid metabolic process","RNA metabolic process","nucleobase-containing compound metabolic process","heterocycle metabolic process","cellular aromatic compound metabolic process","small-subunit processome","ribosomal large subunit biogenesis","cellular nitrogen compound metabolic process","organic cyclic compound metabolic process","ribosomal small subunit biogenesis","intracellular membrane-bounded organelle","maturation of SSU-rRNA","mRNA metabolic process","intracellular organelle","nuclear protein-containing complex","mRNA processing","maturation of LSU-rRNA","protein-containing complex","gene expression","RNA splicing","cellular component biogenesis","membrane-bounded organelle","snoRNA binding","intracellular anatomical structure","cellular component organization or biogenesis","organelle","maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","preribosome, large subunit precursor","spliceosomal complex","90S preribosome","RNA splicing, via transesterification reactions with bulged adenosine as nucleophile","mRNA splicing, via spliceosome","maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","RNA splicing, via transesterification reactions","chromosome","nitrogen compound metabolic process","chromosome organization","RNA helicase activity","macromolecule metabolic process","catalytic activity, acting on RNA","sno(s)RNA-containing ribonucleoprotein complex","preribosome, small subunit precursor","cellular metabolic process","primary metabolic process","mitotic nuclear division","sister chromatid segregation","ribosomal large subunit assembly","nuclear body","ribosome assembly","ribonucleoprotein complex assembly","box C/D RNP complex","ATPase, acting on DNA","ribonucleoprotein complex subunit organization","helicase activity","catalytic step 2 spliceosome","chromosome segregation","organic substance metabolic process","mitotic sister chromatid segregation","ATPase","U2-type spliceosomal complex","mRNA Splicing - Major Pathway","telomerase RNA binding","nuclear division","Pwp2p-containing subcomplex of 90S preribosome","nuclear speck","regulation of chromosome segregation","catalytic complex","negative regulation of mitotic nuclear division","mRNA Splicing","precatalytic spliceosome","nuclear chromosome","chromosomal region","chromosome separation","regulation of mitotic nuclear division","catalytic activity, acting on DNA","organelle fission","negative regulation of nuclear division","nuclear chromosome segregation","DNA metabolic process","Processing of Capped Intron-Containing Pre-mRNA","negative regulation of mitotic metaphase/anaphase transition","regulation of mitotic metaphase/anaphase transition","condensed chromosome","metaphase/anaphase transition of mitotic cell cycle","negative regulation of mitotic sister chromatid separation","negative regulation of sister chromatid segregation","negative regulation of metaphase/anaphase transition of cell cycle","negative regulation of mitotic sister chromatid segregation","regulation of metaphase/anaphase transition of cell cycle","regulation of mitotic sister chromatid separation","metabolic process","negative regulation of chromosome segregation","negative regulation of chromosome separation","metaphase/anaphase transition of cell cycle","cell adhesive protein binding involved in bundle of His cell-Purkinje myocyte communication","mitotic sister chromatid separation","regulation of mitotic sister chromatid segregation","histone kinase activity","histone phosphorylation","purine ribonucleoside triphosphate binding","regulation of nuclear division","telomerase RNA stabilization","negative regulation of chromosome organization","U2-type precatalytic spliceosome","regulation of sister chromatid segregation","regulation of chromosome separation","RNA catabolic process","mitotic metaphase plate congression","purine ribonucleotide binding","cell cycle","ATP binding","U4 snRNA binding","protein binding","nuclear periphery","rRNA modification","purine nucleotide binding","ribonucleotide binding","chromosome, centromeric region","nuclear-transcribed mRNA catabolic process","regulation of mRNA processing","DNA-directed 5'-3' RNA polymerase activity","5'-3' RNA polymerase activity","RNA polymerase activity","adenyl ribonucleotide binding","regulation of RNA splicing","adenyl nucleotide binding","positive regulation of rRNA processing","desmosome","spliceosomal snRNP complex","mRNA Splicing - Minor Pathway","spindle","nucleotide binding","nucleoside phosphate binding","telomere maintenance","mitotic spindle assembly checkpoint signaling","mitotic spindle checkpoint signaling","spindle assembly checkpoint signaling","viral transcription","cytosolic ribosome","spindle checkpoint signaling","maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","dense fibrillar component","regulation of mRNA metabolic process","rRNA binding","bundle of His cell-Purkinje myocyte adhesion involved in cell communication","regulation of chromosome organization","nucleotidyltransferase activity","small nuclear ribonucleoprotein complex","cleavage involved in rRNA processing","Cajal body","telomere organization","metaphase plate congression","euchromatin","pre-snoRNP complex","protein binding involved in heterotypic cell-cell adhesion","regulation of telomere maintenance","U4atac snRNP","nuclear-transcribed mRNA catabolic process, nonsense-mediated decay","regulation of ubiquitin protein ligase activity","cardiac muscle cell-cardiac muscle cell adhesion","pericentric heterochromatin","telomere maintenance via telomere lengthening","endonucleolytic cleavage of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","endonucleolytic cleavage involved in rRNA processing","ribonucleoside-diphosphate reductase complex","positive regulation of telomere maintenance","snoRNA localization","cellular response to DNA damage stimulus","U4 snRNP","t-UTP complex","Sm-like protein family complex","structural constituent of ribosome","nucleobase-containing compound catabolic process","mitotic cell cycle","regulation of rRNA processing","succinate dehydrogenase (ubiquinone) activity","Phosphorylation of Emi1","Telomere Maintenance","Influenza Viral RNA Transcription and Replication","RNA polymerase II, core complex","carbohydrate derivative binding","establishment of chromosome localization","nuclear matrix","protein localization to chromosome","organelle organization","mRNA catabolic process","chromosome localization","U2-type catalytic step 2 spliceosome","lamin filament","cell cycle process"],["9.1e-51","5.0e-49","9.3e-49","1.1e-47","8.9e-47","2.6e-45","3.6e-37","3.9e-37","1.2e-36","2.1e-36","1.1e-35","1.1e-35","1.1e-35","1.5e-35","8.7e-34","4.7e-31","4.8e-30","5.2e-29","7.2e-29","2.5e-28","5.9e-28","5.4e-26","7.0e-25","3.0e-24","3.4e-24","3.5e-24","1.6e-21","2.4e-21","3.5e-21","4.2e-21","2.2e-20","4.8e-20","7.0e-20","7.9e-20","2.0e-19","8.3e-19","2.7e-18","1.8e-17","8.4e-17","3.8e-16","1.4e-14","2.6e-14","3.1e-14","1.8e-13","2.4e-13","2.4e-13","2.6e-13","3.7e-13","5.6e-13","4.4e-12","7.0e-12","1.4e-11","1.5e-11","1.5e-10","4.2e-10","1.2e-09","2.0e-09","2.0e-09","2.0e-09","2.1e-09","2.4e-09","7.9e-09","3.6e-08","7.1e-08","1.8e-07","4.0e-07","4.8e-07","8.8e-07","1.2e-06","1.3e-06","1.6e-06","3.0e-06","3.7e-06","4.7e-06","4.7e-06","5.6e-06","6.8e-06","9.6e-06","1.0e-05","1.1e-05","1.5e-05","1.9e-05","2.5e-05","3.1e-05","3.4e-05","3.8e-05","4.3e-05","5.3e-05","5.4e-05","5.4e-05","5.6e-05","6.8e-05","6.9e-05","9.0e-05","9.5e-05","9.5e-05","1.2e-04","1.5e-04","1.6e-04","1.8e-04","1.9e-04","1.9e-04","2.8e-04","2.8e-04","3.1e-04","3.8e-04","4.2e-04","4.8e-04","5.1e-04","5.5e-04","6.6e-04","6.8e-04","6.8e-04","6.8e-04","6.8e-04","7.5e-04","8.5e-04","9.4e-04","9.5e-04","9.5e-04","9.6e-04","9.7e-04","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.3e-03","1.3e-03","1.5e-03","1.5e-03","1.7e-03","1.9e-03","1.9e-03","2.3e-03","2.3e-03","2.4e-03","2.4e-03","2.5e-03","2.5e-03","2.6e-03","2.6e-03","2.6e-03","2.7e-03","2.8e-03","3.2e-03","3.3e-03","3.4e-03","3.6e-03","4.6e-03","4.6e-03","4.6e-03","5.2e-03","5.2e-03","5.3e-03","5.4e-03","5.5e-03","5.5e-03","5.5e-03","5.9e-03","6.0e-03","6.4e-03","6.8e-03","6.8e-03","6.8e-03","7.1e-03","7.8e-03","7.9e-03","8.8e-03","8.9e-03","9.5e-03","9.6e-03","9.7e-03","1.1e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02","2.1e-02","2.2e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.6e-02","2.7e-02","2.8e-02","2.8e-02","3.0e-02","3.2e-02","3.2e-02","3.3e-02","3.3e-02","3.4e-02","3.5e-02","3.6e-02","3.8e-02","4.0e-02","4.0e-02","4.1e-02","4.3e-02","4.5e-02","4.5e-02","4.6e-02","4.6e-02","4.7e-02","4.7e-02","4.8e-02"],[1010,335,1951,265,275,473,694,439,955,4453,5579,5579,5579,79,535,7607,4252,6210,193,6291,203,4112,183,5094,60,5102,662,5234,4714,5752,5929,5973,39,72,6576,6227,74,12046,50,880,13117,1236,532,29,5541,6192,459,3377,13201,32,14808,6880,14027,37,24,192,28,360,360,15,363,1795,10148,1270,76,9867,394,24,15,10941,10642,346,214,27,816,62,196,7,185,203,163,85,353,11397,185,553,93,178,22,495,6,413,86,1377,48,186,52,237,348,95,110,299,544,56,293,976,237,40,59,239,61,42,42,42,42,62,63,11915,44,44,64,5,65,45,16,38,1846,138,4,92,50,70,70,431,50,1917,1867,1500,6,14767,132,37,1932,1934,200,214,145,43,45,45,1564,152,1577,10,25,61,51,390,2176,2177,169,38,38,38,181,106,39,24,2,384,68,6,299,134,70,27,57,183,65,39,7,11,82,2,123,28,7,23,87,15,15,2,52,5,894,10,3,82,171,573,1098,15,2,6,112,133,15,2282,82,110,94,4136,388,84,30,4,1388],[85,57,105,52,52,61,61,53,68,131,144,144,144,29,54,162,123,145,40,145,40,113,36,125,24,125,57,126,119,132,133,133,16,20,139,134,19,179,16,46,185,49,33,12,110,124,30,87,181,11,192,129,187,11,9,17,9,23,23,8,23,51,155,41,7,150,18,6,6,159,156,20,16,7,22,9,14,4,13,14,9,9,19,160,14,14,9,15,5,22,4,17,10,32,7,15,7,13,12,10,9,8,22,7,16,29,16,7,8,13,8,7,7,7,7,8,8,161,7,7,8,3,8,7,3,4,23,9,3,9,6,8,8,18,7,23,42,20,3,180,9,6,23,23,11,11,10,5,5,5,20,10,20,4,4,6,7,12,24,24,10,6,6,6,10,7,6,5,2,16,6,3,14,7,6,5,5,10,5,5,3,3,6,2,8,5,3,3,6,4,4,2,5,3,24,3,2,6,8,19,27,4,2,2,8,9,3,23,5,7,5,68,15,5,4,2,31]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/BP-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/MF-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/CC-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/scatter-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="halo_act-1" class="section level3 tabset tabset-fade">
<h3><strong>5.2. Halo_act</strong></h3>
<div id="gostplot-1" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in Halo_act were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &lt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_2)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-a2454929a56e939b13f9" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-a2454929a56e939b13f9">{"x":{"data":[{"x":[74.4931449053838,76.5140026949861,83.0648874255928,118.467461156079,118.516292292259,75.9430540258048,75.9130040958479,76.2548220491078,122.189896229492,75.8679292009126,76.5215151774754,115.87941093854,132.658540578231,113.629422433016,76.2848719790647,121.750416003872,132.688590508187,106.793063367819,82.8244879859375,126.795047995388,131.501618274889,159.185116247694,83.2827494177805,94.0293556186209,75.9092478546033,75.8266105472218,76.5102464537415,131.336343660126,107.059756496187,98.7434383806114,85.690500055578,88.6128557438876,130.295864835368,76.641714897303,145.245704988932,98.7509508631006,109.982112184496,144.051220273145,126.952810127662,77.3892069049812,166.047769001604,130.078002843181,128.297544493234,83.2001121103989,92.5306153620199,83.267724452802,121.979546719793,130.934425846953,76.6454711385476,93.6424627704257,93.6650002178934,144.381769502671,126.798804236633,128.312569458212,101.301438668193,76.6266899323246,146.534095735834,101.004695609869,116.964964658234,101.327732356906,89.3227853391196,77.3967193874704,146.522827012101,166.051525242848,130.08551532567,130.292108594124,81.5886846164596,85.9534369427009,146.81581382918,145.640110319616,128.507894002932,119.537989910794,106.935800535114,115.860629732317,142.49238015663,126.133949536336,77.6897062045503,77.6746812395719,116.17239775562,76.4576590763169,77.6483875508596,131.148531597896,121.818028346275,106.601495064344,130.930669605708,96.7826804509229,101.48549448918,108.806408674932,131.817142539437,103.57772086243,130.442358243908,75.8303667884664,119.301346712383,150.771135859759,103.566452138696,83.4517802737881,151.011535299414,83.0611311843482,181.662463855464,158.103318769245,120.255431988515,118.786741661871,165.912544316798,111.807645429379,77.9225931617164,137.879715908244,100.967133197423,129.923996952152,131.535424446091,84.1166349740847,81.4684848966319,83.9250666706094,83.4630489975219,145.369660950004,82.5803323050376,149.122145953373,119.996251342637,132.898940017886,109.501313305186,94.3110737119669,101.000939368624,132.75620285059,121.821784587519,104.941236434224,101.015964333603,165.630826223452,73.5353033880073,118.335992712518,101.042258022315,132.654784336986,131.610549270983,115.582667880216,75.620017278768,100.959620714934,101.823556201195,87.733895292648,119.973713895169,97.8945278593286,83.2714806940466,94.0706742723116,103.585233344919,166.059037725338,131.539180687336,142.477355191651,127.399802835771,87.373296133165,76.6492273797923,170.517696082694,121.81427210503,145.60254790717,104.948748916714,145.74904131571,108.24297248824,129.957803123353,98.7584633455898,115.560130432748,101.305194909438,103.573964621185,85.3223884136058,74.7673505162407,182.62781785533,149.114633470884,75.7251920336172,76.7055709984615,181.827738470227,181.790176057781,136.865530772198,124.139385435446,84.0377539079478,77.9301056442056,158.10707501049,76.739377169663,75.7101670686388,123.294231155408,75.7139233098834,102.661197998744,91.9145917979033,106.560176410653,77.6521437921042,170.510183600205,87.7113578451803,105.294323111218,74.3804576680454,131.603036788494,139.329625028665,132.7148841969,155.207256769648,73.7306279327272,172.448404082426],"y":[14.6841618611522,13.4868508451512,13.46037395865,13.1791192424498,12.9850875001748,12.8460140029309,12.8167094089725,12.7788653071128,12.715580707479,12.6934980229856,12.519189156323,12.4656474506269,12.3596490003902,12.2802821975684,12.2183287579062,12.1622172979381,12.0753712169548,12.0331790215976,12.0200997836511,11.8906411922427,11.8713035516227,11.7674112628315,11.5877987248451,11.4289019783957,10.8687803780232,9.75688242153694,9.68455283296312,9.31834830331757,8.82306374287136,8.67706755562497,8.1872053521357,7.48907893469014,7.46087972207125,7.08446967357355,6.79758277140543,6.77543445094562,6.68522881215758,6.55462428727741,6.41075709814705,6.36773626932377,6.02950053195622,5.96846963567329,5.84132068355559,5.77945610759729,5.7089648217121,5.67955708530305,5.58334150295998,5.55160067820689,5.51125650521824,5.49204385812496,5.48015896432711,5.47402999193576,5.45435049186016,5.40757051317867,5.38790182718222,5.28350194799559,5.27899429322223,5.24489496750199,5.20385886274459,5.13090919836162,5.0246312466556,5.00698039192917,4.91481332976815,4.86950217388066,4.84162926330213,4.82365566796175,4.79711091721755,4.75770971781466,4.74937411265038,4.46004157331088,4.28724156678543,4.27570416396948,4.22371353255668,4.21824741252653,4.18779924454318,4.12979624199957,3.97377326238024,3.92603340741278,3.92587706281959,3.91198810223662,3.91198810223662,3.82928904749336,3.81824574117687,3.46627511149128,3.35706385289753,3.23410296524072,3.23394520203438,3.23271427644493,3.18309102033417,3.18178625498978,3.15949518942512,3.14054453133853,3.12533940044121,3.12361409133244,3.02683108478394,2.97480537218223,2.91537749939876,2.75175177700318,2.72911294576386,2.70738840147754,2.63555851184327,2.61746779969414,2.58071473982001,2.56431568460712,2.53366098964579,2.5227660393805,2.51855243718039,2.45483890017794,2.43165339399596,2.38910749672699,2.38818942602989,2.35956467893769,2.33885354860611,2.33337528724363,2.30244037502244,2.29892082055011,2.29640962325263,2.28714934817544,2.26059925555832,2.25484295765334,2.25318573862693,2.23027298635387,2.17536609811068,2.14731647909365,2.06452291730946,2.01166028728753,1.98950659172547,1.97977990419813,1.96801300541924,1.94285978092856,1.9336799840485,1.91023992270729,1.90278259248575,1.87594103593561,1.85336873667066,1.83901097891475,1.83034816885079,1.82217729349316,1.81676079870428,1.80157018688918,1.79864813862012,1.75619107869305,1.75415331831446,1.74532782255625,1.72972599819816,1.71643414594845,1.70366220675676,1.70304370543471,1.70165646399043,1.69394709881904,1.67340256368949,1.65055507018847,1.63931958259604,1.6382572760292,1.63425896842874,1.62459389563356,1.60965803979372,1.60381869900658,1.5889055017876,1.57583941964157,1.57236106496798,1.56788672833774,1.55489856232783,1.54867392383888,1.5236333960502,1.5236333960502,1.51698114394652,1.51437283168395,1.51081937947105,1.49809795161196,1.49500927107836,1.48682992341732,1.47397884941433,1.46682793323132,1.4620844796517,1.45615354925458,1.45520173819826,1.45175189414637,1.43178893859615,1.42262518164283,1.3983541398222,1.39816217283749,1.38689899550136,1.37110711768939,1.35665769973093,1.34903160737982,1.34390199882449,1.3352424426321,1.31521939520139],"text":["GO:0001775 (1507) <br> cell activation <br> 2.069e-15","GO:0002444 (558) <br> myeloid leukocyte mediated immunity <br> 3.259e-14","GO:0006887 (906) <br> exocytosis <br> 3.464e-14","GO:0043299 (538) <br> leukocyte degranulation <br> 6.620e-14","GO:0043312 (483) <br> neutrophil degranulation <br> 1.035e-13","GO:0002283 (488) <br> neutrophil activation involved in immune response <br> 1.426e-13","GO:0002275 (552) <br> myeloid cell activation involved in immune response <br> 1.525e-13","GO:0002366 (723) <br> leukocyte activation involved in immune response <br> 1.664e-13","GO:0045321 (1341) <br> leukocyte activation <br> 1.925e-13","GO:0002263 (727) <br> cell activation involved in immune response <br> 2.025e-13","GO:0002446 (500) <br> neutrophil mediated immunity <br> 3.026e-13","GO:0042119 (502) <br> neutrophil activation <br> 3.423e-13","GO:0051641 (3519) <br> cellular localization <br> 4.369e-13","GO:0036230 (509) <br> granulocyte activation <br> 5.245e-13","GO:0002376 (3438) <br> immune system process <br> 6.049e-13","GO:0045055 (790) <br> regulated exocytosis <br> 6.883e-13","GO:0051649 (2778) <br> establishment of localization in cell <br> 8.407e-13","GO:0032940 (1351) <br> secretion by cell <br> 9.264e-13","GO:0006810 (5214) <br> transport <br> 9.548e-13","GO:0046903 (1492) <br> secretion <br> 1.286e-12","GO:0051234 (5364) <br> establishment of localization <br> 1.345e-12","GO:0140352 (1413) <br> export from cell <br> 1.708e-12","GO:0006955 (2544) <br> immune response <br> 2.583e-12","GO:0016192 (2193) <br> vesicle-mediated transport <br> 3.725e-12","GO:0002274 (671) <br> myeloid leukocyte activation <br> 1.353e-11","GO:0002252 (1135) <br> immune effector process <br> 1.750e-10","GO:0002443 (888) <br> leukocyte mediated immunity <br> 2.068e-10","GO:0051179 (6938) <br> localization <br> 4.805e-10","GO:0033036 (3148) <br> macromolecule localization <br> 1.503e-09","GO:0019882 (236) <br> antigen processing and presentation <br> 2.103e-09","GO:0008104 (2670) <br> protein localization <br> 6.498e-09","GO:0010033 (3457) <br> response to organic substance <br> 3.243e-08","GO:0050852 (202) <br> T cell receptor signaling pathway <br> 3.460e-08","GO:0002478 (175) <br> antigen processing and presentation of exogenous peptide antigen <br> 8.232e-08","GO:0071310 (2821) <br> cellular response to organic substance <br> 1.594e-07","GO:0019884 (183) <br> antigen processing and presentation of exogenous antigen <br> 1.677e-07","GO:0034613 (1842) <br> cellular protein localization <br> 2.064e-07","GO:0070727 (1856) <br> cellular macromolecule localization <br> 2.789e-07","GO:0048002 (193) <br> antigen processing and presentation of peptide antigen <br> 3.884e-07","GO:0002682 (1734) <br> regulation of immune system process <br> 4.288e-07","GO:1901698 (1178) <br> response to nitrogen compound <br> 9.343e-07","GO:0050776 (1159) <br> regulation of immune response <br> 1.075e-06","GO:0048518 (6474) <br> positive regulation of biological process <br> 1.441e-06","GO:0006928 (2319) <br> movement of cell or subcellular component <br> 1.662e-06","GO:0015031 (1754) <br> protein transport <br> 1.954e-06","GO:0006950 (4321) <br> response to stress <br> 2.091e-06","GO:0045184 (1866) <br> establishment of protein localization <br> 2.610e-06","GO:0051050 (951) <br> positive regulation of transport <br> 2.808e-06","GO:0002479 (75) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent <br> 3.081e-06","GO:0016032 (1011) <br> viral process <br> 3.221e-06","GO:0016043 (6661) <br> cellular component organization <br> 3.310e-06","GO:0070887 (3443) <br> cellular response to chemical stimulus <br> 3.357e-06","GO:0046907 (1723) <br> intracellular transport <br> 3.513e-06","GO:0048522 (5863) <br> positive regulation of cellular process <br> 3.912e-06","GO:0030029 (810) <br> actin filament-based process <br> 4.094e-06","GO:0002474 (99) <br> antigen processing and presentation of peptide antigen via MHC class I <br> 5.206e-06","GO:0071705 (2216) <br> nitrogen compound transport <br> 5.260e-06","GO:0022604 (312) <br> regulation of cell morphogenesis <br> 5.690e-06","GO:0042590 (80) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I <br> 6.254e-06","GO:0030036 (712) <br> actin cytoskeleton organization <br> 7.398e-06","GO:0010243 (1092) <br> response to organonitrogen compound <br> 9.449e-06","GO:0002684 (1079) <br> positive regulation of immune system process <br> 9.841e-06","GO:0071702 (2657) <br> organic substance transport <br> 1.217e-05","GO:1901699 (728) <br> cellular response to nitrogen compound <br> 1.351e-05","GO:0050778 (757) <br> positive regulation of immune response <br> 1.440e-05","GO:0050851 (321) <br> antigen receptor-mediated signaling pathway <br> 1.501e-05","GO:0006457 (230) <br> protein folding <br> 1.595e-05","GO:0008360 (157) <br> regulation of cell shape <br> 1.747e-05","GO:0071840 (6880) <br> cellular component organization or biogenesis <br> 1.781e-05","GO:0071417 (671) <br> cellular response to organonitrogen compound <br> 3.467e-05","GO:0048584 (2368) <br> positive regulation of response to stimulus <br> 5.161e-05","GO:0043933 (2017) <br> protein-containing complex subunit organization <br> 5.300e-05","GO:0032984 (337) <br> protein-containing complex disassembly <br> 5.974e-05","GO:0042110 (495) <br> T cell activation <br> 6.050e-05","GO:0065008 (4116) <br> regulation of biological quality <br> 6.489e-05","GO:0046649 (780) <br> lymphocyte activation <br> 7.417e-05","GO:0002768 (510) <br> immune response-regulating cell surface receptor signaling pathway <br> 1.062e-04","GO:0002764 (513) <br> immune response-regulating signaling pathway <br> 1.186e-04","GO:0042221 (4829) <br> response to chemical <br> 1.186e-04","GO:0002429 (475) <br> immune response-activating cell surface receptor signaling pathway <br> 1.225e-04","GO:0002757 (475) <br> immune response-activating signal transduction <br> 1.225e-04","GO:0051128 (2439) <br> regulation of cellular component organization <br> 1.482e-04","GO:0045088 (316) <br> regulation of innate immune response <br> 1.520e-04","GO:0032879 (2906) <br> regulation of localization <br> 3.418e-04","GO:0051049 (1839) <br> regulation of transport <br> 4.395e-04","GO:0019221 (824) <br> cytokine-mediated signaling pathway <br> 5.833e-04","GO:0030168 (163) <br> platelet activation <br> 5.835e-04","GO:0034097 (1242) <br> response to cytokine <br> 5.852e-04","GO:0051336 (1252) <br> regulation of hydrolase activity <br> 6.560e-04","GO:0031347 (889) <br> regulation of defense response <br> 6.580e-04","GO:0050896 (9487) <br> response to stimulus <br> 6.926e-04","GO:0002253 (566) <br> activation of immune response <br> 7.235e-04","GO:0043624 (228) <br> cellular protein complex disassembly <br> 7.493e-04","GO:0075525 (5) <br> viral translational termination-reinitiation <br> 7.523e-04","GO:0031344 (660) <br> regulation of cell projection organization <br> 9.401e-04","GO:0007010 (1473) <br> cytoskeleton organization <br> 1.060e-03","GO:0080134 (1626) <br> regulation of response to stress <br> 1.215e-03","GO:0006886 (995) <br> intracellular protein transport <br> 1.771e-03","GO:1990778 (339) <br> protein localization to cell periphery <br> 1.866e-03","GO:0120035 (642) <br> regulation of plasma membrane bounded cell projection organization <br> 1.962e-03","GO:0044267 (5280) <br> cellular protein metabolic process <br> 2.314e-03","GO:0043434 (461) <br> response to peptide hormone <br> 2.413e-03","GO:1901652 (551) <br> response to peptide <br> 2.626e-03","GO:0035556 (2886) <br> intracellular signal transduction <br> 2.727e-03","GO:0002831 (426) <br> regulation of response to biotic stimulus <br> 2.926e-03","GO:0060627 (544) <br> regulation of vesicle-mediated transport <br> 3.001e-03","GO:0022411 (577) <br> cellular component disassembly <br> 3.030e-03","GO:0050690 (29) <br> regulation of defense response to virus by virus <br> 3.509e-03","GO:0051246 (2759) <br> regulation of protein metabolic process <br> 3.701e-03","GO:0007212 (41) <br> dopamine receptor signaling pathway <br> 4.082e-03","GO:0006415 (104) <br> translational termination <br> 4.091e-03","GO:0007159 (377) <br> leukocyte cell-cell adhesion <br> 4.370e-03","GO:0007015 (445) <br> actin filament organization <br> 4.583e-03","GO:0071345 (1148) <br> cellular response to cytokine stimulus <br> 4.641e-03","GO:0006734 (43) <br> NADH metabolic process <br> 4.984e-03","GO:0072659 (287) <br> protein localization to plasma membrane <br> 5.024e-03","GO:0044093 (1774) <br> positive regulation of molecular function <br> 5.053e-03","GO:0051716 (7873) <br> cellular response to stimulus <br> 5.162e-03","GO:0034341 (198) <br> response to interferon-gamma <br> 5.488e-03","GO:0016477 (1660) <br> cell migration <br> 5.561e-03","GO:0022603 (1048) <br> regulation of anatomical structure morphogenesis <br> 5.582e-03","GO:0051668 (775) <br> localization within membrane <br> 5.885e-03","GO:0045089 (217) <br> positive regulation of innate immune response <br> 6.678e-03","GO:0032268 (2591) <br> regulation of cellular protein metabolic process <br> 7.123e-03","GO:0022607 (3104) <br> cellular component assembly <br> 8.619e-03","GO:1901564 (6904) <br> organonitrogen compound metabolic process <br> 9.735e-03","GO:0000902 (1045) <br> cell morphogenesis <br> 1.024e-02","GO:0043200 (115) <br> response to amino acid <br> 1.048e-02","GO:0022614 (5) <br> membrane to membrane docking <br> 1.076e-02","GO:0051640 (657) <br> organelle localization <br> 1.141e-02","GO:0051272 (605) <br> positive regulation of cellular component movement <br> 1.165e-02","GO:0040017 (607) <br> positive regulation of locomotion <br> 1.230e-02","GO:0002183 (35) <br> cytoplasmic translational initiation <br> 1.251e-02","GO:0022409 (286) <br> positive regulation of cell-cell adhesion <br> 1.331e-02","GO:0030335 (568) <br> positive regulation of cell migration <br> 1.402e-02","GO:0009725 (958) <br> response to hormone <br> 1.449e-02","GO:0044087 (991) <br> regulation of cellular component biogenesis <br> 1.478e-02","GO:0019538 (5912) <br> protein metabolic process <br> 1.506e-02","GO:0006952 (1956) <br> defense response <br> 1.525e-02","GO:0016236 (337) <br> macroautophagy <br> 1.579e-02","GO:0031349 (364) <br> positive regulation of defense response <br> 1.590e-02","GO:1901701 (1237) <br> cellular response to oxygen-containing compound <br> 1.753e-02","GO:0051247 (1577) <br> positive regulation of protein metabolic process <br> 1.761e-02","GO:0065003 (1724) <br> protein-containing complex assembly <br> 1.798e-02","GO:0048193 (415) <br> Golgi vesicle transport <br> 1.863e-02","GO:0009605 (3114) <br> response to external stimulus <br> 1.921e-02","GO:0002480 (8) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-independent <br> 1.979e-02","GO:1903039 (243) <br> positive regulation of leukocyte cell-cell adhesion <br> 1.981e-02","GO:0045087 (1003) <br> innate immune response <br> 1.988e-02","GO:0071407 (624) <br> cellular response to organic cyclic compound <br> 2.023e-02","GO:0032270 (1488) <br> positive regulation of cellular protein metabolic process <br> 2.121e-02","GO:0071453 (243) <br> cellular response to oxygen levels <br> 2.236e-02","GO:0033554 (2142) <br> cellular response to stress <br> 2.294e-02","GO:0050730 (270) <br> regulation of peptidyl-tyrosine phosphorylation <br> 2.300e-02","GO:0019886 (97) <br> antigen processing and presentation of exogenous peptide antigen via MHC class II <br> 2.321e-02","GO:0040011 (2039) <br> locomotion <br> 2.374e-02","GO:0030030 (1636) <br> cell projection organization <br> 2.457e-02","GO:0031346 (359) <br> positive regulation of cell projection organization <br> 2.490e-02","GO:0007568 (333) <br> aging <br> 2.577e-02","GO:0001885 (70) <br> endothelial cell development <br> 2.656e-02","GO:2000147 (592) <br> positive regulation of cell motility <br> 2.677e-02","GO:0072657 (698) <br> protein localization to membrane <br> 2.705e-02","GO:0002223 (112) <br> stimulatory C-type lectin receptor signaling pathway <br> 2.787e-02","GO:0002495 (101) <br> antigen processing and presentation of peptide antigen via MHC class II <br> 2.827e-02","GO:1990858 (113) <br> cellular response to lectin <br> 2.995e-02","GO:1990840 (113) <br> response to lectin <br> 2.995e-02","GO:0060341 (851) <br> regulation of cellular localization <br> 3.041e-02","GO:0046034 (317) <br> ATP metabolic process <br> 3.059e-02","GO:0007191 (10) <br> adenylate cyclase-activating dopamine receptor signaling pathway <br> 3.084e-02","GO:0002833 (253) <br> positive regulation of response to biotic stimulus <br> 3.176e-02","GO:0120036 (1596) <br> plasma membrane bounded cell projection organization <br> 3.199e-02","GO:0002504 (104) <br> antigen processing and presentation of peptide or polysaccharide antigen via MHC class II <br> 3.260e-02","GO:0002218 (144) <br> activation of innate immune response <br> 3.358e-02","GO:0045785 (442) <br> positive regulation of cell adhesion <br> 3.413e-02","GO:0002220 (115) <br> innate immune response activating cell surface receptor signaling pathway <br> 3.451e-02","GO:0030866 (40) <br> cortical actin cytoskeleton organization <br> 3.498e-02","GO:0014070 (1004) <br> response to organic cyclic compound <br> 3.506e-02","GO:0032868 (293) <br> response to insulin <br> 3.534e-02","GO:0002758 (116) <br> innate immune response-activating signal transduction <br> 3.700e-02","GO:1903037 (341) <br> regulation of leukocyte cell-cell adhesion <br> 3.779e-02","GO:0009719 (1729) <br> response to endogenous stimulus <br> 3.996e-02","GO:0032388 (227) <br> positive regulation of intracellular transport <br> 3.998e-02","GO:0001732 (16) <br> formation of cytoplasmic translation initiation complex <br> 4.103e-02","GO:0051270 (1149) <br> regulation of cellular component movement <br> 4.255e-02","GO:0061028 (51) <br> establishment of endothelial barrier <br> 4.399e-02","GO:0051656 (443) <br> establishment of organelle localization <br> 4.477e-02","GO:0098542 (1222) <br> defense response to other organism <br> 4.530e-02","GO:0001101 (136) <br> response to acid chemical <br> 4.621e-02","GO:1903613 (6) <br> regulation of protein tyrosine phosphatase activity <br> 4.839e-02"],"key":["GO:0001775","GO:0002444","GO:0006887","GO:0043299","GO:0043312","GO:0002283","GO:0002275","GO:0002366","GO:0045321","GO:0002263","GO:0002446","GO:0042119","GO:0051641","GO:0036230","GO:0002376","GO:0045055","GO:0051649","GO:0032940","GO:0006810","GO:0046903","GO:0051234","GO:0140352","GO:0006955","GO:0016192","GO:0002274","GO:0002252","GO:0002443","GO:0051179","GO:0033036","GO:0019882","GO:0008104","GO:0010033","GO:0050852","GO:0002478","GO:0071310","GO:0019884","GO:0034613","GO:0070727","GO:0048002","GO:0002682","GO:1901698","GO:0050776","GO:0048518","GO:0006928","GO:0015031","GO:0006950","GO:0045184","GO:0051050","GO:0002479","GO:0016032","GO:0016043","GO:0070887","GO:0046907","GO:0048522","GO:0030029","GO:0002474","GO:0071705","GO:0022604","GO:0042590","GO:0030036","GO:0010243","GO:0002684","GO:0071702","GO:1901699","GO:0050778","GO:0050851","GO:0006457","GO:0008360","GO:0071840","GO:0071417","GO:0048584","GO:0043933","GO:0032984","GO:0042110","GO:0065008","GO:0046649","GO:0002768","GO:0002764","GO:0042221","GO:0002429","GO:0002757","GO:0051128","GO:0045088","GO:0032879","GO:0051049","GO:0019221","GO:0030168","GO:0034097","GO:0051336","GO:0031347","GO:0050896","GO:0002253","GO:0043624","GO:0075525","GO:0031344","GO:0007010","GO:0080134","GO:0006886","GO:1990778","GO:0120035","GO:0044267","GO:0043434","GO:1901652","GO:0035556","GO:0002831","GO:0060627","GO:0022411","GO:0050690","GO:0051246","GO:0007212","GO:0006415","GO:0007159","GO:0007015","GO:0071345","GO:0006734","GO:0072659","GO:0044093","GO:0051716","GO:0034341","GO:0016477","GO:0022603","GO:0051668","GO:0045089","GO:0032268","GO:0022607","GO:1901564","GO:0000902","GO:0043200","GO:0022614","GO:0051640","GO:0051272","GO:0040017","GO:0002183","GO:0022409","GO:0030335","GO:0009725","GO:0044087","GO:0019538","GO:0006952","GO:0016236","GO:0031349","GO:1901701","GO:0051247","GO:0065003","GO:0048193","GO:0009605","GO:0002480","GO:1903039","GO:0045087","GO:0071407","GO:0032270","GO:0071453","GO:0033554","GO:0050730","GO:0019886","GO:0040011","GO:0030030","GO:0031346","GO:0007568","GO:0001885","GO:2000147","GO:0072657","GO:0002223","GO:0002495","GO:1990858","GO:1990840","GO:0060341","GO:0046034","GO:0007191","GO:0002833","GO:0120036","GO:0002504","GO:0002218","GO:0045785","GO:0002220","GO:0030866","GO:0014070","GO:0032868","GO:0002758","GO:1903037","GO:0009719","GO:0032388","GO:0001732","GO:0051270","GO:0061028","GO:0051656","GO:0098542","GO:0001101","GO:1903613"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[19.9462123686674,18.6847287497765,19.3129380919747,18.6363439324449,18.4924595182474,18.5062609951603,18.6704142203647,19.0237128877517,19.8031784771994,19.0308506328191,18.5387647885811,18.5440992955177,20.9498382841373,18.5625892117027,20.9230640505876,19.1379652435679,20.6760128426016,19.8123220388678,21.395744596271,19.9340018163666,21.4274758345104,19.8674339300097,20.5729683353868,20.3976580599366,18.9268195002008,19.5965245859779,19.287432915256,21.7127740996053,20.8213641129297,17.4986794977524,20.6296556443227,20.9294044050122,17.2731463069687,17.0617353348181,20.6939359660643,17.1279572175344,20.1893119996143,20.1984114205898,17.206351575782,20.1165183285464,19.6428327290729,19.6225996651137,21.636471805168,20.4638403981758,20.1303596037508,21.1841117247357,20.2048660232527,19.3743757472774,15.7373316539529,19.451573873278,21.6679004416094,20.9247361833321,20.1088323582115,21.5266239363287,19.1700431574596,16.1868772799724,20.4100380870974,17.8943821822839,15.8433279821367,19.0038610019127,19.5482837527865,19.5332947906194,20.6239407817799,19.032628413028,19.0830626021267,17.9340698285406,17.4616009111547,16.8995454455618,21.7035374999992,18.9268195002008,20.4885440438041,20.2980535736863,18.0016970365285,18.5253260577865,21.1289356985204,19.1215946463033,18.5652079791051,18.5730308200746,21.3096384784207,18.470049611806,18.470049611806,20.5233846899778,17.9121743965842,20.7285216892983,20.1873524673787,19.191991296678,16.9558134205769,19.7084847763908,19.7184126682615,19.2888644874537,22.0537265707873,18.7035562554922,17.4490032718266,9.79200637188604,18.9052781824991,19.9183441742083,20.0386839143178,19.4314817950327,18.0099018230755,18.8691734015509,21.4098243887321,18.4298227465791,18.6680119996668,20.7204820590056,18.3231184550737,18.6510623608111,18.7289763933658,14.0509285588584,20.6679986229776,14.6957136866834,16.2649666785139,18.1564249614111,18.3821828199127,19.6107216016844,14.781443852525,17.7771707617082,20.1440324362717,21.8513047485316,17.2438757382447,20.0637720125704,19.4967505982493,19.1133238346132,17.3774579655542,20.5944578148066,20.8050634202662,21.7073695277206,19.4931522760457,16.4228177328561,9.79200637188604,18.8993355559252,18.7913735860747,18.7957097408416,14.405947329161,17.7722510341765,18.7082178803185,19.383649545002,19.4264042119455,21.5358724307931,20.2613396475456,18.0016970365285,18.1081960550518,19.7034884746249,20.001517942691,20.1095332573674,18.2875917936538,20.808789697542,11.1749781597015,17.540643442317,19.4415711203883,18.8319514777529,19.930723362667,17.540643442317,20.3697031428378,17.6908552942341,16.1543870489597,20.3110033219573,20.0461208480012,18.0891416461647,17.9851259338563,15.6229705518591,18.7628029787026,18.9781080980935,16.3815087539172,16.2186353293393,16.3954151879507,16.3954151879507,19.2332018170431,17.9165837938067,11.747980750738,17.598331775882,20.0160746648226,16.2649666785139,16.7689423433419,18.3730421667754,16.4228177328561,14.651001049555,19.442826169324,17.8062984506393,16.4363189409718,18.0180536596155,20.1130312162973,17.4426585575357,12.8370677538266,19.6118064881816,15.0832328994516,18.3760965631462,19.6883679423801,16.6818474191902,10.3630624020686],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData76f7c579","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[62.186587712122,67.1194768172047,59.8873279007658,62.0964206606962,57.3401086979889,52.9256801386042,53.3276749095439,56.8479468756235,65.0193359110803,54.8304590999727,66.0825557258088,53.9475733880958,65.259781381549,55.7396435351822,53.6620443919143,67.2359425919629,55.938762440414,55.0746615309174,52.7866726009895,53.64701655001,65.2222117767882,56.5661748399181,56.1078256618373,56.4534660256359,65.3386775515465,54.950681835207,61.0369578064439,65.624206547728,55.5780942347111,62.2692408425955,56.0815269385048,55.574337274235,65.5077407729697,59.5041179322065,61.4690082611922,55.0258210447285,52.9482219014606,59.8760570193376,54.9544387956831,55.7358865747061,59.8835709402897,61.4314386564315,53.8160797714333,59.8723000588615,57.7796730736894,63.648045337314,65.5114977334458,59.8910848612419,59.7445634026751,53.4516546052543,53.4215989214457,65.6091787058237,66.3831125638945,62.7576457044849,56.6262862075352,52.606338498138,66.1576949353302,64.6210981006167,52.5725258538534,52.5950676167098,52.1329614781529,53.4178419609696,53.4478976447782,51.9563843357775,57.6218807336943,59.6243406674408,61.5216057078572,59.898598782194,62.1001776211723,57.3138099746564,56.197992713263,64.6060702587124,61.2586184745321,59.7332925212469,65.4964698915415,56.6187722865831,56.2017496737391,65.3424345120225,56.1190965432655,53.4328698028739,53.8799480995265,54.8680287047335,66.1614518958063,53.9888999533326,57.3476226189411,55.1535577009149,55.9312485194619,59.6055558650604,59.5529584183954,53.4253558819218,63.4188707482736,65.2259687372643,56.092797819933,58.8842194536546,58.072715990823,59.477819208874,60.5447959840784,56.0139016499354,64.4933614444302,53.4854672495389,53.5267938147757,57.9975767813015,55.9124637170815,53.7183987990554,55.1159880961542,59.5492014579193,55.157314661391,53.5080090123953,62.8553266768628,57.1935872394221,57.4415466308429,65.5490673382065,65.312378828214,54.8530008628292,68.5696635609685,61.4351956169075,55.8523523494643,58.8767055327024,66.9128439910207,57.4453035913189,60.9843603597789,57.2912682118,57.3175669351325,55.9049497961294,65.6429913501083,62.8365418744824,55.9199776380337,52.1216905967247,53.4140850004935,53.5568494985843,61.6192866802351,55.8561093099404,53.5042520519193,55.9575472427944,59.2599155012618,54.9431679142549,57.9900628603494,56.6225292470591,56.5398761165855,56.4647369070641,52.9933054271735,61.0745274112046,65.5716091010629,55.8598662704165,54.1579631747558,61.5178487473811],"y":[17,17,17,17,17,17,17,17,17,13.8495363032451,12.9434615284418,12.7231522956229,12.5664757273417,12.5134473860802,12.4732597750046,12.4732597750046,12.4681903439167,12.0313091038516,12.0015042063729,11.9409499929618,11.6862828580966,11.6589615917858,11.4104869706991,11.2149037174021,10.7478687735979,10.6741920662764,10.5368371597511,10.5368371597511,10.2004503865458,9.95208328770714,9.73394652570008,9.4599367144968,8.95501820788617,8.78298099321421,8.78298099321421,8.69581255428931,8.47837799834762,8.11204962807173,7.30823196610155,7.18380127941517,7.1771578179806,7.06444924043577,6.98191391116633,6.78961768876646,6.73776741232231,6.31371961366361,5.88621349088825,5.77936444660151,5.73245298710017,5.68749602418009,5.65600875100483,5.65600875100483,5.6349590932931,5.50587769314808,5.00690861216182,5.00690861216182,4.98181848528021,4.9591706900258,4.9591706900258,4.84762747936105,4.78004321595099,4.78004321595099,4.57845130753515,4.5188416385859,4.49993530939131,4.44641659788219,4.09011343587011,4.08193393794194,4.08193393794194,4.08193393794194,3.88115315230592,3.87600959905532,3.77758602788811,3.70089736467223,3.68635177216226,3.53121755336181,3.4844512385613,3.42072417435534,3.20215277631413,3.14418215924632,3.11557536370051,3.06801914282424,3.03701854192124,2.81802555421597,2.79283992156547,2.74404644411062,2.65689777870388,2.65167077383758,2.65117398492365,2.65117398492365,2.6092190432815,2.6092190432815,2.58914690249976,2.51813120704684,2.42388217867216,2.42296939323993,2.39431322109604,2.3829722427651,2.36152512786257,2.36037841628008,2.35865986819001,2.30029349462663,2.28503993499675,2.25137686379367,2.25137686379367,2.24546221191517,2.24113085152208,2.20844922130975,2.17319154982269,2.17052605397535,2.1704624751977,2.16946070750316,2.1137238383889,2.08003984310973,2.05151035168553,2.0466167006148,2.0466167006148,2.00776488896119,1.95987550158308,1.91633746230783,1.8846727960483,1.85256727907297,1.85256727907297,1.83049027549961,1.81933725933558,1.81465866436373,1.7573111936171,1.73210718735012,1.73210718735012,1.70983742075809,1.68311095672808,1.66262982476352,1.65430790975716,1.65403272700199,1.63838882304207,1.62028355798162,1.61583655273904,1.54049664061447,1.52221050210091,1.4968568668612,1.47096846568394,1.37338290357631,1.36777705411957,1.35940320815201,1.34809763023606,1.30771914958873],"text":["GO:0070062 (2178) <br> extracellular exosome <br> 1.500e-33","GO:1903561 (2263) <br> extracellular vesicle <br> 5.131e-32","GO:0043230 (2265) <br> extracellular organelle <br> 5.564e-32","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 5.564e-32","GO:0031982 (4058) <br> vesicle <br> 2.079e-25","GO:0005615 (3595) <br> extracellular space <br> 1.179e-18","GO:0005737 (11951) <br> cytoplasm <br> 3.218e-18","GO:0031410 (2433) <br> cytoplasmic vesicle <br> 9.275e-18","GO:0097708 (2438) <br> intracellular vesicle <br> 1.060e-17","GO:0009898 (169) <br> cytoplasmic side of plasma membrane <br> 1.414e-14","GO:0099503 (1027) <br> secretory vesicle <br> 1.139e-13","GO:0005925 (420) <br> focal adhesion <br> 1.892e-13","GO:0098562 (195) <br> cytoplasmic side of membrane <br> 2.713e-13","GO:0030055 (427) <br> cell-substrate junction <br> 3.066e-13","GO:0005834 (36) <br> heterotrimeric G-protein complex <br> 3.363e-13","GO:1905360 (36) <br> GTPase complex <br> 3.363e-13","GO:0030141 (857) <br> secretory granule <br> 3.403e-13","GO:0016020 (9829) <br> membrane <br> 9.304e-13","GO:0005576 (4564) <br> extracellular region <br> 9.965e-13","GO:0005829 (5361) <br> cytosol <br> 1.146e-12","GO:0098552 (622) <br> side of membrane <br> 2.059e-12","GO:0031234 (103) <br> extrinsic component of cytoplasmic side of plasma membrane <br> 2.193e-12","GO:0030667 (308) <br> secretory granule membrane <br> 3.886e-12","GO:0031090 (3644) <br> organelle membrane <br> 6.097e-12","GO:0098588 (2165) <br> bounding membrane of organelle <br> 1.787e-11","GO:0012505 (4651) <br> endomembrane system <br> 2.117e-11","GO:0045121 (339) <br> membrane raft <br> 2.905e-11","GO:0098857 (339) <br> membrane microdomain <br> 2.905e-11","GO:0019898 (315) <br> extrinsic component of membrane <br> 6.303e-11","GO:0070161 (838) <br> anchoring junction <br> 1.117e-10","GO:0030659 (789) <br> cytoplasmic vesicle membrane <br> 1.845e-10","GO:0019897 (178) <br> extrinsic component of plasma membrane <br> 3.468e-10","GO:0098796 (1326) <br> membrane protein complex <br> 1.109e-09","GO:0042470 (109) <br> melanosome <br> 1.648e-09","GO:0048770 (109) <br> pigment granule <br> 1.648e-09","GO:0015629 (517) <br> actin cytoskeleton <br> 2.015e-09","GO:0005622 (14808) <br> intracellular anatomical structure <br> 3.324e-09","GO:0043227 (13201) <br> membrane-bounded organelle <br> 7.726e-09","GO:0012506 (1183) <br> vesicle membrane <br> 4.918e-08","GO:0030054 (2107) <br> cell junction <br> 6.549e-08","GO:0043229 (13117) <br> intracellular organelle <br> 6.650e-08","GO:0048471 (745) <br> perinuclear region of cytoplasm <br> 8.621e-08","GO:0005886 (5680) <br> plasma membrane <br> 1.043e-07","GO:0043226 (14027) <br> organelle <br> 1.623e-07","GO:0032991 (5541) <br> protein-containing complex <br> 1.829e-07","GO:0071944 (6178) <br> cell periphery <br> 4.856e-07","GO:0098797 (696) <br> plasma membrane protein complex <br> 1.300e-06","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 1.662e-06","GO:0042995 (2337) <br> cell projection <br> 1.852e-06","GO:0005774 (450) <br> vacuolar membrane <br> 2.054e-06","GO:0005765 (395) <br> lysosomal membrane <br> 2.208e-06","GO:0098852 (395) <br> lytic vacuole membrane <br> 2.208e-06","GO:0120025 (2234) <br> plasma membrane bounded cell projection <br> 2.318e-06","GO:0070820 (163) <br> tertiary granule <br> 3.120e-06","GO:0031254 (13) <br> cell trailing edge <br> 9.842e-06","GO:0001931 (13) <br> uropod <br> 9.842e-06","GO:0101002 (184) <br> ficolin-1-rich granule <br> 1.043e-05","GO:0097517 (70) <br> contractile actin filament bundle <br> 1.099e-05","GO:0001725 (70) <br> stress fiber <br> 1.099e-05","GO:0001772 (44) <br> immunological synapse <br> 1.420e-05","GO:0000323 (723) <br> lytic vacuole <br> 1.659e-05","GO:0005764 (723) <br> lysosome <br> 1.659e-05","GO:0005773 (818) <br> vacuole <br> 2.640e-05","GO:0000139 (725) <br> Golgi membrane <br> 3.028e-05","GO:0032432 (78) <br> actin filament bundle <br> 3.163e-05","GO:0042641 (79) <br> actomyosin <br> 3.578e-05","GO:0055038 (85) <br> recycling endosome membrane <br> 8.126e-05","GO:0043233 (5579) <br> organelle lumen <br> 8.281e-05","GO:0070013 (5579) <br> intracellular organelle lumen <br> 8.281e-05","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 8.281e-05","GO:0030863 (121) <br> cortical cytoskeleton <br> 1.315e-04","GO:0097513 (3) <br> myosin II filament <br> 1.330e-04","GO:0045335 (139) <br> phagocytic vesicle <br> 1.669e-04","GO:0042824 (9) <br> MHC class I peptide loading complex <br> 1.991e-04","GO:0098791 (845) <br> Golgi apparatus subcompartment <br> 2.059e-04","GO:0031252 (426) <br> cell leading edge <br> 2.943e-04","GO:0030864 (92) <br> cortical actin cytoskeleton <br> 3.278e-04","GO:0098590 (1240) <br> plasma membrane region <br> 3.796e-04","GO:0030670 (75) <br> phagocytic vesicle membrane <br> 6.278e-04","GO:0005768 (993) <br> endosome <br> 7.175e-04","GO:0005903 (109) <br> brush border <br> 7.663e-04","GO:0010008 (520) <br> endosome membrane <br> 8.550e-04","GO:0101003 (60) <br> ficolin-1-rich granule membrane <br> 9.183e-04","GO:0005938 (326) <br> cell cortex <br> 1.520e-03","GO:0031984 (1764) <br> organelle subcompartment <br> 1.611e-03","GO:0016459 (57) <br> myosin complex <br> 1.803e-03","GO:0030139 (335) <br> endocytic vesicle <br> 2.203e-03","GO:0042612 (8) <br> MHC class I protein complex <br> 2.230e-03","GO:0042582 (154) <br> azurophil granule <br> 2.233e-03","GO:0005766 (154) <br> primary lysosome <br> 2.233e-03","GO:0071556 (25) <br> integral component of lumenal side of endoplasmic reticulum membrane <br> 2.459e-03","GO:0098553 (25) <br> lumenal side of endoplasmic reticulum membrane <br> 2.459e-03","GO:0030662 (177) <br> coated vesicle membrane <br> 2.575e-03","GO:0035579 (90) <br> specific granule membrane <br> 3.033e-03","GO:0033290 (15) <br> eukaryotic 48S preinitiation complex <br> 3.768e-03","GO:0042175 (1154) <br> nuclear outer membrane-endoplasmic reticulum membrane network <br> 3.776e-03","GO:0044297 (595) <br> cell body <br> 4.034e-03","GO:0030425 (641) <br> dendrite <br> 4.140e-03","GO:0097447 (643) <br> dendritic tree <br> 4.350e-03","GO:0005783 (1995) <br> endoplasmic reticulum <br> 4.361e-03","GO:0005794 (1593) <br> Golgi apparatus <br> 4.379e-03","GO:0033180 (8) <br> proton-transporting V-type ATPase, V1 domain <br> 5.008e-03","GO:0030133 (416) <br> transport vesicle <br> 5.188e-03","GO:0005852 (17) <br> eukaryotic translation initiation factor 3 complex <br> 5.606e-03","GO:0016282 (17) <br> eukaryotic 43S preinitiation complex <br> 5.606e-03","GO:0042581 (158) <br> specific granule <br> 5.682e-03","GO:0016460 (21) <br> myosin II complex <br> 5.739e-03","GO:0005789 (1132) <br> endoplasmic reticulum membrane <br> 6.188e-03","GO:0070993 (18) <br> translation preinitiation complex <br> 6.711e-03","GO:0031680 (12) <br> G-protein beta/gamma-subunit complex <br> 6.753e-03","GO:0032153 (72) <br> cell division site <br> 6.754e-03","GO:0098827 (1137) <br> endoplasmic reticulum subcompartment <br> 6.769e-03","GO:0098576 (33) <br> lumenal side of membrane <br> 7.696e-03","GO:0009986 (896) <br> cell surface <br> 8.317e-03","GO:1990904 (694) <br> ribonucleoprotein complex <br> 8.882e-03","GO:0048475 (97) <br> coated membrane <br> 8.982e-03","GO:0030117 (97) <br> membrane coat <br> 8.982e-03","GO:0035577 (57) <br> azurophil granule membrane <br> 9.823e-03","GO:1902494 (1377) <br> catalytic complex <br> 1.097e-02","GO:0032154 (54) <br> cleavage furrow <br> 1.212e-02","GO:0044853 (115) <br> plasma membrane raft <br> 1.304e-02","GO:0031967 (1244) <br> organelle envelope <br> 1.404e-02","GO:0031975 (1244) <br> envelope <br> 1.404e-02","GO:0030131 (24) <br> clathrin adaptor complex <br> 1.477e-02","GO:0098862 (164) <br> cluster of actin-based cell projections <br> 1.516e-02","GO:0070938 (10) <br> contractile ring <br> 1.532e-02","GO:0030135 (296) <br> coated vesicle <br> 1.749e-02","GO:0000314 (29) <br> organellar small ribosomal subunit <br> 1.853e-02","GO:0005763 (29) <br> mitochondrial small ribosomal subunit <br> 1.853e-02","GO:0005802 (255) <br> trans-Golgi network <br> 1.951e-02","GO:0060187 (8) <br> cell pole <br> 2.074e-02","GO:0030118 (47) <br> clathrin coat <br> 2.175e-02","GO:0005788 (313) <br> endoplasmic reticulum lumen <br> 2.217e-02","GO:0030176 (161) <br> integral component of endoplasmic reticulum membrane <br> 2.218e-02","GO:0036477 (875) <br> somatodendritic compartment <br> 2.299e-02","GO:0010494 (77) <br> cytoplasmic stress granule <br> 2.397e-02","GO:0033178 (17) <br> proton-transporting two-sector ATPase complex, catalytic domain <br> 2.422e-02","GO:0031253 (346) <br> cell projection membrane <br> 2.881e-02","GO:0031227 (169) <br> intrinsic component of endoplasmic reticulum membrane <br> 3.005e-02","GO:0031093 (66) <br> platelet alpha granule lumen <br> 3.185e-02","GO:0005635 (471) <br> nuclear envelope <br> 3.381e-02","GO:0045202 (1348) <br> synapse <br> 4.233e-02","GO:0098835 (7) <br> presynaptic endocytic zone membrane <br> 4.288e-02","GO:0030119 (41) <br> AP-type membrane coat adaptor complex <br> 4.371e-02","GO:0008180 (35) <br> COP9 signalosome <br> 4.486e-02","GO:0055037 (187) <br> recycling endosome <br> 4.924e-02"],"key":["GO:0070062","GO:1903561","GO:0043230","GO:0065010","GO:0031982","GO:0005615","GO:0005737","GO:0031410","GO:0097708","GO:0009898","GO:0099503","GO:0005925","GO:0098562","GO:0030055","GO:0005834","GO:1905360","GO:0030141","GO:0016020","GO:0005576","GO:0005829","GO:0098552","GO:0031234","GO:0030667","GO:0031090","GO:0098588","GO:0012505","GO:0045121","GO:0098857","GO:0019898","GO:0070161","GO:0030659","GO:0019897","GO:0098796","GO:0042470","GO:0048770","GO:0015629","GO:0005622","GO:0043227","GO:0012506","GO:0030054","GO:0043229","GO:0048471","GO:0005886","GO:0043226","GO:0032991","GO:0071944","GO:0098797","GO:0043231","GO:0042995","GO:0005774","GO:0005765","GO:0098852","GO:0120025","GO:0070820","GO:0031254","GO:0001931","GO:0101002","GO:0097517","GO:0001725","GO:0001772","GO:0000323","GO:0005764","GO:0005773","GO:0000139","GO:0032432","GO:0042641","GO:0055038","GO:0043233","GO:0070013","GO:0031974","GO:0030863","GO:0097513","GO:0045335","GO:0042824","GO:0098791","GO:0031252","GO:0030864","GO:0098590","GO:0030670","GO:0005768","GO:0005903","GO:0010008","GO:0101003","GO:0005938","GO:0031984","GO:0016459","GO:0030139","GO:0042612","GO:0042582","GO:0005766","GO:0071556","GO:0098553","GO:0030662","GO:0035579","GO:0033290","GO:0042175","GO:0044297","GO:0030425","GO:0097447","GO:0005783","GO:0005794","GO:0033180","GO:0030133","GO:0005852","GO:0016282","GO:0042581","GO:0016460","GO:0005789","GO:0070993","GO:0031680","GO:0032153","GO:0098827","GO:0098576","GO:0009986","GO:1990904","GO:0048475","GO:0030117","GO:0035577","GO:1902494","GO:0032154","GO:0044853","GO:0031967","GO:0031975","GO:0030131","GO:0098862","GO:0070938","GO:0030135","GO:0000314","GO:0005763","GO:0005802","GO:0060187","GO:0030118","GO:0005788","GO:0030176","GO:0036477","GO:0010494","GO:0033178","GO:0031253","GO:0031227","GO:0031093","GO:0005635","GO:0045202","GO:0098835","GO:0030119","GO:0008180","GO:0055037"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[20.3895089474417,20.43491384342,20.4359600780238,20.4359600780238,21.1127924137548,20.97436852479,22.301288321845,20.520482671709,20.522901551273,17.0098206791988,19.4713253358183,18.3038663648759,17.2214892061892,18.3262982994618,14.4581139285914,14.4581139285914,19.2421674566737,22.0919153891785,21.2460124471743,21.4268505016346,18.8277438199372,16.2496921162547,17.8763374918434,20.9898925305246,20.3823975905089,21.2673252341618,18.0099018230755,18.0099018230755,17.90774964072,19.2135392269153,19.1363383178836,17.0869555296552,19.7893245482402,16.3389366763927,16.3389366763927,18.5833840485582,22.5281903038104,22.4069336180039,19.6480987518461,20.3501024869214,22.4001723500672,19.0624517869717,21.4913424228551,22.4710947059109,21.4637267954854,21.5846981879027,18.9743833960756,22.3097189832527,20.4729795999535,18.3972686497445,18.2202776543847,18.2202776543847,20.4196311626559,16.9558134205769,12.3729643352122,12.3729643352122,17.1360081782449,15.6229705518591,15.6229705518591,14.8225866744181,19.0237128877517,19.0237128877517,19.1826347921651,19.0272871005367,15.801858092204,15.8227421236023,15.9420548318458,21.4713489077623,21.4713489077623,21.4713489077623,16.5019202141693,7.7791008157754,16.715163312453,11.4827422948031,19.2241674808196,18.3231184550737,16.0697362528688,19.7064888615817,15.7373316539529,19.4289457661119,16.3389366763927,18.5910918340989,15.3633750239743,17.9555937940104,20.1372168262823,15.2756954716879,17.9934386332075,11.1749781597015,16.8705055778228,16.8705055778228,13.7618192670139,13.7618192670139,17.0786016380891,16.0344085591459,12.6954028668369,19.6172156871556,18.7694563553214,18.8671354327216,18.8712079238796,20.2849514736126,20.0137886576694,11.1749781597015,18.2908638032579,12.968152232712,12.968152232712,16.9090881701731,13.4111152730112,19.5932234371659,13.0900518224006,12.187219027138,15.6697988614665,19.5987201263685,14.2961523059962,19.2988369642948,18.9706470582724,16.1543870489597,16.1543870489597,15.2756954716879,19.835758359278,15.1825444005545,16.4228177328561,19.710477230295,19.710477230295,13.6808214449962,16.9649668448592,11.747980750738,17.8206171716069,14.0509285588584,14.0509285588584,17.6095673525824,11.1749781597015,14.9397854919929,17.8988536385066,16.9373174877114,19.2686621270864,15.7806675338474,12.968152232712,18.0382058611615,17.0098206791988,15.5245580341656,18.4586896731346,19.8095866429318,10.8097872173079,14.6957136866834,14.405947329161,17.1598725929261],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData76f7c579","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[28.7757887903244,16.9543231164817,15.3202787669038,16.2143076294315,16.2105512056393,4.24258500390343,9.05456388162587,3.75424991092613,19.3434086482783,3.64155719716214,9.0921281195472,22.3297655630241,18.6935473322393,9.35507778499651,25.5790721432192,25.5828285670113,18.7010601798235,36.1083280325681,36.1534051180737,27.4460147679093,22.299714172687,21.169030611255,40.8301527392793,46.8817514684057,40.706190754139,22.3072270202713,27.1229623217858,16.8679253692626,46.8704821970293,36.1646743894501,25.8194832659157,2.12771840893252,16.1128841870439,25.9960351841459,4.24634142769556,27.0515902697353,8.02906018637355,19.362190767239,27.7014515857743,19.2344723583064,9.48279619392904,8.29576627561499,27.0478338459432,28.9448278609704,28.8997507754648,3.68663428266774,40.341817646302,19.9932699643173,27.044077422151,38.0353734379323,8.99070467715961,19.3584343434468,2.10893628997186],"y":[17,12.6271567194248,11.1371665679076,10.8789737497886,10.8558492530957,10.7824726124305,9.75763041074863,8.45950634847854,8.4529041187357,7.72804378865718,7.71538365125733,7.2514365944103,7.2514365944103,7.22259752759318,6.40776895796568,5.93516797450259,5.48429965016278,5.47893809644251,5.41076515046335,5.3052702916526,5.02636079314789,4.85872520720617,4.730714592626,4.53617270701866,4.51920891450032,4.3671043795666,4.29042412083262,4.25356872198731,3.98706655186526,3.87859519059068,3.75018536778684,3.63412089471872,3.55389266591751,3.53201531641517,3.33834285240138,3.26822043640691,2.5952217292898,2.57007375695717,2.39324704354358,2.27018652144167,2.26040185631978,2.16915664074442,1.95691930655678,1.95447804534207,1.86555292131129,1.75464342680511,1.724920693343,1.70176686972025,1.66807738437008,1.66370165200299,1.61509466794424,1.60487203713533,1.47352119395506],"text":["GO:0044877 (1304) <br> protein-containing complex binding <br> 1.466e-18","GO:0017111 (424) <br> nucleoside-triphosphatase activity <br> 2.360e-13","GO:0016462 (482) <br> pyrophosphatase activity <br> 7.292e-12","GO:0016818 (493) <br> hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides <br> 1.321e-11","GO:0016817 (494) <br> hydrolase activity, acting on acid anhydrides <br> 1.394e-11","GO:0003924 (300) <br> GTPase activity <br> 1.650e-11","GO:0005515 (14767) <br> protein binding <br> 1.747e-10","GO:0003779 (449) <br> actin binding <br> 3.471e-09","GO:0019899 (2048) <br> enzyme binding <br> 3.524e-09","GO:0003723 (1951) <br> RNA binding <br> 1.870e-08","GO:0005525 (381) <br> GTP binding <br> 1.926e-08","GO:0032561 (402) <br> guanyl ribonucleotide binding <br> 5.605e-08","GO:0019001 (402) <br> guanyl nucleotide binding <br> 5.605e-08","GO:0008092 (1001) <br> cytoskeletal protein binding <br> 5.990e-08","GO:0035254 (48) <br> glutamate receptor binding <br> 3.910e-07","GO:0035255 (27) <br> ionotropic glutamate receptor binding <br> 1.161e-06","GO:0019003 (73) <br> GDP binding <br> 3.279e-06","GO:0050839 (544) <br> cell adhesion molecule binding <br> 3.319e-06","GO:0051015 (213) <br> actin filament binding <br> 3.884e-06","GO:0043168 (2413) <br> anion binding <br> 4.951e-06","GO:0032553 (1934) <br> ribonucleotide binding <br> 9.411e-06","GO:0031683 (20) <br> G-protein beta/gamma-subunit complex binding <br> 1.384e-05","GO:0097367 (2282) <br> carbohydrate derivative binding <br> 1.859e-05","GO:1901363 (6210) <br> heterocyclic compound binding <br> 2.910e-05","GO:0097159 (6291) <br> organic cyclic compound binding <br> 3.025e-05","GO:0032555 (1917) <br> purine ribonucleotide binding <br> 4.294e-05","GO:0042802 (2063) <br> identical protein binding <br> 5.124e-05","GO:0017076 (1932) <br> purine nucleotide binding <br> 5.577e-05","GO:1901265 (2177) <br> nucleoside phosphate binding <br> 1.030e-04","GO:0051020 (295) <br> GTPase binding <br> 1.323e-04","GO:0035639 (1846) <br> purine ribonucleoside triphosphate binding <br> 1.778e-04","GO:0000166 (2176) <br> nucleotide binding <br> 2.322e-04","GO:0016787 (2357) <br> hydrolase activity <br> 2.793e-04","GO:0036094 (2516) <br> small molecule binding <br> 2.938e-04","GO:0003925 (32) <br> G protein activity <br> 4.588e-04","GO:0042610 (4) <br> CD8 receptor binding <br> 5.392e-04","GO:0005102 (1551) <br> signaling receptor binding <br> 2.540e-03","GO:0019904 (706) <br> protein domain specific binding <br> 2.691e-03","GO:0043621 (56) <br> protein self-association <br> 4.043e-03","GO:0019829 (54) <br> ATPase-coupled cation transmembrane transporter activity <br> 5.368e-03","GO:0008135 (99) <br> translation factor activity, RNA binding <br> 5.490e-03","GO:0005178 (146) <br> integrin binding <br> 6.774e-03","GO:0042609 (9) <br> CD4 receptor binding <br> 1.104e-02","GO:0045296 (333) <br> cadherin binding <br> 1.111e-02","GO:0045182 (154) <br> translation regulator activity <br> 1.363e-02","GO:0003743 (58) <br> translation initiation factor activity <br> 1.759e-02","GO:0090079 (123) <br> translation regulator activity, nucleic acid binding <br> 1.884e-02","GO:0030276 (70) <br> clathrin binding <br> 1.987e-02","GO:0042608 (11) <br> T cell receptor binding <br> 2.147e-02","GO:0060090 (341) <br> molecular adaptor activity <br> 2.169e-02","GO:0005488 (17070) <br> binding <br> 2.426e-02","GO:0019903 (153) <br> protein phosphatase binding <br> 2.484e-02","GO:0000146 (28) <br> microfilament motor activity <br> 3.361e-02"],"key":["GO:0044877","GO:0017111","GO:0016462","GO:0016818","GO:0016817","GO:0003924","GO:0005515","GO:0003779","GO:0019899","GO:0003723","GO:0005525","GO:0032561","GO:0019001","GO:0008092","GO:0035254","GO:0035255","GO:0019003","GO:0050839","GO:0051015","GO:0043168","GO:0032553","GO:0031683","GO:0097367","GO:1901363","GO:0097159","GO:0032555","GO:0042802","GO:0017076","GO:1901265","GO:0051020","GO:0035639","GO:0000166","GO:0016787","GO:0036094","GO:0003925","GO:0042610","GO:0005102","GO:0019904","GO:0043621","GO:0019829","GO:0008135","GO:0005178","GO:0042609","GO:0045296","GO:0045182","GO:0003743","GO:0090079","GO:0030276","GO:0042608","GO:0060090","GO:0005488","GO:0019903","GO:0000146"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[19.7686971273735,18.3167342125249,18.4896805188233,18.5199090563365,18.5226205472543,17.8394624222953,22.525272898967,18.394266179927,20.3162578263421,20.258276019327,18.1708988395135,18.2442535356626,18.2442535356626,19.4390569617437,14.976936429373,13.9127623976552,15.692660304218,18.6510623608111,17.3504409902673,20.5107536570554,20.2477963305085,13.3106655321065,20.4448133217894,21.5904209757188,21.6047679255868,20.2372173548861,20.324960558726,20.2465569336091,20.3889635367376,17.8158620733969,20.1919193878134,20.38841785752,20.4830462803831,20.5599632179123,14.2382740983495,9.00890082870231,19.9812897468844,18.9928919364036,15.2452846314144,15.1825444005545,16.1868772799724,16.7898725428429,11.4827422948031,17.9851259338563,16.8705055778228,15.3054992444299,16.5273136649716,15.6229705518591,11.980531463383,18.0180536596155,22.6771653543307,16.8606852085784,13.9833131530912],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData76f7c579","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[193.957760321422,198.094889825427,190.583425812253,196.024446268019,191.962468980254,193.935214656551,196.783483652041,199.147020852785,194.014124483602,194.138125640398,194.01036687279,190.835185736656,190.985490169136,190.906580342084,199.143263241973,195.904202722035,194.14188325121,194.134368029586,196.170993089686,193.938972267363,197.542521036064,193.597029683471,193.942729878175,193.950245099799,196.715846657425,193.394118699624,191.301129477343,191.755800385595,190.579668201441,199.624237425908,191.079430439436,193.946487488987,194.679221597325,196.61063355469,196.790998873665,195.907960332847,191.1620978773,199.658055923216,190.831428125844,194.619099824333,193.95400271061,191.08694566106,191.128279379992,193.961517932234,199.135748020349,194.713040094633,191.582950288243,194.930981521729,194.570250883777,196.013173435583,190.575910590629,194.773161867625,198.489438960686,198.267739922779,193.379088256376,193.371573034752,194.382370343178,194.765646646001,193.920184213303,196.621906387126,192.022590753246],"y":[7.51410504780635,6.99721001392364,6.6724591719785,6.62038593315955,5.97854751496356,5.49262687573604,5.38270126321733,4.70261871667664,4.69901913797233,4.57628849941618,4.46170040496603,4.46170040496603,4.26797940575954,4.07658302456303,3.98625289822816,3.93352450030046,3.8151085297461,3.8151085297461,3.81351282855483,3.80599959935816,3.77546895081533,3.67872565150262,3.55451413699175,3.55451413699175,3.43945367063326,3.41704348146882,3.20969352242191,3.13205721807499,3.12868116722498,3.10530000368099,3.07656703145163,3.03480845659991,3.0120992973753,2.97666872650178,2.89773694138767,2.80562545359079,2.6978224545319,2.6978224545319,2.69764168071368,2.69764168071368,2.69764168071368,2.6492781783209,2.60941946727887,2.4773907352363,2.45013525392308,2.39709315679458,2.28599838776099,2.27910841030397,2.27132639116958,2.06362276328777,2.04384241466576,1.97159893266253,1.74960351173469,1.74960351173469,1.74713445235578,1.74347989395877,1.68951968324924,1.55201438249962,1.52664459519027,1.37905010656888,1.36655387161109],"text":["REAC:R-HSA-202040 (28) <br> G-protein activation <br> 3.061e-08","REAC:R-HSA-392518 (32) <br> Signal amplification <br> 1.006e-07","REAC:R-HSA-392170 (22) <br> ADP signalling through P2Y purinoceptor 12 <br> 2.126e-07","REAC:R-HSA-6798695 (476) <br> Neutrophil degranulation <br> 2.397e-07","REAC:R-HSA-6814122 (42) <br> Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding <br> 1.051e-06","REAC:R-HSA-418597 (48) <br> G alpha (z) signalling events <br> 3.216e-06","REAC:R-HSA-392851 (19) <br> Prostacyclin signalling through prostacyclin receptor <br> 4.143e-06","REAC:R-HSA-428930 (24) <br> Thromboxane signalling through TP receptor <br> 1.983e-05","REAC:R-HSA-977443 (60) <br> GABA receptor activation <br> 2.000e-05","REAC:R-HSA-381676 (42) <br> Glucagon-like Peptide-1 (GLP1) regulates insulin secretion <br> 2.653e-05","REAC:R-HSA-977444 (43) <br> GABA B receptor activation <br> 3.454e-05","REAC:R-HSA-991365 (43) <br> Activation of GABAB receptors <br> 3.454e-05","REAC:R-HSA-400042 (28) <br> Adrenaline,noradrenaline inhibits insulin secretion <br> 5.395e-05","REAC:R-HSA-451326 (30) <br> Activation of kainate receptors upon glutamate binding <br> 8.383e-05","REAC:R-HSA-456926 (31) <br> Thrombin signalling through proteinase activated receptors (PARs) <br> 1.032e-04","REAC:R-HSA-164940 (9) <br> Nef mediated downregulation of MHC class I complex cell surface expression <br> 1.165e-04","REAC:R-HSA-420092 (33) <br> Glucagon-type ligand receptors <br> 1.531e-04","REAC:R-HSA-163359 (33) <br> Glucagon signaling in metabolic regulation <br> 1.531e-04","REAC:R-HSA-111885 (90) <br> Opioid Signalling <br> 1.536e-04","REAC:R-HSA-8964315 (18) <br> G beta:gamma signalling through BTK <br> 1.563e-04","REAC:R-HSA-422356 (78) <br> Regulation of insulin secretion <br> 1.677e-04","REAC:R-HSA-9009391 (73) <br> Extra-nuclear estrogen signaling <br> 2.095e-04","REAC:R-HSA-8964616 (20) <br> G beta:gamma signalling through CDC42 <br> 2.789e-04","REAC:R-HSA-418217 (20) <br> G beta:gamma signalling through PLC beta <br> 2.789e-04","REAC:R-HSA-500657 (21) <br> Presynaptic function of Kainate receptors <br> 3.635e-04","REAC:R-HSA-1236974 (83) <br> ER-Phagosome pathway <br> 3.828e-04","REAC:R-HSA-3858494 (143) <br> Beta-catenin independent WNT signaling <br> 6.170e-04","REAC:R-HSA-390466 (95) <br> Chaperonin-mediated protein folding <br> 7.378e-04","REAC:R-HSA-418592 (24) <br> ADP signalling through P2Y purinoceptor 1 <br> 7.436e-04","REAC:R-HSA-432040 (43) <br> Vasopressin regulates renal water homeostasis via Aquaporins <br> 7.847e-04","REAC:R-HSA-983170 (25) <br> Antigen Presentation: Folding, assembly and peptide loading of class I MHC <br> 8.384e-04","REAC:R-HSA-392451 (25) <br> G beta:gamma signalling through PI3Kgamma <br> 9.230e-04","REAC:R-HSA-168249 (1090) <br> Innate Immune System <br> 9.725e-04","REAC:R-HSA-76002 (258) <br> Platelet activation, signaling and aggregation <br> 1.055e-03","REAC:R-HSA-391251 (102) <br> Protein folding <br> 1.266e-03","REAC:R-HSA-164938 (20) <br> Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters <br> 1.564e-03","REAC:R-HSA-168316 (2) <br> Assembly of Viral Components at the Budding Site <br> 2.005e-03","REAC:R-HSA-168268 (2) <br> Virus Assembly and Release <br> 2.005e-03","REAC:R-HSA-1296041 (29) <br> Activation of G protein gated Potassium channels <br> 2.006e-03","REAC:R-HSA-997272 (29) <br> Inhibition  of voltage gated Ca2+ channels via Gbeta/gamma subunits <br> 2.006e-03","REAC:R-HSA-1296059 (29) <br> G protein gated Potassium channels <br> 2.006e-03","REAC:R-HSA-1236975 (99) <br> Antigen processing-Cross presentation <br> 2.242e-03","REAC:R-HSA-445717 (52) <br> Aquaporin-mediated transport <br> 2.458e-03","REAC:R-HSA-397795 (32) <br> G-protein beta:gamma signalling <br> 3.331e-03","REAC:R-HSA-164952 (26) <br> The role of Nef in HIV-1 replication and disease pathogenesis <br> 3.547e-03","REAC:R-HSA-163685 (107) <br> Integration of energy metabolism <br> 4.008e-03","REAC:R-HSA-4086398 (59) <br> Ca2+ pathway <br> 5.176e-03","REAC:R-HSA-1296065 (35) <br> Inwardly rectifying K+ channels <br> 5.259e-03","REAC:R-HSA-168256 (2038) <br> Immune System <br> 5.354e-03","REAC:R-HSA-112314 (196) <br> Neurotransmitter receptors and postsynaptic signal transmission <br> 8.637e-03","REAC:R-HSA-9660821 (133) <br> ADORA2B mediated anti-inflammatory cytokines production <br> 9.040e-03","REAC:R-HSA-877300 (87) <br> Interferon gamma signaling <br> 1.068e-02","REAC:R-HSA-9670439 (17) <br> Signaling by phosphorylated juxtamembrane, extracellular and kinase domain KIT mutants <br> 1.780e-02","REAC:R-HSA-9669938 (17) <br> Signaling by KIT in disease <br> 1.780e-02","REAC:R-HSA-3928663 (28) <br> EPHA-mediated growth cone collapse <br> 1.790e-02","REAC:R-HSA-2682334 (90) <br> EPH-Ephrin signaling <br> 1.805e-02","REAC:R-HSA-109582 (673) <br> Hemostasis <br> 2.044e-02","REAC:R-HSA-913531 (193) <br> Interferon Signaling <br> 2.805e-02","REAC:R-HSA-416482 (80) <br> G alpha (12/13) signalling events <br> 2.974e-02","REAC:R-HSA-418346 (85) <br> Platelet homeostasis <br> 4.178e-02","REAC:R-HSA-1280215 (696) <br> Cytokine Signaling in Immune system <br> 4.300e-02"],"key":["REAC:R-HSA-202040","REAC:R-HSA-392518","REAC:R-HSA-392170","REAC:R-HSA-6798695","REAC:R-HSA-6814122","REAC:R-HSA-418597","REAC:R-HSA-392851","REAC:R-HSA-428930","REAC:R-HSA-977443","REAC:R-HSA-381676","REAC:R-HSA-977444","REAC:R-HSA-991365","REAC:R-HSA-400042","REAC:R-HSA-451326","REAC:R-HSA-456926","REAC:R-HSA-164940","REAC:R-HSA-420092","REAC:R-HSA-163359","REAC:R-HSA-111885","REAC:R-HSA-8964315","REAC:R-HSA-422356","REAC:R-HSA-9009391","REAC:R-HSA-8964616","REAC:R-HSA-418217","REAC:R-HSA-500657","REAC:R-HSA-1236974","REAC:R-HSA-3858494","REAC:R-HSA-390466","REAC:R-HSA-418592","REAC:R-HSA-432040","REAC:R-HSA-983170","REAC:R-HSA-392451","REAC:R-HSA-168249","REAC:R-HSA-76002","REAC:R-HSA-391251","REAC:R-HSA-164938","REAC:R-HSA-168316","REAC:R-HSA-168268","REAC:R-HSA-1296041","REAC:R-HSA-997272","REAC:R-HSA-1296059","REAC:R-HSA-1236975","REAC:R-HSA-445717","REAC:R-HSA-397795","REAC:R-HSA-164952","REAC:R-HSA-163685","REAC:R-HSA-4086398","REAC:R-HSA-1296065","REAC:R-HSA-168256","REAC:R-HSA-112314","REAC:R-HSA-9660821","REAC:R-HSA-877300","REAC:R-HSA-9670439","REAC:R-HSA-9669938","REAC:R-HSA-3928663","REAC:R-HSA-2682334","REAC:R-HSA-109582","REAC:R-HSA-913531","REAC:R-HSA-416482","REAC:R-HSA-418346","REAC:R-HSA-1280215"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,102,204,1)","opacity":0.8,"size":[13.9833131530912,14.2382740983495,13.5059251363754,18.4728732854611,14.73917293128,14.976936429373,13.2039079443259,13.6808214449962,15.3633750239743,14.73917293128,14.781443852525,14.781443852525,13.9833131530912,14.1158314099108,14.1782205386673,11.4827422948031,14.2961523059962,14.2961523059962,16.0344085591459,13.0900518224006,15.801858092204,15.692660304218,13.3106655321065,13.3106655321065,13.4111152730112,15.9033751977876,16.7583551431596,16.121131665261,13.6808214449962,14.781443852525,13.7618192670139,13.7618192670139,19.5459903341358,17.6262397302166,16.2342495236679,13.3106655321065,3.77952755905512,3.77952755905512,14.0509285588584,14.0509285588584,14.0509285588584,16.1868772799724,15.1170689685676,14.2382740983495,13.83902557929,16.309816295931,15.3347186146137,14.405947329161,20.3104179536764,17.2289935746564,16.6476991969387,15.9797164726717,12.968152232712,12.968152232712,13.9833131530912,16.0344085591459,18.9306948804635,17.206351575782,15.8433279821367,15.9420548318458,18.9743833960756],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,102,204,1)"}},"hoveron":"points","set":"SharedData76f7c579","name":"REAC","legendgroup":"REAC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"17247d9d4658":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"17241b9058e0":{"x":{},"xend":{},"y":{},"yend":{}},"17241748f06":{"x":{},"xend":{},"y":{},"yend":{}},"17244d075701":{"x":{},"xend":{},"y":{},"yend":{}},"17244f7f4f54":{"x":{},"xend":{},"y":{},"yend":{}},"17245f115d38":{"x":{},"y":{}},"172437d6176c":{"yintercept":{}}},"cur_data":"17247d9d4658","visdat":{"17247d9d4658":["function (y) ","x"],"17241b9058e0":["function (y) ","x"],"17241748f06":["function (y) ","x"],"17244d075701":["function (y) ","x"],"17244f7f4f54":["function (y) ","x"],"17245f115d38":["function (y) ","x"],"172437d6176c":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData76f7c579"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results-1" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-2b7856457827ab1876f7" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2b7856457827ab1876f7">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449"],["GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:MF","GO:BP","GO:MF","GO:MF","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:MF","GO:MF","GO:BP","GO:CC","GO:MF","GO:MF","REAC","GO:BP","GO:BP","GO:CC","GO:MF","GO:MF","GO:MF","GO:CC","GO:CC","GO:BP","GO:CC","REAC","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","REAC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","REAC","REAC","GO:CC","REAC","GO:MF","GO:MF","GO:CC","GO:CC","REAC","REAC","GO:BP","GO:CC","GO:MF","GO:MF","GO:BP","GO:BP","REAC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","REAC","GO:MF","REAC","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","REAC","REAC","REAC","REAC","GO:CC","REAC","GO:MF","GO:CC","GO:CC","REAC","GO:MF","REAC","REAC","GO:MF","GO:MF","GO:CC","GO:CC","GO:BP","REAC","GO:CC","REAC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:CC","REAC","REAC","GO:CC","GO:CC","REAC","GO:BP","REAC","REAC","GO:BP","GO:BP","REAC","GO:CC","REAC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","REAC","REAC","REAC","REAC","GO:CC","GO:CC","GO:CC","GO:CC","REAC","GO:BP","GO:BP","REAC","GO:CC","GO:CC","GO:MF","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","REAC","GO:BP","GO:CC","GO:CC","REAC","GO:CC","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","REAC","REAC","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:CC","GO:CC","GO:CC","REAC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","REAC","REAC","REAC","GO:BP","REAC","GO:CC","GO:CC","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:MF","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:CC","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:CC","REAC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC"],[2743,4056,2131,2719,1453,278,7129,385,1322,3497,561,785,1099,2843,12268,12281,3780,947,939,1030,550,13259,927,3982,3561,1101,1027,474,4087,1080,11579,16046,10980,1038,13142,16054,9160,850,2779,241,470,14485,15738,23108,3551,1247,2901,5762,1125,1217,3547,3785,938,3784,598,3582,817,2437,3658,984,2765,1879,916,1118,1098,983,15694,3627,9231,2029,2552,837,7017,284,468,4618,3542,2128,438,1889,908,4320,15417,818,5413,4445,1959,1026,2130,1133,2542,2009,515,19397,2127,7019,1570,10009,10,1458,19079,14527,6278,1332,3132,24935,377,15359,6279,3628,14885,2879,2132,2093,5363,418,2897,410,3654,3860,13203,15587,1134,2895,902,5659,4447,5665,9081,19167,14486,9093,14889,7698,1660,6775,1129,19740,7619,11868,7705,5405,4509,1334,1263,193,3800,3391,184,19737,24936,5104,190,15361,15416,2450,67,409,3612,19815,10338,2289,923,417,956,11949,10305,20,1528,922,77,19502,2061,5407,6689,14941,12553,117,3959,9198,11574,18664,14309,2566,2134,2720,1446,96,11946,2288,1412,1426,1408,11657,1084,1401,1149,9096,3387,15644,13160,957,955,1497,903,2496,1862,6342,2090,3624,812,35,904,906,3758,6389,1261,1150,9109,1642,3583,758,15586,599,6670,6495,7747,9696,201,1128,15822,8304,15456,413,917,322,9,12490,20868,532,2416,142,795,3801,905,8301,1100,1614,2946,20932,1662,561,1427,1455,2842,871,29092,22820,164,2425,76,1084,907,1078,2056,2042,411,144,12744,12353,155,3071,3552,1606,1121,24899,4623,10495,1474,17436,7609,1864,909,15318,2286,15747,1648,2022,1109,2306,6843,3123,2418,1100,3357,427,3072,438,2949,19430,2714,1628,20429,12675,16110,276,1073,1167,1071,4589,9881,1993,5837,7618,489,861,2041,872,16072,433,13161,2921,1414,1480,3638,1677,8667,3575,791,7622,1455,4442,2543,1057,8,24824,1862,306,12233,1125,7629,4001,6669,7174,16045,15767,1481,11500,861,2423,7607,7162,7837,1440,1447,4086,1071,12669,6791,3663,2898,2916,5773,8306,1075,24938,450,15748,2114,2055,754,18660,752,64,408,14646,10208,3990,446,1135,26125,4791,13159,19492,1021,2592,8669,6668,9594,1058,432,1085,19531,9546,1964,15327,7021,11494,815,1626,1862,7699,4622,8303,3444,634,29349,20427,889,1123,1150,1262,898,29136,29126,1240,17166,13778,3102,1476,1220,22821,1159,885,30,296,13553,886,8060,5199,9098,1402,26123,4080,8761,531,1617,2447,15765,3644,393,1059,17822,16061,606,22049,358,26639,2565],["extracellular exosome","extracellular vesicle","extracellular organelle","extracellular membrane-bounded organelle","vesicle","extracellular space","protein-containing complex binding","cytoplasm","cytoplasmic vesicle","intracellular vesicle","cell activation","cytoplasmic side of plasma membrane","myeloid leukocyte mediated immunity","exocytosis","leukocyte degranulation","neutrophil degranulation","secretory vesicle","neutrophil activation involved in immune response","myeloid cell activation involved in immune response","leukocyte activation involved in immune response","focal adhesion","leukocyte activation","cell activation involved in immune response","nucleoside-triphosphatase activity","cytoplasmic side of membrane","neutrophil mediated immunity","cell-substrate junction","heterotrimeric G-protein complex","GTPase complex","secretory granule","neutrophil activation","cellular localization","granulocyte activation","immune system process","regulated exocytosis","establishment of localization in cell","secretion by cell","membrane","transport","extracellular region","cytosol","secretion","establishment of localization","export from cell","side of membrane","extrinsic component of cytoplasmic side of plasma membrane","immune response","vesicle-mediated transport","secretory granule membrane","organelle membrane","pyrophosphatase activity","hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides","myeloid leukocyte activation","hydrolase activity, acting on acid anhydrides","GTPase activity","bounding membrane of organelle","endomembrane system","membrane raft","membrane microdomain","extrinsic component of membrane","anchoring junction","protein binding","immune effector process","cytoplasmic vesicle membrane","leukocyte mediated immunity","extrinsic component of plasma membrane","localization","membrane protein complex","macromolecule localization","melanosome","pigment granule","actin cytoskeleton","antigen processing and presentation","intracellular anatomical structure","actin binding","enzyme binding","protein localization","membrane-bounded organelle","RNA binding","GTP binding","G-protein activation","response to organic substance","T cell receptor signaling pathway","vesicle membrane","guanyl ribonucleotide binding","guanyl nucleotide binding","cytoskeletal protein binding","cell junction","intracellular organelle","antigen processing and presentation of exogenous peptide antigen","perinuclear region of cytoplasm","Signal amplification","plasma membrane","cellular response to organic substance","organelle","antigen processing and presentation of exogenous antigen","protein-containing complex","cellular protein localization","ADP signalling through P2Y purinoceptor 12","Neutrophil degranulation","cellular macromolecule localization","antigen processing and presentation of peptide antigen","glutamate receptor binding","regulation of immune system process","cell periphery","response to nitrogen compound","Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding","regulation of immune response","ionotropic glutamate receptor binding","plasma membrane protein complex","positive regulation of biological process","movement of cell or subcellular component","intracellular membrane-bounded organelle","cell projection","protein transport","vacuolar membrane","response to stress","lysosomal membrane","lytic vacuole membrane","plasma membrane bounded cell projection","establishment of protein localization","positive regulation of transport","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","tertiary granule","G alpha (z) signalling events","viral process","GDP binding","cellular component organization","cell adhesion molecule binding","cellular response to chemical stimulus","intracellular transport","actin filament binding","positive regulation of cellular process","actin filament-based process","Prostacyclin signalling through prostacyclin receptor","anion binding","antigen processing and presentation of peptide antigen via MHC class I","nitrogen compound transport","regulation of cell morphogenesis","antigen processing and presentation of exogenous peptide antigen via MHC class I","actin cytoskeleton organization","ribonucleotide binding","response to organonitrogen compound","positive regulation of immune system process","cell trailing edge","uropod","ficolin-1-rich granule","contractile actin filament bundle","stress fiber","organic substance transport","cellular response to nitrogen compound","G-protein beta/gamma-subunit complex binding","immunological synapse","positive regulation of immune response","antigen receptor-mediated signaling pathway","protein folding","lytic vacuole","lysosome","regulation of cell shape","cellular component organization or biogenesis","carbohydrate derivative binding","Thromboxane signalling through TP receptor","GABA receptor activation","vacuole","Glucagon-like Peptide-1 (GLP1) regulates insulin secretion","heterocyclic compound binding","organic cyclic compound binding","Golgi membrane","actin filament bundle","GABA B receptor activation","Activation of GABAB receptors","cellular response to organonitrogen compound","actomyosin","purine ribonucleotide binding","identical protein binding","positive regulation of response to stimulus","protein-containing complex subunit organization","Adrenaline,noradrenaline inhibits insulin secretion","purine nucleotide binding","protein-containing complex disassembly","T cell activation","regulation of biological quality","lymphocyte activation","recycling endosome membrane","organelle lumen","intracellular organelle lumen","membrane-enclosed lumen","Activation of kainate receptors upon glutamate binding","nucleoside phosphate binding","Thrombin signalling through proteinase activated receptors (PARs)","immune response-regulating cell surface receptor signaling pathway","Nef mediated downregulation of MHC class I complex cell surface expression","immune response-regulating signaling pathway","response to chemical","immune response-activating cell surface receptor signaling pathway","immune response-activating signal transduction","cortical cytoskeleton","GTPase binding","myosin II filament","regulation of cellular component organization","regulation of innate immune response","Glucagon-type ligand receptors","Glucagon signaling in metabolic regulation","Opioid Signalling","G beta:gamma signalling through BTK","phagocytic vesicle","Regulation of insulin secretion","purine ribonucleoside triphosphate binding","MHC class I peptide loading complex","Golgi apparatus subcompartment","Extra-nuclear estrogen signaling","nucleotide binding","G beta:gamma signalling through CDC42","G beta:gamma signalling through PLC beta","hydrolase activity","small molecule binding","cell leading edge","cortical actin cytoskeleton","regulation of localization","Presynaptic function of Kainate receptors","plasma membrane region","ER-Phagosome pathway","regulation of transport","G protein activity","CD8 receptor binding","cytokine-mediated signaling pathway","platelet activation","response to cytokine","Beta-catenin independent WNT signaling","phagocytic vesicle membrane","regulation of hydrolase activity","regulation of defense response","response to stimulus","endosome","activation of immune response","Chaperonin-mediated protein folding","ADP signalling through P2Y purinoceptor 1","cellular protein complex disassembly","viral translational termination-reinitiation","brush border","Vasopressin regulates renal water homeostasis via Aquaporins","Antigen Presentation: Folding, assembly and peptide loading of class I MHC","endosome membrane","ficolin-1-rich granule membrane","G beta:gamma signalling through PI3Kgamma","regulation of cell projection organization","Innate Immune System","Platelet activation, signaling and aggregation","cytoskeleton organization","regulation of response to stress","Protein folding","cell cortex","Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters","organelle subcompartment","intracellular protein transport","myosin complex","protein localization to cell periphery","regulation of plasma membrane bounded cell projection organization","Assembly of Viral Components at the Budding Site","Virus Assembly and Release","Activation of G protein gated Potassium channels","Inhibition  of voltage gated Ca2+ channels via Gbeta/gamma subunits","G protein gated Potassium channels","endocytic vesicle","MHC class I protein complex","azurophil granule","primary lysosome","Antigen processing-Cross presentation","cellular protein metabolic process","response to peptide hormone","Aquaporin-mediated transport","integral component of lumenal side of endoplasmic reticulum membrane","lumenal side of endoplasmic reticulum membrane","signaling receptor binding","coated vesicle membrane","response to peptide","protein domain specific binding","intracellular signal transduction","regulation of response to biotic stimulus","regulation of vesicle-mediated transport","cellular component disassembly","specific granule membrane","G-protein beta:gamma signalling","regulation of defense response to virus by virus","The role of Nef in HIV-1 replication and disease pathogenesis","regulation of protein metabolic process","eukaryotic 48S preinitiation complex","nuclear outer membrane-endoplasmic reticulum membrane network","Integration of energy metabolism","cell body","protein self-association","dopamine receptor signaling pathway","translational termination","dendrite","dendritic tree","endoplasmic reticulum","leukocyte cell-cell adhesion","Golgi apparatus","actin filament organization","cellular response to cytokine stimulus","NADH metabolic process","proton-transporting V-type ATPase, V1 domain","protein localization to plasma membrane","positive regulation of molecular function","cellular response to stimulus","Ca2+ pathway","transport vesicle","Inwardly rectifying K+ channels","Immune System","ATPase-coupled cation transmembrane transporter activity","response to interferon-gamma","translation factor activity, RNA binding","cell migration","regulation of anatomical structure morphogenesis","eukaryotic translation initiation factor 3 complex","eukaryotic 43S preinitiation complex","specific granule","myosin II complex","localization within membrane","endoplasmic reticulum membrane","positive regulation of innate immune response","translation preinitiation complex","G-protein beta/gamma-subunit complex","cell division site","endoplasmic reticulum subcompartment","integrin binding","regulation of cellular protein metabolic process","lumenal side of membrane","cell surface","cellular component assembly","Neurotransmitter receptors and postsynaptic signal transmission","ribonucleoprotein complex","coated membrane","membrane coat","ADORA2B mediated anti-inflammatory cytokines production","organonitrogen compound metabolic process","azurophil granule membrane","cell morphogenesis","response to amino acid","Interferon gamma signaling","membrane to membrane docking","catalytic complex","CD4 receptor binding","cadherin binding","organelle localization","positive regulation of cellular component movement","cleavage furrow","positive regulation of locomotion","cytoplasmic translational initiation","plasma membrane raft","positive regulation of cell-cell adhesion","translation regulator activity","positive regulation of cell migration","organelle envelope","envelope","response to hormone","clathrin adaptor complex","regulation of cellular component biogenesis","protein metabolic process","cluster of actin-based cell projections","defense response","contractile ring","macroautophagy","positive regulation of defense response","coated vesicle","cellular response to oxygen-containing compound","translation initiation factor activity","positive regulation of protein metabolic process","Signaling by phosphorylated juxtamembrane, extracellular and kinase domain KIT mutants","Signaling by KIT in disease","EPHA-mediated growth cone collapse","protein-containing complex assembly","EPH-Ephrin signaling","organellar small ribosomal subunit","mitochondrial small ribosomal subunit","Golgi vesicle transport","translation regulator activity, nucleic acid binding","response to external stimulus","trans-Golgi network","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-independent","positive regulation of leukocyte cell-cell adhesion","clathrin binding","innate immune response","cellular response to organic cyclic compound","Hemostasis","cell pole","positive regulation of cellular protein metabolic process","T cell receptor binding","molecular adaptor activity","clathrin coat","endoplasmic reticulum lumen","integral component of endoplasmic reticulum membrane","cellular response to oxygen levels","cellular response to stress","somatodendritic compartment","regulation of peptidyl-tyrosine phosphorylation","antigen processing and presentation of exogenous peptide antigen via MHC class II","locomotion","cytoplasmic stress granule","proton-transporting two-sector ATPase complex, catalytic domain","binding","cell projection organization","protein phosphatase binding","positive regulation of cell projection organization","aging","endothelial cell development","positive regulation of cell motility","protein localization to membrane","stimulatory C-type lectin receptor signaling pathway","Interferon Signaling","antigen processing and presentation of peptide antigen via MHC class II","cell projection membrane","G alpha (12/13) signalling events","cellular response to lectin","response to lectin","intrinsic component of endoplasmic reticulum membrane","regulation of cellular localization","ATP metabolic process","adenylate cyclase-activating dopamine receptor signaling pathway","positive regulation of response to biotic stimulus","platelet alpha granule lumen","plasma membrane bounded cell projection organization","antigen processing and presentation of peptide or polysaccharide antigen via MHC class II","activation of innate immune response","microfilament motor activity","nuclear envelope","positive regulation of cell adhesion","innate immune response activating cell surface receptor signaling pathway","cortical actin cytoskeleton organization","response to organic cyclic compound","response to insulin","innate immune response-activating signal transduction","regulation of leukocyte cell-cell adhesion","response to endogenous stimulus","positive regulation of intracellular transport","formation of cytoplasmic translation initiation complex","Platelet homeostasis","synapse","regulation of cellular component movement","presynaptic endocytic zone membrane","Cytokine Signaling in Immune system","AP-type membrane coat adaptor complex","establishment of endothelial barrier","establishment of organelle localization","COP9 signalosome","defense response to other organism","response to acid chemical","regulation of protein tyrosine phosphatase activity","recycling endosome"],["1.5e-33","5.1e-32","5.6e-32","5.6e-32","2.1e-25","1.2e-18","1.5e-18","3.2e-18","9.3e-18","1.1e-17","2.1e-15","1.4e-14","3.3e-14","3.5e-14","6.6e-14","1.0e-13","1.1e-13","1.4e-13","1.5e-13","1.7e-13","1.9e-13","1.9e-13","2.0e-13","2.4e-13","2.7e-13","3.0e-13","3.1e-13","3.4e-13","3.4e-13","3.4e-13","3.4e-13","4.4e-13","5.2e-13","6.0e-13","6.9e-13","8.4e-13","9.3e-13","9.3e-13","9.5e-13","1.0e-12","1.1e-12","1.3e-12","1.3e-12","1.7e-12","2.1e-12","2.2e-12","2.6e-12","3.7e-12","3.9e-12","6.1e-12","7.3e-12","1.3e-11","1.4e-11","1.4e-11","1.7e-11","1.8e-11","2.1e-11","2.9e-11","2.9e-11","6.3e-11","1.1e-10","1.7e-10","1.8e-10","1.8e-10","2.1e-10","3.5e-10","4.8e-10","1.1e-09","1.5e-09","1.6e-09","1.6e-09","2.0e-09","2.1e-09","3.3e-09","3.5e-09","3.5e-09","6.5e-09","7.7e-09","1.9e-08","1.9e-08","3.1e-08","3.2e-08","3.5e-08","4.9e-08","5.6e-08","5.6e-08","6.0e-08","6.5e-08","6.7e-08","8.2e-08","8.6e-08","1.0e-07","1.0e-07","1.6e-07","1.6e-07","1.7e-07","1.8e-07","2.1e-07","2.1e-07","2.4e-07","2.8e-07","3.9e-07","3.9e-07","4.3e-07","4.9e-07","9.3e-07","1.1e-06","1.1e-06","1.2e-06","1.3e-06","1.4e-06","1.7e-06","1.7e-06","1.9e-06","2.0e-06","2.1e-06","2.1e-06","2.2e-06","2.2e-06","2.3e-06","2.6e-06","2.8e-06","3.1e-06","3.1e-06","3.2e-06","3.2e-06","3.3e-06","3.3e-06","3.3e-06","3.4e-06","3.5e-06","3.9e-06","3.9e-06","4.1e-06","4.1e-06","5.0e-06","5.2e-06","5.3e-06","5.7e-06","6.3e-06","7.4e-06","9.4e-06","9.4e-06","9.8e-06","9.8e-06","9.8e-06","1.0e-05","1.1e-05","1.1e-05","1.2e-05","1.4e-05","1.4e-05","1.4e-05","1.4e-05","1.5e-05","1.6e-05","1.7e-05","1.7e-05","1.7e-05","1.8e-05","1.9e-05","2.0e-05","2.0e-05","2.6e-05","2.7e-05","2.9e-05","3.0e-05","3.0e-05","3.2e-05","3.5e-05","3.5e-05","3.5e-05","3.6e-05","4.3e-05","5.1e-05","5.2e-05","5.3e-05","5.4e-05","5.6e-05","6.0e-05","6.0e-05","6.5e-05","7.4e-05","8.1e-05","8.3e-05","8.3e-05","8.3e-05","8.4e-05","1.0e-04","1.0e-04","1.1e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.3e-04","1.3e-04","1.3e-04","1.5e-04","1.5e-04","1.5e-04","1.5e-04","1.5e-04","1.6e-04","1.7e-04","1.7e-04","1.8e-04","2.0e-04","2.1e-04","2.1e-04","2.3e-04","2.8e-04","2.8e-04","2.8e-04","2.9e-04","2.9e-04","3.3e-04","3.4e-04","3.6e-04","3.8e-04","3.8e-04","4.4e-04","4.6e-04","5.4e-04","5.8e-04","5.8e-04","5.9e-04","6.2e-04","6.3e-04","6.6e-04","6.6e-04","6.9e-04","7.2e-04","7.2e-04","7.4e-04","7.4e-04","7.5e-04","7.5e-04","7.7e-04","7.8e-04","8.4e-04","8.6e-04","9.2e-04","9.2e-04","9.4e-04","9.7e-04","1.1e-03","1.1e-03","1.2e-03","1.3e-03","1.5e-03","1.6e-03","1.6e-03","1.8e-03","1.8e-03","1.9e-03","2.0e-03","2.0e-03","2.0e-03","2.0e-03","2.0e-03","2.0e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.3e-03","2.4e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.6e-03","2.6e-03","2.7e-03","2.7e-03","2.9e-03","3.0e-03","3.0e-03","3.0e-03","3.3e-03","3.5e-03","3.5e-03","3.7e-03","3.8e-03","3.8e-03","4.0e-03","4.0e-03","4.0e-03","4.1e-03","4.1e-03","4.1e-03","4.3e-03","4.4e-03","4.4e-03","4.4e-03","4.6e-03","4.6e-03","5.0e-03","5.0e-03","5.0e-03","5.1e-03","5.2e-03","5.2e-03","5.2e-03","5.3e-03","5.4e-03","5.4e-03","5.5e-03","5.5e-03","5.6e-03","5.6e-03","5.6e-03","5.6e-03","5.7e-03","5.7e-03","5.9e-03","6.2e-03","6.7e-03","6.7e-03","6.8e-03","6.8e-03","6.8e-03","6.8e-03","7.1e-03","7.7e-03","8.3e-03","8.6e-03","8.6e-03","8.9e-03","9.0e-03","9.0e-03","9.0e-03","9.7e-03","9.8e-03","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.1e-02","2.1e-02","2.1e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.5e-02","2.5e-02","2.6e-02","2.7e-02","2.7e-02","2.7e-02","2.8e-02","2.8e-02","2.8e-02","2.9e-02","3.0e-02","3.0e-02","3.0e-02","3.0e-02","3.0e-02","3.1e-02","3.1e-02","3.2e-02","3.2e-02","3.2e-02","3.3e-02","3.4e-02","3.4e-02","3.4e-02","3.4e-02","3.5e-02","3.5e-02","3.5e-02","3.5e-02","3.7e-02","3.8e-02","4.0e-02","4.0e-02","4.1e-02","4.2e-02","4.2e-02","4.3e-02","4.3e-02","4.3e-02","4.4e-02","4.4e-02","4.5e-02","4.5e-02","4.5e-02","4.6e-02","4.8e-02","4.9e-02"],[2178,2263,2265,2265,4058,3595,1304,11951,2433,2438,1507,169,558,906,538,483,1027,488,552,723,420,1341,727,424,195,500,427,36,36,857,502,3519,509,3438,790,2778,1351,9829,5214,4564,5361,1492,5364,1413,622,103,2544,2193,308,3644,482,493,671,494,300,2165,4651,339,339,315,838,14767,1135,789,888,178,6938,1326,3148,109,109,517,236,14808,449,2048,2670,13201,1951,381,28,3457,202,1183,402,402,1001,2107,13117,175,745,32,5680,2821,14027,183,5541,1842,22,476,1856,193,48,1734,6178,1178,42,1159,27,696,6474,2319,12046,2337,1754,450,4321,395,395,2234,1866,951,75,163,48,1011,73,6661,544,3443,1723,213,5863,810,19,2413,99,2216,312,80,712,1934,1092,1079,13,13,184,70,70,2657,728,20,44,757,321,230,723,723,157,6880,2282,24,60,818,42,6210,6291,725,78,43,43,671,79,1917,2063,2368,2017,28,1932,337,495,4116,780,85,5579,5579,5579,30,2177,31,510,9,513,4829,475,475,121,295,3,2439,316,33,33,90,18,139,78,1846,9,845,73,2176,20,20,2357,2516,426,92,2906,21,1240,83,1839,32,4,824,163,1242,143,75,1252,889,9487,993,566,95,24,228,5,109,43,25,520,60,25,660,1090,258,1473,1626,102,326,20,1764,995,57,339,642,2,2,29,29,29,335,8,154,154,99,5280,461,52,25,25,1551,177,551,706,2886,426,544,577,90,32,29,26,2759,15,1154,107,595,56,41,104,641,643,1995,377,1593,445,1148,43,8,287,1774,7873,59,416,35,2038,54,198,99,1660,1048,17,17,158,21,775,1132,217,18,12,72,1137,146,2591,33,896,3104,196,694,97,97,133,6904,57,1045,115,87,5,1377,9,333,657,605,54,607,35,115,286,154,568,1244,1244,958,24,991,5912,164,1956,10,337,364,296,1237,58,1577,17,17,28,1724,90,29,29,415,123,3114,255,8,243,70,1003,624,673,8,1488,11,341,47,313,161,243,2142,875,270,97,2039,77,17,17070,1636,153,359,333,70,592,698,112,193,101,346,80,113,113,169,851,317,10,253,66,1596,104,144,28,471,442,115,40,1004,293,116,341,1729,227,16,85,1348,1149,7,696,41,51,443,35,1222,136,6,187],[110,110,110,110,130,117,67,230,86,86,66,21,38,49,37,35,49,35,37,42,33,59,42,30,21,35,33,10,10,44,35,102,35,100,44,88,58,178,127,115,133,61,131,59,33,14,81,75,26,96,30,30,38,30,24,69,112,25,25,23,42,252,48,38,42,17,148,34,88,15,15,31,21,248,29,68,78,231,52,23,8,90,19,43,23,23,41,61,228,17,32,8,116,68,237,17,96,59,7,34,59,17,9,60,121,46,8,44,6,19,132,68,212,61,57,17,94,16,16,59,59,39,11,14,8,36,10,140,26,81,54,17,122,36,6,68,12,63,17,11,33,58,42,39,5,5,13,10,10,69,30,5,7,29,20,16,20,20,12,141,64,6,8,21,9,113,114,28,10,7,7,28,10,56,58,57,61,6,56,21,24,92,33,9,100,100,100,6,60,6,24,3,24,109,23,23,11,17,3,63,19,6,6,11,5,11,11,53,4,29,8,59,5,5,52,65,20,9,70,5,36,11,51,6,3,31,12,40,13,8,39,34,156,28,24,8,5,15,3,8,6,6,19,6,5,26,43,14,46,47,8,16,3,44,35,6,17,25,2,2,5,5,5,15,3,8,8,11,114,20,6,4,4,44,8,23,23,64,21,22,25,8,5,5,3,70,3,35,11,22,7,5,8,23,23,51,18,40,21,36,7,2,15,50,124,6,17,5,64,3,9,6,45,29,3,3,10,4,27,34,12,3,3,7,34,10,66,4,25,69,10,17,4,4,8,112,5,32,10,7,3,30,3,15,24,23,6,23,4,8,11,8,22,28,28,29,2,31,121,8,53,3,15,15,11,34,4,45,5,5,5,48,8,4,4,18,6,74,11,3,10,4,33,22,21,3,43,3,14,3,12,7,12,56,25,15,5,50,7,2,259,41,10,16,14,7,22,24,9,10,5,14,6,9,9,7,29,16,3,12,2,40,5,10,4,18,13,9,4,29,14,9,14,42,13,3,6,32,33,2,31,2,6,18,4,37,10,3,9]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap-1" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp-1" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/BP-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf-1" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/MF-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc-1" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/CC-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot-1" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/scatter-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="network-analysis" class="section level2">
<h2><strong>6. Network analysis</strong></h2>
<p>Overview from <strong>STRING Database</strong>: In STRING, each protein-protein interaction is annotated with one or more ‘scores’. Importantly, these scores do <strong>not</strong> indicate the strength or the specificity of the interaction. Instead, they are indicators of <strong>confidence</strong>, i.e. how likely STRING judges an interaction to be true, given the available evidence. All scores rank from 0 to 1, with 1 being the highest possible confidence. A score of 0.5 would indicate that roughly every second interaction might be erroneous (i.e., a false positive).</p>
<pre class="r"><code>if(leng_dep != 0){
  
  string_db &lt;- STRINGdb$new(version=&quot;11&quot;, species=9606, score_threshold=400)

  string_mapped &lt;- string_db$map(string_res, &quot;Gene.name&quot;, removeUnmappedRows = TRUE, quiet = T)

  hits &lt;- string_mapped$STRING_id[1:leng_dep]

  string_db$plot_network(hits, add_link = T)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/Halo_nt_vs_Halo_act.Rmd/stringdb-1.png" width="1440" style="display: block; margin: auto;" /></p>
<div id="cytoscape-network" class="section level3">
<h3><strong>6.1. Cytoscape Network</strong></h3>
<p>Cytoscape is an open source software platform for visualizing complex networks and integrating these with any type of attribute data. In this presentation all proteins showing at least one confidence level from the STRING database are shown. The <strong>color</strong> code is relative to <strong>fold change</strong> and the <strong>size</strong> is directly proportional to the <strong>significance (p-value)</strong>.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre><code>[1] &quot;Enable Cytoscape option to see this results.&quot;</code></pre>
<pre class="r"><code>if(n_edge != 0){
  knitr::include_graphics(paste0(&quot;assets/network_FC_pvalue&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
</div>
<div id="cytoscape-enrichmentmap" class="section level3">
<h3><strong>6.2. Cytoscape EnrichmentMap</strong></h3>
<p>An <strong>enrichment map</strong> is a different sort of network. Instead of nodes representing genes, nodes represent pathways or functions. Edges between these pathways or functions represent shared genes or pathway crosstalk. An enrichment map is a way to visualize your enrichment results to help reduce redundancy and uncover main themes.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre><code>[1] &quot;Enable Cytoscape option to see this results.&quot;</code></pre>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant) %&gt;% pull(name)

if(!params$Imputation){
  
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  pull(name) %&gt;% 
  unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  
  print(&quot;NOT enough data.&quot;)
  
}else{
  
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  
  if(is.null(GO)){
    
    print(&quot;NOT enough data.&quot;)

    enrich_map = F
    
  }else{
    
    enrich_map = T
    
    gem &lt;- GO$result[,c(&quot;term_id&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;intersection&quot;)]
    colnames(gem) &lt;- c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;Genes&quot;)
    gem$FDR &lt;- gem$p.Val
    gem$Phenotype = &quot;+1&quot;
    gem &lt;- gem[,c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;FDR&quot;, &quot;Phenotype&quot;, &quot;Genes&quot;)]
    head(gem)
  
    write.table(gem, here::here(&quot;output/enrichmentmap.txt&quot;),col.name=TRUE,sep=&quot;\t&quot;,row.names=FALSE,quote=FALSE)
    
  }
}</code></pre>
<pre class="r"><code>if(enrich_map){
  knitr::include_graphics(paste0(&quot;assets/network_enrichmentmap&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
</div>
</div>
<div id="section" class="section level2 unnumbered">
<h2></h2>
<pre class="r"><code>saveWorkbook(excel, here::here(paste0(&quot;output/results_table_&quot;, test, &quot;.xlsx&quot;)), overwrite = T)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] forcats_0.5.1               stringr_1.4.0              
 [3] dplyr_1.0.7                 purrr_0.3.4                
 [5] readr_1.4.0                 tidyr_1.1.3                
 [7] tibble_3.1.2                tidyverse_1.3.1            
 [9] openxlsx_4.2.4              here_1.0.1                 
[11] RColorBrewer_1.1-2          RCy3_2.10.2                
[13] STRINGdb_2.2.2              naniar_0.6.1               
[15] visdat_0.5.3                ggrepel_0.9.1              
[17] ggplot2_3.3.5               UpSetR_1.4.0               
[19] ggstatsplot_0.8.0           datapasta_3.1.0            
[21] patchwork_1.1.1             DT_0.18                    
[23] rrvgo_1.2.0                 org.Hs.eg.db_3.12.0        
[25] AnnotationDbi_1.52.0        gprofiler2_0.2.0           
[27] enrichR_3.0                 preprocessCore_1.52.1      
[29] SummarizedExperiment_1.20.0 Biobase_2.50.0             
[31] GenomicRanges_1.42.0        GenomeInfoDb_1.26.2        
[33] IRanges_2.24.1              S4Vectors_0.28.1           
[35] BiocGenerics_0.36.0         MatrixGenerics_1.2.1       
[37] matrixStats_0.59.0          DEP_1.12.0                 
[39] MBQN_2.2.0                  workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.3            R.methodsS3_1.8.1        
  [3] coda_0.19-4               bit64_4.0.5              
  [5] knitr_1.33                DelayedArray_0.16.2      
  [7] R.utils_2.10.1            data.table_1.14.0        
  [9] RCurl_1.98-1.3            doParallel_1.0.16        
 [11] generics_0.1.0            RSQLite_2.2.7            
 [13] correlation_0.6.1         chron_2.3-56             
 [15] bit_4.0.4                 lubridate_1.7.10         
 [17] xml2_1.3.2                httpuv_1.6.1             
 [19] assertthat_0.2.1          WRS2_1.1-2               
 [21] xfun_0.24                 hms_1.1.0                
 [23] jquerylib_0.1.4           evaluate_0.14            
 [25] promises_1.2.0.1          fansi_0.5.0              
 [27] readxl_1.3.1              caTools_1.18.2           
 [29] dbplyr_2.1.1              igraph_1.2.6             
 [31] DBI_1.1.1                 htmlwidgets_1.5.3        
 [33] reshape_0.8.8             kSamples_1.2-9           
 [35] Rmpfr_0.8-4               paletteer_1.3.0          
 [37] hash_2.2.6.1              ellipsis_0.3.2           
 [39] crosstalk_1.1.1           backports_1.2.1          
 [41] insight_0.14.2            gridBase_0.4-7           
 [43] vctrs_0.3.8               imputeLCMD_2.0           
 [45] Cairo_1.5-12.2            cachem_1.0.5             
 [47] withr_2.4.2               treemap_2.4-2            
 [49] cluster_2.1.0             lazyeval_0.2.2           
 [51] crayon_1.4.1              labeling_0.4.2           
 [53] pkgconfig_2.0.3           SuppDists_1.1-9.5        
 [55] slam_0.1-48               statsExpressions_1.1.0   
 [57] ProtGenerics_1.22.0       wordcloud_2.6            
 [59] rlang_0.4.11              RJSONIO_1.3-1.4          
 [61] lifecycle_1.0.0           MatrixModels_0.5-0       
 [63] sandwich_3.0-1            affyio_1.60.0            
 [65] BiocFileCache_1.14.0      modelr_0.1.8             
 [67] cellranger_1.1.0          rprojroot_2.0.2          
 [69] graph_1.68.0              Matrix_1.2-18            
 [71] mc2d_0.1-21               zoo_1.8-9                
 [73] reprex_2.0.0              whisker_0.4              
 [75] GlobalOptions_0.1.2       pheatmap_1.0.12          
 [77] png_0.1-7                 viridisLite_0.4.0        
 [79] rjson_0.2.20              mzR_2.24.1               
 [81] PMCMRplus_1.9.0           parameters_0.14.0        
 [83] bitops_1.0-7              shinydashboard_0.7.1     
 [85] R.oo_1.24.0               KernSmooth_2.23-17       
 [87] blob_1.2.1                shape_1.4.6              
 [89] multcompView_0.1-8        tmvtnorm_1.4-10          
 [91] ggsignif_0.6.2            scales_1.1.1             
 [93] memoise_2.0.0             magrittr_2.0.1           
 [95] plyr_1.8.6                gplots_3.1.1             
 [97] zlibbioc_1.36.0           compiler_4.0.3           
 [99] plotrix_3.8-1             pcaMethods_1.82.0        
[101] clue_0.3-59               cli_3.0.0                
[103] affy_1.68.0               XVector_0.30.0           
[105] pbapply_1.4-3             MASS_7.3-53              
[107] tidyselect_1.1.1          vsn_3.58.0               
[109] stringi_1.6.2             highr_0.9                
[111] yaml_2.2.1                GOSemSim_2.16.1          
[113] norm_1.0-9.5              MALDIquant_1.19.3        
[115] grid_4.0.3                sass_0.4.0               
[117] tools_4.0.3               rstudioapi_0.13          
[119] circlize_0.4.13           foreach_1.5.1            
[121] git2r_0.28.0              gridExtra_2.3            
[123] ipmisc_6.0.2              pairwiseComparisons_3.1.6
[125] farver_2.1.0              mzID_1.28.0              
[127] digest_0.6.27             BiocManager_1.30.16      
[129] shiny_1.6.0               proto_1.0.0              
[131] BWStest_0.2.2             Rcpp_1.0.6               
[133] broom_0.7.8               BayesFactor_0.9.12-4.2   
[135] performance_0.7.2         later_1.2.0              
[137] ncdf4_1.17                httr_1.4.2               
[139] MSnbase_2.15.7            ComplexHeatmap_2.6.2     
[141] effectsize_0.4.5          colorspace_2.0-2         
[143] rvest_1.0.0               XML_3.99-0.6             
[145] fs_1.5.0                  rematch2_2.1.2           
[147] gmm_1.6-6                 plotly_4.9.4.1           
[149] xtable_1.8-4              gmp_0.6-2                
[151] jsonlite_1.7.2            zeallot_0.1.0            
[153] NLP_0.2-1                 R6_2.5.0                 
[155] tm_0.7-8                  gsubfn_0.7               
[157] pillar_1.6.1              htmltools_0.5.1.1        
[159] mime_0.11                 glue_1.4.2               
[161] fastmap_1.1.0             BiocParallel_1.24.1      
[163] codetools_0.2-16          mvtnorm_1.1-2            
[165] utf8_1.2.1                lattice_0.20-41          
[167] bslib_0.2.5.1             sqldf_0.4-11             
[169] curl_4.3.2                gtools_3.9.2             
[171] zip_2.2.0                 GO.db_3.12.1             
[173] limma_3.46.0              rmarkdown_2.9            
[175] munsell_0.5.0             GetoptLong_1.0.5         
[177] GenomeInfoDbData_1.2.4    iterators_1.0.13         
[179] impute_1.64.0             haven_2.4.1              
[181] gtable_0.3.0              bayestestR_0.10.0        </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
