<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit at IEO" />


<title>LFQ_Analysis</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.18/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.9.4.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Template_LFQ</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LFQ_Analysis</h1>
<h4 class="author">Fabio Bedin | MS-Unit at IEO</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-29
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>SKAP2_HaloTag/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20211221code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20211221)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20211221code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20211221)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong2865ca5"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 2865ca5 </a>
</p>
</div>
<div id="strongRepositoryversionstrong2865ca5" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 2865ca5. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  Counts.pdf
    Untracked:  Intensity_Distributions.pdf
    Untracked:  code/custom_functions.R
    Untracked:  data/expdesign.tsv
    Untracked:  data/proteinGroups.txt
    Untracked:  output/enrichmentmap.txt
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_vs_Halo.png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_vs_Halo.png
    Untracked:  output/output.zip
    Untracked:  output/results_table_Halo._vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_C._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_C._vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_C_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_C..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_N_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_N_vs_Halo.xlsx

Unstaged changes:
    Modified:   analysis/_site.yml
    Deleted:    analysis/about.Rmd
    Deleted:    analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/NSCAP2_vs_CSCAP2.Rmd</code>) and HTML (<code>docs/NSCAP2_vs_CSCAP2.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
2865ca5
</td>
<td>
FabioBedin
</td>
<td>
2021-12-29
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>## Proteomics
library(&quot;MBQN&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;enrichR&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)

## Plots
library(&quot;DT&quot;)
library(&quot;patchwork&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)

## Networks
library(&quot;STRINGdb&quot;)
library(&quot;RCy3&quot;)
library(&quot;RColorBrewer&quot;)

## Other
library(&quot;here&quot;)
library(&quot;openxlsx&quot;)
library(&quot;tidyverse&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(paste0(&quot;data/&quot;, params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot; &amp; data$Peptides &gt;= params$n_peptides,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<pre class="r"><code>cond_1 &lt;- params$test_1

cond_2 &lt;- params$test_2

conditions&lt;-c(cond_1,cond_2)

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)

expdesign &lt;- read.table(here::here(paste0(&quot;data/&quot;, params$expdesign)), header = T, stringsAsFactors = F)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
if(max(expdesign$replicate)&lt;=3){
       threshold&lt;-0
     } else if(max(expdesign$replicate)&lt;6){
       threshold&lt;-1
     } else if (max(expdesign$replicate)&gt;=6){
       threshold&lt;-trunc(max(expdesign$replicate)*0.25) ## 0.25 serve per avere sempre il 75% di valid vales
     }

data_filt &lt;- filter_missval(data_se, thr = threshold)</code></pre>
<div id="skap2_n.-vs-skap2_c." class="section level1">
<h1><strong>SKAP2_N.</strong> vs <strong>SKAP2_C.</strong></h1>
<hr />
</div>
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>First, we need to wrangling the original dataset. From the <strong>proteinGroups</strong> obtained by <a href="https://www.maxquant.org/">MaxQuant</a> software, I remove proteins that are marked form the software as potential contaminant, only identify by site and reverse.<br />
We che also filter the initial datasets by excluding all proteins identified by less than 2 peptides.<br />
Now, we generate some quality control plots to investigate the structure of the dataset and observe the effect of filters.</p>
<div id="normalization-strategy" class="section level4">
<h4><strong>Normalization strategy:</strong></h4>
<pre class="r"><code>if(params$Normalization == &quot;VSN&quot;){
  data_filt &lt;- normalize_vsn(data_filt)
  print(&quot;VSN normalization is used&quot;)
} else if(params$Normalization == &quot;MBQN&quot;){
  data_filt &lt;- normalize_MBQN(data_filt)
  print(&quot;MBQN normalization is used&quot;)
} else {
  print(&quot;Max-LFQ normalization is used&quot;)
}</code></pre>
<p>[1] “Max-LFQ normalization is used”</p>
</div>
<div id="quality-control-plots" class="section level2 tabset tabset-fade">
<h2><strong>1. Quality control plots</strong></h2>
<div id="proteins-per-samples" class="section level3 tabset">
<h3>Proteins per samples</h3>
<p>This plot show the number of proteins identify in each samples <em>after and before User define</em> filters:</p>
<div id="filtered" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_filt)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/proteins-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_se)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/proteins-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="distributions" class="section level3 tabset">
<h3>Distributions</h3>
<p>This plot is useful for checking the distributions of the samples <em>after and before User define</em> filters:</p>
<div id="filtered-1" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_normalization(data_filt)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/normalization-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-1" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_normalization(data_se)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/normalization-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="missing-data" class="section level3 tabset">
<h3>Missing data</h3>
<p>This plot we can explore the missing data pattern <em>after and before User define</em> filters. Notice that the percentages of missingness are provided in the data. These are accurate to 1 decimal place. Also the dataset is arranged by columns with most missingness.</p>
<div id="filtered-2" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/missing-data-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-2" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>assay(data_se) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/missing-data-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample-cvs" class="section level3 tabset">
<h3>Sample CVs</h3>
<p>The coefficient of variation (CV) is a statistical measure of the dispersion of data points in a data series around the mean. The coefficient of variation represents the ratio of the standard deviation to the mean, and it is a useful statistic for comparing the degree of variation from one data series to another, even if the means are drastically different from one another.</p>
<div id="filtered-3" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_cvs(data_filt)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/CVs-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-3" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_cvs(data_se)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/CVs-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="upset-plot" class="section level2 tabset tabset-fade">
<h2><strong>2. Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within the 2 conditions or in common between all condition.</p>
<div id="skap2_n." class="section level3">
<h3>SKAP2_N.</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_1)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_1)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/upset-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="skap2_c." class="section level3">
<h3>SKAP2_C.</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_2)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_2)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/upset-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="both-conditions" class="section level3">
<h3>Both conditions</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()
n_sets &lt;- length(define_set)
n_each &lt;- n_sets / 2

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;red3&quot;, &quot;royalblue1&quot;), each = n_each),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/upset-both-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="imputation" class="section level2">
<h2><strong>3. Imputation</strong></h2>
<p>An idealized version of a label-free discovery mass spectrometry proteomics experiment would provide absolute abundance measurements for a whole proteome, across varying conditions. Unfortunately, this ideal is not realized. Measurements are made on peptides requiring an inferential step to obtain protein level estimates. The inference is complicated by experimental factors that necessitate relative abundance estimation and result in widespread non-ignorable missing data. Relative abundance on the log scale takes the form of parameter contrasts. In a complete-case analysis, contrast estimates may be biased by missing data and a substantial amount of useful information will often go unused.</p>
<pre class="r"><code>data_imputed &lt;- mix_imputation_mean(data_filt, cond1 = cond_1, cond2 = cond_2)

plot_imputation(data_filt, data_imputed)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/imputation-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>data_filt &lt;- data_imputed</code></pre>
</div>
<div id="differential-enrichment-analysis" class="section level2">
<h2><strong>4. Differential enrichment analysis</strong></h2>
<p>Protein-wise linear models combined with empirical Bayes statistics are used for the differential enrichment analysis (or differential expression analysis) and False discovery rate (FDR) is ajusted with <em>Benjamini-Hochberg</em> procedure. Significant proteins are are define with <strong>FDR = 0.05</strong> and <strong>Fold change = 1</strong></p>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results&lt;-get_results(dep)</code></pre>
<p>There are <strong>497</strong> significant proteins in <strong>SKAP2_N.</strong> vs <strong>SKAP2_C.</strong> comparison.</p>
<p>The results from the previous analysis can be easily visualized by a number of functions. These visualizations assist in the determination of the optimal cutoffs to be used, highlight the most interesting samples and contrasts, and pinpoint differentially enriched/expressed proteins.</p>
<div id="visualization-of-the-results" class="section level3 tabset tabset-fade">
<h3><strong>4.1. Visualization of the results</strong></h3>
<div id="pca" class="section level4">
<h4>PCA</h4>
<p>The PCA plot can be used to get a high-level overview of the data. This can be very useful to observe batch effects, such as clear differences between replicates.</p>
<pre class="r"><code>if(params$Imputation){
  if(nrow(dep) &gt;= 500){
    n_value &lt;- 500
  }else {
    n_value &lt;- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
} else {
  var &lt;- apply(assay(dep), 1, sd)
  df &lt;- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value &lt;- df %&gt;% as.data.frame() %&gt;% drop_na() %&gt;% nrow()
  if(n_value &gt;= 500){
    n_value &lt;- 500
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
}</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/PCA-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano" class="section level4">
<h4>Volcano</h4>
<p>Volcano plots allows to inspect the enrichment of proteins between the two samples (x axis) and their corresponding adjusted p value (y axis).</p>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T, label_size=5, adjusted = F)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/volcano-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-and-unique" class="section level4">
<h4>Volcano and unique</h4>
<pre class="r"><code>plot_volcano_2(dep, contrast=test, add_names=T, label_size=5, adjusted = F, unique_vec = &quot;SKAP2&quot;)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/volcano-unique-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="correlation" class="section level4">
<h4>Correlation</h4>
<pre class="r"><code>plot_cor(dep, significant = TRUE, lower = 0, upper = 1, pal = &quot;Reds&quot;)</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/correlation-matrix-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-heatmap" class="section level4">
<h4>Cluster heatmap</h4>
<p>The heatmap representation gives an overview of all significant proteins (rows) in all samples (columns). This allows to see general trends, for example if one sample or replicate is really different compared to the others. Additionally, the clustering of samples (columns) can indicate closer related samples and clustering of proteins (rows) indicates similarly behaving proteins.</p>
<pre class="r"><code>if(results %&gt;% filter(significant) %&gt;% nrow() &lt;= 100){
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = T, indicate = &quot;condition&quot;, col_limit = 5)
}else {
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = F, indicate = &quot;condition&quot;, col_limit = 5)
}</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/cluster-heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="result-table" class="section level3">
<h3><strong>4.2. Result table</strong></h3>
<p>In this table are summarized the results of DEP analysis, sorted by p.value. Proteins that are UP regulated are colored with a red box in Fold change column, in the orhter hand proteins that are DOWN regulated are colored with blue.</p>
<pre class="r"><code>results %&gt;% 
  select(starts_with(&quot;name&quot;) | starts_with(test)) %&gt;% 
  filter(across(ends_with(&quot;significant&quot;))) %&gt;% 
  arrange(across(ends_with(&quot;p.adj&quot;))) %&gt;%
  mutate(across(2:3, format, scientific = T, digits = 2)) %&gt;%
  dplyr::rename_with(~ tolower(gsub(paste0(test,&quot;_&quot;), &quot;&quot;, .x)), .cols = starts_with(test)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(Fold.Change = ratio, Gene.name = name) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:5)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;Fold.Change&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-2a0dd1ba32e0541469b5" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2a0dd1ba32e0541469b5">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497"],["CYBRD1","KRI1","RGS19","SVIP","CTDSP1","EIF3K","A8K968","hCG_2004507","MFGE8","RAP1A","FARP2","UTP11L","KRAS","ALCAM","AAK1","ASF1B","SELPLG","PYCRL","GNAQ","HLA-C","ITGB1","MORF4L1","RRAS","CC2D1A","FYN","RCBTB2","EMG1","INCENP","CCNY","hCG_1994888","TRAFD1","ARPC5L","PPP6R3","TBCE","CSNK1G2","STXBP6","KIAA1429","PLXNA1","GPC4","HTATSF1","SFSWAP","ING5","BRD1","HRAS","CCDC59","MFI2","REL","MAK16","RALB","PHIP","PTPRA","GABRA2","PRRC2A","KDM3A","RPRD1B","STX4","TJAP1","GYPC","PPIG","IGF1R","DGKA","DUSP11","WDR43","IMP3","GPX1","DOCK9","DHX37","DDX52","PTPRG","PHF14","GNG2","C12orf75","MYO1C","MRPL19","DDX10","FMNL3","MTFMT","PFKM","GNG4","NOP9","ZDHHC5","NDNL2","WDR46.1","AATF","NME1","YES1","PWP2","IL2RG","YY1","PRAM1","DBN1","GNAO1","BST2","GNB2","LCK","GNAI2","MPHOSPH10","FLOT2","LAMTOR3","MYO1G","POGZ","ABCC1","MFAP1","EPB41L2","SPEN","FLOT1","MKI67","LAT","RP9","OSBP","RRAS2","HIST2H2AC","NHP2L1","DAAM1","H2AFY","EPB41","EPB41L3","DPP4","GNB1","LIMA1","WDR36","INPP5A","RSL1D1","GLIPR2","DDX56","UTP3","RFTN1","RRP12","NOC3L","C1orf198","NVL","RP2","TSN","HIST1H4A","PRKAR2A","RALA","RRP1","BOP1","THY1","POLR2E","EFR3A","PAG1","SF3B6","PARN","CALM2","SERPINB9","EBNA1BP2","IK","PRNP","NRAS","KTN1","CD44","HIST1H2BN","PDCD11","DNTTIP2","DOCK8","PTPRC","SNAP23","CD6","GNAI3","RCL1","VIM","ULBP2","RBM19","NOP58","PHF5A","CSNK2B-LY6G5B-1181","GNL3","SCRIB","LGALS8","GNG5","KRR1","NOB1","GNA13","CSNK2A1","SIT1","MYH9","WDR75","GNAS","BAZ1B","CD70","RRP9","RPS12","UTP15","LIN7C","MYH10","MYL6","AHNAK","LMNB2","CD48","KDM2A","RPF2","TBL3","HNRNPC","BANF1","EIF3G","HIST1H2AA","NOP2","NOP56","SON","RBM22","GNL2","NSA2","STX3","TOP2B","NOP16","B2M","RAB6A","GRK6","MYL12A","SPTAN1","WDR74","ICAM3","LACTB","AP1M1","ITM2A","DDX18","PSMB2","EPB41L5","ZNF326","RNPS1","LRCH4","CHUK","RAB35","ARHGAP30","SAP18","BLM","WDR12","PRPF38A","SPTBN1","CBX5","CD2BP2","DCAF13","TRBC1","NPEPPS","SEPT1","PSMA2","RGS1","FAM120A","FXYD5","NOL7","RALY","SAFB2","THOC2","NOL6","IFITM2","SLC3A2","SP140L","RAC2","RAP2B","PDCD6","DHX33","H2AFV","PNN","HLA-A","UTP14A","ACTN4","ATP6V1B2","CD97","DDX49","NIFK","PES1","TOP2A","PTBP3","TTC4","CAPG","HNRNPM.1","HNRNPUL2","NCBP1","RPRC1","LMNB1","SUZ12","CMSS1","GNB2L1","FTSJ3","ITGAL","DDX27","EIF3B","DDX47","ANXA1","GMFG","HEATR1","GOLGA7","MARCKSL1","PEX19","RPL21","SLC7A5","HIST1H1B","CENPV","RPL27","ITGB2","EZR","GYG1","HLA-B","PNPO","BSG","HNRNPK.1","EIF4B","BYSL","DDX54","NOL10","ERH","DKC1","HMGCL","CIRH1A","CSNK1G3","KIAA0020","PDIA6","SLC16A1","BAG3","GTPBP4","NOL11","CEBPZ","HLA-C.1","RRS1","AKR1A1","HLA-G","CDV3","UTP18","FMNL1.1","UTRN","NOM1","RPL7L1","DDX41","MARCKS","PTMA","MAGOHB","KIFC1","RRBP1","SMC1A","SDHB","LAMTOR1","SLC9A3R1","CBX3","FBL","QARS","UTP6","BCLAF1","EEF1D","PDIA3","SKAP2","CDC20","ACSL1","EXOSC4","NUPL1","RAP1B.1","STOM","LMAN1","TOMM40","HLA-DRA","ACAP1","RRP8","DUT.1","ELAVL1","RAP2C","RRP7A","POLR2A","RRP15","POLD1","USP16","PCK2","TRA2A","ARPC1B","NAT10","RAD21","SRFBP1","POLR2J3","RASA4B","PPP1R9B","COX4I1","LGALS3","RFK","LAMTOR2","EIF3M","FUBP1","HIST2H3PS2","USP36","PSMA6","PCM1","CCDC86","TMED4","UBC","LRRC8C","ADRM1","LAMP1","PRKAA1","STXBP2","ERBB2IP","SCAF11","CAPRIN1","KLC2","CYCS","YME1L1","RBMX","RBM28","RRP1B","UTP20","BRIX1","NOC2L","THRAP3","CHTOP","PSMA7","AIMP2","NHP2","PSMA5","PSMB4","TMSB4X","POLR2B","CLTC","RCOR1","DDX39A.1","LAS1L","SLC1A5","COX7A2","EXOSC10","CD5","PANK4","ATRX","SMARCA5","POLR2G","AP1B1","DLST","DDX39B","EIF2S3","EIF4A3","GOT2","MRTO4","S100A8","MRPS6","STMN1","GLS.1","HLA-DRB1","HSCB","NAP1L1","ESF1","FKBP1A","DBI","FAM208A","MPHOSPH8","SAFB","ZC3H14","NOP14","SRSF9","ALYREF","EIF3L","EIF3F","INA","EIF3CL","KIAA0101","SYNGR2","CALCOCO2","TRIM24","TFRC","MSN","EIF4EBP1","HSPA4","DCTN2","NUSAP1","ZYX","LGALS9","MATR3","ATP1A1","SEC61B","TBCB","CDCA8","APMAP","GSTK1","RHOG","STMN2","RPLP1","HNRNPU","POLR2L","SMARCC2","UQCRC2","PGAM1","ACIN1","PSMA3","ALKBH4","SMC3","NIP7","PSMB1","SMS","STT3A","PRKCSH","CD82","PABPN1","SNRPB2","Pe1Fe10","CROCC","ATP1A3","RPS7","TDP2","PPIB","RAB11B","BCAP31","DHX9","SOD1"],["2.8e-12","2.4e-12","2.5e-12","2.6e-12","5.3e-12","6.5e-12","1.4e-11","1.5e-11","1.5e-11","1.4e-11","2.1e-11","2.1e-11","2.4e-11","3.2e-11","3.5e-11","4.4e-11","5.3e-11","7.1e-11","9.4e-11","9.7e-11","9.2e-11","8.7e-11","1.0e-10","1.3e-10","1.3e-10","1.3e-10","1.5e-10","1.4e-10","1.7e-10","1.7e-10","2.0e-10","2.3e-10","4.0e-10","4.0e-10","5.3e-10","8.4e-10","9.9e-10","1.1e-09","1.3e-09","1.3e-09","1.2e-09","1.4e-09","1.5e-09","1.5e-09","1.6e-09","1.7e-09","2.1e-09","2.3e-09","2.6e-09","2.9e-09","4.4e-09","5.1e-09","5.0e-09","5.4e-09","6.4e-09","7.6e-09","7.9e-09","1.0e-08","1.1e-08","1.2e-08","2.1e-08","2.7e-08","2.8e-08","3.2e-08","3.6e-08","3.8e-08","4.4e-08","4.5e-08","4.7e-08","5.1e-08","5.6e-08","5.9e-08","6.8e-08","7.2e-08","7.6e-08","8.3e-08","8.5e-08","8.6e-08","1.1e-07","1.2e-07","1.7e-07","1.9e-07","2.0e-07","2.1e-07","2.1e-07","2.1e-07","2.3e-07","2.5e-07","2.6e-07","4.3e-07","4.8e-07","4.8e-07","5.2e-07","5.1e-07","7.2e-07","7.3e-07","8.6e-07","8.8e-07","9.2e-07","9.5e-07","9.9e-07","1.0e-06","1.0e-06","1.1e-06","1.3e-06","1.4e-06","1.4e-06","1.5e-06","1.5e-06","1.6e-06","1.6e-06","1.8e-06","2.0e-06","2.3e-06","2.3e-06","2.4e-06","2.5e-06","2.6e-06","2.8e-06","2.8e-06","3.5e-06","3.6e-06","3.8e-06","4.0e-06","4.1e-06","4.3e-06","5.3e-06","5.5e-06","6.3e-06","6.6e-06","7.0e-06","7.2e-06","7.5e-06","7.7e-06","7.7e-06","8.4e-06","8.6e-06","8.8e-06","9.3e-06","9.7e-06","1.0e-05","1.0e-05","1.1e-05","1.1e-05","1.1e-05","1.2e-05","1.3e-05","1.3e-05","1.4e-05","1.4e-05","1.4e-05","1.5e-05","1.5e-05","1.6e-05","1.6e-05","1.8e-05","1.8e-05","2.0e-05","2.0e-05","2.1e-05","2.1e-05","2.1e-05","2.2e-05","2.3e-05","2.3e-05","2.5e-05","2.5e-05","2.6e-05","2.7e-05","2.7e-05","2.8e-05","3.0e-05","3.0e-05","3.1e-05","3.1e-05","3.1e-05","3.2e-05","3.3e-05","3.3e-05","3.7e-05","3.8e-05","3.8e-05","3.9e-05","4.1e-05","4.6e-05","4.6e-05","4.8e-05","5.1e-05","5.1e-05","5.3e-05","5.3e-05","5.4e-05","5.6e-05","5.6e-05","5.7e-05","5.8e-05","5.9e-05","6.4e-05","6.5e-05","6.6e-05","6.7e-05","7.3e-05","7.6e-05","7.9e-05","7.9e-05","8.0e-05","8.2e-05","8.4e-05","8.7e-05","8.7e-05","9.3e-05","9.2e-05","9.8e-05","9.7e-05","9.9e-05","1.1e-04","1.1e-04","1.1e-04","1.1e-04","1.1e-04","1.2e-04","1.2e-04","1.2e-04","1.3e-04","1.4e-04","1.4e-04","1.5e-04","1.5e-04","1.6e-04","1.8e-04","1.8e-04","1.8e-04","1.8e-04","1.9e-04","2.0e-04","2.0e-04","2.0e-04","2.0e-04","2.1e-04","2.2e-04","2.2e-04","2.2e-04","2.2e-04","2.2e-04","2.2e-04","2.3e-04","2.3e-04","2.3e-04","2.3e-04","2.4e-04","2.4e-04","2.4e-04","2.5e-04","2.7e-04","2.9e-04","3.0e-04","3.0e-04","3.1e-04","3.2e-04","3.1e-04","3.1e-04","3.1e-04","3.1e-04","3.2e-04","3.2e-04","3.3e-04","3.3e-04","3.3e-04","3.3e-04","3.4e-04","3.4e-04","3.4e-04","3.5e-04","3.6e-04","3.8e-04","3.8e-04","3.9e-04","3.9e-04","3.9e-04","4.0e-04","4.2e-04","4.3e-04","4.4e-04","4.5e-04","4.6e-04","4.5e-04","4.5e-04","5.0e-04","5.1e-04","5.2e-04","5.2e-04","5.3e-04","5.4e-04","5.4e-04","5.5e-04","5.6e-04","5.8e-04","6.0e-04","6.1e-04","6.1e-04","6.1e-04","6.2e-04","6.3e-04","6.4e-04","6.4e-04","6.5e-04","6.8e-04","6.8e-04","6.8e-04","7.0e-04","7.3e-04","7.4e-04","7.5e-04","7.6e-04","7.9e-04","8.1e-04","9.1e-04","9.2e-04","9.2e-04","9.3e-04","9.3e-04","9.6e-04","9.7e-04","9.7e-04","9.8e-04","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.8e-03","1.8e-03","1.9e-03","1.9e-03","1.9e-03","2.0e-03","2.0e-03","2.0e-03","2.0e-03","2.1e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.4e-03","2.5e-03","2.6e-03","2.6e-03","2.7e-03","2.7e-03","2.8e-03","2.9e-03","2.9e-03","3.0e-03","3.0e-03","3.1e-03","3.1e-03","3.3e-03","3.3e-03","3.4e-03","3.4e-03","3.4e-03","3.6e-03","3.7e-03","3.7e-03","3.7e-03","3.8e-03","4.0e-03","4.1e-03","4.2e-03","4.4e-03","4.4e-03","4.5e-03","4.5e-03","4.6e-03","4.6e-03","4.7e-03","4.9e-03","4.9e-03","5.0e-03","5.1e-03","5.1e-03","5.3e-03","5.4e-03","5.4e-03","5.5e-03","5.6e-03","5.6e-03","5.6e-03","5.7e-03","5.7e-03","5.8e-03","5.8e-03","5.8e-03","5.9e-03","5.9e-03","6.1e-03","6.2e-03","6.2e-03","6.4e-03","6.6e-03","6.6e-03","6.9e-03","7.0e-03","7.0e-03","7.1e-03","7.5e-03","7.6e-03","7.8e-03","7.9e-03","8.0e-03","8.3e-03","8.4e-03","8.4e-03","8.6e-03","8.6e-03","8.8e-03","8.9e-03","8.9e-03","9.2e-03","9.3e-03","9.2e-03","9.3e-03","9.3e-03","9.6e-03","9.7e-03","9.8e-03","9.9e-03","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02"],["1.7e-09","1.7e-09","1.7e-09","1.7e-09","2.6e-09","2.7e-09","3.8e-09","3.8e-09","3.8e-09","3.8e-09","4.4e-09","4.4e-09","4.6e-09","5.6e-09","5.7e-09","6.8e-09","7.8e-09","9.8e-09","1.1e-08","1.1e-08","1.1e-08","1.1e-08","1.1e-08","1.3e-08","1.3e-08","1.3e-08","1.3e-08","1.3e-08","1.4e-08","1.4e-08","1.6e-08","1.8e-08","2.9e-08","2.9e-08","3.8e-08","5.8e-08","6.7e-08","7.5e-08","7.8e-08","7.8e-08","7.8e-08","8.5e-08","8.6e-08","8.6e-08","9.0e-08","9.0e-08","1.1e-07","1.2e-07","1.3e-07","1.4e-07","2.1e-07","2.4e-07","2.4e-07","2.5e-07","2.9e-07","3.4e-07","3.4e-07","4.4e-07","4.4e-07","4.9e-07","8.6e-07","1.1e-06","1.1e-06","1.2e-06","1.4e-06","1.4e-06","1.6e-06","1.7e-06","1.7e-06","1.8e-06","2.0e-06","2.0e-06","2.3e-06","2.4e-06","2.5e-06","2.7e-06","2.7e-06","2.7e-06","3.3e-06","3.6e-06","5.2e-06","5.8e-06","5.9e-06","6.1e-06","6.1e-06","6.1e-06","6.4e-06","7.0e-06","7.2e-06","1.2e-05","1.3e-05","1.3e-05","1.4e-05","1.4e-05","1.9e-05","1.9e-05","2.2e-05","2.2e-05","2.3e-05","2.3e-05","2.4e-05","2.5e-05","2.5e-05","2.6e-05","3.0e-05","3.3e-05","3.3e-05","3.5e-05","3.5e-05","3.6e-05","3.6e-05","3.9e-05","4.4e-05","5.0e-05","5.0e-05","5.2e-05","5.3e-05","5.4e-05","5.9e-05","5.9e-05","7.2e-05","7.4e-05","7.6e-05","8.1e-05","8.2e-05","8.6e-05","1.0e-04","1.1e-04","1.2e-04","1.3e-04","1.3e-04","1.4e-04","1.4e-04","1.4e-04","1.4e-04","1.5e-04","1.6e-04","1.6e-04","1.7e-04","1.7e-04","1.8e-04","1.8e-04","1.8e-04","1.9e-04","2.0e-04","2.1e-04","2.2e-04","2.2e-04","2.3e-04","2.3e-04","2.4e-04","2.5e-04","2.5e-04","2.5e-04","2.6e-04","2.9e-04","2.9e-04","3.2e-04","3.2e-04","3.2e-04","3.2e-04","3.2e-04","3.4e-04","3.5e-04","3.5e-04","3.7e-04","3.8e-04","3.8e-04","3.9e-04","4.0e-04","4.1e-04","4.3e-04","4.3e-04","4.4e-04","4.4e-04","4.4e-04","4.4e-04","4.6e-04","4.6e-04","5.1e-04","5.2e-04","5.2e-04","5.3e-04","5.6e-04","6.1e-04","6.2e-04","6.4e-04","6.7e-04","6.7e-04","6.9e-04","6.9e-04","7.0e-04","7.2e-04","7.2e-04","7.3e-04","7.3e-04","7.4e-04","8.0e-04","8.1e-04","8.2e-04","8.3e-04","8.9e-04","9.3e-04","9.6e-04","9.6e-04","9.6e-04","9.8e-04","9.9e-04","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.6e-03","1.6e-03","1.6e-03","1.8e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","2.0e-03","2.1e-03","2.1e-03","2.1e-03","2.1e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.4e-03","2.4e-03","2.6e-03","2.7e-03","2.9e-03","2.9e-03","2.9e-03","2.9e-03","2.9e-03","2.9e-03","2.9e-03","2.9e-03","2.9e-03","2.9e-03","3.0e-03","3.0e-03","3.0e-03","3.0e-03","3.0e-03","3.0e-03","3.0e-03","3.1e-03","3.2e-03","3.3e-03","3.3e-03","3.4e-03","3.4e-03","3.4e-03","3.4e-03","3.5e-03","3.6e-03","3.7e-03","3.7e-03","3.7e-03","3.7e-03","3.7e-03","4.1e-03","4.2e-03","4.2e-03","4.2e-03","4.3e-03","4.3e-03","4.3e-03","4.4e-03","4.4e-03","4.6e-03","4.7e-03","4.7e-03","4.7e-03","4.7e-03","4.8e-03","4.9e-03","4.9e-03","4.9e-03","5.0e-03","5.1e-03","5.1e-03","5.1e-03","5.2e-03","5.4e-03","5.4e-03","5.5e-03","5.6e-03","5.7e-03","5.9e-03","6.5e-03","6.6e-03","6.6e-03","6.6e-03","6.6e-03","6.8e-03","6.8e-03","6.8e-03","6.8e-03","7.0e-03","7.1e-03","7.3e-03","7.3e-03","7.3e-03","7.6e-03","7.6e-03","7.7e-03","7.7e-03","7.7e-03","7.7e-03","7.8e-03","7.9e-03","7.9e-03","7.9e-03","7.9e-03","8.0e-03","8.1e-03","8.1e-03","8.3e-03","8.3e-03","8.4e-03","8.4e-03","8.8e-03","8.8e-03","8.8e-03","8.8e-03","8.9e-03","8.9e-03","8.9e-03","8.9e-03","9.1e-03","9.2e-03","9.2e-03","9.3e-03","9.4e-03","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","2.0e-02","2.0e-02","2.0e-02","2.1e-02","2.1e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.4e-02","2.4e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.7e-02","2.7e-02","2.7e-02","2.8e-02","2.8e-02","2.9e-02","2.9e-02","3.0e-02","3.0e-02","3.0e-02","3.2e-02","3.2e-02","3.3e-02","3.3e-02","3.3e-02","3.4e-02","3.5e-02","3.5e-02","3.5e-02","3.5e-02","3.6e-02","3.6e-02","3.6e-02","3.7e-02","3.7e-02","3.7e-02","3.7e-02","3.7e-02","3.8e-02","3.8e-02","3.8e-02","3.9e-02","4.0e-02","4.1e-02","4.1e-02","4.1e-02","4.2e-02","4.2e-02","4.2e-02","4.3e-02","4.3e-02","4.5e-02","4.5e-02","4.7e-02","4.7e-02","4.7e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.9e-02","4.9e-02","5.0e-02","5.0e-02","5.0e-02"],["True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True"],[5.82,-5.3,5.34,5.98,5.47,5.46,5.84,4.38,3.96,6.1,3.74,-5.52,4.28,6.36,3.76,-3.82,5.01,4.3,5.25,5.23,4.49,-3.77,3.85,3.06,5.45,4.51,-4.28,-5.15,5.34,5.82,2.87,4.43,3.96,3.91,6.72,3.06,2.85,3.16,2.67,2.86,2.82,-3.47,-3.85,4.53,-3.54,3.63,3.03,-6.59,4.55,-3.4,4.09,5.61,2.2,3.11,-4.55,4.27,-2.18,5.79,-3.05,2.57,-3.52,-3.97,-3.77,-6.11,-2.36,3.12,-2.95,-3.02,5.42,-3.43,6.15,6.44,3.8,-1.98,-5.79,2.84,-2.83,4.43,6.65,-3.66,4.26,-2.24,-4.26,-5.3,2.53,5.09,-4.87,1.6,-4.53,-7.07,3.08,4.36,4.88,4.56,6.29,3.91,-6.01,5.66,2.74,3.75,-1.84,4.58,-4.01,7.22,-3.72,5.72,-3.45,4.54,-3.19,3.48,6.83,-4.08,-3.24,7.73,-4.92,7.58,7.51,5.57,3.72,1.47,-4.59,4.95,-3.32,4.44,-4.52,-5.03,4.16,-3.01,-2.44,2.1,-3.78,4.69,2,-3.16,1.92,4.6,-3.76,-3.48,3.54,-1.24,5.97,4.92,-1.37,-2.74,3.43,-2.51,-3.06,-1.86,4.14,2.51,2.92,4.51,-3.06,-5.46,-4.8,1.37,4.22,6.01,3.98,5.48,-3.49,-2.82,3.79,-3.48,-2.57,-2.29,1.61,-1.62,4.52,4.95,4.66,-4.35,-1.16,5.51,1.25,1.1,1.3,-3.66,2.17,-4.43,1.54,-3.4,-1.05,-4.72,1.51,2.6,1.21,1.2,-2.43,6.26,-1.8,-4.77,-5.27,-2.79,-4.34,1.19,-5.1,-2.48,-2.48,-2.92,-1.37,-2.05,-2.31,2.46,-2.7,-4.65,3.38,1.5,1.69,1.69,1.67,-2.41,5.44,3.14,1.39,1.82,-2.63,1.56,1.16,-3.67,-2.34,2,2.22,2.46,1.16,-1.98,-1.27,-3.77,-1.41,2.84,-4.26,1.22,-3.24,-1.27,1.22,-1.08,1.33,-2.34,-1.15,1.27,-2.15,-4.43,-1.73,1.57,-2.46,3.37,1.79,-1.97,1.28,2.37,1.47,-1.27,-3.21,-2.62,1.78,-5.9,1.72,1.09,1.23,-1.53,-4.55,-2.9,-5.36,-1.68,-1.11,-1.15,-1.06,-2.66,-1.1,-1.74,-2.07,-1.14,-1.11,-1,-3.23,3.31,-4.86,2.58,-1.01,1.05,-1.34,-5.54,1.24,2.97,-1.45,1.23,1.23,-1.43,-2.81,1.05,2.81,3.47,1.4,2.9,1.52,3.6,-1.65,2.31,-2.88,-2.81,-3.46,-1.72,-1.62,-1.69,-3.02,1.96,-3.43,1.16,4.68,1.04,-3.61,-1.75,-2.39,4.36,-3.43,-1.53,1.29,2.4,-2.52,3.82,1.1,-1.13,-3.77,-1.08,1.84,3.38,-1.4,-1.33,1,-1.17,-1.19,2.85,1.33,-2,-1.74,-5.27,-3.08,-2.14,1.09,1.26,-1.52,-1.14,-2.31,1.22,-1.28,4.19,3.27,-1.83,1.94,2.19,1.41,-1.07,1.31,-1.02,2.47,-1.69,-3.49,-3.04,-1.05,-1.13,-1.09,-1.17,1.05,-1.75,-2.2,-1.44,-1.6,-1.27,2.49,2.09,1.56,-1.76,1.52,1.62,1.75,-2.16,-1.86,1.68,2.32,-1.8,1.44,-1.17,1.84,-1.18,2.17,1.39,4.23,2.63,-2.74,1.2,1.01,1.03,3.76,-2.16,-1.57,-1.35,-2.87,-2.02,-2.45,-1.88,-1.11,1.02,1.31,-1.99,1.76,2.45,2.75,-2.53,1.3,-1.01,-1.01,-1.24,2.56,1.67,-1.01,2.87,-1.14,-1.31,-3.09,-1.23,1.11,1.29,-1.21,1.14,-2.05,2.48,-1.52,-2.56,-1.54,1.66,1.15,2.34,-1.43,1.04,-1.11,1.14,1.27,-1.89,-2.02,-1.62,-1.51,-3.17,-1.98,2.2,2.6,1.69,-1.2,1.2,-1.34,1.67,3.2,-1.08,1.56,1.91,1.03,1.81,1.06,-1.1,1.23,1.76,-2.02,2.05,-1.42,1.08,-2.26,2.19,-1.33,1.02,2.51,1.75,-1.25,-1.25,1.2,2.14,2.32,-1.61,1.31,1.12,-1.86,-1.22,1.24,-1.08,1.07,2.8,1.27,-1.15,-1.5,-1.35,1.45,1.08,1.12,-1.12,1.19,1.25,1.35,-1.2,1.49]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene.name<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>Fold.Change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[5]; $(this.api().cell(row, 5).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[4]; $(this.api().cell(row, 4).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
<div id="gene-ontology" class="section level2">
<h2><strong>5. Gene Ontology</strong></h2>
<p>Now I perfom a gene onology analysis (GO or by branch <a href="GO:MF" class="uri">GO:MF</a>, <a href="GO:BP" class="uri">GO:BP</a>, <a href="GO:CC" class="uri">GO:CC</a>) and a REAC ontology.<br />
The enrichment results are visualized with an interactive <strong>Manhattan-like-plot</strong> named “Gostplot”.</p>
<div id="skap2_n.-1" class="section level3 tabset tabset-fade">
<h3><strong>5.1. SKAP2_N.</strong></h3>
<div id="gostplot" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in SKAP2_N. were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &gt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_1)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-ed4906484ed42468d86d" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-ed4906484ed42468d86d">{"x":{"data":[{"x":[82.8244879859375,74.4931449053838,131.501618274889,131.336343660126,94.0293556186209,76.2848719790647,122.189896229492,159.185116247694,98.7434383806114,83.0648874255928,126.795047995388,126.952810127662,106.793063367819,76.641714897303,98.7509508631006,93.6424627704257,83.2827494177805,121.750416003872,77.3892069049812,85.690500055578,83.9250666706094,130.295864835368,130.078002843181,75.620017278768,118.467461156079,132.658540578231,76.6454711385476,75.9130040958479,76.2548220491078,75.8679292009126,76.5140026949861,116.964964658234,75.8266105472218,132.75620285059,76.6266899323246,106.605251305588,74.3804576680454,76.5102464537415,77.3967193874704,109.982112184496,106.601495064344,144.051220273145,115.860629732317,107.059756496187,77.6897062045503,130.08551532567,76.4576590763169,77.6483875508596,77.6746812395719,132.688590508187,113.629422433016,118.516292292259,75.9430540258048,130.337183489059,181.662463855464,149.114633470884,155.248575423339,83.2001121103989,88.6128557438876,144.381769502671,76.5215151774754,115.87941093854,75.9092478546033,126.133949536336,173.158333677658,145.245704988932,146.81581382918,173.150821195169,83.910041705631,101.027233057337,128.507894002932,76.6604961035261,128.297544493234,93.6650002178934,75.8303667884664,136.865530772198,175.028941817476,101.459200800467,83.9513603593217,130.930669605708,170.510183600205,170.517696082694,94.3110737119669,130.292108594124,100.959620714934,142.49238015663,101.301438668193,115.560130432748,100.952108232444,75.7251920336172,181.827738470227,181.790176057781,128.312569458212,75.7139233098834,119.969957653925,77.6521437921042,130.934425846953,98.747194621856,116.17239775562,101.327732356906,101.200020154589,175.036454299965,101.015964333603,83.9062854643864,123.294231155408,178.849039163248,178.841526680759,146.534095735834,149.122145953373,111.807645429379,76.6492273797923,92.5306153620199,121.979546719793,85.9534369427009,119.996251342637,142.477355191651,108.806408674932,101.004695609869,77.4342817999165,130.363477177771,119.537989910794,83.9476041180771,132.771227815569,129.495785450266,96.7826804509229,89.961346350704,75.7101670686388,131.546693169825,84.289422071337,121.818028346275,145.369660950004,111.845207841825,101.057282987294,121.821784587519,77.9301056442056,87.373296133165,84.2856658300924,130.344695971548,146.766982693,75.6125047962788,130.453626967642,81.4609724141427,77.4530630061396,128.504137761688,101.215045119567,115.841848526094,109.501313305186,142.496136397874,84.2969345538262,121.81427210503,146.522827012101,170.664189491234,130.111809014382,72.6563429367676,166.047769001604,90.4346327475254,83.2714806940466,131.467812103688,74.3917263917793,114.493357919278,142.868004281091,81.5886846164596,132.898940017886,77.9225931617164,155.207256769648,130.442358243908,74.8499878236222,136.835480842241,131.471568344933,118.095593272862,74.8349628586437,101.000939368624,88.4626060941031,172.159173506591,177.000968470898,77.4417942824058,109.997137149475,101.48549448918,137.879715908244,131.554205652314,77.4680879711181,130.352208454038,126.798804236633,120.555931288084,83.4517802737881,136.005351527182,76.5590775899215,166.051525242848,132.7148841969,74.8462315823776,131.148531597896,74.8575003061114,152.206020015201,76.6717648272599,76.6642523447707,132.654784336986,170.656677008745,103.57772086243,130.108052773138,142.488623915385,74.8312066173991,93.7025626303395,159.988951874041,89.3227853391196,101.196263913344,115.909460868497,150.771135859759,83.267724452802,83.9438478768325,74.3842139092901,103.585233344919,105.294323111218,77.4756004536073,77.3929631462258,98.7547071043452,129.923996952152,96.6512120073614,74.8424753411329,102.661197998744,158.10707501049,131.817142539437,143.923508070828,73.5353033880073,126.137705777581,74.8387190998883,106.800575850308,103.555183414962,107.46167430936,145.640110319616,130.123077738116,101.305194909438,152.705600100735,77.4455505236504,145.373417191249,169.04900575605,149.242345673201,84.1166349740847,90.4308765062808,168.868706176309,152.713112583224,110.121093110547,101.03850178107,120.255431988515,103.558939656206,74.48187618165,75.1579996056805,84.0377539079478,81.8027903674026,114.03134024619,105.286810628729,81.4572161728981,106.522613998207,106.515101515718,104.569368551008,116.255035063002,98.7584633455898,83.1362560092405,143.931020553317,166.059037725338,102.657441757499,130.119321496872,123.8576673421,129.007474088466,177.560648416345,82.8320004684268,131.603036788494,82.1145583907055,152.70184385949,130.081759084425,76.7055709984615,83.0611311843482,103.562695897451,117.990418518013,131.535424446091,152.115870225331,105.346910488643,77.5169191072981,101.81979995995,76.739377169663,103.33356518153,74.4706074579162,127.399802835771,122.588057801421,167.287328612326,77.4906254185858,114.444526783098,82.8282442271822,129.860140850993,77.877518266781,77.4868691773411,145.760310039444,143.566665152589,88.4588498528584,106.804332091553,104.561856068518,97.8945278593286,110.117336869302,104.941236434224,171.749743210928,74.4668512166716,115.759211218712,108.855239811112,107.409086931936,83.9363353943433,140.738215495395,77.449306764895,116.251278821757,119.973713895169,77.498137901075,131.610549270983,83.0836686318159,115.582667880216,166.055281484093,113.813478254002,119.286321747405,83.2264057991112,139.314600063686,176.407482354249,126.336786563545,182.180825147221,116.360209817851,132.850108881706,145.501129393566,101.493006971669,101.823556201195,115.563886673993,77.4718442123627,184.76136288227,173.398733117313,172.448404082426,132.868890087929,118.343505195007,77.5131628660534,119.293834229894,152.517788038504,182.62030537284,85.4200506859658,77.4793566948519,118.471217397324,76.2360408428847,107.348987072022,130.919400881974],"y":[17,17,17,17,15.423279980731,15.2038121710957,14.535891762541,14.1080263169424,13.8880723472361,13.8409389196334,13.5872753935098,13.3422463172517,13.1017335484484,13.0598190093979,12.6564989145653,12.4932590940423,11.9370068521396,11.7737454524278,11.4859149668691,11.2678307676826,11.1274306078039,11.0104119587946,10.9873832482455,10.8577035069809,10.8470529884397,10.7697468613245,10.6775123062207,10.5205339901161,10.4744353433944,10.3970150656992,10.3837333956591,10.2361666152574,10.0738671792165,10.071290307211,10.0427322773827,9.86627710160164,9.73401903502959,9.72520438233403,9.70441816689967,9.70228082475313,9.65386400980409,9.55085920974461,9.55034985224728,9.5245737654475,9.40055360252713,9.36202296266775,9.34761359921509,9.34761359921509,9.33635881886832,9.30235156376463,9.17649337820529,8.99753980532624,8.88261444685442,8.80568379558518,8.7807358873469,8.77651517363439,8.72665833828399,8.68138967271398,8.67401251439017,8.64090197704686,8.61268819611282,8.5684884428851,8.10888336140142,8.10185889101475,8.05396392452799,8.05055944729514,7.96836011740323,7.94859955189904,7.9480755322297,7.86880313744563,7.80497599918858,7.64524608826508,7.58427575391931,7.56880101127033,7.51738047917678,7.48923389790348,7.44856695257477,7.2222213026817,7.2090104607993,7.17421272949116,7.16813799191268,7.14139372290936,7.14106624728232,7.08621465760257,7.07041029427555,7.06716237249279,6.96382451604047,6.84845264721689,6.83844604226312,6.79157536738804,6.74218866141563,6.74218866141563,6.64818601132727,6.64489101611115,6.62682863793061,6.59696201448116,6.55411315803877,6.54433935178734,6.52933144946777,6.52797332481943,6.45391694385284,6.44849461217268,6.4437804351837,6.33694603661798,6.28848381202199,6.21130079851163,6.20815792280222,6.17402007731777,6.15261888131337,6.11464495118632,6.08533104004336,6.04398591522273,5.97033982186818,5.94078954153027,5.93538269151229,5.91976427790001,5.78429423446604,5.72751609992533,5.71060016628513,5.66632796400421,5.66368889658372,5.5660035746666,5.53103730982869,5.53103730982869,5.44874223081224,5.43694616670801,5.41881764205516,5.39613356833865,5.37245267934751,5.30453245211787,5.26154987489976,5.16510931766194,5.13137481438568,5.07010008480505,5.03999920768025,5.01732634231817,5.01080893638238,4.99902994621019,4.98448130307416,4.96267332518292,4.95588717749719,4.92175316338228,4.92028934445648,4.86086316537784,4.7533209939067,4.71795210283185,4.71130486979752,4.69276513624786,4.61116261395849,4.5869597101573,4.58679023539704,4.56456688969983,4.48524908124958,4.4520761584143,4.42974978022741,4.35889420118439,4.34906290092018,4.20092286187654,4.19275694693568,4.17560583745944,4.15218326105578,4.13229501962893,4.11222814637958,4.03243874568881,4.02087928028099,3.99198956472273,3.99196270490094,3.93858930163555,3.91018634712627,3.90680706305783,3.90133530895344,3.89796239878058,3.89294054736202,3.89192236858986,3.87384597424387,3.80831766610275,3.74871619474185,3.73123033140043,3.72035549132599,3.71722414444904,3.7099939597612,3.63855167784997,3.63854440534744,3.57597646064337,3.57519127535809,3.54774357632879,3.51016633089003,3.44719919333183,3.43938223696289,3.42835505782634,3.42055243363551,3.41203337452226,3.40068296877605,3.38631634549174,3.38631634549174,3.38172184960911,3.36046387274059,3.28546812047912,3.24580123012569,3.2206745171766,3.20799438613786,3.17860879514214,3.15284304453082,3.14156539986583,3.0764173409276,3.02830575376789,2.98114791117167,2.90343986163215,2.8666973636324,2.8666973636324,2.85700050078932,2.84674874784604,2.84167448686613,2.80325628218023,2.79902767987345,2.79093990407885,2.7648738164443,2.75532634077816,2.72780015387387,2.71840982683763,2.65865269226576,2.65783239018429,2.59934736835831,2.55665665653023,2.55190692283938,2.49934802830804,2.49683532350614,2.47013489654444,2.44247419355323,2.44106402857384,2.43024131390046,2.41888578258642,2.37843806037543,2.37241039155013,2.36044933070531,2.34466114792644,2.32432524099232,2.30308205761427,2.29529480112223,2.2195719025847,2.20584608926635,2.1529538735215,2.14714497336653,2.12799641087573,2.11883882160595,2.11618421505245,2.09623440430983,2.08934547571827,2.08934547571827,2.08840753596578,2.07071965617429,2.07056754691724,2.07056754691724,2.06751119855864,2.05390611574354,2.04998499708259,2.02552797013692,2.00401455736048,2.00220237171563,1.99778652548903,1.97920059341028,1.97662845395691,1.97395299656435,1.96733399796505,1.96474694444811,1.95918134957181,1.95587263248687,1.94606159070403,1.93734773341145,1.93287889212563,1.92837367777662,1.91075861393869,1.88771233657086,1.88525106559932,1.88453554857603,1.87770217561107,1.86261693031186,1.85349684636566,1.84840084854303,1.81910111939488,1.79763333487256,1.78785615603955,1.75596600781748,1.75424636782126,1.74991031036238,1.7394668810169,1.73836180554003,1.71972437411106,1.7187871164583,1.71304051254279,1.71157980454345,1.70038883923563,1.68558600331739,1.67964926868181,1.66301416737073,1.65614390391564,1.64982500613966,1.64741185561743,1.64598302338977,1.64179364700049,1.63666152611189,1.61495789142527,1.61409681090712,1.60269478708966,1.60181794285946,1.58688391163678,1.56904672641552,1.56581682246173,1.55892658761855,1.55261378562977,1.53784598329023,1.53044031351308,1.52840766720974,1.52498164842093,1.52244142814586,1.52023625461823,1.50637297251088,1.50059288613812,1.49826398308581,1.48919314857221,1.48861036794063,1.48454951874617,1.48452089715253,1.46885111771526,1.45920564494918,1.41682572996727,1.41619360775998,1.40990790690088,1.38947235421861,1.380172333813,1.3778464238636,1.37221464929838,1.36706197896684,1.35533935467095,1.34888752379021,1.34366691568841,1.34127016201318,1.34099828464509,1.3365910112092,1.32945639972462,1.32581860030643,1.3108562311157],"text":["GO:0006810 (5214) <br> transport <br> 4.782e-18","GO:0001775 (1507) <br> cell activation <br> 6.471e-18","GO:0051234 (5364) <br> establishment of localization <br> 6.665e-18","GO:0051179 (6938) <br> localization <br> 7.986e-18","GO:0016192 (2193) <br> vesicle-mediated transport <br> 3.773e-16","GO:0002376 (3438) <br> immune system process <br> 6.254e-16","GO:0045321 (1341) <br> leukocyte activation <br> 2.911e-15","GO:0140352 (1413) <br> export from cell <br> 7.798e-15","GO:0019882 (236) <br> antigen processing and presentation <br> 1.294e-14","GO:0006887 (906) <br> exocytosis <br> 1.442e-14","GO:0046903 (1492) <br> secretion <br> 2.587e-14","GO:0048002 (193) <br> antigen processing and presentation of peptide antigen <br> 4.547e-14","GO:0032940 (1351) <br> secretion by cell <br> 7.912e-14","GO:0002478 (175) <br> antigen processing and presentation of exogenous peptide antigen <br> 8.713e-14","GO:0019884 (183) <br> antigen processing and presentation of exogenous antigen <br> 2.205e-13","GO:0016032 (1011) <br> viral process <br> 3.212e-13","GO:0006955 (2544) <br> immune response <br> 1.156e-12","GO:0045055 (790) <br> regulated exocytosis <br> 1.684e-12","GO:0002682 (1734) <br> regulation of immune system process <br> 3.267e-12","GO:0008104 (2670) <br> protein localization <br> 5.397e-12","GO:0007159 (377) <br> leukocyte cell-cell adhesion <br> 7.457e-12","GO:0050852 (202) <br> T cell receptor signaling pathway <br> 9.763e-12","GO:0050776 (1159) <br> regulation of immune response <br> 1.029e-11","GO:0002183 (35) <br> cytoplasmic translational initiation <br> 1.388e-11","GO:0043299 (538) <br> leukocyte degranulation <br> 1.422e-11","GO:0051641 (3519) <br> cellular localization <br> 1.699e-11","GO:0002479 (75) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent <br> 2.101e-11","GO:0002275 (552) <br> myeloid cell activation involved in immune response <br> 3.016e-11","GO:0002366 (723) <br> leukocyte activation involved in immune response <br> 3.354e-11","GO:0002263 (727) <br> cell activation involved in immune response <br> 4.009e-11","GO:0002444 (558) <br> myeloid leukocyte mediated immunity <br> 4.133e-11","GO:0042590 (80) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I <br> 5.805e-11","GO:0002252 (1135) <br> immune effector process <br> 8.436e-11","GO:0051668 (775) <br> localization within membrane <br> 8.486e-11","GO:0002474 (99) <br> antigen processing and presentation of peptide antigen via MHC class I <br> 9.063e-11","GO:0032880 (912) <br> regulation of protein localization <br> 1.361e-10","GO:0001732 (16) <br> formation of cytoplasmic translation initiation complex <br> 1.845e-10","GO:0002443 (888) <br> leukocyte mediated immunity <br> 1.883e-10","GO:0002684 (1079) <br> positive regulation of immune system process <br> 1.975e-10","GO:0034613 (1842) <br> cellular protein localization <br> 1.985e-10","GO:0032879 (2906) <br> regulation of localization <br> 2.219e-10","GO:0070727 (1856) <br> cellular macromolecule localization <br> 2.813e-10","GO:0042110 (495) <br> T cell activation <br> 2.816e-10","GO:0033036 (3148) <br> macromolecule localization <br> 2.988e-10","GO:0002768 (510) <br> immune response-regulating cell surface receptor signaling pathway <br> 3.976e-10","GO:0050778 (757) <br> positive regulation of immune response <br> 4.345e-10","GO:0002429 (475) <br> immune response-activating cell surface receptor signaling pathway <br> 4.491e-10","GO:0002757 (475) <br> immune response-activating signal transduction <br> 4.491e-10","GO:0002764 (513) <br> immune response-regulating signaling pathway <br> 4.609e-10","GO:0051649 (2778) <br> establishment of localization in cell <br> 4.985e-10","GO:0036230 (509) <br> granulocyte activation <br> 6.660e-10","GO:0043312 (483) <br> neutrophil degranulation <br> 1.006e-09","GO:0002283 (488) <br> neutrophil activation involved in immune response <br> 1.310e-09","GO:0050863 (341) <br> regulation of T cell activation <br> 1.564e-09","GO:1990778 (339) <br> protein localization to cell periphery <br> 1.657e-09","GO:0072657 (698) <br> protein localization to membrane <br> 1.673e-09","GO:0098609 (898) <br> cell-cell adhesion <br> 1.876e-09","GO:0006928 (2319) <br> movement of cell or subcellular component <br> 2.083e-09","GO:0010033 (3457) <br> response to organic substance <br> 2.118e-09","GO:0070887 (3443) <br> cellular response to chemical stimulus <br> 2.286e-09","GO:0002446 (500) <br> neutrophil mediated immunity <br> 2.440e-09","GO:0042119 (502) <br> neutrophil activation <br> 2.701e-09","GO:0002274 (671) <br> myeloid leukocyte activation <br> 7.782e-09","GO:0046649 (780) <br> lymphocyte activation <br> 7.909e-09","GO:1903829 (307) <br> positive regulation of cellular protein localization <br> 8.832e-09","GO:0071310 (2821) <br> cellular response to organic substance <br> 8.901e-09","GO:0071840 (6880) <br> cellular component organization or biogenesis <br> 1.076e-08","GO:1903827 (563) <br> regulation of cellular protein localization <br> 1.126e-08","GO:0007155 (1502) <br> cell adhesion <br> 1.127e-08","GO:0022610 (1509) <br> biological adhesion <br> 1.353e-08","GO:0048584 (2368) <br> positive regulation of response to stimulus <br> 1.567e-08","GO:0002483 (19) <br> antigen processing and presentation of endogenous peptide antigen <br> 2.263e-08","GO:0048518 (6474) <br> positive regulation of biological process <br> 2.604e-08","GO:0016043 (6661) <br> cellular component organization <br> 2.699e-08","GO:0002253 (566) <br> activation of immune response <br> 3.038e-08","GO:0060341 (851) <br> regulation of cellular localization <br> 3.242e-08","GO:1904375 (128) <br> regulation of protein localization to cell periphery <br> 3.560e-08","GO:0030155 (756) <br> regulation of cell adhesion <br> 5.995e-08","GO:0007166 (3226) <br> cell surface receptor signaling pathway <br> 6.180e-08","GO:0051049 (1839) <br> regulation of transport <br> 6.696e-08","GO:1903037 (341) <br> regulation of leukocyte cell-cell adhesion <br> 6.790e-08","GO:1903039 (243) <br> positive regulation of leukocyte cell-cell adhesion <br> 7.221e-08","GO:0016477 (1660) <br> cell migration <br> 7.227e-08","GO:0050851 (321) <br> antigen receptor-mediated signaling pathway <br> 8.199e-08","GO:0022409 (286) <br> positive regulation of cell-cell adhesion <br> 8.503e-08","GO:0065008 (4116) <br> regulation of biological quality <br> 8.567e-08","GO:0030029 (810) <br> actin filament-based process <br> 1.087e-07","GO:0040011 (2039) <br> locomotion <br> 1.418e-07","GO:0022407 (450) <br> regulation of cell-cell adhesion <br> 1.451e-07","GO:0002223 (112) <br> stimulatory C-type lectin receptor signaling pathway <br> 1.616e-07","GO:1990858 (113) <br> cellular response to lectin <br> 1.811e-07","GO:1990840 (113) <br> response to lectin <br> 1.811e-07","GO:0048522 (5863) <br> positive regulation of cellular process <br> 2.248e-07","GO:0002220 (115) <br> innate immune response activating cell surface receptor signaling pathway <br> 2.265e-07","GO:0044085 (3377) <br> cellular component biogenesis <br> 2.361e-07","GO:0002758 (116) <br> innate immune response-activating signal transduction <br> 2.530e-07","GO:0051050 (951) <br> positive regulation of transport <br> 2.792e-07","GO:0019883 (26) <br> antigen processing and presentation of endogenous antigen <br> 2.855e-07","GO:0042221 (4829) <br> response to chemical <br> 2.956e-07","GO:0030036 (712) <br> actin cytoskeleton organization <br> 2.965e-07","GO:0023052 (6937) <br> signaling <br> 3.516e-07","GO:1904377 (71) <br> positive regulation of protein localization to cell periphery <br> 3.560e-07","GO:0022607 (3104) <br> cellular component assembly <br> 3.599e-07","GO:0007154 (6961) <br> cell communication <br> 4.603e-07","GO:0045785 (442) <br> positive regulation of cell adhesion <br> 5.147e-07","GO:1905477 (132) <br> positive regulation of protein localization to membrane <br> 6.148e-07","GO:1905475 (203) <br> regulation of protein localization to membrane <br> 6.192e-07","GO:0071705 (2216) <br> nitrogen compound transport <br> 6.699e-07","GO:0072659 (287) <br> protein localization to plasma membrane <br> 7.037e-07","GO:0035556 (2886) <br> intracellular signal transduction <br> 7.680e-07","GO:0002480 (8) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-independent <br> 8.216e-07","GO:0015031 (1754) <br> protein transport <br> 9.037e-07","GO:0045184 (1866) <br> establishment of protein localization <br> 1.071e-06","GO:0008360 (157) <br> regulation of cell shape <br> 1.146e-06","GO:0044093 (1774) <br> positive regulation of molecular function <br> 1.160e-06","GO:0065003 (1724) <br> protein-containing complex assembly <br> 1.203e-06","GO:0034097 (1242) <br> response to cytokine <br> 1.643e-06","GO:0022604 (312) <br> regulation of cell morphogenesis <br> 1.873e-06","GO:0002694 (617) <br> regulation of leukocyte activation <br> 1.947e-06","GO:0050870 (221) <br> positive regulation of T cell activation <br> 2.156e-06","GO:0043933 (2017) <br> protein-containing complex subunit organization <br> 2.169e-06","GO:0007165 (6447) <br> signal transduction <br> 2.716e-06","GO:0051674 (1840) <br> localization of cell <br> 2.944e-06","GO:0048870 (1840) <br> cell motility <br> 2.944e-06","GO:0019221 (824) <br> cytokine-mediated signaling pathway <br> 3.558e-06","GO:0010499 (23) <br> proteasomal ubiquitin-independent protein catabolic process <br> 3.656e-06","GO:0002218 (144) <br> activation of innate immune response <br> 3.812e-06","GO:0051249 (523) <br> regulation of lymphocyte activation <br> 4.017e-06","GO:0007265 (352) <br> Ras protein signal transduction <br> 4.242e-06","GO:0045088 (316) <br> regulation of innate immune response <br> 4.960e-06","GO:0071345 (1148) <br> cellular response to cytokine stimulus <br> 5.476e-06","GO:0035567 (151) <br> non-canonical Wnt signaling pathway <br> 6.837e-06","GO:0022618 (196) <br> ribonucleoprotein complex assembly <br> 7.390e-06","GO:0045089 (217) <br> positive regulation of innate immune response <br> 8.509e-06","GO:0002833 (253) <br> positive regulation of response to biotic stimulus <br> 9.120e-06","GO:0009605 (3114) <br> response to external stimulus <br> 9.609e-06","GO:0007264 (520) <br> small GTPase mediated signal transduction <br> 9.754e-06","GO:0050865 (662) <br> regulation of cell activation <br> 1.002e-05","GO:0071826 (203) <br> ribonucleoprotein complex subunit organization <br> 1.036e-05","GO:0002181 (149) <br> cytoplasmic translation <br> 1.090e-05","GO:0050900 (511) <br> leukocyte migration <br> 1.107e-05","GO:0006413 (203) <br> translational initiation <br> 1.197e-05","GO:0002699 (228) <br> positive regulation of immune effector process <br> 1.201e-05","GO:0048583 (4416) <br> regulation of response to stimulus <br> 1.378e-05","GO:0023056 (1814) <br> positive regulation of signaling <br> 1.765e-05","GO:0042098 (206) <br> T cell proliferation <br> 1.914e-05","GO:0034341 (198) <br> response to interferon-gamma <br> 1.944e-05","GO:0065009 (3191) <br> regulation of molecular function <br> 2.029e-05","GO:0007267 (1735) <br> cell-cell signaling <br> 2.448e-05","GO:0045087 (1003) <br> innate immune response <br> 2.588e-05","GO:0071702 (2657) <br> organic substance transport <br> 2.589e-05","GO:1903078 (64) <br> positive regulation of protein localization to plasma membrane <br> 2.725e-05","GO:0050790 (2452) <br> regulation of catalytic activity <br> 3.272e-05","GO:0000165 (965) <br> MAPK cascade <br> 3.531e-05","GO:1901698 (1178) <br> response to nitrogen compound <br> 3.717e-05","GO:0010647 (1809) <br> positive regulation of cell communication <br> 4.376e-05","GO:0006952 (1956) <br> defense response <br> 4.476e-05","GO:0051222 (329) <br> positive regulation of protein transport <br> 6.296e-05","GO:0001738 (152) <br> morphogenesis of a polarized epithelium <br> 6.416e-05","GO:0038093 (238) <br> Fc receptor signaling pathway <br> 6.674e-05","GO:0070201 (595) <br> regulation of establishment of protein localization <br> 7.044e-05","GO:0006457 (230) <br> protein folding <br> 7.374e-05","GO:0051716 (7873) <br> cellular response to stimulus <br> 7.723e-05","GO:0002831 (426) <br> regulation of response to biotic stimulus <br> 9.280e-05","GO:0098542 (1222) <br> defense response to other organism <br> 9.531e-05","GO:0050896 (9487) <br> response to stimulus <br> 1.019e-04","GO:0001914 (35) <br> regulation of T cell mediated cytotoxicity <br> 1.019e-04","GO:0060333 (88) <br> interferon-gamma-mediated signaling pathway <br> 1.152e-04","GO:0051223 (570) <br> regulation of protein transport <br> 1.230e-04","GO:0043085 (1383) <br> positive regulation of catalytic activity <br> 1.239e-04","GO:0001910 (75) <br> regulation of leukocyte mediated cytotoxicity <br> 1.255e-04","GO:0022603 (1048) <br> regulation of anatomical structure morphogenesis <br> 1.265e-04","GO:0009967 (1626) <br> positive regulation of signal transduction <br> 1.280e-04","GO:1903533 (82) <br> regulation of protein targeting <br> 1.283e-04","GO:1904951 (345) <br> positive regulation of establishment of protein localization <br> 1.337e-04","GO:0002696 (414) <br> positive regulation of leukocyte activation <br> 1.555e-04","GO:0034622 (1150) <br> cellular protein-containing complex assembly <br> 1.784e-04","GO:0030168 (163) <br> platelet activation <br> 1.857e-04","GO:0060627 (544) <br> regulation of vesicle-mediated transport <br> 1.904e-04","GO:0051251 (366) <br> positive regulation of lymphocyte activation <br> 1.918e-04","GO:0002703 (212) <br> regulation of leukocyte mediated immunity <br> 1.950e-04","GO:0050867 (426) <br> positive regulation of cell activation <br> 2.299e-04","GO:0046907 (1723) <br> intracellular transport <br> 2.299e-04","GO:0044419 (1768) <br> biological process involved in interspecies interaction between organisms <br> 2.655e-04","GO:0007010 (1473) <br> cytoskeleton organization <br> 2.660e-04","GO:0060071 (110) <br> Wnt signaling pathway, planar cell polarity pathway <br> 2.833e-04","GO:0002456 (102) <br> T cell mediated immunity <br> 3.089e-04","GO:1901699 (728) <br> cellular response to nitrogen compound <br> 3.571e-04","GO:0051656 (443) <br> establishment of organelle localization <br> 3.636e-04","GO:0001913 (42) <br> T cell mediated cytotoxicity <br> 3.729e-04","GO:0051128 (2439) <br> regulation of cellular component organization <br> 3.797e-04","GO:0001916 (26) <br> positive regulation of T cell mediated cytotoxicity <br> 3.872e-04","GO:0090175 (114) <br> regulation of establishment of planar polarity <br> 3.975e-04","GO:0002486 (3) <br> antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway, TAP-independent <br> 4.109e-04","GO:0002484 (3) <br> antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway <br> 4.109e-04","GO:0051640 (657) <br> organelle localization <br> 4.152e-04","GO:1903076 (107) <br> regulation of protein localization to plasma membrane <br> 4.360e-04","GO:0031347 (889) <br> regulation of defense response <br> 5.182e-04","GO:0050789 (12122) <br> regulation of biological process <br> 5.678e-04","GO:0065007 (12802) <br> biological regulation <br> 6.016e-04","GO:0001909 (113) <br> leukocyte mediated cytotoxicity <br> 6.194e-04","GO:0016055 (533) <br> Wnt signaling pathway <br> 6.628e-04","GO:0198738 (535) <br> cell-cell signaling by wnt <br> 7.033e-04","GO:0010243 (1092) <br> response to organonitrogen compound <br> 7.218e-04","GO:0023051 (3669) <br> regulation of signaling <br> 8.387e-04","GO:0042129 (177) <br> regulation of T cell proliferation <br> 9.369e-04","GO:0075525 (5) <br> viral translational termination-reinitiation <br> 1.044e-03","GO:0006950 (4321) <br> response to stress <br> 1.249e-03","GO:0007164 (130) <br> establishment of tissue polarity <br> 1.359e-03","GO:0001736 (130) <br> establishment of planar polarity <br> 1.359e-03","GO:0031349 (364) <br> positive regulation of defense response <br> 1.390e-03","GO:0032388 (227) <br> positive regulation of intracellular transport <br> 1.423e-03","GO:0002705 (127) <br> positive regulation of leukocyte mediated immunity <br> 1.440e-03","GO:0002683 (614) <br> negative regulation of immune system process <br> 1.573e-03","GO:0019885 (13) <br> antigen processing and presentation of endogenous peptide antigen via MHC class I <br> 1.588e-03","GO:0050690 (29) <br> regulation of defense response to virus by virus <br> 1.618e-03","GO:0019081 (20) <br> viral translation <br> 1.718e-03","GO:0001912 (52) <br> positive regulation of leukocyte mediated cytotoxicity <br> 1.757e-03","GO:0030866 (40) <br> cortical actin cytoskeleton organization <br> 1.872e-03","GO:0120036 (1596) <br> plasma membrane bounded cell projection organization <br> 1.912e-03","GO:0051336 (1252) <br> regulation of hydrolase activity <br> 2.195e-03","GO:0070661 (327) <br> leukocyte proliferation <br> 2.199e-03","GO:0000902 (1045) <br> cell morphogenesis <br> 2.516e-03","GO:0046651 (296) <br> lymphocyte proliferation <br> 2.776e-03","GO:0001911 (20) <br> negative regulation of leukocyte mediated cytotoxicity <br> 2.806e-03","GO:0032943 (299) <br> mononuclear cell proliferation <br> 3.167e-03","GO:0031341 (109) <br> regulation of cell killing <br> 3.185e-03","GO:0033238 (83) <br> regulation of cellular amine metabolic process <br> 3.387e-03","GO:0071417 (671) <br> cellular response to organonitrogen compound <br> 3.610e-03","GO:0050794 (11517) <br> regulation of cellular process <br> 3.622e-03","GO:0030030 (1636) <br> cell projection organization <br> 3.713e-03","GO:0090314 (31) <br> positive regulation of protein targeting to membrane <br> 3.812e-03","GO:0002697 (421) <br> regulation of immune effector process <br> 4.184e-03","GO:0071346 (177) <br> cellular response to interferon-gamma <br> 4.242e-03","GO:1902600 (160) <br> proton transmembrane transport <br> 4.361e-03","GO:0072697 (11) <br> protein localization to cell cortex <br> 4.522e-03","GO:0007212 (41) <br> dopamine receptor signaling pathway <br> 4.739e-03","GO:0010646 (3634) <br> regulation of cell communication <br> 4.976e-03","GO:1902533 (1063) <br> positive regulation of intracellular signal transduction <br> 5.066e-03","GO:0090316 (184) <br> positive regulation of intracellular protein transport <br> 6.032e-03","GO:0034695 (25) <br> response to prostaglandin E <br> 6.225e-03","GO:0022613 (473) <br> ribonucleoprotein complex biogenesis <br> 7.031e-03","GO:0044267 (5280) <br> cellular protein metabolic process <br> 7.126e-03","GO:0031342 (24) <br> negative regulation of cell killing <br> 7.447e-03","GO:0001771 (14) <br> immunological synapse formation <br> 7.606e-03","GO:0002009 (566) <br> morphogenesis of an epithelium <br> 7.653e-03","GO:0007191 (10) <br> adenylate cyclase-activating dopamine receptor signaling pathway <br> 8.012e-03","GO:0006521 (66) <br> regulation of cellular amino acid metabolic process <br> 8.141e-03","GO:0036388 (66) <br> pre-replicative complex assembly <br> 8.141e-03","GO:0032386 (365) <br> regulation of intracellular transport <br> 8.158e-03","GO:0006412 (804) <br> translation <br> 8.497e-03","GO:0032831 (5) <br> positive regulation of CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation <br> 8.500e-03","GO:0032829 (5) <br> regulation of CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation <br> 8.500e-03","GO:0032103 (518) <br> positive regulation of response to external stimulus <br> 8.560e-03","GO:0042270 (6) <br> protection from natural killer cell mediated cytotoxicity <br> 8.833e-03","GO:0019886 (97) <br> antigen processing and presentation of exogenous peptide antigen via MHC class II <br> 8.913e-03","GO:0006909 (388) <br> phagocytosis <br> 9.429e-03","GO:0070663 (253) <br> regulation of leukocyte proliferation <br> 9.908e-03","GO:1901701 (1237) <br> cellular response to oxygen-containing compound <br> 9.949e-03","GO:0030865 (62) <br> cortical cytoskeleton organization <br> 1.005e-02","GO:0050793 (2621) <br> regulation of developmental process <br> 1.049e-02","GO:0045953 (16) <br> negative regulation of natural killer cell mediated cytotoxicity <br> 1.055e-02","GO:0048729 (675) <br> tissue morphogenesis <br> 1.062e-02","GO:1905114 (639) <br> cell surface receptor signaling pathway involved in cell-cell signaling <br> 1.078e-02","GO:0006812 (1210) <br> cation transport <br> 1.085e-02","GO:0051270 (1149) <br> regulation of cellular component movement <br> 1.099e-02","GO:0006605 (400) <br> protein targeting <br> 1.107e-02","GO:0090313 (37) <br> regulation of protein targeting to membrane <br> 1.132e-02","GO:0050777 (373) <br> negative regulation of immune response <br> 1.155e-02","GO:0002495 (101) <br> antigen processing and presentation of peptide antigen via MHC class II <br> 1.167e-02","GO:0006886 (995) <br> intracellular protein transport <br> 1.179e-02","GO:0031343 (69) <br> positive regulation of cell killing <br> 1.228e-02","GO:0043043 (834) <br> peptide biosynthetic process <br> 1.295e-02","GO:0051246 (2759) <br> regulation of protein metabolic process <br> 1.302e-02","GO:0090150 (385) <br> establishment of protein localization to membrane <br> 1.305e-02","GO:0032418 (79) <br> lysosome localization <br> 1.325e-02","GO:0002716 (17) <br> negative regulation of natural killer cell mediated immunity <br> 1.372e-02","GO:0030334 (1008) <br> regulation of cell migration <br> 1.401e-02","GO:0002504 (104) <br> antigen processing and presentation of peptide or polysaccharide antigen via MHC class II <br> 1.418e-02","GO:0031175 (1021) <br> neuron projection development <br> 1.517e-02","GO:0001768 (12) <br> establishment of T cell polarity <br> 1.594e-02","GO:0048193 (415) <br> Golgi vesicle transport <br> 1.630e-02","GO:0045591 (18) <br> positive regulation of regulatory T cell differentiation <br> 1.754e-02","GO:1902036 (74) <br> regulation of hematopoietic stem cell differentiation <br> 1.761e-02","GO:0002709 (80) <br> regulation of T cell mediated immunity <br> 1.779e-02","GO:0038061 (196) <br> NIK/NF-kappaB signaling <br> 1.822e-02","GO:0006811 (1610) <br> ion transport <br> 1.827e-02","GO:0050670 (232) <br> regulation of lymphocyte proliferation <br> 1.907e-02","GO:0002819 (172) <br> regulation of adaptive immune response <br> 1.911e-02","GO:0002708 (107) <br> positive regulation of lymphocyte mediated immunity <br> 1.936e-02","GO:0071456 (214) <br> cellular response to hypoxia <br> 1.943e-02","GO:0070498 (105) <br> interleukin-1-mediated signaling pathway <br> 1.993e-02","GO:0009966 (3251) <br> regulation of signal transduction <br> 2.063e-02","GO:0032944 (234) <br> regulation of mononuclear cell proliferation <br> 2.091e-02","GO:0032101 (1238) <br> regulation of response to external stimulus <br> 2.173e-02","GO:0019538 (5912) <br> protein metabolic process <br> 2.207e-02","GO:0034694 (32) <br> response to prostaglandin <br> 2.240e-02","GO:0032268 (2591) <br> regulation of cellular protein metabolic process <br> 2.252e-02","GO:1903416 (5) <br> response to glycoside <br> 2.260e-02","GO:0001767 (13) <br> establishment of lymphocyte polarity <br> 2.281e-02","GO:0042060 (555) <br> wound healing <br> 2.309e-02","GO:0034113 (60) <br> heterotypic cell-cell adhesion <br> 2.427e-02","GO:0033209 (179) <br> tumor necrosis factor-mediated signaling pathway <br> 2.432e-02","GO:0007162 (308) <br> negative regulation of cell adhesion <br> 2.496e-02","GO:0061418 (78) <br> regulation of transcription from RNA polymerase II promoter in response to hypoxia <br> 2.501e-02","GO:0002698 (107) <br> negative regulation of immune effector process <br> 2.589e-02","GO:0042269 (41) <br> regulation of natural killer cell mediated cytotoxicity <br> 2.697e-02","GO:0044087 (991) <br> regulation of cellular component biogenesis <br> 2.718e-02","GO:0002711 (52) <br> positive regulation of T cell mediated immunity <br> 2.761e-02","GO:0051272 (605) <br> positive regulation of cellular component movement <br> 2.801e-02","GO:0006892 (144) <br> post-Golgi vesicle-mediated transport <br> 2.898e-02","GO:0040017 (607) <br> positive regulation of locomotion <br> 2.948e-02","GO:1901700 (1748) <br> response to oxygen-containing compound <br> 2.962e-02","GO:0036294 (224) <br> cellular response to decreased oxygen levels <br> 2.986e-02","GO:0043618 (111) <br> regulation of transcription from RNA polymerase II promoter in response to stress <br> 3.003e-02","GO:0006935 (654) <br> chemotaxis <br> 3.018e-02","GO:0061024 (1016) <br> membrane organization <br> 3.116e-02","GO:1904778 (6) <br> positive regulation of protein localization to cell cortex <br> 3.158e-02","GO:0046718 (143) <br> viral entry into host cell <br> 3.175e-02","GO:2000027 (185) <br> regulation of animal organ morphogenesis <br> 3.242e-02","GO:0042330 (657) <br> taxis <br> 3.246e-02","GO:0051701 (243) <br> biological process involved in interaction with host <br> 3.277e-02","GO:0071380 (17) <br> cellular response to prostaglandin E stimulus <br> 3.277e-02","GO:0030177 (186) <br> positive regulation of Wnt signaling pathway <br> 3.397e-02","GO:0030335 (568) <br> positive regulation of cell migration <br> 3.474e-02","GO:0040012 (1113) <br> regulation of locomotion <br> 3.830e-02","GO:0002704 (55) <br> negative regulation of leukocyte mediated immunity <br> 3.835e-02","GO:2000736 (115) <br> regulation of stem cell differentiation <br> 3.891e-02","GO:1903905 (2) <br> positive regulation of establishment of T cell polarity <br> 4.079e-02","GO:1903613 (6) <br> regulation of protein tyrosine phosphatase activity <br> 4.167e-02","GO:0051707 (1600) <br> response to other organism <br> 4.189e-02","GO:0043207 (1601) <br> response to external biotic stimulus <br> 4.244e-02","GO:0002715 (45) <br> regulation of natural killer cell mediated immunity <br> 4.295e-02","GO:0043620 (117) <br> regulation of DNA-templated transcription in response to stress <br> 4.412e-02","GO:0090263 (153) <br> positive regulation of canonical Wnt signaling pathway <br> 4.478e-02","GO:2000145 (1070) <br> regulation of cell motility <br> 4.532e-02","GO:0007596 (351) <br> blood coagulation <br> 4.558e-02","GO:0002706 (156) <br> regulation of lymphocyte mediated immunity <br> 4.560e-02","GO:0043300 (46) <br> regulation of leukocyte degranulation <br> 4.607e-02","GO:0002361 (8) <br> CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation <br> 4.683e-02","GO:0033157 (256) <br> regulation of intracellular protein transport <br> 4.723e-02","GO:0051046 (650) <br> regulation of secretion <br> 4.888e-02"],"key":["GO:0006810","GO:0001775","GO:0051234","GO:0051179","GO:0016192","GO:0002376","GO:0045321","GO:0140352","GO:0019882","GO:0006887","GO:0046903","GO:0048002","GO:0032940","GO:0002478","GO:0019884","GO:0016032","GO:0006955","GO:0045055","GO:0002682","GO:0008104","GO:0007159","GO:0050852","GO:0050776","GO:0002183","GO:0043299","GO:0051641","GO:0002479","GO:0002275","GO:0002366","GO:0002263","GO:0002444","GO:0042590","GO:0002252","GO:0051668","GO:0002474","GO:0032880","GO:0001732","GO:0002443","GO:0002684","GO:0034613","GO:0032879","GO:0070727","GO:0042110","GO:0033036","GO:0002768","GO:0050778","GO:0002429","GO:0002757","GO:0002764","GO:0051649","GO:0036230","GO:0043312","GO:0002283","GO:0050863","GO:1990778","GO:0072657","GO:0098609","GO:0006928","GO:0010033","GO:0070887","GO:0002446","GO:0042119","GO:0002274","GO:0046649","GO:1903829","GO:0071310","GO:0071840","GO:1903827","GO:0007155","GO:0022610","GO:0048584","GO:0002483","GO:0048518","GO:0016043","GO:0002253","GO:0060341","GO:1904375","GO:0030155","GO:0007166","GO:0051049","GO:1903037","GO:1903039","GO:0016477","GO:0050851","GO:0022409","GO:0065008","GO:0030029","GO:0040011","GO:0022407","GO:0002223","GO:1990858","GO:1990840","GO:0048522","GO:0002220","GO:0044085","GO:0002758","GO:0051050","GO:0019883","GO:0042221","GO:0030036","GO:0023052","GO:1904377","GO:0022607","GO:0007154","GO:0045785","GO:1905477","GO:1905475","GO:0071705","GO:0072659","GO:0035556","GO:0002480","GO:0015031","GO:0045184","GO:0008360","GO:0044093","GO:0065003","GO:0034097","GO:0022604","GO:0002694","GO:0050870","GO:0043933","GO:0007165","GO:0051674","GO:0048870","GO:0019221","GO:0010499","GO:0002218","GO:0051249","GO:0007265","GO:0045088","GO:0071345","GO:0035567","GO:0022618","GO:0045089","GO:0002833","GO:0009605","GO:0007264","GO:0050865","GO:0071826","GO:0002181","GO:0050900","GO:0006413","GO:0002699","GO:0048583","GO:0023056","GO:0042098","GO:0034341","GO:0065009","GO:0007267","GO:0045087","GO:0071702","GO:1903078","GO:0050790","GO:0000165","GO:1901698","GO:0010647","GO:0006952","GO:0051222","GO:0001738","GO:0038093","GO:0070201","GO:0006457","GO:0051716","GO:0002831","GO:0098542","GO:0050896","GO:0001914","GO:0060333","GO:0051223","GO:0043085","GO:0001910","GO:0022603","GO:0009967","GO:1903533","GO:1904951","GO:0002696","GO:0034622","GO:0030168","GO:0060627","GO:0051251","GO:0002703","GO:0050867","GO:0046907","GO:0044419","GO:0007010","GO:0060071","GO:0002456","GO:1901699","GO:0051656","GO:0001913","GO:0051128","GO:0001916","GO:0090175","GO:0002486","GO:0002484","GO:0051640","GO:1903076","GO:0031347","GO:0050789","GO:0065007","GO:0001909","GO:0016055","GO:0198738","GO:0010243","GO:0023051","GO:0042129","GO:0075525","GO:0006950","GO:0007164","GO:0001736","GO:0031349","GO:0032388","GO:0002705","GO:0002683","GO:0019885","GO:0050690","GO:0019081","GO:0001912","GO:0030866","GO:0120036","GO:0051336","GO:0070661","GO:0000902","GO:0046651","GO:0001911","GO:0032943","GO:0031341","GO:0033238","GO:0071417","GO:0050794","GO:0030030","GO:0090314","GO:0002697","GO:0071346","GO:1902600","GO:0072697","GO:0007212","GO:0010646","GO:1902533","GO:0090316","GO:0034695","GO:0022613","GO:0044267","GO:0031342","GO:0001771","GO:0002009","GO:0007191","GO:0006521","GO:0036388","GO:0032386","GO:0006412","GO:0032831","GO:0032829","GO:0032103","GO:0042270","GO:0019886","GO:0006909","GO:0070663","GO:1901701","GO:0030865","GO:0050793","GO:0045953","GO:0048729","GO:1905114","GO:0006812","GO:0051270","GO:0006605","GO:0090313","GO:0050777","GO:0002495","GO:0006886","GO:0031343","GO:0043043","GO:0051246","GO:0090150","GO:0032418","GO:0002716","GO:0030334","GO:0002504","GO:0031175","GO:0001768","GO:0048193","GO:0045591","GO:1902036","GO:0002709","GO:0038061","GO:0006811","GO:0050670","GO:0002819","GO:0002708","GO:0071456","GO:0070498","GO:0009966","GO:0032944","GO:0032101","GO:0019538","GO:0034694","GO:0032268","GO:1903416","GO:0001767","GO:0042060","GO:0034113","GO:0033209","GO:0007162","GO:0061418","GO:0002698","GO:0042269","GO:0044087","GO:0002711","GO:0051272","GO:0006892","GO:0040017","GO:1901700","GO:0036294","GO:0043618","GO:0006935","GO:0061024","GO:1904778","GO:0046718","GO:2000027","GO:0042330","GO:0051701","GO:0071380","GO:0030177","GO:0030335","GO:0040012","GO:0002704","GO:2000736","GO:1903905","GO:1903613","GO:0051707","GO:0043207","GO:0002715","GO:0043620","GO:0090263","GO:2000145","GO:0007596","GO:0002706","GO:0043300","GO:0002361","GO:0033157","GO:0051046"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[21.5357213433502,20.0746712736595,21.5677047149222,21.8552699322811,20.5297041084802,21.0592849240276,19.9305008488025,19.9952668691908,17.6076905446195,19.4363650611617,20.0623636974978,17.3130398134153,19.9397170643226,17.1672744658673,17.2340225412223,19.5761024288984,20.706407382327,19.2600018955208,20.2463304681304,20.7635451231059,18.2706623897546,17.380365289039,19.7484871750406,14.4903838270411,18.7543947502514,21.0862719030328,15.8323472021764,18.788735757365,19.1448417015984,19.1520361624938,18.8031640286352,15.9391857668738,19.7222049058623,19.2351646883273,16.2854648740137,19.4448117412001,12.9090380872033,19.4106572227896,19.6584726927001,20.319702550628,20.8631967480583,20.3288742747175,18.6424947383594,20.9567768885572,18.6826935575456,19.2046630030309,18.5867790711158,18.5867790711158,18.6905785580485,20.8102706711732,18.6800539816722,18.609367044494,18.6232781866,18.1311916030541,18.1229749928138,19.0988745407174,19.4250082065263,20.5964123251416,21.0656756582917,21.0609703433762,18.6560402520018,18.6614171464117,19.0471784086146,19.2435012190827,17.9837617719465,20.8283362100316,21.8459599395529,18.8150591419289,20.0705834441879,20.0763023228695,20.6213122635352,13.2787931541743,21.7783613472339,21.8100397125636,18.82214113588,19.3559952095453,16.6906051657902,19.2029454719691,20.9853092903616,20.3177274481399,18.1311916030541,17.6499879306058,20.1931650350581,18.0465404446264,17.8834358538621,21.2667924080501,19.2923346973961,20.4423608203817,18.5134197986091,16.481642870719,16.4956598039246,16.4956598039246,21.6676406379136,16.5232800867865,21.0385096214352,16.5368885741886,19.4982908940975,13.9189573720891,21.4489310346629,19.1248320746054,21.8551101159863,15.7408670843248,20.9403466721754,21.8589389746917,18.4890008143628,16.7382817982581,17.3876399430454,20.5421825061326,17.8883946730703,20.8550932356453,11.2337416908317,20.2602817247612,20.3353801650254,17.0037958315268,20.2740632002794,20.2392898942552,19.835054721058,18.006537444193,18.9366475606211,17.5121875085122,20.4293081746705,21.773707326355,20.3183861996269,20.3183861996269,19.3144572343938,13.6736623489524,16.8721549697602,18.7165034152978,18.1754578485376,18.0244710336975,19.7365147296747,16.9446117361575,17.3358617234104,17.4855057979174,17.7081346500581,20.944102558088,18.7087830831335,19.0294427494667,17.3876399430454,16.9242986933942,18.6853274967924,17.3876399430454,17.5576195962344,21.3472335340474,20.3011325306085,17.4092279114231,17.3508621392231,20.9725984395191,20.2470321260701,19.5660201951071,20.7577848508211,15.5656749705695,20.6627410824846,19.5169123222971,19.768881008958,20.2977840946643,20.3923025232438,18.0810777484509,16.9546559960072,17.6199154662783,18.8885648714925,17.5703173351792,21.9949013322691,18.4386804137039,19.814778040641,22.1989315786353,14.4903838270411,16.0952695414637,18.8315204925904,19.9687227772095,15.8323472021764,19.621638123732,20.1678775892171,15.9798150349669,18.1474673384407,18.3995647489471,19.7387007173567,17.0605109063983,18.7692301298177,18.2296517587411,17.4513777342553,18.4386804137039,20.2385834258228,20.2699463460657,20.0465816412379,16.4531820242158,16.3332135329498,19.1538280687713,18.4920794806701,14.8262572073822,20.6564297500945,13.9189573720891,16.5095380077466,7.81088103584182,7.81088103584182,19.0194760795912,16.409380751435,19.4121001701208,22.4637053352364,22.5221566726976,16.4956598039246,18.7418935907743,18.7469093243631,19.6735807557026,21.1345415277369,17.1842747871435,9.83978094370953,21.3224068576028,16.7146488256242,16.7146488256242,18.2220502611965,17.5512244674609,16.6784250132177,18.9302015947495,12.4412469479671,14.1325441121217,13.3863990273633,15.2071559698249,14.7373847204614,20.1450886888246,19.8450614989918,18.0725305762401,19.6180112096102,17.9321863039161,13.3863990273633,17.9464587784159,16.4387325196073,15.9997101118596,19.0471784086146,22.4087088361335,20.1753736160522,14.2608475416207,18.4225304952707,17.1842747871435,17.0324495724136,12.0456958427095,14.7824526398553,21.1234693785035,19.6396061076506,17.2421374740836,13.8411375859653,18.5810671803456,21.5499130124727,13.7594961631324,12.6110489969071,18.82214113588,11.8112973047217,15.6178829039023,15.6178829039023,18.2258567164575,19.2827264956164,9.83978094370953,9.83978094370953,18.7036091768081,10.4153745295623,16.2527164787856,18.3103826663661,17.7081346500581,19.8300187191642,15.5115658618445,20.7416746444029,12.9090380872033,19.0549781704639,18.9829013053904,19.8024395839391,19.7376082365745,18.3523314408885,14.5938600426124,18.255902947563,16.3174752697517,19.5558507009536,15.6928968665451,19.3300112439233,20.8021927713396,18.2996733657727,15.9184363351098,13.0411641506074,19.5723317249652,16.3641747633204,19.5885846994232,12.254025725742,18.4028714608879,13.1640323425788,15.8100072347003,15.9391857668738,17.3358617234104,20.1557802914872,17.582893606206,17.1413646793065,16.409380751435,17.4651348098225,16.3794048207368,20.9942986487379,17.5953506241288,19.8310276725812,21.6769626200154,14.3213782810468,20.7280676150601,9.83978094370953,12.4412469479671,18.7959711236385,15.45541914919,17.2010629177462,17.9883493723974,15.8973863612294,16.409380751435,14.7824526398553,19.5507327718851,15.2071559698249,18.9106562544295,16.8721549697602,18.9150268638603,20.2561143588398,17.531850546536,16.4674845293838,19.0134564485586,19.5823601537695,10.4153745295623,16.8614836447151,17.250203574472,19.0194760795912,17.6499879306058,13.0411641506074,17.2582214035679,18.8268398015238,19.6975806690925,15.3051018762464,16.5232800867865,3.77952755905512,10.4153745295623,20.1481536865209,20.1489186491777,14.9507227107922,16.5503659342062,16.9646268148802,19.6478977043997,18.171496771787,16.9941087522152,14.990083183191,11.2337416908317,17.7250852434335,19.0053834439718],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[62.186587712122,67.1194768172047,59.8873279007658,62.0964206606962,57.3401086979889,52.9256801386042,52.7866726009895,53.8160797714333,53.9475733880958,55.7396435351822,63.648045337314,65.2222117767882,62.2692408425955,55.7358865747061,55.0746615309174,56.8479468756235,65.0193359110803,61.0369578064439,65.624206547728,65.5114977334458,53.3276749095439,56.0815269385048,67.2359425919629,53.6620443919143,53.64701655001,65.259781381549,65.3424345120225,54.8304590999727,65.5077407729697,59.5041179322065,61.4690082611922,57.7796730736894,56.1078256618373,56.5661748399181,65.3386775515465,55.574337274235,58.072715990823,56.6187722865831,55.938762440414,66.0825557258088,54.9544387956831,55.1159880961542,53.7183987990554,62.8553266768628,66.3831125638945,55.0258210447285,59.7445634026751,55.5780942347111,53.4215989214457,65.6091787058237,53.4516546052543,54.950681835207,52.5950676167098,59.8723000588615,59.6017989045843,65.2259687372643,63.4188707482736,55.9312485194619,56.6225292470591,62.7576457044849,54.8530008628292,59.8760570193376,54.8417299814009,56.4534660256359,59.6055558650604,61.2586184745321,65.312378828214,53.4178419609696,52.1329614781529,59.5492014579193,61.0632565297764,61.5216057078572,53.6808291942947,53.4478976447782,53.4328698028739,61.0745274112046,55.1197450566303,56.6262862075352,52.606338498138,55.73212961423,56.1040687013612,54.8680287047335,56.1190965432655,54.9581957561592,52.9482219014606,56.197992713263,56.6338001284874,61.5178487473811,56.092797819933,58.8842194536546,55.9199776380337,56.0777699780287,59.8835709402897,62.761402664961,53.9888999533326,53.7296696804836,54.8267021394967,59.4890900903022,55.1235020171064,52.602581537662,56.5361191561095,51.9563843357775,55.9162206775576,55.9575472427944,55.5104689461418,55.1310159380585,63.4075998668454,56.5398761165855,66.9128439910207,60.522254221222,61.0594995693003,53.8987329019068,57.6218807336943,56.2017496737391,59.6243406674408,53.5267938147757,55.9124637170815,52.2494272529112,53.8799480995265,67.243456512915,55.1535577009149,61.6568562849958,65.4964698915415,53.8198367319093,56.5323621956334,65.7744849667708,56.6751266937241,53.9024898623829,59.7483203631512,52.5725258538534,64.6210981006167,57.6932629827397,59.8910848612419,57.1935872394221,52.5762828143295,63.6630731792183,67.239699552439,58.5047664455713,56.6413140494395,53.8123228109572,59.7332925212469,57.2161290022785,56.6788836542002,55.9237345985097,55.9537902823183,53.8311076133375,53.8761911390504,57.2198859627546,56.0139016499354,64.4933614444302,57.6970199432158,65.6429913501083,65.627963508204,61.4314386564315,55.9049497961294,54.1542062142797,67.5928538371898,55.5142259066179,66.77007949293,61.2924311188168,55.8561093099404,54.1579631747558,59.2599155012618,63.8471642425458,56.0026307685072],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.053001574527,14.4411940150231,14.379478170174,14.2886234269289,14.2886234269289,14.1798164151282,13.4519235465993,13.4167655305895,13.2508022738748,12.6945802617553,12.4972192836307,12.4972192836307,12.3226610061341,11.7610277688418,11.4589165497266,11.3185822674873,11.2055232989201,10.9705406809711,10.6849802077375,10.6772158110811,10.6175500665656,10.5667263689128,10.4926633603242,10.4926633603242,10.2800080806762,9.68498115481477,9.50433261460083,9.28192933614394,9.15671193325684,9.09799530243039,9.09799530243039,8.99212192120092,8.12362599715292,8.01479704490989,7.81764719866713,7.80575755864039,7.66511844088969,7.66511844088969,7.65728047587883,7.52222616828959,7.42508152734303,7.31201497733212,7.28444121950699,7.14768796704545,7.10832326547862,7.03402569318647,6.98469925767872,6.7325814073405,6.72642773312083,6.72642773312083,6.68760085086806,6.63657710549647,6.55699495102504,6.42276366014275,6.41972549681808,6.23079230800888,6.21882640085513,6.14746421931625,6.1222446346427,6.1222446346427,6.10007812181417,5.45102007648312,5.34867134740031,5.2498521863033,5.17847245352875,5.17365711223556,4.86540041948982,4.85947046882881,4.76012401736591,4.67814301996719,4.53267742858809,4.42371530752923,4.36039005330948,4.32388234764146,4.23341825977972,4.22507880646806,4.17089295996881,4.1186831602622,3.88649369652682,3.74064678814742,3.7341717830512,3.65747890863379,3.60797849379713,3.60739691657299,3.51315978520786,3.36427579491171,3.35682303233943,3.32921431930081,3.32125999780773,3.31681517530271,3.29999192944036,3.27868455584681,3.22783702087891,3.1473337633742,3.1152883998515,3.10982245621506,3.05364883335954,3.02384267465111,3.02381042617761,2.92451006595994,2.77167741151089,2.76917276903177,2.7573965253671,2.71102066459445,2.58701290198078,2.45049269523595,2.40300723858164,2.40250620949792,2.3731247721495,2.30827661398165,2.29786097944384,2.29786097944384,2.22626718486735,2.13233373323689,2.08578643714658,2.05395416281679,1.96133001180389,1.95694743181089,1.92162562306363,1.90988139033139,1.89618463076736,1.8509883883024,1.83310303632937,1.75029987654266,1.74144689686361,1.72943162976705,1.71808063394479,1.70327559077133,1.65534930848737,1.59043757031349,1.57144669454467,1.51548415114953,1.50810283599425,1.4922131780614,1.45943621302541,1.43871636512942,1.43551476541722,1.41843627553584,1.40964637025288,1.38816803923253,1.37609803777189,1.35982089846972,1.3585290375364,1.31970464516592,1.30644517112698,1.3045166344597],"text":["GO:0070062 (2178) <br> extracellular exosome <br> 3.648e-57","GO:1903561 (2263) <br> extracellular vesicle <br> 3.473e-56","GO:0043230 (2265) <br> extracellular organelle <br> 3.858e-56","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 3.858e-56","GO:0031982 (4058) <br> vesicle <br> 3.534e-42","GO:0005615 (3595) <br> extracellular space <br> 7.686e-35","GO:0005576 (4564) <br> extracellular region <br> 1.074e-26","GO:0005886 (5680) <br> plasma membrane <br> 1.900e-24","GO:0005925 (420) <br> focal adhesion <br> 6.802e-24","GO:0030055 (427) <br> cell-substrate junction <br> 1.335e-23","GO:0071944 (6178) <br> cell periphery <br> 1.460e-21","GO:0098552 (622) <br> side of membrane <br> 1.915e-21","GO:0070161 (838) <br> anchoring junction <br> 9.255e-21","GO:0030054 (2107) <br> cell junction <br> 2.963e-19","GO:0016020 (9829) <br> membrane <br> 9.569e-19","GO:0031410 (2433) <br> cytoplasmic vesicle <br> 2.147e-18","GO:0097708 (2438) <br> intracellular vesicle <br> 2.458e-18","GO:0045121 (339) <br> membrane raft <br> 6.838e-17","GO:0098857 (339) <br> membrane microdomain <br> 6.838e-17","GO:0098797 (696) <br> plasma membrane protein complex <br> 8.851e-16","GO:0005737 (11951) <br> cytoplasm <br> 3.621e-15","GO:0030659 (789) <br> cytoplasmic vesicle membrane <br> 4.174e-15","GO:1905360 (36) <br> GTPase complex <br> 5.145e-15","GO:0005834 (36) <br> heterotrimeric G-protein complex <br> 5.145e-15","GO:0005829 (5361) <br> cytosol <br> 6.610e-15","GO:0098562 (195) <br> cytoplasmic side of membrane <br> 3.532e-14","GO:0098590 (1240) <br> plasma membrane region <br> 3.830e-14","GO:0009898 (169) <br> cytoplasmic side of plasma membrane <br> 5.613e-14","GO:0098796 (1326) <br> membrane protein complex <br> 2.020e-13","GO:0042470 (109) <br> melanosome <br> 3.183e-13","GO:0048770 (109) <br> pigment granule <br> 3.183e-13","GO:0032991 (5541) <br> protein-containing complex <br> 4.757e-13","GO:0030667 (308) <br> secretory granule membrane <br> 1.734e-12","GO:0031234 (103) <br> extrinsic component of cytoplasmic side of plasma membrane <br> 3.476e-12","GO:0098588 (2165) <br> bounding membrane of organelle <br> 4.802e-12","GO:0019897 (178) <br> extrinsic component of plasma membrane <br> 6.230e-12","GO:0033290 (15) <br> eukaryotic 48S preinitiation complex <br> 1.070e-11","GO:0031252 (426) <br> cell leading edge <br> 2.065e-11","GO:0030141 (857) <br> secretory granule <br> 2.103e-11","GO:0099503 (1027) <br> secretory vesicle <br> 2.412e-11","GO:0012506 (1183) <br> vesicle membrane <br> 2.712e-11","GO:0016282 (17) <br> eukaryotic 43S preinitiation complex <br> 3.216e-11","GO:0005852 (17) <br> eukaryotic translation initiation factor 3 complex <br> 3.216e-11","GO:0070993 (18) <br> translation preinitiation complex <br> 5.248e-11","GO:0120025 (2234) <br> plasma membrane bounded cell projection <br> 2.065e-10","GO:0015629 (517) <br> actin cytoskeleton <br> 3.131e-10","GO:0042995 (2337) <br> cell projection <br> 5.225e-10","GO:0019898 (315) <br> extrinsic component of membrane <br> 6.971e-10","GO:0005765 (395) <br> lysosomal membrane <br> 7.980e-10","GO:0098852 (395) <br> lytic vacuole membrane <br> 7.980e-10","GO:0005774 (450) <br> vacuolar membrane <br> 1.018e-09","GO:0012505 (4651) <br> endomembrane system <br> 7.523e-09","GO:0001772 (44) <br> immunological synapse <br> 9.665e-09","GO:0043226 (14027) <br> organelle <br> 1.522e-08","GO:0042611 (24) <br> MHC protein complex <br> 1.564e-08","GO:0098553 (25) <br> lumenal side of endoplasmic reticulum membrane <br> 2.162e-08","GO:0071556 (25) <br> integral component of lumenal side of endoplasmic reticulum membrane <br> 2.162e-08","GO:0030139 (335) <br> endocytic vesicle <br> 2.202e-08","GO:0031253 (346) <br> cell projection membrane <br> 3.005e-08","GO:0070820 (163) <br> tertiary granule <br> 3.758e-08","GO:0009986 (896) <br> cell surface <br> 4.875e-08","GO:0043227 (13201) <br> membrane-bounded organelle <br> 5.195e-08","GO:0009925 (250) <br> basal plasma membrane <br> 7.117e-08","GO:0031090 (3644) <br> organelle membrane <br> 7.792e-08","GO:0042612 (8) <br> MHC class I protein complex <br> 9.246e-08","GO:0045335 (139) <br> phagocytic vesicle <br> 1.036e-07","GO:0098576 (33) <br> lumenal side of membrane <br> 1.851e-07","GO:0005764 (723) <br> lysosome <br> 1.877e-07","GO:0000323 (723) <br> lytic vacuole <br> 1.877e-07","GO:0042581 (158) <br> specific granule <br> 2.053e-07","GO:0045178 (268) <br> basal part of cell <br> 2.309e-07","GO:0055038 (85) <br> recycling endosome membrane <br> 2.773e-07","GO:0005839 (23) <br> proteasome core complex <br> 3.778e-07","GO:0005773 (818) <br> vacuole <br> 3.804e-07","GO:0005768 (993) <br> endosome <br> 5.878e-07","GO:0045202 (1348) <br> synapse <br> 6.042e-07","GO:0016323 (225) <br> basolateral plasma membrane <br> 7.121e-07","GO:0031254 (13) <br> cell trailing edge <br> 7.547e-07","GO:0001931 (13) <br> uropod <br> 7.547e-07","GO:0030027 (203) <br> lamellipodium <br> 7.942e-07","GO:0030666 (190) <br> endocytic vesicle membrane <br> 3.540e-06","GO:0010008 (520) <br> endosome membrane <br> 4.481e-06","GO:0030670 (75) <br> phagocytic vesicle membrane <br> 5.625e-06","GO:0012507 (57) <br> ER to Golgi transport vesicle membrane <br> 6.630e-06","GO:0005622 (14808) <br> intracellular anatomical structure <br> 6.704e-06","GO:0030863 (121) <br> cortical cytoskeleton <br> 1.363e-05","GO:0031256 (179) <br> leading edge membrane <br> 1.382e-05","GO:0055037 (187) <br> recycling endosome <br> 1.737e-05","GO:0030662 (177) <br> coated vesicle membrane <br> 2.098e-05","GO:0035579 (90) <br> specific granule membrane <br> 2.933e-05","GO:0030135 (296) <br> coated vesicle <br> 3.770e-05","GO:0030658 (210) <br> transport vesicle membrane <br> 4.361e-05","GO:0043229 (13117) <br> intracellular organelle <br> 4.744e-05","GO:0070821 (72) <br> tertiary granule membrane <br> 5.842e-05","GO:0005938 (326) <br> cell cortex <br> 5.956e-05","GO:0005856 (2339) <br> cytoskeleton <br> 6.747e-05","GO:0009897 (409) <br> external side of plasma membrane <br> 7.609e-05","GO:0042383 (141) <br> sarcolemma <br> 1.299e-04","GO:0016324 (364) <br> apical plasma membrane <br> 1.817e-04","GO:0001917 (63) <br> photoreceptor inner segment <br> 1.844e-04","GO:0031226 (1725) <br> intrinsic component of plasma membrane <br> 2.200e-04","GO:0000139 (725) <br> Golgi membrane <br> 2.466e-04","GO:0030134 (90) <br> COPII-coated ER to Golgi transport vesicle <br> 2.469e-04","GO:0030176 (161) <br> integral component of endoplasmic reticulum membrane <br> 3.068e-04","GO:0019773 (10) <br> proteasome core complex, alpha-subunit complex <br> 4.322e-04","GO:0016328 (67) <br> lateral plasma membrane <br> 4.397e-04","GO:0071541 (7) <br> eukaryotic translation initiation factor 3 complex, eIF3m <br> 4.686e-04","GO:0031227 (169) <br> intrinsic component of endoplasmic reticulum membrane <br> 4.772e-04","GO:1902494 (1377) <br> catalytic complex <br> 4.822e-04","GO:0044291 (76) <br> cell-cell contact zone <br> 5.012e-04","GO:0045177 (433) <br> apical part of cell <br> 5.264e-04","GO:0005911 (496) <br> cell-cell junction <br> 5.918e-04","GO:0032432 (78) <br> actin filament bundle <br> 7.123e-04","GO:0030864 (92) <br> cortical actin cytoskeleton <br> 7.669e-04","GO:0042641 (79) <br> actomyosin <br> 7.766e-04","GO:0005794 (1593) <br> Golgi apparatus <br> 8.838e-04","GO:0030133 (416) <br> transport vesicle <br> 9.466e-04","GO:0000502 (67) <br> proteasome complex <br> 9.467e-04","GO:0005903 (109) <br> brush border <br> 1.190e-03","GO:1905369 (73) <br> endopeptidase complex <br> 1.692e-03","GO:0016459 (57) <br> myosin complex <br> 1.701e-03","GO:0060342 (4) <br> photoreceptor inner segment membrane <br> 1.748e-03","GO:0098791 (845) <br> Golgi apparatus subcompartment <br> 1.945e-03","GO:0005887 (1643) <br> integral component of plasma membrane <br> 2.588e-03","GO:0031225 (172) <br> anchored component of membrane <br> 3.544e-03","GO:0098978 (354) <br> glutamatergic synapse <br> 3.954e-03","GO:0031300 (419) <br> intrinsic component of organelle membrane <br> 3.958e-03","GO:0005912 (171) <br> adherens junction <br> 4.235e-03","GO:0043005 (1384) <br> neuron projection <br> 4.917e-03","GO:0001725 (70) <br> stress fiber <br> 5.037e-03","GO:0097517 (70) <br> contractile actin filament bundle <br> 5.037e-03","GO:0032588 (98) <br> trans-Golgi network membrane <br> 5.939e-03","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 7.373e-03","GO:0031680 (12) <br> G-protein beta/gamma-subunit complex <br> 8.208e-03","GO:0001726 (180) <br> ruffle <br> 8.832e-03","GO:0071986 (10) <br> Ragulator complex <br> 1.093e-02","GO:1905368 (97) <br> peptidase complex <br> 1.104e-02","GO:0034687 (2) <br> integrin alphaL-beta2 complex <br> 1.198e-02","GO:0031258 (22) <br> lamellipodium membrane <br> 1.231e-02","GO:0005885 (11) <br> Arp2/3 protein complex <br> 1.270e-02","GO:0042824 (9) <br> MHC class I peptide loading complex <br> 1.409e-02","GO:0031901 (167) <br> early endosome membrane <br> 1.469e-02","GO:0031301 (389) <br> integral component of organelle membrane <br> 1.777e-02","GO:0030136 (194) <br> clathrin-coated vesicle <br> 1.814e-02","GO:0030175 (106) <br> filopodium <br> 1.865e-02","GO:0005890 (12) <br> sodium:potassium-exchanging ATPase complex <br> 1.914e-02","GO:0005902 (89) <br> microvillus <br> 1.980e-02","GO:0031902 (149) <br> late endosome membrane <br> 2.211e-02","GO:0030425 (641) <br> dendrite <br> 2.568e-02","GO:0097447 (643) <br> dendritic tree <br> 2.683e-02","GO:0032589 (61) <br> neuron projection membrane <br> 3.052e-02","GO:0098862 (164) <br> cluster of actin-based cell projections <br> 3.104e-02","GO:0098858 (217) <br> actin-based cell projection <br> 3.219e-02","GO:0048471 (745) <br> perinuclear region of cytoplasm <br> 3.472e-02","GO:0030131 (24) <br> clathrin adaptor complex <br> 3.642e-02","GO:0008091 (9) <br> spectrin <br> 3.668e-02","GO:1990184 (2) <br> amino acid transport complex <br> 3.816e-02","GO:0019774 (11) <br> proteasome core complex, beta-subunit complex <br> 3.894e-02","GO:0140534 (128) <br> endoplasmic reticulum protein-containing complex <br> 4.091e-02","GO:0046658 (63) <br> anchored component of plasma membrane <br> 4.206e-02","GO:0030118 (47) <br> clathrin coat <br> 4.367e-02","GO:0008180 (35) <br> COP9 signalosome <br> 4.380e-02","GO:0036477 (875) <br> somatodendritic compartment <br> 4.790e-02","GO:0090533 (19) <br> cation-transporting ATPase complex <br> 4.938e-02","GO:0030315 (53) <br> T-tubule <br> 4.960e-02"],"key":["GO:0070062","GO:1903561","GO:0043230","GO:0065010","GO:0031982","GO:0005615","GO:0005576","GO:0005886","GO:0005925","GO:0030055","GO:0071944","GO:0098552","GO:0070161","GO:0030054","GO:0016020","GO:0031410","GO:0097708","GO:0045121","GO:0098857","GO:0098797","GO:0005737","GO:0030659","GO:1905360","GO:0005834","GO:0005829","GO:0098562","GO:0098590","GO:0009898","GO:0098796","GO:0042470","GO:0048770","GO:0032991","GO:0030667","GO:0031234","GO:0098588","GO:0019897","GO:0033290","GO:0031252","GO:0030141","GO:0099503","GO:0012506","GO:0016282","GO:0005852","GO:0070993","GO:0120025","GO:0015629","GO:0042995","GO:0019898","GO:0005765","GO:0098852","GO:0005774","GO:0012505","GO:0001772","GO:0043226","GO:0042611","GO:0098553","GO:0071556","GO:0030139","GO:0031253","GO:0070820","GO:0009986","GO:0043227","GO:0009925","GO:0031090","GO:0042612","GO:0045335","GO:0098576","GO:0005764","GO:0000323","GO:0042581","GO:0045178","GO:0055038","GO:0005839","GO:0005773","GO:0005768","GO:0045202","GO:0016323","GO:0031254","GO:0001931","GO:0030027","GO:0030666","GO:0010008","GO:0030670","GO:0012507","GO:0005622","GO:0030863","GO:0031256","GO:0055037","GO:0030662","GO:0035579","GO:0030135","GO:0030658","GO:0043229","GO:0070821","GO:0005938","GO:0005856","GO:0009897","GO:0042383","GO:0016324","GO:0001917","GO:0031226","GO:0000139","GO:0030134","GO:0030176","GO:0019773","GO:0016328","GO:0071541","GO:0031227","GO:1902494","GO:0044291","GO:0045177","GO:0005911","GO:0032432","GO:0030864","GO:0042641","GO:0005794","GO:0030133","GO:0000502","GO:0005903","GO:1905369","GO:0016459","GO:0060342","GO:0098791","GO:0005887","GO:0031225","GO:0098978","GO:0031300","GO:0005912","GO:0043005","GO:0001725","GO:0097517","GO:0032588","GO:0043231","GO:0031680","GO:0001726","GO:0071986","GO:1905368","GO:0034687","GO:0031258","GO:0005885","GO:0042824","GO:0031901","GO:0031301","GO:0030136","GO:0030175","GO:0005890","GO:0005902","GO:0031902","GO:0030425","GO:0097447","GO:0032589","GO:0098862","GO:0098858","GO:0048471","GO:0030131","GO:0008091","GO:1990184","GO:0019774","GO:0140534","GO:0046658","GO:0030118","GO:0008180","GO:0036477","GO:0090533","GO:0030315"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[20.5214902439301,20.5672559227953,20.5683104706776,20.5683104706776,21.250520850314,21.1109970585917,21.384799437161,21.6320787809657,18.41927534827,18.4418855248262,21.7261763418418,18.9473154826326,19.3361763823889,20.4817706635075,22.2374238416628,20.6535046726708,20.6559427724165,18.1229749928138,18.1229749928138,19.0951202425796,22.4484604312869,19.2583620424432,14.5429649371964,14.5429649371964,21.5670744132122,17.3282977259613,19.833042946904,17.114947301444,19.9165368378714,16.4387325196073,16.4387325196073,21.604243722507,17.9883493723974,16.3487788308752,20.5143223808468,17.1926950579195,12.766247544801,18.4386804137039,19.3650320892764,19.5960108346389,19.774188875036,13.0411641506074,13.0411641506074,13.1640323425788,20.5518518073364,18.7010140522786,20.6056241461277,18.0200111191535,18.3350224500158,18.3350224500158,18.5134197986091,21.4062815734808,14.9103337480098,22.6196160803639,13.7594961631324,13.8411375859653,13.8411375859653,18.1063809880406,18.1515039320334,17.0605109063983,19.4221518872928,22.5549451746932,17.6909608332441,21.1266444165659,11.2337416908317,16.8179486159229,14.3797163833018,19.1448417015984,19.1448417015984,17.0134143816997,17.7907627740747,16.0386970905277,13.6736623489524,19.3050263840062,19.5532945209938,19.9369599331965,17.5383402391376,12.4412469479671,12.4412469479671,17.3876399430454,17.2898209035835,18.7087830831335,15.8323472021764,15.3670429037096,22.6771653543307,16.603011108722,17.2010629177462,17.2661915130826,17.1842747871435,16.1317846515619,17.9321863039161,17.437476203366,22.5481301823792,15.7642778006896,18.0682354374322,20.6066429891266,18.3828990739611,16.8398886539314,18.2220502611965,15.5388662640176,20.2399959227273,19.1484443147305,16.1317846515619,17.0418680066494,11.8112973047217,15.6433129553031,10.8656489767313,17.114947301444,19.9633396074552,15.8543507531641,18.4609454905787,18.6452157443254,15.8973863612294,16.1673930557028,15.9184363351098,20.1427845172809,18.4061694695621,15.6433129553031,16.4387325196073,15.7873208982415,15.3670429037096,9.05045292005097,19.3468890873649,20.180591598588,17.1413646793065,18.1833431094616,18.4160117151406,17.1326162234276,19.9696175275459,15.717077397889,15.717077397889,16.2691849528541,22.4569580819102,12.254025725742,17.209378989379,11.8112973047217,16.2527164787856,3.77952755905512,13.5832101454551,12.0456958427095,11.5439512898152,17.0970441678342,18.3139322979379,17.3206905810587,16.394472388579,12.254025725742,16.1136431816444,16.9242986933942,18.9870200973406,18.9911249481172,15.4837562909781,17.0697370628997,17.4855057979174,19.1838884163435,13.7594961631324,11.5439512898152,3.77952755905512,12.0456958427095,16.6906051657902,15.5388662640176,15.0284638159893,14.4903838270411,19.391737283763,13.2787931541743,15.2405126341149],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[28.7757887903244,16.9543231164817,4.24258500390343,15.3202787669038,16.2143076294315,16.2105512056393,9.35507778499651,3.68663428266774,18.7010601798235,9.48279619392904,36.1083280325681,9.0921281195472,3.75424991092613,28.8997507754648,18.6935473322393,22.3297655630241,9.05456388162587,40.341817646302,28.9448278609704,21.169030611255,8.02906018637355,4.24634142769556,27.0515902697353,36.1534051180737,27.044077422151,27.0403209983589,40.8301527392793,16.1128841870439,19.3434086482783,16.8679253692626,22.299714172687,21.0075043881932,25.8194832659157,22.3072270202713,46.8704821970293,2.12771840893252,25.5790721432192,25.9960351841459,22.5363688715914,20.2712453249352,26.8487433849601,27.070372388696,13.9341583876067,21.6536092804401,3.64155719716214,29.669817652852,47.5616641747818,19.3471650720704,38.0353734379323,27.4460147679093,19.2344723583064,8.85922984443495,10.4782484988443,27.0478338459432,8.36713832766552,36.0557380994782,3.06306793317365,10.4820049226364,3.91577613398786,46.3333135947543,22.3785990723218,19.362190767239,19.3509214958626,41.1682308805713,2.10893628997186,27.6826694668137],"y":[17,12.8461108462391,11.9208510068443,11.3507799382996,11.0915734193438,11.0683567558089,10.9693839886783,9.50893037740043,9.35669376264523,8.88634157001901,8.78767764048335,8.70284982122704,8.35861790452005,8.27131716652941,8.21197633307425,8.21197633307425,7.99559654400057,7.93280756947757,7.27058677834636,6.72833187782171,6.45058430780059,6.40197457629601,5.9826136237533,5.53337798614583,5.50011729425649,5.11021774364376,5.10424157140482,5.04564266560708,4.80125203933995,4.41243256786597,4.39889998006179,4.13293523373854,4.12999972921352,4.08347117232385,3.68773602264053,3.30072386639765,3.29523378803958,2.94979280848292,2.92027753913682,2.60684849395983,2.57986673448344,2.48611089926975,2.44357036644594,2.39706518405263,2.28558046834612,2.19569430037697,2.17997399751265,2.15035827749867,2.1241775589581,2.06342093878065,1.94589132906274,1.90943436609025,1.90943436609025,1.90767920021312,1.87161611064465,1.80450686765968,1.72701202082485,1.70390698670488,1.69201894882709,1.69013979526294,1.68695184755275,1.56279679291144,1.52742852400374,1.52713093652082,1.51006683342885,1.49296437884589],"text":["GO:0044877 (1304) <br> protein-containing complex binding <br> 7.483e-21","GO:0017111 (424) <br> nucleoside-triphosphatase activity <br> 1.425e-13","GO:0003924 (300) <br> GTPase activity <br> 1.200e-12","GO:0016462 (482) <br> pyrophosphatase activity <br> 4.459e-12","GO:0016818 (493) <br> hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides <br> 8.099e-12","GO:0016817 (494) <br> hydrolase activity, acting on acid anhydrides <br> 8.544e-12","GO:0008092 (1001) <br> cytoskeletal protein binding <br> 1.073e-11","GO:0003743 (58) <br> translation initiation factor activity <br> 3.098e-10","GO:0019003 (73) <br> GDP binding <br> 4.399e-10","GO:0008135 (99) <br> translation factor activity, RNA binding <br> 1.299e-09","GO:0050839 (544) <br> cell adhesion molecule binding <br> 1.631e-09","GO:0005525 (381) <br> GTP binding <br> 1.982e-09","GO:0003779 (449) <br> actin binding <br> 4.379e-09","GO:0045182 (154) <br> translation regulator activity <br> 5.354e-09","GO:0019001 (402) <br> guanyl nucleotide binding <br> 6.138e-09","GO:0032561 (402) <br> guanyl ribonucleotide binding <br> 6.138e-09","GO:0005515 (14767) <br> protein binding <br> 1.010e-08","GO:0090079 (123) <br> translation regulator activity, nucleic acid binding <br> 1.167e-08","GO:0045296 (333) <br> cadherin binding <br> 5.363e-08","GO:0031683 (20) <br> G-protein beta/gamma-subunit complex binding <br> 1.869e-07","GO:0005102 (1551) <br> signaling receptor binding <br> 3.543e-07","GO:0003925 (32) <br> G protein activity <br> 3.963e-07","GO:0042610 (4) <br> CD8 receptor binding <br> 1.041e-06","GO:0051015 (213) <br> actin filament binding <br> 2.928e-06","GO:0042608 (11) <br> T cell receptor binding <br> 3.161e-06","GO:0042605 (30) <br> peptide antigen binding <br> 7.759e-06","GO:0097367 (2282) <br> carbohydrate derivative binding <br> 7.866e-06","GO:0016787 (2357) <br> hydrolase activity <br> 9.002e-06","GO:0019899 (2048) <br> enzyme binding <br> 1.580e-05","GO:0017076 (1932) <br> purine nucleotide binding <br> 3.869e-05","GO:0032553 (1934) <br> ribonucleotide binding <br> 3.991e-05","GO:0031369 (34) <br> translation initiation factor binding <br> 7.363e-05","GO:0035639 (1846) <br> purine ribonucleoside triphosphate binding <br> 7.413e-05","GO:0032555 (1917) <br> purine ribonucleotide binding <br> 8.251e-05","GO:1901265 (2177) <br> nucleoside phosphate binding <br> 2.052e-04","GO:0000166 (2176) <br> nucleotide binding <br> 5.004e-04","GO:0035254 (48) <br> glutamate receptor binding <br> 5.067e-04","GO:0036094 (2516) <br> small molecule binding <br> 1.123e-03","GO:0033218 (387) <br> amide binding <br> 1.201e-03","GO:0030507 (27) <br> spectrin binding <br> 2.473e-03","GO:0042277 (312) <br> peptide binding <br> 2.631e-03","GO:0042626 (120) <br> ATPase-coupled transmembrane transporter activity <br> 3.265e-03","GO:0015399 (123) <br> primary active transmembrane transporter activity <br> 3.601e-03","GO:0031821 (9) <br> G protein-coupled serotonin receptor binding <br> 4.008e-03","GO:0003723 (1951) <br> RNA binding <br> 5.181e-03","GO:0046977 (9) <br> TAP binding <br> 6.372e-03","GO:1990239 (6) <br> steroid hormone binding <br> 6.607e-03","GO:0019900 (757) <br> kinase binding <br> 7.074e-03","GO:0060090 (341) <br> molecular adaptor activity <br> 7.513e-03","GO:0043168 (2413) <br> anion binding <br> 8.641e-03","GO:0019829 (54) <br> ATPase-coupled cation transmembrane transporter activity <br> 1.133e-02","GO:0005391 (8) <br> P-type sodium:potassium-exchanging transporter activity <br> 1.232e-02","GO:0008554 (8) <br> P-type sodium transporter activity <br> 1.232e-02","GO:0042609 (9) <br> CD4 receptor binding <br> 1.237e-02","GO:0005200 (104) <br> structural constituent of cytoskeleton <br> 1.344e-02","GO:0050780 (17) <br> dopamine receptor binding <br> 1.569e-02","GO:0001618 (76) <br> virus receptor activity <br> 1.875e-02","GO:0008556 (10) <br> P-type potassium transmembrane transporter activity <br> 1.977e-02","GO:0003823 (163) <br> antigen binding <br> 2.032e-02","GO:0140272 (77) <br> exogenous protein binding <br> 2.041e-02","GO:0032794 (15) <br> GTPase activating protein binding <br> 2.056e-02","GO:0019904 (706) <br> protein domain specific binding <br> 2.737e-02","GO:0019901 (677) <br> protein kinase binding <br> 2.969e-02","GO:0098772 (1115) <br> molecular function regulator <br> 2.971e-02","GO:0000146 (28) <br> microfilament motor activity <br> 3.090e-02","GO:0043548 (32) <br> phosphatidylinositol 3-kinase binding <br> 3.214e-02"],"key":["GO:0044877","GO:0017111","GO:0003924","GO:0016462","GO:0016818","GO:0016817","GO:0008092","GO:0003743","GO:0019003","GO:0008135","GO:0050839","GO:0005525","GO:0003779","GO:0045182","GO:0019001","GO:0032561","GO:0005515","GO:0090079","GO:0045296","GO:0031683","GO:0005102","GO:0003925","GO:0042610","GO:0051015","GO:0042608","GO:0042605","GO:0097367","GO:0016787","GO:0019899","GO:0017076","GO:0032553","GO:0031369","GO:0035639","GO:0032555","GO:1901265","GO:0000166","GO:0035254","GO:0036094","GO:0033218","GO:0030507","GO:0042277","GO:0042626","GO:0015399","GO:0031821","GO:0003723","GO:0046977","GO:1990239","GO:0019900","GO:0060090","GO:0043168","GO:0019829","GO:0005391","GO:0008554","GO:0042609","GO:0005200","GO:0050780","GO:0001618","GO:0008556","GO:0003823","GO:0140272","GO:0032794","GO:0019904","GO:0019901","GO:0098772","GO:0000146","GO:0043548"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[19.8957455135249,18.4322454425848,17.9511812971334,18.6065659634108,18.6370346939429,18.6397677301037,19.5634860592025,15.3970834943308,15.7873208982415,16.2854648740137,18.7692301298177,18.2852512758846,18.5103934714585,16.9745252158235,18.3591888413143,18.3591888413143,22.6742247680722,16.6286063334144,18.0980022367888,13.3863990273633,20.1100273733119,14.3213782810468,9.05045292005097,17.4582741483727,12.0456958427095,14.1979626749156,20.577234061457,20.6157708154208,20.4476570765472,20.3774023473002,20.3786515923268,14.4360075836772,20.3223306568802,20.3679885571932,20.5209404994456,20.5203904843139,15.0659099511043,20.6932989274541,18.3068230037595,13.9932800960407,18.006537444193,16.5900349098891,16.6286063334144,11.5439512898152,20.3892145517334,11.5439512898152,10.4153745295623,19.2046630030309,18.1311916030541,20.6436983522132,15.2731516632256,11.2337416908317,11.2337416908317,11.5439512898152,16.3641747633204,13.0411641506074,15.8543507531641,11.8112973047217,17.0605109063983,15.8760274247573,12.766247544801,19.1137758499688,19.0588592513543,19.6998408459147,14.064391440917,14.3213782810468],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[198.094889825427,193.394118699624,193.957760321422,191.962468980254,190.91785317452,191.08694566106,190.583425812253,199.147020852785,191.301129477343,197.756704852347,199.353689447445,191.229734871916,199.143263241973,193.935214656551,196.783483652041,198.838896766201,193.288905596888,195.975597327463,193.875092883559,193.457998083427,194.382370343178,191.079430439436,190.579668201441,194.014124483602,193.597029683471,193.544423132103,191.613011174739,190.962944504264,190.835185736656,194.01036687279,197.542521036064,190.985490169136,191.755800385595,196.790998873665,196.024446268019,199.849694074628,194.14188325121,194.134368029586,193.938972267363,195.904202722035,195.002376127157,194.138125640398,194.06673103497,193.281390375264,193.942729878175,193.950245099799,196.715846657425,196.61063355469,194.431219283734,196.170993089686,196.23487247349,197.429792711704,196.896211976401,199.624237425908,193.946487488987,194.773161867625,195.194014278568,191.458949131447,199.349931836633,196.899969587213,194.713040094633,195.314257824552,198.034768052435,190.850216179904,194.570250883777,192.022590753246,193.95400271061,190.831428125844,194.619099824333,195.776443954427,191.695678612603,194.679221597325,192.139076688418,190.906580342084,191.128279379992,195.907960332847,198.49695418231,193.961517932234,195.998142992335,196.013173435583,199.135748020349,194.930981521729,191.582950288243,196.31002468973,198.444347630942,191.988772255938,197.486156873884,197.426035100892,195.182741446132,196.396449738406,197.565066700936,193.833759164627,199.541569988044,191.21846203948,199.537812377232,199.706904863772,199.883512571936,199.879754961124,192.052651639742,190.575910590629,195.949294051779,197.358398106276,199.635510258344,199.316113339325,194.912193467669,199.868482128688,198.267739922779,198.489438960686,191.195916374608,190.673608471741,197.44106554414,193.371573034752,193.044660894108,197.813069014527,193.608302515907,196.561784614134,194.765646646001,194.397400786426,196.809786927725,193.04841850492,198.568348787738,198.925321814877,195.464562257032,194.393643175614,191.504040461191,190.568395369005,197.839372290211,193.055933726544,194.032912537662,197.809311403715,193.059691337356,192.31192678577,194.769404256813,193.920184213303,194.600311770273,191.214704428668,197.5537938685,192.04889402893,194.36733989993,197.820584236151,199.875997350312,199.8722397395,196.219842030242,194.359824678306,196.621906387126,191.267310980035],"y":[8.55711169891288,8.24679346469871,7.42140225501224,7.38796449388245,7.17589802912,7.03668399353811,6.58924371508946,6.28826578943441,6.07158649499264,5.81755559694435,5.81755559694435,5.57066022971536,5.42995597148062,5.40138502920778,5.30908203451561,5.20521604693927,5.09446865580363,5.08520903431659,4.83496305313588,4.73000428057976,4.68294787819838,4.63567106309356,4.62935148013092,4.60865224830025,4.49353749181963,4.46175518841791,4.46175518841791,4.39521678674199,4.37999357714458,4.37999357714458,4.30253122431336,4.19499346628655,4.13767904619232,3.87032330916993,3.8664276665673,3.86243881174653,3.74247383516524,3.74247383516524,3.7420055078604,3.62474465799079,3.60432117270713,3.58435496308802,3.57466025036449,3.53667145961752,3.4906567466958,3.4906567466958,3.37566459564965,3.3614925258885,3.30829478492083,3.22356111966336,3.20767842080867,3.170966116869,3.0658045519034,3.03336651824114,2.97129240590581,2.94721433765837,2.94487790728782,2.91843070175744,2.91143626552976,2.90220011402242,2.89692391381836,2.89189063013287,2.88716525986754,2.87998531294919,2.80362359497169,2.79050471357668,2.63439827165149,2.63439827165149,2.63439827165149,2.614567910084,2.60079386564447,2.56258502922831,2.55861273949075,2.55832047379048,2.53811557240505,2.49819563527389,2.45728381251619,2.41435155343742,2.25051402442563,2.24253516090752,2.21938175590982,2.21627323585848,2.2151831663057,2.14497101079762,2.12774844648207,2.11548480092039,2.11548480092039,2.10003651922464,2.09839823538183,2.07786111724591,2.05749046686232,2.02509756882974,2.00082172935012,2.00082172935012,2.00082172935012,2.00082172935012,2.00082172935012,2.00082172935012,1.98094723408043,1.95876858331031,1.94542470603605,1.94542470603605,1.94542470603605,1.94352355421933,1.92376813482711,1.91497213285571,1.91352682803519,1.91352682803519,1.90076839443347,1.89124873646309,1.89124873646309,1.84972677878382,1.83824612980612,1.83824612980612,1.83824612980612,1.81710848375646,1.78849804272491,1.7863719368627,1.75729349925993,1.73558374354519,1.73558374354519,1.7284234058721,1.68584148348286,1.6371072676478,1.6371072676478,1.62673153312965,1.61503786584273,1.58934522917959,1.58934522917959,1.58934522917959,1.58934522917959,1.54252138180841,1.459619026683,1.45728989299336,1.43071471157249,1.40736469090559,1.40257952166641,1.37126474899522,1.36398705098241,1.33004215825284,1.32140171234139,1.32140171234139,1.32140171234139,1.31256249913854,1.3100418888801,1.30518682075876],"text":["REAC:R-HSA-392518 (32) <br> Signal amplification <br> 2.773e-09","REAC:R-HSA-1236974 (83) <br> ER-Phagosome pathway <br> 5.665e-09","REAC:R-HSA-202040 (28) <br> G-protein activation <br> 3.790e-08","REAC:R-HSA-6814122 (42) <br> Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding <br> 4.093e-08","REAC:R-HSA-72662 (60) <br> Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S <br> 6.670e-08","REAC:R-HSA-1236975 (99) <br> Antigen processing-Cross presentation <br> 9.190e-08","REAC:R-HSA-392170 (22) <br> ADP signalling through P2Y purinoceptor 12 <br> 2.575e-07","REAC:R-HSA-428930 (24) <br> Thromboxane signalling through TP receptor <br> 5.149e-07","REAC:R-HSA-3858494 (143) <br> Beta-catenin independent WNT signaling <br> 8.480e-07","REAC:R-HSA-72702 (59) <br> Ribosomal scanning and start codon recognition <br> 1.522e-06","REAC:R-HSA-72649 (59) <br> Translation initiation complex formation <br> 1.522e-06","REAC:R-HSA-422475 (549) <br> Axon guidance <br> 2.687e-06","REAC:R-HSA-456926 (31) <br> Thrombin signalling through proteinase activated receptors (PARs) <br> 3.716e-06","REAC:R-HSA-418597 (48) <br> G alpha (z) signalling events <br> 3.968e-06","REAC:R-HSA-392851 (19) <br> Prostacyclin signalling through prostacyclin receptor <br> 4.908e-06","REAC:R-HSA-202403 (120) <br> TCR signaling <br> 6.234e-06","REAC:R-HSA-1168372 (80) <br> Downstream signaling events of B Cell Receptor (BCR) <br> 8.045e-06","REAC:R-HSA-9675108 (574) <br> Nervous system development <br> 8.218e-06","REAC:R-HSA-72695 (52) <br> Formation of the ternary complex, and subsequently, the 43S complex <br> 1.462e-05","REAC:R-HSA-1236977 (11) <br> Endosomal/Vacuolar pathway <br> 1.862e-05","REAC:R-HSA-109582 (673) <br> Hemostasis <br> 2.075e-05","REAC:R-HSA-983170 (25) <br> Antigen Presentation: Folding, assembly and peptide loading of class I MHC <br> 2.314e-05","REAC:R-HSA-418592 (24) <br> ADP signalling through P2Y purinoceptor 1 <br> 2.348e-05","REAC:R-HSA-977443 (60) <br> GABA receptor activation <br> 2.462e-05","REAC:R-HSA-9009391 (73) <br> Extra-nuclear estrogen signaling <br> 3.210e-05","REAC:R-HSA-72613 (120) <br> Eukaryotic Translation Initiation <br> 3.453e-05","REAC:R-HSA-72737 (120) <br> Cap-dependent Translation Initiation <br> 3.453e-05","REAC:R-HSA-1280218 (799) <br> Adaptive Immune System <br> 4.025e-05","REAC:R-HSA-991365 (43) <br> Activation of GABAB receptors <br> 4.169e-05","REAC:R-HSA-977444 (43) <br> GABA B receptor activation <br> 4.169e-05","REAC:R-HSA-422356 (78) <br> Regulation of insulin secretion <br> 4.983e-05","REAC:R-HSA-400042 (28) <br> Adrenaline,noradrenaline inhibits insulin secretion <br> 6.383e-05","REAC:R-HSA-390466 (95) <br> Chaperonin-mediated protein folding <br> 7.283e-05","REAC:R-HSA-391251 (102) <br> Protein folding <br> 1.348e-04","REAC:R-HSA-6798695 (476) <br> Neutrophil degranulation <br> 1.360e-04","REAC:R-HSA-166208 (24) <br> mTORC1-mediated signalling <br> 1.373e-04","REAC:R-HSA-420092 (33) <br> Glucagon-type ligand receptors <br> 1.809e-04","REAC:R-HSA-163359 (33) <br> Glucagon signaling in metabolic regulation <br> 1.809e-04","REAC:R-HSA-8964315 (18) <br> G beta:gamma signalling through BTK <br> 1.811e-04","REAC:R-HSA-164940 (9) <br> Nef mediated downregulation of MHC class I complex cell surface expression <br> 2.373e-04","REAC:R-HSA-156827 (112) <br> L13a-mediated translational silencing of Ceruloplasmin expression <br> 2.487e-04","REAC:R-HSA-381676 (42) <br> Glucagon-like Peptide-1 (GLP1) regulates insulin secretion <br> 2.604e-04","REAC:R-HSA-72706 (113) <br> GTP hydrolysis and joining of the 60S ribosomal subunit <br> 2.663e-04","REAC:R-HSA-202424 (98) <br> Downstream TCR signaling <br> 2.906e-04","REAC:R-HSA-8964616 (20) <br> G beta:gamma signalling through CDC42 <br> 3.231e-04","REAC:R-HSA-418217 (20) <br> G beta:gamma signalling through PLC beta <br> 3.231e-04","REAC:R-HSA-500657 (21) <br> Presynaptic function of Kainate receptors <br> 4.211e-04","REAC:R-HSA-76002 (258) <br> Platelet activation, signaling and aggregation <br> 4.350e-04","REAC:R-HSA-162909 (125) <br> Host Interactions of HIV factors <br> 4.917e-04","REAC:R-HSA-111885 (90) <br> Opioid Signalling <br> 5.976e-04","REAC:R-HSA-4086400 (92) <br> PCP/CE pathway <br> 6.199e-04","REAC:R-HSA-5658442 (67) <br> Regulation of RAS by GAPs <br> 6.746e-04","REAC:R-HSA-5673001 (264) <br> RAF/MAP kinase cascade <br> 8.594e-04","REAC:R-HSA-432040 (43) <br> Vasopressin regulates renal water homeostasis via Aquaporins <br> 9.260e-04","REAC:R-HSA-392451 (25) <br> G beta:gamma signalling through PI3Kgamma <br> 1.068e-03","REAC:R-HSA-877300 (87) <br> Interferon gamma signaling <br> 1.129e-03","REAC:R-HSA-5684996 (269) <br> MAPK1/MAPK3 signaling <br> 1.135e-03","REAC:R-HSA-5621481 (136) <br> C-type lectin receptors (CLRs) <br> 1.207e-03","REAC:R-HSA-72766 (292) <br> Translation <br> 1.226e-03","REAC:R-HSA-9649913 (3) <br> RAS GTPase cycle mutants <br> 1.253e-03","REAC:R-HSA-163685 (107) <br> Integration of energy metabolism <br> 1.268e-03","REAC:R-HSA-165159 (41) <br> MTOR signalling <br> 1.283e-03","REAC:R-HSA-373755 (64) <br> Semaphorin interactions <br> 1.297e-03","REAC:R-HSA-1169091 (67) <br> Activation of NF-kappaB in B cells <br> 1.318e-03","REAC:R-HSA-168256 (2038) <br> Immune System <br> 1.572e-03","REAC:R-HSA-1280215 (696) <br> Cytokine Signaling in Immune system <br> 1.620e-03","REAC:R-HSA-1296059 (29) <br> G protein gated Potassium channels <br> 2.321e-03","REAC:R-HSA-1296041 (29) <br> Activation of G protein gated Potassium channels <br> 2.321e-03","REAC:R-HSA-997272 (29) <br> Inhibition  of voltage gated Ca2+ channels via Gbeta/gamma subunits <br> 2.321e-03","REAC:R-HSA-5676590 (59) <br> NIK-->noncanonical NF-kB signaling <br> 2.429e-03","REAC:R-HSA-202733 (190) <br> Cell surface interactions at the vascular wall <br> 2.507e-03","REAC:R-HSA-168249 (1090) <br> Innate Immune System <br> 2.738e-03","REAC:R-HSA-5607761 (60) <br> Dectin-1 mediated noncanonical NF-kB signaling <br> 2.763e-03","REAC:R-HSA-451326 (30) <br> Activation of kainate receptors upon glutamate binding <br> 2.765e-03","REAC:R-HSA-445717 (52) <br> Aquaporin-mediated transport <br> 2.897e-03","REAC:R-HSA-164938 (20) <br> Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters <br> 3.175e-03","REAC:R-HSA-983705 (163) <br> Signaling by the B Cell Receptor (BCR) <br> 3.489e-03","REAC:R-HSA-397795 (32) <br> G-protein beta:gamma signalling <br> 3.852e-03","REAC:R-HSA-112316 (400) <br> Neuronal System <br> 5.617e-03","REAC:R-HSA-112314 (196) <br> Neurotransmitter receptors and postsynaptic signal transmission <br> 5.721e-03","REAC:R-HSA-164952 (26) <br> The role of Nef in HIV-1 replication and disease pathogenesis <br> 6.034e-03","REAC:R-HSA-1296065 (35) <br> Inwardly rectifying K+ channels <br> 6.078e-03","REAC:R-HSA-4086398 (59) <br> Ca2+ pathway <br> 6.093e-03","REAC:R-HSA-1257604 (252) <br> PIP3 activates AKT signaling <br> 7.162e-03","REAC:R-HSA-195721 (328) <br> Signaling by WNT <br> 7.452e-03","REAC:R-HSA-1236978 (50) <br> Cross-presentation of soluble exogenous antigens (endosomes) <br> 7.665e-03","REAC:R-HSA-211733 (50) <br> Regulation of activated PAK-2p34 by proteasome mediated degradation <br> 7.665e-03","REAC:R-HSA-8948751 (69) <br> Regulation of PTEN stability and activity <br> 7.943e-03","REAC:R-HSA-5683057 (308) <br> MAPK family signaling cascades <br> 7.973e-03","REAC:R-HSA-6807070 (138) <br> PTEN Regulation <br> 8.359e-03","REAC:R-HSA-350562 (51) <br> Regulation of ornithine decarboxylase (ODC) <br> 8.760e-03","REAC:R-HSA-72689 (102) <br> Formation of a pool of free 40S subunits <br> 9.438e-03","REAC:R-HSA-75815 (52) <br> Ubiquitin-dependent degradation of Cyclin D <br> 9.981e-03","REAC:R-HSA-349425 (52) <br> Autodegradation of the E3 ubiquitin ligase COP1 <br> 9.981e-03","REAC:R-HSA-69601 (52) <br> Ubiquitin Mediated Degradation of Phosphorylated Cdc25A <br> 9.981e-03","REAC:R-HSA-180534 (52) <br> Vpu mediated degradation of CD4 <br> 9.981e-03","REAC:R-HSA-69613 (52) <br> p53-Independent G1/S DNA damage checkpoint <br> 9.981e-03","REAC:R-HSA-69610 (52) <br> p53-Independent DNA Damage Response <br> 9.981e-03","REAC:R-HSA-2424491 (25) <br> DAP12 signaling <br> 1.045e-02","REAC:R-HSA-9660821 (133) <br> ADORA2B mediated anti-inflammatory cytokines production <br> 1.100e-02","REAC:R-HSA-9604323 (53) <br> Negative regulation of NOTCH4 signaling <br> 1.134e-02","REAC:R-HSA-169911 (53) <br> Regulation of Apoptosis <br> 1.134e-02","REAC:R-HSA-180585 (53) <br> Vif-mediated degradation of APOBEC3G <br> 1.134e-02","REAC:R-HSA-8878159 (94) <br> Transcriptional regulation by RUNX3 <br> 1.139e-02","REAC:R-HSA-9006925 (292) <br> Intracellular signaling by second messengers <br> 1.192e-02","REAC:R-HSA-171007 (11) <br> p38MAPK events <br> 1.216e-02","REAC:R-HSA-9669938 (17) <br> Signaling by KIT in disease <br> 1.220e-02","REAC:R-HSA-9670439 (17) <br> Signaling by phosphorylated juxtamembrane, extracellular and kinase domain KIT mutants <br> 1.220e-02","REAC:R-HSA-4608870 (64) <br> Asymmetric localization of PCP proteins <br> 1.257e-02","REAC:R-HSA-450408 (54) <br> AUF1 (hnRNP D0) binds and destabilizes mRNA <br> 1.285e-02","REAC:R-HSA-8941858 (54) <br> Regulation of RUNX3 expression and activity <br> 1.285e-02","REAC:R-HSA-2682334 (90) <br> EPH-Ephrin signaling <br> 1.413e-02","REAC:R-HSA-4641257 (55) <br> Degradation of AXIN <br> 1.451e-02","REAC:R-HSA-174113 (55) <br> SCF-beta-TrCP mediated degradation of Emi1 <br> 1.451e-02","REAC:R-HSA-8854050 (55) <br> FBXL7 down-regulates AURKA during mitotic entry and in early mitosis <br> 1.451e-02","REAC:R-HSA-202427 (23) <br> Phosphorylation of CD3 and TCR zeta chains <br> 1.524e-02","REAC:R-HSA-913531 (193) <br> Interferon Signaling <br> 1.627e-02","REAC:R-HSA-5362768 (56) <br> Hh mutants are degraded by ERAD <br> 1.635e-02","REAC:R-HSA-6794362 (86) <br> Protein-protein interactions at synapses <br> 1.749e-02","REAC:R-HSA-4641258 (57) <br> Degradation of DVL <br> 1.838e-02","REAC:R-HSA-69541 (57) <br> Stabilization of p53 <br> 1.838e-02","REAC:R-HSA-5668541 (100) <br> TNFR2 non-canonical NF-kB pathway <br> 1.869e-02","REAC:R-HSA-351202 (58) <br> Metabolism of polyamines <br> 2.061e-02","REAC:R-HSA-5387390 (59) <br> Hh mutants abrogate ligand secretion <br> 2.306e-02","REAC:R-HSA-68827 (59) <br> CDT1 association with the CDC6:ORC:origin complex <br> 2.306e-02","REAC:R-HSA-382556 (103) <br> ABC-family proteins mediated transport <br> 2.362e-02","REAC:R-HSA-2428933 (6) <br> SHC-related events triggered by IGF1R <br> 2.426e-02","REAC:R-HSA-5610780 (60) <br> Degradation of GLI1 by the proteasome <br> 2.574e-02","REAC:R-HSA-5610785 (60) <br> GLI3 is processed to GLI3R by the proteasome <br> 2.574e-02","REAC:R-HSA-187577 (60) <br> SCF(Skp2)-mediated degradation of p27/p21 <br> 2.574e-02","REAC:R-HSA-5610783 (60) <br> Degradation of GLI2 by the proteasome <br> 2.574e-02","REAC:R-HSA-5678895 (61) <br> Defective CFTR causes cystic fibrosis <br> 2.867e-02","REAC:R-HSA-909733 (68) <br> Interferon alpha/beta signaling <br> 3.470e-02","REAC:R-HSA-416482 (80) <br> G alpha (12/13) signalling events <br> 3.489e-02","REAC:R-HSA-5663205 (909) <br> Infectious disease <br> 3.709e-02","REAC:R-HSA-174084 (64) <br> Autodegradation of Cdh1 by Cdh1:APC/C <br> 3.914e-02","REAC:R-HSA-450531 (86) <br> Regulation of mRNA stability by proteins that bind AU-rich elements <br> 3.957e-02","REAC:R-HSA-2172127 (33) <br> DAP12 interactions <br> 4.253e-02","REAC:R-HSA-5358346 (65) <br> Hedgehog ligand biogenesis <br> 4.325e-02","REAC:R-HSA-399955 (14) <br> SEMA3A-Plexin repulsion signaling by inhibiting Integrin adhesion <br> 4.677e-02","REAC:R-HSA-69580 (66) <br> p53-Dependent G1/S DNA damage checkpoint <br> 4.771e-02","REAC:R-HSA-69563 (66) <br> p53-Dependent G1 DNA Damage Response <br> 4.771e-02","REAC:R-HSA-1234176 (66) <br> Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha <br> 4.771e-02","REAC:R-HSA-5610787 (113) <br> Hedgehog 'off' state <br> 4.869e-02","REAC:R-HSA-418346 (85) <br> Platelet homeostasis <br> 4.897e-02","REAC:R-HSA-210991 (25) <br> Basigin interactions <br> 4.952e-02"],"key":["REAC:R-HSA-392518","REAC:R-HSA-1236974","REAC:R-HSA-202040","REAC:R-HSA-6814122","REAC:R-HSA-72662","REAC:R-HSA-1236975","REAC:R-HSA-392170","REAC:R-HSA-428930","REAC:R-HSA-3858494","REAC:R-HSA-72702","REAC:R-HSA-72649","REAC:R-HSA-422475","REAC:R-HSA-456926","REAC:R-HSA-418597","REAC:R-HSA-392851","REAC:R-HSA-202403","REAC:R-HSA-1168372","REAC:R-HSA-9675108","REAC:R-HSA-72695","REAC:R-HSA-1236977","REAC:R-HSA-109582","REAC:R-HSA-983170","REAC:R-HSA-418592","REAC:R-HSA-977443","REAC:R-HSA-9009391","REAC:R-HSA-72613","REAC:R-HSA-72737","REAC:R-HSA-1280218","REAC:R-HSA-991365","REAC:R-HSA-977444","REAC:R-HSA-422356","REAC:R-HSA-400042","REAC:R-HSA-390466","REAC:R-HSA-391251","REAC:R-HSA-6798695","REAC:R-HSA-166208","REAC:R-HSA-420092","REAC:R-HSA-163359","REAC:R-HSA-8964315","REAC:R-HSA-164940","REAC:R-HSA-156827","REAC:R-HSA-381676","REAC:R-HSA-72706","REAC:R-HSA-202424","REAC:R-HSA-8964616","REAC:R-HSA-418217","REAC:R-HSA-500657","REAC:R-HSA-76002","REAC:R-HSA-162909","REAC:R-HSA-111885","REAC:R-HSA-4086400","REAC:R-HSA-5658442","REAC:R-HSA-5673001","REAC:R-HSA-432040","REAC:R-HSA-392451","REAC:R-HSA-877300","REAC:R-HSA-5684996","REAC:R-HSA-5621481","REAC:R-HSA-72766","REAC:R-HSA-9649913","REAC:R-HSA-163685","REAC:R-HSA-165159","REAC:R-HSA-373755","REAC:R-HSA-1169091","REAC:R-HSA-168256","REAC:R-HSA-1280215","REAC:R-HSA-1296059","REAC:R-HSA-1296041","REAC:R-HSA-997272","REAC:R-HSA-5676590","REAC:R-HSA-202733","REAC:R-HSA-168249","REAC:R-HSA-5607761","REAC:R-HSA-451326","REAC:R-HSA-445717","REAC:R-HSA-164938","REAC:R-HSA-983705","REAC:R-HSA-397795","REAC:R-HSA-112316","REAC:R-HSA-112314","REAC:R-HSA-164952","REAC:R-HSA-1296065","REAC:R-HSA-4086398","REAC:R-HSA-1257604","REAC:R-HSA-195721","REAC:R-HSA-1236978","REAC:R-HSA-211733","REAC:R-HSA-8948751","REAC:R-HSA-5683057","REAC:R-HSA-6807070","REAC:R-HSA-350562","REAC:R-HSA-72689","REAC:R-HSA-75815","REAC:R-HSA-349425","REAC:R-HSA-69601","REAC:R-HSA-180534","REAC:R-HSA-69613","REAC:R-HSA-69610","REAC:R-HSA-2424491","REAC:R-HSA-9660821","REAC:R-HSA-9604323","REAC:R-HSA-169911","REAC:R-HSA-180585","REAC:R-HSA-8878159","REAC:R-HSA-9006925","REAC:R-HSA-171007","REAC:R-HSA-9669938","REAC:R-HSA-9670439","REAC:R-HSA-4608870","REAC:R-HSA-450408","REAC:R-HSA-8941858","REAC:R-HSA-2682334","REAC:R-HSA-4641257","REAC:R-HSA-174113","REAC:R-HSA-8854050","REAC:R-HSA-202427","REAC:R-HSA-913531","REAC:R-HSA-5362768","REAC:R-HSA-6794362","REAC:R-HSA-4641258","REAC:R-HSA-69541","REAC:R-HSA-5668541","REAC:R-HSA-351202","REAC:R-HSA-5387390","REAC:R-HSA-68827","REAC:R-HSA-382556","REAC:R-HSA-2428933","REAC:R-HSA-5610780","REAC:R-HSA-5610785","REAC:R-HSA-187577","REAC:R-HSA-5610783","REAC:R-HSA-5678895","REAC:R-HSA-909733","REAC:R-HSA-416482","REAC:R-HSA-5663205","REAC:R-HSA-174084","REAC:R-HSA-450531","REAC:R-HSA-2172127","REAC:R-HSA-5358346","REAC:R-HSA-399955","REAC:R-HSA-69580","REAC:R-HSA-69563","REAC:R-HSA-1234176","REAC:R-HSA-5610787","REAC:R-HSA-418346","REAC:R-HSA-210991"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,102,204,1)","opacity":0.8,"size":[14.3213782810468,15.9997101118596,14.064391440917,14.8262572073822,15.45541914919,16.2854648740137,13.5832101454551,13.7594961631324,16.8614836447151,15.426535038788,15.426535038788,18.7814574429451,14.2608475416207,15.0659099511043,13.2787931541743,16.5900349098891,15.9391857668738,18.8408284790331,15.2071559698249,12.0456958427095,19.0510845822719,13.8411375859653,13.7594961631324,15.45541914919,15.7873208982415,16.5900349098891,16.5900349098891,19.2746601311804,14.8688640092558,14.8688640092558,15.8973863612294,14.064391440917,16.2191968510427,16.3332135329498,18.5896251814072,13.7594961631324,14.3797163833018,14.3797163833018,13.1640323425788,11.5439512898152,16.481642870719,14.8262572073822,16.4956598039246,16.2691849528541,13.3863990273633,13.3863990273633,13.4876469321386,17.7362643582832,16.6537390526195,16.1317846515619,16.1673930557028,15.6433129553031,17.7692368338489,14.8688640092558,13.8411375859653,16.0766579870883,17.7960889502372,16.7843679978129,17.912906815234,7.81088103584182,16.409380751435,14.7824526398553,15.5656749705695,15.6433129553031,20.4417708008214,19.0951202425796,14.1325441121217,14.1325441121217,14.1325441121217,15.426535038788,17.2898209035835,19.6712691137704,15.45541914919,14.1979626749156,15.2071559698249,13.3863990273633,17.0605109063983,14.3213782810468,18.3523314408885,17.3358617234104,13.9189573720891,14.4903838270411,15.426535038788,17.7024351253793,18.0768113156737,15.1381652632347,15.1381652632347,15.6928968665451,17.9883493723974,16.8068458716563,15.1730510425948,16.3332135329498,15.2071559698249,15.2071559698249,15.2071559698249,15.2071559698249,15.2071559698249,15.2071559698249,13.8411375859653,16.7499484371089,15.2405126341149,15.2405126341149,15.2405126341149,16.2021368290015,17.912906815234,12.0456958427095,13.0411641506074,13.0411641506074,15.5656749705695,15.2731516632256,15.2731516632256,16.1317846515619,15.3051018762464,15.3051018762464,15.3051018762464,13.6736623489524,17.3130398134153,15.3363904218573,16.0578025627736,15.3670429037096,15.3670429037096,16.3015603275795,15.3970834943308,15.426535038788,15.426535038788,16.3487788308752,10.4153745295623,15.45541914919,15.45541914919,15.45541914919,15.45541914919,15.4837562909781,15.6683130595324,15.9391857668738,19.4405959398567,15.5656749705695,16.0578025627736,14.3797163833018,15.592008586327,12.6110489969071,15.6178829039023,15.6178829039023,15.6178829039023,16.4956598039246,16.0386970905277,13.8411375859653],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,102,204,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"REAC","legendgroup":"REAC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"19fc13662a8e":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"19fc1adb4e87":{"x":{},"xend":{},"y":{},"yend":{}},"19fc632260a5":{"x":{},"xend":{},"y":{},"yend":{}},"19fc5a238c":{"x":{},"xend":{},"y":{},"yend":{}},"19fc38526ebf":{"x":{},"xend":{},"y":{},"yend":{}},"19fc35b073c0":{"x":{},"y":{}},"19fc1321d23":{"yintercept":{}}},"cur_data":"19fc13662a8e","visdat":{"19fc13662a8e":["function (y) ","x"],"19fc1adb4e87":["function (y) ","x"],"19fc632260a5":["function (y) ","x"],"19fc5a238c":["function (y) ","x"],"19fc38526ebf":["function (y) ","x"],"19fc35b073c0":["function (y) ","x"],"19fc1321d23":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData2a0dd1ba"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;%
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-e4a421697d58abce2053" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-e4a421697d58abce2053">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546","547","548","549","550","551","552","553","554","555","556","557","558","559","560","561","562","563","564","565","566","567","568","569","570","571","572","573","574","575","576","577","578","579","580","581","582","583","584","585","586","587","588","589","590","591","592","593","594","595","596","597","598","599","600","601","602","603","604","605","606","607","608","609","610","611","612","613","614","615","616","617","618","619","620","621","622","623","624","625","626","627","628","629","630","631","632","633","634","635","636","637","638","639","640","641","642","643","644","645","646","647","648","649","650","651","652","653","654","655","656","657","658","659","660","661","662","663","664","665","666","667","668","669","670","671","672","673","674","675","676","677","678","679","680","681","682","683","684","685","686","687","688","689","690","691","692","693","694","695","696","697","698","699","700","701","702","703","704","705","706","707","708","709","710","711","712","713","714","715","716","717","718","719"],["GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:MF","GO:MF","REAC","GO:MF","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","REAC","REAC","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:BP","GO:CC","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:MF","REAC","REAC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","REAC","GO:MF","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","REAC","REAC","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:CC","GO:MF","GO:MF","REAC","REAC","REAC","GO:CC","GO:BP","GO:BP","GO:CC","REAC","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:BP","REAC","GO:MF","GO:BP","GO:MF","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","REAC","REAC","REAC","GO:BP","GO:BP","REAC","REAC","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","REAC","GO:CC","GO:CC","REAC","REAC","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:CC","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:CC","REAC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","REAC","GO:MF","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC","REAC","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:CC","REAC","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:MF","REAC","REAC","GO:CC","GO:MF","REAC","REAC","GO:BP","REAC","REAC","REAC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","REAC","REAC","REAC","REAC","GO:MF","REAC","GO:BP","GO:CC","GO:MF","REAC","REAC","REAC","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:BP","GO:MF","GO:BP","REAC","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:MF","REAC","REAC","GO:CC","GO:BP","REAC","REAC","REAC","GO:BP","GO:MF","GO:MF","GO:BP","GO:MF","GO:BP","REAC","GO:CC","GO:BP","REAC","GO:MF","GO:BP","GO:BP","REAC","REAC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","REAC","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","REAC","REAC","REAC","REAC","REAC","REAC","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","REAC","GO:CC","GO:BP","GO:BP","GO:MF","REAC","REAC","REAC","REAC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","REAC","REAC","REAC","GO:BP","GO:CC","GO:MF","GO:MF","GO:MF","REAC","GO:CC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","REAC","GO:BP","REAC","REAC","REAC","GO:CC","GO:BP","REAC","GO:MF","GO:BP","REAC","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","REAC","GO:CC","REAC","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:MF","GO:MF","GO:MF","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","REAC","REAC","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:BP","REAC","GO:CC","GO:CC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","REAC","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","REAC","GO:BP","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","REAC","REAC","REAC","GO:CC","REAC","GO:BP","REAC","GO:CC","REAC","GO:CC"],[2743,4056,2131,2719,1453,278,241,515,550,1027,3132,3551,7129,2765,1026,850,1322,3497,2779,561,15738,15694,2437,3658,5762,1038,3628,13259,385,1118,4087,474,470,23108,7017,2843,14485,3561,3583,14527,785,9160,1133,3982,3627,7019,2029,2552,5659,1570,2901,598,13142,1125,1332,1247,3547,3582,3542,983,3072,3785,3784,15417,15359,1648,1959,861,12268,16046,1261,1134,1080,3780,818,939,861,489,1030,927,1099,2921,11868,916,16072,1129,9110,531,1098,1334,10009,3860,9109,19079,11574,9231,450,837,1412,15361,4447,1084,1401,1408,16054,2093,10980,984,410,3654,12281,418,1993,947,15428,9081,29092,20427,22060,1889,2879,4320,19167,1101,11579,2009,468,7162,758,4445,5413,817,938,14309,26828,19397,190,1879,19815,26826,3068,10208,7625,2127,2055,14941,3552,3071,1078,1138,14885,5665,1262,917,17166,27326,2895,908,377,791,2128,7174,7740,3079,99,15586,26123,788,26125,5837,1217,15416,7607,18664,144,2056,2496,7698,11494,7605,889,29136,29126,3575,5104,409,67,2041,14889,886,2444,12668,1402,10,2566,15587,7018,11657,7705,7671,1606,27328,7622,479,417,599,3067,13553,2289,413,2447,28343,28341,19740,20429,862,1263,193,10495,1025,1135,201,5363,6670,13203,3612,12675,18660,1919,2344,9696,7619,1344,15435,12553,182,3078,9093,16076,15204,6668,1124,6495,4679,2288,885,902,15750,3169,795,1660,13160,19430,1128,2207,819,284,10505,7633,6667,10338,730,1445,13161,3758,1476,3990,3168,15430,19802,859,15459,2416,1349,1149,14940,1265,886,4618,2565,7675,775,11569,9881,18665,1021,1121,142,9,3171,923,13159,19737,26164,1864,812,15368,798,284,72,24935,1075,3959,5405,111,77,922,1117,4805,2898,2130,1862,2896,561,15729,117,534,11210,492,18764,322,5061,2450,6342,784,16110,5407,1474,22049,15456,656,17158,15730,12169,652,7618,4280,26562,2025,27851,1662,1458,2476,1346,10013,957,955,903,863,192,7747,17436,15752,1353,11946,1239,15432,14486,1426,20,1074,1186,956,12824,2946,937,16937,728,1085,1111,904,906,24936,16061,655,15644,658,21250,1141,1139,16045,1642,966,1614,26162,865,3068,1240,4001,1034,35,2300,6278,8304,2443,15367,537,1497,18663,651,1514,5675,1832,23322,1528,4509,1150,2061,7670,1690,438,2416,11587,1073,98,20868,905,6389,1125,1237,532,5468,243,2343,2897,1691,1109,1269,1993,81,3077,532,8306,8761,1355,1071,1333,7020,15318,393,4089,871,6460,2602,654,8060,22821,3624,15822,19045,907,76,1084,1392,4865,306,306,516,6616,1100,424,96,14310,653,155,9162,1427,8298,6675,9338,2116,1238,3178,19502,15371,7699,21383,909,3698,1276,5233,1347,538,19431,25734,20461,3123,2094,4804,184,3391,25686,438,1451,1455,1547,21385,2286,1167,276,10046,7367,12130,7628,4619,12744,1534,2132,8299,2102,9594,558,738,384,1847,1831,1234,3102,2507,11087,8759,1414,1557,2415,9088,9086,8568,6775,1868,185,11679,7021,2862,875,19047,24938,2394,179,2393,2438,2485,2484,8059,401,15370,13703,15074,28000,2781,3136,15765,8,4088,2590,21382,4589,1438,1813,2419,2334,15360,1150,2842,1162,1763,2481,2055,2114,8300,1267,1827,2258,6669,173,514,34,1835,12141,15747,21226,8775,1696,1366,7836,2090,752,1159,665,1934,815,1420,8239,1601,9067,555,1123,14646,1025,1667,13365,25265,1277,1359,1076,11197,2780,666,2135,1084,2230,284,15301,1462,519,1358,19534,2259,531,18950,511,11803,5426,1309,4279,9163,8566,6791,1421,10045,8667,26453,554,1024,255,11547,6,1941,9709,9324,3075,18197,1100,668,928,1933,669,1348,3357,11678,12669,4623,1361,15767,470,2848,11500,24937,4620,10428,11029,12486,2886,1548,30,3663,17818,27693,14363,6838,3659,29230,11707,16097,19465,7749,1124,2542,7837,898,1071,605,1079,4182,11495,1354,29917,967,178,1865,26892,3963,26639,16102,2505,12235,400,1365,1017,1058,606,12488,21333,29347,3470,1356,12269,1936,1025,9308,2483,2482,1510,1964,1015,15583,1617,3185,192,1097],["extracellular exosome","extracellular vesicle","extracellular organelle","extracellular membrane-bounded organelle","vesicle","extracellular space","extracellular region","plasma membrane","focal adhesion","cell-substrate junction","cell periphery","side of membrane","protein-containing complex binding","anchoring junction","cell junction","membrane","cytoplasmic vesicle","intracellular vesicle","transport","cell activation","establishment of localization","localization","membrane raft","membrane microdomain","vesicle-mediated transport","immune system process","plasma membrane protein complex","leukocyte activation","cytoplasm","cytoplasmic vesicle membrane","GTPase complex","heterotrimeric G-protein complex","cytosol","export from cell","antigen processing and presentation","exocytosis","secretion","cytoplasmic side of membrane","plasma membrane region","antigen processing and presentation of peptide antigen","cytoplasmic side of plasma membrane","secretion by cell","antigen processing and presentation of exogenous peptide antigen","nucleoside-triphosphatase activity","membrane protein complex","antigen processing and presentation of exogenous antigen","melanosome","pigment granule","viral process","protein-containing complex","immune response","GTPase activity","regulated exocytosis","secretory granule membrane","regulation of immune system process","extrinsic component of cytoplasmic side of plasma membrane","pyrophosphatase activity","bounding membrane of organelle","protein localization","extrinsic component of plasma membrane","leukocyte cell-cell adhesion","hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides","hydrolase activity, acting on acid anhydrides","T cell receptor signaling pathway","regulation of immune response","eukaryotic 48S preinitiation complex","cytoskeletal protein binding","cytoplasmic translational initiation","leukocyte degranulation","cellular localization","cell leading edge","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","secretory granule","secretory vesicle","vesicle membrane","myeloid cell activation involved in immune response","eukaryotic 43S preinitiation complex","eukaryotic translation initiation factor 3 complex","leukocyte activation involved in immune response","cell activation involved in immune response","myeloid leukocyte mediated immunity","translation preinitiation complex","antigen processing and presentation of exogenous peptide antigen via MHC class I","immune effector process","localization within membrane","antigen processing and presentation of peptide antigen via MHC class I","regulation of protein localization","formation of cytoplasmic translation initiation complex","leukocyte mediated immunity","positive regulation of immune system process","cellular protein localization","plasma membrane bounded cell projection","regulation of localization","cellular macromolecule localization","T cell activation","macromolecule localization","translation initiation factor activity","actin cytoskeleton","immune response-regulating cell surface receptor signaling pathway","positive regulation of immune response","GDP binding","immune response-activating cell surface receptor signaling pathway","immune response-activating signal transduction","immune response-regulating signaling pathway","establishment of localization in cell","cell projection","granulocyte activation","extrinsic component of membrane","lysosomal membrane","lytic vacuole membrane","neutrophil degranulation","vacuolar membrane","translation factor activity, RNA binding","neutrophil activation involved in immune response","regulation of T cell activation","cell adhesion molecule binding","protein localization to cell periphery","protein localization to membrane","cell-cell adhesion","GTP binding","movement of cell or subcellular component","response to organic substance","cellular response to chemical stimulus","neutrophil mediated immunity","neutrophil activation","Signal amplification","actin binding","translation regulator activity","ER-Phagosome pathway","guanyl nucleotide binding","guanyl ribonucleotide binding","endomembrane system","myeloid leukocyte activation","lymphocyte activation","positive regulation of cellular protein localization","cellular response to organic substance","immunological synapse","protein binding","cellular component organization or biogenesis","regulation of cellular protein localization","cell adhesion","translation regulator activity, nucleic acid binding","biological adhesion","organelle","MHC protein complex","positive regulation of response to stimulus","lumenal side of endoplasmic reticulum membrane","integral component of lumenal side of endoplasmic reticulum membrane","endocytic vesicle","antigen processing and presentation of endogenous peptide antigen","positive regulation of biological process","cellular component organization","cell projection membrane","activation of immune response","regulation of cellular localization","regulation of protein localization to cell periphery","tertiary granule","G-protein activation","Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding","cell surface","membrane-bounded organelle","cadherin binding","regulation of cell adhesion","cell surface receptor signaling pathway","Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S","regulation of transport","regulation of leukocyte cell-cell adhesion","basal plasma membrane","positive regulation of leukocyte cell-cell adhesion","cell migration","organelle membrane","antigen receptor-mediated signaling pathway","positive regulation of cell-cell adhesion","regulation of biological quality","Antigen processing-Cross presentation","MHC class I protein complex","phagocytic vesicle","actin filament-based process","locomotion","regulation of cell-cell adhesion","stimulatory C-type lectin receptor signaling pathway","cellular response to lectin","response to lectin","lumenal side of membrane","G-protein beta/gamma-subunit complex binding","lysosome","lytic vacuole","specific granule","positive regulation of cellular process","innate immune response activating cell surface receptor signaling pathway","basal part of cell","cellular component biogenesis","innate immune response-activating signal transduction","ADP signalling through P2Y purinoceptor 12","recycling endosome membrane","positive regulation of transport","antigen processing and presentation of endogenous antigen","response to chemical","actin cytoskeleton organization","signaling","signaling receptor binding","positive regulation of protein localization to cell periphery","cellular component assembly","proteasome core complex","vacuole","G protein activity","cell communication","positive regulation of cell adhesion","Thromboxane signalling through TP receptor","endosome","synapse","positive regulation of protein localization to membrane","regulation of protein localization to membrane","nitrogen compound transport","protein localization to plasma membrane","basolateral plasma membrane","cell trailing edge","uropod","intracellular signal transduction","lamellipodium","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-independent","Beta-catenin independent WNT signaling","protein transport","CD8 receptor binding","establishment of protein localization","regulation of cell shape","positive regulation of molecular function","protein-containing complex assembly","Ribosomal scanning and start codon recognition","Translation initiation complex formation","response to cytokine","regulation of cell morphogenesis","regulation of leukocyte activation","positive regulation of T cell activation","protein-containing complex subunit organization","Axon guidance","signal transduction","actin filament binding","localization of cell","cell motility","T cell receptor binding","endocytic vesicle membrane","cytokine-mediated signaling pathway","proteasomal ubiquitin-independent protein catabolic process","Thrombin signalling through proteinase activated receptors (PARs)","activation of innate immune response","G alpha (z) signalling events","regulation of lymphocyte activation","Ras protein signal transduction","endosome membrane","Prostacyclin signalling through prostacyclin receptor","regulation of innate immune response","cellular response to cytokine stimulus","phagocytic vesicle membrane","TCR signaling","ER to Golgi transport vesicle membrane","intracellular anatomical structure","non-canonical Wnt signaling pathway","ribonucleoprotein complex assembly","peptide antigen binding","carbohydrate derivative binding","Downstream signaling events of B Cell Receptor (BCR)","Nervous system development","positive regulation of innate immune response","hydrolase activity","positive regulation of response to biotic stimulus","response to external stimulus","small GTPase mediated signal transduction","regulation of cell activation","ribonucleoprotein complex subunit organization","cytoplasmic translation","leukocyte migration","translational initiation","positive regulation of immune effector process","cortical cytoskeleton","regulation of response to stimulus","leading edge membrane","Formation of the ternary complex, and subsequently, the 43S complex","enzyme binding","recycling endosome","positive regulation of signaling","Endosomal/Vacuolar pathway","T cell proliferation","response to interferon-gamma","regulation of molecular function","Hemostasis","coated vesicle membrane","Antigen Presentation: Folding, assembly and peptide loading of class I MHC","ADP signalling through P2Y purinoceptor 1","cell-cell signaling","GABA receptor activation","innate immune response","organic substance transport","positive regulation of protein localization to plasma membrane","specific granule membrane","Extra-nuclear estrogen signaling","regulation of catalytic activity","Eukaryotic Translation Initiation","Cap-dependent Translation Initiation","MAPK cascade","response to nitrogen compound","coated vesicle","purine nucleotide binding","ribonucleotide binding","Adaptive Immune System","Activation of GABAB receptors","GABA B receptor activation","transport vesicle membrane","positive regulation of cell communication","defense response","intracellular organelle","Regulation of insulin secretion","tertiary granule membrane","cell cortex","positive regulation of protein transport","Adrenaline,noradrenaline inhibits insulin secretion","morphogenesis of a polarized epithelium","Fc receptor signaling pathway","cytoskeleton","regulation of establishment of protein localization","Chaperonin-mediated protein folding","translation initiation factor binding","protein folding","purine ribonucleoside triphosphate binding","external side of plasma membrane","cellular response to stimulus","purine ribonucleotide binding","regulation of response to biotic stimulus","defense response to other organism","response to stimulus","regulation of T cell mediated cytotoxicity","interferon-gamma-mediated signaling pathway","regulation of protein transport","positive regulation of catalytic activity","regulation of leukocyte mediated cytotoxicity","regulation of anatomical structure morphogenesis","positive regulation of signal transduction","regulation of protein targeting","sarcolemma","positive regulation of establishment of protein localization","Protein folding","Neutrophil degranulation","mTORC1-mediated signalling","positive regulation of leukocyte activation","cellular protein-containing complex assembly","Glucagon-type ligand receptors","Glucagon signaling in metabolic regulation","G beta:gamma signalling through BTK","apical plasma membrane","photoreceptor inner segment","platelet activation","regulation of vesicle-mediated transport","positive regulation of lymphocyte activation","regulation of leukocyte mediated immunity","nucleoside phosphate binding","intrinsic component of plasma membrane","positive regulation of cell activation","intracellular transport","Nef mediated downregulation of MHC class I complex cell surface expression","Golgi membrane","COPII-coated ER to Golgi transport vesicle","L13a-mediated translational silencing of Ceruloplasmin expression","Glucagon-like Peptide-1 (GLP1) regulates insulin secretion","biological process involved in interspecies interaction between organisms","cytoskeleton organization","GTP hydrolysis and joining of the 60S ribosomal subunit","Wnt signaling pathway, planar cell polarity pathway","Downstream TCR signaling","integral component of endoplasmic reticulum membrane","T cell mediated immunity","G beta:gamma signalling through CDC42","G beta:gamma signalling through PLC beta","cellular response to nitrogen compound","establishment of organelle localization","T cell mediated cytotoxicity","regulation of cellular component organization","positive regulation of T cell mediated cytotoxicity","regulation of establishment of planar polarity","antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway, TAP-independent","antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway","organelle localization","Presynaptic function of Kainate receptors","proteasome core complex, alpha-subunit complex","Platelet activation, signaling and aggregation","regulation of protein localization to plasma membrane","lateral plasma membrane","eukaryotic translation initiation factor 3 complex, eIF3m","intrinsic component of endoplasmic reticulum membrane","catalytic complex","Host Interactions of HIV factors","nucleotide binding","cell-cell contact zone","glutamate receptor binding","regulation of defense response","apical part of cell","regulation of biological process","cell-cell junction","Opioid Signalling","biological regulation","leukocyte mediated cytotoxicity","PCP/CE pathway","Wnt signaling pathway","Regulation of RAS by GAPs","cell-cell signaling by wnt","actin filament bundle","response to organonitrogen compound","cortical actin cytoskeleton","actomyosin","regulation of signaling","RAF/MAP kinase cascade","Golgi apparatus","Vasopressin regulates renal water homeostasis via Aquaporins","regulation of T cell proliferation","transport vesicle","proteasome complex","viral translational termination-reinitiation","G beta:gamma signalling through PI3Kgamma","small molecule binding","Interferon gamma signaling","MAPK1/MAPK3 signaling","brush border","amide binding","C-type lectin receptors (CLRs)","Translation","response to stress","RAS GTPase cycle mutants","Integration of energy metabolism","MTOR signalling","Semaphorin interactions","Activation of NF-kappaB in B cells","establishment of tissue polarity","establishment of planar polarity","positive regulation of defense response","positive regulation of intracellular transport","positive regulation of leukocyte mediated immunity","Immune System","negative regulation of immune system process","antigen processing and presentation of endogenous peptide antigen via MHC class I","regulation of defense response to virus by virus","Cytokine Signaling in Immune system","endopeptidase complex","myosin complex","viral translation","photoreceptor inner segment membrane","positive regulation of leukocyte mediated cytotoxicity","cortical actin cytoskeleton organization","plasma membrane bounded cell projection organization","Golgi apparatus subcompartment","regulation of hydrolase activity","leukocyte proliferation","G protein gated Potassium channels","Activation of G protein gated Potassium channels","Inhibition  of voltage gated Ca2+ channels via Gbeta/gamma subunits","NIK--&gt;noncanonical NF-kB signaling","spectrin binding","Cell surface interactions at the vascular wall","cell morphogenesis","integral component of plasma membrane","peptide binding","Innate Immune System","Dectin-1 mediated noncanonical NF-kB signaling","Activation of kainate receptors upon glutamate binding","lymphocyte proliferation","negative regulation of leukocyte mediated cytotoxicity","Aquaporin-mediated transport","mononuclear cell proliferation","Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters","regulation of cell killing","ATPase-coupled transmembrane transporter activity","regulation of cellular amine metabolic process","Signaling by the B Cell Receptor (BCR)","anchored component of membrane","primary active transmembrane transporter activity","cellular response to organonitrogen compound","regulation of cellular process","cell projection organization","positive regulation of protein targeting to membrane","G-protein beta:gamma signalling","glutamatergic synapse","intrinsic component of organelle membrane","G protein-coupled serotonin receptor binding","regulation of immune effector process","adherens junction","cellular response to interferon-gamma","proton transmembrane transport","protein localization to cell cortex","dopamine receptor signaling pathway","neuron projection","regulation of cell communication","stress fiber","contractile actin filament bundle","positive regulation of intracellular signal transduction","RNA binding","Neuronal System","Neurotransmitter receptors and postsynaptic signal transmission","trans-Golgi network membrane","positive regulation of intracellular protein transport","The role of Nef in HIV-1 replication and disease pathogenesis","Inwardly rectifying K+ channels","Ca2+ pathway","response to prostaglandin E","TAP binding","steroid hormone binding","ribonucleoprotein complex biogenesis","kinase binding","cellular protein metabolic process","PIP3 activates AKT signaling","intracellular membrane-bounded organelle","negative regulation of cell killing","Signaling by WNT","molecular adaptor activity","immunological synapse formation","morphogenesis of an epithelium","Cross-presentation of soluble exogenous antigens (endosomes)","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of PTEN stability and activity","MAPK family signaling cascades","adenylate cyclase-activating dopamine receptor signaling pathway","regulation of cellular amino acid metabolic process","pre-replicative complex assembly","regulation of intracellular transport","G-protein beta/gamma-subunit complex","PTEN Regulation","translation","positive regulation of CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation","regulation of CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation","positive regulation of response to external stimulus","anion binding","Regulation of ornithine decarboxylase (ODC)","ruffle","protection from natural killer cell mediated cytotoxicity","antigen processing and presentation of exogenous peptide antigen via MHC class II","phagocytosis","Formation of a pool of free 40S subunits","regulation of leukocyte proliferation","cellular response to oxygen-containing compound","Ubiquitin-dependent degradation of Cyclin D","Autodegradation of the E3 ubiquitin ligase COP1","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Vpu mediated degradation of CD4","p53-Independent G1/S DNA damage checkpoint","p53-Independent DNA Damage Response","cortical cytoskeleton organization","DAP12 signaling","regulation of developmental process","negative regulation of natural killer cell mediated cytotoxicity","tissue morphogenesis","cell surface receptor signaling pathway involved in cell-cell signaling","cation transport","Ragulator complex","regulation of cellular component movement","ADORA2B mediated anti-inflammatory cytokines production","peptidase complex","protein targeting","regulation of protein targeting to membrane","ATPase-coupled cation transmembrane transporter activity","Negative regulation of NOTCH4 signaling","Regulation of Apoptosis","Vif-mediated degradation of APOBEC3G","Transcriptional regulation by RUNX3","negative regulation of immune response","antigen processing and presentation of peptide antigen via MHC class II","intracellular protein transport","Intracellular signaling by second messengers","integrin alphaL-beta2 complex","p38MAPK events","Signaling by KIT in disease","Signaling by phosphorylated juxtamembrane, extracellular and kinase domain KIT mutants","positive regulation of cell killing","lamellipodium membrane","P-type sodium:potassium-exchanging transporter activity","P-type sodium transporter activity","CD4 receptor binding","Asymmetric localization of PCP proteins","Arp2/3 protein complex","AUF1 (hnRNP D0) binds and destabilizes mRNA","Regulation of RUNX3 expression and activity","peptide biosynthetic process","regulation of protein metabolic process","establishment of protein localization to membrane","lysosome localization","structural constituent of cytoskeleton","negative regulation of natural killer cell mediated immunity","regulation of cell migration","MHC class I peptide loading complex","EPH-Ephrin signaling","antigen processing and presentation of peptide or polysaccharide antigen via MHC class II","Degradation of AXIN","SCF-beta-TrCP mediated degradation of Emi1","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","early endosome membrane","neuron projection development","Phosphorylation of CD3 and TCR zeta chains","dopamine receptor binding","establishment of T cell polarity","Interferon Signaling","Golgi vesicle transport","Hh mutants are degraded by ERAD","Protein-protein interactions at synapses","positive regulation of regulatory T cell differentiation","regulation of hematopoietic stem cell differentiation","integral component of organelle membrane","regulation of T cell mediated immunity","clathrin-coated vesicle","NIK/NF-kappaB signaling","ion transport","Degradation of DVL","Stabilization of p53","filopodium","TNFR2 non-canonical NF-kB pathway","virus receptor activity","regulation of lymphocyte proliferation","regulation of adaptive immune response","sodium:potassium-exchanging ATPase complex","positive regulation of lymphocyte mediated immunity","cellular response to hypoxia","P-type potassium transmembrane transporter activity","microvillus","interleukin-1-mediated signaling pathway","antigen binding","exogenous protein binding","GTPase activating protein binding","Metabolism of polyamines","regulation of signal transduction","regulation of mononuclear cell proliferation","regulation of response to external stimulus","protein metabolic process","late endosome membrane","response to prostaglandin","regulation of cellular protein metabolic process","response to glycoside","establishment of lymphocyte polarity","Hh mutants abrogate ligand secretion","CDT1 association with the CDC6:ORC:origin complex","wound healing","ABC-family proteins mediated transport","SHC-related events triggered by IGF1R","heterotypic cell-cell adhesion","tumor necrosis factor-mediated signaling pathway","negative regulation of cell adhesion","regulation of transcription from RNA polymerase II promoter in response to hypoxia","dendrite","Degradation of GLI1 by the proteasome","GLI3 is processed to GLI3R by the proteasome","SCF(Skp2)-mediated degradation of p27/p21","Degradation of GLI2 by the proteasome","negative regulation of immune effector process","dendritic tree","regulation of natural killer cell mediated cytotoxicity","regulation of cellular component biogenesis","protein domain specific binding","positive regulation of T cell mediated immunity","positive regulation of cellular component movement","Defective CFTR causes cystic fibrosis","post-Golgi vesicle-mediated transport","positive regulation of locomotion","response to oxygen-containing compound","protein kinase binding","molecular function regulator","cellular response to decreased oxygen levels","regulation of transcription from RNA polymerase II promoter in response to stress","chemotaxis","neuron projection membrane","microfilament motor activity","cluster of actin-based cell projections","membrane organization","positive regulation of protein localization to cell cortex","viral entry into host cell","phosphatidylinositol 3-kinase binding","actin-based cell projection","regulation of animal organ morphogenesis","taxis","biological process involved in interaction with host","cellular response to prostaglandin E stimulus","positive regulation of Wnt signaling pathway","Interferon alpha/beta signaling","perinuclear region of cytoplasm","positive regulation of cell migration","G alpha (12/13) signalling events","clathrin adaptor complex","spectrin","Infectious disease","amino acid transport complex","regulation of locomotion","negative regulation of leukocyte mediated immunity","regulation of stem cell differentiation","proteasome core complex, beta-subunit complex","Autodegradation of Cdh1 by Cdh1:APC/C","Regulation of mRNA stability by proteins that bind AU-rich elements","positive regulation of establishment of T cell polarity","endoplasmic reticulum protein-containing complex","regulation of protein tyrosine phosphatase activity","response to other organism","anchored component of plasma membrane","response to external biotic stimulus","DAP12 interactions","regulation of natural killer cell mediated immunity","Hedgehog ligand biogenesis","clathrin coat","COP9 signalosome","regulation of DNA-templated transcription in response to stress","positive regulation of canonical Wnt signaling pathway","regulation of cell motility","blood coagulation","regulation of lymphocyte mediated immunity","regulation of leukocyte degranulation","SEMA3A-Plexin repulsion signaling by inhibiting Integrin adhesion","CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation","regulation of intracellular protein transport","p53-Dependent G1/S DNA damage checkpoint","p53-Dependent G1 DNA Damage Response","Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha","somatodendritic compartment","Hedgehog 'off' state","regulation of secretion","Platelet homeostasis","cation-transporting ATPase complex","Basigin interactions","T-tubule"],["3.6e-57","3.5e-56","3.9e-56","3.9e-56","3.5e-42","7.7e-35","1.1e-26","1.9e-24","6.8e-24","1.3e-23","1.5e-21","1.9e-21","7.5e-21","9.3e-21","3.0e-19","9.6e-19","2.1e-18","2.5e-18","4.8e-18","6.5e-18","6.7e-18","8.0e-18","6.8e-17","6.8e-17","3.8e-16","6.3e-16","8.9e-16","2.9e-15","3.6e-15","4.2e-15","5.1e-15","5.1e-15","6.6e-15","7.8e-15","1.3e-14","1.4e-14","2.6e-14","3.5e-14","3.8e-14","4.5e-14","5.6e-14","7.9e-14","8.7e-14","1.4e-13","2.0e-13","2.2e-13","3.2e-13","3.2e-13","3.2e-13","4.8e-13","1.2e-12","1.2e-12","1.7e-12","1.7e-12","3.3e-12","3.5e-12","4.5e-12","4.8e-12","5.4e-12","6.2e-12","7.5e-12","8.1e-12","8.5e-12","9.8e-12","1.0e-11","1.1e-11","1.1e-11","1.4e-11","1.4e-11","1.7e-11","2.1e-11","2.1e-11","2.1e-11","2.4e-11","2.7e-11","3.0e-11","3.2e-11","3.2e-11","3.4e-11","4.0e-11","4.1e-11","5.2e-11","5.8e-11","8.4e-11","8.5e-11","9.1e-11","1.4e-10","1.8e-10","1.9e-10","2.0e-10","2.0e-10","2.1e-10","2.2e-10","2.8e-10","2.8e-10","3.0e-10","3.1e-10","3.1e-10","4.0e-10","4.3e-10","4.4e-10","4.5e-10","4.5e-10","4.6e-10","5.0e-10","5.2e-10","6.7e-10","7.0e-10","8.0e-10","8.0e-10","1.0e-09","1.0e-09","1.3e-09","1.3e-09","1.6e-09","1.6e-09","1.7e-09","1.7e-09","1.9e-09","2.0e-09","2.1e-09","2.1e-09","2.3e-09","2.4e-09","2.7e-09","2.8e-09","4.4e-09","5.4e-09","5.7e-09","6.1e-09","6.1e-09","7.5e-09","7.8e-09","7.9e-09","8.8e-09","8.9e-09","9.7e-09","1.0e-08","1.1e-08","1.1e-08","1.1e-08","1.2e-08","1.4e-08","1.5e-08","1.6e-08","1.6e-08","2.2e-08","2.2e-08","2.2e-08","2.3e-08","2.6e-08","2.7e-08","3.0e-08","3.0e-08","3.2e-08","3.6e-08","3.8e-08","3.8e-08","4.1e-08","4.9e-08","5.2e-08","5.4e-08","6.0e-08","6.2e-08","6.7e-08","6.7e-08","6.8e-08","7.1e-08","7.2e-08","7.2e-08","7.8e-08","8.2e-08","8.5e-08","8.6e-08","9.2e-08","9.2e-08","1.0e-07","1.1e-07","1.4e-07","1.5e-07","1.6e-07","1.8e-07","1.8e-07","1.9e-07","1.9e-07","1.9e-07","1.9e-07","2.1e-07","2.2e-07","2.3e-07","2.3e-07","2.4e-07","2.5e-07","2.6e-07","2.8e-07","2.8e-07","2.9e-07","3.0e-07","3.0e-07","3.5e-07","3.5e-07","3.6e-07","3.6e-07","3.8e-07","3.8e-07","4.0e-07","4.6e-07","5.1e-07","5.1e-07","5.9e-07","6.0e-07","6.1e-07","6.2e-07","6.7e-07","7.0e-07","7.1e-07","7.5e-07","7.5e-07","7.7e-07","7.9e-07","8.2e-07","8.5e-07","9.0e-07","1.0e-06","1.1e-06","1.1e-06","1.2e-06","1.2e-06","1.5e-06","1.5e-06","1.6e-06","1.9e-06","1.9e-06","2.2e-06","2.2e-06","2.7e-06","2.7e-06","2.9e-06","2.9e-06","2.9e-06","3.2e-06","3.5e-06","3.6e-06","3.7e-06","3.7e-06","3.8e-06","4.0e-06","4.0e-06","4.2e-06","4.5e-06","4.9e-06","5.0e-06","5.5e-06","5.6e-06","6.2e-06","6.6e-06","6.7e-06","6.8e-06","7.4e-06","7.8e-06","7.9e-06","8.0e-06","8.2e-06","8.5e-06","9.0e-06","9.1e-06","9.6e-06","9.8e-06","1.0e-05","1.0e-05","1.1e-05","1.1e-05","1.2e-05","1.2e-05","1.4e-05","1.4e-05","1.4e-05","1.5e-05","1.6e-05","1.7e-05","1.8e-05","1.9e-05","1.9e-05","1.9e-05","2.0e-05","2.1e-05","2.1e-05","2.3e-05","2.3e-05","2.4e-05","2.5e-05","2.6e-05","2.6e-05","2.7e-05","2.9e-05","3.2e-05","3.3e-05","3.5e-05","3.5e-05","3.5e-05","3.7e-05","3.8e-05","3.9e-05","4.0e-05","4.0e-05","4.2e-05","4.2e-05","4.4e-05","4.4e-05","4.5e-05","4.7e-05","5.0e-05","5.8e-05","6.0e-05","6.3e-05","6.4e-05","6.4e-05","6.7e-05","6.7e-05","7.0e-05","7.3e-05","7.4e-05","7.4e-05","7.4e-05","7.6e-05","7.7e-05","8.3e-05","9.3e-05","9.5e-05","1.0e-04","1.0e-04","1.2e-04","1.2e-04","1.2e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.4e-04","1.4e-04","1.6e-04","1.8e-04","1.8e-04","1.8e-04","1.8e-04","1.8e-04","1.8e-04","1.9e-04","1.9e-04","1.9e-04","1.9e-04","2.1e-04","2.2e-04","2.3e-04","2.3e-04","2.4e-04","2.5e-04","2.5e-04","2.5e-04","2.6e-04","2.7e-04","2.7e-04","2.7e-04","2.8e-04","2.9e-04","3.1e-04","3.1e-04","3.2e-04","3.2e-04","3.6e-04","3.6e-04","3.7e-04","3.8e-04","3.9e-04","4.0e-04","4.1e-04","4.1e-04","4.2e-04","4.2e-04","4.3e-04","4.4e-04","4.4e-04","4.4e-04","4.7e-04","4.8e-04","4.8e-04","4.9e-04","5.0e-04","5.0e-04","5.1e-04","5.2e-04","5.3e-04","5.7e-04","5.9e-04","6.0e-04","6.0e-04","6.2e-04","6.2e-04","6.6e-04","6.7e-04","7.0e-04","7.1e-04","7.2e-04","7.7e-04","7.8e-04","8.4e-04","8.6e-04","8.8e-04","9.3e-04","9.4e-04","9.5e-04","9.5e-04","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.6e-03","1.6e-03","1.6e-03","1.6e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.8e-03","1.9e-03","1.9e-03","1.9e-03","2.2e-03","2.2e-03","2.3e-03","2.3e-03","2.3e-03","2.4e-03","2.5e-03","2.5e-03","2.5e-03","2.6e-03","2.6e-03","2.7e-03","2.8e-03","2.8e-03","2.8e-03","2.8e-03","2.9e-03","3.2e-03","3.2e-03","3.2e-03","3.3e-03","3.4e-03","3.5e-03","3.5e-03","3.6e-03","3.6e-03","3.6e-03","3.7e-03","3.8e-03","3.9e-03","4.0e-03","4.0e-03","4.0e-03","4.2e-03","4.2e-03","4.2e-03","4.4e-03","4.5e-03","4.7e-03","4.9e-03","5.0e-03","5.0e-03","5.0e-03","5.1e-03","5.2e-03","5.6e-03","5.7e-03","5.9e-03","6.0e-03","6.0e-03","6.1e-03","6.1e-03","6.2e-03","6.4e-03","6.6e-03","7.0e-03","7.1e-03","7.1e-03","7.2e-03","7.4e-03","7.4e-03","7.5e-03","7.5e-03","7.6e-03","7.7e-03","7.7e-03","7.7e-03","7.9e-03","8.0e-03","8.0e-03","8.1e-03","8.1e-03","8.2e-03","8.2e-03","8.4e-03","8.5e-03","8.5e-03","8.5e-03","8.6e-03","8.6e-03","8.8e-03","8.8e-03","8.8e-03","8.9e-03","9.4e-03","9.4e-03","9.9e-03","9.9e-03","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.1e-02","2.1e-02","2.1e-02","2.1e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.5e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.8e-02","2.8e-02","2.9e-02","2.9e-02","2.9e-02","3.0e-02","3.0e-02","3.0e-02","3.0e-02","3.0e-02","3.0e-02","3.1e-02","3.1e-02","3.1e-02","3.1e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.3e-02","3.3e-02","3.4e-02","3.5e-02","3.5e-02","3.5e-02","3.5e-02","3.6e-02","3.7e-02","3.7e-02","3.8e-02","3.8e-02","3.8e-02","3.9e-02","3.9e-02","3.9e-02","4.0e-02","4.1e-02","4.1e-02","4.2e-02","4.2e-02","4.2e-02","4.2e-02","4.3e-02","4.3e-02","4.3e-02","4.4e-02","4.4e-02","4.4e-02","4.5e-02","4.5e-02","4.6e-02","4.6e-02","4.6e-02","4.7e-02","4.7e-02","4.7e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.9e-02","4.9e-02","4.9e-02","4.9e-02","5.0e-02","5.0e-02"],[2178,2263,2265,2265,4058,3595,4564,5680,420,427,6178,622,1304,838,2107,9829,2433,2438,5214,1507,5364,6938,339,339,2193,3438,696,1341,11951,789,36,36,5361,1413,236,906,1492,195,1240,193,169,1351,175,424,1326,183,109,109,1011,5541,2544,300,790,308,1734,103,482,2165,2670,178,377,493,494,202,1159,15,1001,35,538,3519,426,75,857,1027,1183,552,17,17,723,727,558,18,80,1135,775,99,912,16,888,1079,1842,2234,2906,1856,495,3148,58,517,510,757,73,475,475,513,2778,2337,509,315,395,395,483,450,99,488,341,544,339,698,898,381,2319,3457,3443,500,502,32,449,154,83,402,402,4651,671,780,307,2821,44,14767,6880,563,1502,123,1509,14027,24,2368,25,25,335,19,6474,6661,346,566,851,128,163,28,42,896,13201,333,756,3226,60,1839,341,250,243,1660,3644,321,286,4116,99,8,139,810,2039,450,112,113,113,33,20,723,723,158,5863,115,268,3377,116,22,85,951,26,4829,712,6937,1551,71,3104,23,818,32,6961,442,24,993,1348,132,203,2216,287,225,13,13,2886,203,8,143,1754,4,1866,157,1774,1724,59,59,1242,312,617,221,2017,549,6447,213,1840,1840,11,190,824,23,31,144,48,523,352,520,19,316,1148,75,120,57,14808,151,196,30,2282,80,574,217,2357,253,3114,520,662,203,149,511,203,228,121,4416,179,52,2048,187,1814,11,206,198,3191,673,177,25,24,1735,60,1003,2657,64,90,73,2452,120,120,965,1178,296,1932,1934,799,43,43,210,1809,1956,13117,78,72,326,329,28,152,238,2339,595,95,34,230,1846,409,7873,1917,426,1222,9487,35,88,570,1383,75,1048,1626,82,141,345,102,476,24,414,1150,33,33,18,364,63,163,544,366,212,2177,1725,426,1723,9,725,90,112,42,1768,1473,113,110,98,161,102,20,20,728,443,42,2439,26,114,3,3,657,21,10,258,107,67,7,169,1377,125,2176,76,48,889,433,12122,496,90,12802,113,92,533,67,535,78,1092,92,79,3669,264,1593,43,177,416,67,5,25,2516,87,269,109,387,136,292,4321,3,107,41,64,67,130,130,364,227,127,2038,614,13,29,696,73,57,20,4,52,40,1596,845,1252,327,29,29,29,59,27,190,1045,1643,312,1090,60,30,296,20,52,299,20,109,120,83,163,172,123,671,11517,1636,31,32,354,419,9,421,171,177,160,11,41,1384,3634,70,70,1063,1951,400,196,98,184,26,35,59,25,9,6,473,757,5280,252,12046,24,328,341,14,566,50,50,69,308,10,66,66,365,12,138,804,5,5,518,2413,51,180,6,97,388,102,253,1237,52,52,52,52,52,52,62,25,2621,16,675,639,1210,10,1149,133,97,400,37,54,53,53,53,94,373,101,995,292,2,11,17,17,69,22,8,8,9,64,11,54,54,834,2759,385,79,104,17,1008,9,90,104,55,55,55,167,1021,23,17,12,193,415,56,86,18,74,389,80,194,196,1610,57,57,106,100,76,232,172,12,107,214,10,89,105,163,77,15,58,3251,234,1238,5912,149,32,2591,5,13,59,59,555,103,6,60,179,308,78,641,60,60,60,60,107,643,41,991,706,52,605,61,144,607,1748,677,1115,224,111,654,61,28,164,1016,6,143,32,217,185,657,243,17,186,68,745,568,80,24,9,909,2,1113,55,115,11,64,86,2,128,6,1600,63,1601,33,45,65,47,35,117,153,1070,351,156,46,14,8,256,66,66,66,875,113,650,85,19,25,53],[128,129,129,129,149,131,139,146,43,43,148,38,67,54,83,197,86,86,140,70,139,161,32,32,81,107,31,62,213,44,11,11,114,62,25,49,63,22,54,22,20,59,21,30,41,21,18,18,51,123,84,25,43,26,66,14,30,70,83,17,30,30,30,21,46,7,44,9,34,97,29,15,41,45,48,34,7,7,39,39,34,7,15,50,40,16,43,7,43,47,64,66,86,64,32,88,9,31,29,35,13,28,28,29,81,67,31,19,20,20,30,21,10,30,26,31,26,37,42,24,68,90,78,30,30,9,22,11,16,24,24,105,34,38,24,68,10,230,141,32,55,10,55,221,7,72,7,7,23,7,134,137,23,28,38,16,16,8,9,37,212,22,36,72,9,60,24,19,16,52,88,21,17,98,16,5,15,35,62,27,13,13,13,7,6,23,23,15,123,13,19,85,13,7,11,39,7,104,34,133,37,12,80,7,24,8,133,26,7,33,45,15,18,64,21,17,6,6,72,16,5,17,55,4,57,15,55,54,8,8,44,20,30,18,59,34,111,14,52,52,5,15,30,7,7,13,8,27,21,22,6,18,41,10,14,8,223,13,10,6,54,12,34,15,55,16,79,25,30,10,9,26,10,17,12,100,14,7,56,13,53,5,16,13,77,39,10,7,6,50,8,31,68,10,10,9,64,9,9,34,38,12,47,47,37,7,7,13,52,46,204,11,9,18,19,6,13,15,59,26,9,5,15,45,20,123,46,19,34,169,7,8,25,43,9,36,43,10,9,19,9,29,6,16,38,6,6,5,18,6,11,24,15,15,49,30,16,49,3,17,8,8,8,51,44,8,10,11,10,10,5,5,25,21,6,61,6,10,3,3,26,5,4,20,11,4,3,10,32,12,48,8,7,27,19,186,21,8,193,11,11,20,9,20,7,31,9,7,80,19,28,6,13,18,7,3,5,52,8,19,9,15,12,11,73,3,11,7,9,9,10,10,16,13,11,72,25,4,6,34,7,6,4,2,7,5,44,17,36,17,5,5,5,8,5,15,32,27,13,41,8,5,16,5,6,16,3,9,4,8,13,11,4,24,155,44,6,5,16,16,3,19,11,10,6,4,5,37,77,6,6,33,47,14,13,6,12,5,5,6,5,3,2,11,25,89,16,185,5,18,15,4,21,7,7,8,19,4,7,7,17,3,11,14,3,3,21,48,7,9,3,7,17,10,14,31,7,7,7,7,7,7,7,5,51,4,23,20,36,3,34,8,7,18,6,3,7,7,7,9,15,7,31,17,2,4,5,5,7,3,2,2,3,8,2,7,7,14,53,17,8,8,4,31,3,8,7,7,7,7,7,31,5,4,4,10,18,7,6,4,7,11,6,5,11,43,7,7,8,9,7,13,8,2,9,11,2,7,8,7,7,3,7,69,13,36,95,7,5,52,2,4,7,7,16,9,3,7,10,14,7,21,7,7,7,7,8,21,5,30,21,6,22,7,10,22,39,22,27,11,8,23,6,4,9,30,3,10,4,11,10,23,13,4,10,6,22,21,6,2,3,34,2,32,6,8,3,7,8,2,8,3,34,6,34,5,5,7,3,4,8,9,31,14,8,6,4,3,13,7,7,7,25,9,22,6,2,5,4]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/BP-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/MF-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/CC-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/scatter-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="skap2_c.-1" class="section level3 tabset tabset-fade">
<h3><strong>5.2. SKAP2_C.</strong></h3>
<div id="gostplot-1" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in SKAP2_C. were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &lt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_2)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-a2454929a56e939b13f9" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-a2454929a56e939b13f9">{"x":{"data":[{"x":[81.4008725542289,116.213716409311,81.3032102818689,93.7551500077641,101.03850178107,109.813081328489,110.098555663079,93.7476375252749,152.668037688289,80.5482057917015,82.5465261338361,125.739544205652,164.924652869464,110.049724526899,89.893734008301,116.262547545491,119.969957653925,116.266303786735,85.9872431139025,102.037661952138,81.4046287954735,73.1897291935028,72.964354718826,93.7513937665195,72.9718672013152,73.0169420962506,146.81581382918,73.1521667810567,73.1596792635459,82.8132192622037,120.187819646112,118.302186541316,120.184063404868,131.625574235962,73.1747042285244,73.1859729522582,73.1634355047905,146.53033949459,81.4271662429412,73.2235353647043,73.2197791234597,85.6980125380672,73.1033356448767,129.901459504684,116.217472650555,72.6413179717892,118.959528759124,130.941938329442,73.2272916059489,103.172046808011,171.370362845222,73.5240346642735,101.057282987294,171.374119086466,146.766982693,153.227717633736,73.3512475670212,107.07102521992,158.726854815851,104.719618200792,80.9726610523429,152.671793929534,104.704593235814,73.674284314058,73.197241675992,145.670160249573,144.851299658247,81.2769165931566,72.43848094458,131.295025006436,81.4196537604519,130.945694570687,123.823861170899,127.01666622882,83.598273682328,107.236299834683,72.5060932869831,153.231473874981,144.806224763312,103.175803049256,81.2694041106674,81.1867668032859,186.549333714707,72.8366425165091,155.654249477757,182.939585878633,83.4292428263204,183.112372975885,81.0365171535013,129.785016026101,81.4234100016966,81.4346787254304,103.183315531745,81.2881853168904,104.74215564826,129.908971987173,146.958550996476,91.0919749653328,87.2343152071143,118.238330440158,123.793811240942,81.1792543207966,127.628933551693,127.020422470065,129.833847162281,129.837603403525,93.7626624902533,131.509130757379,130.86681350455,174.957573233828,182.947098361122,81.3144790056027,131.29878124768,104.745911889504,110.087286939345,81.321991488092,118.978309965347,83.4029491376081,155.605418341577,113.712059740398,127.512490073109,87.2380714483589,149.234833190712,176.723006618797,83.6132986473065,153.592073034464,153.558266863262,153.554510622018,153.550754380773,168.425469709444,168.970124689913,90.4045828175685,96.7751679684337,72.6676116605015,126.479523730841,126.284199186121,120.262944471005,72.983135925049,174.961329475073,83.3428492776943,103.307271492817,73.6329656603673,83.151280974219,83.6057861648172,97.5414411823349,145.673916490818,81.438434966675,123.703661451071,131.557961893559,149.294933050626],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.5626777291017,13.9270377759483,12.9254611414979,12.8550901303014,12.1039594788338,11.8858818289023,11.2759143706134,10.6198289289505,10.3959041676243,8.69521990699272,8.62020968506321,8.62020968506321,7.82653136472386,6.54931924498184,6.52461186871218,6.44950421319488,5.40934417933031,5.21134231855435,5.20841042912769,5.18131593578346,5.05448597666727,5.02896159190467,4.80095394957753,4.60652048948246,4.56442641904502,4.41251281038038,4.35968507267659,4.26873341363426,4.13662838041309,4.06884910223446,3.94515537281616,3.9427112493302,3.93768652889732,3.9127920064785,3.8368262907599,3.8368262907599,3.77247892642344,3.73817888881936,3.69320051469175,3.62775856875568,3.54647919218729,3.48045872532281,3.45543281053128,3.42562800904328,3.41204437895439,3.4033561881036,3.31453344875581,3.30638941822893,3.25504542847489,3.13855480274385,3.1279325140906,3.10900497399399,3.08565228130384,3.07033438871117,3.04438041625203,3.00741025776996,2.97342472611433,2.96760928932921,2.88971194715989,2.88886679199216,2.85654065865102,2.84869140803357,2.82358467461823,2.80426263653803,2.65077093791524,2.59296999089078,2.57201362039572,2.54652928964914,2.52635239705278,2.52203449844071,2.50277213013948,2.49002809344432,2.41293252651169,2.40961697085883,2.34163831720074,2.29863617191628,2.29863617191628,2.24747175085325,2.24052778305881,2.19958196551504,2.19339666477297,2.16515300551427,2.16421618693862,2.16036829307502,2.15693561525782,2.13450117809794,2.13226082854526,2.11567524536269,2.0820526187779,2.0705475163378,2.06569543687552,2.06569543687552,2.06569543687552,2.06569543687552,2.02615719791898,1.96754620647712,1.92566205520111,1.92566205520111,1.92566205520111,1.92566205520111,1.81225801462637,1.80815755637784,1.80584120484782,1.72163005585785,1.6896249683143,1.65302718211955,1.64330609947363,1.58021285053321,1.57365425366144,1.57181065555114,1.56264052593932,1.56226629548464,1.54557704750635,1.53363058724383,1.47567467908539,1.44649420918245,1.44298479193526,1.44298479193526,1.43493990159189,1.3669594577518,1.35590781986467],"text":["GO:0006396 (1010) <br> RNA processing <br> 3.517e-91","GO:0042254 (335) <br> ribosome biogenesis <br> 1.759e-80","GO:0006364 (265) <br> rRNA processing <br> 3.232e-79","GO:0016072 (275) <br> rRNA metabolic process <br> 6.965e-78","GO:0022613 (473) <br> ribonucleoprotein complex biogenesis <br> 5.105e-75","GO:0034470 (439) <br> ncRNA processing <br> 1.271e-64","GO:0034660 (535) <br> ncRNA metabolic process <br> 6.060e-58","GO:0016070 (4714) <br> RNA metabolic process <br> 6.198e-42","GO:0090304 (5234) <br> nucleic acid metabolic process <br> 4.368e-40","GO:0006139 (5752) <br> nucleobase-containing compound metabolic process <br> 1.611e-39","GO:0006725 (5973) <br> cellular aromatic compound metabolic process <br> 5.022e-38","GO:0046483 (5929) <br> heterocycle metabolic process <br> 1.255e-37","GO:1901360 (6227) <br> organic cyclic compound metabolic process <br> 1.988e-35","GO:0034641 (6576) <br> cellular nitrogen compound metabolic process <br> 1.608e-34","GO:0010467 (6192) <br> gene expression <br> 5.956e-31","GO:0042273 (72) <br> ribosomal large subunit biogenesis <br> 2.527e-29","GO:0044085 (3377) <br> cellular component biogenesis <br> 4.923e-26","GO:0042274 (74) <br> ribosomal small subunit biogenesis <br> 8.162e-26","GO:0008380 (459) <br> RNA splicing <br> 2.550e-25","GO:0030490 (50) <br> maturation of SSU-rRNA <br> 3.821e-25","GO:0006397 (532) <br> mRNA processing <br> 9.107e-25","GO:0000470 (29) <br> maturation of LSU-rRNA <br> 3.818e-23","GO:0000375 (363) <br> RNA splicing, via transesterification reactions <br> 1.639e-21","GO:0016071 (880) <br> mRNA metabolic process <br> 3.593e-21","GO:0000377 (360) <br> RNA splicing, via transesterification reactions with bulged adenosine as nucleophile <br> 1.540e-20","GO:0000398 (360) <br> mRNA splicing, via spliceosome <br> 1.540e-20","GO:0071840 (6880) <br> cellular component organization or biogenesis <br> 2.377e-17","GO:0000460 (36) <br> maturation of 5.8S rRNA <br> 3.420e-17","GO:0000462 (37) <br> maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 4.902e-17","GO:0006807 (10148) <br> nitrogen compound metabolic process <br> 2.737e-16","GO:0044238 (10642) <br> primary metabolic process <br> 1.183e-14","GO:0043170 (9867) <br> macromolecule metabolic process <br> 1.187e-13","GO:0044237 (10941) <br> cellular metabolic process <br> 1.396e-13","GO:0051276 (1270) <br> chromosome organization <br> 7.871e-13","GO:0000466 (24) <br> maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 1.301e-12","GO:0000469 (27) <br> cleavage involved in rRNA processing <br> 5.298e-12","GO:0000463 (15) <br> maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 2.400e-11","GO:0071704 (11397) <br> organic substance metabolic process <br> 4.019e-11","GO:0006403 (222) <br> RNA localization <br> 2.017e-09","GO:0000479 (15) <br> endonucleolytic cleavage of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 2.398e-09","GO:0000478 (15) <br> endonucleolytic cleavage involved in rRNA processing <br> 2.398e-09","GO:0008152 (11915) <br> metabolic process <br> 1.491e-08","GO:0000447 (9) <br> endonucleolytic cleavage in ITS1 to separate SSU-rRNA from 5.8S rRNA and LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 2.823e-07","GO:0050684 (145) <br> regulation of mRNA processing <br> 2.988e-07","GO:0042255 (62) <br> ribosome assembly <br> 3.552e-07","GO:0000154 (37) <br> rRNA modification <br> 3.896e-06","GO:0043484 (152) <br> regulation of RNA splicing <br> 6.147e-06","GO:0051052 (389) <br> regulation of DNA metabolic process <br> 6.189e-06","GO:0000480 (7) <br> endonucleolytic cleavage in 5'-ETS of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 6.587e-06","GO:0031123 (157) <br> RNA 3'-end processing <br> 8.821e-06","GO:1903311 (384) <br> regulation of mRNA metabolic process <br> 9.355e-06","GO:0000819 (214) <br> sister chromatid segregation <br> 1.581e-05","GO:0022618 (196) <br> ribonucleoprotein complex assembly <br> 2.474e-05","GO:1903312 (95) <br> negative regulation of mRNA metabolic process <br> 2.726e-05","GO:0071826 (203) <br> ribonucleoprotein complex subunit organization <br> 3.868e-05","GO:0090501 (157) <br> RNA phosphodiester bond hydrolysis <br> 4.368e-05","GO:0000723 (169) <br> telomere maintenance <br> 5.386e-05","GO:0033044 (299) <br> regulation of chromosome organization <br> 7.301e-05","GO:0140014 (346) <br> mitotic nuclear division <br> 8.534e-05","GO:0032204 (82) <br> regulation of telomere maintenance <br> 1.135e-04","GO:0006259 (976) <br> DNA metabolic process <br> 1.141e-04","GO:0090305 (306) <br> nucleic acid phosphodiester bond hydrolysis <br> 1.154e-04","GO:0032200 (183) <br> telomere organization <br> 1.222e-04","GO:0000967 (5) <br> rRNA 5'-end processing <br> 1.456e-04","GO:0000472 (5) <br> endonucleolytic cleavage to generate mature 5'-end of SSU-rRNA from (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 1.456e-04","GO:0071426 (130) <br> ribonucleoprotein complex export from nucleus <br> 1.689e-04","GO:0071166 (131) <br> ribonucleoprotein complex localization <br> 1.827e-04","GO:0006356 (34) <br> regulation of transcription by RNA polymerase I <br> 2.027e-04","GO:0000027 (27) <br> ribosomal large subunit assembly <br> 2.356e-04","GO:0051168 (202) <br> nuclear export <br> 2.841e-04","GO:0006401 (431) <br> RNA catabolic process <br> 3.308e-04","GO:0051053 (154) <br> negative regulation of DNA metabolic process <br> 3.504e-04","GO:0045943 (23) <br> positive regulation of transcription by RNA polymerase I <br> 3.753e-04","GO:0048024 (107) <br> regulation of mRNA splicing, via spliceosome <br> 3.872e-04","GO:0007059 (353) <br> chromosome segregation <br> 3.950e-04","GO:0033119 (27) <br> negative regulation of RNA splicing <br> 4.847e-04","GO:0000070 (185) <br> mitotic sister chromatid segregation <br> 4.939e-04","GO:0090502 (81) <br> RNA phosphodiester bond hydrolysis, endonucleolytic <br> 5.558e-04","GO:0071103 (359) <br> DNA conformation change <br> 7.269e-04","GO:0031124 (104) <br> mRNA 3'-end processing <br> 7.448e-04","GO:0006354 (104) <br> DNA-templated transcription, elongation <br> 7.780e-04","GO:0006325 (839) <br> chromatin organization <br> 8.210e-04","GO:2001251 (92) <br> negative regulation of chromosome organization <br> 8.505e-04","GO:0000280 (495) <br> nuclear division <br> 9.029e-04","GO:0098813 (293) <br> nuclear chromosome segregation <br> 9.831e-04","GO:2000232 (15) <br> regulation of rRNA processing <br> 1.063e-03","GO:0007004 (75) <br> telomere maintenance via telomerase <br> 1.077e-03","GO:2000278 (112) <br> regulation of DNA biosynthetic process <br> 1.289e-03","GO:0006278 (77) <br> RNA-dependent DNA biosynthetic process <br> 1.292e-03","GO:0050434 (24) <br> positive regulation of viral transcription <br> 1.391e-03","GO:0006402 (388) <br> mRNA catabolic process <br> 1.417e-03","GO:0006405 (129) <br> RNA export from nucleus <br> 1.501e-03","GO:0031126 (12) <br> sno(s)RNA 3'-end processing <br> 1.569e-03","GO:0006360 (68) <br> transcription by RNA polymerase I <br> 2.235e-03","GO:0032210 (54) <br> regulation of telomere maintenance via telomerase <br> 2.553e-03","GO:0050686 (31) <br> negative regulation of mRNA processing <br> 2.679e-03","GO:0071897 (201) <br> DNA biosynthetic process <br> 2.841e-03","GO:0010833 (87) <br> telomere maintenance via telomere lengthening <br> 2.976e-03","GO:0009451 (169) <br> RNA modification <br> 3.006e-03","GO:0043144 (14) <br> sno(s)RNA processing <br> 3.142e-03","GO:0045934 (1569) <br> negative regulation of nucleobase-containing compound metabolic process <br> 3.236e-03","GO:0006323 (246) <br> DNA packaging <br> 3.864e-03","GO:0048285 (544) <br> organelle fission <br> 3.894e-03","GO:0048025 (22) <br> negative regulation of mRNA splicing, via spliceosome <br> 4.554e-03","GO:0050657 (183) <br> nucleic acid transport <br> 5.028e-03","GO:0050658 (183) <br> RNA transport <br> 5.028e-03","GO:0016074 (16) <br> sno(s)RNA metabolic process <br> 5.656e-03","GO:0051236 (186) <br> establishment of RNA localization <br> 5.747e-03","GO:0051028 (151) <br> mRNA transport <br> 6.316e-03","GO:1904356 (63) <br> regulation of telomere maintenance via telomere lengthening <br> 6.406e-03","GO:2000234 (10) <br> positive regulation of rRNA processing <br> 6.837e-03","GO:0006368 (88) <br> transcription elongation from RNA polymerase II promoter <br> 6.851e-03","GO:0051169 (356) <br> nuclear transport <br> 6.912e-03","GO:0032211 (21) <br> negative regulation of telomere maintenance via telomerase <br> 6.967e-03","GO:0034655 (573) <br> nucleobase-containing compound catabolic process <br> 7.337e-03","GO:0006370 (33) <br> 7-methylguanosine mRNA capping <br> 7.375e-03","GO:0043489 (66) <br> RNA stabilization <br> 7.662e-03","GO:0006996 (4136) <br> organelle organization <br> 8.278e-03","GO:0098781 (110) <br> ncRNA transcription <br> 8.501e-03","GO:0036260 (34) <br> RNA capping <br> 8.596e-03","GO:0048239 (34) <br> negative regulation of DNA recombination at telomere <br> 8.596e-03","GO:0009452 (34) <br> 7-methylguanosine RNA capping <br> 8.596e-03","GO:0072695 (34) <br> regulation of DNA recombination at telomere <br> 8.596e-03","GO:1904872 (18) <br> regulation of telomerase RNA localization to Cajal body <br> 9.415e-03","GO:0007064 (27) <br> mitotic sister chromatid cohesion <br> 1.078e-02","GO:0090685 (19) <br> RNA localization to nucleus <br> 1.187e-02","GO:0090672 (19) <br> telomerase RNA localization <br> 1.187e-02","GO:0090671 (19) <br> telomerase RNA localization to Cajal body <br> 1.187e-02","GO:0090670 (19) <br> RNA localization to Cajal body <br> 1.187e-02","GO:1902369 (76) <br> negative regulation of RNA catabolic process <br> 1.541e-02","GO:1902570 (15) <br> protein localization to nucleolus <br> 1.555e-02","GO:0010639 (361) <br> negative regulation of organelle organization <br> 1.564e-02","GO:0019219 (4123) <br> regulation of nucleobase-containing compound metabolic process <br> 1.898e-02","GO:0000184 (123) <br> nuclear-transcribed mRNA catabolic process, nonsense-mediated decay <br> 2.044e-02","GO:0046782 (41) <br> regulation of viral transcription <br> 2.223e-02","GO:0046700 (617) <br> heterocycle catabolic process <br> 2.273e-02","GO:0044270 (623) <br> cellular nitrogen compound catabolic process <br> 2.629e-02","GO:0000380 (76) <br> alternative mRNA splicing, via spliceosome <br> 2.669e-02","GO:1904357 (29) <br> negative regulation of telomere maintenance via telomere lengthening <br> 2.680e-02","GO:0006974 (894) <br> cellular response to DNA damage stimulus <br> 2.738e-02","GO:0031167 (25) <br> rRNA methylation <br> 2.740e-02","GO:0000956 (214) <br> nuclear-transcribed mRNA catabolic process <br> 2.847e-02","GO:0006913 (353) <br> nucleocytoplasmic transport <br> 2.927e-02","GO:0007062 (56) <br> sister chromatid cohesion <br> 3.344e-02","GO:0019439 (636) <br> aromatic compound catabolic process <br> 3.577e-02","GO:0071427 (112) <br> mRNA-containing ribonucleoprotein complex export from nucleus <br> 3.606e-02","GO:0006406 (112) <br> mRNA export from nucleus <br> 3.606e-02","GO:0045910 (76) <br> negative regulation of DNA recombination <br> 3.673e-02","GO:0051252 (3847) <br> regulation of RNA metabolic process <br> 4.296e-02","GO:0072711 (12) <br> cellular response to hydroxyurea <br> 4.406e-02"],"key":["GO:0006396","GO:0042254","GO:0006364","GO:0016072","GO:0022613","GO:0034470","GO:0034660","GO:0016070","GO:0090304","GO:0006139","GO:0006725","GO:0046483","GO:1901360","GO:0034641","GO:0010467","GO:0042273","GO:0044085","GO:0042274","GO:0008380","GO:0030490","GO:0006397","GO:0000470","GO:0000375","GO:0016071","GO:0000377","GO:0000398","GO:0071840","GO:0000460","GO:0000462","GO:0006807","GO:0044238","GO:0043170","GO:0044237","GO:0051276","GO:0000466","GO:0000469","GO:0000463","GO:0071704","GO:0006403","GO:0000479","GO:0000478","GO:0008152","GO:0000447","GO:0050684","GO:0042255","GO:0000154","GO:0043484","GO:0051052","GO:0000480","GO:0031123","GO:1903311","GO:0000819","GO:0022618","GO:1903312","GO:0071826","GO:0090501","GO:0000723","GO:0033044","GO:0140014","GO:0032204","GO:0006259","GO:0090305","GO:0032200","GO:0000967","GO:0000472","GO:0071426","GO:0071166","GO:0006356","GO:0000027","GO:0051168","GO:0006401","GO:0051053","GO:0045943","GO:0048024","GO:0007059","GO:0033119","GO:0000070","GO:0090502","GO:0071103","GO:0031124","GO:0006354","GO:0006325","GO:2001251","GO:0000280","GO:0098813","GO:2000232","GO:0007004","GO:2000278","GO:0006278","GO:0050434","GO:0006402","GO:0006405","GO:0031126","GO:0006360","GO:0032210","GO:0050686","GO:0071897","GO:0010833","GO:0009451","GO:0043144","GO:0045934","GO:0006323","GO:0048285","GO:0048025","GO:0050657","GO:0050658","GO:0016074","GO:0051236","GO:0051028","GO:1904356","GO:2000234","GO:0006368","GO:0051169","GO:0032211","GO:0034655","GO:0006370","GO:0043489","GO:0006996","GO:0098781","GO:0036260","GO:0048239","GO:0009452","GO:0072695","GO:1904872","GO:0007064","GO:0090685","GO:0090672","GO:0090671","GO:0090670","GO:1902369","GO:1902570","GO:0010639","GO:0019219","GO:0000184","GO:0046782","GO:0046700","GO:0044270","GO:0000380","GO:1904357","GO:0006974","GO:0031167","GO:0000956","GO:0006913","GO:0007062","GO:0019439","GO:0071427","GO:0006406","GO:0045910","GO:0051252","GO:0072711"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[19.4503282144806,17.9934386332075,17.6643246110046,17.7168500269786,18.4643827493734,18.3638334737416,18.6289175161382,21.2824953778867,21.4000311246093,21.5053658734699,21.5472727056056,21.5390620738563,21.5934484800124,21.6537323834277,21.5872057726815,15.6697988614665,20.9024525251713,15.7151677983179,18.4239673394241,15.0486221338682,18.6214455917298,14.0509285588584,18.1044082182465,19.2759171540558,18.0929757080159,18.0929757080159,21.7035374999992,14.4581139285914,14.5086078145015,22.1262884949501,22.1773231312919,22.0960714860976,22.2070086720384,19.73606939506,13.6808214449962,13.9127623976552,12.6954028668369,22.2506677513959,17.4104643922402,12.6954028668369,12.6954028668369,22.298075013352,11.4827422948031,16.7794477421486,15.4190791173213,14.5086078145015,16.8507929922937,18.1993537615243,10.8097872173079,16.8995454455618,18.181645451538,17.3572475673179,17.2289935746564,16.121131665261,17.2803636129636,16.8995454455618,17.0098206791988,17.8347771324352,18.0382058611615,15.8836369590203,19.4071640563441,17.8672182940543,17.1279572175344,9.79200637188604,9.79200637188604,16.6126778615783,16.624451291171,14.3519997480295,13.9127623976552,17.2731463069687,18.3389367682336,16.8705055778228,13.5956642813554,16.309816295931,18.0658885682371,13.9127623976552,17.1440106914419,15.8636236908885,18.0891416461647,16.2649666785139,16.2649666785139,19.2150634219582,16.0697362528688,18.5253260577865,17.8062984506393,12.6954028668369,15.7373316539529,16.3815087539172,15.7806675338474,13.6808214449962,18.195832112632,16.6008025798044,12.187219027138,15.5745906356112,15.1825444005545,14.1782205386673,17.265889291748,15.9797164726717,17.0098206791988,12.5414277899272,19.9953323018734,17.5582210891217,18.6510623608111,13.5059251363754,17.1279572175344,17.1279572175344,12.8370677538266,17.1519653138816,16.840827913987,15.4461643032416,11.747980750738,15.9981813072674,18.0775688230189,13.4111152730112,18.7197942104249,14.2961523059962,15.5245580341656,21.1344462088703,16.3532722717548,14.3519997480295,14.3519997480295,14.3519997480295,14.3519997480295,13.0900518224006,13.9127623976552,13.2039079443259,13.2039079443259,13.2039079443259,13.2039079443259,15.7591617451593,12.6954028668369,18.0967981119871,21.1308676162256,16.5273136649716,14.6957136866834,18.8171599959616,18.8298494843551,15.7591617451593,14.0509285588584,19.2959963103746,13.7618192670139,17.3572475673179,18.0658885682371,15.2452846314144,18.8568934935132,16.3815087539172,16.3815087539172,15.7591617451593,21.0519171992385,12.187219027138],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData4d95451b","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[53.3013761862114,57.3363517375128,56.1529091875501,57.3138099746564,62.1001776211723,59.898598782194,68.5696635609685,53.0534167947906,52.9895484666974,59.894841821718,59.8798139798137,57.3814352632257,56.1604231085023,66.7625655719778,56.1641800689783,59.8910848612419,59.8835709402897,57.7796730736894,59.8760570193376,59.8723000588615,53.1886673719292,55.258752594245,52.9482219014606,53.1398268857403,56.1679370294544,55.2700234756732,53.3088901071635,52.0390374662511,62.9041671630517,58.7639967184202,56.878002559432,53.1999382533574,52.2982677391001,58.3056475403395,55.1422868194867,53.2675635419267,53.1510977671685,62.8966532420996,66.9128439910207,65.4513863658286,52.2606981343394,58.3018905798634,52.3020246995762,62.8741114792432,52.294510778624,62.554769838777,61.7507802968976,53.0872294390753,52.5049005652841,56.0589851756483,61.4877930635725,52.2719690157676,53.6132039057254,62.7538887440089,52.5086575257602,56.2468331994519,62.7501317835328,58.3469741055762,57.6632072989311,58.3432171451002,62.8778684397192,62.9004102025757,58.3394601846241,54.1880188585644,56.1716939899305,53.1285560043121,55.262509554721],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,14.9194881331851,14.7436358341714,14.0483307477744,13.4716157937197,10.4846618205682,9.46851468680786,8.59174453944546,8.21930800039331,8.1103062452707,6.76063431104484,6.640083208344,6.45721268131563,6.07219093924065,6.02190769526709,5.92112931392265,5.81643467656577,5.69427289269337,5.16079262379161,4.95589973472049,4.79628008079302,4.49341201627466,4.22462166469567,4.09111265302531,3.81413153959771,3.78312312059088,3.54685593761785,3.29577617464306,2.96511419197214,2.94401553081688,2.88145711972747,2.67548369843909,2.40394874586839,2.27448115880451,2.15141251434275,2.07504592331007,1.99592063990918,1.96340908798364,1.92217245038589,1.60701795580276,1.54671788752285,1.54671788752285,1.46523782007474,1.46523782007474,1.45825888304671,1.43722335218119,1.42474655983295,1.36134410293205,1.32556498619274],"text":["GO:0005730 (955) <br> nucleolus <br> 4.975e-78","GO:0031981 (4453) <br> nuclear lumen <br> 1.686e-68","GO:0030684 (79) <br> preribosome <br> 1.959e-68","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 2.159e-61","GO:0070013 (5579) <br> intracellular organelle lumen <br> 2.159e-61","GO:0043233 (5579) <br> organelle lumen <br> 2.159e-61","GO:1990904 (694) <br> ribonucleoprotein complex <br> 1.379e-58","GO:0005654 (4112) <br> nucleoplasm <br> 1.784e-52","GO:0005634 (7607) <br> nucleus <br> 4.606e-47","GO:0043232 (5094) <br> intracellular non-membrane-bounded organelle <br> 3.977e-36","GO:0043228 (5102) <br> non-membrane-bounded organelle <br> 4.845e-36","GO:0032040 (39) <br> small-subunit processome <br> 2.508e-33","GO:0030686 (28) <br> 90S preribosome <br> 2.389e-31","GO:0140513 (1236) <br> nuclear protein-containing complex <br> 2.361e-26","GO:0030687 (24) <br> preribosome, large subunit precursor <br> 6.764e-24","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 7.399e-24","GO:0043229 (13117) <br> intracellular organelle <br> 4.513e-19","GO:0032991 (5541) <br> protein-containing complex <br> 1.165e-18","GO:0043227 (13201) <br> membrane-bounded organelle <br> 1.435e-18","GO:0043226 (14027) <br> organelle <br> 1.204e-15","GO:0005694 (1795) <br> chromosome <br> 1.805e-15","GO:0016604 (816) <br> nuclear body <br> 8.947e-15","GO:0005622 (14808) <br> intracellular anatomical structure <br> 3.376e-14","GO:0005681 (192) <br> spliceosomal complex <br> 3.276e-11","GO:0030688 (15) <br> preribosome, small subunit precursor <br> 3.400e-10","GO:0016607 (413) <br> nuclear speck <br> 2.560e-09","GO:0005732 (24) <br> sno(s)RNA-containing ribonucleoprotein complex <br> 6.035e-09","GO:0000228 (237) <br> nuclear chromosome <br> 7.757e-09","GO:0071013 (85) <br> catalytic step 2 spliceosome <br> 1.735e-07","GO:0035145 (20) <br> exon-exon junction complex <br> 2.290e-07","GO:0031428 (7) <br> box C/D RNP complex <br> 3.490e-07","GO:0005697 (21) <br> telomerase holoenzyme complex <br> 8.469e-07","GO:0000793 (239) <br> condensed chromosome <br> 9.508e-07","GO:0034399 (132) <br> nuclear periphery <br> 1.199e-06","GO:0016363 (110) <br> nuclear matrix <br> 1.526e-06","GO:0005721 (23) <br> pericentric heterochromatin <br> 2.022e-06","GO:0005684 (93) <br> U2-type spliceosomal complex <br> 6.906e-06","GO:0071011 (52) <br> precatalytic spliceosome <br> 1.107e-05","GO:1902494 (1377) <br> catalytic complex <br> 1.599e-05","GO:0098687 (348) <br> chromosomal region <br> 3.211e-05","GO:0000775 (200) <br> chromosome, centromeric region <br> 5.962e-05","GO:0034388 (6) <br> Pwp2p-containing subcomplex of 90S preribosome <br> 8.108e-05","GO:0000794 (73) <br> condensed nuclear chromosome <br> 1.534e-04","GO:0071005 (50) <br> U2-type precatalytic spliceosome <br> 1.648e-04","GO:0000792 (76) <br> heterochromatin <br> 2.839e-04","GO:0070545 (3) <br> PeBoW complex <br> 5.061e-04","GO:0061574 (4) <br> ASAP complex <br> 1.084e-03","GO:0005665 (15) <br> RNA polymerase II, core complex <br> 1.138e-03","GO:0001650 (140) <br> fibrillar center <br> 1.314e-03","GO:0030496 (197) <br> midbody <br> 2.111e-03","GO:0051233 (36) <br> spindle midzone <br> 3.945e-03","GO:0000781 (160) <br> chromosome, telomeric region <br> 5.315e-03","GO:0005819 (390) <br> spindle <br> 7.056e-03","GO:0070776 (9) <br> MOZ/MORF histone acetyltransferase complex <br> 8.413e-03","GO:0001651 (2) <br> dense fibrillar component <br> 1.009e-02","GO:0030893 (7) <br> meiotic cohesin complex <br> 1.088e-02","GO:0070775 (10) <br> H3 histone acetyltransferase complex <br> 1.196e-02","GO:0034457 (3) <br> Mpp10 complex <br> 2.472e-02","GO:0032545 (2) <br> CURI complex <br> 2.840e-02","GO:0034456 (2) <br> UTP-C complex <br> 2.840e-02","GO:0071006 (12) <br> U2-type catalytic step 1 spliceosome <br> 3.426e-02","GO:0071012 (12) <br> catalytic step 1 spliceosome <br> 3.426e-02","GO:0034455 (3) <br> t-UTP complex <br> 3.481e-02","GO:0008278 (10) <br> cohesin complex <br> 3.654e-02","GO:0030689 (3) <br> Noc complex <br> 3.761e-02","GO:0005677 (9) <br> chromatin silencing complex <br> 4.352e-02","GO:0016605 (102) <br> PML body <br> 4.725e-02"],"key":["GO:0005730","GO:0031981","GO:0030684","GO:0031974","GO:0070013","GO:0043233","GO:1990904","GO:0005654","GO:0005634","GO:0043232","GO:0043228","GO:0032040","GO:0030686","GO:0140513","GO:0030687","GO:0043231","GO:0043229","GO:0032991","GO:0043227","GO:0043226","GO:0005694","GO:0016604","GO:0005622","GO:0005681","GO:0030688","GO:0016607","GO:0005732","GO:0000228","GO:0071013","GO:0035145","GO:0031428","GO:0005697","GO:0000793","GO:0034399","GO:0016363","GO:0005721","GO:0005684","GO:0071011","GO:1902494","GO:0098687","GO:0000775","GO:0034388","GO:0000794","GO:0071005","GO:0000792","GO:0070545","GO:0061574","GO:0005665","GO:0001650","GO:0030496","GO:0051233","GO:0000781","GO:0005819","GO:0070776","GO:0001651","GO:0030893","GO:0070775","GO:0034457","GO:0032545","GO:0034456","GO:0071006","GO:0071012","GO:0034455","GO:0008278","GO:0030689","GO:0005677","GO:0016605"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[19.3796840599657,21.2181834681373,15.8227421236023,21.4713489077623,21.4713489077623,21.4713489077623,18.9706470582724,21.1278301730825,21.8137488730278,21.3696524759062,21.3714121953266,14.6049645781774,13.9833131530912,19.7024866025059,13.6808214449962,22.3097189832527,22.4001723500672,21.4637267954854,22.4069336180039,22.4710947059109,20.1582119259402,19.17949942902,22.5281903038104,17.1987173630513,12.6954028668369,18.2810218264132,13.6808214449962,17.5047579060072,15.9420548318458,13.3106655321065,10.8097872173079,13.4111152730112,17.5168301728758,16.6361245292714,16.3532722717548,13.5956642813554,16.0870759222918,15.1170689685676,19.835758359278,18.0461774501309,17.2585921453936,10.3630624020686,15.692660304218,15.0486221338682,15.7591617451593,7.7791008157754,9.00890082870231,12.6954028668369,16.7260902965966,17.2364555998064,14.4581139285914,16.9279733014706,18.2028655123974,11.4827422948031,3.77952755905512,10.8097872173079,11.747980750738,7.7791008157754,3.77952755905512,3.77952755905512,12.187219027138,12.187219027138,7.7791008157754,11.747980750738,7.7791008157754,11.4827422948031,16.2342495236679],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData4d95451b","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[3.64155719716214,3.49505666926895,46.8817514684057,40.706190754139,3.64531362095428,5.77144948730158,9.36259063258078,46.2469158475352,46.2506722713273,20.2787581725194,16.4659880235044,9.08837169575506,2.12771840893252,46.8704821970293,22.3222527154398,20.3388609531935,38.873055943578,22.3072270202713,16.8679253692626,22.299714172687,25.8194832659157,25.9960351841459,26.9727053701005,27.4460147679093,40.8301527392793,3.51383878822962,27.3746427158588,24.4108243438658,10.5946976364004,46.2581851189116,4.1599436804765,2.76631045359514,2.77006687738728,24.1253361356637,40.9841661147568,3.65658289233068,8.99070467715961,16.1128841870439,19.2607673248514,16.0865892204989,16.8491432503019,46.1830566430689,25.5001872435844,3.60399295924081],"y":[17,17,17,17,17,17,17,17,17,14.634248476706,12.9157673417434,6.62760357457422,6.44785377279137,6.4433872335449,6.33412569924493,6.27618033274108,5.83583495736509,5.40422133911544,5.33760558911171,5.32877261342632,5.18710195714763,5.06302321496034,4.82145461190381,4.16064528702312,3.9485367558876,3.6695582304171,3.47497547915932,3.12675233535191,2.46352041905676,2.46352041905676,2.31106414198618,2.2404923817323,2.2404923817323,2.21239856574095,2.21239856574095,2.19082519299462,2.14366432712919,2.03735092792906,1.96293626894831,1.74913125042838,1.48201071301258,1.47921638927889,1.47921638927889,1.40877100562727],"text":["GO:0003723 (1951) <br> RNA binding <br> 2.010e-80","GO:0003676 (4252) <br> nucleic acid binding <br> 1.867e-53","GO:1901363 (6210) <br> heterocyclic compound binding <br> 3.641e-38","GO:0097159 (6291) <br> organic cyclic compound binding <br> 2.346e-37","GO:0003724 (76) <br> RNA helicase activity <br> 8.309e-21","GO:0004386 (163) <br> helicase activity <br> 1.687e-19","GO:0008094 (185) <br> ATPase, acting on DNA <br> 1.358e-18","GO:0140097 (299) <br> catalytic activity, acting on DNA <br> 8.695e-17","GO:0140098 (394) <br> catalytic activity, acting on RNA <br> 9.507e-17","GO:0030515 (32) <br> snoRNA binding <br> 2.321e-15","GO:0016887 (553) <br> ATPase <br> 1.214e-13","GO:0005524 (1500) <br> ATP binding <br> 2.357e-07","GO:0000166 (2176) <br> nucleotide binding <br> 3.566e-07","GO:1901265 (2177) <br> nucleoside phosphate binding <br> 3.603e-07","GO:0032559 (1564) <br> adenyl ribonucleotide binding <br> 4.633e-07","GO:0030554 (1577) <br> adenyl nucleotide binding <br> 5.294e-07","GO:0070034 (22) <br> telomerase RNA binding <br> 1.459e-06","GO:0032555 (1917) <br> purine ribonucleotide binding <br> 3.943e-06","GO:0017076 (1932) <br> purine nucleotide binding <br> 4.596e-06","GO:0032553 (1934) <br> ribonucleotide binding <br> 4.691e-06","GO:0035639 (1846) <br> purine ribonucleoside triphosphate binding <br> 6.500e-06","GO:0036094 (2516) <br> small molecule binding <br> 8.649e-06","GO:0042393 (240) <br> histone binding <br> 1.509e-05","GO:0043168 (2413) <br> anion binding <br> 6.908e-05","GO:0097367 (2282) <br> carbohydrate derivative binding <br> 1.126e-04","GO:0003682 (573) <br> chromatin binding <br> 2.140e-04","GO:0043021 (150) <br> ribonucleoprotein complex binding <br> 3.350e-04","GO:0034511 (10) <br> U3 snoRNA binding <br> 7.469e-04","GO:0008649 (23) <br> rRNA methyltransferase activity <br> 3.439e-03","GO:0140102 (23) <br> catalytic activity, acting on a rRNA <br> 3.439e-03","GO:0003899 (43) <br> DNA-directed 5'-3' RNA polymerase activity <br> 4.886e-03","GO:0001098 (70) <br> basal transcription machinery binding <br> 5.748e-03","GO:0001099 (70) <br> basal RNA polymerase II transcription machinery binding <br> 5.748e-03","GO:0034062 (45) <br> 5'-3' RNA polymerase activity <br> 6.132e-03","GO:0097747 (45) <br> RNA polymerase activity <br> 6.132e-03","GO:0003727 (91) <br> single-stranded RNA binding <br> 6.444e-03","GO:0005488 (17070) <br> binding <br> 7.183e-03","GO:0016787 (2357) <br> hydrolase activity <br> 9.176e-03","GO:0019843 (68) <br> rRNA binding <br> 1.089e-02","GO:0016779 (134) <br> nucleotidyltransferase activity <br> 1.782e-02","GO:0017069 (47) <br> snRNA binding <br> 3.296e-02","GO:0140034 (73) <br> methylation-dependent protein binding <br> 3.317e-02","GO:0035064 (73) <br> methylated histone binding <br> 3.317e-02","GO:0003712 (509) <br> transcription coregulator activity <br> 3.901e-02"],"key":["GO:0003723","GO:0003676","GO:1901363","GO:0097159","GO:0003724","GO:0004386","GO:0008094","GO:0140097","GO:0140098","GO:0030515","GO:0016887","GO:0005524","GO:0000166","GO:1901265","GO:0032559","GO:0030554","GO:0070034","GO:0032555","GO:0017076","GO:0032553","GO:0035639","GO:0036094","GO:0042393","GO:0043168","GO:0097367","GO:0003682","GO:0043021","GO:0034511","GO:0008649","GO:0140102","GO:0003899","GO:0001098","GO:0001099","GO:0034062","GO:0097747","GO:0003727","GO:0005488","GO:0016787","GO:0019843","GO:0016779","GO:0017069","GO:0140034","GO:0035064","GO:0003712"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[20.258276019327,21.1658574902378,21.5904209757188,21.6047679255868,15.7591617451593,16.9558134205769,17.1440106914419,17.8347771324352,18.2168145994675,14.2382740983495,18.6728117067331,19.9405304557382,20.38841785752,20.3889635367376,19.9914490344426,20.001517942691,13.5059251363754,20.2372173548861,20.2465569336091,20.2477963305085,20.1919193878134,20.5599632179123,17.5228245316566,20.5107536570554,20.4448133217894,18.7197942104249,16.8307889373762,11.747980750738,13.5956642813554,13.5956642813554,14.781443852525,15.6229705518591,15.6229705518591,14.862657240381,14.862657240381,16.0521821733416,22.6771653543307,20.4830462803831,15.5745906356112,16.6591768768421,14.9397854919929,15.692660304218,15.692660304218,18.5625892117027],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData4d95451b","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[199.924846290867,199.936119123303,199.928603901679,195.321773046176,195.411955705664,199.83466363138,199.830906020568,196.734634711485,199.071868636545,198.200102928163,190.714942190673,199.812117966508,191.797134104526,197.925797338887,193.657151456463,197.099122960249,197.940827782135,199.838421242192,199.81587557732,193.664666678087,193.852547218687,193.860062440311,198.211375760599],"y":[17,17,17,17,17,8.94242979624834,8.56914886632383,8.22331360482664,3.35392375664656,3.10580267789165,2.7430503332594,2.73564549375898,2.50225670474723,2.36800707533361,2.34345553696702,2.24237892928728,2.20566747225093,2.18967957247374,2.08922175121329,2.01133027226929,1.86434835922475,1.86434835922475,1.48092578442721],"text":["REAC:R-HSA-6790901 (60) <br> rRNA modification in the nucleus and cytosol <br> 1.593e-44","REAC:R-HSA-8868773 (193) <br> rRNA processing in the nucleus and cytosol <br> 2.550e-40","REAC:R-HSA-72312 (203) <br> rRNA processing <br> 3.998e-39","REAC:R-HSA-6791226 (183) <br> Major pathway of rRNA processing in the nucleolus and cytosol <br> 3.675e-36","REAC:R-HSA-8953854 (662) <br> Metabolism of RNA <br> 1.219e-34","REAC:R-HSA-72163 (178) <br> mRNA Splicing - Major Pathway <br> 1.142e-09","REAC:R-HSA-72172 (186) <br> mRNA Splicing <br> 2.697e-09","REAC:R-HSA-72203 (237) <br> Processing of Capped Intron-Containing Pre-mRNA <br> 5.980e-09","REAC:R-HSA-157579 (112) <br> Telomere Maintenance <br> 4.427e-04","REAC:R-HSA-8851708 (18) <br> Signaling by FGFR2 IIIa TM <br> 7.838e-04","REAC:R-HSA-167242 (21) <br> Abortive elongation of HIV-1 transcript in the absence of Tat <br> 1.807e-03","REAC:R-HSA-72187 (56) <br> mRNA 3'-end processing <br> 1.838e-03","REAC:R-HSA-73886 (139) <br> Chromosome Maintenance <br> 3.146e-03","REAC:R-HSA-3108232 (170) <br> SUMO E3 ligases SUMOylate target proteins <br> 4.285e-03","REAC:R-HSA-6803529 (25) <br> FGFR2 alternative splicing <br> 4.535e-03","REAC:R-HSA-73856 (65) <br> RNA Polymerase II Transcription Termination <br> 5.723e-03","REAC:R-HSA-2990846 (176) <br> SUMOylation <br> 6.228e-03","REAC:R-HSA-72165 (51) <br> mRNA Splicing - Minor Pathway <br> 6.461e-03","REAC:R-HSA-72086 (28) <br> mRNA Capping <br> 8.143e-03","REAC:R-HSA-1839126 (29) <br> FGFR2 mutant receptor activation <br> 9.742e-03","REAC:R-HSA-113418 (31) <br> Formation of the Early Elongation Complex <br> 1.367e-02","REAC:R-HSA-167158 (31) <br> Formation of the HIV-1 Early Elongation Complex <br> 1.367e-02","REAC:R-HSA-5655253 (37) <br> Signaling by FGFR2 in disease <br> 3.304e-02"],"key":["REAC:R-HSA-6790901","REAC:R-HSA-8868773","REAC:R-HSA-72312","REAC:R-HSA-6791226","REAC:R-HSA-8953854","REAC:R-HSA-72163","REAC:R-HSA-72172","REAC:R-HSA-72203","REAC:R-HSA-157579","REAC:R-HSA-8851708","REAC:R-HSA-167242","REAC:R-HSA-72187","REAC:R-HSA-73886","REAC:R-HSA-3108232","REAC:R-HSA-6803529","REAC:R-HSA-73856","REAC:R-HSA-2990846","REAC:R-HSA-72165","REAC:R-HSA-72086","REAC:R-HSA-1839126","REAC:R-HSA-113418","REAC:R-HSA-167158","REAC:R-HSA-5655253"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,102,204,1)","opacity":0.8,"size":[15.3633750239743,17.206351575782,17.2803636129636,17.1279572175344,18.90922365592,17.0869555296552,17.1519653138816,17.5047579060072,16.3815087539172,13.0900518224006,13.4111152730112,15.2452846314144,16.715163312453,17.01861417614,13.7618192670139,15.498887690643,17.0701951198665,15.0832328994516,13.9833131530912,14.0509285588584,14.1782205386673,14.1782205386673,14.5086078145015],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,102,204,1)"}},"hoveron":"points","set":"SharedData4d95451b","name":"REAC","legendgroup":"REAC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"19fc77114d14":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"19fc73594e92":{"x":{},"xend":{},"y":{},"yend":{}},"19fc31e44013":{"x":{},"xend":{},"y":{},"yend":{}},"19fc213078e6":{"x":{},"xend":{},"y":{},"yend":{}},"19fc6b31784e":{"x":{},"xend":{},"y":{},"yend":{}},"19fc2c7c3146":{"x":{},"y":{}},"19fc4210156c":{"yintercept":{}}},"cur_data":"19fc77114d14","visdat":{"19fc77114d14":["function (y) ","x"],"19fc73594e92":["function (y) ","x"],"19fc31e44013":["function (y) ","x"],"19fc213078e6":["function (y) ","x"],"19fc6b31784e":["function (y) ","x"],"19fc2c7c3146":["function (y) ","x"],"19fc4210156c":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData4d95451b"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results-1" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-2b7856457827ab1876f7" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2b7856457827ab1876f7">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284"],["GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:MF","GO:CC","GO:CC","REAC","GO:BP","REAC","GO:BP","GO:BP","REAC","GO:MF","GO:BP","GO:BP","GO:MF","REAC","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:CC","GO:CC","GO:MF","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","REAC","GO:BP","GO:BP","GO:BP","GO:CC","REAC","REAC","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:MF","GO:MF","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:BP","GO:MF","GO:MF","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","REAC","REAC","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:MF","GO:MF","GO:MF","GO:MF","REAC","GO:BP","GO:BP","GO:MF","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","REAC","GO:MF","GO:MF","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC"],[2400,11668,438,2374,378,5689,7628,1452,1137,9964,1446,2720,2134,4442,10040,399,312,295,2496,5687,2499,21373,2173,2497,11949,2705,14204,10305,1271,2133,2129,24636,1295,10027,1464,1139,4661,11681,3961,12668,11682,3621,7894,2401,1140,2132,214,154,5688,439,156,168,1005,2130,1570,1961,2128,19815,204,206,11780,11781,2776,2127,348,4867,899,12726,284,12224,3852,12725,15771,210,213,207,335,19739,1141,2472,2407,223,222,902,2471,1647,380,42,3544,2934,1832,1888,191,15312,1330,11669,35,11946,5411,4883,351,111,1710,9817,868,369,68,5407,3959,5405,12399,15589,6342,224,338,6389,8196,26352,2932,6649,303,4001,7633,26353,3612,19802,21522,257,101,6775,9234,1709,22986,10338,8608,2286,21374,8604,343,216,112,2926,19510,19292,2367,404,14,110,15683,2405,6756,15590,13694,14544,2985,2269,9278,32,2841,21523,19280,8197,5967,2365,2037,2343,30393,120,22168,29432,2940,2627,321,29478,2303,166,15281,2406,2409,8199,45,2466,1112,2370,8614,15314,19853,4980,3953,12207,333,13686,2289,11783,2341,14707,2557,1964,828,14545,576,15294,15295,104,5691,1744,15740,205,206,5891,10379,1968,15569,27307,442,2473,29434,2377,15684,8615,461,1862,10037,2379,12404,2467,2933,2894,22155,11002,14676,3954,20459,3758,27777,830,167,2989,1162,4596,21619,21610,21609,21608,2893,880,882,25568,25713,4797,3751,6493,75,14401,14349,1721,12746,159,27308,2917,8232,1539,1720,332,2866,3954,2040,11763,6257,2987,2927,2933,1719,6697,19511,2410,614,13662,1142,428,15753,332,20475,900],["RNA processing","ribosome biogenesis","RNA binding","rRNA processing","nucleolus","rRNA metabolic process","ribonucleoprotein complex biogenesis","nuclear lumen","preribosome","ncRNA processing","membrane-enclosed lumen","intracellular organelle lumen","organelle lumen","ribonucleoprotein complex","ncRNA metabolic process","nucleic acid binding","nucleoplasm","nucleus","rRNA modification in the nucleus and cytosol","RNA metabolic process","rRNA processing in the nucleus and cytosol","nucleic acid metabolic process","nucleobase-containing compound metabolic process","rRNA processing","heterocyclic compound binding","cellular aromatic compound metabolic process","heterocycle metabolic process","organic cyclic compound binding","Major pathway of rRNA processing in the nucleolus and cytosol","intracellular non-membrane-bounded organelle","non-membrane-bounded organelle","organic cyclic compound metabolic process","Metabolism of RNA","cellular nitrogen compound metabolic process","small-subunit processome","90S preribosome","gene expression","ribosomal large subunit biogenesis","nuclear protein-containing complex","cellular component biogenesis","ribosomal small subunit biogenesis","RNA splicing","maturation of SSU-rRNA","mRNA processing","preribosome, large subunit precursor","intracellular membrane-bounded organelle","maturation of LSU-rRNA","RNA splicing, via transesterification reactions","mRNA metabolic process","RNA helicase activity","RNA splicing, via transesterification reactions with bulged adenosine as nucleophile","mRNA splicing, via spliceosome","helicase activity","intracellular organelle","protein-containing complex","ATPase, acting on DNA","membrane-bounded organelle","cellular component organization or biogenesis","maturation of 5.8S rRNA","maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","catalytic activity, acting on DNA","catalytic activity, acting on RNA","nitrogen compound metabolic process","organelle","chromosome","snoRNA binding","nuclear body","primary metabolic process","intracellular anatomical structure","macromolecule metabolic process","ATPase","cellular metabolic process","chromosome organization","maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","cleavage involved in rRNA processing","maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","spliceosomal complex","organic substance metabolic process","preribosome, small subunit precursor","mRNA Splicing - Major Pathway","RNA localization","endonucleolytic cleavage of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","endonucleolytic cleavage involved in rRNA processing","nuclear speck","mRNA Splicing","Processing of Capped Intron-Containing Pre-mRNA","sno(s)RNA-containing ribonucleoprotein complex","nuclear chromosome","metabolic process","catalytic step 2 spliceosome","exon-exon junction complex","ATP binding","endonucleolytic cleavage in ITS1 to separate SSU-rRNA from 5.8S rRNA and LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","regulation of mRNA processing","box C/D RNP complex","ribosome assembly","nucleotide binding","nucleoside phosphate binding","adenyl ribonucleotide binding","adenyl nucleotide binding","telomerase holoenzyme complex","condensed chromosome","nuclear periphery","telomerase RNA binding","nuclear matrix","pericentric heterochromatin","rRNA modification","purine ribonucleotide binding","purine nucleotide binding","ribonucleotide binding","regulation of RNA splicing","regulation of DNA metabolic process","purine ribonucleoside triphosphate binding","endonucleolytic cleavage in 5'-ETS of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","U2-type spliceosomal complex","small molecule binding","RNA 3'-end processing","regulation of mRNA metabolic process","precatalytic spliceosome","histone binding","sister chromatid segregation","catalytic complex","ribonucleoprotein complex assembly","negative regulation of mRNA metabolic process","chromosomal region","ribonucleoprotein complex subunit organization","RNA phosphodiester bond hydrolysis","telomere maintenance","chromosome, centromeric region","anion binding","regulation of chromosome organization","Pwp2p-containing subcomplex of 90S preribosome","mitotic nuclear division","carbohydrate derivative binding","regulation of telomere maintenance","DNA metabolic process","nucleic acid phosphodiester bond hydrolysis","telomere organization","rRNA 5'-end processing","endonucleolytic cleavage to generate mature 5'-end of SSU-rRNA from (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","condensed nuclear chromosome","U2-type precatalytic spliceosome","ribonucleoprotein complex export from nucleus","ribonucleoprotein complex localization","regulation of transcription by RNA polymerase I","chromatin binding","ribosomal large subunit assembly","heterochromatin","nuclear export","RNA catabolic process","ribonucleoprotein complex binding","negative regulation of DNA metabolic process","positive regulation of transcription by RNA polymerase I","regulation of mRNA splicing, via spliceosome","chromosome segregation","Telomere Maintenance","negative regulation of RNA splicing","mitotic sister chromatid segregation","PeBoW complex","RNA phosphodiester bond hydrolysis, endonucleolytic","DNA conformation change","mRNA 3'-end processing","U3 snoRNA binding","DNA-templated transcription, elongation","Signaling by FGFR2 IIIa TM","chromatin organization","negative regulation of chromosome organization","nuclear division","nuclear chromosome segregation","regulation of rRNA processing","telomere maintenance via telomerase","ASAP complex","RNA polymerase II, core complex","regulation of DNA biosynthetic process","RNA-dependent DNA biosynthetic process","fibrillar center","positive regulation of viral transcription","mRNA catabolic process","RNA export from nucleus","sno(s)RNA 3'-end processing","Abortive elongation of HIV-1 transcript in the absence of Tat","mRNA 3'-end processing","midbody","transcription by RNA polymerase I","regulation of telomere maintenance via telomerase","negative regulation of mRNA processing","DNA biosynthetic process","telomere maintenance via telomere lengthening","RNA modification","sno(s)RNA processing","Chromosome Maintenance","negative regulation of nucleobase-containing compound metabolic process","rRNA methyltransferase activity","catalytic activity, acting on a rRNA","DNA packaging","organelle fission","spindle midzone","SUMO E3 ligases SUMOylate target proteins","FGFR2 alternative splicing","negative regulation of mRNA splicing, via spliceosome","DNA-directed 5'-3' RNA polymerase activity","nucleic acid transport","RNA transport","chromosome, telomeric region","sno(s)RNA metabolic process","RNA Polymerase II Transcription Termination","establishment of RNA localization","basal transcription machinery binding","basal RNA polymerase II transcription machinery binding","5'-3' RNA polymerase activity","RNA polymerase activity","SUMOylation","mRNA transport","regulation of telomere maintenance via telomere lengthening","single-stranded RNA binding","mRNA Splicing - Minor Pathway","positive regulation of rRNA processing","transcription elongation from RNA polymerase II promoter","nuclear transport","negative regulation of telomere maintenance via telomerase","spindle","binding","nucleobase-containing compound catabolic process","7-methylguanosine mRNA capping","RNA stabilization","mRNA Capping","organelle organization","MOZ/MORF histone acetyltransferase complex","ncRNA transcription","RNA capping","negative regulation of DNA recombination at telomere","7-methylguanosine RNA capping","regulation of DNA recombination at telomere","hydrolase activity","regulation of telomerase RNA localization to Cajal body","FGFR2 mutant receptor activation","dense fibrillar component","mitotic sister chromatid cohesion","meiotic cohesin complex","rRNA binding","RNA localization to nucleus","telomerase RNA localization","telomerase RNA localization to Cajal body","RNA localization to Cajal body","H3 histone acetyltransferase complex","Formation of the Early Elongation Complex","Formation of the HIV-1 Early Elongation Complex","negative regulation of RNA catabolic process","protein localization to nucleolus","negative regulation of organelle organization","nucleotidyltransferase activity","regulation of nucleobase-containing compound metabolic process","nuclear-transcribed mRNA catabolic process, nonsense-mediated decay","regulation of viral transcription","heterocycle catabolic process","Mpp10 complex","cellular nitrogen compound catabolic process","alternative mRNA splicing, via spliceosome","negative regulation of telomere maintenance via telomere lengthening","cellular response to DNA damage stimulus","rRNA methylation","CURI complex","UTP-C complex","nuclear-transcribed mRNA catabolic process","nucleocytoplasmic transport","snRNA binding","Signaling by FGFR2 in disease","methylation-dependent protein binding","methylated histone binding","sister chromatid cohesion","U2-type catalytic step 1 spliceosome","catalytic step 1 spliceosome","t-UTP complex","aromatic compound catabolic process","mRNA-containing ribonucleoprotein complex export from nucleus","mRNA export from nucleus","cohesin complex","negative regulation of DNA recombination","Noc complex","transcription coregulator activity","regulation of RNA metabolic process","chromatin silencing complex","cellular response to hydroxyurea","PML body"],["3.5e-91","1.8e-80","2.0e-80","3.2e-79","5.0e-78","7.0e-78","5.1e-75","1.7e-68","2.0e-68","1.3e-64","2.2e-61","2.2e-61","2.2e-61","1.4e-58","6.1e-58","1.9e-53","1.8e-52","4.6e-47","1.6e-44","6.2e-42","2.5e-40","4.4e-40","1.6e-39","4.0e-39","3.6e-38","5.0e-38","1.3e-37","2.3e-37","3.7e-36","4.0e-36","4.8e-36","2.0e-35","1.2e-34","1.6e-34","2.5e-33","2.4e-31","6.0e-31","2.5e-29","2.4e-26","4.9e-26","8.2e-26","2.5e-25","3.8e-25","9.1e-25","6.8e-24","7.4e-24","3.8e-23","1.6e-21","3.6e-21","8.3e-21","1.5e-20","1.5e-20","1.7e-19","4.5e-19","1.2e-18","1.4e-18","1.4e-18","2.4e-17","3.4e-17","4.9e-17","8.7e-17","9.5e-17","2.7e-16","1.2e-15","1.8e-15","2.3e-15","8.9e-15","1.2e-14","3.4e-14","1.2e-13","1.2e-13","1.4e-13","7.9e-13","1.3e-12","5.3e-12","2.4e-11","3.3e-11","4.0e-11","3.4e-10","1.1e-09","2.0e-09","2.4e-09","2.4e-09","2.6e-09","2.7e-09","6.0e-09","6.0e-09","7.8e-09","1.5e-08","1.7e-07","2.3e-07","2.4e-07","2.8e-07","3.0e-07","3.5e-07","3.6e-07","3.6e-07","3.6e-07","4.6e-07","5.3e-07","8.5e-07","9.5e-07","1.2e-06","1.5e-06","1.5e-06","2.0e-06","3.9e-06","3.9e-06","4.6e-06","4.7e-06","6.1e-06","6.2e-06","6.5e-06","6.6e-06","6.9e-06","8.6e-06","8.8e-06","9.4e-06","1.1e-05","1.5e-05","1.6e-05","1.6e-05","2.5e-05","2.7e-05","3.2e-05","3.9e-05","4.4e-05","5.4e-05","6.0e-05","6.9e-05","7.3e-05","8.1e-05","8.5e-05","1.1e-04","1.1e-04","1.1e-04","1.2e-04","1.2e-04","1.5e-04","1.5e-04","1.5e-04","1.6e-04","1.7e-04","1.8e-04","2.0e-04","2.1e-04","2.4e-04","2.8e-04","2.8e-04","3.3e-04","3.3e-04","3.5e-04","3.8e-04","3.9e-04","4.0e-04","4.4e-04","4.8e-04","4.9e-04","5.1e-04","5.6e-04","7.3e-04","7.4e-04","7.5e-04","7.8e-04","7.8e-04","8.2e-04","8.5e-04","9.0e-04","9.8e-04","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.5e-03","1.6e-03","1.8e-03","1.8e-03","2.1e-03","2.2e-03","2.6e-03","2.7e-03","2.8e-03","3.0e-03","3.0e-03","3.1e-03","3.1e-03","3.2e-03","3.4e-03","3.4e-03","3.9e-03","3.9e-03","3.9e-03","4.3e-03","4.5e-03","4.6e-03","4.9e-03","5.0e-03","5.0e-03","5.3e-03","5.7e-03","5.7e-03","5.7e-03","5.7e-03","5.7e-03","6.1e-03","6.1e-03","6.2e-03","6.3e-03","6.4e-03","6.4e-03","6.5e-03","6.8e-03","6.9e-03","6.9e-03","7.0e-03","7.1e-03","7.2e-03","7.3e-03","7.4e-03","7.7e-03","8.1e-03","8.3e-03","8.4e-03","8.5e-03","8.6e-03","8.6e-03","8.6e-03","8.6e-03","9.2e-03","9.4e-03","9.7e-03","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.4e-02","1.4e-02","1.5e-02","1.6e-02","1.6e-02","1.8e-02","1.9e-02","2.0e-02","2.2e-02","2.3e-02","2.5e-02","2.6e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.8e-02","2.8e-02","2.8e-02","2.9e-02","3.3e-02","3.3e-02","3.3e-02","3.3e-02","3.3e-02","3.4e-02","3.4e-02","3.5e-02","3.6e-02","3.6e-02","3.6e-02","3.7e-02","3.7e-02","3.8e-02","3.9e-02","4.3e-02","4.4e-02","4.4e-02","4.7e-02"],[1010,335,1951,265,955,275,473,4453,79,439,5579,5579,5579,694,535,4252,4112,7607,60,4714,193,5234,5752,203,6210,5973,5929,6291,183,5094,5102,6227,662,6576,39,28,6192,72,1236,3377,74,459,50,532,24,12046,29,363,880,76,360,360,163,13117,5541,185,13201,6880,36,37,299,394,10148,14027,1795,32,816,10642,14808,9867,553,10941,1270,24,27,15,192,11397,15,178,222,15,15,413,186,237,24,237,11915,85,20,1500,9,145,7,62,2176,2177,1564,1577,21,239,132,22,110,23,37,1917,1932,1934,152,389,1846,7,93,2516,157,384,52,240,214,1377,196,95,348,203,157,169,200,2413,299,6,346,2282,82,976,306,183,5,5,73,50,130,131,34,573,27,76,202,431,150,154,23,107,353,112,27,185,3,81,359,104,10,104,18,839,92,495,293,15,75,4,15,112,77,140,24,388,129,12,21,56,197,68,54,31,201,87,169,14,139,1569,23,23,246,544,36,170,25,22,43,183,183,160,16,65,186,70,70,45,45,176,151,63,91,51,10,88,356,21,390,17070,573,33,66,28,4136,9,110,34,34,34,34,2357,18,29,2,27,7,68,19,19,19,19,10,31,31,76,15,361,134,4123,123,41,617,3,623,76,29,894,25,2,2,214,353,47,37,73,73,56,12,12,3,636,112,112,10,76,3,509,3847,9,12,102],[120,79,138,73,106,73,84,172,46,75,180,180,180,81,75,156,152,188,36,153,50,158,164,50,165,165,164,165,46,147,147,165,74,168,23,20,159,26,67,111,24,43,21,45,16,200,17,36,54,14,35,35,16,202,128,16,202,146,15,15,17,27,180,204,58,13,40,182,207,173,18,183,51,11,11,9,19,183,8,21,16,8,8,25,21,23,8,19,183,11,7,18,6,15,5,10,27,27,18,18,6,16,13,6,12,4,7,24,24,24,14,17,18,5,10,27,14,21,8,15,15,36,14,11,17,14,13,11,6,19,12,4,18,24,8,31,17,11,4,4,8,7,11,11,7,21,6,4,13,20,11,10,6,10,17,10,6,12,3,9,17,10,4,10,5,17,7,20,15,5,7,3,4,8,7,10,5,18,10,4,5,8,8,8,6,6,10,7,9,4,10,38,4,4,13,20,4,13,5,5,5,9,9,8,4,8,9,7,7,5,5,13,10,6,5,7,4,7,15,4,14,150,21,5,5,5,43,3,8,5,5,5,5,17,4,5,2,5,3,6,4,4,4,4,3,5,5,5,4,9,7,76,8,5,21,2,21,7,4,7,4,2,2,11,14,5,5,6,6,6,3,3,2,21,8,8,3,6,2,16,71,3,2,3]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap-1" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp-1" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/BP-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf-1" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/MF-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc-1" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/CC-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot-1" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/scatter-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="network-analysis" class="section level2">
<h2><strong>6. Network analysis</strong></h2>
<p>Overview from <strong>STRING Database</strong>: In STRING, each protein-protein interaction is annotated with one or more ‘scores’. Importantly, these scores do <strong>not</strong> indicate the strength or the specificity of the interaction. Instead, they are indicators of <strong>confidence</strong>, i.e. how likely STRING judges an interaction to be true, given the available evidence. All scores rank from 0 to 1, with 1 being the highest possible confidence. A score of 0.5 would indicate that roughly every second interaction might be erroneous (i.e., a false positive).</p>
<pre class="r"><code>if(leng_dep != 0){
  
  string_db &lt;- STRINGdb$new(version=&quot;11&quot;, species=9606, score_threshold=400)

  string_mapped &lt;- string_db$map(string_res, &quot;Gene.name&quot;, removeUnmappedRows = TRUE, quiet = T)

  hits &lt;- string_mapped$STRING_id[1:leng_dep]

  string_db$plot_network(hits, add_link = T)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/NSCAP2_vs_CSCAP2.Rmd/stringdb-1.png" width="1440" style="display: block; margin: auto;" /></p>
<div id="cytoscape-network" class="section level3">
<h3><strong>6.1. Cytoscape Network</strong></h3>
<p>Cytoscape is an open source software platform for visualizing complex networks and integrating these with any type of attribute data. In this presentation all proteins showing at least one confidence level from the STRING database are shown. The <strong>color</strong> code is relative to <strong>fold change</strong> and the <strong>size</strong> is directly proportional to the <strong>significance (p-value)</strong>.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>if(n_edge != 0){
  knitr::include_graphics(paste0(&quot;assets/network_FC_pvalue&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_FC_pvalueSKAP2_N._vs_SKAP2_C..png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
<div id="cytoscape-enrichmentmap" class="section level3">
<h3><strong>6.2. Cytoscape EnrichmentMap</strong></h3>
<p>An <strong>enrichment map</strong> is a different sort of network. Instead of nodes representing genes, nodes represent pathways or functions. Edges between these pathways or functions represent shared genes or pathway crosstalk. An enrichment map is a way to visualize your enrichment results to help reduce redundancy and uncover main themes.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant) %&gt;% pull(name)

if(!params$Imputation){
  
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  pull(name) %&gt;% 
  unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  
  print(&quot;NOT enough data.&quot;)
  
}else{
  
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  
  if(is.null(GO)){
    
    print(&quot;NOT enough data.&quot;)

    enrich_map = F
    
  }else{
    
    enrich_map = T
    
    gem &lt;- GO$result[,c(&quot;term_id&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;intersection&quot;)]
    colnames(gem) &lt;- c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;Genes&quot;)
    gem$FDR &lt;- gem$p.Val
    gem$Phenotype = &quot;+1&quot;
    gem &lt;- gem[,c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;FDR&quot;, &quot;Phenotype&quot;, &quot;Genes&quot;)]
    head(gem)
  
    write.table(gem, here::here(&quot;output/enrichmentmap.txt&quot;),col.name=TRUE,sep=&quot;\t&quot;,row.names=FALSE,quote=FALSE)
    
  }
}</code></pre>
<pre class="r"><code>if(enrich_map){
  knitr::include_graphics(paste0(&quot;assets/network_enrichmentmap&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_enrichmentmapSKAP2_N._vs_SKAP2_C..png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="section" class="section level2 unnumbered">
<h2></h2>
<pre class="r"><code>saveWorkbook(excel, here::here(paste0(&quot;output/results_table_&quot;, test, &quot;.xlsx&quot;)), overwrite = T)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] forcats_0.5.1               stringr_1.4.0              
 [3] dplyr_1.0.7                 purrr_0.3.4                
 [5] readr_1.4.0                 tidyr_1.1.3                
 [7] tibble_3.1.2                tidyverse_1.3.1            
 [9] openxlsx_4.2.4              here_1.0.1                 
[11] RColorBrewer_1.1-2          RCy3_2.10.2                
[13] STRINGdb_2.2.2              naniar_0.6.1               
[15] visdat_0.5.3                ggrepel_0.9.1              
[17] ggplot2_3.3.5               UpSetR_1.4.0               
[19] ggstatsplot_0.8.0           datapasta_3.1.0            
[21] patchwork_1.1.1             DT_0.18                    
[23] rrvgo_1.2.0                 org.Hs.eg.db_3.12.0        
[25] AnnotationDbi_1.52.0        gprofiler2_0.2.0           
[27] enrichR_3.0                 preprocessCore_1.52.1      
[29] SummarizedExperiment_1.20.0 Biobase_2.50.0             
[31] GenomicRanges_1.42.0        GenomeInfoDb_1.26.2        
[33] IRanges_2.24.1              S4Vectors_0.28.1           
[35] BiocGenerics_0.36.0         MatrixGenerics_1.2.1       
[37] matrixStats_0.59.0          DEP_1.12.0                 
[39] MBQN_2.2.0                  workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.3            R.methodsS3_1.8.1        
  [3] coda_0.19-4               bit64_4.0.5              
  [5] knitr_1.33                DelayedArray_0.16.2      
  [7] R.utils_2.10.1            data.table_1.14.0        
  [9] RCurl_1.98-1.3            doParallel_1.0.16        
 [11] generics_0.1.0            RSQLite_2.2.7            
 [13] correlation_0.6.1         chron_2.3-56             
 [15] bit_4.0.4                 lubridate_1.7.10         
 [17] xml2_1.3.2                httpuv_1.6.1             
 [19] assertthat_0.2.1          WRS2_1.1-2               
 [21] xfun_0.24                 hms_1.1.0                
 [23] jquerylib_0.1.4           evaluate_0.14            
 [25] promises_1.2.0.1          fansi_0.5.0              
 [27] readxl_1.3.1              caTools_1.18.2           
 [29] dbplyr_2.1.1              igraph_1.2.6             
 [31] DBI_1.1.1                 htmlwidgets_1.5.3        
 [33] reshape_0.8.8             kSamples_1.2-9           
 [35] Rmpfr_0.8-4               paletteer_1.3.0          
 [37] hash_2.2.6.1              ellipsis_0.3.2           
 [39] crosstalk_1.1.1           backports_1.2.1          
 [41] insight_0.14.2            gridBase_0.4-7           
 [43] vctrs_0.3.8               imputeLCMD_2.0           
 [45] Cairo_1.5-12.2            cachem_1.0.5             
 [47] withr_2.4.2               treemap_2.4-2            
 [49] cluster_2.1.0             lazyeval_0.2.2           
 [51] crayon_1.4.1              labeling_0.4.2           
 [53] pkgconfig_2.0.3           SuppDists_1.1-9.5        
 [55] slam_0.1-48               statsExpressions_1.1.0   
 [57] ProtGenerics_1.22.0       wordcloud_2.6            
 [59] rlang_0.4.11              RJSONIO_1.3-1.4          
 [61] lifecycle_1.0.0           MatrixModels_0.5-0       
 [63] sandwich_3.0-1            affyio_1.60.0            
 [65] BiocFileCache_1.14.0      modelr_0.1.8             
 [67] cellranger_1.1.0          rprojroot_2.0.2          
 [69] graph_1.68.0              Matrix_1.2-18            
 [71] mc2d_0.1-21               zoo_1.8-9                
 [73] reprex_2.0.0              whisker_0.4              
 [75] GlobalOptions_0.1.2       pheatmap_1.0.12          
 [77] png_0.1-7                 viridisLite_0.4.0        
 [79] rjson_0.2.20              mzR_2.24.1               
 [81] PMCMRplus_1.9.0           parameters_0.14.0        
 [83] bitops_1.0-7              shinydashboard_0.7.1     
 [85] R.oo_1.24.0               KernSmooth_2.23-17       
 [87] blob_1.2.1                shape_1.4.6              
 [89] multcompView_0.1-8        tmvtnorm_1.4-10          
 [91] ggsignif_0.6.2            scales_1.1.1             
 [93] memoise_2.0.0             magrittr_2.0.1           
 [95] plyr_1.8.6                gplots_3.1.1             
 [97] zlibbioc_1.36.0           compiler_4.0.3           
 [99] plotrix_3.8-1             pcaMethods_1.82.0        
[101] clue_0.3-59               cli_3.0.0                
[103] affy_1.68.0               XVector_0.30.0           
[105] pbapply_1.4-3             MASS_7.3-53              
[107] tidyselect_1.1.1          vsn_3.58.0               
[109] stringi_1.6.2             highr_0.9                
[111] yaml_2.2.1                GOSemSim_2.16.1          
[113] norm_1.0-9.5              MALDIquant_1.19.3        
[115] grid_4.0.3                sass_0.4.0               
[117] tools_4.0.3               rstudioapi_0.13          
[119] circlize_0.4.13           foreach_1.5.1            
[121] git2r_0.28.0              gridExtra_2.3            
[123] ipmisc_6.0.2              pairwiseComparisons_3.1.6
[125] farver_2.1.0              mzID_1.28.0              
[127] digest_0.6.27             BiocManager_1.30.16      
[129] shiny_1.6.0               proto_1.0.0              
[131] BWStest_0.2.2             Rcpp_1.0.6               
[133] broom_0.7.8               BayesFactor_0.9.12-4.2   
[135] performance_0.7.2         later_1.2.0              
[137] ncdf4_1.17                httr_1.4.2               
[139] MSnbase_2.15.7            ComplexHeatmap_2.6.2     
[141] effectsize_0.4.5          colorspace_2.0-2         
[143] rvest_1.0.0               XML_3.99-0.6             
[145] fs_1.5.0                  rematch2_2.1.2           
[147] gmm_1.6-6                 plotly_4.9.4.1           
[149] xtable_1.8-4              gmp_0.6-2                
[151] jsonlite_1.7.2            zeallot_0.1.0            
[153] NLP_0.2-1                 R6_2.5.0                 
[155] tm_0.7-8                  gsubfn_0.7               
[157] pillar_1.6.1              htmltools_0.5.1.1        
[159] mime_0.11                 glue_1.4.2               
[161] fastmap_1.1.0             BiocParallel_1.24.1      
[163] codetools_0.2-16          mvtnorm_1.1-2            
[165] utf8_1.2.1                lattice_0.20-41          
[167] bslib_0.2.5.1             sqldf_0.4-11             
[169] curl_4.3.2                gtools_3.9.2             
[171] zip_2.2.0                 GO.db_3.12.1             
[173] limma_3.46.0              rmarkdown_2.9            
[175] munsell_0.5.0             GetoptLong_1.0.5         
[177] GenomeInfoDbData_1.2.4    iterators_1.0.13         
[179] impute_1.64.0             haven_2.4.1              
[181] gtable_0.3.0              bayestestR_0.10.0        </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
