<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit at IEO" />


<title>LFQ_Analysis</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.18/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.9.4.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Template_LFQ</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LFQ_Analysis</h1>
<h4 class="author">Fabio Bedin | MS-Unit at IEO</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-29
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>SKAP2_HaloTag/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20211221code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20211221)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20211221code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20211221)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong2865ca5"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 2865ca5 </a>
</p>
</div>
<div id="strongRepositoryversionstrong2865ca5" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 2865ca5. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  Counts.pdf
    Untracked:  Intensity_Distributions.pdf
    Untracked:  code/custom_functions.R
    Untracked:  data/expdesign.tsv
    Untracked:  data/proteinGroups.txt
    Untracked:  output/enrichmentmap.txt
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_vs_Halo.png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_vs_Halo.png
    Untracked:  output/output.zip
    Untracked:  output/results_table_Halo._vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_C._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_C._vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_C_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_C..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_N_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_N_vs_Halo.xlsx

Unstaged changes:
    Modified:   analysis/_site.yml
    Deleted:    analysis/about.Rmd
    Deleted:    analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/NSCAP2_act_vs_CSCAP2_act.Rmd</code>) and HTML (<code>docs/NSCAP2_act_vs_CSCAP2_act.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
2865ca5
</td>
<td>
FabioBedin
</td>
<td>
2021-12-29
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>## Proteomics
library(&quot;MBQN&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;enrichR&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)

## Plots
library(&quot;DT&quot;)
library(&quot;patchwork&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)

## Networks
library(&quot;STRINGdb&quot;)
library(&quot;RCy3&quot;)
library(&quot;RColorBrewer&quot;)

## Other
library(&quot;here&quot;)
library(&quot;openxlsx&quot;)
library(&quot;tidyverse&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(paste0(&quot;data/&quot;, params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot; &amp; data$Peptides &gt;= params$n_peptides,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<pre class="r"><code>cond_1 &lt;- params$test_1

cond_2 &lt;- params$test_2

conditions&lt;-c(cond_1,cond_2)

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)

expdesign &lt;- read.table(here::here(paste0(&quot;data/&quot;, params$expdesign)), header = T, stringsAsFactors = F)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
if(max(expdesign$replicate)&lt;=3){
       threshold&lt;-0
     } else if(max(expdesign$replicate)&lt;6){
       threshold&lt;-1
     } else if (max(expdesign$replicate)&gt;=6){
       threshold&lt;-trunc(max(expdesign$replicate)*0.25) ## 0.25 serve per avere sempre il 75% di valid vales
     }

data_filt &lt;- filter_missval(data_se, thr = threshold)</code></pre>
<div id="skap2_n_act-vs-skap2_c_act" class="section level1">
<h1><strong>SKAP2_N_act</strong> vs <strong>SKAP2_C_act</strong></h1>
<hr />
</div>
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>First, we need to wrangling the original dataset. From the <strong>proteinGroups</strong> obtained by <a href="https://www.maxquant.org/">MaxQuant</a> software, I remove proteins that are marked form the software as potential contaminant, only identify by site and reverse.<br />
We che also filter the initial datasets by excluding all proteins identified by less than 2 peptides.<br />
Now, we generate some quality control plots to investigate the structure of the dataset and observe the effect of filters.</p>
<div id="normalization-strategy" class="section level4">
<h4><strong>Normalization strategy:</strong></h4>
<pre class="r"><code>if(params$Normalization == &quot;VSN&quot;){
  data_filt &lt;- normalize_vsn(data_filt)
  print(&quot;VSN normalization is used&quot;)
} else if(params$Normalization == &quot;MBQN&quot;){
  data_filt &lt;- normalize_MBQN(data_filt)
  print(&quot;MBQN normalization is used&quot;)
} else {
  print(&quot;Max-LFQ normalization is used&quot;)
}</code></pre>
<p>[1] “Max-LFQ normalization is used”</p>
</div>
<div id="quality-control-plots" class="section level2 tabset tabset-fade">
<h2><strong>1. Quality control plots</strong></h2>
<div id="proteins-per-samples" class="section level3 tabset">
<h3>Proteins per samples</h3>
<p>This plot show the number of proteins identify in each samples <em>after and before User define</em> filters:</p>
<div id="filtered" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_filt)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/proteins-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_se)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/proteins-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="distributions" class="section level3 tabset">
<h3>Distributions</h3>
<p>This plot is useful for checking the distributions of the samples <em>after and before User define</em> filters:</p>
<div id="filtered-1" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_normalization(data_filt)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/normalization-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-1" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_normalization(data_se)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/normalization-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="missing-data" class="section level3 tabset">
<h3>Missing data</h3>
<p>This plot we can explore the missing data pattern <em>after and before User define</em> filters. Notice that the percentages of missingness are provided in the data. These are accurate to 1 decimal place. Also the dataset is arranged by columns with most missingness.</p>
<div id="filtered-2" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/missing-data-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-2" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>assay(data_se) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/missing-data-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample-cvs" class="section level3 tabset">
<h3>Sample CVs</h3>
<p>The coefficient of variation (CV) is a statistical measure of the dispersion of data points in a data series around the mean. The coefficient of variation represents the ratio of the standard deviation to the mean, and it is a useful statistic for comparing the degree of variation from one data series to another, even if the means are drastically different from one another.</p>
<div id="filtered-3" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_cvs(data_filt)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/CVs-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-3" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_cvs(data_se)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/CVs-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="upset-plot" class="section level2 tabset tabset-fade">
<h2><strong>2. Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within the 2 conditions or in common between all condition.</p>
<div id="skap2_n_act" class="section level3">
<h3>SKAP2_N_act</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_1)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_1)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/upset-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="skap2_c_act" class="section level3">
<h3>SKAP2_C_act</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_2)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_2)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/upset-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="both-conditions" class="section level3">
<h3>Both conditions</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()
n_sets &lt;- length(define_set)
n_each &lt;- n_sets / 2

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;red3&quot;, &quot;royalblue1&quot;), each = n_each),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/upset-both-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="imputation" class="section level2">
<h2><strong>3. Imputation</strong></h2>
<p>An idealized version of a label-free discovery mass spectrometry proteomics experiment would provide absolute abundance measurements for a whole proteome, across varying conditions. Unfortunately, this ideal is not realized. Measurements are made on peptides requiring an inferential step to obtain protein level estimates. The inference is complicated by experimental factors that necessitate relative abundance estimation and result in widespread non-ignorable missing data. Relative abundance on the log scale takes the form of parameter contrasts. In a complete-case analysis, contrast estimates may be biased by missing data and a substantial amount of useful information will often go unused.</p>
<pre class="r"><code>data_imputed &lt;- mix_imputation_mean(data_filt, cond1 = cond_1, cond2 = cond_2)

plot_imputation(data_filt, data_imputed)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/imputation-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>data_filt &lt;- data_imputed</code></pre>
</div>
<div id="differential-enrichment-analysis" class="section level2">
<h2><strong>4. Differential enrichment analysis</strong></h2>
<p>Protein-wise linear models combined with empirical Bayes statistics are used for the differential enrichment analysis (or differential expression analysis) and False discovery rate (FDR) is ajusted with <em>Benjamini-Hochberg</em> procedure. Significant proteins are are define with <strong>FDR = 0.05</strong> and <strong>Fold change = 1</strong></p>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results&lt;-get_results(dep)</code></pre>
<p>There are <strong>546</strong> significant proteins in <strong>SKAP2_N_act</strong> vs <strong>SKAP2_C_act</strong> comparison.</p>
<p>The results from the previous analysis can be easily visualized by a number of functions. These visualizations assist in the determination of the optimal cutoffs to be used, highlight the most interesting samples and contrasts, and pinpoint differentially enriched/expressed proteins.</p>
<div id="visualization-of-the-results" class="section level3 tabset tabset-fade">
<h3><strong>4.1. Visualization of the results</strong></h3>
<div id="pca" class="section level4">
<h4>PCA</h4>
<p>The PCA plot can be used to get a high-level overview of the data. This can be very useful to observe batch effects, such as clear differences between replicates.</p>
<pre class="r"><code>if(params$Imputation){
  if(nrow(dep) &gt;= 500){
    n_value &lt;- 500
  }else {
    n_value &lt;- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
} else {
  var &lt;- apply(assay(dep), 1, sd)
  df &lt;- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value &lt;- df %&gt;% as.data.frame() %&gt;% drop_na() %&gt;% nrow()
  if(n_value &gt;= 500){
    n_value &lt;- 500
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
}</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/PCA-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano" class="section level4">
<h4>Volcano</h4>
<p>Volcano plots allows to inspect the enrichment of proteins between the two samples (x axis) and their corresponding adjusted p value (y axis).</p>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T, label_size=5, adjusted = F)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/volcano-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-and-unique" class="section level4">
<h4>Volcano and unique</h4>
<pre class="r"><code>plot_volcano_2(dep, contrast=test, add_names=T, label_size=5, adjusted = F, unique_vec = &quot;SKAP2&quot;)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/volcano-unique-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="correlation" class="section level4">
<h4>Correlation</h4>
<pre class="r"><code>plot_cor(dep, significant = TRUE, lower = 0, upper = 1, pal = &quot;Reds&quot;)</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/correlation-matrix-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-heatmap" class="section level4">
<h4>Cluster heatmap</h4>
<p>The heatmap representation gives an overview of all significant proteins (rows) in all samples (columns). This allows to see general trends, for example if one sample or replicate is really different compared to the others. Additionally, the clustering of samples (columns) can indicate closer related samples and clustering of proteins (rows) indicates similarly behaving proteins.</p>
<pre class="r"><code>if(results %&gt;% filter(significant) %&gt;% nrow() &lt;= 100){
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = T, indicate = &quot;condition&quot;, col_limit = 5)
}else {
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = F, indicate = &quot;condition&quot;, col_limit = 5)
}</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/cluster-heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="result-table" class="section level3">
<h3><strong>4.2. Result table</strong></h3>
<p>In this table are summarized the results of DEP analysis, sorted by p.value. Proteins that are UP regulated are colored with a red box in Fold change column, in the orhter hand proteins that are DOWN regulated are colored with blue.</p>
<pre class="r"><code>results %&gt;% 
  select(starts_with(&quot;name&quot;) | starts_with(test)) %&gt;% 
  filter(across(ends_with(&quot;significant&quot;))) %&gt;% 
  arrange(across(ends_with(&quot;p.adj&quot;))) %&gt;%
  mutate(across(2:3, format, scientific = T, digits = 2)) %&gt;%
  dplyr::rename_with(~ tolower(gsub(paste0(test,&quot;_&quot;), &quot;&quot;, .x)), .cols = starts_with(test)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(Fold.Change = ratio, Gene.name = name) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:5)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;Fold.Change&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-d1ba32e0541469b5ed49" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-d1ba32e0541469b5ed49">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546"],["PRAM1","SASS6","MRPS18B","ARL8B","ARPC5L","CLDND1","CLTB","KDM1A","PRKCQ","SELPLG","TYMS","HRAS","MTA1","CSNK1G2","PTP4A3","RFK","RGS19","TRPV2","VDAC3","WDR82","LAMTOR2","A8K968","MRPS17","LSS","MRPS14","TFG","STAG2","G3BP2","IRF3","SYF2","RPS6KA1","STX4","SUPT6H","ERLIN2","GNG4","PSPH","ITGB1","TNPO3","TRA2A","CDK11B","PGRMC1","ATP5J2-PTCD1","FMNL3","ECD","hCG_2004507","CTDSP1","CAB39","EIF2S1","ERMP1","IGF2BP1","VDAC2","MBP","NUCB2","TOMM40","BRD3","C17orf85","TRAT1","RP2","ZDHHC5","C1orf21","ERICH5","SLC27A2","NDUFB10","RRAS2","HLA-C","HTATSF1","ERLIN1","MYO1C","HNRNPH3","YES1","hCG_1994888","AQR","FLOT2","LRRC7","STXBP3","MRPS36","PI4KA","SNAP29","LMTK2","GNAI3","NDUFA9","PPP1R16A","GLIPR2","EPB41L2","STXBP2","FLOT1","STOM","SLC4A7","PPP4C","CD58","GNB2","GRAMD1C","CD47","PLXNA1","SPICE1","CD48","GNAI2","GNAO1","SNAP23","GNAQ","SUCLG2","C12orf75","RALA","EMC2","TMED5","MLEC","TOMM22","LCK","LGALS8","MYO1G","SLC25A20","TAF15","ZNF112","HLA-C.1","CALM2","GNAS","GNA13","GNG2","GNG5","BST2","ULBP2","NAT10","DPP4","INPP5A","FYN","LTA4H","PRNP","DAD1","YBX3","IMMT.1","PAG1","PTPRC","SAMM50","RPS3A","GTPBP4","RFTN1","IMMT","LGALS9","RPL19","ITGAL","LAMTOR1","LAMTOR3","PLAU","RAB35","RNPEP","CD44","SEC23A","B2M","EBNA1BP2","TMED9","STAU1","DBN1","RPL22","RAC2","CD6","MTCH2","FYB","LAT","HNRNPC","COX4I1","SLC16A1","THRAP3","UNC45A","ITGB2","FOXRED1","FMNL1.1","PPP1CC","UBTF","HLA-B","PLEKHO1","SLC1A4","SSR1","STX11","CHCHD3","BCAP31","VDAC1","RPS20","RAP1B.1","RAP2C","CLTA","RAB10","CDC42","DUS2","LACTB","RPS4X","DLST","HNRNPUL1","CYBRD1","MARCKS","MARCKSL1","RBMX","PSMA4","THY1","SDHB","RNPS1","BYSL","EFR3A","CLTC","SLC29A1","SRRM2","SVIP","HLA-G","FAM98B","PRPF8","SRSF6","ILF2","JUNB","RPS13","EIF3A","HNRNPU","KARS","CCM2","GNB2L1","RER1","LRCH4","MAP2K3","ACTG1","NAPA","CD82","AGK","ITM2A","SKAP2","IFITM2","UQCRC2","NIP7","P4HA1","VCP","MATR3","TOP1","RAB27A","RPL10","ERBB2IP","PSMD2","XRCC6","SARS","HNRNPAB","LMNB1","MAT2B","APMAP","GNB1","PSMD12","RCL1","RAP2B","BZW1","NOP56","TPD52L2","ECH1","PHB","OGDH","RPN2","FKBP11","EPB41L3","STOML2","ERH","ADH5","LCP2","TMEM43","SRSF1","AIFM1","LMAN1","ILF3","SLC1A5","TDP2","RSL1D1","STX3","H2AFV","STIM1","THNSL1","EIF3E","HSDL2","RALY","SEMA7A","CALCOCO2","RPA2","SEPT7","BCLAF1","HMGCL","KBTBD4","UBXN1","DAAM1","EIF3L","SF3A2","CLINT1","RPL9","HLA-A","EIF3F","HNRNPA2B1","SF3B2","SNRPA","TPR","CHMP1A","MAGOHB","XRCC5","UBA6","MRPS35","HNRNPR","MYH10","RPS7","SCRIB","ZC3HAV1L","ITPR1","STT3B","TMED10","YRDC","MOCS2","SRSF9","PNN","ZGPAT","SARNP","BSG","SPTBN1","PRDX3","PFN2","PHB2","PRPF3","AIMP2","AKR1B1","FH","FKBP4","NOLC1","RHOG","ACP6","LAP3","ILKAP","H2AFY","RGS1","SUPT16H","IGF2BP3","PSMD11","C12orf29","EIF3J","VPS26A","DHX9","HSPA4","PPP1R12A","UGP2","NUP210","SSBP1","LRRC8C","TMED4","SNRPB2","ACIN1","BAZ1B","CEACAM1","CPPED1","HNRNPDL","PSMB1","SNRNP70","TMSB4X","EIF3CL","NAP1L1","WTAP","ATP5F1","UPF1","RBPJ","RMND5A","ISOC2","RAP1A","SEPT6","SCD","SEPT2","EIF3H","CSNK1G3","CYP51A1","DEF6","G6PD","PSMD9","TMED2","SNRNP200","SNRPD3","ALYREF","ACSF3","PCMT1","RPS25","ATP1A1","CUL1","ACTN4","METAP1D","PRPF19","PSMA5","EZR","MCM2","SPRYD4","PSMB4","SON","COPB2","PSMD13","PYCR1","C21orf33","COX5B","HSP90AB1","PSMD14","HNRNPLL","PTPRE","MCM4","COPA","BANF1","JAK1","SRSF2","NONO","PABPN1","RTFDC1","ACYP2","BCL10","CD5","NOP2","GPX4","HNRNPA1","HNRNPUL2","NUDT16L1","EIF4B","QARS","NARS","SRRM1","CHAC2","PSMD8","RTCA","SAP18","HNRNPD","COPB1","EIF4A3","RAB1B","VAMP8","HPRT1","SNRPD2","PNP","EIF3B","UNC13D","LANCL2","ANP32E","CPOX","HM13","ICAM3","TCAF2","C19orf43","EHD1","RAB11B","ATAD3A","TFRC","RABGGTB","SLC3A2","CAPZB","EIF2S2","SFPQ","CDV3","CAP1","PEF1","RPA1","LAMP1","HSP90AA1","UBE2S","RPS19","SMAP1","LANCL1","NAP1L4","PSMB2","SLC7A5","PCNA","SERPINB6","SFXN1","RAD23B","KRR1","EPDR1","PAFAH1B1","RNASE3","GK","PITRM1","PSMA1","TXN","MED16","GLS.1","PA2G4","GRB2","TBCC","PAK2","SRSF7","FARSB","DDX39B","MIF","SSRP1","CDC5L","BOLA2","NUP155","IVNS1ABP","RPN1","UBE2L6","PRDX1","BAG4","HNRNPA0","CANX","DUS3L","IBA57","LAMP2","PSMA3","RPL34","PPIB","COPE","PSMD3","ATP5H","DNPEP","IAH1","FERMT3","GOLGA7","NRAS","LYPLA2","TALDO1","CBR3","GSR","HSP90B1","LYSMD2","TRMT11","YWHAB","FUS","PSMA6","GANAB","CPSF6","MUC1","ADI1","GBP1","G3BP1","PSMC6","PALLD","RTN4","SRBD1","DLAT","SAE1","EIF4H","SLFN11","SF3A3","FTO","SSR4","TRIM28","CRKL","PDHB","PITHD1","RBM14","ATP5B","PTPRG","ETFA","WDR45B","DDX18","MAPKAPK3","RAB7A","ST13","STT3A","GRK6"],["1.6e-12","8.1e-12","1.5e-11","5.5e-11","4.5e-11","5.2e-11","4.2e-11","5.0e-11","2.7e-11","3.7e-11","3.5e-11","6.8e-11","6.6e-11","1.1e-10","1.2e-10","1.1e-10","9.9e-11","1.1e-10","9.8e-11","1.2e-10","1.5e-10","1.7e-10","1.9e-10","2.6e-10","3.9e-10","4.5e-10","5.1e-10","5.4e-10","6.4e-10","6.3e-10","6.8e-10","7.1e-10","7.3e-10","7.6e-10","1.0e-09","1.0e-09","1.1e-09","1.3e-09","1.3e-09","1.7e-09","2.2e-09","2.6e-09","2.7e-09","3.3e-09","3.4e-09","3.7e-09","5.5e-09","5.6e-09","5.5e-09","5.6e-09","6.0e-09","6.9e-09","7.4e-09","1.2e-08","1.4e-08","1.4e-08","1.4e-08","1.7e-08","1.8e-08","2.1e-08","2.3e-08","2.5e-08","2.6e-08","2.7e-08","2.9e-08","4.2e-08","4.4e-08","6.3e-08","8.0e-08","8.1e-08","9.0e-08","9.4e-08","1.0e-07","1.0e-07","1.1e-07","1.3e-07","1.3e-07","1.3e-07","1.5e-07","1.6e-07","1.6e-07","1.7e-07","1.9e-07","2.0e-07","2.6e-07","3.4e-07","3.6e-07","3.9e-07","4.1e-07","4.5e-07","5.3e-07","5.5e-07","6.4e-07","6.5e-07","6.9e-07","7.0e-07","7.7e-07","9.7e-07","9.8e-07","9.9e-07","1.3e-06","1.4e-06","1.6e-06","1.6e-06","2.0e-06","2.2e-06","2.3e-06","2.8e-06","2.9e-06","2.9e-06","3.0e-06","3.1e-06","3.2e-06","3.5e-06","3.8e-06","4.4e-06","4.5e-06","5.7e-06","6.0e-06","6.1e-06","6.1e-06","6.5e-06","7.8e-06","8.3e-06","9.1e-06","9.5e-06","1.1e-05","1.3e-05","1.3e-05","1.3e-05","1.5e-05","1.6e-05","1.7e-05","1.9e-05","1.9e-05","2.1e-05","2.6e-05","2.9e-05","2.9e-05","3.3e-05","3.6e-05","3.6e-05","3.9e-05","4.0e-05","4.0e-05","4.2e-05","4.2e-05","4.4e-05","4.9e-05","5.0e-05","5.3e-05","5.4e-05","6.0e-05","6.2e-05","6.3e-05","6.8e-05","8.4e-05","8.4e-05","8.9e-05","9.3e-05","9.6e-05","9.7e-05","1.0e-04","1.0e-04","1.1e-04","1.1e-04","1.1e-04","1.1e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.4e-04","1.4e-04","1.5e-04","1.5e-04","1.5e-04","1.7e-04","1.8e-04","1.9e-04","2.0e-04","2.1e-04","2.1e-04","2.1e-04","2.1e-04","2.2e-04","2.3e-04","2.3e-04","2.3e-04","2.3e-04","2.4e-04","2.5e-04","2.5e-04","2.7e-04","2.7e-04","2.8e-04","2.9e-04","3.0e-04","3.3e-04","3.4e-04","3.7e-04","4.0e-04","4.1e-04","4.3e-04","4.5e-04","4.5e-04","4.6e-04","5.1e-04","5.2e-04","5.4e-04","5.7e-04","5.7e-04","5.8e-04","5.9e-04","6.0e-04","6.1e-04","6.3e-04","6.6e-04","6.7e-04","6.6e-04","6.8e-04","6.8e-04","7.0e-04","7.0e-04","7.1e-04","7.3e-04","7.5e-04","7.6e-04","7.9e-04","8.0e-04","8.0e-04","8.5e-04","8.7e-04","8.9e-04","9.2e-04","9.2e-04","9.2e-04","9.6e-04","9.8e-04","9.8e-04","9.8e-04","1.0e-03","1.0e-03","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.6e-03","1.7e-03","1.7e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.9e-03","1.9e-03","2.0e-03","2.0e-03","2.0e-03","2.0e-03","2.1e-03","2.1e-03","2.1e-03","2.1e-03","2.1e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.2e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.4e-03","2.4e-03","2.4e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.6e-03","2.6e-03","2.7e-03","2.7e-03","2.7e-03","2.7e-03","2.8e-03","2.8e-03","2.8e-03","2.8e-03","2.9e-03","3.0e-03","3.0e-03","3.0e-03","3.1e-03","3.2e-03","3.1e-03","3.3e-03","3.3e-03","3.4e-03","3.5e-03","3.5e-03","3.5e-03","3.5e-03","3.5e-03","3.4e-03","3.5e-03","3.5e-03","3.6e-03","3.5e-03","3.6e-03","3.7e-03","3.8e-03","3.8e-03","3.9e-03","4.0e-03","4.0e-03","4.0e-03","4.1e-03","4.0e-03","4.1e-03","4.3e-03","4.4e-03","4.2e-03","4.3e-03","4.3e-03","4.3e-03","4.4e-03","4.4e-03","4.5e-03","4.6e-03","4.6e-03","4.5e-03","4.6e-03","4.6e-03","4.7e-03","4.7e-03","4.7e-03","4.6e-03","4.8e-03","4.8e-03","4.9e-03","4.9e-03","5.1e-03","5.1e-03","5.1e-03","5.1e-03","5.2e-03","5.2e-03","5.2e-03","5.2e-03","5.2e-03","5.3e-03","5.5e-03","5.5e-03","5.6e-03","5.6e-03","5.8e-03","5.8e-03","5.9e-03","5.9e-03","6.0e-03","6.0e-03","5.9e-03","5.9e-03","6.1e-03","6.1e-03","6.0e-03","6.1e-03","6.2e-03","6.2e-03","6.3e-03","6.3e-03","6.5e-03","6.5e-03","6.5e-03","6.5e-03","6.6e-03","6.6e-03","6.6e-03","6.7e-03","6.8e-03","6.9e-03","6.9e-03","7.2e-03","7.3e-03","7.3e-03","7.4e-03","7.6e-03","7.5e-03","7.6e-03","7.5e-03","7.5e-03","7.9e-03","7.9e-03","7.9e-03","8.0e-03","8.0e-03","8.1e-03","8.1e-03","8.1e-03","8.2e-03","8.2e-03","8.5e-03","8.5e-03","8.5e-03","8.6e-03","8.7e-03","8.7e-03","8.8e-03","8.9e-03","9.1e-03","9.2e-03","9.2e-03","9.2e-03","9.3e-03","9.3e-03","9.4e-03","9.4e-03","9.5e-03","9.5e-03","9.7e-03","9.8e-03","9.9e-03","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.8e-02"],["3.8e-09","9.5e-09","1.2e-08","1.2e-08","1.2e-08","1.2e-08","1.2e-08","1.2e-08","1.2e-08","1.2e-08","1.2e-08","1.2e-08","1.2e-08","1.4e-08","1.4e-08","1.4e-08","1.4e-08","1.4e-08","1.4e-08","1.4e-08","1.7e-08","1.8e-08","1.9e-08","2.6e-08","3.6e-08","4.1e-08","4.4e-08","4.5e-08","5.0e-08","5.0e-08","5.2e-08","5.2e-08","5.2e-08","5.2e-08","6.6e-08","6.6e-08","7.0e-08","7.7e-08","7.7e-08","1.0e-07","1.3e-07","1.4e-07","1.5e-07","1.8e-07","1.8e-07","1.9e-07","2.6e-07","2.6e-07","2.6e-07","2.6e-07","2.7e-07","3.1e-07","3.2e-07","5.0e-07","5.8e-07","5.8e-07","6.0e-07","6.9e-07","7.1e-07","8.3e-07","8.8e-07","9.3e-07","9.8e-07","9.9e-07","1.1e-06","1.5e-06","1.5e-06","2.2e-06","2.7e-06","2.7e-06","3.0e-06","3.1e-06","3.2e-06","3.3e-06","3.6e-06","3.9e-06","3.9e-06","4.0e-06","4.4e-06","4.6e-06","4.8e-06","4.9e-06","5.3e-06","5.7e-06","7.0e-06","9.2e-06","9.8e-06","1.0e-05","1.1e-05","1.2e-05","1.4e-05","1.4e-05","1.6e-05","1.6e-05","1.7e-05","1.7e-05","1.9e-05","2.3e-05","2.3e-05","2.3e-05","2.9e-05","3.2e-05","3.6e-05","3.6e-05","4.5e-05","4.8e-05","4.9e-05","6.0e-05","6.2e-05","6.2e-05","6.2e-05","6.5e-05","6.6e-05","7.1e-05","7.8e-05","8.9e-05","9.1e-05","1.1e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.5e-04","1.6e-04","1.7e-04","1.8e-04","2.0e-04","2.3e-04","2.4e-04","2.4e-04","2.6e-04","2.8e-04","3.1e-04","3.3e-04","3.4e-04","3.7e-04","4.4e-04","4.8e-04","4.8e-04","5.4e-04","5.9e-04","5.9e-04","6.4e-04","6.5e-04","6.5e-04","6.7e-04","6.7e-04","6.9e-04","7.7e-04","7.8e-04","8.2e-04","8.3e-04","9.2e-04","9.5e-04","9.5e-04","1.0e-03","1.2e-03","1.2e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.6e-03","1.7e-03","1.7e-03","1.8e-03","1.9e-03","1.9e-03","2.0e-03","2.0e-03","2.2e-03","2.2e-03","2.5e-03","2.6e-03","2.6e-03","2.6e-03","2.6e-03","2.6e-03","2.7e-03","2.7e-03","2.8e-03","2.8e-03","2.8e-03","2.8e-03","2.9e-03","2.9e-03","3.1e-03","3.1e-03","3.2e-03","3.3e-03","3.4e-03","3.7e-03","3.7e-03","4.1e-03","4.4e-03","4.4e-03","4.7e-03","4.8e-03","4.8e-03","4.8e-03","5.4e-03","5.4e-03","5.6e-03","5.9e-03","5.9e-03","5.9e-03","6.0e-03","6.1e-03","6.2e-03","6.3e-03","6.6e-03","6.6e-03","6.6e-03","6.6e-03","6.6e-03","6.8e-03","6.8e-03","6.8e-03","7.0e-03","7.2e-03","7.2e-03","7.4e-03","7.4e-03","7.4e-03","7.8e-03","8.0e-03","8.1e-03","8.3e-03","8.3e-03","8.3e-03","8.6e-03","8.7e-03","8.7e-03","8.7e-03","8.7e-03","8.7e-03","8.8e-03","8.8e-03","9.2e-03","9.2e-03","9.3e-03","9.3e-03","9.3e-03","9.5e-03","9.5e-03","9.7e-03","9.7e-03","9.9e-03","9.9e-03","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.1e-02","2.1e-02","2.1e-02","2.1e-02","2.1e-02","2.1e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.6e-02","2.6e-02","2.6e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.9e-02","2.9e-02","2.9e-02","2.9e-02","2.9e-02","3.0e-02","3.0e-02","3.0e-02","3.1e-02","3.1e-02","3.1e-02","3.1e-02","3.1e-02","3.1e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.2e-02","3.3e-02","3.3e-02","3.3e-02","3.3e-02","3.3e-02","3.3e-02","3.3e-02","3.4e-02","3.4e-02","3.4e-02","3.4e-02","3.4e-02","3.4e-02","3.4e-02","3.5e-02","3.5e-02","3.5e-02","3.5e-02","3.5e-02","3.5e-02","3.6e-02","3.6e-02","3.6e-02","3.6e-02","3.6e-02","3.6e-02","3.7e-02","3.7e-02","3.7e-02","3.7e-02","3.7e-02","3.7e-02","3.8e-02","3.8e-02","3.8e-02","3.8e-02","3.9e-02","3.9e-02","3.9e-02","3.9e-02","3.9e-02","3.9e-02","4.0e-02","4.0e-02","4.0e-02","4.0e-02","4.0e-02","4.0e-02","4.0e-02","4.0e-02","4.0e-02","4.1e-02","4.1e-02","4.1e-02","4.1e-02","4.1e-02","4.1e-02","4.2e-02","4.2e-02","4.2e-02","4.2e-02","4.3e-02","4.4e-02","4.4e-02","4.4e-02","4.4e-02","4.4e-02","4.4e-02","4.4e-02","4.4e-02","4.4e-02","4.4e-02","4.5e-02","4.5e-02","4.5e-02","4.5e-02","4.5e-02","4.5e-02","4.5e-02","4.6e-02","4.6e-02","4.6e-02","4.6e-02","4.7e-02","4.7e-02","4.7e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.9e-02","4.9e-02","4.9e-02","4.9e-02","5.0e-02","5.0e-02"],["True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True"],[-7.89,-4.13,3.64,5.33,4.52,4.59,3.6,3.95,3.53,5.07,5.38,4.99,3.33,6.63,4.44,-4.32,4.72,3.93,7.42,3.32,3.95,4.68,4.46,2.93,4.49,5.02,3.27,4.24,5.56,-2.78,3.37,4.65,3.1,4.67,7.72,4.82,4.66,4.53,4.26,3.32,4.66,4.73,4.11,3.21,4.67,5.54,4.05,6.12,2.53,4.56,7.3,2.58,3.69,6.28,3.98,3.56,4.64,4.8,4.05,4.06,2.56,4.2,4.49,7.98,6.09,3.29,3.28,3.49,4.65,5.18,5.1,2.47,5.56,4.96,4.21,4.24,4.05,3.28,4.67,5.6,4.13,3.99,5.95,8.01,7.56,5.51,6.35,5.05,3.4,5.91,4.21,3.75,5.79,3.91,3.63,5.51,4.37,5.65,5.27,4.75,3.77,4.06,4.96,3.51,5.55,5.18,5.41,6.45,4.44,4.27,-1.39,2.4,-1.45,7.81,3.01,2.54,5.5,4.71,5.7,5.17,5.56,1.54,4.56,4.51,5.13,3.77,3.62,4.87,1.27,5.84,5.43,4.27,4.15,1.39,1.96,5.7,2.83,1.95,1.18,3.46,2.6,2.41,2.25,3.71,-1.72,3.95,-1.19,2.8,1.65,2.72,1.65,3,1.5,1.34,3.08,2.46,-3.45,3.34,1.46,2.9,4.44,1.37,1.21,2.92,-1.75,4.85,1.52,1.17,2.86,1.11,4.27,2.18,4.32,2.95,2.23,5.46,-1.6,3.76,3.12,1.6,2.07,3.11,-1.04,2.65,1.24,2.08,2.43,2.81,1.25,2.9,1.37,1.56,3.02,-1.65,1.74,2.06,4.88,1.72,5.15,1.3,2.43,1.9,-1.02,1.06,1.83,1.09,-1.85,1.12,3.02,1.26,1.14,1.84,-1.37,2.26,1.46,-1.25,1.11,2.82,2.21,2.46,1.08,-2.81,3.07,2.5,1.26,1.34,2.35,1.73,2.59,1.3,1.1,3.63,1.16,1.56,4.21,3.49,1.13,-1,3.94,3.81,1.78,1.7,2.7,1.35,1.05,2.48,2.14,3.9,2.5,4.98,2.23,4.96,3.11,1.07,1.6,2.94,2.2,1.76,1.09,3.77,1.31,3.86,-1.31,1.33,3.64,1.77,1.41,-1.45,1.68,-1.26,2.6,4.16,2.43,1.42,1.05,2.06,-1.6,-1.01,-1.12,3.87,3.48,1.29,1.3,1.47,1.36,2.27,2.07,1.64,1.97,2.1,-1.36,1.04,1.24,-1.2,-1.17,1.15,1.88,3.36,2.01,-1.86,2.28,1.5,3.13,-1.46,-1.08,1.81,1.69,-1.02,1.68,3.38,1.62,-1.06,-1.63,3.42,1.32,-1.19,1.56,3.77,2.61,2.28,2.64,-1.77,1.12,-1.23,1.32,-1.52,2.41,1.27,1.77,-1.42,1.14,1.14,1.62,2.15,1.37,3.02,2.1,2.83,-1.16,1.54,1.43,1.62,1.53,1.37,-1.67,1.84,1.67,1.32,-1.6,1.51,1.2,1.07,1.94,1.14,-1.42,-1.07,-1.81,2.01,1.34,1.4,1.31,2.55,2.99,1.77,-1.25,-1,-1.11,2.39,1.14,1.13,1.77,-1.27,-1.32,1.02,2.44,1.42,1.37,-1.42,1.44,1.69,2.65,1.27,-1.29,2.89,1.93,2.61,2.14,1.84,-1,2.39,1.41,1.01,-1.08,-1.04,2.18,3.09,3.15,1.03,1.98,1.13,1.09,-1.33,-1.68,-1.12,3.64,1.36,-1.63,1.63,1.98,-1.44,1.52,-2.76,1.83,1.08,-1.1,1.41,-1.01,1.04,1.9,1.49,1.58,1.41,1.32,-1.06,1.13,3.82,2.48,1.48,-1.31,1.85,-1.03,2.36,2.14,-1.24,-1.31,3.68,2.1,1.37,1.64,-1.21,1.13,-1.01,1.68,1.19,1.42,1.87,-1.55,1.56,1.91,1.92,-1.03,1.34,-1.27,-1.09,2.13,1.27,1.02,1.19,-1.01,2.93,1.36,1.3,-1.44,1.73,1.69,2.31,-1.05,1.61,-1.9,-1.05,1.77,1.51,-1.01,-1.21,2.01,1.49,-1.13,1.16,-1.3,2.54,1.72,-1.39,-1.15,-1.54,1.35,-1.63,-1.29,-1.07,1.21,1.04,-1.17,-1.34,1.27,1.62,1.16,1.74,1.28,1.16,1.05,-1.32,-1.16,2.2,1.04,1.96,-1.18,2.89,-1.15,-1.19,1.59,-1.46,-1.13,1.36,1.97,1.59,2.88,1.42,2.16,-1.25,1.25,2.88,1.45,-1.09,1.83,-1.42,1.57,1.38,1.44,-1,1.56,-1.09,1.4,1.29,-1.42,1.05,-1.12,1.16,1.47,1.49,2.39,-1.1,1.21,-1,1.4,1.8,1.65,1.44]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene.name<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>Fold.Change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[5]; $(this.api().cell(row, 5).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[4]; $(this.api().cell(row, 4).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
<div id="gene-ontology" class="section level2">
<h2><strong>5. Gene Ontology</strong></h2>
<p>Now I perfom a gene onology analysis (GO or by branch <a href="GO:MF" class="uri">GO:MF</a>, <a href="GO:BP" class="uri">GO:BP</a>, <a href="GO:CC" class="uri">GO:CC</a>) and a REAC ontology.<br />
The enrichment results are visualized with an interactive <strong>Manhattan-like-plot</strong> named “Gostplot”.</p>
<div id="skap2_n_act-1" class="section level3 tabset tabset-fade">
<h3><strong>5.1. SKAP2_N_act</strong></h3>
<div id="gostplot" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in SKAP2_N_act were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &gt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_1)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-06484ed42468d86de4a4" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-06484ed42468d86de4a4">{"x":{"data":[{"x":[132.688590508187,93.7513937665195,132.658540578231,93.6424627704257,118.467461156079,75.9130040958479,118.516292292259,76.5140026949861,72.9718672013152,73.0169420962506,75.9430540258048,72.964354718826,113.629422433016,76.5215151774754,82.8244879859375,115.87941093854,131.501618274889,121.750416003872,85.9872431139025,81.4008725542289,126.798804236633,74.4931449053838,130.295864835368,171.370362845222,146.534095735834,131.336343660126,81.4046287954735,83.0648874255928,75.9092478546033,107.059756496187,76.2548220491078,76.6454711385476,75.8679292009126,76.6266899323246,101.03850178107,75.8266105472218,81.4234100016966,116.964964658234,130.08551532567,121.818028346275,77.6483875508596,76.4576590763169,122.189896229492,75.7101670686388,77.6897062045503,121.821784587519,75.7251920336172,81.4196537604519,77.6746812395719,76.5102464537415,75.8303667884664,181.790176057781,181.827738470227,77.3967193874704,114.493357919278,75.7139233098834,77.6521437921042,98.7434383806114,109.982112184496,144.051220273145,83.2827494177805,114.03134024619,146.522827012101,130.078002843181,119.969957653925,77.9301056442056,106.793063367819,85.690500055578,94.0293556186209,121.81427210503,146.81581382918,75.620017278768,119.537989910794,74.3804576680454,142.477355191651,101.057282987294,145.369660950004,159.185116247694,126.952810127662,76.641714897303,109.997137149475,146.766982693,110.087286939345,108.806408674932,98.7509508631006,130.292108594124,77.9225931617164,126.795047995388,111.845207841825,126.284199186121,155.207256769648,75.6125047962788,120.262944471005,74.3917263917793,81.8027903674026,83.0611311843482,97.5414411823349,76.2848719790647,136.005351527182,107.46167430936,83.9438478768325,74.3842139092901,152.206020015201,96.7826804509229,129.833847162281,129.837603403525,132.75620285059,167.287328612326,131.509130757379,164.928409110709,120.555931288084,83.267724452802,114.444526783098,77.3892069049812,103.585233344919,92.5306153620199,88.6128557438876,140.738215495395,118.974553724102,132.868890087929,118.343505195007,121.979546719793,103.247171632903,130.86681350455,114.500870401767,93.6650002178934,139.295818857463,149.114633470884,91.5539926384203,118.970797482857,87.3808086156542,119.293834229894,169.514779670382,101.015964333603,81.4271662429412,120.251675747271,103.243415391659,136.437319270312,143.566665152589,93.4133320545042,165.514382744869,145.245704988932,119.286321747405,144.381769502671,86.1825676586224,81.4609724141427,184.76136288227,109.501313305186,83.2714806940466,129.901459504684,182.180825147221,87.373296133165,171.374119086466,80.9764172935875,104.569368551008,81.4346787254304,152.517788038504,145.377173432493,72.6563429367676,108.24297248824,143.664327424949,118.959528759124,82.1145583907055,120.007520066371,152.115870225331,80.9801735348321,116.17239775562,87.0389906623944,90.1491584129347,96.6474557661168,103.57772086243,111.807645429379,89.6383096036672,145.373417191249,107.799736021376,101.493006971669,127.01666622882,90.3670204051223,72.6676116605015,169.511023429138,128.507894002932,151.011535299414,90.29189558023,81.438434966675,145.673916490818,128.297544493234,89.961346350704,81.4572161728981,116.018391864591,83.9250666706094,88.2635253081385,113.813478254002,142.481111432896,101.48549448918,75.8003168585095,107.409086931936,76.6492273797923,107.236299834683,72.983135925049,167.118297756319,86.1788114173778,131.29878124768,72.5549244231631,117.990418518013,120.21035709358,119.237490611225,142.49238015663,146.759470210511,145.760310039444,151.909276956877,136.865530772198,145.74904131571,97.8945278593286,121.476210393015,81.7952778849133,165.668388635898,145.670160249573,93.7025626303395,127.020422470065,103.513864761271,120.255431988515,144.851299658247,159.988951874041,145.410979603695,128.312569458212,83.151280974219,81.3032102818689,103.795582854617,136.835480842241,113.809722012758,173.150821195169,81.5886846164596,100.952108232444,129.469491761553,93.7551500077641,157.840381882122,75.1579996056805,89.9575901094594,167.107029032585,85.4200506859658,109.978355943252,138.619695433433,118.283405335093,85.4313194096996,74.2489892244839,130.186933839275,119.23373436998,131.295025006436,101.496763212913,83.9513603593217,144.610900218592,89.893734008301,101.459200800467,143.517834016409,96.6512120073614,103.558939656206,91.4675990897942,77.4530630061396,149.039508645992,115.759211218712,123.778786275963,96.5760871824691,83.102449838039,100.959620714934,167.114541515074,149.122145953373,177.560648416345,116.213716409311,82.1408520794178,181.662463855464,104.561856068518,129.908971987173,103.175803049256,90.3632641638777,170.517696082694,132.850108881706,155.248575423339,127.591371139246,170.510183600205,101.000939368624,129.007474088466,106.605251305588,130.442358243908,136.001595285937,76.6604961035261,108.855239811112,81.7990341261579,178.841526680759,175.028941817476,136.85050580722,139.314600063686,145.414735844939,130.337183489059,130.382258383994,87.4446647168127,84.5486027172153,74.8387190998883,183.112372975885,105.286810628729,101.470469524201,178.849039163248,123.8576673421,120.232894541048,76.6717648272599,76.6642523447707,167.10327279134,132.898940017886,120.510856393149,109.497557063941,165.630826223452,121.299667054518,77.5169191072981,127.557564968045,119.425302673456,94.4575671205069,115.860629732317,149.058289852215,126.133949536336,145.384685914982,132.654784336986,83.5644675111265,118.471217397324,82.1446083206624,82.148364561907,127.399802835771,173.158333677658,143.961070483274,132.7148841969,101.425394629266,73.6329656603673,121.292154572029,166.047769001604,72.8291300340199,121.701584867692,83.4442677912988,130.363477177771,170.547746012651,150.771135859759,98.747194621856,74.8312066173991,168.440494674423,158.760660987053,118.302186541316,87.3958335806327,143.878433175892,123.297987396653,116.243766339268,118.978309965347,84.1729785927539,72.8028363453076,98.7547071043452,166.051525242848,75.7402169985957,74.8462315823776,90.1454021716901,85.6980125380672,121.720366073915,72.7615176916168,96.7864366921675,116.251278821757,146.958550996476,81.5623909277473,88.4626060941031,168.425469709444,83.0799123905713,83.4029491376081,112.254638137488,175.036454299965,101.027233057337,105.294323111218,142.075437378478,74.8499878236222,77.5131628660534,77.4680879711181,145.60254790717,144.498212981254,170.656677008745,110.049724526899,128.504137761688,130.201958804253,109.813081328489,74.8236941349099,96.6436995248722,73.3512475670212,72.9868921662937,116.255035063002,100.933327026221,136.550006507651,84.1166349740847,131.535424446091,77.4455505236504,149.024483681013,109.283451312998,83.9476041180771,74.8349628586437,98.5894324895822,83.910041705631,74.6283695901899,91.0919749653328,84.289422071337,82.1934394568424,106.601495064344,165.634582464696,132.692346749432,103.172046808011,83.1062060792836,165.638338705941,120.184063404868,96.6587244898506,88.4663623353477,146.53033949459,103.555183414962,81.4722411378765,77.4342817999165,170.664189491234,83.0761561493267,80.3603937294708,77.4718442123627,83.1362560092405,104.081057189208,150.759867136025,94.6040605290468,83.0686436668375,89.3227853391196,128.301300734478,104.704593235814,132.906452500375,118.794254144361],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.9779088220951,15.8692266077119,15.8593268499579,15.7689210265341,15.6601980090826,15.5535055763631,15.309450464312,14.8469349226635,14.3792280571041,14.3180140983669,14.2772456399735,14.1603344310158,14.1335322162122,14.118554992104,14.0419474681678,13.8765266817088,13.8197694208589,13.7132121973995,13.5869503435845,13.5655880701379,13.56213293344,13.5137457423918,13.4968938348828,13.4464594437263,13.4451628288131,13.3718061624717,13.363465730158,13.360336711728,13.0734770722904,13.0475876315026,13.0402518551666,12.8887757934502,12.682338108102,12.6762774834377,12.5097549019435,11.9618256607976,11.8608704032503,11.8031204480107,11.7353134362692,11.7075473668975,11.6931250096315,11.5621970003493,11.5490845369791,11.3818643019603,11.3512705532747,11.339283015471,11.1400451865438,11.0765440278005,11.0765440278005,11.0403047694663,10.9964947283263,10.8901026073585,10.8901026073585,10.879851821228,10.7115250057703,10.7108116066835,10.7091877878845,10.7057993844085,10.6613236591684,10.4728314977037,10.4638126392229,10.4432766470424,10.4339277704907,10.4058029093958,10.3262438067252,10.2137108487192,10.1850235230644,10.1721175063904,10.1405522770162,10.1075415717249,10.0974214745546,9.97398503105206,9.91113450610788,9.89445313339131,9.8498681532054,9.79710620597604,9.73930331924168,9.69040492136543,9.67850295643416,9.64736891517511,9.64461433130458,9.63784368909932,9.54548689165387,9.53872364983897,9.37673374978168,9.35808206311289,9.32109075533608,9.21897029516797,9.18682483427283,8.94758099732831,8.90892503011593,8.801305490485,8.78762299780909,8.68812984708287,8.59288155571234,8.53262892676577,8.40335347521342,8.113674171651,8.10610850254948,7.99254383100557,7.95464113332587,7.77809992831773,7.73563292811605,7.63675328977333,7.44369197244288,7.35964491203661,7.16109259345375,7.12624265142318,7.09560631245385,7.09229324180317,7.03550397950333,6.9008639258967,6.88135517298889,6.84079222770289,6.81900683438248,6.73017665723094,6.55839260315545,6.52907239143226,6.52129325303237,6.42912674118101,6.38295052205043,6.32991203558954,6.21237006569344,6.17703382218309,6.16717155983066,6.14227201089173,6.10704951689785,6.08592870545293,6.07687949438585,6.02407660022719,6.00512706883761,6.00512706883761,5.92395913840057,5.85197450565603,5.84542730581794,5.82931522801821,5.80199202146571,5.76673973770357,5.73514775182775,5.73491354132688,5.67405262249645,5.65824407899791,5.58134260313387,5.56624506231402,5.53624916519974,5.46460472971323,5.44139429222876,5.42002542454218,5.39226940968287,5.38351025016338,5.37665497612645,5.35774536350181,5.30284672721438,5.30021446501144,5.23642414857442,5.22371025645213,5.21756792930997,5.16137668457915,5.14263901682978,5.11362725418405,5.07468460896372,5.03390370803001,5.0133611112065,4.98178075321003,4.95373519701465,4.94763027221601,4.94576387269737,4.93289737488153,4.93015924289385,4.91815075528277,4.91467404797467,4.85712515115538,4.85140216831682,4.83120544433395,4.77703562985027,4.68869869759675,4.62878370729686,4.62817773999421,4.596502399389,4.57635362329919,4.55167733796993,4.54689685151035,4.5087255013629,4.4811770063503,4.4548037677446,4.41122628793149,4.37777256903059,4.34266606299879,4.3302061193003,4.31658621392674,4.30207839209016,4.26653160575889,4.24599400024658,4.23975416163101,4.22826935763853,4.22373557335178,4.2097743497972,4.18242801279992,4.17544850841379,4.06794720469367,4.05877927289406,4.05523429676322,4.02318419053301,4.01586888311888,3.991564188265,3.98282991102798,3.97341213181348,3.95730198545049,3.94359325596293,3.9184524455302,3.91296606488577,3.89152946091796,3.88530672702521,3.83576467219256,3.72046875446792,3.69762783639889,3.68553811834656,3.68455710072284,3.62140812198295,3.58216746243503,3.55336390552244,3.52117114477055,3.51646207103895,3.40496503464113,3.38376467268306,3.3816920949181,3.37038590844868,3.37006225009885,3.36390268742533,3.30322899294164,3.29947140372374,3.27710854969408,3.27501365416103,3.267925220905,3.24000958938137,3.2170336117603,3.20064090792259,3.19317737088123,3.1617642392795,3.14864481802401,3.14391236345895,3.12640545673857,3.12182599628959,3.11321837760957,3.09648360413286,3.07739785002229,3.06620490110889,3.04107030381437,3.02728378916267,3.02492406287499,3.02488978904357,3.02488978904357,3.01301577341904,2.99432494200825,2.98968674070761,2.97398599874475,2.92324380730856,2.91539234141679,2.87953873531697,2.86980935294687,2.83910433737792,2.83727141541226,2.8356691964014,2.83415307054777,2.82685486275148,2.82558855715436,2.81676034595166,2.80538368902439,2.80028839498283,2.77378496145132,2.77177959978653,2.75912731698319,2.75532401557903,2.74728161382617,2.7372622699611,2.71446787080047,2.70254667279972,2.63246479798713,2.62558887421143,2.61349110777186,2.60818036762256,2.60818036762256,2.60553394749458,2.56993779959543,2.56123896721626,2.54243252596972,2.52371160987132,2.52053344850121,2.48697908820104,2.47714574878386,2.46856933366568,2.46231877928435,2.45729514042998,2.44921542517979,2.42856225331327,2.41755532977767,2.34435346474799,2.34028769301917,2.27174741441387,2.26401322771977,2.16523745654018,2.14833263229167,2.14217342958837,2.13982883182327,2.13817722158596,2.12948300929001,2.11460369671855,2.06811856781499,2.05845087404396,2.02616216419653,2.01131075384754,1.98295428675653,1.96961284225359,1.93382280650469,1.92770729990744,1.91304398108814,1.89906482635191,1.89564503024862,1.87214233550726,1.87094655479429,1.86978078253647,1.8602326675245,1.81961537403803,1.81092890393467,1.79695843404739,1.79620956094141,1.75843207207198,1.75747218423836,1.7468473071807,1.74133573585336,1.7347521770015,1.73124593248653,1.72097874950666,1.7066018873757,1.6968240055198,1.68940010475223,1.67672819258183,1.66527205426757,1.6443090015602,1.63561465982369,1.6131279689279,1.60717699298194,1.58901646167486,1.58827608083618,1.57803064519917,1.55862507619495,1.55610164522265,1.55443568606473,1.54315408143459,1.54164714282926,1.53825246814372,1.53775311070151,1.53536781056131,1.5292621376076,1.51951177268128,1.47715025069465,1.47685368640052,1.469292424247,1.46701307677895,1.4631848701339,1.4505024147571,1.44529489183568,1.43945305393842,1.42730668143411,1.42365519498051,1.41338793830572,1.40877466711024,1.398222684503,1.3682293215796,1.35500387114422,1.34427656428844,1.31977067511482,1.31938547423997,1.31641744372423,1.30552273983998],"text":["GO:0051649 (2778) <br> establishment of localization in cell <br> 4.697e-31","GO:0016071 (880) <br> mRNA metabolic process <br> 5.530e-31","GO:0051641 (3519) <br> cellular localization <br> 2.293e-29","GO:0016032 (1011) <br> viral process <br> 1.661e-28","GO:0043299 (538) <br> leukocyte degranulation <br> 3.901e-23","GO:0002275 (552) <br> myeloid cell activation involved in immune response <br> 1.604e-22","GO:0043312 (483) <br> neutrophil degranulation <br> 2.291e-22","GO:0002444 (558) <br> myeloid leukocyte mediated immunity <br> 2.899e-22","GO:0000377 (360) <br> RNA splicing, via transesterification reactions with bulged adenosine as nucleophile <br> 3.601e-22","GO:0000398 (360) <br> mRNA splicing, via spliceosome <br> 3.601e-22","GO:0002283 (488) <br> neutrophil activation involved in immune response <br> 3.917e-22","GO:0000375 (363) <br> RNA splicing, via transesterification reactions <br> 5.197e-22","GO:0036230 (509) <br> granulocyte activation <br> 5.390e-22","GO:0002446 (500) <br> neutrophil mediated immunity <br> 1.380e-21","GO:0006810 (5214) <br> transport <br> 1.388e-21","GO:0042119 (502) <br> neutrophil activation <br> 1.696e-21","GO:0051234 (5364) <br> establishment of localization <br> 3.663e-21","GO:0045055 (790) <br> regulated exocytosis <br> 4.406e-21","GO:0008380 (459) <br> RNA splicing <br> 1.829e-20","GO:0006396 (1010) <br> RNA processing <br> 1.885e-20","GO:0046907 (1723) <br> intracellular transport <br> 2.722e-20","GO:0001775 (1507) <br> cell activation <br> 9.403e-20","GO:0050852 (202) <br> T cell receptor signaling pathway <br> 1.502e-19","GO:1903311 (384) <br> regulation of mRNA metabolic process <br> 1.773e-19","GO:0071705 (2216) <br> nitrogen compound transport <br> 3.494e-19","GO:0051179 (6938) <br> localization <br> 4.008e-19","GO:0006397 (532) <br> mRNA processing <br> 5.213e-19","GO:0006887 (906) <br> exocytosis <br> 7.044e-19","GO:0002274 (671) <br> myeloid leukocyte activation <br> 1.043e-18","GO:0033036 (3148) <br> macromolecule localization <br> 1.790e-18","GO:0002366 (723) <br> leukocyte activation involved in immune response <br> 2.208e-18","GO:0002479 (75) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent <br> 2.915e-18","GO:0002263 (727) <br> cell activation involved in immune response <br> 2.971e-18","GO:0002474 (99) <br> antigen processing and presentation of peptide antigen via MHC class I <br> 3.572e-18","GO:0022613 (473) <br> ribonucleoprotein complex biogenesis <br> 6.732e-18","GO:0002252 (1135) <br> immune effector process <br> 9.795e-18","GO:0006402 (388) <br> mRNA catabolic process <br> 1.141e-17","GO:0042590 (80) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I <br> 1.312e-17","GO:0050778 (757) <br> positive regulation of immune response <br> 1.359e-17","GO:0045088 (316) <br> regulation of innate immune response <br> 1.431e-17","GO:0002757 (475) <br> immune response-activating signal transduction <br> 2.845e-17","GO:0002429 (475) <br> immune response-activating cell surface receptor signaling pathway <br> 2.845e-17","GO:0045321 (1341) <br> leukocyte activation <br> 4.222e-17","GO:0002218 (144) <br> activation of innate immune response <br> 4.422e-17","GO:0002768 (510) <br> immune response-regulating cell surface receptor signaling pathway <br> 6.401e-17","GO:0045089 (217) <br> positive regulation of innate immune response <br> 6.706e-17","GO:0002223 (112) <br> stimulatory C-type lectin receptor signaling pathway <br> 7.308e-17","GO:0006401 (431) <br> RNA catabolic process <br> 7.972e-17","GO:0002764 (513) <br> immune response-regulating signaling pathway <br> 8.049e-17","GO:0002443 (888) <br> leukocyte mediated immunity <br> 8.258e-17","GO:0002253 (566) <br> activation of immune response <br> 9.045e-17","GO:1990840 (113) <br> response to lectin <br> 9.058e-17","GO:1990858 (113) <br> cellular response to lectin <br> 9.058e-17","GO:0002684 (1079) <br> positive regulation of immune system process <br> 1.052e-16","GO:0038093 (238) <br> Fc receptor signaling pathway <br> 1.351e-16","GO:0002220 (115) <br> innate immune response activating cell surface receptor signaling pathway <br> 1.383e-16","GO:0002758 (116) <br> innate immune response-activating signal transduction <br> 1.702e-16","GO:0019882 (236) <br> antigen processing and presentation <br> 2.187e-16","GO:0034613 (1842) <br> cellular protein localization <br> 2.796e-16","GO:0070727 (1856) <br> cellular macromolecule localization <br> 4.904e-16","GO:0006955 (2544) <br> immune response <br> 1.423e-15","GO:0036388 (66) <br> pre-replicative complex assembly <br> 4.176e-15","GO:0071702 (2657) <br> organic substance transport <br> 4.808e-15","GO:0050776 (1159) <br> regulation of immune response <br> 5.281e-15","GO:0044085 (3377) <br> cellular component biogenesis <br> 6.913e-15","GO:0002833 (253) <br> positive regulation of response to biotic stimulus <br> 7.353e-15","GO:0032940 (1351) <br> secretion by cell <br> 7.611e-15","GO:0008104 (2670) <br> protein localization <br> 9.079e-15","GO:0016192 (2193) <br> vesicle-mediated transport <br> 1.329e-14","GO:0045087 (1003) <br> innate immune response <br> 1.514e-14","GO:0071840 (6880) <br> cellular component organization or biogenesis <br> 1.935e-14","GO:0002183 (35) <br> cytoplasmic translational initiation <br> 2.589e-14","GO:0043933 (2017) <br> protein-containing complex subunit organization <br> 2.719e-14","GO:0001732 (16) <br> formation of cytoplasmic translation initiation complex <br> 2.741e-14","GO:0065003 (1724) <br> protein-containing complex assembly <br> 3.064e-14","GO:0022618 (196) <br> ribonucleoprotein complex assembly <br> 3.185e-14","GO:0071345 (1148) <br> cellular response to cytokine stimulus <br> 3.577e-14","GO:0140352 (1413) <br> export from cell <br> 3.588e-14","GO:0048002 (193) <br> antigen processing and presentation of peptide antigen <br> 4.248e-14","GO:0002478 (175) <br> antigen processing and presentation of exogenous peptide antigen <br> 4.330e-14","GO:0034622 (1150) <br> cellular protein-containing complex assembly <br> 4.362e-14","GO:0071826 (203) <br> ribonucleoprotein complex subunit organization <br> 8.444e-14","GO:0034655 (573) <br> nucleobase-containing compound catabolic process <br> 8.962e-14","GO:0034097 (1242) <br> response to cytokine <br> 9.115e-14","GO:0019884 (183) <br> antigen processing and presentation of exogenous antigen <br> 1.292e-13","GO:0050851 (321) <br> antigen receptor-mediated signaling pathway <br> 2.078e-13","GO:0002831 (426) <br> regulation of response to biotic stimulus <br> 2.107e-13","GO:0046903 (1492) <br> secretion <br> 3.092e-13","GO:0035567 (151) <br> non-canonical Wnt signaling pathway <br> 1.092e-12","GO:0046700 (617) <br> heterocycle catabolic process <br> 1.378e-12","GO:0098542 (1222) <br> defense response to other organism <br> 1.574e-12","GO:0002181 (149) <br> cytoplasmic translation <br> 1.839e-12","GO:0044270 (623) <br> cellular nitrogen compound catabolic process <br> 1.961e-12","GO:0001738 (152) <br> morphogenesis of a polarized epithelium <br> 2.027e-12","GO:0006521 (66) <br> regulation of cellular amino acid metabolic process <br> 2.740e-12","GO:0006886 (995) <br> intracellular protein transport <br> 2.824e-12","GO:0019439 (636) <br> aromatic compound catabolic process <br> 4.151e-12","GO:0002376 (3438) <br> immune system process <br> 4.454e-12","GO:0060071 (110) <br> Wnt signaling pathway, planar cell polarity pathway <br> 4.578e-12","GO:0033238 (83) <br> regulation of cellular amine metabolic process <br> 7.244e-12","GO:0007164 (130) <br> establishment of tissue polarity <br> 8.384e-12","GO:0001736 (130) <br> establishment of planar polarity <br> 8.384e-12","GO:0090175 (114) <br> regulation of establishment of planar polarity <br> 9.114e-12","GO:0019221 (824) <br> cytokine-mediated signaling pathway <br> 1.008e-11","GO:0050657 (183) <br> nucleic acid transport <br> 1.288e-11","GO:0050658 (183) <br> RNA transport <br> 1.288e-11","GO:0051668 (775) <br> localization within membrane <br> 1.319e-11","GO:1902036 (74) <br> regulation of hematopoietic stem cell differentiation <br> 1.943e-11","GO:0051236 (186) <br> establishment of RNA localization <br> 1.946e-11","GO:1901361 (664) <br> organic cyclic compound catabolic process <br> 1.953e-11","GO:0044419 (1768) <br> biological process involved in interspecies interaction between organisms <br> 1.969e-11","GO:0006950 (4321) <br> response to stress <br> 2.181e-11","GO:0038061 (196) <br> NIK/NF-kappaB signaling <br> 3.366e-11","GO:0002682 (1734) <br> regulation of immune system process <br> 3.437e-11","GO:0031349 (364) <br> positive regulation of defense response <br> 3.603e-11","GO:0015031 (1754) <br> protein transport <br> 3.682e-11","GO:0010033 (3457) <br> response to organic substance <br> 3.928e-11","GO:0061418 (78) <br> regulation of transcription from RNA polymerase II promoter in response to hypoxia <br> 4.718e-11","GO:0043488 (247) <br> regulation of mRNA stability <br> 6.113e-11","GO:0051707 (1600) <br> response to other organism <br> 6.531e-11","GO:0043207 (1601) <br> response to external biotic stimulus <br> 6.728e-11","GO:0045184 (1866) <br> establishment of protein localization <br> 7.235e-11","GO:0031146 (95) <br> SCF-dependent proteasomal ubiquitin-dependent protein catabolic process <br> 7.807e-11","GO:0051028 (151) <br> mRNA transport <br> 7.991e-11","GO:0038095 (167) <br> Fc-epsilon receptor signaling pathway <br> 1.062e-10","GO:0016043 (6661) <br> cellular component organization <br> 1.227e-10","GO:0061013 (255) <br> regulation of mRNA catabolic process <br> 1.275e-10","GO:0072657 (698) <br> protein localization to membrane <br> 1.413e-10","GO:0010972 (99) <br> negative regulation of G2/M transition of mitotic cell cycle <br> 1.595e-10","GO:0043487 (259) <br> regulation of RNA stability <br> 1.823e-10","GO:0009607 (1639) <br> response to biotic stimulus <br> 2.040e-10","GO:0043620 (117) <br> regulation of DNA-templated transcription in response to stress <br> 2.097e-10","GO:1902750 (101) <br> negative regulation of cell cycle G2/M phase transition <br> 2.252e-10","GO:0022607 (3104) <br> cellular component assembly <br> 2.267e-10","GO:0006403 (222) <br> RNA localization <br> 2.302e-10","GO:0044265 (1244) <br> cellular macromolecule catabolic process <br> 2.848e-10","GO:0031145 (87) <br> anaphase-promoting complex-dependent catabolic process <br> 2.893e-10","GO:0060218 (89) <br> hematopoietic stem cell differentiation <br> 4.200e-10","GO:0070498 (105) <br> interleukin-1-mediated signaling pathway <br> 4.384e-10","GO:0015931 (246) <br> nucleobase-containing compound transport <br> 4.774e-10","GO:1901532 (91) <br> regulation of hematopoietic progenitor cell differentiation <br> 6.040e-10","GO:0071310 (2821) <br> cellular response to organic substance <br> 6.504e-10","GO:0043618 (111) <br> regulation of transcription from RNA polymerase II promoter in response to stress <br> 1.128e-09","GO:0070887 (3443) <br> cellular response to chemical stimulus <br> 1.233e-09","GO:0009057 (1479) <br> macromolecule catabolic process <br> 1.580e-09","GO:0006413 (203) <br> translational initiation <br> 1.631e-09","GO:2000736 (115) <br> regulation of stem cell differentiation <br> 2.051e-09","GO:0034341 (198) <br> response to interferon-gamma <br> 2.553e-09","GO:0006952 (1956) <br> defense response <br> 2.933e-09","GO:0050684 (145) <br> regulation of mRNA processing <br> 3.950e-09","GO:2000027 (185) <br> regulation of animal organ morphogenesis <br> 7.697e-09","GO:0009605 (3114) <br> response to external stimulus <br> 7.832e-09","GO:1903312 (95) <br> negative regulation of mRNA metabolic process <br> 1.017e-08","GO:0006260 (338) <br> DNA replication <br> 1.110e-08","GO:0032103 (518) <br> positive regulation of response to external stimulus <br> 1.667e-08","GO:0006405 (129) <br> RNA export from nucleus <br> 1.838e-08","GO:0090263 (153) <br> positive regulation of canonical Wnt signaling pathway <br> 2.308e-08","GO:0071347 (187) <br> cellular response to interleukin-1 <br> 3.600e-08","GO:0000165 (965) <br> MAPK cascade <br> 4.369e-08","GO:0033554 (2142) <br> cellular response to stress <br> 6.901e-08","GO:0070555 (217) <br> response to interleukin-1 <br> 7.478e-08","GO:0043484 (152) <br> regulation of RNA splicing <br> 8.024e-08","GO:0006605 (400) <br> protein targeting <br> 8.085e-08","GO:0044106 (166) <br> cellular amine metabolic process <br> 9.215e-08","GO:0090150 (385) <br> establishment of protein localization to membrane <br> 1.256e-07","GO:0006261 (216) <br> DNA-dependent DNA replication <br> 1.314e-07","GO:0042221 (4829) <br> response to chemical <br> 1.443e-07","GO:0009308 (171) <br> amine metabolic process <br> 1.517e-07","GO:0010565 (196) <br> regulation of cellular ketone metabolic process <br> 1.861e-07","GO:0019080 (237) <br> viral gene expression <br> 2.764e-07","GO:0031347 (889) <br> regulation of defense response <br> 2.958e-07","GO:0035556 (2886) <br> intracellular signal transduction <br> 3.011e-07","GO:0010389 (204) <br> regulation of G2/M transition of mitotic cell cycle <br> 3.723e-07","GO:0071346 (177) <br> cellular response to interferon-gamma <br> 4.140e-07","GO:0033365 (974) <br> protein localization to organelle <br> 4.678e-07","GO:0030177 (186) <br> positive regulation of Wnt signaling pathway <br> 6.132e-07","GO:0048024 (107) <br> regulation of mRNA splicing, via spliceosome <br> 6.652e-07","GO:0010629 (1470) <br> negative regulation of gene expression <br> 6.805e-07","GO:0000184 (123) <br> nuclear-transcribed mRNA catabolic process, nonsense-mediated decay <br> 7.207e-07","GO:1902749 (213) <br> regulation of cell cycle G2/M phase transition <br> 7.815e-07","GO:0048584 (2368) <br> positive regulation of response to stimulus <br> 8.205e-07","GO:0080134 (1626) <br> regulation of response to stress <br> 8.378e-07","GO:0010608 (928) <br> posttranscriptional regulation of gene expression <br> 9.461e-07","GO:0006406 (112) <br> mRNA export from nucleus <br> 9.883e-07","GO:0071427 (112) <br> mRNA-containing ribonucleoprotein complex export from nucleus <br> 9.883e-07","GO:0048518 (6474) <br> positive regulation of biological process <br> 1.191e-06","GO:0010499 (23) <br> proteasomal ubiquitin-independent protein catabolic process <br> 1.406e-06","GO:0006412 (804) <br> translation <br> 1.427e-06","GO:0042180 (274) <br> cellular ketone metabolic process <br> 1.481e-06","GO:0007159 (377) <br> leukocyte cell-cell adhesion <br> 1.578e-06","GO:0009894 (1081) <br> regulation of catabolic process <br> 1.711e-06","GO:0036294 (224) <br> cellular response to decreased oxygen levels <br> 1.840e-06","GO:0065004 (300) <br> protein-DNA complex assembly <br> 1.841e-06","GO:0030168 (163) <br> platelet activation <br> 2.118e-06","GO:0002244 (175) <br> hematopoietic progenitor cell differentiation <br> 2.197e-06","GO:0033209 (179) <br> tumor necrosis factor-mediated signaling pathway <br> 2.622e-06","GO:0002480 (8) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-independent <br> 2.715e-06","GO:0033119 (27) <br> negative regulation of RNA splicing <br> 2.909e-06","GO:0000380 (76) <br> alternative mRNA splicing, via spliceosome <br> 3.431e-06","GO:1901991 (260) <br> negative regulation of mitotic cell cycle phase transition <br> 3.619e-06","GO:0009056 (2779) <br> catabolic process <br> 3.802e-06","GO:0051169 (356) <br> nuclear transport <br> 4.053e-06","GO:0000086 (262) <br> G2/M transition of mitotic cell cycle <br> 4.135e-06","GO:0043043 (834) <br> peptide biosynthetic process <br> 4.201e-06","GO:0044248 (2381) <br> cellular catabolic process <br> 4.388e-06","GO:0043604 (975) <br> amide biosynthetic process <br> 4.979e-06","GO:0065008 (4116) <br> regulation of biological quality <br> 5.009e-06","GO:0071824 (346) <br> protein-DNA complex subunit organization <br> 5.802e-06","GO:0071456 (214) <br> cellular response to hypoxia <br> 5.974e-06","GO:0090090 (189) <br> negative regulation of canonical Wnt signaling pathway <br> 6.059e-06","GO:0060341 (851) <br> regulation of cellular localization <br> 6.896e-06","GO:0071453 (243) <br> cellular response to oxygen levels <br> 7.200e-06","GO:0019538 (5912) <br> protein metabolic process <br> 7.698e-06","GO:0044839 (273) <br> cell cycle G2/M phase transition <br> 8.420e-06","GO:0006518 (998) <br> peptide metabolic process <br> 9.249e-06","GO:1901575 (2289) <br> organic substance catabolic process <br> 9.697e-06","GO:0071426 (130) <br> ribonucleoprotein complex export from nucleus <br> 1.043e-05","GO:0016055 (533) <br> Wnt signaling pathway <br> 1.112e-05","GO:0048025 (22) <br> negative regulation of mRNA splicing, via spliceosome <br> 1.128e-05","GO:0031329 (913) <br> regulation of cellular catabolic process <br> 1.133e-05","GO:0044267 (5280) <br> cellular protein metabolic process <br> 1.167e-05","GO:0071166 (131) <br> ribonucleoprotein complex localization <br> 1.174e-05","GO:0198738 (535) <br> cell-cell signaling by wnt <br> 1.207e-05","GO:0071356 (308) <br> cellular response to tumor necrosis factor <br> 1.217e-05","GO:0048522 (5863) <br> positive regulation of cellular process <br> 1.390e-05","GO:0006913 (353) <br> nucleocytoplasmic transport <br> 1.408e-05","GO:0006364 (265) <br> rRNA processing <br> 1.475e-05","GO:0031503 (289) <br> protein-containing complex localization <br> 1.671e-05","GO:0060333 (88) <br> interferon-gamma-mediated signaling pathway <br> 2.048e-05","GO:0036293 (382) <br> response to decreased oxygen levels <br> 2.351e-05","GO:1903827 (563) <br> regulation of cellular protein localization <br> 2.354e-05","GO:0006457 (230) <br> protein folding <br> 2.532e-05","GO:0022407 (450) <br> regulation of cell-cell adhesion <br> 2.652e-05","GO:0048863 (261) <br> stem cell differentiation <br> 2.808e-05","GO:0016072 (275) <br> rRNA metabolic process <br> 2.839e-05","GO:0110096 (12) <br> cellular response to aldehyde <br> 3.099e-05","GO:0002009 (566) <br> morphogenesis of an epithelium <br> 3.302e-05","GO:0010498 (501) <br> proteasomal protein catabolic process <br> 3.509e-05","GO:1901988 (329) <br> negative regulation of cell cycle phase transition <br> 3.879e-05","GO:0007596 (351) <br> blood coagulation <br> 4.190e-05","GO:0034612 (332) <br> response to tumor necrosis factor <br> 4.543e-05","GO:0060828 (302) <br> regulation of canonical Wnt signaling pathway <br> 4.675e-05","GO:0043161 (442) <br> proteasome-mediated ubiquitin-dependent protein catabolic process <br> 4.824e-05","GO:0007599 (355) <br> hemostasis <br> 4.988e-05","GO:0001666 (367) <br> response to hypoxia <br> 5.413e-05","GO:0050817 (358) <br> coagulation <br> 5.676e-05","GO:0043603 (1284) <br> cellular amide metabolic process <br> 5.758e-05","GO:0051168 (202) <br> nuclear export <br> 5.912e-05","GO:0030178 (220) <br> negative regulation of Wnt signaling pathway <br> 5.974e-05","GO:0007166 (3226) <br> cell surface receptor signaling pathway <br> 6.169e-05","GO:0070972 (144) <br> protein localization to endoplasmic reticulum <br> 6.570e-05","GO:0010467 (6192) <br> gene expression <br> 6.677e-05","GO:0030155 (756) <br> regulation of cell adhesion <br> 8.552e-05","GO:0070482 (410) <br> response to oxygen levels <br> 8.734e-05","GO:0019081 (20) <br> viral translation <br> 8.806e-05","GO:0031342 (24) <br> negative regulation of cell killing <br> 9.480e-05","GO:0010948 (400) <br> negative regulation of cell cycle process <br> 9.641e-05","GO:0002699 (228) <br> positive regulation of immune effector process <br> 1.020e-04","GO:0072594 (523) <br> establishment of protein localization to organelle <br> 1.040e-04","GO:0042060 (555) <br> wound healing <br> 1.063e-04","GO:0045930 (318) <br> negative regulation of mitotic cell cycle <br> 1.103e-04","GO:0019058 (386) <br> viral life cycle <br> 1.139e-04","GO:0006897 (700) <br> endocytosis <br> 1.207e-04","GO:0022409 (286) <br> positive regulation of cell-cell adhesion <br> 1.222e-04","GO:1901990 (453) <br> regulation of mitotic cell cycle phase transition <br> 1.284e-04","GO:0072659 (287) <br> protein localization to plasma membrane <br> 1.302e-04","GO:1905114 (639) <br> cell surface receptor signaling pathway involved in cell-cell signaling <br> 1.460e-04","GO:0042254 (335) <br> ribosome biogenesis <br> 1.903e-04","GO:0006612 (219) <br> protein targeting to membrane <br> 2.006e-04","GO:1990778 (339) <br> protein localization to cell periphery <br> 2.063e-04","GO:0032101 (1238) <br> regulation of response to external stimulus <br> 2.067e-04","GO:0050686 (31) <br> negative regulation of mRNA processing <br> 2.391e-04","GO:0031124 (104) <br> mRNA 3'-end processing <br> 2.617e-04","GO:0010628 (1206) <br> positive regulation of gene expression <br> 2.797e-04","GO:1903039 (243) <br> positive regulation of leukocyte cell-cell adhesion <br> 3.012e-04","GO:0051701 (243) <br> biological process involved in interaction with host <br> 3.045e-04","GO:0098609 (898) <br> cell-cell adhesion <br> 3.936e-04","GO:0048268 (18) <br> clathrin coat assembly <br> 4.133e-04","GO:1903037 (341) <br> regulation of leukocyte cell-cell adhesion <br> 4.152e-04","GO:0022603 (1048) <br> regulation of anatomical structure morphogenesis <br> 4.262e-04","GO:0048729 (675) <br> tissue morphogenesis <br> 4.265e-04","GO:0032880 (912) <br> regulation of protein localization <br> 4.326e-04","GO:0050896 (9487) <br> response to stimulus <br> 4.975e-04","GO:0060070 (349) <br> canonical Wnt signaling pathway <br> 5.018e-04","GO:0002483 (19) <br> antigen processing and presentation of endogenous peptide antigen <br> 5.283e-04","GO:0034113 (60) <br> heterotypic cell-cell adhesion <br> 5.309e-04","GO:0006520 (344) <br> cellular amino acid metabolic process <br> 5.396e-04","GO:1905475 (203) <br> regulation of protein localization to membrane <br> 5.754e-04","GO:1904375 (128) <br> regulation of protein localization to cell periphery <br> 6.067e-04","GO:0060337 (96) <br> type I interferon signaling pathway <br> 6.300e-04","GO:0061024 (1016) <br> membrane organization <br> 6.409e-04","GO:0071357 (97) <br> cellular response to type I interferon <br> 6.890e-04","GO:0050863 (341) <br> regulation of T cell activation <br> 7.102e-04","GO:0050878 (519) <br> regulation of body fluid levels <br> 7.179e-04","GO:0009628 (1264) <br> response to abiotic stimulus <br> 7.475e-04","GO:0007346 (612) <br> regulation of mitotic cell cycle <br> 7.554e-04","GO:0001911 (20) <br> negative regulation of leukocyte mediated cytotoxicity <br> 7.705e-04","GO:2000278 (112) <br> regulation of DNA biosynthetic process <br> 8.008e-04","GO:0032386 (365) <br> regulation of intracellular transport <br> 8.368e-04","GO:0030163 (983) <br> protein catabolic process <br> 8.586e-04","GO:1905477 (132) <br> positive regulation of protein localization to membrane <br> 9.098e-04","GO:0045953 (16) <br> negative regulation of natural killer cell mediated cytotoxicity <br> 9.391e-04","GO:0044257 (822) <br> cellular protein catabolic process <br> 9.442e-04","GO:0002486 (3) <br> antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway, TAP-independent <br> 9.443e-04","GO:0002484 (3) <br> antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway <br> 9.443e-04","GO:1901987 (543) <br> regulation of cell cycle phase transition <br> 9.705e-04","GO:0051716 (7873) <br> cellular response to stimulus <br> 1.013e-03","GO:0044403 (331) <br> biological process involved in symbiotic interaction <br> 1.024e-03","GO:0034340 (102) <br> response to type I interferon <br> 1.062e-03","GO:1901564 (6904) <br> organonitrogen compound metabolic process <br> 1.193e-03","GO:0044772 (588) <br> mitotic cell cycle phase transition <br> 1.215e-03","GO:0002716 (17) <br> negative regulation of natural killer cell mediated immunity <br> 1.320e-03","GO:0048255 (57) <br> mRNA stabilization <br> 1.350e-03","GO:0043687 (362) <br> post-translational protein modification <br> 1.448e-03","GO:0016579 (292) <br> protein deubiquitination <br> 1.455e-03","GO:0042110 (495) <br> T cell activation <br> 1.460e-03","GO:0072599 (115) <br> establishment of protein localization to endoplasmic reticulum <br> 1.465e-03","GO:0046649 (780) <br> lymphocyte activation <br> 1.490e-03","GO:0071349 (49) <br> cellular response to interleukin-12 <br> 1.494e-03","GO:0051640 (657) <br> organelle localization <br> 1.525e-03","GO:0007049 (1867) <br> cell cycle <br> 1.565e-03","GO:0043300 (46) <br> regulation of leukocyte degranulation <br> 1.584e-03","GO:0006613 (102) <br> cotranslational protein targeting to membrane <br> 1.684e-03","GO:0006614 (96) <br> SRP-dependent cotranslational protein targeting to membrane <br> 1.691e-03","GO:0048193 (415) <br> Golgi vesicle transport <br> 1.741e-03","GO:1903829 (307) <br> positive regulation of cellular protein localization <br> 1.757e-03","GO:0070671 (50) <br> response to interleukin-12 <br> 1.789e-03","GO:0051656 (443) <br> establishment of organelle localization <br> 1.831e-03","GO:0030111 (380) <br> regulation of Wnt signaling pathway <br> 1.930e-03","GO:0000956 (214) <br> nuclear-transcribed mRNA catabolic process <br> 1.984e-03","GO:0044770 (681) <br> cell cycle phase transition <br> 2.331e-03","GO:1901698 (1178) <br> response to nitrogen compound <br> 2.368e-03","GO:0000278 (1098) <br> mitotic cell cycle <br> 2.435e-03","GO:0045040 (5) <br> protein insertion into mitochondrial outer membrane <br> 2.465e-03","GO:0007008 (5) <br> outer mitochondrial membrane organization <br> 2.465e-03","GO:0050870 (221) <br> positive regulation of T cell activation <br> 2.480e-03","GO:1903047 (956) <br> mitotic cell cycle process <br> 2.692e-03","GO:0075525 (5) <br> viral translational termination-reinitiation <br> 2.746e-03","GO:0019883 (26) <br> antigen processing and presentation of endogenous antigen <br> 2.868e-03","GO:0001909 (113) <br> leukocyte mediated cytotoxicity <br> 2.994e-03","GO:1902373 (64) <br> negative regulation of mRNA catabolic process <br> 3.016e-03","GO:0140029 (79) <br> exocytic process <br> 3.259e-03","GO:0043170 (9867) <br> macromolecule metabolic process <br> 3.333e-03","GO:0009611 (680) <br> response to wounding <br> 3.400e-03","GO:0070646 (310) <br> protein modification by small protein removal <br> 3.449e-03","GO:0045786 (487) <br> negative regulation of cell cycle <br> 3.489e-03","GO:0042267 (65) <br> natural killer cell mediated cytotoxicity <br> 3.555e-03","GO:0043489 (66) <br> RNA stabilization <br> 3.728e-03","GO:0007229 (112) <br> integrin-mediated signaling pathway <br> 3.823e-03","GO:0000245 (59) <br> spliceosomal complex assembly <br> 4.525e-03","GO:0019885 (13) <br> antigen processing and presentation of endogenous peptide antigen via MHC class I <br> 4.568e-03","GO:1901699 (728) <br> cellular response to nitrogen compound <br> 5.349e-03","GO:0002228 (68) <br> natural killer cell mediated immunity <br> 5.445e-03","GO:0001913 (42) <br> T cell mediated cytotoxicity <br> 6.835e-03","GO:0010564 (815) <br> regulation of cell cycle process <br> 7.107e-03","GO:0008152 (11915) <br> metabolic process <br> 7.208e-03","GO:0045047 (111) <br> protein targeting to ER <br> 7.247e-03","GO:0000209 (343) <br> protein polyubiquitination <br> 7.275e-03","GO:0019222 (7175) <br> regulation of metabolic process <br> 7.422e-03","GO:0042269 (41) <br> regulation of natural killer cell mediated cytotoxicity <br> 7.681e-03","GO:0071897 (201) <br> DNA biosynthetic process <br> 8.548e-03","GO:0006446 (82) <br> regulation of translational initiation <br> 8.741e-03","GO:0009967 (1626) <br> positive regulation of signal transduction <br> 9.415e-03","GO:1902369 (76) <br> negative regulation of RNA catabolic process <br> 9.743e-03","GO:0006891 (36) <br> intra-Golgi vesicle-mediated transport <br> 1.040e-02","GO:0006996 (4136) <br> organelle organization <br> 1.072e-02","GO:0035722 (47) <br> interleukin-12-mediated signaling pathway <br> 1.165e-02","GO:1904377 (71) <br> positive regulation of protein localization to cell periphery <br> 1.181e-02","GO:0022610 (1509) <br> biological adhesion <br> 1.222e-02","GO:0032388 (227) <br> positive regulation of intracellular transport <br> 1.262e-02","GO:0062012 (430) <br> regulation of small molecule metabolic process <br> 1.272e-02","GO:0001914 (35) <br> regulation of T cell mediated cytotoxicity <br> 1.342e-02","GO:0002715 (45) <br> regulation of natural killer cell mediated immunity <br> 1.346e-02","GO:0002703 (212) <br> regulation of leukocyte mediated immunity <br> 1.350e-02","GO:0071407 (624) <br> cellular response to organic cyclic compound <br> 1.380e-02","GO:0070934 (5) <br> CRD-mediated mRNA stabilization <br> 1.515e-02","GO:1903076 (107) <br> regulation of protein localization to plasma membrane <br> 1.546e-02","GO:0034641 (6576) <br> cellular nitrogen compound metabolic process <br> 1.596e-02","GO:0048583 (4416) <br> regulation of response to stimulus <br> 1.599e-02","GO:0050821 (195) <br> protein stabilization <br> 1.744e-02","GO:0034470 (439) <br> ncRNA processing <br> 1.748e-02","GO:0001906 (196) <br> cell killing <br> 1.791e-02","GO:0019079 (131) <br> viral genome replication <br> 1.814e-02","GO:0000723 (169) <br> telomere maintenance <br> 1.842e-02","GO:0000381 (60) <br> regulation of alternative mRNA splicing, via spliceosome <br> 1.857e-02","GO:0042270 (6) <br> protection from natural killer cell mediated cytotoxicity <br> 1.901e-02","GO:0022402 (1388) <br> cell cycle process <br> 1.965e-02","GO:0060255 (6622) <br> regulation of macromolecule metabolic process <br> 2.010e-02","GO:0007212 (41) <br> dopamine receptor signaling pathway <br> 2.045e-02","GO:0051246 (2759) <br> regulation of protein metabolic process <br> 2.105e-02","GO:0002697 (421) <br> regulation of immune effector process <br> 2.161e-02","GO:0072583 (47) <br> clathrin-dependent endocytosis <br> 2.268e-02","GO:0034248 (577) <br> regulation of cellular amide metabolic process <br> 2.314e-02","GO:0007165 (6447) <br> signal transduction <br> 2.437e-02","GO:0001910 (75) <br> regulation of leukocyte mediated cytotoxicity <br> 2.471e-02","GO:0019752 (1033) <br> carboxylic acid metabolic process <br> 2.576e-02","GO:0007155 (1502) <br> cell adhesion <br> 2.581e-02","GO:0001819 (498) <br> positive regulation of cytokine production <br> 2.642e-02","GO:0010833 (87) <br> telomere maintenance via telomere lengthening <br> 2.763e-02","GO:0007265 (352) <br> Ras protein signal transduction <br> 2.779e-02","GO:0006626 (102) <br> protein targeting to mitochondrion <br> 2.790e-02","GO:0032879 (2906) <br> regulation of localization <br> 2.863e-02","GO:1901565 (1375) <br> organonitrogen compound catabolic process <br> 2.873e-02","GO:0051650 (212) <br> establishment of vesicle localization <br> 2.896e-02","GO:0031123 (157) <br> RNA 3'-end processing <br> 2.899e-02","GO:0006898 (335) <br> receptor-mediated endocytosis <br> 2.915e-02","GO:1901566 (1889) <br> organonitrogen compound biosynthetic process <br> 2.956e-02","GO:0044237 (10941) <br> cellular metabolic process <br> 3.023e-02","GO:0019083 (181) <br> viral transcription <br> 3.333e-02","GO:0009968 (1394) <br> negative regulation of signal transduction <br> 3.335e-02","GO:0071704 (11397) <br> organic substance metabolic process <br> 3.394e-02","GO:0031341 (109) <br> regulation of cell killing <br> 3.412e-02","GO:0006417 (504) <br> regulation of translation <br> 3.442e-02","GO:0002694 (617) <br> regulation of leukocyte activation <br> 3.544e-02","GO:1903078 (64) <br> positive regulation of protein localization to plasma membrane <br> 3.587e-02","GO:0006890 (87) <br> retrograde vesicle-mediated transport, Golgi to endoplasmic reticulum <br> 3.635e-02","GO:0006082 (1098) <br> organic acid metabolic process <br> 3.738e-02","GO:0002704 (55) <br> negative regulation of leukocyte mediated immunity <br> 3.770e-02","GO:0006909 (388) <br> phagocytosis <br> 3.860e-02","GO:0031647 (302) <br> regulation of protein stability <br> 3.901e-02","GO:0075522 (11) <br> IRES-dependent viral translational initiation <br> 3.997e-02","GO:0017157 (208) <br> regulation of exocytosis <br> 4.283e-02","GO:0006888 (208) <br> endoplasmic reticulum to Golgi vesicle-mediated transport <br> 4.416e-02","GO:0010243 (1092) <br> response to organonitrogen compound <br> 4.526e-02","GO:0048519 (5861) <br> negative regulation of biological process <br> 4.789e-02","GO:0032200 (183) <br> telomere organization <br> 4.793e-02","GO:0051726 (1120) <br> regulation of cell cycle <br> 4.826e-02","GO:0043436 (1066) <br> oxoacid metabolic process <br> 4.949e-02"],"key":["GO:0051649","GO:0016071","GO:0051641","GO:0016032","GO:0043299","GO:0002275","GO:0043312","GO:0002444","GO:0000377","GO:0000398","GO:0002283","GO:0000375","GO:0036230","GO:0002446","GO:0006810","GO:0042119","GO:0051234","GO:0045055","GO:0008380","GO:0006396","GO:0046907","GO:0001775","GO:0050852","GO:1903311","GO:0071705","GO:0051179","GO:0006397","GO:0006887","GO:0002274","GO:0033036","GO:0002366","GO:0002479","GO:0002263","GO:0002474","GO:0022613","GO:0002252","GO:0006402","GO:0042590","GO:0050778","GO:0045088","GO:0002757","GO:0002429","GO:0045321","GO:0002218","GO:0002768","GO:0045089","GO:0002223","GO:0006401","GO:0002764","GO:0002443","GO:0002253","GO:1990840","GO:1990858","GO:0002684","GO:0038093","GO:0002220","GO:0002758","GO:0019882","GO:0034613","GO:0070727","GO:0006955","GO:0036388","GO:0071702","GO:0050776","GO:0044085","GO:0002833","GO:0032940","GO:0008104","GO:0016192","GO:0045087","GO:0071840","GO:0002183","GO:0043933","GO:0001732","GO:0065003","GO:0022618","GO:0071345","GO:0140352","GO:0048002","GO:0002478","GO:0034622","GO:0071826","GO:0034655","GO:0034097","GO:0019884","GO:0050851","GO:0002831","GO:0046903","GO:0035567","GO:0046700","GO:0098542","GO:0002181","GO:0044270","GO:0001738","GO:0006521","GO:0006886","GO:0019439","GO:0002376","GO:0060071","GO:0033238","GO:0007164","GO:0001736","GO:0090175","GO:0019221","GO:0050657","GO:0050658","GO:0051668","GO:1902036","GO:0051236","GO:1901361","GO:0044419","GO:0006950","GO:0038061","GO:0002682","GO:0031349","GO:0015031","GO:0010033","GO:0061418","GO:0043488","GO:0051707","GO:0043207","GO:0045184","GO:0031146","GO:0051028","GO:0038095","GO:0016043","GO:0061013","GO:0072657","GO:0010972","GO:0043487","GO:0009607","GO:0043620","GO:1902750","GO:0022607","GO:0006403","GO:0044265","GO:0031145","GO:0060218","GO:0070498","GO:0015931","GO:1901532","GO:0071310","GO:0043618","GO:0070887","GO:0009057","GO:0006413","GO:2000736","GO:0034341","GO:0006952","GO:0050684","GO:2000027","GO:0009605","GO:1903312","GO:0006260","GO:0032103","GO:0006405","GO:0090263","GO:0071347","GO:0000165","GO:0033554","GO:0070555","GO:0043484","GO:0006605","GO:0044106","GO:0090150","GO:0006261","GO:0042221","GO:0009308","GO:0010565","GO:0019080","GO:0031347","GO:0035556","GO:0010389","GO:0071346","GO:0033365","GO:0030177","GO:0048024","GO:0010629","GO:0000184","GO:1902749","GO:0048584","GO:0080134","GO:0010608","GO:0006406","GO:0071427","GO:0048518","GO:0010499","GO:0006412","GO:0042180","GO:0007159","GO:0009894","GO:0036294","GO:0065004","GO:0030168","GO:0002244","GO:0033209","GO:0002480","GO:0033119","GO:0000380","GO:1901991","GO:0009056","GO:0051169","GO:0000086","GO:0043043","GO:0044248","GO:0043604","GO:0065008","GO:0071824","GO:0071456","GO:0090090","GO:0060341","GO:0071453","GO:0019538","GO:0044839","GO:0006518","GO:1901575","GO:0071426","GO:0016055","GO:0048025","GO:0031329","GO:0044267","GO:0071166","GO:0198738","GO:0071356","GO:0048522","GO:0006913","GO:0006364","GO:0031503","GO:0060333","GO:0036293","GO:1903827","GO:0006457","GO:0022407","GO:0048863","GO:0016072","GO:0110096","GO:0002009","GO:0010498","GO:1901988","GO:0007596","GO:0034612","GO:0060828","GO:0043161","GO:0007599","GO:0001666","GO:0050817","GO:0043603","GO:0051168","GO:0030178","GO:0007166","GO:0070972","GO:0010467","GO:0030155","GO:0070482","GO:0019081","GO:0031342","GO:0010948","GO:0002699","GO:0072594","GO:0042060","GO:0045930","GO:0019058","GO:0006897","GO:0022409","GO:1901990","GO:0072659","GO:1905114","GO:0042254","GO:0006612","GO:1990778","GO:0032101","GO:0050686","GO:0031124","GO:0010628","GO:1903039","GO:0051701","GO:0098609","GO:0048268","GO:1903037","GO:0022603","GO:0048729","GO:0032880","GO:0050896","GO:0060070","GO:0002483","GO:0034113","GO:0006520","GO:1905475","GO:1904375","GO:0060337","GO:0061024","GO:0071357","GO:0050863","GO:0050878","GO:0009628","GO:0007346","GO:0001911","GO:2000278","GO:0032386","GO:0030163","GO:1905477","GO:0045953","GO:0044257","GO:0002486","GO:0002484","GO:1901987","GO:0051716","GO:0044403","GO:0034340","GO:1901564","GO:0044772","GO:0002716","GO:0048255","GO:0043687","GO:0016579","GO:0042110","GO:0072599","GO:0046649","GO:0071349","GO:0051640","GO:0007049","GO:0043300","GO:0006613","GO:0006614","GO:0048193","GO:1903829","GO:0070671","GO:0051656","GO:0030111","GO:0000956","GO:0044770","GO:1901698","GO:0000278","GO:0045040","GO:0007008","GO:0050870","GO:1903047","GO:0075525","GO:0019883","GO:0001909","GO:1902373","GO:0140029","GO:0043170","GO:0009611","GO:0070646","GO:0045786","GO:0042267","GO:0043489","GO:0007229","GO:0000245","GO:0019885","GO:1901699","GO:0002228","GO:0001913","GO:0010564","GO:0008152","GO:0045047","GO:0000209","GO:0019222","GO:0042269","GO:0071897","GO:0006446","GO:0009967","GO:1902369","GO:0006891","GO:0006996","GO:0035722","GO:1904377","GO:0022610","GO:0032388","GO:0062012","GO:0001914","GO:0002715","GO:0002703","GO:0071407","GO:0070934","GO:1903076","GO:0034641","GO:0048583","GO:0050821","GO:0034470","GO:0001906","GO:0019079","GO:0000723","GO:0000381","GO:0042270","GO:0022402","GO:0060255","GO:0007212","GO:0051246","GO:0002697","GO:0072583","GO:0034248","GO:0007165","GO:0001910","GO:0019752","GO:0007155","GO:0001819","GO:0010833","GO:0007265","GO:0006626","GO:0032879","GO:1901565","GO:0051650","GO:0031123","GO:0006898","GO:1901566","GO:0044237","GO:0019083","GO:0009968","GO:0071704","GO:0031341","GO:0006417","GO:0002694","GO:1903078","GO:0006890","GO:0006082","GO:0002704","GO:0006909","GO:0031647","GO:0075522","GO:0017157","GO:0006888","GO:0010243","GO:0048519","GO:0032200","GO:0051726","GO:0043436"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[20.6760128426016,19.2759171540558,20.9498382841373,19.451573873278,18.6363439324449,18.6704142203647,18.4924595182474,18.6847287497765,18.0929757080159,18.0929757080159,18.5062609951603,18.1044082182465,18.5625892117027,18.5387647885811,21.395744596271,18.5440992955177,21.4274758345104,19.1379652435679,18.4239673394241,19.4503282144806,20.1088323582115,19.9462123686674,17.2731463069687,18.181645451538,20.4100380870974,21.7127740996053,18.6214455917298,19.3129380919747,18.9268195002008,20.8213641129297,19.0237128877517,15.7373316539529,19.0308506328191,16.1868772799724,18.4643827493734,19.5965245859779,18.195832112632,15.8433279821367,19.0830626021267,17.9121743965842,18.470049611806,18.470049611806,19.8031784771994,16.7689423433419,18.5652079791051,17.3774579655542,16.3815087539172,18.3389367682336,18.5730308200746,19.287432915256,18.7035562554922,16.3954151879507,16.3954151879507,19.5332947906194,17.5108080471375,16.4228177328561,16.4363189409718,17.4986794977524,20.1893119996143,20.1984114205898,20.5729683353868,15.5245580341656,20.6239407817799,19.6225996651137,20.9024525251713,17.598331775882,19.8123220388678,20.6296556443227,20.3976580599366,19.4415711203883,21.7035374999992,14.405947329161,20.2980535736863,12.8370677538266,20.1095332573674,17.2289935746564,19.6107216016844,19.8674339300097,17.206351575782,17.0617353348181,19.6128903566504,17.2803636129636,18.7197942104249,19.7084847763908,17.1279572175344,17.9340698285406,18.3231184550737,19.9340018163666,16.840827913987,18.8171599959616,19.6883679423801,16.820675004284,18.8298494843551,16.8507929922937,15.5245580341656,19.4314817950327,18.8568934935132,20.9230640505876,16.3532722717548,15.9033751977876,16.6126778615783,16.6126778615783,16.4091839862407,19.191991296678,17.1279572175344,17.1279572175344,19.1133238346132,15.7151677983179,17.1519653138816,18.9131562034715,20.1399480363029,21.1841117247357,17.2289935746564,20.1165183285464,18.1081960550518,20.1303596037508,20.9294044050122,15.801858092204,17.5640277699944,20.0191155003357,20.0198744325906,20.2048660232527,16.121131665261,16.840827913987,16.9920586807013,21.6679004416094,17.6095673525824,18.9781080980935,16.1868772799724,17.631749719545,20.048342405881,16.4496900554146,16.2186353293393,20.8050634202662,17.4104643922402,19.710477230295,15.9797164726717,16.0164101032087,16.2800766733809,17.5582210891217,16.0521821733416,20.6939359660643,16.3674620265117,20.9247361833321,19.9233120507587,17.2803636129636,16.4228177328561,17.2438757382447,20.2613396475456,16.7794477421486,17.1440106914419,20.808789697542,16.121131665261,18.0058060899366,18.5859587150382,16.6008025798044,16.8606852085784,17.1598725929261,19.3928503778594,20.3697031428378,17.3774579655542,16.8507929922937,18.2374501939285,16.9830886832557,18.1852072362727,17.3707557108492,21.3096384784207,17.0273503124218,17.2289935746564,17.5047579060072,19.2888644874537,20.7204820590056,17.2875416251238,17.0786016380891,19.4045745794367,17.1519653138816,16.309816295931,19.915852066474,16.5273136649716,17.3504409902673,20.4885440438041,20.0386839143178,19.3433767765611,16.3815087539172,16.3815087539172,21.636471805168,13.5956642813554,19.1605106996624,17.7116929576767,18.1564249614111,19.5356134332147,17.4234373663266,17.8394624222953,16.9558134205769,17.0617353348181,17.0952574234415,11.1749781597015,13.9127623976552,15.7591617451593,17.6372362975318,20.6764330190036,18.0775688230189,17.6481399798121,19.2074226898582,20.495006256239,19.4058700354974,21.1289356985204,18.0382058611615,17.3572475673179,17.1755472645488,19.2332018170431,17.540643442317,21.5358724307931,17.7065151123898,19.4352755293199,20.4484382631651,16.6126778615783,18.6239413228969,13.5059251363754,19.3227090673339,21.4098243887321,16.624451291171,18.6289175161382,17.8763374918434,21.5266239363287,18.0658885682371,17.6643246110046,17.7869538601274,15.9981813072674,18.1744913095535,18.6965300907626,17.4616009111547,18.3972686497445,17.6426996548312,17.7168500269786,12.187219027138,18.7035562554922,18.5414349449467,17.9683348657126,18.0580410920994,17.9809490074102,17.8487816883629,18.3730421667754,18.073687436303,18.119491495218,18.0852958577034,19.7496169904031,17.2731463069687,17.39736148716,20.849671586304,16.7689423433419,21.5872057726815,19.0813586108141,18.27110133951,13.3106655321065,13.6808214449962,18.2374501939285,17.4490032718266,18.5987513048536,18.6775925483423,17.9209779347506,18.1887589046007,18.9818212341931,17.7722510341765,18.4062325191928,17.7771707617082,18.863049110292,17.9934386332075,17.3907605242587,18.0099018230755,19.7044894741966,14.1782205386673,16.2649666785139,19.6720172509667,17.540643442317,17.540643442317,19.3016707664123,13.0900518224006,18.0180536596155,19.4967505982493,18.9345577744754,19.3213181846121,22.0537265707873,18.0501444205764,13.2039079443259,15.3633750239743,18.0301835617948,17.2803636129636,16.5888237442405,16.1378571469232,19.4577822668577,16.1543870489597,18.0180536596155,18.5885279695768,19.7302139461961,18.8064825080221,13.3106655321065,16.3815087539172,18.1119725030245,19.4161822734211,16.6361245292714,12.8370677538266,19.1888806838206,7.7791008157754,7.7791008157754,18.6486216054544,21.8513047485316,17.9767582464177,16.2342495236679,21.7073695277206,18.7538745275872,12.968152232712,15.2756954716879,18.1006089262143,17.8014896682064,18.5253260577865,16.4228177328561,19.1215946463033,15.0132022257977,18.8993355559252,20.2055094409634,14.9017074238205,16.2342495236679,16.1378571469232,18.2875917936538,17.8717860566539,15.0486221338682,18.3760965631462,18.1672960274771,17.3572475673179,18.9460723109363,19.6428327290729,19.5551369029075,9.79200637188604,9.79200637188604,17.4039293372047,19.3810073827861,9.79200637188604,13.83902557929,16.3954151879507,15.4727616697066,15.8227421236023,22.0960714860976,18.9441608893166,17.8853918269963,18.5035130791456,15.498887690643,15.5245580341656,16.3815087539172,15.3347186146137,12.3729643352122,19.032628413028,15.5745906356112,14.73917293128,19.1779286245093,22.298075013352,16.3674620265117,18.0261532007279,21.7496835896761,14.6957136866834,17.265889291748,15.8836369590203,20.0386839143178,15.7591617451593,14.4581139285914,21.1344462088703,14.9397854919929,15.6465726979379,19.9478305598874,17.4426585575357,18.3357892079078,14.405947329161,14.862657240381,17.3435989423967,18.8319514777529,9.79200637188604,16.309816295931,21.6537323834277,21.2087426859582,17.2214892061892,18.3638334737416,17.2289935746564,16.624451291171,17.0098206791988,15.3633750239743,10.3630624020686,19.8455306846866,21.6614237786868,14.6957136866834,20.6679986229776,18.3070958506964,14.9397854919929,18.7289763933658,21.63185447316,15.7373316539529,19.4786465831394,19.942156764631,18.5334069604083,15.9797164726717,18.0619709426399,16.2342495236679,20.7285216892983,19.8339725461272,17.3435989423967,16.8995454455618,17.9934386332075,20.2195718163645,22.2070086720384,17.1117076782815,19.8508259681122,22.2506677513959,16.3389366763927,18.5494106764228,18.8171599959616,15.4727616697066,15.9797164726717,19.5551369029075,15.2142427416402,18.195832112632,17.8487816883629,11.980531463383,17.3158688224472,17.3158688224472,19.5482837527865,21.5262447040054,17.1279572175344,19.5799237913336,19.5181096039992],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[62.186587712122,67.1194768172047,59.8873279007658,62.0964206606962,57.7796730736894,68.5696635609685,57.3401086979889,59.8910848612419,59.8760570193376,59.8835709402897,52.9256801386042,59.8723000588615,56.8479468756235,65.0193359110803,52.9482219014606,55.7396435351822,66.0825557258088,53.9475733880958,55.938762440414,61.4690082611922,59.5041179322065,62.9041671630517,53.1398268857403,55.7358865747061,53.3276749095439,66.9128439910207,52.7866726009895,56.1078256618373,62.2692408425955,59.898598782194,62.1001776211723,57.3138099746564,58.072715990823,62.8553266768628,52.2494272529112,56.0815269385048,67.243456512915,53.6620443919143,67.2359425919629,57.3363517375128,56.4534660256359,55.0746615309174,53.0534167947906,65.5077407729697,53.7183987990554,55.1159880961542,53.64701655001,67.239699552439,65.3386775515465,65.624206547728,61.0369578064439,62.7576457044849,66.7625655719778,59.5492014579193,54.950681835207,66.1576949353302,55.2700234756732,61.0745274112046,54.9544387956831,55.8636232308926,56.5661748399181,55.258752594245,65.2222117767882,53.1510977671685,65.5114977334458,58.8842194536546,55.9312485194619,54.8304590999727,58.7639967184202,65.259781381549,56.0852838989808,54.2368593447533,61.2586184745321,62.761402664961,63.4188707482736,65.2259687372643,53.6770722338186,61.4351956169075,55.8523523494643,56.092797819933,52.9895484666974,53.6808291942947,55.9124637170815,64.6473968239492,59.6055558650604,55.9199776380337,55.6870460885172,67.19085906625,55.574337274235,65.312378828214,56.0740130175526,56.1190965432655,53.54182165668,63.4075998668454,61.4314386564315,62.8741114792432,55.5780942347111,66.458251773416,62.8966532420996,65.6091787058237,53.4215989214457,53.4516546052543,65.3424345120225,55.886164993749,61.5216057078572,66.77007949293,55.5104689461418,58.647530943662,53.6845861547707,59.477819208874,61.6493423640436,53.5080090123953,57.343865658465,65.5490673382065,56.096554780409,53.8160797714333,53.4478976447782,59.7445634026751,59.894841821718,53.4328698028739,59.8798139798137,55.0370919261567,66.3831125638945,56.6751266937241,60.6499908774085,59.6017989045843,54.8417299814009,51.9563843357775,53.3013761862114,62.8816254001953,55.8824080332729,56.0777699780287,54.8680287047335,59.5529584183954,53.4253558819218,55.6945600094693,62.9004102025757,62.8778684397192,55.9274915589858,53.1586116881206,65.5152546939219,61.0632565297764,65.4964698915415,55.9162206775576,55.9575472427944,55.0258210447285,53.6883431152468,65.5002268520176,61.5178487473811,53.4178419609696,52.1329614781529,54.1805049376123,56.1040687013612,56.5398761165855,58.3056475403395,61.0857982926328,63.648045337314,55.9011928356533,57.3476226189411,56.6788836542002,56.0139016499354,54.8530008628292,64.4933614444302,61.0594995693003,56.0589851756483,66.1426670934259,55.8561093099404,54.9694666375874,56.6225292470591,53.5267938147757,64.3881665511002,60.5447959840784,62.8327849140064,53.4854672495389,55.1197450566303,65.7744849667708,55.9087067566054,55.1235020171064,60.0601480826651,53.9888999533326,54.9581957561592,52.5950676167098,66.1614518958063,61.7507802968976,60.0826898455216,63.4038429063693,59.2599155012618,65.4513863658286,56.250590159928,61.1308818183457,53.075958557647,59.8497582960051,55.1422868194867,56.6262862075352,52.606338498138,52.2719690157676,56.6187722865831,59.4853331298261,65.5039838124936,59.2561585407857,65.627963508204,56.197992713263,57.1935872394221,67.6379373629026,58.5047664455713,52.2118576481505,53.5230368542996,66.1652088562823,64.5196601677628,53.3427027514482,63.6630731792183,55.6983169699454,55.9350054799379,58.9368169003196,65.5190116543979,56.5060634723009,62.3894635778299,57.3175669351325,57.2912682118,53.2863483443071,65.5716091010629,58.9030042560349],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.9608687575114,15.4402857205865,15.4402857205865,15.4402857205865,14.9828188801384,14.0003046354339,13.6876092253919,13.4359616426754,13.0019078302607,12.9564888616565,12.9564888616565,12.9392327788108,12.5730307540481,12.3798119217609,12.1649623764037,12.1522171726817,11.926678879628,11.926678879628,11.4091593490862,10.807453014016,10.8017021626049,10.7737467191067,10.7737467191067,10.5981920485258,10.4857368348138,9.96716930946532,9.85539506594371,9.42810845999515,9.41631763710079,9.1838563886344,9.09799314022984,8.97989958002444,8.92977578485728,8.87983893756292,8.75485405174813,8.45101138519246,7.73849848352509,7.68630728820265,7.63850534269826,7.53126411662226,7.47417387509558,7.45770632876312,7.34075920805769,7.22398924928789,7.17655982916908,7.02777408811631,6.97710146290046,6.97710146290046,6.96663883428768,6.9457614072373,6.9457614072373,6.94016417503497,6.91969307009962,6.89255412169161,6.85498360983908,6.63094748667006,6.55090355451581,6.46916850844611,6.45306794098528,6.22505344302176,6.13422834048051,6.04874477914872,5.90788093104997,5.7449257534579,5.46525012441609,5.35330551620455,5.3021136946399,5.2150156506708,5.15941785146239,5.10012797597441,5.04149033041833,5.04118067358831,5.04118067358831,4.90807420730955,4.84983813990376,4.84120991386262,4.78742604437307,4.51831991079439,4.51059681383057,4.44130253900756,4.40175803337856,4.39679942647668,4.3538228154926,4.33036142440014,4.31061276014081,4.27197329001315,4.21446997195588,4.11672327483629,4.07181237280945,3.97688363219016,3.93730405977789,3.93219934174513,3.89495819438816,3.89016920724424,3.83349688718972,3.76751292602889,3.75588624510849,3.70981823339445,3.69559121331962,3.61922678142982,3.56040290646305,3.5434119620153,3.53161307807799,3.5296354194724,3.50746173279456,3.46677235732318,3.46677235732318,3.41390357642709,3.40503691367788,3.40503691367788,3.39831508780826,3.35059649253018,3.26122907991455,3.24150891335906,3.2375630981065,3.20485320352922,3.15406249318498,3.14874903041906,3.13815919213729,3.13643715498446,3.08862412784379,3.06344814700793,3.06344814700793,3.04176798176234,2.99845416936534,2.98037878629464,2.96826095482456,2.82187845551813,2.80185503939258,2.77741990285615,2.77661964859526,2.75796604271826,2.74560255013029,2.73769977659898,2.72628027680579,2.71251346772959,2.65777036473291,2.65522162383026,2.60788604554555,2.57742376453728,2.50816547351008,2.49737891393676,2.48705879074707,2.47253186685204,2.46655218028248,2.45570798645999,2.44113446846446,2.43311478654201,2.40457519118189,2.34253459927181,2.33054715281043,2.30410395202475,2.26821785839011,2.24441011642512,2.24324000775732,2.24311078141588,2.23281270232419,2.20103032975683,2.1112467177773,2.07609799968095,2.03491378141223,2.01114397647057,1.9351495406693,1.89114258854775,1.85951678949848,1.83238532334087,1.83238532334087,1.81245973044072,1.81108449781732,1.80576308032382,1.77877499996589,1.76302598638697,1.73543614792687,1.73049541566857,1.71849858411963,1.63623400578119,1.61735625574313,1.59892094973995,1.58209534169529,1.58094267642436,1.57313019884647,1.5608696846441,1.54218477191936,1.51957183751916,1.51339105217247,1.47009455486161,1.46668580111652,1.45790347646445,1.40334216732093,1.37898355489062,1.37898355489062,1.36355835855527,1.33666870854107,1.31904789228505],"text":["GO:0070062 (2178) <br> extracellular exosome <br> 4.926e-49","GO:1903561 (2263) <br> extracellular vesicle <br> 8.225e-47","GO:0043230 (2265) <br> extracellular organelle <br> 9.249e-47","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 9.249e-47","GO:0032991 (5541) <br> protein-containing complex <br> 2.856e-39","GO:1990904 (694) <br> ribonucleoprotein complex <br> 6.435e-38","GO:0031982 (4058) <br> vesicle <br> 4.577e-37","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 1.190e-28","GO:0043227 (13201) <br> membrane-bounded organelle <br> 3.581e-26","GO:0043229 (13117) <br> intracellular organelle <br> 5.662e-25","GO:0005615 (3595) <br> extracellular space <br> 7.967e-25","GO:0043226 (14027) <br> organelle <br> 3.891e-22","GO:0031410 (2433) <br> cytoplasmic vesicle <br> 5.921e-22","GO:0097708 (2438) <br> intracellular vesicle <br> 7.183e-22","GO:0005622 (14808) <br> intracellular anatomical structure <br> 4.424e-21","GO:0030055 (427) <br> cell-substrate junction <br> 2.582e-20","GO:0099503 (1027) <br> secretory vesicle <br> 3.313e-20","GO:0005925 (420) <br> focal adhesion <br> 1.069e-19","GO:0030141 (857) <br> secretory granule <br> 1.222e-19","GO:0048770 (109) <br> pigment granule <br> 1.082e-18","GO:0042470 (109) <br> melanosome <br> 1.082e-18","GO:0071013 (85) <br> catalytic step 2 spliceosome <br> 1.970e-18","GO:0005681 (192) <br> spliceosomal complex <br> 2.867e-18","GO:0030054 (2107) <br> cell junction <br> 4.560e-18","GO:0005737 (11951) <br> cytoplasm <br> 8.886e-18","GO:1902494 (1377) <br> catalytic complex <br> 2.606e-17","GO:0005576 (4564) <br> extracellular region <br> 5.269e-17","GO:0030667 (308) <br> secretory granule membrane <br> 7.006e-17","GO:0070161 (838) <br> anchoring junction <br> 1.094e-16","GO:0043233 (5579) <br> organelle lumen <br> 3.628e-16","GO:0070013 (5579) <br> intracellular organelle lumen <br> 3.628e-16","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 3.628e-16","GO:0033290 (15) <br> eukaryotic 48S preinitiation complex <br> 1.040e-15","GO:0070993 (18) <br> translation preinitiation complex <br> 9.993e-15","GO:0000502 (67) <br> proteasome complex <br> 2.053e-14","GO:0030659 (789) <br> cytoplasmic vesicle membrane <br> 3.665e-14","GO:1905369 (73) <br> endopeptidase complex <br> 9.956e-14","GO:0005834 (36) <br> heterotrimeric G-protein complex <br> 1.105e-13","GO:1905360 (36) <br> GTPase complex <br> 1.105e-13","GO:0031981 (4453) <br> nuclear lumen <br> 1.150e-13","GO:0031090 (3644) <br> organelle membrane <br> 2.673e-13","GO:0016020 (9829) <br> membrane <br> 4.170e-13","GO:0005654 (4112) <br> nucleoplasm <br> 6.840e-13","GO:0098796 (1326) <br> membrane protein complex <br> 7.043e-13","GO:0005852 (17) <br> eukaryotic translation initiation factor 3 complex <br> 1.184e-12","GO:0016282 (17) <br> eukaryotic 43S preinitiation complex <br> 1.184e-12","GO:0005829 (5361) <br> cytosol <br> 3.898e-12","GO:1905368 (97) <br> peptidase complex <br> 1.558e-11","GO:0098588 (2165) <br> bounding membrane of organelle <br> 1.579e-11","GO:0098857 (339) <br> membrane microdomain <br> 1.684e-11","GO:0045121 (339) <br> membrane raft <br> 1.684e-11","GO:0070820 (163) <br> tertiary granule <br> 2.522e-11","GO:0140513 (1236) <br> nuclear protein-containing complex <br> 3.268e-11","GO:0042581 (158) <br> specific granule <br> 1.079e-10","GO:0012505 (4651) <br> endomembrane system <br> 1.395e-10","GO:0101002 (184) <br> ficolin-1-rich granule <br> 3.732e-10","GO:0016607 (413) <br> nuclear speck <br> 3.834e-10","GO:0045202 (1348) <br> synapse <br> 6.549e-10","GO:0012506 (1183) <br> vesicle membrane <br> 7.980e-10","GO:0030120 (55) <br> vesicle coat <br> 1.047e-09","GO:0031234 (103) <br> extrinsic component of cytoplasmic side of plasma membrane <br> 1.176e-09","GO:0016604 (816) <br> nuclear body <br> 1.319e-09","GO:0098552 (622) <br> side of membrane <br> 1.759e-09","GO:0005684 (93) <br> U2-type spliceosomal complex <br> 3.540e-09","GO:0098797 (696) <br> plasma membrane protein complex <br> 1.826e-08","GO:0035579 (90) <br> specific granule membrane <br> 2.059e-08","GO:0030139 (335) <br> endocytic vesicle <br> 2.299e-08","GO:0009898 (169) <br> cytoplasmic side of plasma membrane <br> 2.943e-08","GO:0035145 (20) <br> exon-exon junction complex <br> 3.356e-08","GO:0098562 (195) <br> cytoplasmic side of membrane <br> 3.486e-08","GO:0030660 (52) <br> Golgi-associated vesicle membrane <br> 4.563e-08","GO:0008541 (8) <br> proteasome regulatory particle, lid subcomplex <br> 5.971e-08","GO:0045335 (139) <br> phagocytic vesicle <br> 6.659e-08","GO:0070821 (72) <br> tertiary granule membrane <br> 9.380e-08","GO:0071556 (25) <br> integral component of lumenal side of endoplasmic reticulum membrane <br> 1.054e-07","GO:0098553 (25) <br> lumenal side of endoplasmic reticulum membrane <br> 1.054e-07","GO:0005838 (22) <br> proteasome regulatory particle <br> 1.080e-07","GO:0048475 (97) <br> coated membrane <br> 1.133e-07","GO:0030117 (97) <br> membrane coat <br> 1.133e-07","GO:0030662 (177) <br> coated vesicle membrane <br> 1.148e-07","GO:0005634 (7607) <br> nucleus <br> 1.203e-07","GO:0005839 (23) <br> proteasome core complex <br> 1.281e-07","GO:0030133 (416) <br> transport vesicle <br> 1.396e-07","GO:0097525 (61) <br> spliceosomal snRNP complex <br> 2.339e-07","GO:0042612 (8) <br> MHC class I protein complex <br> 2.813e-07","GO:0030135 (296) <br> coated vesicle <br> 3.395e-07","GO:0022624 (25) <br> proteasome accessory complex <br> 3.523e-07","GO:1904813 (124) <br> ficolin-1-rich granule lumen <br> 5.956e-07","GO:0019897 (178) <br> extrinsic component of plasma membrane <br> 7.341e-07","GO:0098576 (33) <br> lumenal side of membrane <br> 8.938e-07","GO:0030532 (70) <br> small nuclear ribonucleoprotein complex <br> 1.236e-06","GO:0030670 (75) <br> phagocytic vesicle membrane <br> 1.799e-06","GO:0005798 (95) <br> Golgi-associated vesicle <br> 3.426e-06","GO:0071541 (7) <br> eukaryotic translation initiation factor 3 complex, eIF3m <br> 4.433e-06","GO:0048471 (745) <br> perinuclear region of cytoplasm <br> 4.988e-06","GO:0071005 (50) <br> U2-type precatalytic spliceosome <br> 6.095e-06","GO:0019898 (315) <br> extrinsic component of membrane <br> 6.928e-06","GO:0120114 (82) <br> Sm-like protein family complex <br> 7.941e-06","GO:0071011 (52) <br> precatalytic spliceosome <br> 9.089e-06","GO:0098852 (395) <br> lytic vacuole membrane <br> 9.095e-06","GO:0005765 (395) <br> lysosomal membrane <br> 9.095e-06","GO:0005774 (450) <br> vacuolar membrane <br> 1.236e-05","GO:0098590 (1240) <br> plasma membrane region <br> 1.413e-05","GO:0030126 (13) <br> COPI vesicle coat <br> 1.441e-05","GO:0055038 (85) <br> recycling endosome membrane <br> 1.631e-05","GO:0140534 (128) <br> endoplasmic reticulum protein-containing complex <br> 3.032e-05","GO:0019773 (10) <br> proteasome core complex, alpha-subunit complex <br> 3.086e-05","GO:0034774 (320) <br> secretory granule lumen <br> 3.620e-05","GO:0005840 (247) <br> ribosome <br> 3.965e-05","GO:0042175 (1154) <br> nuclear outer membrane-endoplasmic reticulum membrane network <br> 4.011e-05","GO:0060205 (324) <br> cytoplasmic vesicle lumen <br> 4.428e-05","GO:0005789 (1132) <br> endoplasmic reticulum membrane <br> 4.673e-05","GO:0031983 (326) <br> vesicle lumen <br> 4.891e-05","GO:0098827 (1137) <br> endoplasmic reticulum subcompartment <br> 5.346e-05","GO:0030663 (18) <br> COPI-coated vesicle membrane <br> 6.103e-05","GO:0005886 (5680) <br> plasma membrane <br> 7.643e-05","GO:0005773 (818) <br> vacuole <br> 8.476e-05","GO:0042995 (2337) <br> cell projection <br> 1.055e-04","GO:0043232 (5094) <br> intracellular non-membrane-bounded organelle <br> 1.155e-04","GO:0005768 (993) <br> endosome <br> 1.169e-04","GO:0043228 (5102) <br> non-membrane-bounded organelle <br> 1.274e-04","GO:0015935 (76) <br> small ribosomal subunit <br> 1.288e-04","GO:0120025 (2234) <br> plasma membrane bounded cell projection <br> 1.467e-04","GO:0031300 (419) <br> intrinsic component of organelle membrane <br> 1.708e-04","GO:0044391 (191) <br> ribosomal subunit <br> 1.754e-04","GO:0042611 (24) <br> MHC protein complex <br> 1.951e-04","GO:0009925 (250) <br> basal plasma membrane <br> 2.016e-04","GO:0000139 (725) <br> Golgi membrane <br> 2.403e-04","GO:0005730 (955) <br> nucleolus <br> 2.752e-04","GO:0071007 (30) <br> U2-type catalytic step 2 spliceosome <br> 2.861e-04","GO:0030125 (28) <br> clathrin vesicle coat <br> 2.940e-04","GO:0030658 (210) <br> transport vesicle membrane <br> 2.954e-04","GO:0010008 (520) <br> endosome membrane <br> 3.108e-04","GO:0042582 (154) <br> azurophil granule <br> 3.414e-04","GO:0005766 (154) <br> primary lysosome <br> 3.414e-04","GO:0022626 (106) <br> cytosolic ribosome <br> 3.856e-04","GO:0071012 (12) <br> catalytic step 1 spliceosome <br> 3.935e-04","GO:0071006 (12) <br> U2-type catalytic step 1 spliceosome <br> 3.935e-04","GO:0030137 (28) <br> COPI-coated vesicle <br> 3.997e-04","GO:0005686 (21) <br> U2 snRNP <br> 4.461e-04","GO:0098798 (272) <br> mitochondrial protein-containing complex <br> 5.480e-04","GO:0045178 (268) <br> basal part of cell <br> 5.734e-04","GO:0098791 (845) <br> Golgi apparatus subcompartment <br> 5.787e-04","GO:0030134 (90) <br> COPII-coated ER to Golgi transport vesicle <br> 6.239e-04","GO:0030176 (161) <br> integral component of endoplasmic reticulum membrane <br> 7.014e-04","GO:0015629 (517) <br> actin cytoskeleton <br> 7.100e-04","GO:0005844 (72) <br> polysome <br> 7.275e-04","GO:0098793 (515) <br> presynapse <br> 7.304e-04","GO:0055037 (187) <br> recycling endosome <br> 8.154e-04","GO:0005764 (723) <br> lysosome <br> 8.641e-04","GO:0000323 (723) <br> lytic vacuole <br> 8.641e-04","GO:0008250 (13) <br> oligosaccharyltransferase complex <br> 9.083e-04","GO:0030666 (190) <br> endocytic vesicle membrane <br> 1.004e-03","GO:0031227 (169) <br> intrinsic component of endoplasmic reticulum membrane <br> 1.046e-03","GO:0034399 (132) <br> nuclear periphery <br> 1.076e-03","GO:0045239 (14) <br> tricarboxylic acid cycle enzyme complex <br> 1.507e-03","GO:0071944 (6178) <br> cell periphery <br> 1.578e-03","GO:0030130 (13) <br> clathrin coat of trans-Golgi network vesicle <br> 1.669e-03","GO:0031984 (1764) <br> organelle subcompartment <br> 1.673e-03","GO:0031301 (389) <br> integral component of organelle membrane <br> 1.746e-03","GO:0030425 (641) <br> dendrite <br> 1.796e-03","GO:0009986 (896) <br> cell surface <br> 1.829e-03","GO:0097447 (643) <br> dendritic tree <br> 1.878e-03","GO:0045177 (433) <br> apical part of cell <br> 1.939e-03","GO:0030496 (197) <br> midbody <br> 2.199e-03","GO:0099631 (2) <br> postsynaptic endocytic zone cytoplasmic component <br> 2.212e-03","GO:0030118 (47) <br> clathrin coat <br> 2.467e-03","GO:0012510 (15) <br> trans-Golgi network transport vesicle membrane <br> 2.646e-03","GO:0031253 (346) <br> cell projection membrane <br> 3.103e-03","GO:0005794 (1593) <br> Golgi apparatus <br> 3.181e-03","GO:0097386 (34) <br> glial cell projection <br> 3.258e-03","GO:0044297 (595) <br> cell body <br> 3.369e-03","GO:0070937 (6) <br> CRD-mediated mRNA stability complex <br> 3.415e-03","GO:0005783 (1995) <br> endoplasmic reticulum <br> 3.502e-03","GO:0016323 (225) <br> basolateral plasma membrane <br> 3.621e-03","GO:0098978 (354) <br> glutamatergic synapse <br> 3.689e-03","GO:0030132 (17) <br> clathrin coat of coated pit <br> 3.939e-03","GO:0016324 (364) <br> apical plasma membrane <br> 4.544e-03","GO:0043596 (35) <br> nuclear replication fork <br> 4.671e-03","GO:0005938 (326) <br> cell cortex <br> 4.965e-03","GO:0012507 (57) <br> ER to Golgi transport vesicle membrane <br> 5.392e-03","GO:0001772 (44) <br> immunological synapse <br> 5.696e-03","GO:0101003 (60) <br> ficolin-1-rich granule membrane <br> 5.712e-03","GO:0061574 (4) <br> ASAP complex <br> 5.713e-03","GO:0043614 (2) <br> multi-eIF complex <br> 5.850e-03","GO:0071540 (2) <br> eukaryotic translation initiation factor 3 complex, eIF3e <br> 6.295e-03","GO:0036477 (875) <br> somatodendritic compartment <br> 7.740e-03","GO:0098687 (348) <br> chromosomal region <br> 8.393e-03","GO:0030894 (24) <br> replisome <br> 9.228e-03","GO:0045252 (6) <br> oxoglutarate dehydrogenase complex <br> 9.747e-03","GO:0005662 (13) <br> DNA replication factor A complex <br> 1.161e-02","GO:0043209 (49) <br> myelin sheath <br> 1.285e-02","GO:0016363 (110) <br> nuclear matrix <br> 1.382e-02","GO:0031254 (13) <br> cell trailing edge <br> 1.471e-02","GO:0001931 (13) <br> uropod <br> 1.471e-02","GO:0000781 (160) <br> chromosome, telomeric region <br> 1.540e-02","GO:0031252 (426) <br> cell leading edge <br> 1.545e-02","GO:0042382 (5) <br> paraspeckles <br> 1.564e-02","GO:0098794 (641) <br> postsynapse <br> 1.664e-02","GO:0036464 (238) <br> cytoplasmic ribonucleoprotein granule <br> 1.726e-02","GO:0098858 (217) <br> actin-based cell projection <br> 1.839e-02","GO:0030863 (121) <br> cortical cytoskeleton <br> 1.860e-02","GO:0031680 (12) <br> G-protein beta/gamma-subunit complex <br> 1.912e-02","GO:1990204 (114) <br> oxidoreductase complex <br> 2.311e-02","GO:0034687 (2) <br> integrin alphaL-beta2 complex <br> 2.413e-02","GO:0000421 (41) <br> autophagosome membrane <br> 2.518e-02","GO:0005793 (133) <br> endoplasmic reticulum-Golgi intermediate compartment <br> 2.618e-02","GO:0101004 (2) <br> cytolytic granule membrane <br> 2.625e-02","GO:0097454 (3) <br> Schwann cell microvillus <br> 2.672e-02","GO:0005742 (23) <br> mitochondrial outer membrane translocase complex <br> 2.749e-02","GO:0071986 (10) <br> Ragulator complex <br> 2.870e-02","GO:0022627 (46) <br> cytosolic small ribosomal subunit <br> 3.023e-02","GO:0030140 (33) <br> trans-Golgi network transport vesicle <br> 3.066e-02","GO:0035770 (252) <br> ribonucleoprotein granule <br> 3.388e-02","GO:0098799 (24) <br> outer mitochondrial membrane protein complex <br> 3.414e-02","GO:0031201 (52) <br> SNARE complex <br> 3.484e-02","GO:0070382 (228) <br> exocytic vesicle <br> 3.951e-02","GO:0031975 (1244) <br> envelope <br> 4.178e-02","GO:0031967 (1244) <br> organelle envelope <br> 4.178e-02","GO:0005726 (5) <br> perichromatin fibrils <br> 4.330e-02","GO:0098835 (7) <br> presynaptic endocytic zone membrane <br> 4.606e-02","GO:0035632 (2) <br> mitochondrial prohibitin complex <br> 4.797e-02"],"key":["GO:0070062","GO:1903561","GO:0043230","GO:0065010","GO:0032991","GO:1990904","GO:0031982","GO:0043231","GO:0043227","GO:0043229","GO:0005615","GO:0043226","GO:0031410","GO:0097708","GO:0005622","GO:0030055","GO:0099503","GO:0005925","GO:0030141","GO:0048770","GO:0042470","GO:0071013","GO:0005681","GO:0030054","GO:0005737","GO:1902494","GO:0005576","GO:0030667","GO:0070161","GO:0043233","GO:0070013","GO:0031974","GO:0033290","GO:0070993","GO:0000502","GO:0030659","GO:1905369","GO:0005834","GO:1905360","GO:0031981","GO:0031090","GO:0016020","GO:0005654","GO:0098796","GO:0005852","GO:0016282","GO:0005829","GO:1905368","GO:0098588","GO:0098857","GO:0045121","GO:0070820","GO:0140513","GO:0042581","GO:0012505","GO:0101002","GO:0016607","GO:0045202","GO:0012506","GO:0030120","GO:0031234","GO:0016604","GO:0098552","GO:0005684","GO:0098797","GO:0035579","GO:0030139","GO:0009898","GO:0035145","GO:0098562","GO:0030660","GO:0008541","GO:0045335","GO:0070821","GO:0071556","GO:0098553","GO:0005838","GO:0048475","GO:0030117","GO:0030662","GO:0005634","GO:0005839","GO:0030133","GO:0097525","GO:0042612","GO:0030135","GO:0022624","GO:1904813","GO:0019897","GO:0098576","GO:0030532","GO:0030670","GO:0005798","GO:0071541","GO:0048471","GO:0071005","GO:0019898","GO:0120114","GO:0071011","GO:0098852","GO:0005765","GO:0005774","GO:0098590","GO:0030126","GO:0055038","GO:0140534","GO:0019773","GO:0034774","GO:0005840","GO:0042175","GO:0060205","GO:0005789","GO:0031983","GO:0098827","GO:0030663","GO:0005886","GO:0005773","GO:0042995","GO:0043232","GO:0005768","GO:0043228","GO:0015935","GO:0120025","GO:0031300","GO:0044391","GO:0042611","GO:0009925","GO:0000139","GO:0005730","GO:0071007","GO:0030125","GO:0030658","GO:0010008","GO:0042582","GO:0005766","GO:0022626","GO:0071012","GO:0071006","GO:0030137","GO:0005686","GO:0098798","GO:0045178","GO:0098791","GO:0030134","GO:0030176","GO:0015629","GO:0005844","GO:0098793","GO:0055037","GO:0005764","GO:0000323","GO:0008250","GO:0030666","GO:0031227","GO:0034399","GO:0045239","GO:0071944","GO:0030130","GO:0031984","GO:0031301","GO:0030425","GO:0009986","GO:0097447","GO:0045177","GO:0030496","GO:0099631","GO:0030118","GO:0012510","GO:0031253","GO:0005794","GO:0097386","GO:0044297","GO:0070937","GO:0005783","GO:0016323","GO:0098978","GO:0030132","GO:0016324","GO:0043596","GO:0005938","GO:0012507","GO:0001772","GO:0101003","GO:0061574","GO:0043614","GO:0071540","GO:0036477","GO:0098687","GO:0030894","GO:0045252","GO:0005662","GO:0043209","GO:0016363","GO:0031254","GO:0001931","GO:0000781","GO:0031252","GO:0042382","GO:0098794","GO:0036464","GO:0098858","GO:0030863","GO:0031680","GO:1990204","GO:0034687","GO:0000421","GO:0005793","GO:0101004","GO:0097454","GO:0005742","GO:0071986","GO:0022627","GO:0030140","GO:0035770","GO:0098799","GO:0031201","GO:0070382","GO:0031975","GO:0031967","GO:0005726","GO:0098835","GO:0035632"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[20.3895089474417,20.43491384342,20.4359600780238,20.4359600780238,21.4637267954854,18.9706470582724,21.1127924137548,22.3097189832527,22.4069336180039,22.4001723500672,20.97436852479,22.4710947059109,20.520482671709,20.522901551273,22.5281903038104,18.3262982994618,19.4713253358183,18.3038663648759,19.2421674566737,16.3389366763927,16.3389366763927,15.9420548318458,17.1987173630513,20.3501024869214,22.301288321845,19.835758359278,21.2460124471743,17.8763374918434,19.2135392269153,21.4713489077623,21.4713489077623,21.4713489077623,12.6954028668369,13.0900518224006,15.5497876137808,19.1363383178836,15.692660304218,14.4581139285914,14.4581139285914,21.2181834681373,20.9898925305246,22.0919153891785,21.1278301730825,19.7893245482402,12.968152232712,12.968152232712,21.4268505016346,16.1543870489597,20.3823975905089,18.0099018230755,18.0099018230755,16.9558134205769,19.7024866025059,16.9090881701731,21.2673252341618,17.1360081782449,18.2810218264132,19.8095866429318,19.6480987518461,15.2142427416402,16.2496921162547,19.17949942902,18.8277438199372,16.0870759222918,18.9743833960756,16.0344085591459,17.9934386332075,17.0098206791988,13.3106655321065,17.2214892061892,15.1170689685676,11.1749781597015,16.715163312453,15.6697988614665,13.7618192670139,13.7618192670139,13.5059251363754,16.1543870489597,16.1543870489597,17.0786016380891,21.8137488730278,13.5956642813554,18.2908638032579,15.3914887766185,11.1749781597015,17.8206171716069,13.7618192670139,16.5398373229612,17.0869555296552,14.2961523059962,15.6229705518591,15.7373316539529,16.121131665261,10.8097872173079,19.0624517869717,15.0486221338682,17.90774964072,15.8836369590203,15.1170689685676,18.2202776543847,18.2202776543847,18.3972686497445,19.7064888615817,12.3729643352122,15.9420548318458,16.5888237442405,11.747980750738,17.9297208522418,17.5640277699944,19.6172156871556,17.9470280058521,19.5932234371659,17.9555937940104,19.5987201263685,13.0900518224006,21.4913424228551,19.1826347921651,20.4729795999535,21.3696524759062,19.4289457661119,21.3714121953266,15.7591617451593,20.4196311626559,18.3006284598251,17.1910389050914,13.6808214449962,17.5812933447843,19.0272871005367,19.3796840599657,14.1158314099108,13.9833131530912,17.3298070009327,18.5910918340989,16.8705055778228,16.8705055778228,16.2950254596216,12.187219027138,12.187219027138,13.9833131530912,13.4111152730112,17.7013163294157,17.6803085206584,19.2241674808196,16.0344085591459,16.9373174877114,18.5833840485582,15.6697988614665,18.5782183914119,17.1598725929261,19.0237128877517,19.0237128877517,12.3729643352122,17.1833157067251,17.0098206791988,16.6361245292714,12.5414277899272,21.5846981879027,12.3729643352122,20.1372168262823,18.1993537615243,18.8671354327216,19.2988369642948,18.8712079238796,18.3452080104898,17.2364555998064,3.77952755905512,14.9397854919929,12.6954028668369,18.0382058611615,20.0137886576694,14.3519997480295,18.7694563553214,10.3630624020686,20.2849514736126,17.429875898974,18.0697940419775,12.968152232712,18.1081960550518,14.405947329161,17.9555937940104,15.2756954716879,14.8225866744181,15.3633750239743,9.00890082870231,3.77952755905512,3.77952755905512,19.2686621270864,18.0461774501309,13.6808214449962,10.3630624020686,12.3729643352122,15.0132022257977,16.3532722717548,12.3729643352122,12.3729643352122,16.9279733014706,18.3231184550737,9.79200637188604,18.8671354327216,17.5108080471375,17.3774579655542,16.5019202141693,12.187219027138,16.4091839862407,3.77952755905512,14.6957136866834,16.6476991969387,3.77952755905512,7.7791008157754,13.5956642813554,11.747980750738,14.9017074238205,14.2961523059962,17.5926771820558,13.6808214449962,15.1170689685676,17.4490032718266,19.710477230295,19.710477230295,9.79200637188604,10.8097872173079,3.77952755905512],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[3.64155719716214,9.05456388162587,40.706190754139,28.7757887903244,46.8817514684057,3.68663428266774,19.3434086482783,4.24258500390343,9.48279619392904,16.9543231164817,28.8997507754648,9.0921281195472,18.7010601798235,40.341817646302,3.49505666926895,18.6935473322393,22.3297655630241,15.3202787669038,16.2143076294315,16.2105512056393,4.24634142769556,16.1128841870439,25.8194832659157,21.169030611255,40.8301527392793,25.9960351841459,3.65658289233068,27.0515902697353,22.3072270202713,16.8679253692626,22.299714172687,46.8704821970293,3.64907004474641,3.66033931612281,27.4460147679093,2.12771840893252,36.1083280325681,3.68287785887561,33.45253641153,28.9448278609704,8.35962548008125,25.5790721432192,6.37247729404287,25.5828285670113,26.8149355708309,13.5885673987305,6.36120802266647,3.75424991092613,8.99070467715961,27.3633734444824,38.0353734379323,3.64531362095428,3.66409573991494,27.1229623217858,40.9766532671725,20.4853614810867,3.57018514511161,40.9916789623411,9.35507778499651,21.0075043881932,21.6536092804401,24.1178232880794,27.4009376824037,29.669817652852,5.44839704117814,2.11269271376399,19.9932699643173,16.4659880235044,38.985748657342,25.9547145224325,27.0403209983589,41.1907694233241,19.3509214958626,38.9556972670049,8.02906018637355,38.392233698185,24.3845293773208,21.0826328640359,8.36713832766552,27.3746427158588,15.5381513468475],"y":[17,17,14.9854470006897,14.1844785242178,14.0500314622869,13.5086973893557,11.6719251484468,11.4385734607567,10.5629318969608,10.4247666845153,9.57541548071352,9.46381682263775,9.45554306222099,9.4118127556425,9.32632559465899,8.85194501642636,8.85194501642636,8.84297434201324,8.57089422398686,8.54655726555218,7.91832026684322,6.61213671785816,6.09603546008205,5.95667580409696,5.95549525576473,5.9491642449523,5.60382065423814,5.40368236148785,5.06047171975328,4.93426790866412,4.91757265168375,4.85244017364239,4.62919463504748,4.57435670910052,4.52995029349091,4.4928359594338,4.33854541765465,4.11316903096791,3.97951029971181,3.90330798494355,3.61339772955454,3.51984839063301,3.47075556516036,3.44470375953168,3.34860927966468,3.27153295490034,3.00034216618782,2.917374611736,2.62153577003718,2.60587690767283,2.5897641606528,2.57347643858702,2.5586498991658,2.55820657680673,2.46198916786945,2.45753174270527,2.39651346445988,2.24228764724179,2.13653952270677,2.08296305088956,1.99160716741619,1.93475146085485,1.88068804228393,1.85679838877133,1.8211596051161,1.79229643977144,1.7080217658096,1.69445848452906,1.55056434955812,1.54816289927296,1.5395795524803,1.45632007433163,1.42667917203323,1.42632213994889,1.41945255215951,1.40000960974182,1.38804752851019,1.35041276261047,1.34262762679199,1.34051729623362,1.33349765886188],"text":["GO:0003723 (1951) <br> RNA binding <br> 2.961e-38","GO:0005515 (14767) <br> protein binding <br> 7.112e-20","GO:0097159 (6291) <br> organic cyclic compound binding <br> 1.034e-15","GO:0044877 (1304) <br> protein-containing complex binding <br> 6.539e-15","GO:1901363 (6210) <br> heterocyclic compound binding <br> 8.912e-15","GO:0003743 (58) <br> translation initiation factor activity <br> 3.100e-14","GO:0019899 (2048) <br> enzyme binding <br> 2.129e-12","GO:0003924 (300) <br> GTPase activity <br> 3.643e-12","GO:0008135 (99) <br> translation factor activity, RNA binding <br> 2.736e-11","GO:0017111 (424) <br> nucleoside-triphosphatase activity <br> 3.760e-11","GO:0045182 (154) <br> translation regulator activity <br> 2.658e-10","GO:0005525 (381) <br> GTP binding <br> 3.437e-10","GO:0019003 (73) <br> GDP binding <br> 3.503e-10","GO:0090079 (123) <br> translation regulator activity, nucleic acid binding <br> 3.874e-10","GO:0003676 (4252) <br> nucleic acid binding <br> 4.717e-10","GO:0019001 (402) <br> guanyl nucleotide binding <br> 1.406e-09","GO:0032561 (402) <br> guanyl ribonucleotide binding <br> 1.406e-09","GO:0016462 (482) <br> pyrophosphatase activity <br> 1.436e-09","GO:0016818 (493) <br> hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides <br> 2.686e-09","GO:0016817 (494) <br> hydrolase activity, acting on acid anhydrides <br> 2.841e-09","GO:0003925 (32) <br> G protein activity <br> 1.207e-08","GO:0016787 (2357) <br> hydrolase activity <br> 2.443e-07","GO:0035639 (1846) <br> purine ribonucleoside triphosphate binding <br> 8.016e-07","GO:0031683 (20) <br> G-protein beta/gamma-subunit complex binding <br> 1.105e-06","GO:0097367 (2282) <br> carbohydrate derivative binding <br> 1.108e-06","GO:0036094 (2516) <br> small molecule binding <br> 1.124e-06","GO:0003727 (91) <br> single-stranded RNA binding <br> 2.490e-06","GO:0042610 (4) <br> CD8 receptor binding <br> 3.947e-06","GO:0032555 (1917) <br> purine ribonucleotide binding <br> 8.700e-06","GO:0017076 (1932) <br> purine nucleotide binding <br> 1.163e-05","GO:0032553 (1934) <br> ribonucleotide binding <br> 1.209e-05","GO:1901265 (2177) <br> nucleoside phosphate binding <br> 1.405e-05","GO:0003725 (78) <br> double-stranded RNA binding <br> 2.349e-05","GO:0003729 (553) <br> mRNA binding <br> 2.665e-05","GO:0043168 (2413) <br> anion binding <br> 2.952e-05","GO:0000166 (2176) <br> nucleotide binding <br> 3.215e-05","GO:0050839 (544) <br> cell adhesion molecule binding <br> 4.586e-05","GO:0003735 (171) <br> structural constituent of ribosome <br> 7.706e-05","GO:0048027 (27) <br> mRNA 5'-UTR binding <br> 1.048e-04","GO:0045296 (333) <br> cadherin binding <br> 1.249e-04","GO:0005198 (701) <br> structural molecule activity <br> 2.436e-04","GO:0035254 (48) <br> glutamate receptor binding <br> 3.021e-04","GO:0004579 (5) <br> dolichyl-diphosphooligosaccharide-protein glycotransferase activity <br> 3.383e-04","GO:0035255 (27) <br> ionotropic glutamate receptor binding <br> 3.592e-04","GO:0042162 (36) <br> telomeric DNA binding <br> 4.481e-04","GO:0015288 (5) <br> porin activity <br> 5.351e-04","GO:0004576 (6) <br> oligosaccharyl transferase activity <br> 9.992e-04","GO:0003779 (449) <br> actin binding <br> 1.210e-03","GO:0005488 (17070) <br> binding <br> 2.390e-03","GO:0043008 (8) <br> ATP-dependent protein binding <br> 2.478e-03","GO:0060090 (341) <br> molecular adaptor activity <br> 2.572e-03","GO:0003724 (76) <br> RNA helicase activity <br> 2.670e-03","GO:0003730 (192) <br> mRNA 3'-UTR binding <br> 2.763e-03","GO:0042802 (2063) <br> identical protein binding <br> 2.766e-03","GO:0097718 (34) <br> disordered domain specific binding <br> 3.452e-03","GO:0030619 (8) <br> U1 snRNA binding <br> 3.487e-03","GO:0003697 (121) <br> single-stranded DNA binding <br> 4.013e-03","GO:0098505 (10) <br> G-rich strand telomeric DNA binding <br> 5.724e-03","GO:0008092 (1001) <br> cytoskeletal protein binding <br> 7.302e-03","GO:0031369 (34) <br> translation initiation factor binding <br> 8.261e-03","GO:0031821 (9) <br> G protein-coupled serotonin receptor binding <br> 1.020e-02","GO:0034057 (2) <br> RNA strand-exchange activity <br> 1.162e-02","GO:0043047 (12) <br> single-stranded telomeric DNA binding <br> 1.316e-02","GO:0046977 (9) <br> TAP binding <br> 1.391e-02","GO:0004298 (14) <br> threonine-type endopeptidase activity <br> 1.510e-02","GO:0000149 (113) <br> SNARE binding <br> 1.613e-02","GO:0030276 (70) <br> clathrin binding <br> 1.959e-02","GO:0016887 (553) <br> ATPase <br> 2.021e-02","GO:0070182 (18) <br> DNA polymerase binding <br> 2.815e-02","GO:0036002 (37) <br> pre-mRNA binding <br> 2.830e-02","GO:0042605 (30) <br> peptide antigen binding <br> 2.887e-02","GO:0098847 (15) <br> sequence-specific single stranded DNA binding <br> 3.497e-02","GO:0019901 (677) <br> protein kinase binding <br> 3.744e-02","GO:0070122 (17) <br> isopeptidase activity <br> 3.747e-02","GO:0005102 (1551) <br> signaling receptor binding <br> 3.807e-02","GO:0061752 (2) <br> telomeric repeat-containing RNA binding <br> 3.981e-02","GO:0034458 (4) <br> 3'-5' RNA helicase activity <br> 4.092e-02","GO:0031489 (17) <br> myosin V binding <br> 4.463e-02","GO:0005200 (104) <br> structural constituent of cytoskeleton <br> 4.543e-02","GO:0043021 (150) <br> ribonucleoprotein complex binding <br> 4.565e-02","GO:0016620 (38) <br> oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor <br> 4.640e-02"],"key":["GO:0003723","GO:0005515","GO:0097159","GO:0044877","GO:1901363","GO:0003743","GO:0019899","GO:0003924","GO:0008135","GO:0017111","GO:0045182","GO:0005525","GO:0019003","GO:0090079","GO:0003676","GO:0019001","GO:0032561","GO:0016462","GO:0016818","GO:0016817","GO:0003925","GO:0016787","GO:0035639","GO:0031683","GO:0097367","GO:0036094","GO:0003727","GO:0042610","GO:0032555","GO:0017076","GO:0032553","GO:1901265","GO:0003725","GO:0003729","GO:0043168","GO:0000166","GO:0050839","GO:0003735","GO:0048027","GO:0045296","GO:0005198","GO:0035254","GO:0004579","GO:0035255","GO:0042162","GO:0015288","GO:0004576","GO:0003779","GO:0005488","GO:0043008","GO:0060090","GO:0003724","GO:0003730","GO:0042802","GO:0097718","GO:0030619","GO:0003697","GO:0098505","GO:0008092","GO:0031369","GO:0031821","GO:0034057","GO:0043047","GO:0046977","GO:0004298","GO:0000149","GO:0030276","GO:0016887","GO:0070182","GO:0036002","GO:0042605","GO:0098847","GO:0019901","GO:0070122","GO:0005102","GO:0061752","GO:0034458","GO:0031489","GO:0005200","GO:0043021","GO:0016620"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[20.258276019327,22.525272898967,21.6047679255868,19.7686971273735,21.5904209757188,15.3054992444299,20.3162578263421,17.8394624222953,16.1868772799724,18.3167342125249,16.8705055778228,18.1708988395135,15.692660304218,16.5273136649716,21.1658574902378,18.2442535356626,18.2442535356626,18.4896805188233,18.5199090563365,18.5226205472543,14.2382740983495,20.4830462803831,20.1919193878134,13.3106655321065,20.4448133217894,20.5599632179123,16.0521821733416,9.00890082870231,20.2372173548861,20.2465569336091,20.2477963305085,20.3889635367376,15.801858092204,18.6728117067331,20.5107536570554,20.38841785752,18.6510623608111,17.0273503124218,13.9127623976552,17.9851259338563,18.9836734866874,14.976936429373,9.79200637188604,13.9127623976552,14.4581139285914,9.79200637188604,10.3630624020686,18.394266179927,22.6771653543307,11.1749781597015,18.0180536596155,15.7591617451593,17.1987173630513,20.324960558726,14.3519997480295,11.1749781597015,16.5019202141693,11.747980750738,19.4390569617437,14.3519997480295,11.4827422948031,3.77952755905512,12.187219027138,11.4827422948031,12.5414277899272,16.3954151879507,15.6229705518591,18.6728117067331,13.0900518224006,14.5086078145015,14.1158314099108,12.6954028668369,18.9384082597841,12.968152232712,19.9812897468844,3.77952755905512,9.00890082870231,12.968152232712,16.2649666785139,16.8307889373762,14.5575278577922],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[195.411955705664,199.83466363138,199.830906020568,191.301129477343,196.734634711485,193.394118699624,197.508702538756,197.486156873884,198.838896766201,198.399256301198,192.31192678577,195.949294051779,197.358398106276,196.17850831131,193.608302515907,197.813069014527,196.23487247349,197.429792711704,191.08694566106,191.458949131447,191.184643542172,191.988772255938,195.776443954427,196.024446268019,197.565066700936,197.809311403715,192.139076688418,194.032912537662,193.059691337356,193.055933726544,193.288905596888,199.706904863772,199.537812377232,191.21846203948,199.879754961124,199.541569988044,199.883512571936,199.635510258344,192.101500580298,195.975597327463,197.44106554414,190.673608471741,193.044660894108,194.397400786426,191.229734871916,198.568348787738,193.04841850492,190.560880147381,190.850216179904,194.431219283734,195.201529500192,195.464562257032,199.353689447445,197.756704852347,199.131990409537,191.504040461191,194.393643175614,190.91785317452,198.617197728294,190.568395369005,195.002376127157,194.06673103497,197.437307933328,194.679221597325,198.658531447226,191.195916374608,191.214704428668,193.281390375264,194.36733989993,193.544423132103,191.613011174739,196.219842030242,199.875997350312,199.8722397395,192.097742969486,193.976548375482,190.628517141997,197.426035100892,193.980305986294,197.354640495464,198.339134528206,193.875092883559,193.063448948168,192.007560309998,193.781152613259,191.500282850379,191.665617726107,191.515313293627,192.000045088374,196.396449738406,190.639789974433,190.636032363621,196.760937987169,191.725739499099,190.632274752809,190.621001920373,197.557551479312,190.801367239348,198.094889825427,195.626139521947,195.182741446132,199.316113339325,194.600311770273,194.363582289118,197.5537938685,196.896211976401,194.359824678306,198.925321814877,194.788192310873,195.194014278568,198.49695418231,197.775492906407,193.833759164627,192.022590753246,193.957760321422,198.444347630942,191.10573371512,199.312355728513,194.002851651166,199.522781933984,195.629897132759,190.583425812253,195.419470927288,193.623332959155,199.147020852785,193.240056656332,191.079430439436,196.31002468973,199.349931836633,191.962468980254,198.252709479531,193.99157881873,193.221268602272,194.322248570186,194.780677089249,199.143263241973,196.783483652041,196.61063355469,193.935214656551,194.382370343178,193.942729878175,194.912193467669,198.263982311967,197.200578452173,193.457998083427,193.837516775439,199.812117966508,194.089276699842,191.737012331535,190.579668201441,191.733254720723,194.570250883777,190.835185736656,194.01036687279,193.597029683471,190.985490169136,195.622381911135,198.061071328119,195.633654743571,197.099122960249,197.542521036064,199.42132644206,193.938972267363,193.961517932234,194.607826991897,190.962944504264,194.14188325121,194.134368029586,193.950245099799,199.936119123303,194.138125640398,196.715846657425,197.907009284827,199.413811220436,196.058264765327,196.065779986951,194.014124483602,199.928603901679,193.138601164408,197.696583079356,194.773161867625,192.052651639742,193.946487488987,199.624237425908,196.170993089686,194.769404256813,191.729497109911,191.755800385595,191.834710212646,191.673132947731,198.019737609187,193.95400271061,194.619099824333,190.831428125844,194.713040094633,195.321773046176,193.465513305051,190.906580342084,196.790998873665,199.725692917832,191.128279379992,191.582950288243,194.04794298091,199.729450528644,192.04889402893,191.676890558543,194.34855184587,191.473979574695,194.930981521729,194.765646646001,199.470175382616,199.64678309078,196.464086733022,191.489010017943,198.835139155389,195.885414667975,198.305316030898,193.371573034752,199.308598117701,198.015979998375,193.540665521291,193.548180742915,191.695678612603,194.093034310654,193.375330645564,194.096791921466,196.062022376139],"y":[17,14.5736798293486,13.9367554970376,13.8626726882712,12.8967725532862,12.6051699344843,12.2973937244996,12.1765834784843,12.0973583036316,11.8592358725551,11.7824232512607,11.6894612088589,11.6894612088589,11.6848956142537,11.3839425223814,11.3839425223814,11.235824847404,10.9866705804055,10.9409124337171,10.923484093069,10.8627546011119,10.8544991670639,10.8132638077569,10.7976847822131,10.7006246842973,10.6782079307323,10.6782079307323,10.6782079307323,10.6782079307323,10.6782079307323,10.6452329470874,10.5505515299198,10.5505515299198,10.5505515299198,10.5505515299198,10.5505515299198,10.5505515299198,10.4041051102379,10.3435504994762,10.3195071567286,10.2611227080908,10.2611227080908,10.121452419727,9.98495220902787,9.90125013663563,9.8514890637518,9.8514890637518,9.84086002515475,9.8057536868956,9.80017249355323,9.74173333057987,9.72093824077595,9.7107788816823,9.7107788816823,9.63386915632869,9.59318258922226,9.59318258922226,9.57746075883192,9.5562797110066,9.54448713025421,9.45425579871529,9.37156992030621,9.35679439079813,9.27212094585178,9.22483080559248,8.99269121819667,8.99269121819667,8.94487559694536,8.8796019455266,8.8175717324754,8.8175717324754,8.76865602701755,8.76865602701755,8.76865602701755,8.63533566757388,8.55290187396526,8.55290187396526,8.44795667821956,8.42615156332198,8.35963048880723,8.35963048880723,8.32347495510134,8.17845417720374,8.17845417720374,8.07887461612589,8.0462763404159,8.0462763404159,8.02177499658072,8.00259832633317,7.99031086629764,7.95010061510293,7.95010061510293,7.87510582015382,7.85551935237476,7.76248498533744,7.74819193365869,7.74819193365869,7.6709520535379,7.46323973602655,7.34569750328523,7.28771037302926,7.27015078650074,6.99158558997796,6.98789029833101,6.90811759049026,6.88485158160479,6.87290565667268,6.82849224967267,6.75808042772395,6.69522092462986,6.60049617857124,6.56872493798133,6.53834370190107,6.50879862920273,6.44773100800705,6.12312972536804,6.07218438061171,6.06698241323265,5.86315286356341,5.83390111042314,5.77980521082888,5.7324995497385,5.66423336782754,5.53615714035729,5.43360945851263,5.35271307059163,5.10724461968253,5.03077368914794,4.96078465143353,4.92803553193872,4.90124352484422,4.77954724087501,4.73563075815654,4.71804185942679,4.63656197042036,4.58260031992894,4.57126688949921,4.50572305281477,4.44890284622737,4.44687663179218,4.42055115918443,4.35813683429272,4.30098962145761,4.23230594399129,4.22826801369384,4.14108987353335,4.0101097616324,3.96653849096512,3.90340001323203,3.89664238279363,3.63370121570492,3.5655312698142,3.54512834087001,3.54512834087001,3.47398498408382,3.46636382405286,3.3614987458582,3.35987874996763,3.33144629610319,3.26799943100674,3.25785889723479,3.15461520654397,3.12394737571215,3.10190764456237,3.07354104794305,3.04283532236819,3.01893667108715,3.01893667108715,2.87458122177622,2.8392251774704,2.78878616739725,2.76057974713994,2.65553441610018,2.55019813031615,2.5431778530681,2.5431778530681,2.5384003475593,2.51578785293552,2.50711969349855,2.47267836795756,2.39323734427402,2.38356307588661,2.36016604799905,2.31999099428752,2.31535477835255,2.30250872565307,2.24482452509114,2.1412776887777,2.13325937296777,2.09609733587053,2.06265199950335,2.02722359058896,2.02722359058896,2.02722359058896,2.00664516769662,1.98419438314169,1.96805082022233,1.95213263633763,1.91480552881645,1.86235135404735,1.83385819167993,1.78367259808871,1.77708643111592,1.66365090287998,1.64671387486864,1.63661313343203,1.63464199964505,1.62484632835024,1.61501308926709,1.60910011240755,1.60573597751957,1.59392999021765,1.59392999021765,1.55462186922127,1.52056880584154,1.49728032095634,1.49728032095634,1.49319487284188,1.42828406072584,1.41848575644291,1.41848575644291,1.41848575644291,1.4062355506545,1.3919584539844,1.3919584539844,1.35728183055377,1.3344422396973],"text":["REAC:R-HSA-8953854 (662) <br> Metabolism of RNA <br> 5.057e-20","REAC:R-HSA-72163 (178) <br> mRNA Splicing - Major Pathway <br> 2.669e-15","REAC:R-HSA-72172 (186) <br> mRNA Splicing <br> 1.157e-14","REAC:R-HSA-3858494 (143) <br> Beta-catenin independent WNT signaling <br> 1.372e-14","REAC:R-HSA-72203 (237) <br> Processing of Capped Intron-Containing Pre-mRNA <br> 1.268e-13","REAC:R-HSA-1236974 (83) <br> ER-Phagosome pathway <br> 2.482e-13","REAC:R-HSA-9010553 (171) <br> Regulation of expression of SLITs and ROBOs <br> 5.042e-13","REAC:R-HSA-211733 (50) <br> Regulation of activated PAK-2p34 by proteasome mediated degradation <br> 6.659e-13","REAC:R-HSA-202403 (120) <br> TCR signaling <br> 7.992e-13","REAC:R-HSA-376176 (217) <br> Signaling by ROBO receptors <br> 1.383e-12","REAC:R-HSA-5678895 (61) <br> Defective CFTR causes cystic fibrosis <br> 1.650e-12","REAC:R-HSA-9604323 (53) <br> Negative regulation of NOTCH4 signaling <br> 2.044e-12","REAC:R-HSA-169911 (53) <br> Regulation of Apoptosis <br> 2.044e-12","REAC:R-HSA-68949 (71) <br> Orc1 removal from chromatin <br> 2.066e-12","REAC:R-HSA-8854050 (55) <br> FBXL7 down-regulates AURKA during mitotic entry and in early mitosis <br> 4.131e-12","REAC:R-HSA-174113 (55) <br> SCF-beta-TrCP mediated degradation of Emi1 <br> 4.131e-12","REAC:R-HSA-4086400 (92) <br> PCP/CE pathway <br> 5.810e-12","REAC:R-HSA-5658442 (67) <br> Regulation of RAS by GAPs <br> 1.031e-11","REAC:R-HSA-1236975 (99) <br> Antigen processing-Cross presentation <br> 1.146e-11","REAC:R-HSA-5621481 (136) <br> C-type lectin receptors (CLRs) <br> 1.193e-11","REAC:R-HSA-68867 (68) <br> Assembly of the pre-replicative complex <br> 1.372e-11","REAC:R-HSA-1236978 (50) <br> Cross-presentation of soluble exogenous antigens (endosomes) <br> 1.398e-11","REAC:R-HSA-5676590 (59) <br> NIK-->noncanonical NF-kB signaling <br> 1.537e-11","REAC:R-HSA-6798695 (476) <br> Neutrophil degranulation <br> 1.593e-11","REAC:R-HSA-350562 (51) <br> Regulation of ornithine decarboxylase (ODC) <br> 1.992e-11","REAC:R-HSA-187577 (60) <br> SCF(Skp2)-mediated degradation of p27/p21 <br> 2.098e-11","REAC:R-HSA-5607761 (60) <br> Dectin-1 mediated noncanonical NF-kB signaling <br> 2.098e-11","REAC:R-HSA-5610785 (60) <br> GLI3 is processed to GLI3R by the proteasome <br> 2.098e-11","REAC:R-HSA-5610783 (60) <br> Degradation of GLI2 by the proteasome <br> 2.098e-11","REAC:R-HSA-5610780 (60) <br> Degradation of GLI1 by the proteasome <br> 2.098e-11","REAC:R-HSA-1168372 (80) <br> Downstream signaling events of B Cell Receptor (BCR) <br> 2.263e-11","REAC:R-HSA-180534 (52) <br> Vpu mediated degradation of CD4 <br> 2.815e-11","REAC:R-HSA-69601 (52) <br> Ubiquitin Mediated Degradation of Phosphorylated Cdc25A <br> 2.815e-11","REAC:R-HSA-349425 (52) <br> Autodegradation of the E3 ubiquitin ligase COP1 <br> 2.815e-11","REAC:R-HSA-69610 (52) <br> p53-Independent DNA Damage Response <br> 2.815e-11","REAC:R-HSA-75815 (52) <br> Ubiquitin-dependent degradation of Cyclin D <br> 2.815e-11","REAC:R-HSA-69613 (52) <br> p53-Independent G1/S DNA damage checkpoint <br> 2.815e-11","REAC:R-HSA-180585 (53) <br> Vif-mediated degradation of APOBEC3G <br> 3.944e-11","REAC:R-HSA-69002 (85) <br> DNA Replication Pre-Initiation <br> 4.534e-11","REAC:R-HSA-9675108 (574) <br> Nervous system development <br> 4.792e-11","REAC:R-HSA-8941858 (54) <br> Regulation of RUNX3 expression and activity <br> 5.481e-11","REAC:R-HSA-450408 (54) <br> AUF1 (hnRNP D0) binds and destabilizes mRNA <br> 5.481e-11","REAC:R-HSA-4641257 (55) <br> Degradation of AXIN <br> 7.560e-11","REAC:R-HSA-5362768 (56) <br> Hh mutants are degraded by ERAD <br> 1.035e-10","REAC:R-HSA-422475 (549) <br> Axon guidance <br> 1.255e-10","REAC:R-HSA-69541 (57) <br> Stabilization of p53 <br> 1.408e-10","REAC:R-HSA-4641258 (57) <br> Degradation of DVL <br> 1.408e-10","REAC:R-HSA-5619084 (77) <br> ABC transporter disorders <br> 1.443e-10","REAC:R-HSA-1169091 (67) <br> Activation of NF-kappaB in B cells <br> 1.564e-10","REAC:R-HSA-162909 (125) <br> Host Interactions of HIV factors <br> 1.584e-10","REAC:R-HSA-5687128 (89) <br> MAPK6/MAPK4 signaling <br> 1.812e-10","REAC:R-HSA-351202 (58) <br> Metabolism of polyamines <br> 1.901e-10","REAC:R-HSA-72649 (59) <br> Translation initiation complex formation <br> 1.946e-10","REAC:R-HSA-72702 (59) <br> Ribosomal scanning and start codon recognition <br> 1.946e-10","REAC:R-HSA-8852276 (79) <br> The role of GTSE1 in G2/M progression after G2 checkpoint <br> 2.323e-10","REAC:R-HSA-68827 (59) <br> CDT1 association with the CDC6:ORC:origin complex <br> 2.552e-10","REAC:R-HSA-5387390 (59) <br> Hh mutants abrogate ligand secretion <br> 2.552e-10","REAC:R-HSA-72662 (60) <br> Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S <br> 2.646e-10","REAC:R-HSA-69052 (91) <br> Switching of origins to a post-replicative state <br> 2.778e-10","REAC:R-HSA-382556 (103) <br> ABC-family proteins mediated transport <br> 2.854e-10","REAC:R-HSA-156827 (112) <br> L13a-mediated translational silencing of Ceruloplasmin expression <br> 3.514e-10","REAC:R-HSA-72706 (113) <br> GTP hydrolysis and joining of the 60S ribosomal subunit <br> 4.250e-10","REAC:R-HSA-8939902 (71) <br> Regulation of RUNX2 expression and activity <br> 4.397e-10","REAC:R-HSA-168249 (1090) <br> Innate Immune System <br> 5.344e-10","REAC:R-HSA-69239 (120) <br> Synthesis of DNA <br> 5.959e-10","REAC:R-HSA-4608870 (64) <br> Asymmetric localization of PCP proteins <br> 1.017e-09","REAC:R-HSA-174084 (64) <br> Autodegradation of Cdh1 by Cdh1:APC/C <br> 1.017e-09","REAC:R-HSA-202424 (98) <br> Downstream TCR signaling <br> 1.135e-09","REAC:R-HSA-5358346 (65) <br> Hedgehog ligand biogenesis <br> 1.319e-09","REAC:R-HSA-72613 (120) <br> Eukaryotic Translation Initiation <br> 1.522e-09","REAC:R-HSA-72737 (120) <br> Cap-dependent Translation Initiation <br> 1.522e-09","REAC:R-HSA-1234176 (66) <br> Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha <br> 1.704e-09","REAC:R-HSA-69580 (66) <br> p53-Dependent G1/S DNA damage checkpoint <br> 1.704e-09","REAC:R-HSA-69563 (66) <br> p53-Dependent G1 DNA Damage Response <br> 1.704e-09","REAC:R-HSA-69306 (128) <br> DNA Replication <br> 2.316e-09","REAC:R-HSA-69615 (68) <br> G1/S DNA Damage Checkpoints <br> 2.800e-09","REAC:R-HSA-174154 (68) <br> APC/C:Cdc20 mediated degradation of Securin <br> 2.800e-09","REAC:R-HSA-8948751 (69) <br> Regulation of PTEN stability and activity <br> 3.565e-09","REAC:R-HSA-69206 (131) <br> G1/S Transition <br> 3.748e-09","REAC:R-HSA-176408 (81) <br> Regulation of APC/C activators between G1/S and early anaphase <br> 4.369e-09","REAC:R-HSA-9013694 (81) <br> Signaling by NOTCH4 <br> 4.369e-09","REAC:R-HSA-72695 (52) <br> Formation of the ternary complex, and subsequently, the 43S complex <br> 4.748e-09","REAC:R-HSA-195253 (83) <br> Degradation of beta-catenin by the destruction complex <br> 6.630e-09","REAC:R-HSA-69202 (83) <br> Cyclin E associated events during G1/S transition  <br> 6.630e-09","REAC:R-HSA-2454202 (182) <br> Fc epsilon receptor (FCERI) signaling <br> 8.339e-09","REAC:R-HSA-69017 (73) <br> CDK-mediated phosphorylation and removal of Cdc6 <br> 8.989e-09","REAC:R-HSA-174184 (73) <br> Cdc20:Phospho-APC/C mediated degradation of Cyclin A <br> 8.989e-09","REAC:R-HSA-5607764 (97) <br> CLEC7A (Dectin-1) signaling <br> 9.511e-09","REAC:R-HSA-69656 (85) <br> Cyclin A:Cdk2-associated events at S phase entry <br> 9.940e-09","REAC:R-HSA-6807070 (138) <br> PTEN Regulation <br> 1.023e-08","REAC:R-HSA-179419 (74) <br> APC:Cdc20 mediated degradation of cell cycle proteins prior to satisfation of the cell cycle checkpoint <br> 1.122e-08","REAC:R-HSA-174178 (74) <br> APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1 <br> 1.122e-08","REAC:R-HSA-5357801 (186) <br> Programmed Cell Death <br> 1.333e-08","REAC:R-HSA-1234174 (75) <br> Cellular response to hypoxia <br> 1.395e-08","REAC:R-HSA-176409 (76) <br> APC/C:Cdc20 mediated degradation of mitotic proteins <br> 1.728e-08","REAC:R-HSA-174143 (88) <br> APC/C-mediated degradation of cell cycle proteins <br> 1.786e-08","REAC:R-HSA-453276 (88) <br> Regulation of mitotic cell cycle <br> 1.786e-08","REAC:R-HSA-176814 (77) <br> Activation of APC/C and APC/C:Cdc20 mediated degradation of mitotic proteins <br> 2.133e-08","REAC:R-HSA-392518 (32) <br> Signal amplification <br> 3.442e-08","REAC:R-HSA-453279 (148) <br> Mitotic G1 phase and G1/S transition <br> 4.511e-08","REAC:R-HSA-5683057 (308) <br> MAPK family signaling cascades <br> 5.156e-08","REAC:R-HSA-8878159 (94) <br> Transcriptional regulation by RUNX3 <br> 5.368e-08","REAC:R-HSA-5663205 (909) <br> Infectious disease <br> 1.020e-07","REAC:R-HSA-5632684 (85) <br> Hedgehog 'on' state <br> 1.028e-07","REAC:R-HSA-450531 (86) <br> Regulation of mRNA stability by proteins that bind AU-rich elements <br> 1.236e-07","REAC:R-HSA-5673001 (264) <br> RAF/MAP kinase cascade <br> 1.304e-07","REAC:R-HSA-5610787 (113) <br> Hedgehog 'off' state <br> 1.340e-07","REAC:R-HSA-5668541 (100) <br> TNFR2 non-canonical NF-kB pathway <br> 1.484e-07","REAC:R-HSA-9020702 (101) <br> Interleukin-1 signaling <br> 1.745e-07","REAC:R-HSA-5684996 (269) <br> MAPK1/MAPK3 signaling <br> 2.017e-07","REAC:R-HSA-983705 (163) <br> Signaling by the B Cell Receptor (BCR) <br> 2.509e-07","REAC:R-HSA-69242 (162) <br> S Phase <br> 2.699e-07","REAC:R-HSA-72689 (102) <br> Formation of a pool of free 40S subunits <br> 2.895e-07","REAC:R-HSA-1280215 (696) <br> Cytokine Signaling in Immune system <br> 3.099e-07","REAC:R-HSA-202040 (28) <br> G-protein activation <br> 3.567e-07","REAC:R-HSA-195721 (328) <br> Signaling by WNT <br> 7.531e-07","REAC:R-HSA-109581 (174) <br> Apoptosis <br> 8.469e-07","REAC:R-HSA-8878166 (118) <br> Transcriptional regulation by RUNX2 <br> 8.571e-07","REAC:R-HSA-69275 (196) <br> G2/M Transition <br> 1.370e-06","REAC:R-HSA-5689603 (101) <br> UCH proteinases <br> 1.466e-06","REAC:R-HSA-453274 (198) <br> Mitotic G2-G2/M phases <br> 1.660e-06","REAC:R-HSA-392170 (22) <br> ADP signalling through P2Y purinoceptor 12 <br> 1.851e-06","REAC:R-HSA-71291 (370) <br> Metabolism of amino acids and derivatives <br> 2.167e-06","REAC:R-HSA-2871837 (136) <br> FCERI mediated NF-kB activation <br> 2.910e-06","REAC:R-HSA-428930 (24) <br> Thromboxane signalling through TP receptor <br> 3.685e-06","REAC:R-HSA-5619115 (173) <br> Disorders of transmembrane transporters <br> 4.439e-06","REAC:R-HSA-983170 (25) <br> Antigen Presentation: Folding, assembly and peptide loading of class I MHC <br> 7.812e-06","REAC:R-HSA-1257604 (252) <br> PIP3 activates AKT signaling <br> 9.316e-06","REAC:R-HSA-72766 (292) <br> Translation <br> 1.094e-05","REAC:R-HSA-6814122 (42) <br> Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding <br> 1.180e-05","REAC:R-HSA-5358351 (149) <br> Signaling by Hedgehog <br> 1.255e-05","REAC:R-HSA-69481 (168) <br> G2/M Checkpoints <br> 1.661e-05","REAC:R-HSA-5663202 (380) <br> Diseases of signal transduction by growth factor receptors and second messengers <br> 1.838e-05","REAC:R-HSA-162906 (226) <br> HIV Infection <br> 1.914e-05","REAC:R-HSA-446652 (138) <br> Interleukin-1 family signaling <br> 2.309e-05","REAC:R-HSA-456926 (31) <br> Thrombin signalling through proteinase activated receptors (PARs) <br> 2.615e-05","REAC:R-HSA-392851 (19) <br> Prostacyclin signalling through prostacyclin receptor <br> 2.684e-05","REAC:R-HSA-76002 (258) <br> Platelet activation, signaling and aggregation <br> 3.121e-05","REAC:R-HSA-418597 (48) <br> G alpha (z) signalling events <br> 3.557e-05","REAC:R-HSA-109582 (673) <br> Hemostasis <br> 3.574e-05","REAC:R-HSA-8964616 (20) <br> G beta:gamma signalling through CDC42 <br> 3.797e-05","REAC:R-HSA-9006925 (292) <br> Intracellular signaling by second messengers <br> 4.384e-05","REAC:R-HSA-449147 (459) <br> Signaling by Interleukins <br> 5.000e-05","REAC:R-HSA-8939236 (130) <br> RUNX1 regulates transcription of genes involved in differentiation of HSCs <br> 5.857e-05","REAC:R-HSA-1236977 (11) <br> Endosomal/Vacuolar pathway <br> 5.912e-05","REAC:R-HSA-196025 (11) <br> Formation of annular gap junctions <br> 7.226e-05","REAC:R-HSA-72187 (56) <br> mRNA 3'-end processing <br> 9.770e-05","REAC:R-HSA-190873 (12) <br> Gap junction degradation <br> 1.080e-04","REAC:R-HSA-2262752 (582) <br> Cellular responses to stress <br> 1.249e-04","REAC:R-HSA-418592 (24) <br> ADP signalling through P2Y purinoceptor 1 <br> 1.269e-04","REAC:R-HSA-8953897 (596) <br> Cellular responses to external stimuli <br> 2.324e-04","REAC:R-HSA-168256 (2038) <br> Immune System <br> 2.719e-04","REAC:R-HSA-991365 (43) <br> Activation of GABAB receptors <br> 2.850e-04","REAC:R-HSA-977444 (43) <br> GABA B receptor activation <br> 2.850e-04","REAC:R-HSA-9009391 (73) <br> Extra-nuclear estrogen signaling <br> 3.357e-04","REAC:R-HSA-400042 (28) <br> Adrenaline,noradrenaline inhibits insulin secretion <br> 3.417e-04","REAC:R-HSA-68882 (228) <br> Mitotic Anaphase <br> 4.350e-04","REAC:R-HSA-2467813 (188) <br> Separation of Sister Chromatids <br> 4.366e-04","REAC:R-HSA-2555396 (229) <br> Mitotic Metaphase and Anaphase <br> 4.662e-04","REAC:R-HSA-73856 (65) <br> RNA Polymerase II Transcription Termination <br> 5.395e-04","REAC:R-HSA-422356 (78) <br> Regulation of insulin secretion <br> 5.523e-04","REAC:R-HSA-159236 (71) <br> Transport of Mature mRNA derived from an Intron-Containing Transcript <br> 7.005e-04","REAC:R-HSA-8964315 (18) <br> G beta:gamma signalling through BTK <br> 7.517e-04","REAC:R-HSA-397795 (32) <br> G-protein beta:gamma signalling <br> 7.908e-04","REAC:R-HSA-168255 (152) <br> Influenza Infection <br> 8.442e-04","REAC:R-HSA-1280218 (799) <br> Adaptive Immune System <br> 9.061e-04","REAC:R-HSA-420092 (33) <br> Glucagon-type ligand receptors <br> 9.573e-04","REAC:R-HSA-163359 (33) <br> Glucagon signaling in metabolic regulation <br> 9.573e-04","REAC:R-HSA-418217 (20) <br> G beta:gamma signalling through PLC beta <br> 1.335e-03","REAC:R-HSA-8868773 (193) <br> rRNA processing in the nucleus and cytosol <br> 1.448e-03","REAC:R-HSA-381676 (42) <br> Glucagon-like Peptide-1 (GLP1) regulates insulin secretion <br> 1.626e-03","REAC:R-HSA-500657 (21) <br> Presynaptic function of Kainate receptors <br> 1.735e-03","REAC:R-HSA-1799339 (113) <br> SRP-dependent cotranslational protein targeting to membrane <br> 2.210e-03","REAC:R-HSA-72202 (80) <br> Transport of Mature Transcript to Cytoplasm <br> 2.817e-03","REAC:R-HSA-975957 (116) <br> Nonsense Mediated Decay (NMD) enhanced by the Exon Junction Complex (EJC) <br> 2.863e-03","REAC:R-HSA-927802 (116) <br> Nonsense-Mediated Decay (NMD) <br> 2.863e-03","REAC:R-HSA-977443 (60) <br> GABA receptor activation <br> 2.895e-03","REAC:R-HSA-72312 (203) <br> rRNA processing <br> 3.049e-03","REAC:R-HSA-1643685 (1616) <br> Disease <br> 3.111e-03","REAC:R-HSA-9633012 (102) <br> Response of EIF2AK4 (GCN2) to amino acid deficiency <br> 3.368e-03","REAC:R-HSA-877300 (87) <br> Interferon gamma signaling <br> 4.044e-03","REAC:R-HSA-2424491 (25) <br> DAP12 signaling <br> 4.135e-03","REAC:R-HSA-392451 (25) <br> G beta:gamma signalling through PI3Kgamma <br> 4.363e-03","REAC:R-HSA-432040 (43) <br> Vasopressin regulates renal water homeostasis via Aquaporins <br> 4.786e-03","REAC:R-HSA-111885 (90) <br> Opioid Signalling <br> 4.838e-03","REAC:R-HSA-909733 (68) <br> Interferon alpha/beta signaling <br> 4.983e-03","REAC:R-HSA-9711097 (157) <br> Cellular response to starvation <br> 5.691e-03","REAC:R-HSA-390466 (95) <br> Chaperonin-mediated protein folding <br> 7.223e-03","REAC:R-HSA-983169 (370) <br> Class I MHC mediated antigen processing & presentation <br> 7.358e-03","REAC:R-HSA-69620 (290) <br> Cell Cycle Checkpoints <br> 8.015e-03","REAC:R-HSA-399954 (16) <br> Sema3A PAK dependent Axon repulsion <br> 8.657e-03","REAC:R-HSA-1296059 (29) <br> G protein gated Potassium channels <br> 9.392e-03","REAC:R-HSA-997272 (29) <br> Inhibition  of voltage gated Ca2+ channels via Gbeta/gamma subunits <br> 9.392e-03","REAC:R-HSA-1296041 (29) <br> Activation of G protein gated Potassium channels <br> 9.392e-03","REAC:R-HSA-163685 (107) <br> Integration of energy metabolism <br> 9.848e-03","REAC:R-HSA-6791226 (183) <br> Major pathway of rRNA processing in the nucleolus and cytosol <br> 1.037e-02","REAC:R-HSA-168275 (2) <br> Entry of Influenza Virion into Host Cell via Endocytosis <br> 1.076e-02","REAC:R-HSA-451326 (30) <br> Activation of kainate receptors upon glutamate binding <br> 1.117e-02","REAC:R-HSA-391251 (102) <br> Protein folding <br> 1.217e-02","REAC:R-HSA-5140745 (13) <br> WNT5A-dependent internalization of FZD2, FZD5 and ROR2 <br> 1.373e-02","REAC:R-HSA-445717 (52) <br> Aquaporin-mediated transport <br> 1.466e-02","REAC:R-HSA-4086398 (59) <br> Ca2+ pathway <br> 1.646e-02","REAC:R-HSA-114604 (35) <br> GPVI-mediated activation cascade <br> 1.671e-02","REAC:R-HSA-5099900 (15) <br> WNT5A-dependent internalization of FZD4 <br> 2.169e-02","REAC:R-HSA-2172127 (33) <br> DAP12 interactions <br> 2.256e-02","REAC:R-HSA-69278 (549) <br> Cell Cycle, Mitotic <br> 2.309e-02","REAC:R-HSA-3371511 (12) <br> HSF1 activation <br> 2.319e-02","REAC:R-HSA-5621575 (20) <br> CD209 (DC-SIGN) signaling <br> 2.372e-02","REAC:R-HSA-1296065 (35) <br> Inwardly rectifying K+ channels <br> 2.427e-02","REAC:R-HSA-913531 (193) <br> Interferon Signaling <br> 2.460e-02","REAC:R-HSA-382551 (720) <br> Transport of small molecules <br> 2.479e-02","REAC:R-HSA-192823 (90) <br> Viral mRNA Translation <br> 2.547e-02","REAC:R-HSA-156902 (90) <br> Peptide chain elongation <br> 2.547e-02","REAC:R-HSA-389359 (11) <br> CD28 dependent Vav1 pathway <br> 2.789e-02","REAC:R-HSA-201681 (232) <br> TCF dependent signaling in response to WNT <br> 3.016e-02","REAC:R-HSA-8951664 (233) <br> Neddylation <br> 3.182e-02","REAC:R-HSA-157118 (233) <br> Signaling by NOTCH <br> 3.182e-02","REAC:R-HSA-2682334 (90) <br> EPH-Ephrin signaling <br> 3.212e-02","REAC:R-HSA-8878171 (236) <br> Transcriptional regulation by RUNX1 <br> 3.730e-02","REAC:R-HSA-2408557 (94) <br> Selenocysteine synthesis <br> 3.815e-02","REAC:R-HSA-156842 (94) <br> Eukaryotic Translation Elongation <br> 3.815e-02","REAC:R-HSA-72764 (94) <br> Eukaryotic Translation Termination <br> 3.815e-02","REAC:R-HSA-202733 (190) <br> Cell surface interactions at the vascular wall <br> 3.924e-02","REAC:R-HSA-190828 (49) <br> Gap junction trafficking <br> 4.055e-02","REAC:R-HSA-3928665 (49) <br> EPH-ephrin mediated repulsion of cells <br> 4.055e-02","REAC:R-HSA-157858 (50) <br> Gap junction trafficking and regulation <br> 4.393e-02","REAC:R-HSA-975956 (96) <br> Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) <br> 4.630e-02"],"key":["REAC:R-HSA-8953854","REAC:R-HSA-72163","REAC:R-HSA-72172","REAC:R-HSA-3858494","REAC:R-HSA-72203","REAC:R-HSA-1236974","REAC:R-HSA-9010553","REAC:R-HSA-211733","REAC:R-HSA-202403","REAC:R-HSA-376176","REAC:R-HSA-5678895","REAC:R-HSA-9604323","REAC:R-HSA-169911","REAC:R-HSA-68949","REAC:R-HSA-8854050","REAC:R-HSA-174113","REAC:R-HSA-4086400","REAC:R-HSA-5658442","REAC:R-HSA-1236975","REAC:R-HSA-5621481","REAC:R-HSA-68867","REAC:R-HSA-1236978","REAC:R-HSA-5676590","REAC:R-HSA-6798695","REAC:R-HSA-350562","REAC:R-HSA-187577","REAC:R-HSA-5607761","REAC:R-HSA-5610785","REAC:R-HSA-5610783","REAC:R-HSA-5610780","REAC:R-HSA-1168372","REAC:R-HSA-180534","REAC:R-HSA-69601","REAC:R-HSA-349425","REAC:R-HSA-69610","REAC:R-HSA-75815","REAC:R-HSA-69613","REAC:R-HSA-180585","REAC:R-HSA-69002","REAC:R-HSA-9675108","REAC:R-HSA-8941858","REAC:R-HSA-450408","REAC:R-HSA-4641257","REAC:R-HSA-5362768","REAC:R-HSA-422475","REAC:R-HSA-69541","REAC:R-HSA-4641258","REAC:R-HSA-5619084","REAC:R-HSA-1169091","REAC:R-HSA-162909","REAC:R-HSA-5687128","REAC:R-HSA-351202","REAC:R-HSA-72649","REAC:R-HSA-72702","REAC:R-HSA-8852276","REAC:R-HSA-68827","REAC:R-HSA-5387390","REAC:R-HSA-72662","REAC:R-HSA-69052","REAC:R-HSA-382556","REAC:R-HSA-156827","REAC:R-HSA-72706","REAC:R-HSA-8939902","REAC:R-HSA-168249","REAC:R-HSA-69239","REAC:R-HSA-4608870","REAC:R-HSA-174084","REAC:R-HSA-202424","REAC:R-HSA-5358346","REAC:R-HSA-72613","REAC:R-HSA-72737","REAC:R-HSA-1234176","REAC:R-HSA-69580","REAC:R-HSA-69563","REAC:R-HSA-69306","REAC:R-HSA-69615","REAC:R-HSA-174154","REAC:R-HSA-8948751","REAC:R-HSA-69206","REAC:R-HSA-176408","REAC:R-HSA-9013694","REAC:R-HSA-72695","REAC:R-HSA-195253","REAC:R-HSA-69202","REAC:R-HSA-2454202","REAC:R-HSA-69017","REAC:R-HSA-174184","REAC:R-HSA-5607764","REAC:R-HSA-69656","REAC:R-HSA-6807070","REAC:R-HSA-179419","REAC:R-HSA-174178","REAC:R-HSA-5357801","REAC:R-HSA-1234174","REAC:R-HSA-176409","REAC:R-HSA-174143","REAC:R-HSA-453276","REAC:R-HSA-176814","REAC:R-HSA-392518","REAC:R-HSA-453279","REAC:R-HSA-5683057","REAC:R-HSA-8878159","REAC:R-HSA-5663205","REAC:R-HSA-5632684","REAC:R-HSA-450531","REAC:R-HSA-5673001","REAC:R-HSA-5610787","REAC:R-HSA-5668541","REAC:R-HSA-9020702","REAC:R-HSA-5684996","REAC:R-HSA-983705","REAC:R-HSA-69242","REAC:R-HSA-72689","REAC:R-HSA-1280215","REAC:R-HSA-202040","REAC:R-HSA-195721","REAC:R-HSA-109581","REAC:R-HSA-8878166","REAC:R-HSA-69275","REAC:R-HSA-5689603","REAC:R-HSA-453274","REAC:R-HSA-392170","REAC:R-HSA-71291","REAC:R-HSA-2871837","REAC:R-HSA-428930","REAC:R-HSA-5619115","REAC:R-HSA-983170","REAC:R-HSA-1257604","REAC:R-HSA-72766","REAC:R-HSA-6814122","REAC:R-HSA-5358351","REAC:R-HSA-69481","REAC:R-HSA-5663202","REAC:R-HSA-162906","REAC:R-HSA-446652","REAC:R-HSA-456926","REAC:R-HSA-392851","REAC:R-HSA-76002","REAC:R-HSA-418597","REAC:R-HSA-109582","REAC:R-HSA-8964616","REAC:R-HSA-9006925","REAC:R-HSA-449147","REAC:R-HSA-8939236","REAC:R-HSA-1236977","REAC:R-HSA-196025","REAC:R-HSA-72187","REAC:R-HSA-190873","REAC:R-HSA-2262752","REAC:R-HSA-418592","REAC:R-HSA-8953897","REAC:R-HSA-168256","REAC:R-HSA-991365","REAC:R-HSA-977444","REAC:R-HSA-9009391","REAC:R-HSA-400042","REAC:R-HSA-68882","REAC:R-HSA-2467813","REAC:R-HSA-2555396","REAC:R-HSA-73856","REAC:R-HSA-422356","REAC:R-HSA-159236","REAC:R-HSA-8964315","REAC:R-HSA-397795","REAC:R-HSA-168255","REAC:R-HSA-1280218","REAC:R-HSA-420092","REAC:R-HSA-163359","REAC:R-HSA-418217","REAC:R-HSA-8868773","REAC:R-HSA-381676","REAC:R-HSA-500657","REAC:R-HSA-1799339","REAC:R-HSA-72202","REAC:R-HSA-975957","REAC:R-HSA-927802","REAC:R-HSA-977443","REAC:R-HSA-72312","REAC:R-HSA-1643685","REAC:R-HSA-9633012","REAC:R-HSA-877300","REAC:R-HSA-2424491","REAC:R-HSA-392451","REAC:R-HSA-432040","REAC:R-HSA-111885","REAC:R-HSA-909733","REAC:R-HSA-9711097","REAC:R-HSA-390466","REAC:R-HSA-983169","REAC:R-HSA-69620","REAC:R-HSA-399954","REAC:R-HSA-1296059","REAC:R-HSA-997272","REAC:R-HSA-1296041","REAC:R-HSA-163685","REAC:R-HSA-6791226","REAC:R-HSA-168275","REAC:R-HSA-451326","REAC:R-HSA-391251","REAC:R-HSA-5140745","REAC:R-HSA-445717","REAC:R-HSA-4086398","REAC:R-HSA-114604","REAC:R-HSA-5099900","REAC:R-HSA-2172127","REAC:R-HSA-69278","REAC:R-HSA-3371511","REAC:R-HSA-5621575","REAC:R-HSA-1296065","REAC:R-HSA-913531","REAC:R-HSA-382551","REAC:R-HSA-192823","REAC:R-HSA-156902","REAC:R-HSA-389359","REAC:R-HSA-201681","REAC:R-HSA-8951664","REAC:R-HSA-157118","REAC:R-HSA-2682334","REAC:R-HSA-8878171","REAC:R-HSA-2408557","REAC:R-HSA-156842","REAC:R-HSA-72764","REAC:R-HSA-202733","REAC:R-HSA-190828","REAC:R-HSA-3928665","REAC:R-HSA-157858","REAC:R-HSA-975956"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,102,204,1)","opacity":0.8,"size":[18.90922365592,17.0869555296552,17.1519653138816,16.7583551431596,17.5047579060072,15.9033751977876,17.0273503124218,15.0486221338682,16.4890463101257,17.3774579655542,15.3914887766185,15.1501626734972,15.1501626734972,15.6465726979379,15.2142427416402,15.2142427416402,16.0697362528688,15.5497876137808,16.1868772799724,16.6818474191902,15.5745906356112,15.0486221338682,15.3347186146137,18.4728732854611,15.0832328994516,15.3633750239743,15.3633750239743,15.3633750239743,15.3633750239743,15.3633750239743,15.8433279821367,15.1170689685676,15.1170689685676,15.1170689685676,15.1170689685676,15.1170689685676,15.1170689685676,15.1501626734972,15.9420548318458,18.7220962814027,15.1825444005545,15.1825444005545,15.2142427416402,15.2452846314144,18.6631932828122,15.2756954716879,15.2756954716879,15.7806675338474,15.5497876137808,16.552248256339,16.0164101032087,15.3054992444299,15.3347186146137,15.3347186146137,15.8227421236023,15.3347186146137,15.3347186146137,15.3633750239743,16.0521821733416,16.2496921162547,16.3815087539172,16.3954151879507,15.6465726979379,19.5459903341358,16.4890463101257,15.4727616697066,15.4727616697066,16.1707256977144,15.498887690643,16.4890463101257,16.4890463101257,15.5245580341656,15.5245580341656,15.5245580341656,16.5888237442405,15.5745906356112,15.5745906356112,15.5989806420055,16.624451291171,15.8636236908885,15.8636236908885,15.1170689685676,15.9033751977876,15.9033751977876,17.1198572429448,15.692660304218,15.692660304218,16.1543870489597,15.9420548318458,16.7041480940405,15.7151677983179,15.7151677983179,17.1519653138816,15.7373316539529,15.7591617451593,15.9981813072674,15.9981813072674,15.7806675338474,14.2382740983495,16.8104850340226,17.8763374918434,16.1042061318618,19.3171356175398,15.9420548318458,15.9610096906275,17.6589522750826,16.3954151879507,16.2028458488432,16.2186353293393,17.6855927091729,16.9558134205769,16.9465972450821,16.2342495236679,18.9743833960756,13.9833131530912,17.9641020663474,17.0532216312128,16.4629334550975,17.2289935746564,16.2186353293393,17.2438757382447,13.5059251363754,18.1306861948291,16.6818474191902,13.6808214449962,17.0446533454794,13.7618192670139,17.5926771820558,17.8014896682064,14.73917293128,16.820675004284,17.0009690943617,18.1672960274771,17.4362828639663,16.7041480940405,14.1782205386673,13.2039079443259,17.6262397302166,14.976936429373,18.9306948804635,13.3106655321065,17.8014896682064,18.4239673394241,16.6126778615783,11.980531463383,11.980531463383,15.2452846314144,12.187219027138,18.7403571890374,13.6808214449962,18.7716660414332,20.3104179536764,14.781443852525,14.781443852525,15.692660304218,13.9833131530912,17.4490032718266,17.1677330667511,17.4553172949697,15.498887690643,15.801858092204,15.6465726979379,13.0900518224006,14.2382740983495,16.8507929922937,19.1525079244994,14.2961523059962,14.2961523059962,13.3106655321065,17.206351575782,14.73917293128,13.4111152730112,16.3954151879507,15.8433279821367,16.4363189409718,16.4363189409718,15.3633750239743,17.2803636129636,20.0311976668916,16.2342495236679,15.9797164726717,13.7618192670139,13.7618192670139,14.781443852525,16.0344085591459,15.5745906356112,16.8995454455618,16.121131665261,18.1306861948291,17.7918175061017,12.8370677538266,14.0509285588584,14.0509285588584,14.0509285588584,16.309816295931,17.1279572175344,3.77952755905512,14.1158314099108,16.2342495236679,12.3729643352122,15.1170689685676,15.3347186146137,14.405947329161,12.6954028668369,14.2961523059962,18.6631932828122,12.187219027138,13.3106655321065,14.405947329161,17.206351575782,19.0183314066768,16.0344085591459,16.0344085591459,11.980531463383,17.4740780401283,17.4802721022218,17.4802721022218,16.0344085591459,17.4986794977524,16.1042061318618,16.1042061318618,16.1042061318618,17.1833157067251,15.0132022257977,15.0132022257977,15.0486221338682,16.1378571469232],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,102,204,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"REAC","legendgroup":"REAC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"29d43bee26ec":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"29d44af428b3":{"x":{},"xend":{},"y":{},"yend":{}},"29d4177d26ae":{"x":{},"xend":{},"y":{},"yend":{}},"29d44a9616cd":{"x":{},"xend":{},"y":{},"yend":{}},"29d451a16541":{"x":{},"xend":{},"y":{},"yend":{}},"29d41a7c0b":{"x":{},"y":{}},"29d438743437":{"yintercept":{}}},"cur_data":"29d43bee26ec","visdat":{"29d43bee26ec":["function (y) ","x"],"29d44af428b3":["function (y) ","x"],"29d4177d26ae":["function (y) ","x"],"29d44a9616cd":["function (y) ","x"],"29d451a16541":["function (y) ","x"],"29d41a7c0b":["function (y) ","x"],"29d438743437":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedDatad1ba32e0"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;%
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-21697d58abce2053a245" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-21697d58abce2053a245">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546","547","548","549","550","551","552","553","554","555","556","557","558","559","560","561","562","563","564","565","566","567","568","569","570","571","572","573","574","575","576","577","578","579","580","581","582","583","584","585","586","587","588","589","590","591","592","593","594","595","596","597","598","599","600","601","602","603","604","605","606","607","608","609","610","611","612","613","614","615","616","617","618","619","620","621","622","623","624","625","626","627","628","629","630","631","632","633","634","635","636","637","638","639","640","641","642","643","644","645","646","647","648","649","650","651","652","653","654","655","656","657","658","659","660","661","662","663","664","665","666","667","668","669","670","671","672","673","674","675","676","677","678","679","680","681","682","683","684","685","686","687","688","689","690","691","692","693","694","695","696","697","698","699","700","701","702","703","704","705","706","707","708","709","710","711","712","713","714","715","716","717","718","719","720","721","722","723","724","725","726","727","728","729","730","731","732","733","734","735","736","737","738","739","740","741","742","743","744","745","746","747","748","749","750","751","752","753","754","755","756","757","758","759","760","761","762","763","764","765","766","767","768","769","770","771","772","773","774","775","776","777","778","779","780","781","782","783","784","785","786","787","788","789","790","791","792","793","794","795","796","797","798","799","800","801","802","803","804","805","806","807","808","809","810","811","812","813","814","815","816","817","818","819","820","821","822","823","824","825","826","827","828","829","830","831","832","833","834","835","836","837","838","839","840","841","842","843","844","845","846","847","848","849","850","851","852","853","854","855","856","857","858","859","860","861","862","863","864","865","866","867","868","869","870","871","872","873","874","875","876","877","878","879","880","881","882","883","884","885","886","887","888","889","890","891","892","893","894","895","896","897","898","899","900","901","902","903","904","905","906","907","908","909","910","911","912","913","914","915","916","917","918","919","920","921","922","923","924","925","926","927","928","929","930","931","932","933","934","935","936","937","938","939","940","941","942","943","944","945","946","947","948","949","950","951","952","953","954","955","956","957","958"],["GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","REAC","GO:MF","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","REAC","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","REAC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:CC","REAC","REAC","GO:CC","GO:CC","REAC","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:BP","REAC","GO:BP","GO:BP","GO:BP","REAC","REAC","REAC","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","REAC","REAC","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","REAC","GO:BP","GO:BP","GO:BP","REAC","REAC","REAC","GO:CC","GO:CC","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","REAC","REAC","REAC","REAC","GO:BP","REAC","GO:CC","GO:MF","REAC","REAC","REAC","REAC","REAC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","REAC","REAC","GO:BP","REAC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:CC","REAC","REAC","GO:BP","REAC","REAC","REAC","GO:BP","REAC","GO:BP","REAC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","REAC","REAC","GO:MF","REAC","GO:BP","REAC","GO:BP","GO:MF","GO:MF","REAC","GO:CC","GO:CC","GO:MF","GO:BP","REAC","GO:BP","REAC","GO:MF","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:CC","GO:CC","REAC","REAC","GO:CC","GO:BP","REAC","GO:CC","GO:BP","GO:CC","REAC","GO:MF","GO:MF","GO:MF","REAC","REAC","GO:BP","GO:BP","REAC","REAC","REAC","GO:CC","GO:BP","REAC","GO:BP","GO:MF","REAC","REAC","GO:MF","GO:BP","GO:CC","REAC","REAC","GO:BP","REAC","REAC","REAC","REAC","REAC","GO:BP","GO:BP","REAC","REAC","REAC","REAC","REAC","GO:BP","REAC","GO:BP","REAC","REAC","GO:MF","REAC","REAC","GO:BP","REAC","REAC","REAC","GO:CC","GO:BP","GO:CC","REAC","GO:CC","GO:BP","GO:CC","GO:CC","REAC","GO:CC","GO:BP","GO:BP","REAC","GO:CC","REAC","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","REAC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","REAC","GO:BP","GO:CC","REAC","GO:BP","REAC","GO:CC","GO:BP","REAC","GO:BP","REAC","GO:BP","REAC","GO:CC","GO:MF","REAC","REAC","GO:BP","GO:CC","REAC","GO:BP","GO:BP","REAC","GO:CC","GO:CC","REAC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:MF","GO:BP","GO:BP","REAC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:MF","GO:BP","GO:CC","REAC","GO:BP","GO:BP","REAC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","REAC","GO:BP","GO:MF","GO:BP","GO:CC","REAC","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","REAC","GO:BP","REAC","REAC","GO:BP","REAC","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:MF","REAC","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","REAC","GO:MF","GO:BP","GO:BP","REAC","REAC","GO:CC","REAC","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","REAC","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","REAC","GO:MF","REAC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:MF","GO:BP","REAC","GO:CC","GO:BP","REAC","REAC","GO:CC","GO:CC","GO:CC","GO:BP","GO:MF","GO:BP","GO:CC","REAC","GO:MF","GO:CC","GO:CC","REAC","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:CC","GO:MF","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","REAC","GO:BP","GO:CC","REAC","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:CC","GO:CC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","REAC","GO:CC","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","REAC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:MF","GO:CC","GO:MF","GO:BP","GO:BP","GO:MF","GO:MF","REAC","REAC","REAC","GO:BP","REAC","GO:BP","GO:BP","REAC","GO:CC","REAC","GO:CC","GO:CC","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","REAC","REAC","REAC","GO:BP","GO:CC","GO:BP","GO:CC","REAC","REAC","GO:CC","REAC","GO:BP","GO:CC","GO:BP","REAC","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:MF","REAC","GO:BP","GO:BP","GO:CC","REAC","GO:MF","GO:CC","GO:BP","REAC","GO:BP","GO:CC","REAC","REAC","REAC","GO:BP","GO:BP","GO:CC","REAC","GO:MF","REAC","GO:BP","GO:BP","REAC","REAC","GO:CC","GO:MF","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:MF","REAC","GO:CC","GO:CC","GO:MF","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","REAC","GO:CC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:BP","REAC","GO:CC","GO:BP","REAC","REAC","GO:CC","REAC","GO:BP","REAC","GO:BP","REAC","GO:CC","REAC","REAC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:MF","GO:MF","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:CC","REAC","REAC","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:MF","GO:MF","GO:BP","GO:MF","REAC","REAC","REAC","GO:BP","GO:BP","REAC","GO:CC","GO:MF","GO:BP","REAC","REAC","GO:MF","GO:CC","GO:CC","GO:BP","GO:CC","REAC","GO:BP","GO:MF","GO:BP","GO:MF","GO:MF","GO:CC","REAC","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP"],[2743,4056,2131,2719,1570,438,4442,1453,16054,5688,16046,2132,5659,2128,2130,278,12268,939,12281,1099,156,168,2127,947,154,10980,1322,3497,1101,2779,11579,15738,13142,284,3621,2400,1027,14486,3780,1295,1879,561,550,1080,15417,26352,19740,15694,2401,2843,938,2552,2029,9231,2934,1030,335,1134,927,1129,1026,7628,385,916,2406,11868,15361,13160,4001,1401,1084,13259,885,241,1412,13161,1125,889,2405,1408,1098,917,29126,29136,1334,2765,11210,886,1402,7017,10009,2134,2720,1446,19079,10305,1648,2901,2472,11087,19737,15359,7129,12668,1476,9160,11949,3542,2921,2471,5762,201,13159,19815,98,861,12553,531,18660,450,7633,19430,23108,1118,14527,1133,10013,19802,10037,9696,4089,474,4087,1452,1647,7019,15416,1474,758,1217,14485,850,1853,1847,312,3627,2207,10505,489,861,14349,2090,22049,470,859,12746,534,1438,1813,1499,4618,2507,2842,598,470,815,1934,6697,1038,16937,1514,9338,3077,532,21250,6495,1832,144,243,15294,15295,16072,170,384,1392,4088,3582,1458,3658,2437,25265,15740,24637,12824,1868,1933,424,928,669,668,2897,730,2895,1993,2438,2393,179,2484,2394,2485,3961,11197,1332,8306,5363,3982,4320,2419,414,18197,1445,1835,34,12403,16102,12235,13203,665,8216,15569,1025,11212,2041,5665,182,17813,817,2135,666,20427,4,81,1034,5103,1239,12402,1309,2344,1919,3992,12488,25858,7622,2407,2285,255,1024,99,7162,2148,12743,6,8215,1889,4447,1186,3800,902,10208,17052,937,18950,1834,399,5598,1100,2159,24793,19397,2447,818,173,178,1060,12486,728,1247,19167,899,1017,4445,5413,3547,798,284,3673,2416,1510,2483,2482,3551,29917,413,9881,3785,913,22,3784,2898,338,1831,914,15312,1812,2074,886,670,389,29230,3990,861,254,298,258,387,26353,1557,2287,25,24,599,1654,314,8568,23,20,1866,3628,2409,1864,68,1078,21333,785,1832,2009,3561,19432,72,1352,1119,1234,2334,627,2496,9546,18976,12399,2590,12678,2896,1079,1016,3071,3552,478,2543,1057,1121,295,1865,21226,479,1690,2288,1015,1073,11657,2230,3901,1129,4729,1237,3398,3758,2116,1924,6459,2056,875,8304,10495,393,1075,1013,908,4593,19431,9428,4075,7749,14544,4787,75,983,2102,25857,6342,14941,20932,149,2333,3575,4767,2410,19511,5104,10338,6389,14885,1116,920,4679,2415,2389,11616,3072,1353,4227,1128,11029,18661,10,7747,1297,909,442,9324,1135,9278,819,442,159,25220,2289,3672,6670,15684,45,12141,12732,3068,717,12473,2542,18664,19800,19534,21171,2926,17166,984,19531,6791,142,3880,13069,5407,2932,3654,410,2505,1534,24834,19510,2343,5675,14545,8287,3959,12744,19292,377,23322,5405,19441,418,2051,14889,11946,2866,3583,1066,2374,2566,917,8362,712,1005,17158,1127,440,11028,26826,2450,2288,7605,443,1660,15197,5689,6775,3963,966,22750,1614,35,738,4678,902,1021,1801,904,25217,480,2022,3470,1162,2600,10008,9081,433,17633,12219,1454,3473,2054,3638,496,15388,12472,1771,15683,775,7750,1122,3079,19228,4661,876,515,449,417,7740,18937,6460,8299,5080,2466,1349,20407,8374,2093,11547,943,13682,6440,2133,413,2853,7607,317,7174,9,2129,25219,840,20429,28000,3860,1276,2334,11668,2055,2597,788,29092,8566,316,15314,20,1694,8197,1071,378,4786,77,922,2928,1065,1117,26125,6278,16097,795,812,1165,2042,411,117,6279,1015,2933,2927,22060,1077,14697,26123,7618,15074,9110,1351,2000,340,6607,1354,15456,16936,1138,9709,3086,1744,2506,3629,1862,2444,28341,3624,27326,1074,17162,17818,19442,2362,1085,837,15428,15440,481,3625,4009,903,3238,653,909,29478,2565,8759,1081,7743,409,67,111,612,28343,13703,12738,1141,1139,957,955,25216,1162,1124,16110,12812,1240,9880,1710,24824,468,13022,1366,906,14688,2499,12523,5876,11574,20412,14309,19434,2450,16045,2976,3132,12269,956,1070,1455,2598,2599,1642,14646,1277,26828,19055,1100,791,16061,3357,7731,2443,332,1112,1959,3796,13020,24935,1862,118,13129,2944,1058,6753,15435,9594,822,439,26133,20868,444,6689,2360,1467,1469,7018,923,651,25572,2497,1262,690,438,3329,22995,12224,1903,2306,3996,2915,19033,10377,4922,13554,427,11676,862,3698,12404,3138,1072,419,1125,401,905,111,863,7020,2177,2416,1497,561,1124,24936,819,893,315,190,3801,2627,10381,2183,3067,655,4728,3544,322,13134,100,1959,343,6496,11678,1964,300,5061,3612,19853,1989,2443,1163,907,1084,76,4280,25568,2462,1109,5233,1271,2847,2933,777,96,318,5889,10614,27328,1662,7625,8761,18553,2121,6763,656,1365,1353,2443,19492,868,7367,155,1263,193,919,19198,104,1261,26162,2024,10027,14940,31,276,3626,932,1963,15392,9964,649,6458,3659,257,160,1149,11679,1414,4791,7600,17082,3852,3123,15747,1347,2444,400,20403,301,4194,9823,1012,247,1763,1167,3078,1123,652,2375,88,2422,1575,6976,3068,437,3802,597,3364,389,4980,3169,251,2611,9847,6378,9109,3136,24825,6667,16055,8196,2854,24826,2206,1016,12725,1079,1421,2065,752,6462,4281,1878,19739,8298,3630,2419,1231,10434,1344,26164,2846,2332,2123,4620,9839,1354,1606,1988,797,799,2862,8438,306,2797,9689,20865,944,753,5960,1447,1440,5915,374,945,2844,5081,4509,1696,6756,3644,1468,3605,14886,8604,1869,16112,12355],["extracellular exosome","extracellular vesicle","extracellular organelle","extracellular membrane-bounded organelle","protein-containing complex","RNA binding","ribonucleoprotein complex","vesicle","establishment of localization in cell","mRNA metabolic process","cellular localization","intracellular membrane-bounded organelle","viral process","membrane-bounded organelle","intracellular organelle","extracellular space","leukocyte degranulation","myeloid cell activation involved in immune response","neutrophil degranulation","myeloid leukocyte mediated immunity","RNA splicing, via transesterification reactions with bulged adenosine as nucleophile","mRNA splicing, via spliceosome","organelle","neutrophil activation involved in immune response","RNA splicing, via transesterification reactions","granulocyte activation","cytoplasmic vesicle","intracellular vesicle","neutrophil mediated immunity","transport","neutrophil activation","establishment of localization","regulated exocytosis","intracellular anatomical structure","RNA splicing","RNA processing","cell-substrate junction","intracellular transport","secretory vesicle","Metabolism of RNA","protein binding","cell activation","focal adhesion","secretory granule","T cell receptor signaling pathway","regulation of mRNA metabolic process","nitrogen compound transport","localization","mRNA processing","exocytosis","myeloid leukocyte activation","pigment granule","melanosome","macromolecule localization","catalytic step 2 spliceosome","leukocyte activation involved in immune response","spliceosomal complex","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","cell activation involved in immune response","antigen processing and presentation of peptide antigen via MHC class I","cell junction","ribonucleoprotein complex biogenesis","cytoplasm","immune effector process","mRNA catabolic process","antigen processing and presentation of exogenous peptide antigen via MHC class I","positive regulation of immune response","regulation of innate immune response","catalytic complex","immune response-activating signal transduction","immune response-activating cell surface receptor signaling pathway","leukocyte activation","activation of innate immune response","extracellular region","immune response-regulating cell surface receptor signaling pathway","positive regulation of innate immune response","secretory granule membrane","stimulatory C-type lectin receptor signaling pathway","RNA catabolic process","immune response-regulating signaling pathway","leukocyte mediated immunity","activation of immune response","response to lectin","cellular response to lectin","positive regulation of immune system process","anchoring junction","Fc receptor signaling pathway","innate immune response activating cell surface receptor signaling pathway","innate immune response-activating signal transduction","antigen processing and presentation","cellular protein localization","organelle lumen","intracellular organelle lumen","membrane-enclosed lumen","cellular macromolecule localization","organic cyclic compound binding","eukaryotic 48S preinitiation complex","immune response","mRNA Splicing - Major Pathway","pre-replicative complex assembly","organic substance transport","regulation of immune response","protein-containing complex binding","cellular component biogenesis","positive regulation of response to biotic stimulus","secretion by cell","heterocyclic compound binding","protein localization","translation preinitiation complex","mRNA Splicing","vesicle-mediated transport","Beta-catenin independent WNT signaling","innate immune response","cellular component organization or biogenesis","proteasome complex","cytoplasmic translational initiation","protein-containing complex subunit organization","formation of cytoplasmic translation initiation complex","protein-containing complex assembly","translation initiation factor activity","ribonucleoprotein complex assembly","cellular response to cytokine stimulus","export from cell","cytoplasmic vesicle membrane","antigen processing and presentation of peptide antigen","antigen processing and presentation of exogenous peptide antigen","cellular protein-containing complex assembly","ribonucleoprotein complex subunit organization","nucleobase-containing compound catabolic process","response to cytokine","endopeptidase complex","heterotrimeric G-protein complex","GTPase complex","nuclear lumen","Processing of Capped Intron-Containing Pre-mRNA","antigen processing and presentation of exogenous antigen","antigen receptor-mediated signaling pathway","regulation of response to biotic stimulus","ER-Phagosome pathway","organelle membrane","secretion","membrane","Regulation of expression of SLITs and ROBOs","Regulation of activated PAK-2p34 by proteasome mediated degradation","nucleoplasm","membrane protein complex","TCR signaling","non-canonical Wnt signaling pathway","eukaryotic translation initiation factor 3 complex","eukaryotic 43S preinitiation complex","heterocycle catabolic process","Signaling by ROBO receptors","defense response to other organism","Defective CFTR causes cystic fibrosis","cytoplasmic translation","cellular nitrogen compound catabolic process","morphogenesis of a polarized epithelium","Negative regulation of NOTCH4 signaling","Regulation of Apoptosis","Orc1 removal from chromatin","enzyme binding","regulation of cellular amino acid metabolic process","intracellular protein transport","GTPase activity","cytosol","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","SCF-beta-TrCP mediated degradation of Emi1","aromatic compound catabolic process","immune system process","Wnt signaling pathway, planar cell polarity pathway","PCP/CE pathway","regulation of cellular amine metabolic process","establishment of tissue polarity","establishment of planar polarity","regulation of establishment of planar polarity","cytokine-mediated signaling pathway","Regulation of RAS by GAPs","Antigen processing-Cross presentation","C-type lectin receptors (CLRs)","nucleic acid transport","RNA transport","localization within membrane","Assembly of the pre-replicative complex","Cross-presentation of soluble exogenous antigens (endosomes)","NIK--&gt;noncanonical NF-kB signaling","peptidase complex","bounding membrane of organelle","Neutrophil degranulation","membrane microdomain","membrane raft","regulation of hematopoietic stem cell differentiation","establishment of RNA localization","organic cyclic compound catabolic process","biological process involved in interspecies interaction between organisms","Regulation of ornithine decarboxylase (ODC)","SCF(Skp2)-mediated degradation of p27/p21","Dectin-1 mediated noncanonical NF-kB signaling","GLI3 is processed to GLI3R by the proteasome","Degradation of GLI2 by the proteasome","Degradation of GLI1 by the proteasome","response to stress","Downstream signaling events of B Cell Receptor (BCR)","tertiary granule","translation factor activity, RNA binding","Vpu mediated degradation of CD4","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Autodegradation of the E3 ubiquitin ligase COP1","p53-Independent DNA Damage Response","Ubiquitin-dependent degradation of Cyclin D","p53-Independent G1/S DNA damage checkpoint","nuclear protein-containing complex","NIK/NF-kappaB signaling","regulation of immune system process","positive regulation of defense response","protein transport","nucleoside-triphosphatase activity","response to organic substance","Vif-mediated degradation of APOBEC3G","DNA Replication Pre-Initiation","regulation of transcription from RNA polymerase II promoter in response to hypoxia","Nervous system development","Regulation of RUNX3 expression and activity","AUF1 (hnRNP D0) binds and destabilizes mRNA","regulation of mRNA stability","response to other organism","response to external biotic stimulus","establishment of protein localization","Degradation of AXIN","SCF-dependent proteasomal ubiquitin-dependent protein catabolic process","mRNA transport","Hh mutants are degraded by ERAD","Fc-epsilon receptor signaling pathway","specific granule","cellular component organization","Axon guidance","regulation of mRNA catabolic process","endomembrane system","Stabilization of p53","Degradation of DVL","protein localization to membrane","ABC transporter disorders","Activation of NF-kappaB in B cells","Host Interactions of HIV factors","negative regulation of G2/M transition of mitotic cell cycle","MAPK6/MAPK4 signaling","regulation of RNA stability","Metabolism of polyamines","Translation initiation complex formation","Ribosomal scanning and start codon recognition","response to biotic stimulus","regulation of DNA-templated transcription in response to stress","negative regulation of cell cycle G2/M phase transition","cellular component assembly","RNA localization","The role of GTSE1 in G2/M progression after G2 checkpoint","CDT1 association with the CDC6:ORC:origin complex","Hh mutants abrogate ligand secretion","Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S","translation regulator activity","Switching of origins to a post-replicative state","cellular macromolecule catabolic process","ABC-family proteins mediated transport","anaphase-promoting complex-dependent catabolic process","GTP binding","GDP binding","L13a-mediated translational silencing of Ceruloplasmin expression","ficolin-1-rich granule","nuclear speck","translation regulator activity, nucleic acid binding","hematopoietic stem cell differentiation","GTP hydrolysis and joining of the 60S ribosomal subunit","interleukin-1-mediated signaling pathway","Regulation of RUNX2 expression and activity","nucleic acid binding","nucleobase-containing compound transport","Innate Immune System","Synthesis of DNA","regulation of hematopoietic progenitor cell differentiation","cellular response to organic substance","synapse","vesicle membrane","Asymmetric localization of PCP proteins","Autodegradation of Cdh1 by Cdh1:APC/C","vesicle coat","regulation of transcription from RNA polymerase II promoter in response to stress","Downstream TCR signaling","extrinsic component of cytoplasmic side of plasma membrane","cellular response to chemical stimulus","nuclear body","Hedgehog ligand biogenesis","guanyl nucleotide binding","guanyl ribonucleotide binding","pyrophosphatase activity","Eukaryotic Translation Initiation","Cap-dependent Translation Initiation","macromolecule catabolic process","translational initiation","Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha","p53-Dependent G1/S DNA damage checkpoint","p53-Dependent G1 DNA Damage Response","side of membrane","regulation of stem cell differentiation","DNA Replication","response to interferon-gamma","hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides","G1/S DNA Damage Checkpoints","APC/C:Cdc20 mediated degradation of Securin","hydrolase activity, acting on acid anhydrides","defense response","U2-type spliceosomal complex","Regulation of PTEN stability and activity","G1/S Transition","regulation of mRNA processing","Regulation of APC/C activators between G1/S and early anaphase","Signaling by NOTCH4","Formation of the ternary complex, and subsequently, the 43S complex","Degradation of beta-catenin by the destruction complex","Cyclin E associated events during G1/S transition ","regulation of animal organ morphogenesis","response to external stimulus","Fc epsilon receptor (FCERI) signaling","CDK-mediated phosphorylation and removal of Cdc6","Cdc20:Phospho-APC/C mediated degradation of Cyclin A","CLEC7A (Dectin-1) signaling","Cyclin A:Cdk2-associated events at S phase entry","negative regulation of mRNA metabolic process","PTEN Regulation","DNA replication","APC:Cdc20 mediated degradation of cell cycle proteins prior to satisfation of the cell cycle checkpoint","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","G protein activity","Programmed Cell Death","Cellular response to hypoxia","positive regulation of response to external stimulus","APC/C:Cdc20 mediated degradation of mitotic proteins","APC/C-mediated degradation of cell cycle proteins","Regulation of mitotic cell cycle","plasma membrane protein complex","RNA export from nucleus","specific granule membrane","Activation of APC/C and APC/C:Cdc20 mediated degradation of mitotic proteins","endocytic vesicle","positive regulation of canonical Wnt signaling pathway","cytoplasmic side of plasma membrane","exon-exon junction complex","Signal amplification","cytoplasmic side of membrane","cellular response to interleukin-1","MAPK cascade","Mitotic G1 phase and G1/S transition","Golgi-associated vesicle membrane","MAPK family signaling cascades","Transcriptional regulation by RUNX3","proteasome regulatory particle, lid subcomplex","phagocytic vesicle","cellular response to stress","response to interleukin-1","regulation of RNA splicing","protein targeting","cellular amine metabolic process","tertiary granule membrane","Infectious disease","Hedgehog 'on' state","integral component of lumenal side of endoplasmic reticulum membrane","lumenal side of endoplasmic reticulum membrane","proteasome regulatory particle","coated membrane","membrane coat","coated vesicle membrane","nucleus","Regulation of mRNA stability by proteins that bind AU-rich elements","establishment of protein localization to membrane","proteasome core complex","RAF/MAP kinase cascade","DNA-dependent DNA replication","Hedgehog 'off' state","transport vesicle","response to chemical","TNFR2 non-canonical NF-kB pathway","amine metabolic process","Interleukin-1 signaling","regulation of cellular ketone metabolic process","MAPK1/MAPK3 signaling","spliceosomal snRNP complex","hydrolase activity","Signaling by the B Cell Receptor (BCR)","S Phase","viral gene expression","MHC class I protein complex","Formation of a pool of free 40S subunits","regulation of defense response","intracellular signal transduction","Cytokine Signaling in Immune system","coated vesicle","proteasome accessory complex","G-protein activation","regulation of G2/M transition of mitotic cell cycle","cellular response to interferon-gamma","protein localization to organelle","ficolin-1-rich granule lumen","positive regulation of Wnt signaling pathway","regulation of mRNA splicing, via spliceosome","negative regulation of gene expression","nuclear-transcribed mRNA catabolic process, nonsense-mediated decay","extrinsic component of plasma membrane","Signaling by WNT","regulation of cell cycle G2/M phase transition","purine ribonucleoside triphosphate binding","positive regulation of response to stimulus","regulation of response to stress","Apoptosis","Transcriptional regulation by RUNX2","lumenal side of membrane","posttranscriptional regulation of gene expression","mRNA export from nucleus","mRNA-containing ribonucleoprotein complex export from nucleus","G-protein beta/gamma-subunit complex binding","carbohydrate derivative binding","small molecule binding","positive regulation of biological process","small nuclear ribonucleoprotein complex","G2/M Transition","proteasomal ubiquitin-independent protein catabolic process","translation","UCH proteinases","cellular ketone metabolic process","leukocyte cell-cell adhesion","Mitotic G2-G2/M phases","regulation of catabolic process","phagocytic vesicle membrane","cellular response to decreased oxygen levels","protein-DNA complex assembly","ADP signalling through P2Y purinoceptor 12","platelet activation","Metabolism of amino acids and derivatives","hematopoietic progenitor cell differentiation","single-stranded RNA binding","tumor necrosis factor-mediated signaling pathway","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-independent","negative regulation of RNA splicing","FCERI mediated NF-kB activation","Golgi-associated vesicle","alternative mRNA splicing, via spliceosome","negative regulation of mitotic cell cycle phase transition","Thromboxane signalling through TP receptor","catabolic process","CD8 receptor binding","nuclear transport","G2/M transition of mitotic cell cycle","peptide biosynthetic process","cellular catabolic process","eukaryotic translation initiation factor 3 complex, eIF3m","Disorders of transmembrane transporters","amide biosynthetic process","perinuclear region of cytoplasm","regulation of biological quality","protein-DNA complex subunit organization","cellular response to hypoxia","negative regulation of canonical Wnt signaling pathway","U2-type precatalytic spliceosome","regulation of cellular localization","extrinsic component of membrane","cellular response to oxygen levels","protein metabolic process","Antigen Presentation: Folding, assembly and peptide loading of class I MHC","Sm-like protein family complex","cell cycle G2/M phase transition","purine ribonucleotide binding","precatalytic spliceosome","lytic vacuole membrane","lysosomal membrane","peptide metabolic process","PIP3 activates AKT signaling","organic substance catabolic process","ribonucleoprotein complex export from nucleus","Translation","Wnt signaling pathway","negative regulation of mRNA splicing, via spliceosome","regulation of cellular catabolic process","purine nucleotide binding","cellular protein metabolic process","ribonucleoprotein complex localization","Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding","cell-cell signaling by wnt","ribonucleotide binding","cellular response to tumor necrosis factor","vacuolar membrane","Signaling by Hedgehog","positive regulation of cellular process","nucleoside phosphate binding","nucleocytoplasmic transport","plasma membrane region","COPI vesicle coat","rRNA processing","recycling endosome membrane","G2/M Checkpoints","protein-containing complex localization","Diseases of signal transduction by growth factor receptors and second messengers","HIV Infection","interferon-gamma-mediated signaling pathway","Interleukin-1 family signaling","double-stranded RNA binding","response to decreased oxygen levels","regulation of cellular protein localization","protein folding","Thrombin signalling through proteinase activated receptors (PARs)","regulation of cell-cell adhesion","mRNA binding","Prostacyclin signalling through prostacyclin receptor","stem cell differentiation","rRNA metabolic process","anion binding","endoplasmic reticulum protein-containing complex","proteasome core complex, alpha-subunit complex","cellular response to aldehyde","Platelet activation, signaling and aggregation","nucleotide binding","morphogenesis of an epithelium","proteasomal protein catabolic process","G alpha (z) signalling events","Hemostasis","secretory granule lumen","G beta:gamma signalling through CDC42","negative regulation of cell cycle phase transition","ribosome","nuclear outer membrane-endoplasmic reticulum membrane network","blood coagulation","Intracellular signaling by second messengers","cytoplasmic vesicle lumen","response to tumor necrosis factor","cell adhesion molecule binding","endoplasmic reticulum membrane","regulation of canonical Wnt signaling pathway","proteasome-mediated ubiquitin-dependent protein catabolic process","vesicle lumen","hemostasis","Signaling by Interleukins","endoplasmic reticulum subcompartment","response to hypoxia","coagulation","cellular amide metabolic process","RUNX1 regulates transcription of genes involved in differentiation of HSCs","nuclear export","Endosomal/Vacuolar pathway","negative regulation of Wnt signaling pathway","COPI-coated vesicle membrane","cell surface receptor signaling pathway","protein localization to endoplasmic reticulum","gene expression","Formation of annular gap junctions","plasma membrane","structural constituent of ribosome","vacuole","regulation of cell adhesion","response to oxygen levels","viral translation","negative regulation of cell killing","negative regulation of cell cycle process","mRNA 3'-end processing","positive regulation of immune effector process","establishment of protein localization to organelle","mRNA 5'-UTR binding","cell projection","wound healing","Gap junction degradation","negative regulation of mitotic cell cycle","viral life cycle","intracellular non-membrane-bounded organelle","endosome","endocytosis","positive regulation of cell-cell adhesion","Cellular responses to stress","cadherin binding","ADP signalling through P2Y purinoceptor 1","non-membrane-bounded organelle","regulation of mitotic cell cycle phase transition","small ribosomal subunit","protein localization to plasma membrane","cell surface receptor signaling pathway involved in cell-cell signaling","plasma membrane bounded cell projection","intrinsic component of organelle membrane","ribosomal subunit","ribosome biogenesis","MHC protein complex","protein targeting to membrane","basal plasma membrane","protein localization to cell periphery","regulation of response to external stimulus","Cellular responses to external stimuli","negative regulation of mRNA processing","Golgi membrane","structural molecule activity","mRNA 3'-end processing","Immune System","nucleolus","positive regulation of gene expression","Activation of GABAB receptors","GABA B receptor activation","U2-type catalytic step 2 spliceosome","clathrin vesicle coat","transport vesicle membrane","positive regulation of leukocyte cell-cell adhesion","glutamate receptor binding","biological process involved in interaction with host","endosome membrane","Extra-nuclear estrogen signaling","dolichyl-diphosphooligosaccharide-protein glycotransferase activity","azurophil granule","primary lysosome","Adrenaline,noradrenaline inhibits insulin secretion","ionotropic glutamate receptor binding","cytosolic ribosome","catalytic step 1 spliceosome","U2-type catalytic step 1 spliceosome","cell-cell adhesion","COPI-coated vesicle","clathrin coat assembly","regulation of leukocyte cell-cell adhesion","regulation of anatomical structure morphogenesis","tissue morphogenesis","regulation of protein localization","Mitotic Anaphase","Separation of Sister Chromatids","U2 snRNP","telomeric DNA binding","Mitotic Metaphase and Anaphase","response to stimulus","canonical Wnt signaling pathway","antigen processing and presentation of endogenous peptide antigen","heterotypic cell-cell adhesion","porin activity","RNA Polymerase II Transcription Termination","cellular amino acid metabolic process","mitochondrial protein-containing complex","Regulation of insulin secretion","basal part of cell","regulation of protein localization to membrane","Golgi apparatus subcompartment","regulation of protein localization to cell periphery","COPII-coated ER to Golgi transport vesicle","type I interferon signaling pathway","membrane organization","cellular response to type I interferon","Transport of Mature mRNA derived from an Intron-Containing Transcript","integral component of endoplasmic reticulum membrane","actin cytoskeleton","regulation of T cell activation","regulation of body fluid levels","polysome","presynapse","response to abiotic stimulus","G beta:gamma signalling through BTK","regulation of mitotic cell cycle","negative regulation of leukocyte mediated cytotoxicity","G-protein beta:gamma signalling","regulation of DNA biosynthetic process","recycling endosome","regulation of intracellular transport","Influenza Infection","protein catabolic process","lysosome","lytic vacuole","Adaptive Immune System","oligosaccharyltransferase complex","positive regulation of protein localization to membrane","negative regulation of natural killer cell mediated cytotoxicity","cellular protein catabolic process","antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway, TAP-independent","antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway","Glucagon-type ligand receptors","Glucagon signaling in metabolic regulation","regulation of cell cycle phase transition","oligosaccharyl transferase activity","endocytic vesicle membrane","cellular response to stimulus","biological process involved in symbiotic interaction","intrinsic component of endoplasmic reticulum membrane","response to type I interferon","nuclear periphery","organonitrogen compound metabolic process","actin binding","mitotic cell cycle phase transition","negative regulation of natural killer cell mediated immunity","G beta:gamma signalling through PLC beta","mRNA stabilization","rRNA processing in the nucleus and cytosol","post-translational protein modification","protein deubiquitination","T cell activation","establishment of protein localization to endoplasmic reticulum","lymphocyte activation","cellular response to interleukin-12","tricarboxylic acid cycle enzyme complex","organelle localization","cell cycle","cell periphery","regulation of leukocyte degranulation","Glucagon-like Peptide-1 (GLP1) regulates insulin secretion","clathrin coat of trans-Golgi network vesicle","organelle subcompartment","cotranslational protein targeting to membrane","SRP-dependent cotranslational protein targeting to membrane","Presynaptic function of Kainate receptors","Golgi vesicle transport","integral component of organelle membrane","positive regulation of cellular protein localization","response to interleukin-12","dendrite","cell surface","establishment of organelle localization","dendritic tree","regulation of Wnt signaling pathway","apical part of cell","nuclear-transcribed mRNA catabolic process","midbody","SRP-dependent cotranslational protein targeting to membrane","postsynaptic endocytic zone cytoplasmic component","cell cycle phase transition","response to nitrogen compound","binding","mitotic cell cycle","protein insertion into mitochondrial outer membrane","outer mitochondrial membrane organization","clathrin coat","ATP-dependent protein binding","positive regulation of T cell activation","molecular adaptor activity","trans-Golgi network transport vesicle membrane","RNA helicase activity","mitotic cell cycle process","viral translational termination-reinitiation","mRNA 3'-UTR binding","identical protein binding","Transport of Mature Transcript to Cytoplasm","Nonsense Mediated Decay (NMD) enhanced by the Exon Junction Complex (EJC)","Nonsense-Mediated Decay (NMD)","antigen processing and presentation of endogenous antigen","GABA receptor activation","leukocyte mediated cytotoxicity","negative regulation of mRNA catabolic process","rRNA processing","cell projection membrane","Disease","Golgi apparatus","glial cell projection","exocytic process","macromolecule metabolic process","Response of EIF2AK4 (GCN2) to amino acid deficiency","cell body","response to wounding","CRD-mediated mRNA stability complex","protein modification by small protein removal","disordered domain specific binding","U1 snRNA binding","negative regulation of cell cycle","endoplasmic reticulum","natural killer cell mediated cytotoxicity","basolateral plasma membrane","glutamatergic synapse","RNA stabilization","integrin-mediated signaling pathway","clathrin coat of coated pit","single-stranded DNA binding","Interferon gamma signaling","DAP12 signaling","G beta:gamma signalling through PI3Kgamma","spliceosomal complex assembly","apical plasma membrane","antigen processing and presentation of endogenous peptide antigen via MHC class I","nuclear replication fork","Vasopressin regulates renal water homeostasis via Aquaporins","Opioid Signalling","cell cortex","Interferon alpha/beta signaling","cellular response to nitrogen compound","ER to Golgi transport vesicle membrane","natural killer cell mediated immunity","Cellular response to starvation","immunological synapse","ficolin-1-rich granule membrane","ASAP complex","G-rich strand telomeric DNA binding","multi-eIF complex","eukaryotic translation initiation factor 3 complex, eIF3e","T cell mediated cytotoxicity","regulation of cell cycle process","metabolic process","Chaperonin-mediated protein folding","protein targeting to ER","protein polyubiquitination","cytoskeletal protein binding","Class I MHC mediated antigen processing &amp; presentation","regulation of metabolic process","regulation of natural killer cell mediated cytotoxicity","somatodendritic compartment","Cell Cycle Checkpoints","translation initiation factor binding","chromosomal region","DNA biosynthetic process","Sema3A PAK dependent Axon repulsion","regulation of translational initiation","replisome","G protein gated Potassium channels","Inhibition  of voltage gated Ca2+ channels via Gbeta/gamma subunits","Activation of G protein gated Potassium channels","positive regulation of signal transduction","negative regulation of RNA catabolic process","oxoglutarate dehydrogenase complex","Integration of energy metabolism","G protein-coupled serotonin receptor binding","Major pathway of rRNA processing in the nucleolus and cytosol","intra-Golgi vesicle-mediated transport","organelle organization","Entry of Influenza Virion into Host Cell via Endocytosis","Activation of kainate receptors upon glutamate binding","DNA replication factor A complex","RNA strand-exchange activity","interleukin-12-mediated signaling pathway","positive regulation of protein localization to cell periphery","Protein folding","biological adhesion","positive regulation of intracellular transport","regulation of small molecule metabolic process","myelin sheath","single-stranded telomeric DNA binding","regulation of T cell mediated cytotoxicity","regulation of natural killer cell mediated immunity","regulation of leukocyte mediated immunity","WNT5A-dependent internalization of FZD2, FZD5 and ROR2","cellular response to organic cyclic compound","nuclear matrix","TAP binding","Aquaporin-mediated transport","cell trailing edge","uropod","threonine-type endopeptidase activity","CRD-mediated mRNA stabilization","chromosome, telomeric region","cell leading edge","regulation of protein localization to plasma membrane","paraspeckles","cellular nitrogen compound metabolic process","regulation of response to stimulus","SNARE binding","Ca2+ pathway","postsynapse","GPVI-mediated activation cascade","cytoplasmic ribonucleoprotein granule","protein stabilization","ncRNA processing","cell killing","viral genome replication","actin-based cell projection","telomere maintenance","regulation of alternative mRNA splicing, via spliceosome","cortical cytoskeleton","protection from natural killer cell mediated cytotoxicity","G-protein beta/gamma-subunit complex","clathrin binding","cell cycle process","regulation of macromolecule metabolic process","ATPase","dopamine receptor signaling pathway","regulation of protein metabolic process","regulation of immune effector process","WNT5A-dependent internalization of FZD4","DAP12 interactions","clathrin-dependent endocytosis","Cell Cycle, Mitotic","oxidoreductase complex","regulation of cellular amide metabolic process","HSF1 activation","CD209 (DC-SIGN) signaling","integrin alphaL-beta2 complex","Inwardly rectifying K+ channels","signal transduction","Interferon Signaling","regulation of leukocyte mediated cytotoxicity","Transport of small molecules","autophagosome membrane","Viral mRNA Translation","Peptide chain elongation","carboxylic acid metabolic process","cell adhesion","endoplasmic reticulum-Golgi intermediate compartment","cytolytic granule membrane","positive regulation of cytokine production","Schwann cell microvillus","mitochondrial outer membrane translocase complex","telomere maintenance via telomere lengthening","Ras protein signal transduction","CD28 dependent Vav1 pathway","protein targeting to mitochondrion","DNA polymerase binding","pre-mRNA binding","regulation of localization","Ragulator complex","organonitrogen compound catabolic process","peptide antigen binding","establishment of vesicle localization","RNA 3'-end processing","receptor-mediated endocytosis","organonitrogen compound biosynthetic process","TCF dependent signaling in response to WNT","cytosolic small ribosomal subunit","cellular metabolic process","trans-Golgi network transport vesicle","Neddylation","Signaling by NOTCH","EPH-Ephrin signaling","viral transcription","negative regulation of signal transduction","ribonucleoprotein granule","organic substance metabolic process","regulation of cell killing","outer mitochondrial membrane protein complex","regulation of translation","SNARE complex","sequence-specific single stranded DNA binding","regulation of leukocyte activation","positive regulation of protein localization to plasma membrane","retrograde vesicle-mediated transport, Golgi to endoplasmic reticulum","Transcriptional regulation by RUNX1","organic acid metabolic process","protein kinase binding","isopeptidase activity","negative regulation of leukocyte mediated immunity","signaling receptor binding","Selenocysteine synthesis","Eukaryotic Translation Elongation","Eukaryotic Translation Termination","phagocytosis","regulation of protein stability","Cell surface interactions at the vascular wall","exocytic vesicle","telomeric repeat-containing RNA binding","IRES-dependent viral translational initiation","Gap junction trafficking","EPH-ephrin mediated repulsion of cells","3'-5' RNA helicase activity","envelope","organelle envelope","regulation of exocytosis","perichromatin fibrils","Gap junction trafficking and regulation","endoplasmic reticulum to Golgi vesicle-mediated transport","myosin V binding","response to organonitrogen compound","structural constituent of cytoskeleton","ribonucleoprotein complex binding","presynaptic endocytic zone membrane","Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC)","oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor","negative regulation of biological process","telomere organization","mitochondrial prohibitin complex","regulation of cell cycle","oxoacid metabolic process"],["4.9e-49","8.2e-47","9.2e-47","9.2e-47","2.9e-39","3.0e-38","6.4e-38","4.6e-37","4.7e-31","5.5e-31","2.3e-29","1.2e-28","1.7e-28","3.6e-26","5.7e-25","8.0e-25","3.9e-23","1.6e-22","2.3e-22","2.9e-22","3.6e-22","3.6e-22","3.9e-22","3.9e-22","5.2e-22","5.4e-22","5.9e-22","7.2e-22","1.4e-21","1.4e-21","1.7e-21","3.7e-21","4.4e-21","4.4e-21","1.8e-20","1.9e-20","2.6e-20","2.7e-20","3.3e-20","5.1e-20","7.1e-20","9.4e-20","1.1e-19","1.2e-19","1.5e-19","1.8e-19","3.5e-19","4.0e-19","5.2e-19","7.0e-19","1.0e-18","1.1e-18","1.1e-18","1.8e-18","2.0e-18","2.2e-18","2.9e-18","2.9e-18","3.0e-18","3.6e-18","4.6e-18","6.7e-18","8.9e-18","9.8e-18","1.1e-17","1.3e-17","1.4e-17","1.4e-17","2.6e-17","2.8e-17","2.8e-17","4.2e-17","4.4e-17","5.3e-17","6.4e-17","6.7e-17","7.0e-17","7.3e-17","8.0e-17","8.0e-17","8.3e-17","9.0e-17","9.1e-17","9.1e-17","1.1e-16","1.1e-16","1.4e-16","1.4e-16","1.7e-16","2.2e-16","2.8e-16","3.6e-16","3.6e-16","3.6e-16","4.9e-16","1.0e-15","1.0e-15","1.4e-15","2.7e-15","4.2e-15","4.8e-15","5.3e-15","6.5e-15","6.9e-15","7.4e-15","7.6e-15","8.9e-15","9.1e-15","1.0e-14","1.2e-14","1.3e-14","1.4e-14","1.5e-14","1.9e-14","2.1e-14","2.6e-14","2.7e-14","2.7e-14","3.1e-14","3.1e-14","3.2e-14","3.6e-14","3.6e-14","3.7e-14","4.2e-14","4.3e-14","4.4e-14","8.4e-14","9.0e-14","9.1e-14","1.0e-13","1.1e-13","1.1e-13","1.2e-13","1.3e-13","1.3e-13","2.1e-13","2.1e-13","2.5e-13","2.7e-13","3.1e-13","4.2e-13","5.0e-13","6.7e-13","6.8e-13","7.0e-13","8.0e-13","1.1e-12","1.2e-12","1.2e-12","1.4e-12","1.4e-12","1.6e-12","1.7e-12","1.8e-12","2.0e-12","2.0e-12","2.0e-12","2.0e-12","2.1e-12","2.1e-12","2.7e-12","2.8e-12","3.6e-12","3.9e-12","4.1e-12","4.1e-12","4.2e-12","4.5e-12","4.6e-12","5.8e-12","7.2e-12","8.4e-12","8.4e-12","9.1e-12","1.0e-11","1.0e-11","1.1e-11","1.2e-11","1.3e-11","1.3e-11","1.3e-11","1.4e-11","1.4e-11","1.5e-11","1.6e-11","1.6e-11","1.6e-11","1.7e-11","1.7e-11","1.9e-11","1.9e-11","2.0e-11","2.0e-11","2.0e-11","2.1e-11","2.1e-11","2.1e-11","2.1e-11","2.1e-11","2.2e-11","2.3e-11","2.5e-11","2.7e-11","2.8e-11","2.8e-11","2.8e-11","2.8e-11","2.8e-11","2.8e-11","3.3e-11","3.4e-11","3.4e-11","3.6e-11","3.7e-11","3.8e-11","3.9e-11","3.9e-11","4.5e-11","4.7e-11","4.8e-11","5.5e-11","5.5e-11","6.1e-11","6.5e-11","6.7e-11","7.2e-11","7.6e-11","7.8e-11","8.0e-11","1.0e-10","1.1e-10","1.1e-10","1.2e-10","1.3e-10","1.3e-10","1.4e-10","1.4e-10","1.4e-10","1.4e-10","1.4e-10","1.6e-10","1.6e-10","1.6e-10","1.8e-10","1.8e-10","1.9e-10","1.9e-10","1.9e-10","2.0e-10","2.1e-10","2.3e-10","2.3e-10","2.3e-10","2.3e-10","2.6e-10","2.6e-10","2.6e-10","2.7e-10","2.8e-10","2.8e-10","2.9e-10","2.9e-10","3.4e-10","3.5e-10","3.5e-10","3.7e-10","3.8e-10","3.9e-10","4.2e-10","4.3e-10","4.4e-10","4.4e-10","4.7e-10","4.8e-10","5.3e-10","6.0e-10","6.0e-10","6.5e-10","6.5e-10","8.0e-10","1.0e-09","1.0e-09","1.0e-09","1.1e-09","1.1e-09","1.2e-09","1.2e-09","1.3e-09","1.3e-09","1.4e-09","1.4e-09","1.4e-09","1.5e-09","1.5e-09","1.6e-09","1.6e-09","1.7e-09","1.7e-09","1.7e-09","1.8e-09","2.1e-09","2.3e-09","2.6e-09","2.7e-09","2.8e-09","2.8e-09","2.8e-09","2.9e-09","3.5e-09","3.6e-09","3.7e-09","4.0e-09","4.4e-09","4.4e-09","4.7e-09","6.6e-09","6.6e-09","7.7e-09","7.8e-09","8.3e-09","9.0e-09","9.0e-09","9.5e-09","9.9e-09","1.0e-08","1.0e-08","1.1e-08","1.1e-08","1.1e-08","1.2e-08","1.3e-08","1.4e-08","1.7e-08","1.7e-08","1.8e-08","1.8e-08","1.8e-08","1.8e-08","2.1e-08","2.1e-08","2.3e-08","2.3e-08","2.9e-08","3.4e-08","3.4e-08","3.5e-08","3.6e-08","4.4e-08","4.5e-08","4.6e-08","5.2e-08","5.4e-08","6.0e-08","6.7e-08","6.9e-08","7.5e-08","8.0e-08","8.1e-08","9.2e-08","9.4e-08","1.0e-07","1.0e-07","1.1e-07","1.1e-07","1.1e-07","1.1e-07","1.1e-07","1.1e-07","1.2e-07","1.2e-07","1.3e-07","1.3e-07","1.3e-07","1.3e-07","1.3e-07","1.4e-07","1.4e-07","1.5e-07","1.5e-07","1.7e-07","1.9e-07","2.0e-07","2.3e-07","2.4e-07","2.5e-07","2.7e-07","2.8e-07","2.8e-07","2.9e-07","3.0e-07","3.0e-07","3.1e-07","3.4e-07","3.5e-07","3.6e-07","3.7e-07","4.1e-07","4.7e-07","6.0e-07","6.1e-07","6.7e-07","6.8e-07","7.2e-07","7.3e-07","7.5e-07","7.8e-07","8.0e-07","8.2e-07","8.4e-07","8.5e-07","8.6e-07","8.9e-07","9.5e-07","9.9e-07","9.9e-07","1.1e-06","1.1e-06","1.1e-06","1.2e-06","1.2e-06","1.4e-06","1.4e-06","1.4e-06","1.5e-06","1.5e-06","1.6e-06","1.7e-06","1.7e-06","1.8e-06","1.8e-06","1.8e-06","1.9e-06","2.1e-06","2.2e-06","2.2e-06","2.5e-06","2.6e-06","2.7e-06","2.9e-06","2.9e-06","3.4e-06","3.4e-06","3.6e-06","3.7e-06","3.8e-06","3.9e-06","4.1e-06","4.1e-06","4.2e-06","4.4e-06","4.4e-06","4.4e-06","5.0e-06","5.0e-06","5.0e-06","5.8e-06","6.0e-06","6.1e-06","6.1e-06","6.9e-06","6.9e-06","7.2e-06","7.7e-06","7.8e-06","7.9e-06","8.4e-06","8.7e-06","9.1e-06","9.1e-06","9.1e-06","9.2e-06","9.3e-06","9.7e-06","1.0e-05","1.1e-05","1.1e-05","1.1e-05","1.1e-05","1.2e-05","1.2e-05","1.2e-05","1.2e-05","1.2e-05","1.2e-05","1.2e-05","1.2e-05","1.3e-05","1.4e-05","1.4e-05","1.4e-05","1.4e-05","1.4e-05","1.5e-05","1.6e-05","1.7e-05","1.7e-05","1.8e-05","1.9e-05","2.0e-05","2.3e-05","2.3e-05","2.4e-05","2.4e-05","2.5e-05","2.6e-05","2.7e-05","2.7e-05","2.7e-05","2.8e-05","2.8e-05","3.0e-05","3.0e-05","3.1e-05","3.1e-05","3.1e-05","3.2e-05","3.3e-05","3.5e-05","3.6e-05","3.6e-05","3.6e-05","3.8e-05","3.9e-05","4.0e-05","4.0e-05","4.2e-05","4.4e-05","4.4e-05","4.5e-05","4.6e-05","4.7e-05","4.7e-05","4.8e-05","4.9e-05","5.0e-05","5.0e-05","5.3e-05","5.4e-05","5.7e-05","5.8e-05","5.9e-05","5.9e-05","5.9e-05","6.0e-05","6.1e-05","6.2e-05","6.6e-05","6.7e-05","7.2e-05","7.6e-05","7.7e-05","8.5e-05","8.6e-05","8.7e-05","8.8e-05","9.5e-05","9.6e-05","9.8e-05","1.0e-04","1.0e-04","1.0e-04","1.1e-04","1.1e-04","1.1e-04","1.1e-04","1.1e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.5e-04","1.5e-04","1.7e-04","1.8e-04","1.9e-04","2.0e-04","2.0e-04","2.0e-04","2.1e-04","2.1e-04","2.3e-04","2.4e-04","2.4e-04","2.4e-04","2.6e-04","2.7e-04","2.8e-04","2.8e-04","2.9e-04","2.9e-04","2.9e-04","2.9e-04","3.0e-04","3.0e-04","3.0e-04","3.0e-04","3.1e-04","3.4e-04","3.4e-04","3.4e-04","3.4e-04","3.4e-04","3.6e-04","3.9e-04","3.9e-04","3.9e-04","3.9e-04","4.0e-04","4.1e-04","4.2e-04","4.3e-04","4.3e-04","4.3e-04","4.4e-04","4.4e-04","4.5e-04","4.5e-04","4.7e-04","5.0e-04","5.0e-04","5.3e-04","5.3e-04","5.4e-04","5.4e-04","5.4e-04","5.5e-04","5.5e-04","5.7e-04","5.8e-04","5.8e-04","6.1e-04","6.2e-04","6.3e-04","6.4e-04","6.9e-04","7.0e-04","7.0e-04","7.1e-04","7.1e-04","7.2e-04","7.3e-04","7.3e-04","7.5e-04","7.5e-04","7.6e-04","7.7e-04","7.9e-04","8.0e-04","8.2e-04","8.4e-04","8.4e-04","8.6e-04","8.6e-04","8.6e-04","9.1e-04","9.1e-04","9.1e-04","9.4e-04","9.4e-04","9.4e-04","9.4e-04","9.6e-04","9.6e-04","9.7e-04","1.0e-03","1.0e-03","1.0e-03","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.6e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.9e-03","1.9e-03","1.9e-03","2.0e-03","2.2e-03","2.2e-03","2.2e-03","2.3e-03","2.4e-03","2.4e-03","2.4e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.6e-03","2.6e-03","2.7e-03","2.7e-03","2.7e-03","2.8e-03","2.8e-03","2.8e-03","2.9e-03","2.9e-03","2.9e-03","2.9e-03","3.0e-03","3.0e-03","3.0e-03","3.1e-03","3.1e-03","3.2e-03","3.3e-03","3.3e-03","3.3e-03","3.4e-03","3.4e-03","3.4e-03","3.4e-03","3.4e-03","3.5e-03","3.5e-03","3.5e-03","3.5e-03","3.6e-03","3.6e-03","3.7e-03","3.7e-03","3.8e-03","3.9e-03","4.0e-03","4.0e-03","4.1e-03","4.4e-03","4.5e-03","4.5e-03","4.6e-03","4.7e-03","4.8e-03","4.8e-03","5.0e-03","5.0e-03","5.3e-03","5.4e-03","5.4e-03","5.7e-03","5.7e-03","5.7e-03","5.7e-03","5.7e-03","5.9e-03","6.3e-03","6.8e-03","7.1e-03","7.2e-03","7.2e-03","7.2e-03","7.3e-03","7.3e-03","7.4e-03","7.4e-03","7.7e-03","7.7e-03","8.0e-03","8.3e-03","8.4e-03","8.5e-03","8.7e-03","8.7e-03","9.2e-03","9.4e-03","9.4e-03","9.4e-03","9.4e-03","9.7e-03","9.7e-03","9.8e-03","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.1e-02","2.2e-02","2.2e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.3e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.7e-02","2.7e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.9e-02","2.9e-02","2.9e-02","2.9e-02","2.9e-02","2.9e-02","2.9e-02","3.0e-02","3.0e-02","3.0e-02","3.0e-02","3.1e-02","3.2e-02","3.2e-02","3.2e-02","3.3e-02","3.3e-02","3.4e-02","3.4e-02","3.4e-02","3.4e-02","3.4e-02","3.5e-02","3.5e-02","3.5e-02","3.6e-02","3.6e-02","3.7e-02","3.7e-02","3.7e-02","3.7e-02","3.8e-02","3.8e-02","3.8e-02","3.8e-02","3.8e-02","3.9e-02","3.9e-02","3.9e-02","4.0e-02","4.0e-02","4.0e-02","4.1e-02","4.1e-02","4.1e-02","4.2e-02","4.2e-02","4.3e-02","4.3e-02","4.4e-02","4.4e-02","4.5e-02","4.5e-02","4.5e-02","4.6e-02","4.6e-02","4.6e-02","4.6e-02","4.8e-02","4.8e-02","4.8e-02","4.8e-02","4.9e-02"],[2178,2263,2265,2265,5541,1951,694,4058,2778,880,3519,12046,1011,13201,13117,3595,538,552,483,558,360,360,14027,488,363,509,2433,2438,500,5214,502,5364,790,14808,459,1010,427,1723,1027,662,14767,1507,420,857,202,384,2216,6938,532,906,671,109,109,3148,85,723,192,75,727,99,2107,473,11951,1135,388,80,757,316,1377,475,475,1341,144,4564,510,217,308,112,431,513,888,566,113,113,1079,838,238,115,116,236,1842,5579,5579,5579,1856,6291,15,2544,178,66,2657,1159,1304,3377,253,1351,6210,2670,18,186,2193,143,1003,6880,67,35,2017,16,1724,58,196,1148,1413,789,193,175,1150,203,573,1242,73,36,36,4453,237,183,321,426,83,3644,1492,9829,171,50,4112,1326,120,151,17,17,617,217,1222,61,149,623,152,53,53,71,2048,66,995,300,5361,55,55,636,3438,110,92,83,130,130,114,824,67,99,136,183,183,775,68,50,59,97,2165,476,339,339,74,186,664,1768,51,60,60,60,60,60,4321,80,163,99,52,52,52,52,52,52,1236,196,1734,364,1754,424,3457,53,85,78,574,54,54,247,1600,1601,1866,55,95,151,56,167,158,6661,549,255,4651,57,57,698,77,67,125,99,89,259,58,59,59,1639,117,101,3104,222,79,59,59,60,154,91,1244,103,87,381,73,112,184,413,123,89,113,105,71,4252,246,1090,120,91,2821,1348,1183,64,64,55,111,98,103,3443,816,65,402,402,482,120,120,1479,203,66,66,66,622,115,128,198,493,68,68,494,1956,93,69,131,145,81,81,52,83,83,185,3114,182,73,73,97,85,95,138,338,74,74,32,186,75,518,76,88,88,696,129,90,77,335,153,169,20,32,195,187,965,148,52,308,94,8,139,2142,217,152,400,166,72,909,85,25,25,22,97,97,177,7607,86,385,23,264,216,113,416,4829,100,171,101,196,269,61,2357,163,162,237,8,102,889,2886,696,296,25,28,204,177,974,124,186,107,1470,123,178,328,213,1846,2368,1626,174,118,33,928,112,112,20,2282,2516,6474,70,196,23,804,101,274,377,198,1081,75,224,300,22,163,370,175,91,179,8,27,136,95,76,260,24,2779,4,356,262,834,2381,7,173,975,745,4116,346,214,189,50,851,315,243,5912,25,82,273,1917,52,395,395,998,252,2289,130,292,533,22,913,1932,5280,131,42,535,1934,308,450,149,5863,2177,353,1240,13,265,85,168,289,380,226,88,138,78,382,563,230,31,450,553,19,261,275,2413,128,10,12,258,2176,566,501,48,673,320,20,329,247,1154,351,292,324,332,544,1132,302,442,326,355,459,1137,367,358,1284,130,202,11,220,18,3226,144,6192,11,5680,171,818,756,410,20,24,400,56,228,523,27,2337,555,12,318,386,5094,993,700,286,582,333,24,5102,453,76,287,639,2234,419,191,335,24,219,250,339,1238,596,31,725,701,104,2038,955,1206,43,43,30,28,210,243,48,243,520,73,5,154,154,28,27,106,12,12,898,28,18,341,1048,675,912,228,188,21,36,229,9487,349,19,60,5,65,344,272,78,268,203,845,128,90,96,1016,97,71,161,517,341,519,72,515,1264,18,612,20,32,112,187,365,152,983,723,723,799,13,132,16,822,3,3,33,33,543,6,190,7873,331,169,102,132,6904,449,588,17,20,57,193,362,292,495,115,780,49,14,657,1867,6178,46,42,13,1764,102,96,21,415,389,307,50,641,896,443,643,380,433,214,197,113,2,681,1178,17070,1098,5,5,47,8,221,341,15,76,956,5,192,2063,80,116,116,26,60,113,64,203,346,1616,1593,34,79,9867,102,595,680,6,310,34,8,487,1995,65,225,354,66,112,17,121,87,25,25,59,364,13,35,43,90,326,68,728,57,68,157,44,60,4,10,2,2,42,815,11915,95,111,343,1001,370,7175,41,875,290,34,348,201,16,82,24,29,29,29,1626,76,6,107,9,183,36,4136,2,30,13,2,47,71,102,1509,227,430,49,12,35,45,212,13,624,110,9,52,13,13,14,5,160,426,107,5,6576,4416,113,59,641,35,238,195,439,196,131,217,169,60,121,6,12,70,1388,6622,553,41,2759,421,15,33,47,549,114,577,12,20,2,35,6447,193,75,720,41,90,90,1033,1502,133,2,498,3,23,87,352,11,102,18,37,2906,10,1375,30,212,157,335,1889,232,46,10941,33,233,233,90,181,1394,252,11397,109,24,504,52,15,617,64,87,236,1098,677,17,55,1551,94,94,94,388,302,190,228,2,11,49,49,4,1244,1244,208,5,50,208,17,1092,104,150,7,96,38,5861,183,2,1120,1066],[157,157,157,157,251,143,79,209,165,90,186,367,93,381,378,162,62,62,58,62,47,47,387,58,47,59,132,132,58,218,58,221,72,396,53,74,44,108,78,76,397,100,43,70,32,49,123,256,55,74,63,27,27,153,23,65,33,21,65,23,105,48,346,82,41,21,54,35,81,43,43,89,26,170,44,32,38,23,42,44,70,46,23,23,78,56,31,23,23,31,106,205,205,205,106,200,9,100,35,18,130,63,81,142,32,85,196,130,9,35,114,28,58,241,17,11,107,9,97,12,29,61,86,56,26,25,69,29,44,63,17,11,11,170,38,25,31,35,21,147,87,245,30,17,159,76,24,22,8,8,44,33,59,18,23,44,22,17,17,19,101,16,67,30,150,17,17,44,125,19,21,17,20,20,19,46,18,21,24,27,27,57,18,16,17,17,100,46,33,33,16,27,44,71,16,17,17,17,17,17,124,19,24,12,16,16,16,16,16,16,70,23,72,31,91,34,110,16,20,16,52,16,16,25,66,66,94,16,17,24,16,21,23,225,50,25,167,16,16,52,18,17,22,17,19,25,16,17,17,66,18,17,131,28,18,16,16,17,13,19,60,20,16,31,16,22,20,36,12,16,22,17,17,152,29,71,22,16,93,66,62,16,16,8,17,19,12,104,47,16,31,31,34,22,22,65,23,16,16,16,28,17,22,19,34,16,16,34,71,16,16,22,22,17,17,15,17,17,20,98,24,16,16,18,17,18,21,32,16,16,10,24,16,33,16,17,17,27,20,16,16,30,18,13,9,9,14,20,44,22,7,31,17,6,19,71,21,21,33,18,14,62,16,7,7,8,8,8,13,198,16,32,8,28,21,18,32,128,17,18,17,19,28,12,75,21,22,22,5,18,42,94,46,26,8,8,19,16,56,14,18,17,60,16,12,30,19,63,77,61,21,18,7,45,17,17,6,73,77,209,12,22,8,41,16,21,27,22,45,13,19,23,7,14,35,17,10,17,5,9,18,7,14,20,7,88,4,29,20,41,79,4,20,45,26,136,24,18,17,10,49,14,19,151,7,12,20,62,10,18,18,45,25,76,17,28,28,8,38,62,137,17,8,28,62,21,19,18,190,67,28,54,4,21,11,20,25,31,22,10,17,12,23,37,22,7,24,31,6,19,21,71,15,5,3,19,66,29,27,8,32,20,6,21,18,51,18,26,20,21,30,50,20,25,20,18,32,50,22,18,51,16,20,5,17,4,89,15,169,4,144,15,42,31,23,5,7,24,12,21,33,7,79,22,4,20,28,140,37,14,19,40,22,6,140,24,10,24,29,76,26,15,22,5,19,19,26,45,40,8,19,32,13,90,45,57,7,7,7,4,12,17,6,21,24,9,4,15,15,6,5,11,5,5,36,4,4,20,38,30,47,20,18,6,8,20,194,20,5,9,4,12,20,20,10,19,19,20,15,11,9,50,9,13,9,14,20,20,9,29,43,5,27,6,6,11,16,26,17,37,36,36,44,5,15,5,33,3,3,6,6,25,4,16,167,24,9,9,13,163,18,26,5,5,7,19,20,18,29,12,41,9,5,36,66,113,9,7,3,63,12,11,5,27,23,23,9,19,40,28,19,20,10,16,14,15,2,28,30,402,46,4,4,4,4,15,22,3,6,35,3,10,76,13,14,14,5,7,13,7,19,13,76,34,5,11,212,13,19,22,3,18,5,4,24,68,10,16,22,7,9,3,12,9,6,5,9,21,4,6,6,8,19,8,22,8,10,16,7,6,3,4,2,2,7,30,270,8,11,18,33,24,166,6,37,22,4,21,17,5,6,5,5,5,5,49,7,3,10,3,17,4,134,2,5,4,2,8,10,8,42,18,20,7,4,5,6,17,3,19,10,3,6,3,3,4,3,13,22,12,3,167,105,11,7,28,7,16,14,22,16,13,11,15,9,6,3,3,4,50,154,18,5,78,25,3,6,4,32,8,31,5,5,2,5,138,12,7,49,6,11,11,34,41,11,2,28,2,5,9,19,4,11,4,6,100,3,42,4,16,13,13,58,17,6,250,3,17,17,5,13,42,16,258,8,5,28,7,4,32,9,10,17,35,31,4,7,31,11,11,11,19,15,12,15,2,3,4,4,2,47,47,16,2,4,16,4,26,2,6,2,11,5,134,15,2,35,34]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/BP-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/MF-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/CC-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/scatter-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="skap2_c_act-1" class="section level3 tabset tabset-fade">
<h3><strong>5.2. SKAP2_C_act</strong></h3>
<div id="gostplot-1" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in SKAP2_C_act were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &lt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_2)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-4929a56e939b13f92b78" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-4929a56e939b13f92b78">{"x":{"data":[{"x":[135.704852227612,82.7493631610453,82.625407199973,81.4121412779627,78.3320234573793,119.23373436998],"y":[5.01951391286474,3.08723791450137,2.31412677563258,2.29966450343271,1.55559363336133,1.49978731900884],"text":["GO:0055114 (475) <br> obsolete oxidation-reduction process <br> 9.561e-06","GO:0006790 (388) <br> sulfur compound metabolic process <br> 8.180e-04","GO:0006749 (64) <br> glutathione metabolic process <br> 4.851e-03","GO:0006399 (191) <br> tRNA metabolic process <br> 5.016e-03","GO:0002943 (6) <br> tRNA dihydrouridine synthesis <br> 2.782e-02","GO:0043603 (1284) <br> cellular amide metabolic process <br> 3.164e-02"],"key":["GO:0055114","GO:0006790","GO:0006749","GO:0006399","GO:0002943","GO:0043603"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[18.1149289122283,17.7792885131994,14.3283099999864,16.5339902938323,3.77952755905512,19.6619312349507],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedDataf94cd961","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[53.64701655001,53.3276749095439],"y":[5.87101187587596,1.87479669619532],"text":["GO:0005829 (5361) <br> cytosol <br> 1.346e-06","GO:0005737 (11951) <br> cytoplasm <br> 1.334e-02"],"key":["GO:0005829","GO:0005737"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[21.6522731484229,22.6771653543307],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedDataf94cd961","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[15.354086581033,3.91953255777999,35.9730967760513,17.0181823209479,46.2544286951195,15.7071904174935,27.1229623217858,15.5644463133924],"y":[4.69694837623215,4.42029474493622,3.18141250055285,2.03227811001643,1.85584827615698,1.83577974392428,1.8229527370364,1.6251970490698],"text":["GO:0016491 (770) <br> oxidoreductase activity <br> 2.009e-05","GO:0003824 (5682) <br> catalytic activity <br> 3.799e-05","GO:0050661 (52) <br> NADP binding <br> 6.585e-04","GO:0017150 (6) <br> tRNA dihydrouridine synthase activity <br> 9.284e-03","GO:0140101 (131) <br> catalytic activity, acting on a tRNA <br> 1.394e-02","GO:0016668 (12) <br> oxidoreductase activity, acting on a sulfur group of donors, NAD(P) as acceptor <br> 1.460e-02","GO:0042802 (2063) <br> identical protein binding <br> 1.503e-02","GO:0016627 (65) <br> oxidoreductase activity, acting on the CH-CH group of donors <br> 2.370e-02"],"key":["GO:0016491","GO:0003824","GO:0050661","GO:0017150","GO:0140101","GO:0016668","GO:0042802","GO:0016627"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[18.8862406459521,21.7285875638015,13.8550340449329,3.77952755905512,15.8190557063088,9.48779874313622,20.3488263412166,14.362799825053],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedDataf94cd961","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"29d43958d25":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"29d44d4a19fc":{"x":{},"xend":{},"y":{},"yend":{}},"29d4360e64e0":{"x":{},"xend":{},"y":{},"yend":{}},"29d46ea53566":{"x":{},"xend":{},"y":{},"yend":{}},"29d441c02d44":{"x":{},"xend":{},"y":{},"yend":{}},"29d4245724f3":{"x":{},"y":{}},"29d44e154a60":{"yintercept":{}}},"cur_data":"29d43958d25","visdat":{"29d43958d25":["function (y) ","x"],"29d44d4a19fc":["function (y) ","x"],"29d4360e64e0":["function (y) ","x"],"29d46ea53566":["function (y) ","x"],"29d441c02d44":["function (y) ","x"],"29d4245724f3":["function (y) ","x"],"29d44e154a60":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedDataf94cd961"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results-1" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-56457827ab1876f7c579" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-56457827ab1876f7c579">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16"],["GO:CC","GO:BP","GO:MF","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:MF","GO:MF","GO:MF","GO:MF","GO:BP","GO:BP"],[470,16857,3556,512,9045,2759,2726,2403,3999,385,11782,3650,6689,3612,1583,12472],["cytosol","obsolete oxidation-reduction process","oxidoreductase activity","catalytic activity","NADP binding","sulfur compound metabolic process","glutathione metabolic process","tRNA metabolic process","tRNA dihydrouridine synthase activity","cytoplasm","catalytic activity, acting on a tRNA","oxidoreductase activity, acting on a sulfur group of donors, NAD(P) as acceptor","identical protein binding","oxidoreductase activity, acting on the CH-CH group of donors","tRNA dihydrouridine synthesis","cellular amide metabolic process"],["1.3e-06","9.6e-06","2.0e-05","3.8e-05","6.6e-04","8.2e-04","4.9e-03","5.0e-03","9.3e-03","1.3e-02","1.4e-02","1.5e-02","1.5e-02","2.4e-02","2.8e-02","3.2e-02"],[5361,475,770,5682,52,388,64,191,6,11951,131,12,2063,65,6,1284],[55,10,11,45,4,8,4,5,2,67,4,3,13,3,2,10]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap-1" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp-1" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/BP-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf-1" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/MF-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc-1" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<pre><code>[1] &quot;NOT enough data.&quot;</code></pre>
</div>
</div>
<div id="scatterplot-1" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/scatter-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="network-analysis" class="section level2">
<h2><strong>6. Network analysis</strong></h2>
<p>Overview from <strong>STRING Database</strong>: In STRING, each protein-protein interaction is annotated with one or more ‘scores’. Importantly, these scores do <strong>not</strong> indicate the strength or the specificity of the interaction. Instead, they are indicators of <strong>confidence</strong>, i.e. how likely STRING judges an interaction to be true, given the available evidence. All scores rank from 0 to 1, with 1 being the highest possible confidence. A score of 0.5 would indicate that roughly every second interaction might be erroneous (i.e., a false positive).</p>
<pre class="r"><code>if(leng_dep != 0){
  
  string_db &lt;- STRINGdb$new(version=&quot;11&quot;, species=9606, score_threshold=400)

  string_mapped &lt;- string_db$map(string_res, &quot;Gene.name&quot;, removeUnmappedRows = TRUE, quiet = T)

  hits &lt;- string_mapped$STRING_id[1:leng_dep]

  string_db$plot_network(hits, add_link = T)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/NSCAP2_act_vs_CSCAP2_act.Rmd/stringdb-1.png" width="1440" style="display: block; margin: auto;" /></p>
<div id="cytoscape-network" class="section level3">
<h3><strong>6.1. Cytoscape Network</strong></h3>
<p>Cytoscape is an open source software platform for visualizing complex networks and integrating these with any type of attribute data. In this presentation all proteins showing at least one confidence level from the STRING database are shown. The <strong>color</strong> code is relative to <strong>fold change</strong> and the <strong>size</strong> is directly proportional to the <strong>significance (p-value)</strong>.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>if(n_edge != 0){
  knitr::include_graphics(paste0(&quot;assets/network_FC_pvalue&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_FC_pvalueSKAP2_N_act_vs_SKAP2_C_act.png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
<div id="cytoscape-enrichmentmap" class="section level3">
<h3><strong>6.2. Cytoscape EnrichmentMap</strong></h3>
<p>An <strong>enrichment map</strong> is a different sort of network. Instead of nodes representing genes, nodes represent pathways or functions. Edges between these pathways or functions represent shared genes or pathway crosstalk. An enrichment map is a way to visualize your enrichment results to help reduce redundancy and uncover main themes.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant) %&gt;% pull(name)

if(!params$Imputation){
  
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  pull(name) %&gt;% 
  unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  
  print(&quot;NOT enough data.&quot;)
  
}else{
  
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  
  if(is.null(GO)){
    
    print(&quot;NOT enough data.&quot;)

    enrich_map = F
    
  }else{
    
    enrich_map = T
    
    gem &lt;- GO$result[,c(&quot;term_id&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;intersection&quot;)]
    colnames(gem) &lt;- c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;Genes&quot;)
    gem$FDR &lt;- gem$p.Val
    gem$Phenotype = &quot;+1&quot;
    gem &lt;- gem[,c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;FDR&quot;, &quot;Phenotype&quot;, &quot;Genes&quot;)]
    head(gem)
  
    write.table(gem, here::here(&quot;output/enrichmentmap.txt&quot;),col.name=TRUE,sep=&quot;\t&quot;,row.names=FALSE,quote=FALSE)
    
  }
}</code></pre>
<pre class="r"><code>if(enrich_map){
  knitr::include_graphics(paste0(&quot;assets/network_enrichmentmap&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_enrichmentmapSKAP2_N_act_vs_SKAP2_C_act.png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="section" class="section level2 unnumbered">
<h2></h2>
<pre class="r"><code>saveWorkbook(excel, here::here(paste0(&quot;output/results_table_&quot;, test, &quot;.xlsx&quot;)), overwrite = T)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] forcats_0.5.1               stringr_1.4.0              
 [3] dplyr_1.0.7                 purrr_0.3.4                
 [5] readr_1.4.0                 tidyr_1.1.3                
 [7] tibble_3.1.2                tidyverse_1.3.1            
 [9] openxlsx_4.2.4              here_1.0.1                 
[11] RColorBrewer_1.1-2          RCy3_2.10.2                
[13] STRINGdb_2.2.2              naniar_0.6.1               
[15] visdat_0.5.3                ggrepel_0.9.1              
[17] ggplot2_3.3.5               UpSetR_1.4.0               
[19] ggstatsplot_0.8.0           datapasta_3.1.0            
[21] patchwork_1.1.1             DT_0.18                    
[23] rrvgo_1.2.0                 org.Hs.eg.db_3.12.0        
[25] AnnotationDbi_1.52.0        gprofiler2_0.2.0           
[27] enrichR_3.0                 preprocessCore_1.52.1      
[29] SummarizedExperiment_1.20.0 Biobase_2.50.0             
[31] GenomicRanges_1.42.0        GenomeInfoDb_1.26.2        
[33] IRanges_2.24.1              S4Vectors_0.28.1           
[35] BiocGenerics_0.36.0         MatrixGenerics_1.2.1       
[37] matrixStats_0.59.0          DEP_1.12.0                 
[39] MBQN_2.2.0                  workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.3            R.methodsS3_1.8.1        
  [3] coda_0.19-4               bit64_4.0.5              
  [5] knitr_1.33                DelayedArray_0.16.2      
  [7] R.utils_2.10.1            data.table_1.14.0        
  [9] RCurl_1.98-1.3            doParallel_1.0.16        
 [11] generics_0.1.0            RSQLite_2.2.7            
 [13] correlation_0.6.1         chron_2.3-56             
 [15] bit_4.0.4                 lubridate_1.7.10         
 [17] xml2_1.3.2                httpuv_1.6.1             
 [19] assertthat_0.2.1          WRS2_1.1-2               
 [21] xfun_0.24                 hms_1.1.0                
 [23] jquerylib_0.1.4           evaluate_0.14            
 [25] promises_1.2.0.1          fansi_0.5.0              
 [27] readxl_1.3.1              caTools_1.18.2           
 [29] dbplyr_2.1.1              igraph_1.2.6             
 [31] DBI_1.1.1                 htmlwidgets_1.5.3        
 [33] reshape_0.8.8             kSamples_1.2-9           
 [35] Rmpfr_0.8-4               paletteer_1.3.0          
 [37] hash_2.2.6.1              ellipsis_0.3.2           
 [39] crosstalk_1.1.1           backports_1.2.1          
 [41] insight_0.14.2            gridBase_0.4-7           
 [43] vctrs_0.3.8               imputeLCMD_2.0           
 [45] Cairo_1.5-12.2            cachem_1.0.5             
 [47] withr_2.4.2               treemap_2.4-2            
 [49] cluster_2.1.0             lazyeval_0.2.2           
 [51] crayon_1.4.1              labeling_0.4.2           
 [53] pkgconfig_2.0.3           SuppDists_1.1-9.5        
 [55] slam_0.1-48               statsExpressions_1.1.0   
 [57] ProtGenerics_1.22.0       wordcloud_2.6            
 [59] rlang_0.4.11              RJSONIO_1.3-1.4          
 [61] lifecycle_1.0.0           MatrixModels_0.5-0       
 [63] sandwich_3.0-1            affyio_1.60.0            
 [65] BiocFileCache_1.14.0      modelr_0.1.8             
 [67] cellranger_1.1.0          rprojroot_2.0.2          
 [69] graph_1.68.0              Matrix_1.2-18            
 [71] mc2d_0.1-21               zoo_1.8-9                
 [73] reprex_2.0.0              whisker_0.4              
 [75] GlobalOptions_0.1.2       pheatmap_1.0.12          
 [77] png_0.1-7                 viridisLite_0.4.0        
 [79] rjson_0.2.20              mzR_2.24.1               
 [81] PMCMRplus_1.9.0           parameters_0.14.0        
 [83] bitops_1.0-7              shinydashboard_0.7.1     
 [85] R.oo_1.24.0               KernSmooth_2.23-17       
 [87] blob_1.2.1                shape_1.4.6              
 [89] multcompView_0.1-8        tmvtnorm_1.4-10          
 [91] ggsignif_0.6.2            scales_1.1.1             
 [93] memoise_2.0.0             magrittr_2.0.1           
 [95] plyr_1.8.6                gplots_3.1.1             
 [97] zlibbioc_1.36.0           compiler_4.0.3           
 [99] plotrix_3.8-1             pcaMethods_1.82.0        
[101] clue_0.3-59               cli_3.0.0                
[103] affy_1.68.0               XVector_0.30.0           
[105] pbapply_1.4-3             MASS_7.3-53              
[107] tidyselect_1.1.1          vsn_3.58.0               
[109] stringi_1.6.2             highr_0.9                
[111] yaml_2.2.1                GOSemSim_2.16.1          
[113] norm_1.0-9.5              MALDIquant_1.19.3        
[115] grid_4.0.3                sass_0.4.0               
[117] tools_4.0.3               rstudioapi_0.13          
[119] circlize_0.4.13           foreach_1.5.1            
[121] git2r_0.28.0              gridExtra_2.3            
[123] ipmisc_6.0.2              pairwiseComparisons_3.1.6
[125] farver_2.1.0              mzID_1.28.0              
[127] digest_0.6.27             BiocManager_1.30.16      
[129] shiny_1.6.0               proto_1.0.0              
[131] BWStest_0.2.2             Rcpp_1.0.6               
[133] broom_0.7.8               BayesFactor_0.9.12-4.2   
[135] performance_0.7.2         later_1.2.0              
[137] ncdf4_1.17                httr_1.4.2               
[139] MSnbase_2.15.7            ComplexHeatmap_2.6.2     
[141] effectsize_0.4.5          colorspace_2.0-2         
[143] rvest_1.0.0               XML_3.99-0.6             
[145] fs_1.5.0                  rematch2_2.1.2           
[147] gmm_1.6-6                 plotly_4.9.4.1           
[149] xtable_1.8-4              gmp_0.6-2                
[151] jsonlite_1.7.2            zeallot_0.1.0            
[153] NLP_0.2-1                 R6_2.5.0                 
[155] tm_0.7-8                  gsubfn_0.7               
[157] pillar_1.6.1              htmltools_0.5.1.1        
[159] mime_0.11                 glue_1.4.2               
[161] fastmap_1.1.0             BiocParallel_1.24.1      
[163] codetools_0.2-16          mvtnorm_1.1-2            
[165] utf8_1.2.1                lattice_0.20-41          
[167] bslib_0.2.5.1             sqldf_0.4-11             
[169] curl_4.3.2                gtools_3.9.2             
[171] zip_2.2.0                 GO.db_3.12.1             
[173] limma_3.46.0              rmarkdown_2.9            
[175] munsell_0.5.0             GetoptLong_1.0.5         
[177] GenomeInfoDbData_1.2.4    iterators_1.0.13         
[179] impute_1.64.0             haven_2.4.1              
[181] gtable_0.3.0              bayestestR_0.10.0        </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
