<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit at IEO" />


<title>LFQ_Analysis</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.18/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.9.4.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Template_LFQ</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LFQ_Analysis</h1>
<h4 class="author">Fabio Bedin | MS-Unit at IEO</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-29
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>SKAP2_HaloTag/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20211221code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20211221)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20211221code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20211221)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong2865ca5"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 2865ca5 </a>
</p>
</div>
<div id="strongRepositoryversionstrong2865ca5" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 2865ca5. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  Counts.pdf
    Untracked:  Intensity_Distributions.pdf
    Untracked:  code/custom_functions.R
    Untracked:  data/expdesign.tsv
    Untracked:  data/proteinGroups.txt
    Untracked:  output/enrichmentmap.txt
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_Halo..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_FC_pvalueSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_Halo_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_FC_pvalueSKAP2_N_vs_Halo.png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_C._vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_C_act_vs_Halo_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_Halo..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_C..png
    Untracked:  output/network_enrichmentmapSKAP2_N._vs_SKAP2_N_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_act_vs_SKAP2_C_act.png
    Untracked:  output/network_enrichmentmapSKAP2_N_vs_Halo.png
    Untracked:  output/output.zip
    Untracked:  output/results_table_Halo._vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_C._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_C._vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_C_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N._vs_Halo..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_C..xlsx
    Untracked:  output/results_table_SKAP2_N._vs_SKAP2_N_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_Halo_act.xlsx
    Untracked:  output/results_table_SKAP2_N_act_vs_SKAP2_C_act.xlsx
    Untracked:  output/results_table_SKAP2_N_vs_Halo.xlsx

Unstaged changes:
    Modified:   analysis/_site.yml
    Deleted:    analysis/about.Rmd
    Deleted:    analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Template_LFQ.Rmd</code>) and HTML (<code>docs/Template_LFQ.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
2865ca5
</td>
<td>
FabioBedin
</td>
<td>
2021-12-29
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>## Proteomics
library(&quot;MBQN&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;enrichR&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)

## Plots
library(&quot;DT&quot;)
library(&quot;patchwork&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)

## Networks
library(&quot;STRINGdb&quot;)
library(&quot;RCy3&quot;)
library(&quot;RColorBrewer&quot;)

## Other
library(&quot;here&quot;)
library(&quot;openxlsx&quot;)
library(&quot;tidyverse&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(paste0(&quot;data/&quot;, params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot; &amp; data$Peptides &gt;= params$n_peptides,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<pre class="r"><code>cond_1 &lt;- params$test_1

cond_2 &lt;- params$test_2

conditions&lt;-c(cond_1,cond_2)

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)

expdesign &lt;- read.table(here::here(paste0(&quot;data/&quot;, params$expdesign)), header = T, stringsAsFactors = F)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
if(max(expdesign$replicate)&lt;=3){
       threshold&lt;-0
     } else if(max(expdesign$replicate)&lt;6){
       threshold&lt;-1
     } else if (max(expdesign$replicate)&gt;=6){
       threshold&lt;-trunc(max(expdesign$replicate)*0.25) ## 0.25 serve per avere sempre il 75% di valid vales
     }

data_filt &lt;- filter_missval(data_se, thr = threshold)</code></pre>
<div id="skap2_n.-vs-halo." class="section level1">
<h1><strong>SKAP2_N.</strong> vs <strong>Halo.</strong></h1>
<hr />
</div>
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>First, we need to wrangling the original dataset. From the <strong>proteinGroups</strong> obtained by <a href="https://www.maxquant.org/">MaxQuant</a> software, I remove proteins that are marked form the software as potential contaminant, only identify by site and reverse.<br />
We che also filter the initial datasets by excluding all proteins identified by less than 2 peptides.<br />
Now, we generate some quality control plots to investigate the structure of the dataset and observe the effect of filters.</p>
<div id="normalization-strategy" class="section level4">
<h4><strong>Normalization strategy:</strong></h4>
<pre class="r"><code>if(params$Normalization == &quot;VSN&quot;){
  data_filt &lt;- normalize_vsn(data_filt)
  print(&quot;VSN normalization is used&quot;)
} else if(params$Normalization == &quot;MBQN&quot;){
  data_filt &lt;- normalize_MBQN(data_filt)
  print(&quot;MBQN normalization is used&quot;)
} else {
  print(&quot;Max-LFQ normalization is used&quot;)
}</code></pre>
<p>[1] “Max-LFQ normalization is used”</p>
</div>
<div id="quality-control-plots" class="section level2 tabset tabset-fade">
<h2><strong>1. Quality control plots</strong></h2>
<div id="proteins-per-samples" class="section level3 tabset">
<h3>Proteins per samples</h3>
<p>This plot show the number of proteins identify in each samples <em>after and before User define</em> filters:</p>
<div id="filtered" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_filt)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/proteins-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_se)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/proteins-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="distributions" class="section level3 tabset">
<h3>Distributions</h3>
<p>This plot is useful for checking the distributions of the samples <em>after and before User define</em> filters:</p>
<div id="filtered-1" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_normalization(data_filt)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/normalization-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-1" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_normalization(data_se)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/normalization-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="missing-data" class="section level3 tabset">
<h3>Missing data</h3>
<p>This plot we can explore the missing data pattern <em>after and before User define</em> filters. Notice that the percentages of missingness are provided in the data. These are accurate to 1 decimal place. Also the dataset is arranged by columns with most missingness.</p>
<div id="filtered-2" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/missing-data-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-2" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>assay(data_se) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/missing-data-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample-cvs" class="section level3 tabset">
<h3>Sample CVs</h3>
<p>The coefficient of variation (CV) is a statistical measure of the dispersion of data points in a data series around the mean. The coefficient of variation represents the ratio of the standard deviation to the mean, and it is a useful statistic for comparing the degree of variation from one data series to another, even if the means are drastically different from one another.</p>
<div id="filtered-3" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_cvs(data_filt)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/CVs-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-3" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_cvs(data_se)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/CVs-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="upset-plot" class="section level2 tabset tabset-fade">
<h2><strong>2. Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within the 2 conditions or in common between all condition.</p>
<div id="skap2_n." class="section level3">
<h3>SKAP2_N.</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_1)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_1)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/upset-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="halo." class="section level3">
<h3>Halo.</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_2)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_2)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/upset-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="both-conditions" class="section level3">
<h3>Both conditions</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()
n_sets &lt;- length(define_set)
n_each &lt;- n_sets / 2

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;red3&quot;, &quot;royalblue1&quot;), each = n_each),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/upset-both-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="imputation" class="section level2">
<h2><strong>3. Imputation</strong></h2>
<p>An idealized version of a label-free discovery mass spectrometry proteomics experiment would provide absolute abundance measurements for a whole proteome, across varying conditions. Unfortunately, this ideal is not realized. Measurements are made on peptides requiring an inferential step to obtain protein level estimates. The inference is complicated by experimental factors that necessitate relative abundance estimation and result in widespread non-ignorable missing data. Relative abundance on the log scale takes the form of parameter contrasts. In a complete-case analysis, contrast estimates may be biased by missing data and a substantial amount of useful information will often go unused.</p>
<pre class="r"><code>data_imputed &lt;- mix_imputation_mean(data_filt, cond1 = cond_1, cond2 = cond_2)

plot_imputation(data_filt, data_imputed)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/imputation-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>data_filt &lt;- data_imputed</code></pre>
</div>
<div id="differential-enrichment-analysis" class="section level2">
<h2><strong>4. Differential enrichment analysis</strong></h2>
<p>Protein-wise linear models combined with empirical Bayes statistics are used for the differential enrichment analysis (or differential expression analysis) and False discovery rate (FDR) is ajusted with <em>Benjamini-Hochberg</em> procedure. Significant proteins are are define with <strong>FDR = 0.05</strong> and <strong>Fold change = 1</strong></p>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results&lt;-get_results(dep)</code></pre>
<p>There are <strong>525</strong> significant proteins in <strong>SKAP2_N.</strong> vs <strong>Halo.</strong> comparison.</p>
<p>The results from the previous analysis can be easily visualized by a number of functions. These visualizations assist in the determination of the optimal cutoffs to be used, highlight the most interesting samples and contrasts, and pinpoint differentially enriched/expressed proteins.</p>
<div id="visualization-of-the-results" class="section level3 tabset tabset-fade">
<h3><strong>4.1. Visualization of the results</strong></h3>
<div id="pca" class="section level4">
<h4>PCA</h4>
<p>The PCA plot can be used to get a high-level overview of the data. This can be very useful to observe batch effects, such as clear differences between replicates.</p>
<pre class="r"><code>if(params$Imputation){
  if(nrow(dep) &gt;= 500){
    n_value &lt;- 500
  }else {
    n_value &lt;- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
} else {
  var &lt;- apply(assay(dep), 1, sd)
  df &lt;- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value &lt;- df %&gt;% as.data.frame() %&gt;% drop_na() %&gt;% nrow()
  if(n_value &gt;= 500){
    n_value &lt;- 500
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/PCA-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano" class="section level4">
<h4>Volcano</h4>
<p>Volcano plots allows to inspect the enrichment of proteins between the two samples (x axis) and their corresponding adjusted p value (y axis).</p>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T, label_size=5, adjusted = F)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/volcano-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-and-unique" class="section level4">
<h4>Volcano and unique</h4>
<pre class="r"><code>plot_volcano_2(dep, contrast=test, add_names=T, label_size=5, adjusted = F, unique_vec = &quot;SKAP2&quot;)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/volcano-unique-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="correlation" class="section level4">
<h4>Correlation</h4>
<pre class="r"><code>plot_cor(dep, significant = TRUE, lower = 0, upper = 1, pal = &quot;Reds&quot;)</code></pre>
<p><img src="figure/Template_LFQ.Rmd/correlation-matrix-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-heatmap" class="section level4">
<h4>Cluster heatmap</h4>
<p>The heatmap representation gives an overview of all significant proteins (rows) in all samples (columns). This allows to see general trends, for example if one sample or replicate is really different compared to the others. Additionally, the clustering of samples (columns) can indicate closer related samples and clustering of proteins (rows) indicates similarly behaving proteins.</p>
<pre class="r"><code>if(results %&gt;% filter(significant) %&gt;% nrow() &lt;= 100){
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = T, indicate = &quot;condition&quot;, col_limit = 5)
}else {
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = F, indicate = &quot;condition&quot;, col_limit = 5)
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/cluster-heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="result-table" class="section level3">
<h3><strong>4.2. Result table</strong></h3>
<p>In this table are summarized the results of DEP analysis, sorted by p.value. Proteins that are UP regulated are colored with a red box in Fold change column, in the orhter hand proteins that are DOWN regulated are colored with blue.</p>
<pre class="r"><code>results %&gt;% 
  select(starts_with(&quot;name&quot;) | starts_with(test)) %&gt;% 
  filter(across(ends_with(&quot;significant&quot;))) %&gt;% 
  arrange(across(ends_with(&quot;p.adj&quot;))) %&gt;%
  mutate(across(2:3, format, scientific = T, digits = 2)) %&gt;%
  dplyr::rename_with(~ tolower(gsub(paste0(test,&quot;_&quot;), &quot;&quot;, .x)), .cols = starts_with(test)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(Fold.Change = ratio, Gene.name = name) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:5)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;Fold.Change&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-d1ba32e0541469b5ed49" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-d1ba32e0541469b5ed49">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525"],["ZNF112","CSNK2B-LY6G5B-1181","CYBRD1","RCN2","RGS19","SVIP","FYB","ACADM","H1FX","CTDSP1","EIF3K","A8K968","DNAJA3","RAP1A","SASS6","C1orf198","AATF","hCG_2004507","IMP3","MFGE8","ATP6V1B2","EPB41L5","FARP2","KRAS","PWP1","ACTR1A","LAMTOR3","RPS6KA4","SIT1","DDX10","MORF4L1","PWP2","SELPLG","U2AF1","COMMD2","ANKRD44","CAST","GNAQ","PYCRL","HLA-C","ITGB1","LRRFIP2","NAPG","FYN","COPZ1","hCG_1994888","RRAS","USP19","UTP11L","RCBTB2","CCNY","MELK","UTP14A","ARAP1","ASF1B","PDIA3","WDR5","CNOT2","CC2D1A","MRPS17","ARPC5L","SLC25A10","WDR43","CSNK1G2","DYNC1LI1","TRAFD1","UBR2","GOLGA7","MLF2","TBCE","MFAP1","TMOD3","THY1","CPSF6","GRK6","RBM12B","DDX52","SPAG9","HRAS","PTPRJ","STXBP6","MKRN2","GZMA","KPNA3","MRPS35","PLXNA1","KIAA1429","MFI2","RALB","ABHD12","RPP40","HTATSF1","SFSWAP","GABRA2","CHTOP","GPC4","SPN","KLC2","PTPRA","PAFAH1B1","SIRT6","CROCC","PARG","TOMM40","GYPC","LCK","PIK3CD","RHOG","STX4","PIK3R4","CLTB","ERLIN1","KDM3A","SEMA7A","DOCK7","MRPS14","LAT","PPT1","RSAD1","PDE4DIP","YES1","LAMTOR2","GNG2","GPX1","IL2RG","IGF2R","PPAT","SLK","YME1L1","C12orf75","PTPRG","HAGH","PRRC2A","UTP3","ALDH5A1","MKI67","GNG4","IGF1R","PXN","PI4KA","TMCO1","BST2","LGALS3","RAP2B","SDHB","APIP","EPB41L2","PFKM","ECH1","DOCK9","MYO1C","SLC38A1","LGALS8","MAEA","MYH10","ZDHHC5","CD82","ACSL1","GNAI2","ALDH18A1","SCRIB","FLOT2","EPB41L3","RHOA","EIF3B","EIF3G","HIST2H2AC","FHOD1","GNB2L1","CALM2","SRPR","WDR74","DPP3","CLTA","NHP2L1","DAAM1","SNAP23","RRAS2","RPS16","CLDND1","GNB1","FLOT1","DOCK8","MYL12A","ABCC1","PARP12","RPS12","RAP2C","SKAP2","UGDH","POLR2C","RFTN1","RP2","SPTBN1","HIST1H4A","CD48","MYH9","GNAO1","MYL6","CLPX","C1orf21","ALCAM","GNAS","RALA","DPP4","THUMPD1","GNA13","EFR3A","INPP5A","OSBP","CD6","SAP18","PRKAR2A","SVIL","PDCD6","GNAI3","ARL6IP4","RAD21","ANKHD1","GNB2","ULBP2","RGS1","BAG2","H2AFV","UBA2","ICAM3","CD44","RBM22","SF3B6","FAM133B","USP16","PRNP","GNG5","TUFM","RRP12","KTN1","GATAD2B","NOL6","TMEM43","YTHDF1","IK","DLST","HNRNPM.1","GLIPR2","LIMA1","RAB35","EPB41","CBX3","B2M","KDM2A","PCM1","KIFC1","BCL3","SLC25A5","CD70","TOP2B","UQCRC2","RPL21","RAB6A","TRMT1L","TSN","CPSF7","RPS20","CAPNS1","MYO1G","PHF5A","ISOC2","PPP1R9B","EMC2","RPL35A","RCL1","PANK4","CDV3","H2AFY","PPP1CA","GNB2L1.1","PSMD5","ACTN4","CCM2","LACTB","RRP7A","FLAD1","NCBP1","ADRM1","TWF2","DDX56","DBT","LMNB2","U2SURP","NOP58","DDX39A.1","RAP1B.1","IFITM2","RP9","CD2BP2","IPO7","QARS","Pe1Fe10","PAG1","COPB2","CWC15","ERBB2IP","FMNL1.1","NOSIP","AP1M1","NOP16","RPL14","NUPL1","MARCKSL1","ARPC2","TRBC1","TCF25","ACAP1","BYSL","MAPK9","HNRNPC","KARS","RNPS1","PNPT1","ITGAL","API5","SFN","LAMTOR1","AP1B1","CDC20","SRP9","SDHA","SYNGR2","GIGYF2","PTMA","PPP3CA","PPP1R16B","DDOST","NADK2","PGRMC2","CALCOCO2","AKR1A1","DDX49","PARN","WDR36","ACTN1","CSNK1D","ITGB2","HARS","ARF5","TIGAR","CHUK","RPL27","PSMA6","PSMC1","DKC1","POGZ","NRAS","PPIP5K2","DHX36","STOM","RFK","PGAM1","NOP56","KRR1","EIF5A","PRKAA1","BANF1","HLA-DRA","HLA-B","VIM","RAC2","ITM2A","KIAA0101","CAPRIN1","CCDC84","ESF1","SAFB","BAZ1B","TIPRL","LMNB1","YRDC","HIST2H3PS2","CBX5","EBNA1BP2","ESYT1","CD247","NAP1L1","PSMD7","RRS1","UFM1","HLA-DRB1","TOP2A","SPTAN1","EIF3L","RPL7","RAB11B","KDM4B","CLTC","HNRNPK.1","NOP14","ADAR","CPNE7","KIAA0020","GEMIN4","RALY","DBN1","LIN7C","SET","SERPINB9","PSMA5","SRBD1","EIF3M","CPPED1","EIF3CL","SLC3A2","STXBP2","METAP1D","WDR3","CORO2A","DNTTIP2","C9orf142","COX4I1","VDAC3","RRP9","RPL7L1","UNC45A","HLA-A","PTBP3","STMN2","BCAP31","LRCH4","HIST1H2BN","DDX39B","UBE2L3","HSPA4","SLC16A1","DDX18","EZR","HIST1H1B","NOP2","HLA-C.1","EXOSC4","RSL1D1","WDR12","ENO1","EIF4B","DYNC1H1","GTPBP4","UPF2","ZW10","PTPN23","RPL5","WDR75","TRIM65","GRAPL","TALDO1","POLR2A","RRP1","EIF3F","BSG","SEC61B","RAB7A","NIFK","FTSJ3","PLA2G4A","PRKCSH","RASAL3","MARCKS","UTP6","BCLAF1","NME1","PNN","HIST1H2AA","HLA-G","ADO","KIN","VCP","MTCH2","THOC2","ESYT2","UTP18","FKBP4","RAB27A","AFG3L2","HSP90AB2P","RBMX","S100A9","GRPEL1","SCAF11","CHAC2","ZNF326","LRBA","MRPS6","RPS7","EPDR1","STX11","APMAP","DSP","PNP","NARF","HMGCL","FBL","GK","HEATR1","IQGAP1","NAA10","PTPRC","DCTN1","KIF22","PSMD1","UGP2","PAK2","DSG1","LAMP1","RAB1B","APEX1","PPIB","ATP1A1","TGDS","DSC1","GOT2","NGDN","PSMA3","FUBP1","SON","GNL2","PDCD11","CD47","WDR1"],["2.1e-12","8.5e-12","1.3e-11","6.6e-12","1.1e-11","1.2e-11","1.7e-11","2.1e-11","2.2e-11","2.7e-11","3.4e-11","7.0e-11","6.1e-11","7.2e-11","6.2e-11","7.7e-11","9.7e-11","1.0e-10","9.4e-11","1.1e-10","1.2e-10","1.4e-10","1.7e-10","1.7e-10","1.7e-10","1.9e-10","1.9e-10","2.8e-10","2.8e-10","3.8e-10","3.7e-10","3.6e-10","3.6e-10","3.3e-10","5.0e-10","6.1e-10","6.0e-10","6.2e-10","5.9e-10","6.4e-10","7.3e-10","7.4e-10","7.7e-10","8.2e-10","1.0e-09","1.0e-09","1.0e-09","1.0e-09","1.0e-09","1.1e-09","1.1e-09","1.1e-09","1.1e-09","1.4e-09","1.4e-09","1.4e-09","1.5e-09","1.8e-09","1.9e-09","1.9e-09","2.0e-09","2.0e-09","2.2e-09","2.7e-09","3.3e-09","3.6e-09","4.2e-09","4.3e-09","4.3e-09","4.5e-09","4.9e-09","6.1e-09","6.2e-09","6.7e-09","8.3e-09","8.8e-09","1.1e-08","1.2e-08","1.4e-08","1.4e-08","1.7e-08","1.9e-08","2.0e-08","2.1e-08","2.1e-08","2.2e-08","2.4e-08","2.4e-08","2.4e-08","2.6e-08","3.1e-08","3.2e-08","3.2e-08","3.3e-08","3.5e-08","3.8e-08","4.0e-08","4.6e-08","5.0e-08","5.4e-08","5.3e-08","6.2e-08","6.3e-08","6.3e-08","6.5e-08","6.7e-08","6.7e-08","6.9e-08","8.1e-08","8.4e-08","8.8e-08","9.3e-08","1.2e-07","1.4e-07","1.4e-07","1.4e-07","1.5e-07","1.6e-07","1.6e-07","1.7e-07","1.8e-07","2.0e-07","2.1e-07","2.2e-07","2.2e-07","2.5e-07","2.5e-07","2.9e-07","2.9e-07","3.1e-07","3.2e-07","3.5e-07","3.7e-07","3.9e-07","4.3e-07","4.5e-07","5.2e-07","5.3e-07","5.4e-07","6.1e-07","6.1e-07","6.3e-07","6.5e-07","6.5e-07","7.7e-07","7.9e-07","7.9e-07","8.4e-07","8.5e-07","9.0e-07","9.5e-07","1.3e-06","1.4e-06","1.6e-06","1.7e-06","1.8e-06","2.0e-06","2.1e-06","2.5e-06","2.5e-06","2.6e-06","2.6e-06","3.4e-06","3.7e-06","3.9e-06","3.9e-06","4.0e-06","4.1e-06","4.3e-06","4.9e-06","5.9e-06","6.0e-06","6.1e-06","6.5e-06","6.6e-06","7.0e-06","7.1e-06","7.1e-06","7.4e-06","7.5e-06","7.7e-06","7.9e-06","8.0e-06","8.5e-06","8.6e-06","9.4e-06","9.3e-06","1.1e-05","1.1e-05","1.2e-05","1.2e-05","1.5e-05","1.5e-05","1.6e-05","1.6e-05","1.7e-05","1.7e-05","1.8e-05","1.8e-05","1.8e-05","1.9e-05","1.9e-05","1.9e-05","2.0e-05","2.0e-05","2.0e-05","2.1e-05","2.2e-05","2.5e-05","2.5e-05","2.6e-05","2.7e-05","2.7e-05","2.7e-05","2.8e-05","2.9e-05","2.9e-05","3.0e-05","3.0e-05","3.1e-05","3.1e-05","3.2e-05","3.3e-05","3.4e-05","3.5e-05","3.5e-05","3.6e-05","3.6e-05","3.7e-05","3.8e-05","3.8e-05","3.9e-05","4.0e-05","4.0e-05","4.1e-05","4.2e-05","4.7e-05","4.7e-05","4.9e-05","5.0e-05","5.1e-05","5.4e-05","5.6e-05","5.8e-05","5.8e-05","5.9e-05","7.0e-05","7.1e-05","7.5e-05","7.5e-05","7.4e-05","7.5e-05","7.6e-05","7.7e-05","7.8e-05","7.8e-05","7.8e-05","8.0e-05","8.5e-05","8.5e-05","8.5e-05","9.0e-05","9.3e-05","9.5e-05","9.5e-05","9.4e-05","9.7e-05","1.0e-04","1.0e-04","1.0e-04","1.1e-04","1.1e-04","1.1e-04","1.2e-04","1.2e-04","1.2e-04","1.2e-04","1.3e-04","1.3e-04","1.3e-04","1.4e-04","1.4e-04","1.4e-04","1.5e-04","1.5e-04","1.6e-04","1.6e-04","1.7e-04","1.7e-04","1.7e-04","1.8e-04","1.9e-04","1.9e-04","2.1e-04","2.1e-04","2.1e-04","2.1e-04","2.1e-04","2.3e-04","2.5e-04","2.5e-04","2.5e-04","2.5e-04","2.5e-04","2.6e-04","2.6e-04","2.7e-04","2.7e-04","2.8e-04","2.8e-04","2.9e-04","2.9e-04","3.0e-04","3.2e-04","3.2e-04","3.2e-04","3.4e-04","3.4e-04","3.4e-04","3.6e-04","3.6e-04","3.6e-04","3.9e-04","4.1e-04","4.1e-04","4.3e-04","4.6e-04","4.6e-04","4.7e-04","4.7e-04","4.9e-04","5.2e-04","5.5e-04","5.8e-04","5.8e-04","5.9e-04","6.1e-04","6.1e-04","6.2e-04","6.3e-04","6.6e-04","6.6e-04","6.6e-04","6.7e-04","6.8e-04","7.0e-04","7.1e-04","7.2e-04","7.3e-04","7.4e-04","7.5e-04","7.5e-04","7.6e-04","7.7e-04","7.7e-04","7.7e-04","7.8e-04","7.9e-04","8.0e-04","8.6e-04","8.7e-04","8.8e-04","9.1e-04","9.2e-04","9.3e-04","9.6e-04","9.7e-04","9.8e-04","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.9e-03","1.9e-03","1.9e-03","2.0e-03","2.0e-03","2.1e-03","2.0e-03","2.1e-03","2.1e-03","2.1e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.3e-03","2.4e-03","2.4e-03","2.5e-03","2.5e-03","2.6e-03","2.6e-03","2.6e-03","2.6e-03","2.7e-03","2.7e-03","2.8e-03","2.8e-03","2.9e-03","3.0e-03","3.0e-03","3.1e-03","3.1e-03","3.2e-03","3.3e-03","3.3e-03","3.5e-03","3.5e-03","3.6e-03","3.8e-03","3.8e-03","4.0e-03","4.1e-03","4.7e-03","4.8e-03","4.9e-03","5.0e-03","5.2e-03","5.2e-03","5.2e-03","5.3e-03","5.5e-03","5.5e-03","5.6e-03","5.6e-03","5.6e-03","5.9e-03","5.9e-03","6.1e-03","6.2e-03","6.4e-03","6.8e-03","6.9e-03","7.0e-03","7.0e-03","7.0e-03","7.0e-03","7.1e-03","7.1e-03","7.2e-03","7.2e-03","7.2e-03","7.5e-03","7.4e-03","7.6e-03","7.6e-03","7.6e-03","7.9e-03","7.9e-03","8.0e-03","8.2e-03","8.3e-03","8.7e-03","8.8e-03","8.8e-03","8.8e-03","8.9e-03","9.0e-03","9.1e-03","9.3e-03","9.3e-03","9.4e-03","9.7e-03","9.6e-03","9.7e-03","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02"],["4.9e-09","5.0e-09","5.0e-09","5.0e-09","5.0e-09","5.0e-09","5.8e-09","5.8e-09","5.8e-09","6.3e-09","7.2e-09","1.1e-08","1.1e-08","1.1e-08","1.1e-08","1.1e-08","1.3e-08","1.3e-08","1.3e-08","1.3e-08","1.4e-08","1.5e-08","1.6e-08","1.6e-08","1.6e-08","1.7e-08","1.7e-08","2.3e-08","2.3e-08","2.6e-08","2.6e-08","2.6e-08","2.6e-08","2.6e-08","3.4e-08","3.7e-08","3.7e-08","3.7e-08","3.7e-08","3.8e-08","4.1e-08","4.1e-08","4.2e-08","4.4e-08","5.0e-08","5.0e-08","5.0e-08","5.0e-08","5.0e-08","5.1e-08","5.1e-08","5.1e-08","5.1e-08","5.9e-08","5.9e-08","6.1e-08","6.3e-08","7.3e-08","7.4e-08","7.6e-08","7.7e-08","7.7e-08","8.4e-08","1.0e-07","1.2e-07","1.3e-07","1.5e-07","1.5e-07","1.5e-07","1.5e-07","1.6e-07","2.0e-07","2.0e-07","2.2e-07","2.6e-07","2.7e-07","3.3e-07","3.6e-07","4.2e-07","4.2e-07","4.8e-07","5.4e-07","5.7e-07","5.9e-07","5.9e-07","6.0e-07","6.4e-07","6.4e-07","6.4e-07","6.7e-07","8.1e-07","8.2e-07","8.2e-07","8.3e-07","8.6e-07","9.4e-07","9.7e-07","1.1e-06","1.2e-06","1.3e-06","1.3e-06","1.4e-06","1.4e-06","1.4e-06","1.4e-06","1.5e-06","1.5e-06","1.5e-06","1.7e-06","1.8e-06","1.9e-06","1.9e-06","2.5e-06","2.8e-06","2.8e-06","2.9e-06","3.0e-06","3.2e-06","3.2e-06","3.4e-06","3.4e-06","3.9e-06","3.9e-06","4.2e-06","4.2e-06","4.6e-06","4.7e-06","5.2e-06","5.2e-06","5.6e-06","5.8e-06","6.3e-06","6.5e-06","6.8e-06","7.4e-06","7.8e-06","9.0e-06","9.0e-06","9.1e-06","1.0e-05","1.0e-05","1.0e-05","1.1e-05","1.1e-05","1.3e-05","1.3e-05","1.3e-05","1.3e-05","1.3e-05","1.4e-05","1.5e-05","2.1e-05","2.2e-05","2.5e-05","2.5e-05","2.7e-05","3.0e-05","3.2e-05","3.6e-05","3.7e-05","3.7e-05","3.8e-05","5.0e-05","5.3e-05","5.5e-05","5.6e-05","5.6e-05","5.8e-05","6.0e-05","6.8e-05","8.1e-05","8.2e-05","8.3e-05","8.8e-05","8.9e-05","9.4e-05","9.5e-05","9.5e-05","9.7e-05","9.8e-05","1.0e-04","1.0e-04","1.0e-04","1.1e-04","1.1e-04","1.2e-04","1.2e-04","1.3e-04","1.4e-04","1.4e-04","1.5e-04","1.8e-04","1.8e-04","1.9e-04","2.0e-04","2.1e-04","2.1e-04","2.1e-04","2.1e-04","2.1e-04","2.2e-04","2.2e-04","2.2e-04","2.3e-04","2.3e-04","2.3e-04","2.4e-04","2.5e-04","2.8e-04","2.8e-04","2.9e-04","3.0e-04","3.0e-04","3.0e-04","3.0e-04","3.1e-04","3.1e-04","3.2e-04","3.3e-04","3.3e-04","3.3e-04","3.4e-04","3.4e-04","3.5e-04","3.6e-04","3.7e-04","3.7e-04","3.7e-04","3.8e-04","3.9e-04","3.9e-04","4.0e-04","4.0e-04","4.0e-04","4.1e-04","4.2e-04","4.6e-04","4.6e-04","4.8e-04","4.9e-04","4.9e-04","5.2e-04","5.4e-04","5.6e-04","5.6e-04","5.6e-04","6.6e-04","6.7e-04","6.9e-04","6.9e-04","6.9e-04","7.0e-04","7.0e-04","7.1e-04","7.1e-04","7.1e-04","7.1e-04","7.2e-04","7.6e-04","7.6e-04","7.6e-04","7.9e-04","8.2e-04","8.3e-04","8.3e-04","8.3e-04","8.4e-04","8.8e-04","8.9e-04","8.9e-04","9.0e-04","9.2e-04","9.5e-04","9.8e-04","1.0e-03","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.6e-03","1.6e-03","1.7e-03","1.8e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","2.0e-03","2.0e-03","2.0e-03","2.0e-03","2.1e-03","2.1e-03","2.1e-03","2.2e-03","2.2e-03","2.3e-03","2.3e-03","2.4e-03","2.4e-03","2.4e-03","2.4e-03","2.5e-03","2.5e-03","2.6e-03","2.7e-03","2.9e-03","2.9e-03","3.0e-03","3.2e-03","3.2e-03","3.2e-03","3.2e-03","3.3e-03","3.5e-03","3.6e-03","3.8e-03","3.8e-03","3.8e-03","3.9e-03","4.0e-03","4.0e-03","4.1e-03","4.2e-03","4.2e-03","4.2e-03","4.2e-03","4.2e-03","4.4e-03","4.4e-03","4.5e-03","4.5e-03","4.5e-03","4.6e-03","4.6e-03","4.6e-03","4.7e-03","4.7e-03","4.7e-03","4.7e-03","4.7e-03","4.8e-03","5.1e-03","5.1e-03","5.2e-03","5.3e-03","5.4e-03","5.4e-03","5.6e-03","5.6e-03","5.7e-03","5.9e-03","6.1e-03","6.1e-03","6.4e-03","6.4e-03","6.4e-03","7.2e-03","7.4e-03","7.4e-03","7.5e-03","7.6e-03","8.0e-03","8.0e-03","8.1e-03","8.1e-03","8.1e-03","8.1e-03","8.4e-03","8.5e-03","8.5e-03","8.7e-03","8.7e-03","8.8e-03","8.9e-03","9.0e-03","9.2e-03","9.2e-03","9.2e-03","9.4e-03","9.6e-03","9.6e-03","9.6e-03","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.8e-02","2.0e-02","2.0e-02","2.0e-02","2.1e-02","2.1e-02","2.1e-02","2.1e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.3e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.6e-02","2.6e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.8e-02","2.8e-02","2.8e-02","2.9e-02","2.9e-02","2.9e-02","2.9e-02","3.0e-02","3.1e-02","3.1e-02","3.1e-02","3.1e-02","3.1e-02","3.1e-02","3.1e-02","3.2e-02","3.2e-02","3.2e-02","3.3e-02","3.3e-02","3.3e-02","3.4e-02","3.5e-02","3.5e-02","3.5e-02","3.6e-02","3.6e-02","3.6e-02","3.7e-02","3.7e-02","3.7e-02","3.8e-02","3.8e-02","3.8e-02","3.8e-02","3.9e-02","4.0e-02","4.0e-02","4.1e-02","4.1e-02","4.1e-02","4.1e-02","4.2e-02","4.2e-02","4.3e-02","4.4e-02","4.4e-02","4.4e-02","4.5e-02","4.5e-02","4.5e-02","4.7e-02","4.9e-02","5.0e-02","5.0e-02"],["True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True"],[-8.21,5.41,4.95,-4.79,4.46,5.11,11.3,4.19,4.51,4.6,4.58,4.97,3.47,5.23,2.98,4.6,-3.24,3.51,-4.8,3.09,3.91,3.69,2.87,3.41,3.24,3.13,5.21,3.23,3.34,-3.53,-3.22,-2.97,4.13,-4.53,3.09,3.89,2.91,4.38,3.43,4.35,3.61,3.71,2.44,4.58,3.12,4.94,2.98,2.72,-3.73,3.63,4.47,-4.5,-4.88,2.46,-2.99,4.93,4.24,2.95,2.19,3.65,3.56,2.39,-3.67,5.84,3.32,1.99,3.41,3.99,3.01,3.04,-2.66,4.81,7.4,3.34,4.11,2.4,-3.09,2.68,3.66,2.9,2.19,2.81,3.62,3.56,2.73,2.28,1.98,2.75,3.67,2.54,2.87,1.99,1.95,4.73,1.85,1.8,4.41,3.19,3.21,2.85,2.41,6.15,3.25,4.78,4.92,7.08,3.45,4.2,3.4,1.97,2.7,2.84,2.24,3.12,1.98,4.4,5.24,2.6,-2.41,2.06,5.24,3.18,5.61,-1.8,1.36,2.38,2.31,2.18,9.98,5.57,4.55,2.43,1.32,-3.28,2.76,-2.69,5.78,1.69,-2.74,2.72,2.25,4.71,3.72,4.52,-2.12,2.63,9.17,3.56,3.4,2.25,2.92,3.54,4.89,2.02,4,3.39,3.75,-2.79,4.1,1.25,4.86,5.19,6.26,-5.2,3.38,1.26,-3.78,1.67,-1.39,3.79,-2.75,-2.1,1.85,3.53,-2.44,6.98,6.67,6.52,-1.08,3.58,3.81,4.85,1.44,1.6,3.7,1.34,-1.44,4.37,4.32,-1.05,-1.61,5.25,4.55,3.26,-2.8,4,1.22,4.3,1.2,2.07,3.08,1.88,2.12,4.15,5.52,-1.17,6.14,5.48,4.08,2.61,3.97,-1.87,1.94,2.97,2.01,5.22,-1.33,-1.47,1.72,6.02,3.74,-2.9,1.29,-2.94,2.59,5.43,4.3,-1.58,-1.19,-1.23,-1.32,3.86,5.48,-1.15,-1.98,2.57,1.32,-1.88,3.02,-1.17,-1.47,2.2,-1.38,3.65,1.08,3.23,6.46,-1.71,3.16,-1.86,3.49,-1.47,-1.08,-1.16,1.35,-2.15,3.52,1.56,1.62,-1.01,1.78,2.39,-1.83,-1.58,4.82,-1.47,-2.49,3.82,1.31,1.13,-2.32,-1.36,3.13,-3.95,1.06,-1.23,1.53,2.24,2.98,3.12,-1.4,-1.08,-1.09,-1.4,1.02,-3.12,-1.74,-1.45,1.09,-2.01,-1.04,4.17,3.35,-2.84,1.12,1.02,-6.17,-1.15,4.9,3.03,-1.25,3.35,4.34,-1.27,1.58,-2.37,1.19,-1.33,3.17,1.03,-1.18,1.6,1.63,-2.01,-1.23,-2.17,1.47,-1.66,1.6,3.42,3.12,-1.26,2.91,1.86,-1.37,-1.24,-1.43,1.84,1.61,4.24,1.19,2.11,1.98,-1.32,-1.23,4.57,-1.33,-1.39,-1.69,-2.87,1.1,-1.11,3.3,3.8,-1.18,-1.07,2.06,1.11,2.83,1.2,-1.2,-1.02,1.98,4.59,-1.08,3.67,-2.06,2.66,-1.58,-2.06,1.29,1.28,-2.98,2.28,2.71,-2.03,1.11,1.12,-1.72,1.27,-1.04,-1.22,-1.2,-2.72,-1,-1.56,-1.17,-1.69,-3.22,-1.93,1.2,-1.67,1.24,1.51,-1.52,-1.03,2.53,-3.78,1.43,2.94,1.06,1.87,-1.02,1.7,-1.27,-1.57,-1.12,-1.05,-2.15,1.15,-3.3,2.75,1.34,2.9,-1.74,1.71,-1.27,1.25,-1.19,1.59,1.69,4.26,-1.27,-1.59,-1.02,-2.57,-1.24,1.71,1.38,-1.86,-2.67,1.13,1.29,-1.24,2.52,1.45,1.39,-2.07,-1.16,1.36,3.3,3.9,-1.48,3,-1,-1.53,3.49,1.07,-2.07,-2.29,-2.15,1.73,1.49,-2.43,-1.06,-1.88,1.3,1.07,-1.73,1.11,-1.29,2.3,-2.26,-2.3,1.48,2.58,-1.48,1.33,-2.28,-1.99,1.35,2.95,1.13,1.22,-2.12,-1.42,1.29,-1.6,-1.25,1.06,-1.03,-1.02,1.71,1.72,1.39,1.3,-1.22,1.91,1.06,-1.29,1.82,-1.61,-3.2,1.15,-1.26,-1.05,-2.2,1.08,-1.16,1.66,1.09,2.12,1.9,-3.17,2.56,-1.07,-1.74,-1.08,1.6,-3.12,1.21,1.01,3.32,1.2,-1.16,2.28,2.24,1.24,-3.02,1.78,1.21,1.63,1.16,1.85,-1.15,-2.82,2.19,-2.1,1.31,1.24,-1.64,-1.08,-3.47,1.53,1.28]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene.name<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>Fold.Change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[5]; $(this.api().cell(row, 5).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[4]; $(this.api().cell(row, 4).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
<div id="gene-ontology" class="section level2">
<h2><strong>5. Gene Ontology</strong></h2>
<p>Now I perfom a gene onology analysis (GO or by branch <a href="GO:MF" class="uri">GO:MF</a>, <a href="GO:BP" class="uri">GO:BP</a>, <a href="GO:CC" class="uri">GO:CC</a>) and a REAC ontology.<br />
The enrichment results are visualized with an interactive <strong>Manhattan-like-plot</strong> named “Gostplot”.</p>
<div id="skap2_n.-1" class="section level3 tabset tabset-fade">
<h3><strong>5.1. SKAP2_N.</strong></h3>
<div id="gostplot" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in SKAP2_N. were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &gt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_1)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-06484ed42468d86de4a4" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-06484ed42468d86de4a4">{"x":{"data":[{"x":[98.7434383806114,74.4931449053838,132.658540578231,94.0293556186209,126.952810127662,76.641714897303,98.7509508631006,159.185116247694,126.795047995388,106.793063367819,122.189896229492,118.467461156079,83.0648874255928,132.688590508187,75.9130040958479,76.5140026949861,131.336343660126,121.750416003872,76.2848719790647,131.501618274889,118.516292292259,113.629422433016,82.8244879859375,75.9430540258048,76.5215151774754,76.2548220491078,115.87941093854,75.8679292009126,93.6424627704257,75.9092478546033,130.295864835368,76.5102464537415,85.690500055578,75.8266105472218,146.81581382918,109.982112184496,144.051220273145,107.059756496187,83.2827494177805,77.6897062045503,77.6746812395719,93.6650002178934,132.75620285059,77.6483875508596,76.4576590763169,74.3804576680454,76.6454711385476,130.078002843181,83.9250666706094,119.969957653925,116.964964658234,126.798804236633,76.6266899323246,115.860629732317,77.3967193874704,75.620017278768,121.979546719793,149.114633470884,92.5306153620199,130.08551532567,83.910041705631,75.8303667884664,77.3892069049812,101.027233057337,146.534095735834,130.292108594124,83.2001121103989,126.133949536336,101.015964333603,76.6604961035261,131.148531597896,101.459200800467,130.337183489059,155.248575423339,75.7251920336172,181.827738470227,181.790176057781,101.301438668193,75.7139233098834,77.6521437921042,123.294231155408,146.522827012101,170.510183600205,83.0611311843482,100.959620714934,98.7584633455898,98.747194621856,114.493357919278,76.7055709984615,76.6492273797923,76.739377169663,101.327732356906,128.312569458212,170.517696082694,181.662463855464,94.3110737119669,101.057282987294,100.952108232444,132.654784336986,130.934425846953,142.49238015663,146.766982693,145.245704988932,75.7101670686388,119.973713895169,109.501313305186,75.6125047962788,128.297544493234,115.560130432748,106.601495064344,111.845207841825,175.028941817476,83.9513603593217,108.806408674932,119.537989910794,130.363477177771,111.807645429379,84.2856658300924,88.6128557438876,136.835480842241,121.821784587519,121.818028346275,96.7826804509229,77.9301056442056,132.771227815569,129.495785450266,145.369660950004,77.4342817999165,175.036454299965,121.81427210503,106.605251305588,81.4609724141427,83.7259858846449,132.7148841969,84.289422071337,74.8462315823776,72.6563429367676,127.399802835771,144.381769502671,83.4517802737881,101.004695609869,178.849039163248,173.150821195169,82.1145583907055,142.477355191651,81.5886846164596,109.997137149475,178.841526680759,128.507894002932,101.000939368624,173.158333677658,115.841848526094,129.923996952152,131.546693169825,76.5590775899215,130.344695971548,136.865530772198,74.8499878236222,149.122145953373,155.207256769648,132.681078025698,76.6717648272599,76.6642523447707,101.48549448918,87.373296133165,131.156044080385,83.2714806940466,77.9225931617164,130.111809014382,119.996251342637,74.8575003061114,115.909460868497,145.373417191249,74.3842139092901,83.9438478768325,137.879715908244,101.03850178107,150.771135859759,101.200020154589,103.585233344919,152.115870225331,132.230329076345,178.743864408399,158.103318769245,158.10707501049,166.047769001604,83.9062854643864,102.657441757499,77.4417942824058,136.005351527182,130.453626967642,83.267724452802,159.988951874041,98.7547071043452,81.4234100016966,170.664189491234,152.206020015201,96.6512120073614,120.255431988515,103.566452138696,130.352208454038,142.496136397874,85.9534369427009,101.305194909438,102.661197998744,131.603036788494,83.1362560092405,74.3917263917793,131.535424446091,106.894481881424,73.5353033880073,84.2969345538262,103.57772086243,131.554205652314,74.8349628586437,120.555931288084,74.8312066173991,89.3227853391196,131.467812103688,74.8424753411329,77.4530630061396,83.4630489975219,101.81979995995,105.346910488643,93.7025626303395,131.817142539437,118.095593272862,165.630826223452,75.1579996056805,149.242345673201,108.855239811112,139.314600063686,121.299667054518,84.1166349740847,116.255035063002,83.9476041180771,166.051525242848,152.705600100735,106.845650745244,177.000968470898,100.9483519912,81.4196537604519,105.294323111218,101.823556201195,116.17239775562,114.500870401767,130.930669605708,136.959436803313,154.636308100467,107.46167430936,110.121093110547,102.935403609601,172.159173506591,128.504137761688,120.21035709358,82.1408520794178,143.931020553317,89.6383096036672,101.342757321884,119.981226377659,142.868004281091,118.366042642475,77.4756004536073,101.215045119567,84.1729785927539,104.941236434224,182.62030537284,143.923508070828,121.885640688678,131.471568344933,127.591371139246,182.62781785533,169.511023429138,129.860140850993,126.137705777581,114.03134024619,81.8027903674026,184.76136288227],"y":[17,17,17,15.8977649888645,15.7729377599443,15.7496037883513,15.2466142309211,15.1321859024601,14.9978721707426,14.9922255601507,14.8240917225424,14.7791376258025,14.7107366260028,14.3561440353754,14.3528441309388,14.3039997143705,14.0332749436826,13.9103804169912,13.8298706141616,13.3565218414395,13.3112751412828,13.2994240847121,13.292843728971,13.1554534696557,13.0560971876636,12.7615799084759,12.7296018100416,12.6668532679548,12.2527767551915,11.9037739844916,11.6190370163313,11.5625823008474,10.9826530027827,10.9203137772842,10.8879538205011,10.7352975407064,10.5554225766406,10.5331415040701,10.2695987383946,10.0132728541551,9.93954861775925,9.67255585158682,9.36570546825746,9.35975443154855,9.35975443154855,8.95420899188044,8.88743781572543,8.76495307103949,8.55184989210515,8.53603510639101,8.46417377398684,8.39302356894794,8.31891526175926,8.2765858725419,8.1828263689299,7.99005075759243,7.88372733381145,7.77987356225339,7.74371950169479,7.73978568145542,7.33384018324614,7.33036737133767,7.27277014975484,7.24931423285531,7.2089398803057,7.19540400208922,7.13240198716866,7.02160846340313,6.95754324431737,6.91667175953185,6.91042976230461,6.72120038116104,6.64461745015689,6.48703650528759,6.41821965707345,6.36565978896668,6.36565978896668,6.33942963221341,6.26214513641173,6.21117060266763,6.14473998654371,6.02562895523438,5.93579841867599,5.89419662065609,5.88197864488027,5.86236350656215,5.81974380379313,5.74097217237016,5.67134259477538,5.55464938848497,5.5335793287968,5.5232851780722,5.51397382944501,5.50148481628646,5.49375747685291,5.4337340228867,5.35691291692628,5.31485540262742,5.30781897944704,5.2075017029626,5.20515357246063,5.18359944746408,5.05911404269632,4.96193723144529,4.93146144675203,4.9284477049318,4.89181190958547,4.799895424083,4.78263719276229,4.69668180397129,4.69395178396906,4.64847576755951,4.6446409169111,4.56634771467545,4.53318131992064,4.51585471835471,4.50346934889409,4.40975624488088,4.37572588320272,4.35020616935906,4.33039030276888,4.2255660033221,4.21709905511816,4.17961534300885,4.16638971296471,4.16638971296471,4.08415454368888,4.00473085041553,3.92792861108921,3.90497810513125,3.90270887794487,3.8804271815469,3.87953014654194,3.83561678668303,3.78609891559592,3.78079611257017,3.76959281980308,3.73654752295179,3.70189799507126,3.69780656328128,3.60170089622929,3.59899583462302,3.59673272905078,3.54863352179527,3.49282460405764,3.48584934908094,3.48485755098054,3.44246306163442,3.43568385877646,3.41046061099578,3.39413325801663,3.3807485560088,3.37346331975105,3.34329380765376,3.3420097832629,3.32348655798195,3.3201419804219,3.30584459317853,3.29791352526173,3.15152108181763,3.12959675415084,3.0460790881135,3.0460790881135,3.01604431765856,2.96633472168066,2.95881983908079,2.94225123485717,2.85086725936601,2.83644836154461,2.82148387791295,2.70609487097036,2.70158667649182,2.69814333900551,2.68193467821294,2.68193467821294,2.6743525777412,2.65680628593258,2.62804413222252,2.62089305737578,2.59893551258587,2.59763329194461,2.59619446270613,2.55881460467762,2.53658259019134,2.53612212895282,2.52269459715743,2.51882055792182,2.4995569047754,2.47646096357105,2.4662253927462,2.40848195005244,2.40631283110246,2.39417802176115,2.36147699208911,2.34359448428666,2.33584620826149,2.32376700043097,2.30901803050152,2.30764768126237,2.29211250081078,2.28662955761278,2.25044410460814,2.23827319286328,2.2072948522559,2.20649768998691,2.18111679168215,2.16933875349575,2.14632302255349,2.12364259415271,2.11095848248584,2.10992470064876,2.10899794690689,2.08335029670697,2.05083456603862,2.01621937207797,2.00096896065405,1.99096474268281,1.98382221325767,1.97613528700715,1.96142068044225,1.9584981964581,1.95767155997425,1.94997752935883,1.92758208428681,1.92453155208112,1.91447853225657,1.90946038089707,1.90603521810727,1.83638710145699,1.83244131773545,1.82906529271907,1.80529109903241,1.79591000324834,1.78071891686413,1.72268307665085,1.71658433586868,1.7145164215215,1.70845286140932,1.70125325901849,1.68078139049836,1.67891396827978,1.66021010091958,1.65231280330146,1.65023980778551,1.6271311192137,1.62164520700665,1.60693057342402,1.60315033918188,1.59463596730564,1.5938784135472,1.58270527603076,1.57423420258401,1.57274996628811,1.57041538513908,1.57008619872947,1.54925401138497,1.54074104129427,1.52514090813898,1.5167878576422,1.50677932019871,1.49798487059234,1.45747577696778,1.43086246302251,1.40311396085867,1.3931037232892,1.38785063434413,1.37276643477724,1.3718327264887,1.37073535551356,1.35693788169646,1.35181497376908,1.34659220433088,1.3376152932052,1.33539396013732,1.3309048896715,1.31117255634442,1.31117255634442,1.30138586155115],"text":["GO:0019882 (236) <br> antigen processing and presentation <br> 6.022e-18","GO:0001775 (1507) <br> cell activation <br> 1.598e-17","GO:0051641 (3519) <br> cellular localization <br> 9.952e-17","GO:0016192 (2193) <br> vesicle-mediated transport <br> 1.265e-16","GO:0048002 (193) <br> antigen processing and presentation of peptide antigen <br> 1.687e-16","GO:0002478 (175) <br> antigen processing and presentation of exogenous peptide antigen <br> 1.780e-16","GO:0019884 (183) <br> antigen processing and presentation of exogenous antigen <br> 5.667e-16","GO:0140352 (1413) <br> export from cell <br> 7.376e-16","GO:0046903 (1492) <br> secretion <br> 1.005e-15","GO:0032940 (1351) <br> secretion by cell <br> 1.018e-15","GO:0045321 (1341) <br> leukocyte activation <br> 1.499e-15","GO:0043299 (538) <br> leukocyte degranulation <br> 1.663e-15","GO:0006887 (906) <br> exocytosis <br> 1.947e-15","GO:0051649 (2778) <br> establishment of localization in cell <br> 4.404e-15","GO:0002275 (552) <br> myeloid cell activation involved in immune response <br> 4.438e-15","GO:0002444 (558) <br> myeloid leukocyte mediated immunity <br> 4.966e-15","GO:0051179 (6938) <br> localization <br> 9.262e-15","GO:0045055 (790) <br> regulated exocytosis <br> 1.229e-14","GO:0002376 (3438) <br> immune system process <br> 1.480e-14","GO:0051234 (5364) <br> establishment of localization <br> 4.400e-14","GO:0043312 (483) <br> neutrophil degranulation <br> 4.883e-14","GO:0036230 (509) <br> granulocyte activation <br> 5.019e-14","GO:0006810 (5214) <br> transport <br> 5.095e-14","GO:0002283 (488) <br> neutrophil activation involved in immune response <br> 6.991e-14","GO:0002446 (500) <br> neutrophil mediated immunity <br> 8.788e-14","GO:0002366 (723) <br> leukocyte activation involved in immune response <br> 1.731e-13","GO:0042119 (502) <br> neutrophil activation <br> 1.864e-13","GO:0002263 (727) <br> cell activation involved in immune response <br> 2.154e-13","GO:0016032 (1011) <br> viral process <br> 5.588e-13","GO:0002274 (671) <br> myeloid leukocyte activation <br> 1.248e-12","GO:0050852 (202) <br> T cell receptor signaling pathway <br> 2.404e-12","GO:0002443 (888) <br> leukocyte mediated immunity <br> 2.738e-12","GO:0008104 (2670) <br> protein localization <br> 1.041e-11","GO:0002252 (1135) <br> immune effector process <br> 1.201e-11","GO:0071840 (6880) <br> cellular component organization or biogenesis <br> 1.294e-11","GO:0034613 (1842) <br> cellular protein localization <br> 1.840e-11","GO:0070727 (1856) <br> cellular macromolecule localization <br> 2.783e-11","GO:0033036 (3148) <br> macromolecule localization <br> 2.930e-11","GO:0006955 (2544) <br> immune response <br> 5.375e-11","GO:0002768 (510) <br> immune response-regulating cell surface receptor signaling pathway <br> 9.699e-11","GO:0002764 (513) <br> immune response-regulating signaling pathway <br> 1.149e-10","GO:0016043 (6661) <br> cellular component organization <br> 2.125e-10","GO:0051668 (775) <br> localization within membrane <br> 4.308e-10","GO:0002757 (475) <br> immune response-activating signal transduction <br> 4.368e-10","GO:0002429 (475) <br> immune response-activating cell surface receptor signaling pathway <br> 4.368e-10","GO:0001732 (16) <br> formation of cytoplasmic translation initiation complex <br> 1.111e-09","GO:0002479 (75) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent <br> 1.296e-09","GO:0050776 (1159) <br> regulation of immune response <br> 1.718e-09","GO:0007159 (377) <br> leukocyte cell-cell adhesion <br> 2.806e-09","GO:0044085 (3377) <br> cellular component biogenesis <br> 2.910e-09","GO:0042590 (80) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I <br> 3.434e-09","GO:0046907 (1723) <br> intracellular transport <br> 4.046e-09","GO:0002474 (99) <br> antigen processing and presentation of peptide antigen via MHC class I <br> 4.798e-09","GO:0042110 (495) <br> T cell activation <br> 5.289e-09","GO:0002684 (1079) <br> positive regulation of immune system process <br> 6.564e-09","GO:0002183 (35) <br> cytoplasmic translational initiation <br> 1.023e-08","GO:0045184 (1866) <br> establishment of protein localization <br> 1.307e-08","GO:0072657 (698) <br> protein localization to membrane <br> 1.660e-08","GO:0015031 (1754) <br> protein transport <br> 1.804e-08","GO:0050778 (757) <br> positive regulation of immune response <br> 1.821e-08","GO:0007155 (1502) <br> cell adhesion <br> 4.636e-08","GO:0002253 (566) <br> activation of immune response <br> 4.673e-08","GO:0002682 (1734) <br> regulation of immune system process <br> 5.336e-08","GO:0022610 (1509) <br> biological adhesion <br> 5.632e-08","GO:0071705 (2216) <br> nitrogen compound transport <br> 6.181e-08","GO:0050851 (321) <br> antigen receptor-mediated signaling pathway <br> 6.377e-08","GO:0006928 (2319) <br> movement of cell or subcellular component <br> 7.372e-08","GO:0046649 (780) <br> lymphocyte activation <br> 9.515e-08","GO:0022607 (3104) <br> cellular component assembly <br> 1.103e-07","GO:0002483 (19) <br> antigen processing and presentation of endogenous peptide antigen <br> 1.212e-07","GO:0051128 (2439) <br> regulation of cellular component organization <br> 1.229e-07","GO:0030155 (756) <br> regulation of cell adhesion <br> 1.900e-07","GO:0050863 (341) <br> regulation of T cell activation <br> 2.267e-07","GO:0098609 (898) <br> cell-cell adhesion <br> 3.258e-07","GO:0002223 (112) <br> stimulatory C-type lectin receptor signaling pathway <br> 3.818e-07","GO:1990858 (113) <br> cellular response to lectin <br> 4.309e-07","GO:1990840 (113) <br> response to lectin <br> 4.309e-07","GO:0030029 (810) <br> actin filament-based process <br> 4.577e-07","GO:0002220 (115) <br> innate immune response activating cell surface receptor signaling pathway <br> 5.468e-07","GO:0002758 (116) <br> innate immune response-activating signal transduction <br> 6.149e-07","GO:0045785 (442) <br> positive regulation of cell adhesion <br> 7.166e-07","GO:0071702 (2657) <br> organic substance transport <br> 9.427e-07","GO:1903037 (341) <br> regulation of leukocyte cell-cell adhesion <br> 1.159e-06","GO:0006886 (995) <br> intracellular protein transport <br> 1.276e-06","GO:0022409 (286) <br> positive regulation of cell-cell adhesion <br> 1.312e-06","GO:0019886 (97) <br> antigen processing and presentation of exogenous peptide antigen via MHC class II <br> 1.373e-06","GO:0019883 (26) <br> antigen processing and presentation of endogenous antigen <br> 1.514e-06","GO:0038093 (238) <br> Fc receptor signaling pathway <br> 1.816e-06","GO:0002495 (101) <br> antigen processing and presentation of peptide antigen via MHC class II <br> 2.131e-06","GO:0002480 (8) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-independent <br> 2.788e-06","GO:0002504 (104) <br> antigen processing and presentation of peptide or polysaccharide antigen via MHC class II <br> 2.927e-06","GO:0030036 (712) <br> actin cytoskeleton organization <br> 2.997e-06","GO:0048522 (5863) <br> positive regulation of cellular process <br> 3.062e-06","GO:1903039 (243) <br> positive regulation of leukocyte cell-cell adhesion <br> 3.151e-06","GO:1990778 (339) <br> protein localization to cell periphery <br> 3.208e-06","GO:0016477 (1660) <br> cell migration <br> 3.684e-06","GO:0022618 (196) <br> ribonucleoprotein complex assembly <br> 4.396e-06","GO:0022407 (450) <br> regulation of cell-cell adhesion <br> 4.843e-06","GO:0051640 (657) <br> organelle localization <br> 4.922e-06","GO:0051050 (951) <br> positive regulation of transport <br> 6.202e-06","GO:0065008 (4116) <br> regulation of biological quality <br> 6.235e-06","GO:0071826 (203) <br> ribonucleoprotein complex subunit organization <br> 6.552e-06","GO:0071310 (2821) <br> cellular response to organic substance <br> 8.727e-06","GO:0002218 (144) <br> activation of innate immune response <br> 1.092e-05","GO:0044087 (991) <br> regulation of cellular component biogenesis <br> 1.171e-05","GO:0034341 (198) <br> response to interferon-gamma <br> 1.179e-05","GO:0002181 (149) <br> cytoplasmic translation <br> 1.283e-05","GO:0048518 (6474) <br> positive regulation of biological process <br> 1.585e-05","GO:0040011 (2039) <br> locomotion <br> 1.650e-05","GO:0032879 (2906) <br> regulation of localization <br> 2.011e-05","GO:0035567 (151) <br> non-canonical Wnt signaling pathway <br> 2.023e-05","GO:1904375 (128) <br> regulation of protein localization to cell periphery <br> 2.247e-05","GO:0007166 (3226) <br> cell surface receptor signaling pathway <br> 2.267e-05","GO:0034097 (1242) <br> response to cytokine <br> 2.714e-05","GO:0043933 (2017) <br> protein-containing complex subunit organization <br> 2.930e-05","GO:0050870 (221) <br> positive regulation of T cell activation <br> 3.049e-05","GO:0035556 (2886) <br> intracellular signal transduction <br> 3.137e-05","GO:0007264 (520) <br> small GTPase mediated signal transduction <br> 3.893e-05","GO:0010033 (3457) <br> response to organic substance <br> 4.210e-05","GO:0060333 (88) <br> interferon-gamma-mediated signaling pathway <br> 4.465e-05","GO:0045089 (217) <br> positive regulation of innate immune response <br> 4.673e-05","GO:0045088 (316) <br> regulation of innate immune response <br> 5.949e-05","GO:0019221 (824) <br> cytokine-mediated signaling pathway <br> 6.066e-05","GO:0002833 (253) <br> positive regulation of response to biotic stimulus <br> 6.613e-05","GO:0051674 (1840) <br> localization of cell <br> 6.817e-05","GO:0048870 (1840) <br> cell motility <br> 6.817e-05","GO:0071345 (1148) <br> cellular response to cytokine stimulus <br> 8.238e-05","GO:0002694 (617) <br> regulation of leukocyte activation <br> 9.892e-05","GO:1904377 (71) <br> positive regulation of protein localization to cell periphery <br> 1.181e-04","GO:0045087 (1003) <br> innate immune response <br> 1.245e-04","GO:0032880 (912) <br> regulation of protein localization <br> 1.251e-04","GO:0006413 (203) <br> translational initiation <br> 1.317e-04","GO:0007097 (20) <br> nuclear migration <br> 1.320e-04","GO:0051656 (443) <br> establishment of organelle localization <br> 1.460e-04","GO:0007265 (352) <br> Ras protein signal transduction <br> 1.636e-04","GO:0001913 (42) <br> T cell mediated cytotoxicity <br> 1.657e-04","GO:0000165 (965) <br> MAPK cascade <br> 1.700e-04","GO:0048193 (415) <br> Golgi vesicle transport <br> 1.834e-04","GO:0070887 (3443) <br> cellular response to chemical stimulus <br> 1.987e-04","GO:0007010 (1473) <br> cytoskeleton organization <br> 2.005e-04","GO:0022604 (312) <br> regulation of cell morphogenesis <br> 2.502e-04","GO:1905477 (132) <br> positive regulation of protein localization to membrane <br> 2.518e-04","GO:1903827 (563) <br> regulation of cellular protein localization <br> 2.531e-04","GO:0006605 (400) <br> protein targeting <br> 2.827e-04","GO:0065003 (1724) <br> protein-containing complex assembly <br> 3.215e-04","GO:0006457 (230) <br> protein folding <br> 3.267e-04","GO:0034622 (1150) <br> cellular protein-containing complex assembly <br> 3.274e-04","GO:1905475 (203) <br> regulation of protein localization to membrane <br> 3.610e-04","GO:0048584 (2368) <br> positive regulation of response to stimulus <br> 3.667e-04","GO:0022603 (1048) <br> regulation of anatomical structure morphogenesis <br> 3.886e-04","GO:1903829 (307) <br> positive regulation of cellular protein localization <br> 4.035e-04","GO:0042098 (206) <br> T cell proliferation <br> 4.162e-04","GO:0050690 (29) <br> regulation of defense response to virus by virus <br> 4.232e-04","GO:0051249 (523) <br> regulation of lymphocyte activation <br> 4.536e-04","GO:0002456 (102) <br> T cell mediated immunity <br> 4.550e-04","GO:0050865 (662) <br> regulation of cell activation <br> 4.748e-04","GO:0060341 (851) <br> regulation of cellular localization <br> 4.785e-04","GO:0001914 (35) <br> regulation of T cell mediated cytotoxicity <br> 4.945e-04","GO:0072659 (287) <br> protein localization to plasma membrane <br> 5.036e-04","GO:0098542 (1222) <br> defense response to other organism <br> 7.055e-04","GO:0051647 (26) <br> nucleus localization <br> 7.420e-04","GO:0002486 (3) <br> antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway, TAP-independent <br> 8.993e-04","GO:0002484 (3) <br> antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway <br> 8.993e-04","GO:0030168 (163) <br> platelet activation <br> 9.637e-04","GO:0009605 (3114) <br> response to external stimulus <br> 1.081e-03","GO:0051130 (1189) <br> positive regulation of cellular component organization <br> 1.099e-03","GO:0006952 (1956) <br> defense response <br> 1.142e-03","GO:0002831 (426) <br> regulation of response to biotic stimulus <br> 1.410e-03","GO:0050790 (2452) <br> regulation of catalytic activity <br> 1.457e-03","GO:0044093 (1774) <br> positive regulation of molecular function <br> 1.508e-03","GO:0001916 (26) <br> positive regulation of T cell mediated cytotoxicity <br> 1.967e-03","GO:0042129 (177) <br> regulation of T cell proliferation <br> 1.988e-03","GO:0071346 (177) <br> cellular response to interferon-gamma <br> 2.004e-03","GO:0001736 (130) <br> establishment of planar polarity <br> 2.080e-03","GO:0007164 (130) <br> establishment of tissue polarity <br> 2.080e-03","GO:0060627 (544) <br> regulation of vesicle-mediated transport <br> 2.117e-03","GO:0022613 (473) <br> ribonucleoprotein complex biogenesis <br> 2.204e-03","GO:0075525 (5) <br> viral translational termination-reinitiation <br> 2.355e-03","GO:0023052 (6937) <br> signaling <br> 2.394e-03","GO:0031349 (364) <br> positive regulation of defense response <br> 2.518e-03","GO:0090150 (385) <br> establishment of protein localization to membrane <br> 2.526e-03","GO:0051493 (535) <br> regulation of cytoskeleton organization <br> 2.534e-03","GO:1905449 (7) <br> regulation of Fc-gamma receptor signaling pathway involved in phagocytosis <br> 2.762e-03","GO:0120035 (642) <br> regulation of plasma membrane bounded cell projection organization <br> 2.907e-03","GO:0120036 (1596) <br> plasma membrane bounded cell projection organization <br> 2.910e-03","GO:1901698 (1178) <br> response to nitrogen compound <br> 3.001e-03","GO:0007154 (6961) <br> cell communication <br> 3.028e-03","GO:0030865 (62) <br> cortical cytoskeleton organization <br> 3.166e-03","GO:0002696 (414) <br> positive regulation of leukocyte activation <br> 3.338e-03","GO:0060071 (110) <br> Wnt signaling pathway, planar cell polarity pathway <br> 3.418e-03","GO:0050900 (511) <br> leukocyte migration <br> 3.904e-03","GO:0006950 (4321) <br> response to stress <br> 3.924e-03","GO:0198738 (535) <br> cell-cell signaling by wnt <br> 4.035e-03","GO:0019885 (13) <br> antigen processing and presentation of endogenous peptide antigen via MHC class I <br> 4.350e-03","GO:0006402 (388) <br> mRNA catabolic process <br> 4.533e-03","GO:1903078 (64) <br> positive regulation of protein localization to plasma membrane <br> 4.615e-03","GO:0090175 (114) <br> regulation of establishment of planar polarity <br> 4.745e-03","GO:0019081 (20) <br> viral translation <br> 4.909e-03","GO:0044267 (5280) <br> cellular protein metabolic process <br> 4.924e-03","GO:0031344 (660) <br> regulation of cell projection organization <br> 5.104e-03","GO:0050867 (426) <br> positive regulation of cell activation <br> 5.169e-03","GO:0065009 (3191) <br> regulation of molecular function <br> 5.618e-03","GO:0008360 (157) <br> regulation of cell shape <br> 5.777e-03","GO:0030030 (1636) <br> cell projection organization <br> 6.204e-03","GO:0030866 (40) <br> cortical actin cytoskeleton organization <br> 6.216e-03","GO:0051270 (1149) <br> regulation of cellular component movement <br> 6.590e-03","GO:0006909 (388) <br> phagocytosis <br> 6.771e-03","GO:0001738 (152) <br> morphogenesis of a polarized epithelium <br> 7.140e-03","GO:0051246 (2759) <br> regulation of protein metabolic process <br> 7.522e-03","GO:0032970 (404) <br> regulation of actin filament-based process <br> 7.745e-03","GO:0000902 (1045) <br> cell morphogenesis <br> 7.764e-03","GO:0007267 (1735) <br> cell-cell signaling <br> 7.780e-03","GO:0031347 (889) <br> regulation of defense response <br> 8.254e-03","GO:0051251 (366) <br> positive regulation of lymphocyte activation <br> 8.895e-03","GO:0001910 (75) <br> regulation of leukocyte mediated cytotoxicity <br> 9.633e-03","GO:0044419 (1768) <br> biological process involved in interspecies interaction between organisms <br> 9.978e-03","GO:0001909 (113) <br> leukocyte mediated cytotoxicity <br> 1.021e-02","GO:0010243 (1092) <br> response to organonitrogen compound <br> 1.038e-02","GO:0051222 (329) <br> positive regulation of protein transport <br> 1.056e-02","GO:0001912 (52) <br> positive regulation of leukocyte mediated cytotoxicity <br> 1.093e-02","GO:0002699 (228) <br> positive regulation of immune effector process <br> 1.100e-02","GO:0007015 (445) <br> actin filament organization <br> 1.102e-02","GO:0030334 (1008) <br> regulation of cell migration <br> 1.122e-02","GO:0032418 (79) <br> lysosome localization <br> 1.181e-02","GO:0016055 (533) <br> Wnt signaling pathway <br> 1.190e-02","GO:0051336 (1252) <br> regulation of hydrolase activity <br> 1.218e-02","GO:0043085 (1383) <br> positive regulation of catalytic activity <br> 1.232e-02","GO:1901564 (6904) <br> organonitrogen compound metabolic process <br> 1.242e-02","GO:0002009 (566) <br> morphogenesis of an epithelium <br> 1.458e-02","GO:0072697 (11) <br> protein localization to cell cortex <br> 1.471e-02","GO:0034113 (60) <br> heterotypic cell-cell adhesion <br> 1.482e-02","GO:0061024 (1016) <br> membrane organization <br> 1.566e-02","GO:0044772 (588) <br> mitotic cell cycle phase transition <br> 1.600e-02","GO:0007212 (41) <br> dopamine receptor signaling pathway <br> 1.657e-02","GO:0042270 (6) <br> protection from natural killer cell mediated cytotoxicity <br> 1.894e-02","GO:0007165 (6447) <br> signal transduction <br> 1.921e-02","GO:1901699 (728) <br> cellular response to nitrogen compound <br> 1.930e-02","GO:0090314 (31) <br> positive regulation of protein targeting to membrane <br> 1.957e-02","GO:0032956 (365) <br> regulation of actin cytoskeleton organization <br> 1.990e-02","GO:1904951 (345) <br> positive regulation of establishment of protein localization <br> 2.086e-02","GO:0022406 (183) <br> membrane docking <br> 2.095e-02","GO:0006401 (431) <br> RNA catabolic process <br> 2.187e-02","GO:0032388 (227) <br> positive regulation of intracellular transport <br> 2.227e-02","GO:0030335 (568) <br> positive regulation of cell migration <br> 2.237e-02","GO:0042221 (4829) <br> response to chemical <br> 2.360e-02","GO:0038095 (167) <br> Fc-epsilon receptor signaling pathway <br> 2.390e-02","GO:0051049 (1839) <br> regulation of transport <br> 2.472e-02","GO:0060368 (11) <br> regulation of Fc receptor mediated stimulatory signaling pathway <br> 2.494e-02","GO:0097435 (863) <br> supramolecular fiber organization <br> 2.543e-02","GO:0033238 (83) <br> regulation of cellular amine metabolic process <br> 2.548e-02","GO:0034695 (25) <br> response to prostaglandin E <br> 2.614e-02","GO:0031032 (199) <br> actomyosin structure organization <br> 2.665e-02","GO:1903533 (82) <br> regulation of protein targeting <br> 2.675e-02","GO:0048583 (4416) <br> regulation of response to stimulus <br> 2.689e-02","GO:0044248 (2381) <br> cellular catabolic process <br> 2.691e-02","GO:0006612 (219) <br> protein targeting to membrane <br> 2.823e-02","GO:0070663 (253) <br> regulation of leukocyte proliferation <br> 2.879e-02","GO:0010389 (204) <br> regulation of G2/M transition of mitotic cell cycle <br> 2.984e-02","GO:0030042 (62) <br> actin filament depolymerization <br> 3.042e-02","GO:0044089 (546) <br> positive regulation of cellular component biogenesis <br> 3.113e-02","GO:0070201 (595) <br> regulation of establishment of protein localization <br> 3.177e-02","GO:0043244 (126) <br> regulation of protein-containing complex disassembly <br> 3.488e-02","GO:0002705 (127) <br> positive regulation of leukocyte mediated immunity <br> 3.708e-02","GO:0023056 (1814) <br> positive regulation of signaling <br> 3.953e-02","GO:0007229 (112) <br> integrin-mediated signaling pathway <br> 4.045e-02","GO:0032268 (2591) <br> regulation of cellular protein metabolic process <br> 4.094e-02","GO:2000145 (1070) <br> regulation of cell motility <br> 4.239e-02","GO:0070661 (327) <br> leukocyte proliferation <br> 4.248e-02","GO:0045123 (69) <br> cellular extravasation <br> 4.259e-02","GO:0051223 (570) <br> regulation of protein transport <br> 4.396e-02","GO:0048268 (18) <br> clathrin coat assembly <br> 4.448e-02","GO:2000147 (592) <br> positive regulation of cell motility <br> 4.502e-02","GO:1902749 (213) <br> regulation of cell cycle G2/M phase transition <br> 4.596e-02","GO:0050670 (232) <br> regulation of lymphocyte proliferation <br> 4.620e-02","GO:0046651 (296) <br> lymphocyte proliferation <br> 4.668e-02","GO:0036388 (66) <br> pre-replicative complex assembly <br> 4.885e-02","GO:0006521 (66) <br> regulation of cellular amino acid metabolic process <br> 4.885e-02","GO:2000736 (115) <br> regulation of stem cell differentiation <br> 4.996e-02"],"key":["GO:0019882","GO:0001775","GO:0051641","GO:0016192","GO:0048002","GO:0002478","GO:0019884","GO:0140352","GO:0046903","GO:0032940","GO:0045321","GO:0043299","GO:0006887","GO:0051649","GO:0002275","GO:0002444","GO:0051179","GO:0045055","GO:0002376","GO:0051234","GO:0043312","GO:0036230","GO:0006810","GO:0002283","GO:0002446","GO:0002366","GO:0042119","GO:0002263","GO:0016032","GO:0002274","GO:0050852","GO:0002443","GO:0008104","GO:0002252","GO:0071840","GO:0034613","GO:0070727","GO:0033036","GO:0006955","GO:0002768","GO:0002764","GO:0016043","GO:0051668","GO:0002757","GO:0002429","GO:0001732","GO:0002479","GO:0050776","GO:0007159","GO:0044085","GO:0042590","GO:0046907","GO:0002474","GO:0042110","GO:0002684","GO:0002183","GO:0045184","GO:0072657","GO:0015031","GO:0050778","GO:0007155","GO:0002253","GO:0002682","GO:0022610","GO:0071705","GO:0050851","GO:0006928","GO:0046649","GO:0022607","GO:0002483","GO:0051128","GO:0030155","GO:0050863","GO:0098609","GO:0002223","GO:1990858","GO:1990840","GO:0030029","GO:0002220","GO:0002758","GO:0045785","GO:0071702","GO:1903037","GO:0006886","GO:0022409","GO:0019886","GO:0019883","GO:0038093","GO:0002495","GO:0002480","GO:0002504","GO:0030036","GO:0048522","GO:1903039","GO:1990778","GO:0016477","GO:0022618","GO:0022407","GO:0051640","GO:0051050","GO:0065008","GO:0071826","GO:0071310","GO:0002218","GO:0044087","GO:0034341","GO:0002181","GO:0048518","GO:0040011","GO:0032879","GO:0035567","GO:1904375","GO:0007166","GO:0034097","GO:0043933","GO:0050870","GO:0035556","GO:0007264","GO:0010033","GO:0060333","GO:0045089","GO:0045088","GO:0019221","GO:0002833","GO:0051674","GO:0048870","GO:0071345","GO:0002694","GO:1904377","GO:0045087","GO:0032880","GO:0006413","GO:0007097","GO:0051656","GO:0007265","GO:0001913","GO:0000165","GO:0048193","GO:0070887","GO:0007010","GO:0022604","GO:1905477","GO:1903827","GO:0006605","GO:0065003","GO:0006457","GO:0034622","GO:1905475","GO:0048584","GO:0022603","GO:1903829","GO:0042098","GO:0050690","GO:0051249","GO:0002456","GO:0050865","GO:0060341","GO:0001914","GO:0072659","GO:0098542","GO:0051647","GO:0002486","GO:0002484","GO:0030168","GO:0009605","GO:0051130","GO:0006952","GO:0002831","GO:0050790","GO:0044093","GO:0001916","GO:0042129","GO:0071346","GO:0001736","GO:0007164","GO:0060627","GO:0022613","GO:0075525","GO:0023052","GO:0031349","GO:0090150","GO:0051493","GO:1905449","GO:0120035","GO:0120036","GO:1901698","GO:0007154","GO:0030865","GO:0002696","GO:0060071","GO:0050900","GO:0006950","GO:0198738","GO:0019885","GO:0006402","GO:1903078","GO:0090175","GO:0019081","GO:0044267","GO:0031344","GO:0050867","GO:0065009","GO:0008360","GO:0030030","GO:0030866","GO:0051270","GO:0006909","GO:0001738","GO:0051246","GO:0032970","GO:0000902","GO:0007267","GO:0031347","GO:0051251","GO:0001910","GO:0044419","GO:0001909","GO:0010243","GO:0051222","GO:0001912","GO:0002699","GO:0007015","GO:0030334","GO:0032418","GO:0016055","GO:0051336","GO:0043085","GO:1901564","GO:0002009","GO:0072697","GO:0034113","GO:0061024","GO:0044772","GO:0007212","GO:0042270","GO:0007165","GO:1901699","GO:0090314","GO:0032956","GO:1904951","GO:0022406","GO:0006401","GO:0032388","GO:0030335","GO:0042221","GO:0038095","GO:0051049","GO:0060368","GO:0097435","GO:0033238","GO:0034695","GO:0031032","GO:1903533","GO:0048583","GO:0044248","GO:0006612","GO:0070663","GO:0010389","GO:0030042","GO:0044089","GO:0070201","GO:0043244","GO:0002705","GO:0023056","GO:0007229","GO:0032268","GO:2000145","GO:0070661","GO:0045123","GO:0051223","GO:0048268","GO:2000147","GO:1902749","GO:0050670","GO:0046651","GO:0036388","GO:0006521","GO:2000736"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[17.4986794977524,19.9462123686674,20.9498382841373,20.3976580599366,17.206351575782,17.0617353348181,17.1279572175344,19.8674339300097,19.9340018163666,19.8123220388678,19.8031784771994,18.6363439324449,19.3129380919747,20.6760128426016,18.6704142203647,18.6847287497765,21.7127740996053,19.1379652435679,20.9230640505876,21.4274758345104,18.4924595182474,18.5625892117027,21.395744596271,18.5062609951603,18.5387647885811,19.0237128877517,18.5440992955177,19.0308506328191,19.451573873278,18.9268195002008,17.2731463069687,19.287432915256,20.6296556443227,19.5965245859779,21.7035374999992,20.1893119996143,20.1984114205898,20.8213641129297,20.5729683353868,18.5652079791051,18.5730308200746,21.6679004416094,19.1133238346132,18.470049611806,18.470049611806,12.8370677538266,15.7373316539529,19.6225996651137,18.1564249614111,20.9024525251713,15.8433279821367,20.1088323582115,16.1868772799724,18.5253260577865,19.5332947906194,14.405947329161,20.2048660232527,18.9781080980935,20.1303596037508,19.0830626021267,19.942156764631,18.7035562554922,20.1165183285464,19.9478305598874,20.4100380870974,17.9340698285406,20.4638403981758,19.1215946463033,20.8050634202662,13.2039079443259,20.5233846899778,19.0813586108141,18.0180536596155,19.3016707664123,16.3815087539172,16.3954151879507,16.3954151879507,19.1700431574596,16.4228177328561,16.4363189409718,18.3730421667754,20.6239407817799,18.0180536596155,19.4314817950327,17.7722510341765,16.1543870489597,13.83902557929,17.5108080471375,16.2186353293393,11.1749781597015,16.2649666785139,19.0038610019127,21.5266239363287,17.540643442317,18.0099018230755,20.0637720125704,17.2289935746564,18.3972686497445,18.8993355559252,19.3743757472774,21.1289356985204,17.2803636129636,20.6939359660643,16.7689423433419,19.4264042119455,17.2438757382447,16.820675004284,21.636471805168,20.3110033219573,20.7285216892983,16.840827913987,16.5888237442405,20.849671586304,19.7084847763908,20.2980535736863,17.4039293372047,20.7204820590056,18.5910918340989,20.9294044050122,15.9981813072674,17.3774579655542,17.9121743965842,19.191991296678,17.598331775882,20.1880060257544,20.1880060257544,19.6107216016844,18.8171599959616,15.6465726979379,19.4415711203883,19.3213181846121,17.2803636129636,13.3106655321065,18.3760965631462,18.0619709426399,14.73917293128,19.3928503778594,18.2875917936538,20.9247361833321,19.9183441742083,17.8943821822839,16.6361245292714,18.6965300907626,18.2374501939285,20.1095332573674,17.4616009111547,19.6128903566504,17.2803636129636,20.4885440438041,19.4967505982493,17.8717860566539,17.3017813977071,14.0509285588584,18.5987513048536,16.2342495236679,18.90922365592,19.2332018170431,14.405947329161,17.7771707617082,19.6883679423801,13.83902557929,7.7791008157754,7.7791008157754,16.9558134205769,20.808789697542,19.6543863888238,20.2613396475456,18.3231184550737,20.5296462684742,20.1440324362717,13.83902557929,17.0786016380891,17.0786016380891,16.6126778615783,16.6126778615783,18.6510623608111,18.4643827493734,9.79200637188604,21.7126155431844,18.1081960550518,18.1852072362727,18.6289175161382,10.8097872173079,18.8691734015509,20.0160746648226,19.6428327290729,21.7164142179893,15.4190791173213,18.2843111493931,16.3532722717548,18.5678211543158,21.1841117247357,18.6289175161382,12.3729643352122,18.195832112632,15.4727616697066,16.4091839862407,13.3106655321065,21.4098243887321,18.9052781824991,18.3231184550737,20.8370609384428,16.8995454455618,20.0461208480012,14.651001049555,19.6118064881816,18.195832112632,16.8507929922937,20.6679986229776,18.2510199391569,19.4931522760457,20.117214455132,19.2888644874537,18.1157376279914,15.7373316539529,20.1399480363029,16.3954151879507,19.5482837527865,17.9683348657126,15.1170689685676,17.4490032718266,18.3821828199127,19.4478328947959,15.8227421236023,18.6239413228969,19.7184126682615,19.8410990920889,21.7073695277206,18.7035562554922,11.980531463383,15.3633750239743,19.4577822668577,18.7538745275872,14.6957136866834,10.3630624020686,21.63185447316,19.032628413028,14.1782205386673,18.1119725030245,18.034201089098,17.1279572175344,18.3389367682336,17.4426585575357,18.7082178803185,21.3096384784207,16.9920586807013,20.1873524673787,11.980531463383,19.2510654035469,15.9033751977876,13.7618192670139,17.2512544393439,15.8836369590203,21.2087426859582,20.495006256239,17.3907605242587,17.598331775882,17.2875416251238,15.4190791173213,18.6559292444103,18.7694563553214,16.5645483925971,16.5767396110151,20.1708883719482,16.3815087539172,20.5944578148066,19.5228031677339,17.959855073109,15.5989806420055,18.7128616722661,13.0900518224006,18.7628029787026,17.3504409902673,17.4740780401283,17.8206171716069,15.5245580341656,15.5245580341656,16.4228177328561],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[62.186587712122,67.1194768172047,59.8873279007658,62.0964206606962,57.3401086979889,52.9256801386042,53.9475733880958,55.7396435351822,62.2692408425955,55.0746615309174,52.7866726009895,56.8479468756235,65.0193359110803,53.3276749095439,55.7358865747061,56.0815269385048,65.2222117767882,53.64701655001,57.7796730736894,53.8160797714333,54.8304590999727,65.259781381549,56.1078256618373,63.648045337314,66.0825557258088,65.5077407729697,56.5661748399181,53.6620443919143,67.2359425919629,65.5114977334458,55.938762440414,65.624206547728,61.0369578064439,54.9544387956831,65.3386775515465,55.574337274235,55.5780942347111,56.6187722865831,54.950681835207,58.072715990823,59.8760570193376,56.4534660256359,55.1159880961542,53.7183987990554,59.8723000588615,52.5950676167098,62.8553266768628,65.3424345120225,56.092797819933,59.7445634026751,53.9888999533326,59.5041179322065,61.4690082611922,66.3831125638945,55.0258210447285,65.6091787058237,53.4215989214457,53.4516546052543,52.9482219014606,55.9312485194619,62.7576457044849,55.9199776380337,59.6017989045843,63.4188707482736,65.2259687372643,59.5492014579193,53.4328698028739,59.6055558650604,61.0745274112046,53.4478976447782,59.8835709402897,56.1190965432655,61.2586184745321,53.4178419609696,52.1329614781529,55.9124637170815,65.312378828214,56.1040687013612,66.9128439910207,58.8842194536546,61.5216057078572,56.5398761165855,55.8523523494643,61.4351956169075,56.197992713263,56.6262862075352,52.606338498138,56.6225292470591,51.9563843357775,54.8680287047335,54.8530008628292,55.73212961423,66.1539379748541,59.8910848612419,56.0777699780287,65.4964698915415,55.9575472427944,54.9581957561592,62.761402664961,68.5696635609685,56.6338001284874,55.8561093099404,53.8987329019068,53.54182165668,56.0852838989808,53.5267938147757,55.8636232308926,53.7296696804836,57.6932629827397,57.6218807336943,59.6243406674408,61.5178487473811,60.522254221222,56.6751266937241,56.1003117408851,52.5762828143295,65.7744849667708,63.4075998668454,61.4314386564315,56.2017496737391,55.9162206775576,61.0632565297764,52.5725258538534,64.6210981006167,55.9011928356533,56.0139016499354,66.1426670934259,64.4933614444302,54.8417299814009,54.9694666375874,55.1197450566303,53.8799480995265,56.5361191561095,55.9237345985097,55.9087067566054,57.3476226189411,66.1126114096173,52.2494272529112,56.4872786699205,55.1535577009149,54.1579631747558,56.6788836542002,53.5568494985843,59.2599155012618,67.243456512915,59.7483203631512,56.5323621956334,53.8085658504811,52.602581537662,65.6429913501083,57.1935872394221,54.8267021394967,61.2548615140561,53.5080090123953,58.5047664455713,55.8824080332729,65.5490673382065,63.6630731792183,53.8123228109572,53.4854672495389,55.9274915589858,59.477819208874,53.8198367319093,56.6413140494395,59.7332925212469,53.9024898623829,53.440383723826,55.9350054799379,57.4415466308429,55.1310159380585,53.6770722338186,55.6908030489932,57.2198859627546,65.5716091010629],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.8399370560352,15.7278870923031,15.6798605417862,15.5300391514483,15.1912574104494,15.1724051192202,14.9148921875314,14.6221564668037,14.5093827664672,14.1994023565339,13.1248289410545,13.1248289410545,12.8389665217276,12.8200304074515,12.8193798576399,12.8193798576399,12.4427398523689,12.3134432378043,11.9457075219511,10.7741392180911,10.4687452413133,10.2873434270239,10.2134377115144,10.0288907043821,9.91738728756937,9.73628402922528,9.73628402922528,9.71319507735181,9.61348897083933,9.52399053770579,9.07533322903101,8.77336786830299,8.57553383814269,8.5691274027521,8.52611422890357,8.52611422890357,8.48433826398719,8.29586273446902,8.14966203480234,8.14966203480234,7.93491508376221,7.90729620950551,7.60371175821078,7.55809964851197,7.51680487813293,7.10281527174739,6.96271911209217,6.96271911209217,6.92020532249327,6.88043187940901,6.52649213759675,6.46957750810771,6.39553622195122,6.37039827655545,6.32057391347104,6.18103790631995,6.17648677277132,6.17648677277132,6.16360382581995,6.03452323947257,5.70541044016213,5.52989708775279,5.4222248007326,5.31652479038952,5.30087536516586,5.29886228496298,5.29886228496298,5.26473712933539,5.26215194570377,5.26215194570377,5.16767123234411,5.09971145537728,5.06286820303847,5.00045925035761,4.91161674446995,4.69774903111426,4.64083898988528,4.56066066819621,4.55785119723238,4.52131026332395,4.34708651085023,4.2843957040555,4.27510681693462,4.25723294286858,4.20203885645893,3.94084835453694,3.89721150772184,3.81714048657933,3.7982025621432,3.69349376890448,3.63201513591254,3.61452336651298,3.549516887592,3.50673149382184,3.39986749327173,3.38183581621481,3.12517691313675,3.11477324531584,3.03878054941893,3.02163922356146,3.0040971831966,2.97744825052444,2.95902023122887,2.93804188738313,2.87413611866268,2.78348748557779,2.78348748557779,2.73502960466485,2.63210186191688,2.62919312003482,2.61083359762573,2.59905213831787,2.53510215586909,2.47449195532698,2.4167078356402,2.39860697378633,2.3821979318611,2.36232224179645,2.32065074673088,2.31542733702967,2.29805568825159,2.24604770214936,2.18993781024873,2.18869554020798,2.12488956379208,2.11367775534326,2.10681941657424,2.02178974343192,2.01150062534942,2.00985472327442,1.97291627352109,1.86009795282188,1.80390268113863,1.7689808630693,1.74618773470142,1.73719877820805,1.71367488934294,1.7034891122996,1.68892845634073,1.68056706092491,1.65816385649714,1.65051081749014,1.60219027559943,1.59406880572491,1.56980817090164,1.56905350994416,1.55901625029884,1.51736107811435,1.50092023922394,1.48559839779199,1.47168628113053,1.40083732277069,1.38425285580909,1.37618907913556,1.36712183759162,1.36244290618332,1.31079303745517],"text":["GO:0070062 (2178) <br> extracellular exosome <br> 4.266e-51","GO:1903561 (2263) <br> extracellular vesicle <br> 5.673e-49","GO:0043230 (2265) <br> extracellular organelle <br> 6.347e-49","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 6.347e-49","GO:0031982 (4058) <br> vesicle <br> 7.620e-36","GO:0005615 (3595) <br> extracellular space <br> 1.796e-27","GO:0005925 (420) <br> focal adhesion <br> 5.081e-22","GO:0030055 (427) <br> cell-substrate junction <br> 1.014e-21","GO:0070161 (838) <br> anchoring junction <br> 2.397e-21","GO:0016020 (9829) <br> membrane <br> 1.157e-20","GO:0005576 (4564) <br> extracellular region <br> 9.979e-20","GO:0031410 (2433) <br> cytoplasmic vesicle <br> 1.465e-19","GO:0097708 (2438) <br> intracellular vesicle <br> 1.711e-19","GO:0005737 (11951) <br> cytoplasm <br> 2.913e-18","GO:0030054 (2107) <br> cell junction <br> 3.798e-18","GO:0030659 (789) <br> cytoplasmic vesicle membrane <br> 8.003e-18","GO:0098552 (622) <br> side of membrane <br> 1.088e-17","GO:0005829 (5361) <br> cytosol <br> 1.446e-16","GO:0032991 (5541) <br> protein-containing complex <br> 1.871e-16","GO:0005886 (5680) <br> plasma membrane <br> 2.090e-16","GO:0009898 (169) <br> cytoplasmic side of plasma membrane <br> 2.951e-16","GO:0098562 (195) <br> cytoplasmic side of membrane <br> 6.438e-16","GO:0030667 (308) <br> secretory granule membrane <br> 6.723e-16","GO:0071944 (6178) <br> cell periphery <br> 1.216e-15","GO:0099503 (1027) <br> secretory vesicle <br> 2.387e-15","GO:0098796 (1326) <br> membrane protein complex <br> 3.095e-15","GO:0031234 (103) <br> extrinsic component of cytoplasmic side of plasma membrane <br> 6.318e-15","GO:0005834 (36) <br> heterotrimeric G-protein complex <br> 7.502e-14","GO:1905360 (36) <br> GTPase complex <br> 7.502e-14","GO:0098797 (696) <br> plasma membrane protein complex <br> 1.449e-13","GO:0030141 (857) <br> secretory granule <br> 1.513e-13","GO:0098857 (339) <br> membrane microdomain <br> 1.516e-13","GO:0045121 (339) <br> membrane raft <br> 1.516e-13","GO:0012506 (1183) <br> vesicle membrane <br> 3.608e-13","GO:0098588 (2165) <br> bounding membrane of organelle <br> 4.859e-13","GO:0019897 (178) <br> extrinsic component of plasma membrane <br> 1.133e-12","GO:0019898 (315) <br> extrinsic component of membrane <br> 1.682e-11","GO:0031252 (426) <br> cell leading edge <br> 3.398e-11","GO:0012505 (4651) <br> endomembrane system <br> 5.160e-11","GO:0033290 (15) <br> eukaryotic 48S preinitiation complex <br> 6.117e-11","GO:0043227 (13201) <br> membrane-bounded organelle <br> 9.356e-11","GO:0031090 (3644) <br> organelle membrane <br> 1.210e-10","GO:0016282 (17) <br> eukaryotic 43S preinitiation complex <br> 1.835e-10","GO:0005852 (17) <br> eukaryotic translation initiation factor 3 complex <br> 1.835e-10","GO:0043226 (14027) <br> organelle <br> 1.936e-10","GO:0001772 (44) <br> immunological synapse <br> 2.435e-10","GO:0070993 (18) <br> translation preinitiation complex <br> 2.992e-10","GO:0098590 (1240) <br> plasma membrane region <br> 8.407e-10","GO:0030662 (177) <br> coated vesicle membrane <br> 1.685e-09","GO:0042995 (2337) <br> cell projection <br> 2.657e-09","GO:0005938 (326) <br> cell cortex <br> 2.697e-09","GO:0042470 (109) <br> melanosome <br> 2.978e-09","GO:0048770 (109) <br> pigment granule <br> 2.978e-09","GO:0120025 (2234) <br> plasma membrane bounded cell projection <br> 3.278e-09","GO:0015629 (517) <br> actin cytoskeleton <br> 5.060e-09","GO:0098852 (395) <br> lytic vacuole membrane <br> 7.085e-09","GO:0005765 (395) <br> lysosomal membrane <br> 7.085e-09","GO:0005774 (450) <br> vacuolar membrane <br> 1.162e-08","GO:0005622 (14808) <br> intracellular anatomical structure <br> 1.238e-08","GO:0030139 (335) <br> endocytic vesicle <br> 2.491e-08","GO:0070820 (163) <br> tertiary granule <br> 2.766e-08","GO:0030135 (296) <br> coated vesicle <br> 3.042e-08","GO:0042611 (24) <br> MHC protein complex <br> 7.892e-08","GO:0071556 (25) <br> integral component of lumenal side of endoplasmic reticulum membrane <br> 1.090e-07","GO:0098553 (25) <br> lumenal side of endoplasmic reticulum membrane <br> 1.090e-07","GO:0042581 (158) <br> specific granule <br> 1.202e-07","GO:0005768 (993) <br> endosome <br> 1.317e-07","GO:0042612 (8) <br> MHC class I protein complex <br> 2.975e-07","GO:0045202 (1348) <br> synapse <br> 3.392e-07","GO:0005773 (818) <br> vacuole <br> 4.022e-07","GO:0043229 (13117) <br> intracellular organelle <br> 4.262e-07","GO:0030670 (75) <br> phagocytic vesicle membrane <br> 4.780e-07","GO:0045335 (139) <br> phagocytic vesicle <br> 6.591e-07","GO:0005764 (723) <br> lysosome <br> 6.661e-07","GO:0000323 (723) <br> lytic vacuole <br> 6.661e-07","GO:0030133 (416) <br> transport vesicle <br> 6.861e-07","GO:0098576 (33) <br> lumenal side of membrane <br> 9.236e-07","GO:0030666 (190) <br> endocytic vesicle membrane <br> 1.971e-06","GO:1902494 (1377) <br> catalytic complex <br> 2.952e-06","GO:0035579 (90) <br> specific granule membrane <br> 3.782e-06","GO:0055038 (85) <br> recycling endosome membrane <br> 4.825e-06","GO:0031227 (169) <br> intrinsic component of endoplasmic reticulum membrane <br> 5.002e-06","GO:0030117 (97) <br> membrane coat <br> 5.025e-06","GO:0048475 (97) <br> coated membrane <br> 5.025e-06","GO:0030863 (121) <br> cortical cytoskeleton <br> 5.436e-06","GO:0031254 (13) <br> cell trailing edge <br> 5.468e-06","GO:0001931 (13) <br> uropod <br> 5.468e-06","GO:0031253 (346) <br> cell projection membrane <br> 6.797e-06","GO:0000139 (725) <br> Golgi membrane <br> 7.949e-06","GO:0010008 (520) <br> endosome membrane <br> 8.652e-06","GO:0009986 (896) <br> cell surface <br> 9.989e-06","GO:0030027 (203) <br> lamellipodium <br> 1.226e-05","GO:0099738 (44) <br> cell cortex region <br> 2.006e-05","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 2.286e-05","GO:0030658 (210) <br> transport vesicle membrane <br> 2.750e-05","GO:0098791 (845) <br> Golgi apparatus subcompartment <br> 2.768e-05","GO:0030176 (161) <br> integral component of endoplasmic reticulum membrane <br> 3.011e-05","GO:0012507 (57) <br> ER to Golgi transport vesicle membrane <br> 4.497e-05","GO:0070821 (72) <br> tertiary granule membrane <br> 5.195e-05","GO:1990904 (694) <br> ribonucleoprotein complex <br> 5.308e-05","GO:0031256 (179) <br> leading edge membrane <br> 5.531e-05","GO:0030118 (47) <br> clathrin coat <br> 6.280e-05","GO:0005911 (496) <br> cell-cell junction <br> 1.146e-04","GO:0005798 (95) <br> Golgi-associated vesicle <br> 1.267e-04","GO:0030660 (52) <br> Golgi-associated vesicle membrane <br> 1.524e-04","GO:0005794 (1593) <br> Golgi apparatus <br> 1.591e-04","GO:0030120 (55) <br> vesicle coat <br> 2.025e-04","GO:0005856 (2339) <br> cytoskeleton <br> 2.333e-04","GO:0032588 (98) <br> trans-Golgi network membrane <br> 2.429e-04","GO:0032432 (78) <br> actin filament bundle <br> 2.822e-04","GO:0042641 (79) <br> actomyosin <br> 3.114e-04","GO:0055037 (187) <br> recycling endosome <br> 3.982e-04","GO:0044291 (76) <br> cell-cell contact zone <br> 4.151e-04","GO:0031300 (419) <br> intrinsic component of organelle membrane <br> 7.496e-04","GO:0030665 (114) <br> clathrin-coated vesicle membrane <br> 7.678e-04","GO:0001726 (180) <br> ruffle <br> 9.146e-04","GO:0098978 (354) <br> glutamatergic synapse <br> 9.514e-04","GO:0071541 (7) <br> eukaryotic translation initiation factor 3 complex, eIF3m <br> 9.906e-04","GO:0048471 (745) <br> perinuclear region of cytoplasm <br> 1.053e-03","GO:0030864 (92) <br> cortical actin cytoskeleton <br> 1.099e-03","GO:0030134 (90) <br> COPII-coated ER to Golgi transport vesicle <br> 1.153e-03","GO:0045178 (268) <br> basal part of cell <br> 1.336e-03","GO:0001725 (70) <br> stress fiber <br> 1.646e-03","GO:0097517 (70) <br> contractile actin filament bundle <br> 1.646e-03","GO:0030130 (13) <br> clathrin coat of trans-Golgi network vesicle <br> 1.841e-03","GO:0030425 (641) <br> dendrite <br> 2.333e-03","GO:0099631 (2) <br> postsynaptic endocytic zone cytoplasmic component <br> 2.349e-03","GO:0097447 (643) <br> dendritic tree <br> 2.450e-03","GO:0009925 (250) <br> basal plasma membrane <br> 2.517e-03","GO:0012510 (15) <br> trans-Golgi network transport vesicle membrane <br> 2.917e-03","GO:0016323 (225) <br> basolateral plasma membrane <br> 3.354e-03","GO:0005903 (109) <br> brush border <br> 3.831e-03","GO:0031226 (1725) <br> intrinsic component of plasma membrane <br> 3.994e-03","GO:0030136 (194) <br> clathrin-coated vesicle <br> 4.148e-03","GO:0030132 (17) <br> clathrin coat of coated pit <br> 4.342e-03","GO:0031984 (1764) <br> organelle subcompartment <br> 4.779e-03","GO:0099568 (270) <br> cytoplasmic region <br> 4.837e-03","GO:0000502 (67) <br> proteasome complex <br> 5.034e-03","GO:0031143 (18) <br> pseudopodium <br> 5.675e-03","GO:0016459 (57) <br> myosin complex <br> 6.457e-03","GO:0008180 (35) <br> COP9 signalosome <br> 6.476e-03","GO:0031301 (389) <br> integral component of organelle membrane <br> 7.501e-03","GO:0005802 (255) <br> trans-Golgi network <br> 7.697e-03","GO:0036477 (875) <br> somatodendritic compartment <br> 7.820e-03","GO:1905369 (73) <br> endopeptidase complex <br> 9.511e-03","GO:0043005 (1384) <br> neuron projection <br> 9.739e-03","GO:0031225 (172) <br> anchored component of membrane <br> 9.776e-03","GO:0005884 (118) <br> actin filament <br> 1.064e-02","GO:0001917 (63) <br> photoreceptor inner segment <br> 1.380e-02","GO:0098862 (164) <br> cluster of actin-based cell projections <br> 1.571e-02","GO:0031680 (12) <br> G-protein beta/gamma-subunit complex <br> 1.702e-02","GO:0009897 (409) <br> external side of plasma membrane <br> 1.794e-02","GO:0045334 (89) <br> clathrin-coated endocytic vesicle <br> 1.831e-02","GO:0005789 (1132) <br> endoplasmic reticulum membrane <br> 1.933e-02","GO:0034687 (2) <br> integrin alphaL-beta2 complex <br> 1.979e-02","GO:0030125 (28) <br> clathrin vesicle coat <br> 2.047e-02","GO:0098827 (1137) <br> endoplasmic reticulum subcompartment <br> 2.087e-02","GO:0071986 (10) <br> Ragulator complex <br> 2.197e-02","GO:0005885 (11) <br> Arp2/3 protein complex <br> 2.236e-02","GO:0005783 (1995) <br> endoplasmic reticulum <br> 2.499e-02","GO:0030137 (28) <br> COPI-coated vesicle <br> 2.546e-02","GO:0042175 (1154) <br> nuclear outer membrane-endoplasmic reticulum membrane network <br> 2.693e-02","GO:0005887 (1643) <br> integral component of plasma membrane <br> 2.697e-02","GO:0031258 (22) <br> lamellipodium membrane <br> 2.760e-02","GO:0042824 (9) <br> MHC class I peptide loading complex <br> 3.038e-02","GO:0005912 (171) <br> adherens junction <br> 3.156e-02","GO:0005770 (285) <br> late endosome <br> 3.269e-02","GO:0030140 (33) <br> trans-Golgi network transport vesicle <br> 3.375e-02","GO:0032153 (72) <br> cell division site <br> 3.973e-02","GO:0016328 (67) <br> lateral plasma membrane <br> 4.128e-02","GO:0005838 (22) <br> proteasome regulatory particle <br> 4.205e-02","GO:0022625 (57) <br> cytosolic large ribosomal subunit <br> 4.294e-02","GO:0031902 (149) <br> late endosome membrane <br> 4.341e-02","GO:0098835 (7) <br> presynaptic endocytic zone membrane <br> 4.889e-02"],"key":["GO:0070062","GO:1903561","GO:0043230","GO:0065010","GO:0031982","GO:0005615","GO:0005925","GO:0030055","GO:0070161","GO:0016020","GO:0005576","GO:0031410","GO:0097708","GO:0005737","GO:0030054","GO:0030659","GO:0098552","GO:0005829","GO:0032991","GO:0005886","GO:0009898","GO:0098562","GO:0030667","GO:0071944","GO:0099503","GO:0098796","GO:0031234","GO:0005834","GO:1905360","GO:0098797","GO:0030141","GO:0098857","GO:0045121","GO:0012506","GO:0098588","GO:0019897","GO:0019898","GO:0031252","GO:0012505","GO:0033290","GO:0043227","GO:0031090","GO:0016282","GO:0005852","GO:0043226","GO:0001772","GO:0070993","GO:0098590","GO:0030662","GO:0042995","GO:0005938","GO:0042470","GO:0048770","GO:0120025","GO:0015629","GO:0098852","GO:0005765","GO:0005774","GO:0005622","GO:0030139","GO:0070820","GO:0030135","GO:0042611","GO:0071556","GO:0098553","GO:0042581","GO:0005768","GO:0042612","GO:0045202","GO:0005773","GO:0043229","GO:0030670","GO:0045335","GO:0005764","GO:0000323","GO:0030133","GO:0098576","GO:0030666","GO:1902494","GO:0035579","GO:0055038","GO:0031227","GO:0030117","GO:0048475","GO:0030863","GO:0031254","GO:0001931","GO:0031253","GO:0000139","GO:0010008","GO:0009986","GO:0030027","GO:0099738","GO:0043231","GO:0030658","GO:0098791","GO:0030176","GO:0012507","GO:0070821","GO:1990904","GO:0031256","GO:0030118","GO:0005911","GO:0005798","GO:0030660","GO:0005794","GO:0030120","GO:0005856","GO:0032588","GO:0032432","GO:0042641","GO:0055037","GO:0044291","GO:0031300","GO:0030665","GO:0001726","GO:0098978","GO:0071541","GO:0048471","GO:0030864","GO:0030134","GO:0045178","GO:0001725","GO:0097517","GO:0030130","GO:0030425","GO:0099631","GO:0097447","GO:0009925","GO:0012510","GO:0016323","GO:0005903","GO:0031226","GO:0030136","GO:0030132","GO:0031984","GO:0099568","GO:0000502","GO:0031143","GO:0016459","GO:0008180","GO:0031301","GO:0005802","GO:0036477","GO:1905369","GO:0043005","GO:0031225","GO:0005884","GO:0001917","GO:0098862","GO:0031680","GO:0009897","GO:0045334","GO:0005789","GO:0034687","GO:0030125","GO:0098827","GO:0071986","GO:0005885","GO:0005783","GO:0030137","GO:0042175","GO:0005887","GO:0031258","GO:0042824","GO:0005912","GO:0005770","GO:0030140","GO:0032153","GO:0016328","GO:0005838","GO:0022625","GO:0031902","GO:0098835"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[20.3895089474417,20.43491384342,20.4359600780238,20.4359600780238,21.1127924137548,20.97436852479,18.3038663648759,18.3262982994618,19.2135392269153,22.0919153891785,21.2460124471743,20.520482671709,20.522901551273,22.301288321845,20.3501024869214,19.1363383178836,18.8277438199372,21.4268505016346,21.4637267954854,21.4913424228551,17.0098206791988,17.2214892061892,17.8763374918434,21.5846981879027,19.4713253358183,19.7893245482402,16.2496921162547,14.4581139285914,14.4581139285914,18.9743833960756,19.2421674566737,18.0099018230755,18.0099018230755,19.6480987518461,20.3823975905089,17.0869555296552,17.90774964072,18.3231184550737,21.2673252341618,12.6954028668369,22.4069336180039,20.9898925305246,12.968152232712,12.968152232712,22.4710947059109,14.8225866744181,13.0900518224006,19.7064888615817,17.0786016380891,20.4729795999535,17.9555937940104,16.3389366763927,16.3389366763927,20.4196311626559,18.5833840485582,18.2202776543847,18.2202776543847,18.3972686497445,22.5281903038104,17.9934386332075,16.9558134205769,17.8206171716069,13.6808214449962,13.7618192670139,13.7618192670139,16.9090881701731,19.4289457661119,11.1749781597015,19.8095866429318,19.1826347921651,22.4001723500672,15.7373316539529,16.715163312453,19.0237128877517,19.0237128877517,18.2908638032579,14.2961523059962,17.1833157067251,19.835758359278,16.0344085591459,15.9420548318458,17.0098206791988,16.1543870489597,16.1543870489597,16.5019202141693,12.3729643352122,12.3729643352122,18.0382058611615,19.0272871005367,18.5910918340989,19.2988369642948,17.2803636129636,14.8225866744181,22.3097189832527,17.3298070009327,19.2241674808196,16.9373174877114,15.2756954716879,15.6697988614665,18.9706470582724,17.0952574234415,14.9397854919929,18.5280256133466,16.121131665261,15.1170689685676,20.0137886576694,15.2142427416402,20.473990411145,16.1707256977144,15.801858092204,15.8227421236023,17.1598725929261,15.7591617451593,18.3006284598251,16.4091839862407,17.1035079372937,18.0697940419775,10.8097872173079,19.0624517869717,16.0697362528688,16.0344085591459,17.6803085206584,15.6229705518591,15.6229705518591,12.3729643352122,18.8671354327216,3.77952755905512,18.8712079238796,17.5812933447843,12.6954028668369,17.429875898974,16.3389366763927,20.1102337200314,17.2139420304112,12.968152232712,20.1372168262823,17.6908552942341,15.5497876137808,13.0900518224006,15.2756954716879,14.405947329161,18.1993537615243,17.6095673525824,19.2686621270864,15.692660304218,19.841986788873,17.0360298019214,16.4629334550975,15.4461643032416,16.9649668448592,12.187219027138,18.2677768543077,16.0164101032087,19.5932234371659,3.77952755905512,13.9833131530912,19.5987201263685,11.747980750738,11.980531463383,20.2849514736126,13.9833131530912,19.6172156871556,20.0512976958079,13.5059251363754,11.4827422948031,17.0273503124218,17.7673123351907,14.2961523059962,15.6697988614665,15.5497876137808,13.5059251363754,15.2756954716879,16.820675004284,10.8097872173079],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[28.7757887903244,9.05456388162587,16.9543231164817,18.7010601798235,36.1083280325681,4.24258500390343,15.3202787669038,16.2143076294315,16.2105512056393,28.9448278609704,9.35507778499651,4.24634142769556,18.6935473322393,22.3297655630241,9.0921281195472,16.1128841870439,3.68663428266774,3.75424991092613,9.48279619392904,21.169030611255,36.1534051180737,27.0515902697353,19.3434086482783,3.64155719716214,40.341817646302,40.8301527392793,16.8679253692626,25.8194832659157,27.044077422151,27.4460147679093,46.8704821970293,28.8997507754648,27.0403209983589,22.3072270202713,2.12771840893252,22.299714172687,25.9960351841459,41.1682308805713,19.9932699643173,8.02906018637355,22.5363688715914,26.8487433849601,19.3471650720704,8.99070467715961,21.0075043881932,21.6536092804401,25.5790721432192,19.3509214958626,27.1229623217858,38.0353734379323,29.669817652852,19.9256543360589,38.0691812520616,27.0478338459432,27.3746427158588,19.362190767239,8.35962548008125,8.29576627561499],"y":[17,13.727570571771,11.8961129899244,11.5739831021461,10.940532779986,10.8115049831137,10.3055032536949,10.0308620102203,10.0062796107186,9.6012887167267,8.45591510322825,8.40714523395277,7.80508981126959,7.80508981126959,7.58260367574716,6.97744386377045,6.88949944489194,6.3495891012917,6.32782785171497,6.03273138488328,5.68999736432344,5.54488257093449,5.52701318141138,5.51611713450323,5.48669779559305,5.15186482718857,5.12426177628221,5.01935026579866,4.96853461887794,4.95431051382445,4.73724287777658,4.6328812705077,4.47517551735374,4.44871027072676,4.39564985456039,4.31755721596575,4.17907733971419,3.73339171179895,3.08938329448318,2.72390411084435,2.65540290467261,2.55262007931048,2.34596407895162,2.11805454481398,2.04974119254753,2.02302566884688,1.95272413275386,1.9368127464516,1.87270449554281,1.85762433200095,1.84370309068119,1.83382990763067,1.57938402101178,1.57022000934249,1.50504814572943,1.41111661216499,1.3511217236569,1.3048346311429],"text":["GO:0044877 (1304) <br> protein-containing complex binding <br> 2.848e-21","GO:0005515 (14767) <br> protein binding <br> 1.873e-14","GO:0017111 (424) <br> nucleoside-triphosphatase activity <br> 1.270e-12","GO:0019003 (73) <br> GDP binding <br> 2.667e-12","GO:0050839 (544) <br> cell adhesion molecule binding <br> 1.147e-11","GO:0003924 (300) <br> GTPase activity <br> 1.543e-11","GO:0016462 (482) <br> pyrophosphatase activity <br> 4.949e-11","GO:0016818 (493) <br> hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides <br> 9.314e-11","GO:0016817 (494) <br> hydrolase activity, acting on acid anhydrides <br> 9.856e-11","GO:0045296 (333) <br> cadherin binding <br> 2.504e-10","GO:0008092 (1001) <br> cytoskeletal protein binding <br> 3.500e-09","GO:0003925 (32) <br> G protein activity <br> 3.916e-09","GO:0019001 (402) <br> guanyl nucleotide binding <br> 1.566e-08","GO:0032561 (402) <br> guanyl ribonucleotide binding <br> 1.566e-08","GO:0005525 (381) <br> GTP binding <br> 2.615e-08","GO:0016787 (2357) <br> hydrolase activity <br> 1.053e-07","GO:0003743 (58) <br> translation initiation factor activity <br> 1.290e-07","GO:0003779 (449) <br> actin binding <br> 4.471e-07","GO:0008135 (99) <br> translation factor activity, RNA binding <br> 4.701e-07","GO:0031683 (20) <br> G-protein beta/gamma-subunit complex binding <br> 9.274e-07","GO:0051015 (213) <br> actin filament binding <br> 2.042e-06","GO:0042610 (4) <br> CD8 receptor binding <br> 2.852e-06","GO:0019899 (2048) <br> enzyme binding <br> 2.972e-06","GO:0003723 (1951) <br> RNA binding <br> 3.047e-06","GO:0090079 (123) <br> translation regulator activity, nucleic acid binding <br> 3.261e-06","GO:0097367 (2282) <br> carbohydrate derivative binding <br> 7.049e-06","GO:0017076 (1932) <br> purine nucleotide binding <br> 7.512e-06","GO:0035639 (1846) <br> purine ribonucleoside triphosphate binding <br> 9.564e-06","GO:0042608 (11) <br> T cell receptor binding <br> 1.075e-05","GO:0043168 (2413) <br> anion binding <br> 1.111e-05","GO:1901265 (2177) <br> nucleoside phosphate binding <br> 1.831e-05","GO:0045182 (154) <br> translation regulator activity <br> 2.329e-05","GO:0042605 (30) <br> peptide antigen binding <br> 3.348e-05","GO:0032555 (1917) <br> purine ribonucleotide binding <br> 3.559e-05","GO:0000166 (2176) <br> nucleotide binding <br> 4.021e-05","GO:0032553 (1934) <br> ribonucleotide binding <br> 4.813e-05","GO:0036094 (2516) <br> small molecule binding <br> 6.621e-05","GO:0098772 (1115) <br> molecular function regulator <br> 1.848e-04","GO:0030276 (70) <br> clathrin binding <br> 8.140e-04","GO:0005102 (1551) <br> signaling receptor binding <br> 1.888e-03","GO:0033218 (387) <br> amide binding <br> 2.211e-03","GO:0042277 (312) <br> peptide binding <br> 2.801e-03","GO:0019900 (757) <br> kinase binding <br> 4.509e-03","GO:0005488 (17070) <br> binding <br> 7.620e-03","GO:0031369 (34) <br> translation initiation factor binding <br> 8.918e-03","GO:0031821 (9) <br> G protein-coupled serotonin receptor binding <br> 9.484e-03","GO:0035254 (48) <br> glutamate receptor binding <br> 1.115e-02","GO:0019901 (677) <br> protein kinase binding <br> 1.157e-02","GO:0042802 (2063) <br> identical protein binding <br> 1.341e-02","GO:0060090 (341) <br> molecular adaptor activity <br> 1.388e-02","GO:0046977 (9) <br> TAP binding <br> 1.433e-02","GO:0030234 (911) <br> enzyme regulator activity <br> 1.466e-02","GO:0060589 (526) <br> nucleoside-triphosphatase regulator activity <br> 2.634e-02","GO:0042609 (9) <br> CD4 receptor binding <br> 2.690e-02","GO:0043021 (150) <br> ribonucleoprotein complex binding <br> 3.126e-02","GO:0019904 (706) <br> protein domain specific binding <br> 3.880e-02","GO:0005198 (701) <br> structural molecule activity <br> 4.455e-02","GO:0005178 (146) <br> integrin binding <br> 4.956e-02"],"key":["GO:0044877","GO:0005515","GO:0017111","GO:0019003","GO:0050839","GO:0003924","GO:0016462","GO:0016818","GO:0016817","GO:0045296","GO:0008092","GO:0003925","GO:0019001","GO:0032561","GO:0005525","GO:0016787","GO:0003743","GO:0003779","GO:0008135","GO:0031683","GO:0051015","GO:0042610","GO:0019899","GO:0003723","GO:0090079","GO:0097367","GO:0017076","GO:0035639","GO:0042608","GO:0043168","GO:1901265","GO:0045182","GO:0042605","GO:0032555","GO:0000166","GO:0032553","GO:0036094","GO:0098772","GO:0030276","GO:0005102","GO:0033218","GO:0042277","GO:0019900","GO:0005488","GO:0031369","GO:0031821","GO:0035254","GO:0019901","GO:0042802","GO:0060090","GO:0046977","GO:0030234","GO:0060589","GO:0042609","GO:0043021","GO:0019904","GO:0005198","GO:0005178"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[19.7686971273735,22.525272898967,18.3167342125249,15.692660304218,18.6510623608111,17.8394624222953,18.4896805188233,18.5199090563365,18.5226205472543,17.9851259338563,19.4390569617437,14.2382740983495,18.2442535356626,18.2442535356626,18.1708988395135,20.4830462803831,15.3054992444299,18.394266179927,16.1868772799724,13.3106655321065,17.3504409902673,9.00890082870231,20.3162578263421,20.258276019327,16.5273136649716,20.4448133217894,20.2465569336091,20.1919193878134,11.980531463383,20.5107536570554,20.3889635367376,16.8705055778228,14.1158314099108,20.2372173548861,20.38841785752,20.2477963305085,20.5599632179123,19.5743368277986,15.6229705518591,19.9812897468844,18.1923005117823,17.8943821822839,19.0830626021267,22.6771653543307,14.3519997480295,11.4827422948031,14.976936429373,18.9384082597841,20.324960558726,18.0180536596155,11.4827422948031,19.3199256512722,18.6063630422794,11.4827422948031,16.8307889373762,18.9928919364036,18.9836734866874,16.7898725428429],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[198.094889825427,191.301129477343,193.957760321422,193.394118699624,196.024446268019,191.229734871916,190.583425812253,199.147020852785,198.838896766201,195.975597327463,191.08694566106,191.962468980254,199.143263241973,196.783483652041,193.935214656551,193.457998083427,190.962944504264,190.579668201441,197.542521036064,191.079430439436,196.170993089686,194.382370343178,194.014124483602,194.138125640398,194.679221597325,199.353689447445,197.756704852347,194.01036687279,190.835185736656,193.597029683471,196.23487247349,190.91785317452,198.034768052435,190.985490169136,195.002376127157,195.16019578126,193.371573034752,194.06673103497,195.904202722035,193.288905596888,193.938972267363,191.458949131447,193.833759164627,195.265408883996,194.14188325121,194.134368029586,193.544423132103,191.613011174739,193.942729878175,193.950245099799,194.431219283734,196.715846657425,197.429792711704,193.875092883559,196.899969587213,199.135748020349,194.713040094633,196.61063355469,193.946487488987,199.624237425908,196.561784614134,193.281390375264,196.896211976401,197.486156873884,191.755800385595,195.194014278568,195.907960332847,194.773161867625,194.600311770273,190.831428125844,193.95400271061,194.619099824333,197.358398106276,193.837516775439,190.906580342084,196.790998873665,193.465513305051,194.089276699842,191.473979574695,195.971839716651,191.128279379992,199.992483285483,193.961517932234,195.182741446132,199.725692917832,195.776443954427,194.570250883777,192.139076688418,194.930981521729,195.998142992335,192.31192678577,199.729450528644,191.695678612603,191.582950288243,198.49695418231,194.258369186382,198.444347630942,192.052651639742,199.868482128688,191.530343736875,191.988772255938,190.850216179904,197.508702538756,197.565066700936,198.399256301198],"y":[7.61541398048559,6.64907080761727,6.58528140780887,6.52608815616478,6.33935381366172,5.95053699798256,5.85584096375178,5.55666275304449,5.3928713005469,5.3871385108041,5.35792029034089,5.06353857121087,4.70464629508662,4.68008717328046,4.58353004226557,4.21990890014134,4.10037109200821,4.00475814611006,3.99807270566049,3.88172627040378,3.84072454747136,3.81331042043964,3.80173063695869,3.73804937752379,3.73122977942463,3.6762253264196,3.6762253264196,3.66545156876769,3.66545156876769,3.65029337803102,3.63405874571657,3.62521950337156,3.59351023720545,3.57391683510823,3.48484689166879,3.47772780470758,3.4745266084865,3.43711349423023,3.36474257344775,3.29124407228518,3.2180974582905,3.18592624460854,3.18285839862643,3.17291240445747,3.12578734106478,3.12578734106478,3.1174998604835,3.1174998604835,2.96845779351046,2.96845779351046,2.85712410454288,2.85431966293935,2.80921357675106,2.79395181811277,2.58176100021435,2.55479172765584,2.50178761300124,2.48916162396131,2.45336000904114,2.42544073409938,2.41673639413656,2.41135376423341,2.37496751495415,2.31200599824027,2.2690803415027,2.25000288336261,2.23954339013264,2.20335802173101,2.17338968060013,2.11987267911907,2.11987267911907,2.11987267911907,2.1186697388065,2.07179997851605,2.0446456777071,2.04159769719265,1.95844546713742,1.94839563263826,1.94573189181207,1.94330693762024,1.93805150041157,1.90381984493856,1.90237725616086,1.85517979336878,1.83598642001148,1.76911945934235,1.74426572680373,1.71508518441283,1.70684693318527,1.67491798319338,1.66214320649394,1.63740858106546,1.63599661962223,1.62121829230774,1.6079684593512,1.59845722533836,1.49546481393848,1.48592691689312,1.42828594728898,1.38070167577592,1.37035401015203,1.3654351726687,1.35339603654533,1.31424651363419,1.30308090640483],"text":["REAC:R-HSA-392518 (32) <br> Signal amplification <br> 2.424e-08","REAC:R-HSA-3858494 (143) <br> Beta-catenin independent WNT signaling <br> 2.244e-07","REAC:R-HSA-202040 (28) <br> G-protein activation <br> 2.598e-07","REAC:R-HSA-1236974 (83) <br> ER-Phagosome pathway <br> 2.978e-07","REAC:R-HSA-6798695 (476) <br> Neutrophil degranulation <br> 4.578e-07","REAC:R-HSA-422475 (549) <br> Axon guidance <br> 1.121e-06","REAC:R-HSA-392170 (22) <br> ADP signalling through P2Y purinoceptor 12 <br> 1.394e-06","REAC:R-HSA-428930 (24) <br> Thromboxane signalling through TP receptor <br> 2.775e-06","REAC:R-HSA-202403 (120) <br> TCR signaling <br> 4.047e-06","REAC:R-HSA-9675108 (574) <br> Nervous system development <br> 4.101e-06","REAC:R-HSA-1236975 (99) <br> Antigen processing-Cross presentation <br> 4.386e-06","REAC:R-HSA-6814122 (42) <br> Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding <br> 8.639e-06","REAC:R-HSA-456926 (31) <br> Thrombin signalling through proteinase activated receptors (PARs) <br> 1.974e-05","REAC:R-HSA-392851 (19) <br> Prostacyclin signalling through prostacyclin receptor <br> 2.089e-05","REAC:R-HSA-418597 (48) <br> G alpha (z) signalling events <br> 2.609e-05","REAC:R-HSA-1236977 (11) <br> Endosomal/Vacuolar pathway <br> 6.027e-05","REAC:R-HSA-1280218 (799) <br> Adaptive Immune System <br> 7.936e-05","REAC:R-HSA-418592 (24) <br> ADP signalling through P2Y purinoceptor 1 <br> 9.891e-05","REAC:R-HSA-422356 (78) <br> Regulation of insulin secretion <br> 1.004e-04","REAC:R-HSA-983170 (25) <br> Antigen Presentation: Folding, assembly and peptide loading of class I MHC <br> 1.313e-04","REAC:R-HSA-111885 (90) <br> Opioid Signalling <br> 1.443e-04","REAC:R-HSA-109582 (673) <br> Hemostasis <br> 1.537e-04","REAC:R-HSA-977443 (60) <br> GABA receptor activation <br> 1.579e-04","REAC:R-HSA-381676 (42) <br> Glucagon-like Peptide-1 (GLP1) regulates insulin secretion <br> 1.828e-04","REAC:R-HSA-168249 (1090) <br> Innate Immune System <br> 1.857e-04","REAC:R-HSA-72649 (59) <br> Translation initiation complex formation <br> 2.108e-04","REAC:R-HSA-72702 (59) <br> Ribosomal scanning and start codon recognition <br> 2.108e-04","REAC:R-HSA-977444 (43) <br> GABA B receptor activation <br> 2.160e-04","REAC:R-HSA-991365 (43) <br> Activation of GABAB receptors <br> 2.160e-04","REAC:R-HSA-9009391 (73) <br> Extra-nuclear estrogen signaling <br> 2.237e-04","REAC:R-HSA-4086400 (92) <br> PCP/CE pathway <br> 2.322e-04","REAC:R-HSA-72662 (60) <br> Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S <br> 2.370e-04","REAC:R-HSA-373755 (64) <br> Semaphorin interactions <br> 2.550e-04","REAC:R-HSA-400042 (28) <br> Adrenaline,noradrenaline inhibits insulin secretion <br> 2.667e-04","REAC:R-HSA-156827 (112) <br> L13a-mediated translational silencing of Ceruloplasmin expression <br> 3.275e-04","REAC:R-HSA-432720 (35) <br> Lysosome Vesicle Biogenesis <br> 3.329e-04","REAC:R-HSA-2682334 (90) <br> EPH-Ephrin signaling <br> 3.353e-04","REAC:R-HSA-72706 (113) <br> GTP hydrolysis and joining of the 60S ribosomal subunit <br> 3.655e-04","REAC:R-HSA-164940 (9) <br> Nef mediated downregulation of MHC class I complex cell surface expression <br> 4.318e-04","REAC:R-HSA-1168372 (80) <br> Downstream signaling events of B Cell Receptor (BCR) <br> 5.114e-04","REAC:R-HSA-8964315 (18) <br> G beta:gamma signalling through BTK <br> 6.052e-04","REAC:R-HSA-5621481 (136) <br> C-type lectin receptors (CLRs) <br> 6.517e-04","REAC:R-HSA-72689 (102) <br> Formation of a pool of free 40S subunits <br> 6.564e-04","REAC:R-HSA-2132295 (123) <br> MHC class II antigen presentation <br> 6.716e-04","REAC:R-HSA-420092 (33) <br> Glucagon-type ligand receptors <br> 7.485e-04","REAC:R-HSA-163359 (33) <br> Glucagon signaling in metabolic regulation <br> 7.485e-04","REAC:R-HSA-72613 (120) <br> Eukaryotic Translation Initiation <br> 7.630e-04","REAC:R-HSA-72737 (120) <br> Cap-dependent Translation Initiation <br> 7.630e-04","REAC:R-HSA-8964616 (20) <br> G beta:gamma signalling through CDC42 <br> 1.075e-03","REAC:R-HSA-418217 (20) <br> G beta:gamma signalling through PLC beta <br> 1.075e-03","REAC:R-HSA-162909 (125) <br> Host Interactions of HIV factors <br> 1.390e-03","REAC:R-HSA-500657 (21) <br> Presynaptic function of Kainate receptors <br> 1.399e-03","REAC:R-HSA-5658442 (67) <br> Regulation of RAS by GAPs <br> 1.552e-03","REAC:R-HSA-72695 (52) <br> Formation of the ternary complex, and subsequently, the 43S complex <br> 1.607e-03","REAC:R-HSA-9649913 (3) <br> RAS GTPase cycle mutants <br> 2.620e-03","REAC:R-HSA-164952 (26) <br> The role of Nef in HIV-1 replication and disease pathogenesis <br> 2.787e-03","REAC:R-HSA-163685 (107) <br> Integration of energy metabolism <br> 3.149e-03","REAC:R-HSA-76002 (258) <br> Platelet activation, signaling and aggregation <br> 3.242e-03","REAC:R-HSA-392451 (25) <br> G beta:gamma signalling through PI3Kgamma <br> 3.521e-03","REAC:R-HSA-432040 (43) <br> Vasopressin regulates renal water homeostasis via Aquaporins <br> 3.755e-03","REAC:R-HSA-202427 (23) <br> Phosphorylation of CD3 and TCR zeta chains <br> 3.831e-03","REAC:R-HSA-202424 (98) <br> Downstream TCR signaling <br> 3.878e-03","REAC:R-HSA-5673001 (264) <br> RAF/MAP kinase cascade <br> 4.217e-03","REAC:R-HSA-211733 (50) <br> Regulation of activated PAK-2p34 by proteasome mediated degradation <br> 4.875e-03","REAC:R-HSA-390466 (95) <br> Chaperonin-mediated protein folding <br> 5.382e-03","REAC:R-HSA-5684996 (269) <br> MAPK1/MAPK3 signaling <br> 5.623e-03","REAC:R-HSA-164938 (20) <br> Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters <br> 5.760e-03","REAC:R-HSA-877300 (87) <br> Interferon gamma signaling <br> 6.261e-03","REAC:R-HSA-5663205 (909) <br> Infectious disease <br> 6.708e-03","REAC:R-HSA-1296041 (29) <br> Activation of G protein gated Potassium channels <br> 7.588e-03","REAC:R-HSA-1296059 (29) <br> G protein gated Potassium channels <br> 7.588e-03","REAC:R-HSA-997272 (29) <br> Inhibition  of voltage gated Ca2+ channels via Gbeta/gamma subunits <br> 7.588e-03","REAC:R-HSA-169911 (53) <br> Regulation of Apoptosis <br> 7.609e-03","REAC:R-HSA-196025 (11) <br> Formation of annular gap junctions <br> 8.476e-03","REAC:R-HSA-451326 (30) <br> Activation of kainate receptors upon glutamate binding <br> 9.023e-03","REAC:R-HSA-391251 (102) <br> Protein folding <br> 9.087e-03","REAC:R-HSA-168275 (2) <br> Entry of Influenza Virion into Host Cell via Endocytosis <br> 1.100e-02","REAC:R-HSA-190873 (12) <br> Gap junction degradation <br> 1.126e-02","REAC:R-HSA-5621575 (20) <br> CD209 (DC-SIGN) signaling <br> 1.133e-02","REAC:R-HSA-373753 (23) <br> Nephrin family interactions <br> 1.139e-02","REAC:R-HSA-445717 (52) <br> Aquaporin-mediated transport <br> 1.153e-02","REAC:R-HSA-199992 (72) <br> trans-Golgi Network Vesicle Budding <br> 1.248e-02","REAC:R-HSA-397795 (32) <br> G-protein beta:gamma signalling <br> 1.252e-02","REAC:R-HSA-5683057 (308) <br> MAPK family signaling cascades <br> 1.396e-02","REAC:R-HSA-5140745 (13) <br> WNT5A-dependent internalization of FZD2, FZD5 and ROR2 <br> 1.459e-02","REAC:R-HSA-5676590 (59) <br> NIK-->noncanonical NF-kB signaling <br> 1.702e-02","REAC:R-HSA-168256 (2038) <br> Immune System <br> 1.802e-02","REAC:R-HSA-5607761 (60) <br> Dectin-1 mediated noncanonical NF-kB signaling <br> 1.927e-02","REAC:R-HSA-1296065 (35) <br> Inwardly rectifying K+ channels <br> 1.964e-02","REAC:R-HSA-112316 (400) <br> Neuronal System <br> 2.114e-02","REAC:R-HSA-5678895 (61) <br> Defective CFTR causes cystic fibrosis <br> 2.177e-02","REAC:R-HSA-5099900 (15) <br> WNT5A-dependent internalization of FZD4 <br> 2.305e-02","REAC:R-HSA-202733 (190) <br> Cell surface interactions at the vascular wall <br> 2.312e-02","REAC:R-HSA-4086398 (59) <br> Ca2+ pathway <br> 2.392e-02","REAC:R-HSA-983705 (163) <br> Signaling by the B Cell Receptor (BCR) <br> 2.466e-02","REAC:R-HSA-8856688 (134) <br> Golgi-to-ER retrograde transport <br> 2.521e-02","REAC:R-HSA-195721 (328) <br> Signaling by WNT <br> 3.195e-02","REAC:R-HSA-2424491 (25) <br> DAP12 signaling <br> 3.266e-02","REAC:R-HSA-171007 (11) <br> p38MAPK events <br> 3.730e-02","REAC:R-HSA-6807878 (102) <br> COPI-mediated anterograde transport <br> 4.162e-02","REAC:R-HSA-1236978 (50) <br> Cross-presentation of soluble exogenous antigens (endosomes) <br> 4.262e-02","REAC:R-HSA-1169091 (67) <br> Activation of NF-kappaB in B cells <br> 4.311e-02","REAC:R-HSA-9010553 (171) <br> Regulation of expression of SLITs and ROBOs <br> 4.432e-02","REAC:R-HSA-350562 (51) <br> Regulation of ornithine decarboxylase (ODC) <br> 4.850e-02","REAC:R-HSA-376176 (217) <br> Signaling by ROBO receptors <br> 4.976e-02"],"key":["REAC:R-HSA-392518","REAC:R-HSA-3858494","REAC:R-HSA-202040","REAC:R-HSA-1236974","REAC:R-HSA-6798695","REAC:R-HSA-422475","REAC:R-HSA-392170","REAC:R-HSA-428930","REAC:R-HSA-202403","REAC:R-HSA-9675108","REAC:R-HSA-1236975","REAC:R-HSA-6814122","REAC:R-HSA-456926","REAC:R-HSA-392851","REAC:R-HSA-418597","REAC:R-HSA-1236977","REAC:R-HSA-1280218","REAC:R-HSA-418592","REAC:R-HSA-422356","REAC:R-HSA-983170","REAC:R-HSA-111885","REAC:R-HSA-109582","REAC:R-HSA-977443","REAC:R-HSA-381676","REAC:R-HSA-168249","REAC:R-HSA-72649","REAC:R-HSA-72702","REAC:R-HSA-977444","REAC:R-HSA-991365","REAC:R-HSA-9009391","REAC:R-HSA-4086400","REAC:R-HSA-72662","REAC:R-HSA-373755","REAC:R-HSA-400042","REAC:R-HSA-156827","REAC:R-HSA-432720","REAC:R-HSA-2682334","REAC:R-HSA-72706","REAC:R-HSA-164940","REAC:R-HSA-1168372","REAC:R-HSA-8964315","REAC:R-HSA-5621481","REAC:R-HSA-72689","REAC:R-HSA-2132295","REAC:R-HSA-420092","REAC:R-HSA-163359","REAC:R-HSA-72613","REAC:R-HSA-72737","REAC:R-HSA-8964616","REAC:R-HSA-418217","REAC:R-HSA-162909","REAC:R-HSA-500657","REAC:R-HSA-5658442","REAC:R-HSA-72695","REAC:R-HSA-9649913","REAC:R-HSA-164952","REAC:R-HSA-163685","REAC:R-HSA-76002","REAC:R-HSA-392451","REAC:R-HSA-432040","REAC:R-HSA-202427","REAC:R-HSA-202424","REAC:R-HSA-5673001","REAC:R-HSA-211733","REAC:R-HSA-390466","REAC:R-HSA-5684996","REAC:R-HSA-164938","REAC:R-HSA-877300","REAC:R-HSA-5663205","REAC:R-HSA-1296041","REAC:R-HSA-1296059","REAC:R-HSA-997272","REAC:R-HSA-169911","REAC:R-HSA-196025","REAC:R-HSA-451326","REAC:R-HSA-391251","REAC:R-HSA-168275","REAC:R-HSA-190873","REAC:R-HSA-5621575","REAC:R-HSA-373753","REAC:R-HSA-445717","REAC:R-HSA-199992","REAC:R-HSA-397795","REAC:R-HSA-5683057","REAC:R-HSA-5140745","REAC:R-HSA-5676590","REAC:R-HSA-168256","REAC:R-HSA-5607761","REAC:R-HSA-1296065","REAC:R-HSA-112316","REAC:R-HSA-5678895","REAC:R-HSA-5099900","REAC:R-HSA-202733","REAC:R-HSA-4086398","REAC:R-HSA-983705","REAC:R-HSA-8856688","REAC:R-HSA-195721","REAC:R-HSA-2424491","REAC:R-HSA-171007","REAC:R-HSA-6807878","REAC:R-HSA-1236978","REAC:R-HSA-1169091","REAC:R-HSA-9010553","REAC:R-HSA-350562","REAC:R-HSA-376176"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,102,204,1)","opacity":0.8,"size":[14.2382740983495,16.7583551431596,13.9833131530912,15.9033751977876,18.4728732854611,18.6631932828122,13.5059251363754,13.6808214449962,16.4890463101257,18.7220962814027,16.1868772799724,14.73917293128,14.1782205386673,13.2039079443259,14.976936429373,11.980531463383,19.1525079244994,13.6808214449962,15.801858092204,13.7618192670139,16.0344085591459,18.9306948804635,15.3633750239743,14.73917293128,19.5459903341358,15.3347186146137,15.3347186146137,14.781443852525,14.781443852525,15.692660304218,16.0697362528688,15.3633750239743,15.4727616697066,13.9833131530912,16.3815087539172,14.405947329161,16.0344085591459,16.3954151879507,11.4827422948031,15.8433279821367,13.0900518224006,16.6818474191902,16.2342495236679,16.5273136649716,14.2961523059962,14.2961523059962,16.4890463101257,16.4890463101257,13.3106655321065,13.3106655321065,16.552248256339,13.4111152730112,15.5497876137808,15.1170689685676,7.7791008157754,13.83902557929,16.309816295931,17.6262397302166,13.7618192670139,14.781443852525,13.5956642813554,16.1707256977144,17.6589522750826,15.0486221338682,16.121131665261,17.6855927091729,13.3106655321065,15.9797164726717,19.3171356175398,14.0509285588584,14.0509285588584,14.0509285588584,15.1501626734972,11.980531463383,14.1158314099108,16.2342495236679,3.77952755905512,12.187219027138,13.3106655321065,13.5956642813554,15.1170689685676,15.6697988614665,14.2382740983495,17.8763374918434,12.3729643352122,15.3347186146137,20.3104179536764,15.3633750239743,14.405947329161,18.2374501939285,15.3914887766185,12.6954028668369,17.1833157067251,15.3347186146137,16.9558134205769,16.6591768768421,17.9641020663474,13.7618192670139,11.980531463383,16.2342495236679,15.0486221338682,15.5497876137808,17.0273503124218,15.0832328994516,17.3774579655542],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,102,204,1)"}},"hoveron":"points","set":"SharedDatad1ba32e0","name":"REAC","legendgroup":"REAC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"255c4bd25087":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"255c3a142fe0":{"x":{},"xend":{},"y":{},"yend":{}},"255c2b5dfcc":{"x":{},"xend":{},"y":{},"yend":{}},"255c280f5570":{"x":{},"xend":{},"y":{},"yend":{}},"255c40e9281d":{"x":{},"xend":{},"y":{},"yend":{}},"255c1cab698f":{"x":{},"y":{}},"255c4796299f":{"yintercept":{}}},"cur_data":"255c4bd25087","visdat":{"255c4bd25087":["function (y) ","x"],"255c3a142fe0":["function (y) ","x"],"255c2b5dfcc":["function (y) ","x"],"255c280f5570":["function (y) ","x"],"255c40e9281d":["function (y) ","x"],"255c1cab698f":["function (y) ","x"],"255c4796299f":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedDatad1ba32e0"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;%
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-21697d58abce2053a245" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-21697d58abce2053a245">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546","547","548","549","550","551","552","553","554","555","556","557","558","559","560","561","562","563","564","565","566","567","568","569","570","571","572","573","574","575","576","577","578","579","580","581","582","583","584","585","586","587","588","589","590","591","592","593","594","595","596","597","598","599","600","601","602","603","604","605","606","607","608","609","610","611","612","613","614","615"],["GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","REAC","GO:BP","REAC","GO:CC","REAC","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:MF","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","REAC","GO:BP","GO:MF","GO:BP","GO:CC","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","REAC","REAC","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:MF","GO:CC","REAC","GO:CC","GO:BP","GO:MF","GO:CC","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","REAC","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:MF","REAC","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:BP","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","REAC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:MF","REAC","GO:BP","GO:BP","GO:CC","REAC","REAC","REAC","REAC","REAC","REAC","GO:CC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","REAC","REAC","REAC","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","REAC","REAC","REAC","GO:BP","GO:BP","REAC","REAC","GO:CC","REAC","REAC","GO:CC","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","REAC","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","REAC","REAC","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:CC","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:BP","GO:CC","GO:BP","REAC","REAC","REAC","GO:CC","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","REAC","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:CC","REAC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:BP","REAC","REAC","REAC","REAC","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:MF","REAC","REAC","GO:MF","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:MF","GO:BP","REAC","REAC","REAC","REAC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:MF","GO:CC","GO:MF","REAC","GO:MF","GO:BP","REAC","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:CC","GO:CC","REAC","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:BP","REAC","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","REAC","REAC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","REAC","REAC","GO:BP","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","REAC","REAC","GO:CC","GO:CC","GO:BP","GO:BP","REAC","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","REAC","GO:CC","GO:BP","REAC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:MF","REAC","GO:BP"],[2743,4056,2131,2719,1453,278,550,1027,2765,7129,850,241,1322,3497,385,1026,7017,1118,3551,561,16046,5762,470,14527,1133,1570,515,785,7019,3561,1125,23108,14485,9160,3132,13259,12268,2843,3780,3627,16054,939,1099,1247,15694,13142,1038,1879,15738,12281,10980,2779,947,474,4087,1101,3628,1080,3658,2437,1030,11579,927,818,3582,5659,983,938,3982,15417,4447,1098,3542,9081,916,19815,598,984,10009,19079,9231,1261,3547,817,2901,1648,3785,2128,1412,3784,1408,1217,861,489,2127,5665,190,7174,2921,16072,1401,1084,3583,531,1134,1121,15359,2093,561,3072,12668,2029,2552,3860,11868,1959,599,14486,1129,837,11574,1334,3654,410,861,418,284,13203,4445,5413,20427,5363,15361,2009,1078,1889,2895,1075,3068,917,1332,7625,19740,15416,2879,2055,14309,3758,3071,3552,7622,2041,1138,15644,450,413,7740,201,15428,908,2056,758,22060,2447,889,417,2130,29136,29126,468,7698,1458,1993,1128,886,1402,2496,409,67,1073,13553,3575,5104,19737,182,26123,2842,7607,7021,10,7018,11210,1124,9093,1150,2289,1135,6670,1159,4001,4618,7705,438,14889,26125,29092,10208,5837,1864,2207,1445,144,7633,2566,7605,16045,1240,1057,2543,1149,1263,193,15587,18664,19802,1262,10338,3959,20,377,795,19397,6342,791,6668,885,6775,12669,9881,1025,859,14885,11494,11946,2288,3799,9109,10505,1660,27326,3079,2132,7162,902,9696,1117,3624,12553,1085,15435,10495,6667,5407,3168,35,4320,17158,819,13161,5405,2896,4442,1265,13160,775,6495,1058,1476,6389,16076,15204,111,19430,9,1344,1862,537,27328,13159,9110,442,142,2416,3019,1497,16061,1119,1021,923,438,3169,655,72,956,14646,10428,1100,19167,2946,1060,2344,1919,922,77,812,1514,492,99,1547,7619,28343,26826,1993,117,1528,2590,2061,18660,2450,10013,1186,1228,752,28341,937,14941,7618,2565,26828,2300,11569,15318,1426,15750,1111,15430,17166,656,20429,730,903,243,875,1256,22049,16052,957,955,1276,798,284,1123,4791,1141,1139,185,3698,7747,3068,2542,904,906,3990,1150,15646,2898,1074,2444,1034,1642,1474,15368,12675,1832,886,184,3391,1070,1606,658,11587,19431,532,3077,17436,7628,5468,1100,3796,20868,7671,3357,788,8306,21226,15932,1691,28315,2286,6616,22820,22821,822,24935,3067,1109,8059,1614,1346,862,16937,905,2416,1601,532,728,15459,2897,1239,23322,1076,1690,1072,7020,4619,2406,26164,21250,1455,3788,1847,6460,12744,98,8301,15432,322,18665,1237,1226,1427,3612,7699,8060,1125,871,606,15765,1079,2862,534,1277,15747,76,907,1084,1813,1862,446,9187,306,3171,1964,8304,876,15752,5061,96,1662,5233,4089,652,2094,1238,12824,651,4509,15729,513,654,1349,777,2949,6278,7836,943,247,1444,155,4620,8775,5675,15822,12169,24824,2514,909,6689,192,9594,1234,7367,738,2443,4773,20461,9709,17818,3663,13022,3123,1392,1414,784,1071,2495,11679,3078,424,24936,433,21383,1167,1763,9174,1065,27851,3638,7604,1451,470,2405,3136,8761,514,7837,2444,306,11657,11212,276,2116,15586,17191,427,988,21897,1077,9338,10046,9603,8133,26562,14940,6669,12732,2022,516,1267,2597,19047,4593,2090,7709,12671,6756,538,18764,2102,401,415,1079,12241,1355,2481,4623,7675,1480,3138,8667,865,262,478,29347,19045,13178,384,1014,81,1421,15730,1853,14697,1694,29349,25857,15301,14310,1868,11087,2507,3644,1677,2090,29917],["extracellular exosome","extracellular vesicle","extracellular organelle","extracellular membrane-bounded organelle","vesicle","extracellular space","focal adhesion","cell-substrate junction","anchoring junction","protein-containing complex binding","membrane","extracellular region","cytoplasmic vesicle","intracellular vesicle","cytoplasm","cell junction","antigen processing and presentation","cytoplasmic vesicle membrane","side of membrane","cell activation","cellular localization","vesicle-mediated transport","cytosol","antigen processing and presentation of peptide antigen","antigen processing and presentation of exogenous peptide antigen","protein-containing complex","plasma membrane","cytoplasmic side of plasma membrane","antigen processing and presentation of exogenous antigen","cytoplasmic side of membrane","secretory granule membrane","export from cell","secretion","secretion by cell","cell periphery","leukocyte activation","leukocyte degranulation","exocytosis","secretory vesicle","membrane protein complex","establishment of localization in cell","myeloid cell activation involved in immune response","myeloid leukocyte mediated immunity","extrinsic component of cytoplasmic side of plasma membrane","localization","regulated exocytosis","immune system process","protein binding","establishment of localization","neutrophil degranulation","granulocyte activation","transport","neutrophil activation involved in immune response","heterotrimeric G-protein complex","GTPase complex","neutrophil mediated immunity","plasma membrane protein complex","secretory granule","membrane microdomain","membrane raft","leukocyte activation involved in immune response","neutrophil activation","cell activation involved in immune response","vesicle membrane","bounding membrane of organelle","viral process","extrinsic component of plasma membrane","myeloid leukocyte activation","nucleoside-triphosphatase activity","T cell receptor signaling pathway","GDP binding","leukocyte mediated immunity","protein localization","cell adhesion molecule binding","immune effector process","cellular component organization or biogenesis","GTPase activity","extrinsic component of membrane","cellular protein localization","cellular macromolecule localization","macromolecule localization","cell leading edge","pyrophosphatase activity","endomembrane system","immune response","eukaryotic 48S preinitiation complex","hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides","membrane-bounded organelle","immune response-regulating cell surface receptor signaling pathway","hydrolase activity, acting on acid anhydrides","immune response-regulating signaling pathway","organelle membrane","eukaryotic 43S preinitiation complex","eukaryotic translation initiation factor 3 complex","organelle","cellular component organization","immunological synapse","cadherin binding","translation preinitiation complex","localization within membrane","immune response-activating signal transduction","immune response-activating cell surface receptor signaling pathway","plasma membrane region","formation of cytoplasmic translation initiation complex","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent","coated vesicle membrane","regulation of immune response","cell projection","cell cortex","leukocyte cell-cell adhesion","cellular component biogenesis","melanosome","pigment granule","plasma membrane bounded cell projection","antigen processing and presentation of exogenous peptide antigen via MHC class I","cytoskeletal protein binding","G protein activity","intracellular transport","antigen processing and presentation of peptide antigen via MHC class I","actin cytoskeleton","T cell activation","positive regulation of immune system process","lytic vacuole membrane","lysosomal membrane","cytoplasmic translational initiation","vacuolar membrane","intracellular anatomical structure","establishment of protein localization","guanyl nucleotide binding","guanyl ribonucleotide binding","protein localization to membrane","protein transport","positive regulation of immune response","Signal amplification","endocytic vesicle","GTP binding","tertiary granule","coated vesicle","cell adhesion","activation of immune response","regulation of immune system process","biological adhesion","nitrogen compound transport","antigen receptor-mediated signaling pathway","movement of cell or subcellular component","MHC protein complex","lymphocyte activation","hydrolase activity","integral component of lumenal side of endoplasmic reticulum membrane","lumenal side of endoplasmic reticulum membrane","cellular component assembly","specific granule","antigen processing and presentation of endogenous peptide antigen","regulation of cellular component organization","translation initiation factor activity","endosome","regulation of cell adhesion","Beta-catenin independent WNT signaling","regulation of T cell activation","G-protein activation","MHC class I protein complex","ER-Phagosome pathway","cell-cell adhesion","synapse","stimulatory C-type lectin receptor signaling pathway","vacuole","intracellular organelle","cellular response to lectin","response to lectin","actin binding","actin filament-based process","Neutrophil degranulation","translation factor activity, RNA binding","phagocytic vesicle membrane","innate immune response activating cell surface receptor signaling pathway","innate immune response-activating signal transduction","phagocytic vesicle","lysosome","lytic vacuole","transport vesicle","positive regulation of cell adhesion","lumenal side of membrane","G-protein beta/gamma-subunit complex binding","organic substance transport","Axon guidance","regulation of leukocyte cell-cell adhesion","intracellular protein transport","positive regulation of cell-cell adhesion","antigen processing and presentation of exogenous peptide antigen via MHC class II","ADP signalling through P2Y purinoceptor 12","antigen processing and presentation of endogenous antigen","Fc receptor signaling pathway","endocytic vesicle membrane","actin filament binding","antigen processing and presentation of peptide antigen via MHC class II","Thromboxane signalling through TP receptor","antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-independent","CD8 receptor binding","antigen processing and presentation of peptide or polysaccharide antigen via MHC class II","catalytic complex","enzyme binding","actin cytoskeleton organization","RNA binding","positive regulation of cellular process","positive regulation of leukocyte cell-cell adhesion","protein localization to cell periphery","translation regulator activity, nucleic acid binding","cell migration","specific granule membrane","TCR signaling","Nervous system development","Antigen processing-Cross presentation","ribonucleoprotein complex assembly","recycling endosome membrane","regulation of cell-cell adhesion","organelle localization","intrinsic component of endoplasmic reticulum membrane","membrane coat","coated membrane","cortical cytoskeleton","cell trailing edge","uropod","positive regulation of transport","regulation of biological quality","ribonucleoprotein complex subunit organization","cell projection membrane","carbohydrate derivative binding","purine nucleotide binding","Golgi membrane","Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding","endosome membrane","cellular response to organic substance","purine ribonucleoside triphosphate binding","cell surface","T cell receptor binding","activation of innate immune response","anion binding","regulation of cellular component biogenesis","response to interferon-gamma","lamellipodium","cytoplasmic translation","positive regulation of biological process","locomotion","nucleoside phosphate binding","Thrombin signalling through proteinase activated receptors (PARs)","cell cortex region","regulation of localization","non-canonical Wnt signaling pathway","Prostacyclin signalling through prostacyclin receptor","regulation of protein localization to cell periphery","cell surface receptor signaling pathway","intracellular membrane-bounded organelle","translation regulator activity","G alpha (z) signalling events","response to cytokine","transport vesicle membrane","Golgi apparatus subcompartment","protein-containing complex subunit organization","integral component of endoplasmic reticulum membrane","positive regulation of T cell activation","intracellular signal transduction","peptide antigen binding","purine ribonucleotide binding","small GTPase mediated signal transduction","nucleotide binding","response to organic substance","interferon-gamma-mediated signaling pathway","ER to Golgi transport vesicle membrane","positive regulation of innate immune response","ribonucleotide binding","tertiary granule membrane","ribonucleoprotein complex","leading edge membrane","regulation of innate immune response","Endosomal/Vacuolar pathway","cytokine-mediated signaling pathway","clathrin coat","positive regulation of response to biotic stimulus","small molecule binding","localization of cell","cell motility","Adaptive Immune System","cellular response to cytokine stimulus","ADP signalling through P2Y purinoceptor 1","regulation of leukocyte activation","Regulation of insulin secretion","cell-cell junction","positive regulation of protein localization to cell periphery","innate immune response","regulation of protein localization","Golgi-associated vesicle","Antigen Presentation: Folding, assembly and peptide loading of class I MHC","translational initiation","nuclear migration","Opioid Signalling","establishment of organelle localization","Golgi-associated vesicle membrane","Hemostasis","GABA receptor activation","Golgi apparatus","Ras protein signal transduction","T cell mediated cytotoxicity","MAPK cascade","Glucagon-like Peptide-1 (GLP1) regulates insulin secretion","Golgi vesicle transport","molecular function regulator","Innate Immune System","cellular response to chemical stimulus","cytoskeleton organization","vesicle coat","Translation initiation complex formation","Ribosomal scanning and start codon recognition","GABA B receptor activation","Activation of GABAB receptors","Extra-nuclear estrogen signaling","PCP/CE pathway","cytoskeleton","Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S","trans-Golgi network membrane","regulation of cell morphogenesis","positive regulation of protein localization to membrane","regulation of cellular protein localization","Semaphorin interactions","Adrenaline,noradrenaline inhibits insulin secretion","actin filament bundle","protein targeting","actomyosin","protein-containing complex assembly","protein folding","cellular protein-containing complex assembly","L13a-mediated translational silencing of Ceruloplasmin expression","Lysosome Vesicle Biogenesis","EPH-Ephrin signaling","regulation of protein localization to membrane","GTP hydrolysis and joining of the 60S ribosomal subunit","positive regulation of response to stimulus","regulation of anatomical structure morphogenesis","recycling endosome","positive regulation of cellular protein localization","cell-cell contact zone","T cell proliferation","regulation of defense response to virus by virus","Nef mediated downregulation of MHC class I complex cell surface expression","regulation of lymphocyte activation","T cell mediated immunity","regulation of cell activation","regulation of cellular localization","regulation of T cell mediated cytotoxicity","protein localization to plasma membrane","Downstream signaling events of B Cell Receptor (BCR)","G beta:gamma signalling through BTK","C-type lectin receptors (CLRs)","Formation of a pool of free 40S subunits","MHC class II antigen presentation","defense response to other organism","nucleus localization","Glucagon-type ligand receptors","Glucagon signaling in metabolic regulation","intrinsic component of organelle membrane","Eukaryotic Translation Initiation","Cap-dependent Translation Initiation","clathrin-coated vesicle membrane","clathrin binding","antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway, TAP-independent","antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway","ruffle","glutamatergic synapse","platelet activation","eukaryotic translation initiation factor 3 complex, eIF3m","perinuclear region of cytoplasm","G beta:gamma signalling through CDC42","G beta:gamma signalling through PLC beta","response to external stimulus","cortical actin cytoskeleton","positive regulation of cellular component organization","defense response","COPII-coated ER to Golgi transport vesicle","basal part of cell","Host Interactions of HIV factors","Presynaptic function of Kainate receptors","regulation of response to biotic stimulus","regulation of catalytic activity","positive regulation of molecular function","Regulation of RAS by GAPs","Formation of the ternary complex, and subsequently, the 43S complex","stress fiber","contractile actin filament bundle","clathrin coat of trans-Golgi network vesicle","signaling receptor binding","positive regulation of T cell mediated cytotoxicity","regulation of T cell proliferation","cellular response to interferon-gamma","establishment of planar polarity","establishment of tissue polarity","regulation of vesicle-mediated transport","ribonucleoprotein complex biogenesis","amide binding","dendrite","postsynaptic endocytic zone cytoplasmic component","viral translational termination-reinitiation","signaling","dendritic tree","basal plasma membrane","positive regulation of defense response","establishment of protein localization to membrane","regulation of cytoskeleton organization","RAS GTPase cycle mutants","regulation of Fc-gamma receptor signaling pathway involved in phagocytosis","The role of Nef in HIV-1 replication and disease pathogenesis","peptide binding","regulation of plasma membrane bounded cell projection organization","plasma membrane bounded cell projection organization","trans-Golgi network transport vesicle membrane","response to nitrogen compound","cell communication","Integration of energy metabolism","cortical cytoskeleton organization","Platelet activation, signaling and aggregation","positive regulation of leukocyte activation","basolateral plasma membrane","Wnt signaling pathway, planar cell polarity pathway","G beta:gamma signalling through PI3Kgamma","Vasopressin regulates renal water homeostasis via Aquaporins","Phosphorylation of CD3 and TCR zeta chains","brush border","Downstream TCR signaling","leukocyte migration","response to stress","intrinsic component of plasma membrane","cell-cell signaling by wnt","clathrin-coated vesicle","RAF/MAP kinase cascade","clathrin coat of coated pit","antigen processing and presentation of endogenous peptide antigen via MHC class I","kinase binding","mRNA catabolic process","positive regulation of protein localization to plasma membrane","regulation of establishment of planar polarity","organelle subcompartment","cytoplasmic region","Regulation of activated PAK-2p34 by proteasome mediated degradation","viral translation","cellular protein metabolic process","proteasome complex","regulation of cell projection organization","positive regulation of cell activation","Chaperonin-mediated protein folding","regulation of molecular function","MAPK1/MAPK3 signaling","pseudopodium","Nef-mediates down modulation of cell surface receptors by recruiting them to clathrin adapters","regulation of cell shape","cell projection organization","cortical actin cytoskeleton organization","Interferon gamma signaling","myosin complex","COP9 signalosome","regulation of cellular component movement","Infectious disease","phagocytosis","morphogenesis of a polarized epithelium","integral component of organelle membrane","regulation of protein metabolic process","Activation of G protein gated Potassium channels","G protein gated Potassium channels","Inhibition  of voltage gated Ca2+ channels via Gbeta/gamma subunits","Regulation of Apoptosis","binding","trans-Golgi network","regulation of actin filament-based process","cell morphogenesis","cell-cell signaling","somatodendritic compartment","regulation of defense response","Formation of annular gap junctions","positive regulation of lymphocyte activation","translation initiation factor binding","Activation of kainate receptors upon glutamate binding","Protein folding","G protein-coupled serotonin receptor binding","endopeptidase complex","regulation of leukocyte mediated cytotoxicity","neuron projection","anchored component of membrane","biological process involved in interspecies interaction between organisms","leukocyte mediated cytotoxicity","response to organonitrogen compound","positive regulation of protein transport","actin filament","positive regulation of leukocyte mediated cytotoxicity","positive regulation of immune effector process","Entry of Influenza Virion into Host Cell via Endocytosis","actin filament organization","glutamate receptor binding","regulation of cell migration","Gap junction degradation","CD209 (DC-SIGN) signaling","Nephrin family interactions","Aquaporin-mediated transport","protein kinase binding","lysosome localization","Wnt signaling pathway","regulation of hydrolase activity","positive regulation of catalytic activity","organonitrogen compound metabolic process","trans-Golgi Network Vesicle Budding","G-protein beta:gamma signalling","identical protein binding","photoreceptor inner segment","molecular adaptor activity","MAPK family signaling cascades","TAP binding","morphogenesis of an epithelium","WNT5A-dependent internalization of FZD2, FZD5 and ROR2","enzyme regulator activity","protein localization to cell cortex","heterotypic cell-cell adhesion","membrane organization","cluster of actin-based cell projections","mitotic cell cycle phase transition","dopamine receptor signaling pathway","NIK--&gt;noncanonical NF-kB signaling","G-protein beta/gamma-subunit complex","external side of plasma membrane","Immune System","clathrin-coated endocytic vesicle","protection from natural killer cell mediated cytotoxicity","signal transduction","Dectin-1 mediated noncanonical NF-kB signaling","cellular response to nitrogen compound","endoplasmic reticulum membrane","positive regulation of protein targeting to membrane","Inwardly rectifying K+ channels","integrin alphaL-beta2 complex","regulation of actin cytoskeleton organization","clathrin vesicle coat","positive regulation of establishment of protein localization","endoplasmic reticulum subcompartment","membrane docking","Neuronal System","Defective CFTR causes cystic fibrosis","RNA catabolic process","Ragulator complex","positive regulation of intracellular transport","Arp2/3 protein complex","positive regulation of cell migration","WNT5A-dependent internalization of FZD4","Cell surface interactions at the vascular wall","response to chemical","Fc-epsilon receptor signaling pathway","Ca2+ pathway","Signaling by the B Cell Receptor (BCR)","regulation of transport","regulation of Fc receptor mediated stimulatory signaling pathway","endoplasmic reticulum","Golgi-to-ER retrograde transport","supramolecular fiber organization","COPI-coated vesicle","regulation of cellular amine metabolic process","response to prostaglandin E","nucleoside-triphosphatase regulator activity","actomyosin structure organization","regulation of protein targeting","regulation of response to stimulus","CD4 receptor binding","cellular catabolic process","nuclear outer membrane-endoplasmic reticulum membrane network","integral component of plasma membrane","lamellipodium membrane","protein targeting to membrane","regulation of leukocyte proliferation","regulation of G2/M transition of mitotic cell cycle","MHC class I peptide loading complex","actin filament depolymerization","positive regulation of cellular component biogenesis","ribonucleoprotein complex binding","adherens junction","regulation of establishment of protein localization","Signaling by WNT","DAP12 signaling","late endosome","trans-Golgi network transport vesicle","regulation of protein-containing complex disassembly","positive regulation of leukocyte mediated immunity","p38MAPK events","protein domain specific binding","positive regulation of signaling","cell division site","integrin-mediated signaling pathway","regulation of cellular protein metabolic process","lateral plasma membrane","COPI-mediated anterograde transport","proteasome regulatory particle","regulation of cell motility","leukocyte proliferation","cellular extravasation","Cross-presentation of soluble exogenous antigens (endosomes)","cytosolic large ribosomal subunit","Activation of NF-kappaB in B cells","late endosome membrane","regulation of protein transport","Regulation of expression of SLITs and ROBOs","clathrin coat assembly","structural molecule activity","positive regulation of cell motility","regulation of cell cycle G2/M phase transition","regulation of lymphocyte proliferation","lymphocyte proliferation","Regulation of ornithine decarboxylase (ODC)","pre-replicative complex assembly","regulation of cellular amino acid metabolic process","presynaptic endocytic zone membrane","integrin binding","Signaling by ROBO receptors","regulation of stem cell differentiation"],["4.3e-51","5.7e-49","6.3e-49","6.3e-49","7.6e-36","1.8e-27","5.1e-22","1.0e-21","2.4e-21","2.8e-21","1.2e-20","1.0e-19","1.5e-19","1.7e-19","2.9e-18","3.8e-18","6.0e-18","8.0e-18","1.1e-17","1.6e-17","1.0e-16","1.3e-16","1.4e-16","1.7e-16","1.8e-16","1.9e-16","2.1e-16","3.0e-16","5.7e-16","6.4e-16","6.7e-16","7.4e-16","1.0e-15","1.0e-15","1.2e-15","1.5e-15","1.7e-15","1.9e-15","2.4e-15","3.1e-15","4.4e-15","4.4e-15","5.0e-15","6.3e-15","9.3e-15","1.2e-14","1.5e-14","1.9e-14","4.4e-14","4.9e-14","5.0e-14","5.1e-14","7.0e-14","7.5e-14","7.5e-14","8.8e-14","1.4e-13","1.5e-13","1.5e-13","1.5e-13","1.7e-13","1.9e-13","2.2e-13","3.6e-13","4.9e-13","5.6e-13","1.1e-12","1.2e-12","1.3e-12","2.4e-12","2.7e-12","2.7e-12","1.0e-11","1.1e-11","1.2e-11","1.3e-11","1.5e-11","1.7e-11","1.8e-11","2.8e-11","2.9e-11","3.4e-11","4.9e-11","5.2e-11","5.4e-11","6.1e-11","9.3e-11","9.4e-11","9.7e-11","9.9e-11","1.1e-10","1.2e-10","1.8e-10","1.8e-10","1.9e-10","2.1e-10","2.4e-10","2.5e-10","3.0e-10","4.3e-10","4.4e-10","4.4e-10","8.4e-10","1.1e-09","1.3e-09","1.7e-09","1.7e-09","2.7e-09","2.7e-09","2.8e-09","2.9e-09","3.0e-09","3.0e-09","3.3e-09","3.4e-09","3.5e-09","3.9e-09","4.0e-09","4.8e-09","5.1e-09","5.3e-09","6.6e-09","7.1e-09","7.1e-09","1.0e-08","1.2e-08","1.2e-08","1.3e-08","1.6e-08","1.6e-08","1.7e-08","1.8e-08","1.8e-08","2.4e-08","2.5e-08","2.6e-08","2.8e-08","3.0e-08","4.6e-08","4.7e-08","5.3e-08","5.6e-08","6.2e-08","6.4e-08","7.4e-08","7.9e-08","9.5e-08","1.1e-07","1.1e-07","1.1e-07","1.1e-07","1.2e-07","1.2e-07","1.2e-07","1.3e-07","1.3e-07","1.9e-07","2.2e-07","2.3e-07","2.6e-07","3.0e-07","3.0e-07","3.3e-07","3.4e-07","3.8e-07","4.0e-07","4.3e-07","4.3e-07","4.3e-07","4.5e-07","4.6e-07","4.6e-07","4.7e-07","4.8e-07","5.5e-07","6.1e-07","6.6e-07","6.7e-07","6.7e-07","6.9e-07","7.2e-07","9.2e-07","9.3e-07","9.4e-07","1.1e-06","1.2e-06","1.3e-06","1.3e-06","1.4e-06","1.4e-06","1.5e-06","1.8e-06","2.0e-06","2.0e-06","2.1e-06","2.8e-06","2.8e-06","2.9e-06","2.9e-06","3.0e-06","3.0e-06","3.0e-06","3.0e-06","3.1e-06","3.2e-06","3.2e-06","3.3e-06","3.7e-06","3.8e-06","4.0e-06","4.1e-06","4.4e-06","4.4e-06","4.8e-06","4.8e-06","4.9e-06","5.0e-06","5.0e-06","5.0e-06","5.4e-06","5.5e-06","5.5e-06","6.2e-06","6.2e-06","6.6e-06","6.8e-06","7.0e-06","7.5e-06","7.9e-06","8.6e-06","8.7e-06","8.7e-06","9.6e-06","1.0e-05","1.1e-05","1.1e-05","1.1e-05","1.2e-05","1.2e-05","1.2e-05","1.3e-05","1.6e-05","1.6e-05","1.8e-05","2.0e-05","2.0e-05","2.0e-05","2.0e-05","2.1e-05","2.2e-05","2.3e-05","2.3e-05","2.3e-05","2.6e-05","2.7e-05","2.8e-05","2.8e-05","2.9e-05","3.0e-05","3.0e-05","3.1e-05","3.3e-05","3.6e-05","3.9e-05","4.0e-05","4.2e-05","4.5e-05","4.5e-05","4.7e-05","4.8e-05","5.2e-05","5.3e-05","5.5e-05","5.9e-05","6.0e-05","6.1e-05","6.3e-05","6.6e-05","6.6e-05","6.8e-05","6.8e-05","7.9e-05","8.2e-05","9.9e-05","9.9e-05","1.0e-04","1.1e-04","1.2e-04","1.2e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.3e-04","1.4e-04","1.5e-04","1.5e-04","1.5e-04","1.6e-04","1.6e-04","1.6e-04","1.7e-04","1.7e-04","1.8e-04","1.8e-04","1.8e-04","1.9e-04","2.0e-04","2.0e-04","2.0e-04","2.1e-04","2.1e-04","2.2e-04","2.2e-04","2.2e-04","2.3e-04","2.3e-04","2.4e-04","2.4e-04","2.5e-04","2.5e-04","2.5e-04","2.5e-04","2.7e-04","2.8e-04","2.8e-04","3.1e-04","3.2e-04","3.3e-04","3.3e-04","3.3e-04","3.3e-04","3.4e-04","3.6e-04","3.7e-04","3.7e-04","3.9e-04","4.0e-04","4.0e-04","4.2e-04","4.2e-04","4.2e-04","4.3e-04","4.5e-04","4.5e-04","4.7e-04","4.8e-04","4.9e-04","5.0e-04","5.1e-04","6.1e-04","6.5e-04","6.6e-04","6.7e-04","7.1e-04","7.4e-04","7.5e-04","7.5e-04","7.5e-04","7.6e-04","7.6e-04","7.7e-04","8.1e-04","9.0e-04","9.0e-04","9.1e-04","9.5e-04","9.6e-04","9.9e-04","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.6e-03","1.6e-03","1.8e-03","1.9e-03","2.0e-03","2.0e-03","2.0e-03","2.1e-03","2.1e-03","2.1e-03","2.2e-03","2.2e-03","2.3e-03","2.3e-03","2.4e-03","2.4e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.5e-03","2.6e-03","2.8e-03","2.8e-03","2.8e-03","2.9e-03","2.9e-03","2.9e-03","3.0e-03","3.0e-03","3.1e-03","3.2e-03","3.2e-03","3.3e-03","3.4e-03","3.4e-03","3.5e-03","3.8e-03","3.8e-03","3.8e-03","3.9e-03","3.9e-03","3.9e-03","4.0e-03","4.0e-03","4.1e-03","4.2e-03","4.3e-03","4.4e-03","4.5e-03","4.5e-03","4.6e-03","4.7e-03","4.8e-03","4.8e-03","4.9e-03","4.9e-03","4.9e-03","5.0e-03","5.1e-03","5.2e-03","5.4e-03","5.6e-03","5.6e-03","5.7e-03","5.8e-03","5.8e-03","6.2e-03","6.2e-03","6.3e-03","6.5e-03","6.5e-03","6.6e-03","6.7e-03","6.8e-03","7.1e-03","7.5e-03","7.5e-03","7.6e-03","7.6e-03","7.6e-03","7.6e-03","7.6e-03","7.7e-03","7.7e-03","7.8e-03","7.8e-03","7.8e-03","8.3e-03","8.5e-03","8.9e-03","8.9e-03","9.0e-03","9.1e-03","9.5e-03","9.5e-03","9.6e-03","9.7e-03","9.8e-03","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.1e-02","2.1e-02","2.1e-02","2.1e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.3e-02","2.3e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.5e-02","2.6e-02","2.6e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.8e-02","2.8e-02","2.9e-02","3.0e-02","3.0e-02","3.0e-02","3.1e-02","3.1e-02","3.2e-02","3.2e-02","3.2e-02","3.3e-02","3.3e-02","3.4e-02","3.5e-02","3.7e-02","3.7e-02","3.9e-02","4.0e-02","4.0e-02","4.0e-02","4.1e-02","4.1e-02","4.2e-02","4.2e-02","4.2e-02","4.2e-02","4.3e-02","4.3e-02","4.3e-02","4.3e-02","4.3e-02","4.4e-02","4.4e-02","4.4e-02","4.5e-02","4.5e-02","4.6e-02","4.6e-02","4.7e-02","4.9e-02","4.9e-02","4.9e-02","4.9e-02","5.0e-02","5.0e-02","5.0e-02"],[2178,2263,2265,2265,4058,3595,420,427,838,1304,9829,4564,2433,2438,11951,2107,236,789,622,1507,3519,2193,5361,193,175,5541,5680,169,183,195,308,1413,1492,1351,6178,1341,538,906,1027,1326,2778,552,558,103,6938,790,3438,14767,5364,483,509,5214,488,36,36,500,696,857,339,339,723,502,727,1183,2165,1011,178,671,424,202,73,888,2670,544,1135,6880,300,315,1842,1856,3148,426,482,4651,2544,15,493,13201,510,494,513,3644,17,17,14027,6661,44,333,18,775,475,475,1240,16,75,177,1159,2337,326,377,3377,109,109,2234,80,1001,32,1723,99,517,495,1079,395,395,35,450,14808,1866,402,402,698,1754,757,32,335,381,163,296,1502,566,1734,1509,2216,321,2319,24,780,2357,25,25,3104,158,19,2439,58,993,756,143,341,28,8,83,898,1348,112,818,13117,113,113,449,810,476,99,75,115,116,139,723,723,416,442,33,20,2657,549,341,995,286,97,22,26,238,190,213,101,24,8,4,104,1377,2048,712,1951,5863,243,339,123,1660,90,120,574,99,196,85,450,657,169,97,97,121,13,13,951,4116,203,346,2282,1932,725,42,520,2821,1846,896,11,144,2413,991,198,203,149,6474,2039,2177,31,44,2906,151,19,128,3226,12046,154,48,1242,210,845,2017,161,221,2886,30,1917,520,2176,3457,88,57,217,1934,72,694,179,316,11,824,47,253,2516,1840,1840,799,1148,24,617,78,496,71,1003,912,95,25,203,20,90,443,52,673,60,1593,352,42,965,42,415,1115,1090,3443,1473,55,59,59,43,43,73,92,2339,60,98,312,132,563,64,28,78,400,79,1724,230,1150,112,35,90,203,113,2368,1048,187,307,76,206,29,9,523,102,662,851,35,287,80,18,136,102,123,1222,26,33,33,419,120,120,114,70,3,3,180,354,163,7,745,20,20,3114,92,1189,1956,90,268,125,21,426,2452,1774,67,52,70,70,13,1551,26,177,177,130,130,544,473,387,641,2,5,6937,643,250,364,385,535,3,7,26,312,642,1596,15,1178,6961,107,62,258,414,225,110,25,43,23,109,98,511,4321,1725,535,194,264,17,13,757,388,64,114,1764,270,50,20,5280,67,660,426,95,3191,269,18,20,157,1636,40,87,57,35,1149,909,388,152,389,2759,29,29,29,53,17070,255,404,1045,1735,875,889,11,366,34,30,102,9,73,75,1384,172,1768,113,1092,329,118,52,228,2,445,48,1008,12,20,23,52,677,79,533,1252,1383,6904,72,32,2063,63,341,308,9,566,13,911,11,60,1016,164,588,41,59,12,409,2038,89,6,6447,60,728,1132,31,35,2,365,28,345,1137,183,400,61,431,10,227,11,568,15,190,4829,167,59,163,1839,11,1995,134,863,28,83,25,526,199,82,4416,9,2381,1154,1643,22,219,253,204,9,62,546,150,171,595,328,25,285,33,126,127,11,706,1814,72,112,2591,67,102,22,1070,327,69,50,57,67,149,570,171,18,701,592,213,232,296,51,66,66,7,146,217,115],[146,146,146,146,165,150,45,45,63,65,235,148,102,102,276,93,31,54,50,79,127,95,144,27,26,132,161,24,26,26,33,73,75,71,168,71,44,57,58,48,108,44,46,17,189,52,127,309,159,40,41,156,40,11,11,42,32,50,32,32,48,40,48,59,85,54,19,45,33,24,16,54,100,39,61,182,27,25,79,79,110,33,33,132,92,7,33,281,34,33,34,97,7,7,291,175,12,29,7,45,32,32,55,7,15,15,50,79,26,30,106,16,16,76,14,49,10,70,15,34,34,52,21,21,8,22,297,71,27,27,40,68,38,9,25,26,18,17,62,32,68,62,82,24,79,7,43,72,7,7,103,17,7,87,8,40,40,20,27,8,5,16,44,52,14,38,271,14,14,22,32,39,9,12,14,14,16,35,35,26,29,7,6,90,41,26,47,21,11,7,7,19,17,15,11,7,5,4,11,43,71,38,67,127,19,24,9,60,12,16,41,16,12,11,28,34,12,7,7,14,6,6,38,114,12,20,66,59,23,8,25,75,57,38,5,14,78,45,13,17,15,134,67,72,7,6,86,14,6,15,81,250,9,8,42,15,23,71,11,17,80,6,57,27,71,86,9,8,16,57,10,29,15,19,5,33,5,17,78,61,61,43,40,6,31,12,26,11,36,41,6,7,16,6,12,25,5,29,8,52,21,8,39,9,24,39,53,82,55,5,7,7,7,7,9,13,50,7,8,19,14,30,11,6,8,24,8,59,18,45,14,5,11,17,14,64,40,13,21,9,16,7,3,27,11,31,38,6,19,11,5,14,13,11,39,6,6,6,15,14,14,8,5,3,3,12,20,12,3,31,5,5,74,10,46,53,7,16,13,5,20,67,53,10,6,7,7,3,35,6,14,10,11,11,26,23,17,22,2,3,152,22,15,18,21,27,3,4,6,15,30,55,3,38,152,12,9,13,20,14,10,5,6,6,8,11,26,87,32,22,12,21,3,4,30,20,9,10,32,9,8,4,113,8,30,20,8,79,21,2,3,12,55,5,8,6,5,41,44,19,12,13,71,5,5,5,8,315,10,22,39,46,31,30,3,18,4,5,8,3,8,7,43,12,47,11,32,19,10,7,15,2,23,6,37,3,5,6,6,27,9,21,40,41,137,5,5,44,5,17,22,3,24,3,30,4,8,37,9,22,5,8,3,19,74,6,3,125,8,26,28,6,5,2,20,3,19,28,13,15,8,20,3,15,2,25,3,16,113,11,6,13,55,4,56,12,34,3,8,5,15,11,9,93,3,62,28,29,3,14,15,12,3,8,25,9,11,26,20,5,13,3,11,8,4,23,50,7,10,74,3,6,4,37,17,8,7,6,8,9,25,14,3,25,25,12,14,16,7,7,7,2,8,16,9]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/BP-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/MF-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/CC-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/scatter-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="halo.-1" class="section level3 tabset tabset-fade">
<h3><strong>5.2. Halo.</strong></h3>
<div id="gostplot-1" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in Halo. were used to generate these gene ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &lt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_2)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-4929a56e939b13f92b78" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-4929a56e939b13f92b78">{"x":{"data":[{"x":[81.4008725542289,81.3032102818689,116.213716409311,93.7551500077641,101.03850178107,109.813081328489,110.098555663079,80.5482057917015,82.5465261338361,125.739544205652,164.924652869464,110.049724526899,93.7476375252749,152.668037688289,102.037661952138,89.893734008301,116.266303786735,81.4046287954735,93.7513937665195,116.262547545491,82.8132192622037,119.969957653925,73.1596792635459,120.184063404868,73.1897291935028,85.9872431139025,120.187819646112,73.0169420962506,72.9718672013152,72.964354718826,146.53033949459,146.81581382918,73.1634355047905,118.302186541316,85.6980125380672,81.4271662429412,153.546998139529,73.1747042285244,131.295025006436,131.625574235962,72.6676116605015,72.6413179717892,101.057282987294,146.766982693,73.1521667810567,83.598273682328,73.1033356448767,73.2798789833735,110.260074036598,103.183315531745,74.6509070376576],"y":[17,17,17,17,17,17,17,17,17,17,17,17,15.7983279553549,14.8313398529007,12.3913311761008,11.5450800374036,11.4633048459576,10.7404685683751,10.6771560753074,10.050665323601,9.65500582767884,9.43350421376752,9.14619811553134,8.66039253159043,8.50806305568491,8.50590040121066,7.04289300880272,6.87324848173376,6.87324848173376,6.81161056616539,6.27477272650858,5.85095991322945,5.49307507352041,5.07590079597194,4.93369166513963,3.28887486204497,3.05187558865154,2.50521873831793,1.93085666151538,1.91144289504681,1.84213030481419,1.79681463391649,1.79667648138008,1.6739093420512,1.59278046145925,1.42651968004361,1.3505547437129,1.32365975003136,1.32365975003136,1.32360335046773,1.30870779711815],"text":["GO:0006396 (1010) <br> RNA processing <br> 4.946e-46","GO:0006364 (265) <br> rRNA processing <br> 1.989e-42","GO:0042254 (335) <br> ribosome biogenesis <br> 6.892e-42","GO:0016072 (275) <br> rRNA metabolic process <br> 1.154e-41","GO:0022613 (473) <br> ribonucleoprotein complex biogenesis <br> 9.684e-40","GO:0034470 (439) <br> ncRNA processing <br> 3.784e-36","GO:0034660 (535) <br> ncRNA metabolic process <br> 4.571e-32","GO:0006139 (5752) <br> nucleobase-containing compound metabolic process <br> 1.248e-18","GO:0006725 (5973) <br> cellular aromatic compound metabolic process <br> 1.605e-18","GO:0046483 (5929) <br> heterocycle metabolic process <br> 3.909e-18","GO:1901360 (6227) <br> organic cyclic compound metabolic process <br> 6.686e-17","GO:0034641 (6576) <br> cellular nitrogen compound metabolic process <br> 9.999e-17","GO:0016070 (4714) <br> RNA metabolic process <br> 1.591e-16","GO:0090304 (5234) <br> nucleic acid metabolic process <br> 1.475e-15","GO:0030490 (50) <br> maturation of SSU-rRNA <br> 4.061e-13","GO:0010467 (6192) <br> gene expression <br> 2.850e-12","GO:0042274 (74) <br> ribosomal small subunit biogenesis <br> 3.441e-12","GO:0006397 (532) <br> mRNA processing <br> 1.818e-11","GO:0016071 (880) <br> mRNA metabolic process <br> 2.103e-11","GO:0042273 (72) <br> ribosomal large subunit biogenesis <br> 8.899e-11","GO:0006807 (10148) <br> nitrogen compound metabolic process <br> 2.213e-10","GO:0044085 (3377) <br> cellular component biogenesis <br> 3.685e-10","GO:0000462 (37) <br> maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 7.142e-10","GO:0044237 (10941) <br> cellular metabolic process <br> 2.186e-09","GO:0000470 (29) <br> maturation of LSU-rRNA <br> 3.104e-09","GO:0008380 (459) <br> RNA splicing <br> 3.120e-09","GO:0044238 (10642) <br> primary metabolic process <br> 9.060e-08","GO:0000398 (360) <br> mRNA splicing, via spliceosome <br> 1.339e-07","GO:0000377 (360) <br> RNA splicing, via transesterification reactions with bulged adenosine as nucleophile <br> 1.339e-07","GO:0000375 (363) <br> RNA splicing, via transesterification reactions <br> 1.543e-07","GO:0071704 (11397) <br> organic substance metabolic process <br> 5.312e-07","GO:0071840 (6880) <br> cellular component organization or biogenesis <br> 1.409e-06","GO:0000463 (15) <br> maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 3.213e-06","GO:0043170 (9867) <br> macromolecule metabolic process <br> 8.397e-06","GO:0008152 (11915) <br> metabolic process <br> 1.165e-05","GO:0006403 (222) <br> RNA localization <br> 5.142e-04","GO:0090669 (4) <br> telomerase RNA stabilization <br> 8.874e-04","GO:0000466 (24) <br> maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 3.125e-03","GO:0051168 (202) <br> nuclear export <br> 1.173e-02","GO:0051276 (1270) <br> chromosome organization <br> 1.226e-02","GO:0000184 (123) <br> nuclear-transcribed mRNA catabolic process, nonsense-mediated decay <br> 1.438e-02","GO:0000154 (37) <br> rRNA modification <br> 1.597e-02","GO:0022618 (196) <br> ribonucleoprotein complex assembly <br> 1.597e-02","GO:0071826 (203) <br> ribonucleoprotein complex subunit organization <br> 2.119e-02","GO:0000460 (36) <br> maturation of 5.8S rRNA <br> 2.554e-02","GO:0007059 (353) <br> chromosome segregation <br> 3.745e-02","GO:0000447 (9) <br> endonucleolytic cleavage in ITS1 to separate SSU-rRNA from 5.8S rRNA and LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 4.461e-02","GO:0000495 (2) <br> box H/ACA RNA 3'-end processing <br> 4.746e-02","GO:0034964 (2) <br> box H/ACA RNA processing <br> 4.746e-02","GO:0031126 (12) <br> sno(s)RNA 3'-end processing <br> 4.747e-02","GO:0001825 (40) <br> blastocyst formation <br> 4.912e-02"],"key":["GO:0006396","GO:0006364","GO:0042254","GO:0016072","GO:0022613","GO:0034470","GO:0034660","GO:0006139","GO:0006725","GO:0046483","GO:1901360","GO:0034641","GO:0016070","GO:0090304","GO:0030490","GO:0010467","GO:0042274","GO:0006397","GO:0016071","GO:0042273","GO:0006807","GO:0044085","GO:0000462","GO:0044237","GO:0000470","GO:0008380","GO:0044238","GO:0000398","GO:0000377","GO:0000375","GO:0071704","GO:0071840","GO:0000463","GO:0043170","GO:0008152","GO:0006403","GO:0090669","GO:0000466","GO:0051168","GO:0051276","GO:0000184","GO:0000154","GO:0022618","GO:0071826","GO:0000460","GO:0007059","GO:0000447","GO:0000495","GO:0034964","GO:0031126","GO:0001825"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[19.4503282144806,17.6643246110046,17.9934386332075,17.7168500269786,18.4643827493734,18.3638334737416,18.6289175161382,21.5053658734699,21.5472727056056,21.5390620738563,21.5934484800124,21.6537323834277,21.2824953778867,21.4000311246093,15.0486221338682,21.5872057726815,15.7151677983179,18.6214455917298,19.2759171540558,15.6697988614665,22.1262884949501,20.9024525251713,14.5086078145015,22.2070086720384,14.0509285588584,18.4239673394241,22.1773231312919,18.0929757080159,18.0929757080159,18.1044082182465,22.2506677513959,21.7035374999992,12.6954028668369,22.0960714860976,22.298075013352,17.4104643922402,9.00890082870231,13.6808214449962,17.2731463069687,19.73606939506,16.5273136649716,14.5086078145015,17.2289935746564,17.2803636129636,14.4581139285914,18.0658885682371,11.4827422948031,3.77952755905512,3.77952755905512,12.187219027138,14.651001049555],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedDatac255e246","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[56.1529091875501,68.5696635609685,53.3013761862114,57.3363517375128,57.3814352632257,57.3138099746564,62.1001776211723,59.898598782194,53.0534167947906,52.9895484666974,56.1604231085023,59.894841821718,59.8798139798137,59.8910848612419,59.8835709402897,59.8760570193376,66.7625655719778,52.9482219014606,59.8723000588615,56.878002559432,58.3018905798634,53.1398268857403,57.7796730736894,53.3088901071635,56.1641800689783,55.258752594245,62.9041671630517,53.1886673719292,55.2700234756732,53.2675635419267,56.1679370294544,58.3056475403395,53.1510977671685,55.7471574561343,58.7639967184202,52.5086575257602,62.8966532420996,57.6632072989311,58.3432171451002,52.2982677391001,55.1422868194867,52.294510778624,52.2606981343394,60.8190540988317,53.0008193481256,62.8741114792432,52.4898727233798],"y":[17,17,17,17,17,17,17,17,17,17,15.3152734461119,13.6712223361854,13.6237684414588,13.4025103214842,10.1271413194324,10.0208009684976,8.82292621466817,7.9557202406558,7.92961993104032,7.29763724475108,7.20132294040956,7.18888322671566,6.45848969214725,5.75802494953133,4.96884510067049,4.23866905339323,4.10693873817329,3.85927761435119,3.76914927333118,3.65575823563889,3.31998807259357,3.20812858151447,3.0098539851379,2.74638387957287,2.33910740780119,2.29907768444768,2.09381532966643,1.94877931021087,1.94877931021087,1.87877837814117,1.86328509648672,1.82628701854621,1.73567862200883,1.58645363536941,1.55787738065501,1.54464309172483,1.50854335913393],"text":["GO:0030684 (79) <br> preribosome <br> 1.656e-36","GO:1990904 (694) <br> ribonucleoprotein complex <br> 1.673e-35","GO:0005730 (955) <br> nucleolus <br> 2.733e-30","GO:0031981 (4453) <br> nuclear lumen <br> 4.318e-28","GO:0032040 (39) <br> small-subunit processome <br> 1.284e-27","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 3.280e-26","GO:0070013 (5579) <br> intracellular organelle lumen <br> 3.280e-26","GO:0043233 (5579) <br> organelle lumen <br> 3.280e-26","GO:0005654 (4112) <br> nucleoplasm <br> 1.852e-22","GO:0005634 (7607) <br> nucleus <br> 5.529e-22","GO:0030686 (28) <br> 90S preribosome <br> 4.839e-16","GO:0043232 (5094) <br> intracellular non-membrane-bounded organelle <br> 2.132e-14","GO:0043228 (5102) <br> non-membrane-bounded organelle <br> 2.378e-14","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 3.958e-14","GO:0043229 (13117) <br> intracellular organelle <br> 7.462e-11","GO:0043227 (13201) <br> membrane-bounded organelle <br> 9.532e-11","GO:0140513 (1236) <br> nuclear protein-containing complex <br> 1.503e-09","GO:0005622 (14808) <br> intracellular anatomical structure <br> 1.107e-08","GO:0043226 (14027) <br> organelle <br> 1.176e-08","GO:0031428 (7) <br> box C/D RNP complex <br> 5.039e-08","GO:0034388 (6) <br> Pwp2p-containing subcomplex of 90S preribosome <br> 6.290e-08","GO:0005681 (192) <br> spliceosomal complex <br> 6.473e-08","GO:0032991 (5541) <br> protein-containing complex <br> 3.479e-07","GO:0005732 (24) <br> sno(s)RNA-containing ribonucleoprotein complex <br> 1.746e-06","GO:0030687 (24) <br> preribosome, large subunit precursor <br> 1.074e-05","GO:0016604 (816) <br> nuclear body <br> 5.772e-05","GO:0071013 (85) <br> catalytic step 2 spliceosome <br> 7.817e-05","GO:0005694 (1795) <br> chromosome <br> 1.383e-04","GO:0016607 (413) <br> nuclear speck <br> 1.702e-04","GO:0005721 (23) <br> pericentric heterochromatin <br> 2.209e-04","GO:0030688 (15) <br> preribosome, small subunit precursor <br> 4.786e-04","GO:0034399 (132) <br> nuclear periphery <br> 6.193e-04","GO:0005684 (93) <br> U2-type spliceosomal complex <br> 9.776e-04","GO:0030057 (25) <br> desmosome <br> 1.793e-03","GO:0035145 (20) <br> exon-exon junction complex <br> 4.580e-03","GO:0001651 (2) <br> dense fibrillar component <br> 5.023e-03","GO:0071011 (52) <br> precatalytic spliceosome <br> 8.057e-03","GO:0032545 (2) <br> CURI complex <br> 1.125e-02","GO:0034456 (2) <br> UTP-C complex <br> 1.125e-02","GO:0000793 (239) <br> condensed chromosome <br> 1.322e-02","GO:0016363 (110) <br> nuclear matrix <br> 1.370e-02","GO:0000792 (76) <br> heterochromatin <br> 1.492e-02","GO:0000775 (200) <br> chromosome, centromeric region <br> 1.838e-02","GO:0044530 (3) <br> supraspliceosomal complex <br> 2.591e-02","GO:0005638 (4) <br> lamin filament <br> 2.768e-02","GO:0071005 (50) <br> U2-type precatalytic spliceosome <br> 2.853e-02","GO:0001533 (44) <br> cornified envelope <br> 3.101e-02"],"key":["GO:0030684","GO:1990904","GO:0005730","GO:0031981","GO:0032040","GO:0031974","GO:0070013","GO:0043233","GO:0005654","GO:0005634","GO:0030686","GO:0043232","GO:0043228","GO:0043231","GO:0043229","GO:0043227","GO:0140513","GO:0005622","GO:0043226","GO:0031428","GO:0034388","GO:0005681","GO:0032991","GO:0005732","GO:0030687","GO:0016604","GO:0071013","GO:0005694","GO:0016607","GO:0005721","GO:0030688","GO:0034399","GO:0005684","GO:0030057","GO:0035145","GO:0001651","GO:0071011","GO:0032545","GO:0034456","GO:0000793","GO:0016363","GO:0000792","GO:0000775","GO:0044530","GO:0005638","GO:0071005","GO:0001533"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[15.8227421236023,18.9706470582724,19.3796840599657,21.2181834681373,14.6049645781774,21.4713489077623,21.4713489077623,21.4713489077623,21.1278301730825,21.8137488730278,13.9833131530912,21.3696524759062,21.3714121953266,22.3097189832527,22.4001723500672,22.4069336180039,19.7024866025059,22.5281903038104,22.4710947059109,10.8097872173079,10.3630624020686,17.1987173630513,21.4637267954854,13.6808214449962,13.6808214449962,19.17949942902,15.9420548318458,20.1582119259402,18.2810218264132,13.5956642813554,12.6954028668369,16.6361245292714,16.0870759222918,13.7618192670139,13.3106655321065,3.77952755905512,15.1170689685676,3.77952755905512,3.77952755905512,17.5168301728758,16.3532722717548,15.7591617451593,17.2585921453936,7.7791008157754,9.00890082870231,15.0486221338682,14.8225866744181],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedDatac255e246","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[3.64155719716214,3.49505666926895,46.8817514684057,40.706190754139,20.2787581725194,3.64531362095428,5.77144948730158,46.2506722713273,9.36259063258078,46.2469158475352,16.4659880235044,22.3072270202713,16.8679253692626,22.299714172687,38.873055943578,25.8194832659157,22.3222527154398,20.3388609531935,9.08837169575506,2.12771840893252,46.8704821970293,27.4460147679093,40.8301527392793,9.58421963631663,8.99070467715961,25.9960351841459],"y":[17,17,17,17,12.2093233016376,7.34018333648262,5.00168160306185,4.69506202703926,4.62211893287138,4.37348975580469,3.52927920889957,2.81911175147016,2.76286569266562,2.75541137791799,2.69725550549555,2.49492890087574,2.39399832807929,2.34191978992744,2.04874878761161,1.92556436468069,1.92240976632695,1.72984633615705,1.60247118498015,1.52844355405403,1.48235730090676,1.43956531922843],"text":["GO:0003723 (1951) <br> RNA binding <br> 3.344e-44","GO:0003676 (4252) <br> nucleic acid binding <br> 6.862e-25","GO:1901363 (6210) <br> heterocyclic compound binding <br> 4.730e-24","GO:0097159 (6291) <br> organic cyclic compound binding <br> 1.759e-23","GO:0030515 (32) <br> snoRNA binding <br> 6.176e-13","GO:0003724 (76) <br> RNA helicase activity <br> 4.569e-08","GO:0004386 (163) <br> helicase activity <br> 9.961e-06","GO:0140098 (394) <br> catalytic activity, acting on RNA <br> 2.018e-05","GO:0008094 (185) <br> ATPase, acting on DNA <br> 2.387e-05","GO:0140097 (299) <br> catalytic activity, acting on DNA <br> 4.232e-05","GO:0016887 (553) <br> ATPase <br> 2.956e-04","GO:0032555 (1917) <br> purine ribonucleotide binding <br> 1.517e-03","GO:0017076 (1932) <br> purine nucleotide binding <br> 1.726e-03","GO:0032553 (1934) <br> ribonucleotide binding <br> 1.756e-03","GO:0070034 (22) <br> telomerase RNA binding <br> 2.008e-03","GO:0035639 (1846) <br> purine ribonucleoside triphosphate binding <br> 3.199e-03","GO:0032559 (1564) <br> adenyl ribonucleotide binding <br> 4.036e-03","GO:0030554 (1577) <br> adenyl nucleotide binding <br> 4.551e-03","GO:0005524 (1500) <br> ATP binding <br> 8.938e-03","GO:0000166 (2176) <br> nucleotide binding <br> 1.187e-02","GO:1901265 (2177) <br> nucleoside phosphate binding <br> 1.196e-02","GO:0043168 (2413) <br> anion binding <br> 1.863e-02","GO:0097367 (2282) <br> carbohydrate derivative binding <br> 2.498e-02","GO:0008177 (2) <br> succinate dehydrogenase (ubiquinone) activity <br> 2.962e-02","GO:0005488 (17070) <br> binding <br> 3.293e-02","GO:0036094 (2516) <br> small molecule binding <br> 3.634e-02"],"key":["GO:0003723","GO:0003676","GO:1901363","GO:0097159","GO:0030515","GO:0003724","GO:0004386","GO:0140098","GO:0008094","GO:0140097","GO:0016887","GO:0032555","GO:0017076","GO:0032553","GO:0070034","GO:0035639","GO:0032559","GO:0030554","GO:0005524","GO:0000166","GO:1901265","GO:0043168","GO:0097367","GO:0008177","GO:0005488","GO:0036094"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[20.258276019327,21.1658574902378,21.5904209757188,21.6047679255868,14.2382740983495,15.7591617451593,16.9558134205769,18.2168145994675,17.1440106914419,17.8347771324352,18.6728117067331,20.2372173548861,20.2465569336091,20.2477963305085,13.5059251363754,20.1919193878134,19.9914490344426,20.001517942691,19.9405304557382,20.38841785752,20.3889635367376,20.5107536570554,20.4448133217894,3.77952755905512,22.6771653543307,20.5599632179123],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedDatac255e246","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[199.924846290867,199.936119123303,199.928603901679,195.321773046176,195.411955705664,199.83466363138,199.830906020568,196.734634711485],"y":[17,17,17,17,17,1.91002432222694,1.73379101571658,1.42765790392853],"text":["REAC:R-HSA-6790901 (60) <br> rRNA modification in the nucleus and cytosol <br> 1.183e-31","REAC:R-HSA-8868773 (193) <br> rRNA processing in the nucleus and cytosol <br> 2.700e-27","REAC:R-HSA-72312 (203) <br> rRNA processing <br> 1.783e-26","REAC:R-HSA-6791226 (183) <br> Major pathway of rRNA processing in the nucleolus and cytosol <br> 2.728e-24","REAC:R-HSA-8953854 (662) <br> Metabolism of RNA <br> 3.342e-20","REAC:R-HSA-72163 (178) <br> mRNA Splicing - Major Pathway <br> 1.230e-02","REAC:R-HSA-72172 (186) <br> mRNA Splicing <br> 1.846e-02","REAC:R-HSA-72203 (237) <br> Processing of Capped Intron-Containing Pre-mRNA <br> 3.735e-02"],"key":["REAC:R-HSA-6790901","REAC:R-HSA-8868773","REAC:R-HSA-72312","REAC:R-HSA-6791226","REAC:R-HSA-8953854","REAC:R-HSA-72163","REAC:R-HSA-72172","REAC:R-HSA-72203"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,102,204,1)","opacity":0.8,"size":[15.3633750239743,17.206351575782,17.2803636129636,17.1279572175344,18.90922365592,17.0869555296552,17.1519653138816,17.5047579060072],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,102,204,1)"}},"hoveron":"points","set":"SharedDatac255e246","name":"REAC","legendgroup":"REAC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1288841673939],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8850020867322,68.6335318890617],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.3896498084001,186.793489395606],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.549607314945,200.003756117919],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0644420836969,60.259266987897,129.591569602003,195.276681716432],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"255c23507b3d":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"255c5e1f2542":{"x":{},"xend":{},"y":{},"yend":{}},"255c5c522fdd":{"x":{},"xend":{},"y":{},"yend":{}},"255c40c56dd8":{"x":{},"xend":{},"y":{},"yend":{}},"255c757ced":{"x":{},"xend":{},"y":{},"yend":{}},"255c7d7d3268":{"x":{},"y":{}},"255c4be25d4":{"yintercept":{}}},"cur_data":"255c23507b3d","visdat":{"255c23507b3d":["function (y) ","x"],"255c5e1f2542":["function (y) ","x"],"255c5c522fdd":["function (y) ","x"],"255c40c56dd8":["function (y) ","x"],"255c757ced":["function (y) ","x"],"255c7d7d3268":["function (y) ","x"],"255c4be25d4":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedDatac255e246"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results-1" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#6e05e6&#39;)))
}</code></pre>
<div id="htmlwidget-56457827ab1876f7c579" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-56457827ab1876f7c579">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132"],["GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","REAC","GO:CC","GO:CC","GO:CC","REAC","REAC","GO:CC","GO:CC","GO:CC","GO:MF","REAC","GO:MF","GO:MF","GO:CC","GO:CC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:MF","GO:MF","GO:MF","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:MF","GO:MF","GO:CC","GO:MF","GO:BP","GO:MF","GO:MF","GO:MF","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","REAC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","REAC","GO:MF","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:MF","GO:MF","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP"],[2400,438,2374,11668,5689,7628,1137,9964,4442,10040,2496,378,1452,1464,2499,2497,1446,2720,2134,399,1271,11949,10305,312,295,1295,2173,2705,14204,24636,10027,5687,1139,21373,2133,2129,2132,7894,4867,4661,11682,2401,5688,2130,11681,2128,2776,12668,206,3961,12725,214,3621,284,2127,439,1330,1709,335,12726,168,156,154,1570,19739,19815,380,207,12224,1005,1140,3544,11781,1961,11780,899,2934,348,902,369,3852,1141,2407,1710,21607,338,5407,3959,5405,1029,9817,210,6342,5411,4883,1832,167,2932,1888,1539,1720,15683,35,11946,15771,2472,111,868,75,110,68,7633,101,2471,6775,19802,10338,204,2379,298,2926,2020,162,1862,6389,1647,2985,191,238,10083,8199,603],["RNA processing","RNA binding","rRNA processing","ribosome biogenesis","rRNA metabolic process","ribonucleoprotein complex biogenesis","preribosome","ncRNA processing","ribonucleoprotein complex","ncRNA metabolic process","rRNA modification in the nucleus and cytosol","nucleolus","nuclear lumen","small-subunit processome","rRNA processing in the nucleus and cytosol","rRNA processing","membrane-enclosed lumen","intracellular organelle lumen","organelle lumen","nucleic acid binding","Major pathway of rRNA processing in the nucleolus and cytosol","heterocyclic compound binding","organic cyclic compound binding","nucleoplasm","nucleus","Metabolism of RNA","nucleobase-containing compound metabolic process","cellular aromatic compound metabolic process","heterocycle metabolic process","organic cyclic compound metabolic process","cellular nitrogen compound metabolic process","RNA metabolic process","90S preribosome","nucleic acid metabolic process","intracellular non-membrane-bounded organelle","non-membrane-bounded organelle","intracellular membrane-bounded organelle","maturation of SSU-rRNA","snoRNA binding","gene expression","ribosomal small subunit biogenesis","mRNA processing","mRNA metabolic process","intracellular organelle","ribosomal large subunit biogenesis","membrane-bounded organelle","nitrogen compound metabolic process","cellular component biogenesis","maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","nuclear protein-containing complex","cellular metabolic process","maturation of LSU-rRNA","RNA splicing","intracellular anatomical structure","organelle","RNA helicase activity","box C/D RNP complex","Pwp2p-containing subcomplex of 90S preribosome","spliceosomal complex","primary metabolic process","mRNA splicing, via spliceosome","RNA splicing, via transesterification reactions with bulged adenosine as nucleophile","RNA splicing, via transesterification reactions","protein-containing complex","organic substance metabolic process","cellular component organization or biogenesis","sno(s)RNA-containing ribonucleoprotein complex","maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","macromolecule metabolic process","helicase activity","preribosome, large subunit precursor","metabolic process","catalytic activity, acting on RNA","ATPase, acting on DNA","catalytic activity, acting on DNA","nuclear body","catalytic step 2 spliceosome","chromosome","nuclear speck","pericentric heterochromatin","ATPase","preribosome, small subunit precursor","RNA localization","nuclear periphery","telomerase RNA stabilization","U2-type spliceosomal complex","purine ribonucleotide binding","purine nucleotide binding","ribonucleotide binding","desmosome","telomerase RNA binding","maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","purine ribonucleoside triphosphate binding","adenyl ribonucleotide binding","adenyl nucleotide binding","exon-exon junction complex","dense fibrillar component","precatalytic spliceosome","ATP binding","CURI complex","UTP-C complex","nuclear export","nucleotide binding","nucleoside phosphate binding","chromosome organization","mRNA Splicing - Major Pathway","condensed chromosome","nuclear matrix","nuclear-transcribed mRNA catabolic process, nonsense-mediated decay","heterochromatin","rRNA modification","ribonucleoprotein complex assembly","chromosome, centromeric region","mRNA Splicing","anion binding","ribonucleoprotein complex subunit organization","carbohydrate derivative binding","maturation of 5.8S rRNA","supraspliceosomal complex","lamin filament","U2-type precatalytic spliceosome","succinate dehydrogenase (ubiquinone) activity","cornified envelope","binding","small molecule binding","Processing of Capped Intron-Containing Pre-mRNA","chromosome segregation","endonucleolytic cleavage in ITS1 to separate SSU-rRNA from 5.8S rRNA and LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","box H/ACA RNA 3'-end processing","box H/ACA RNA processing","sno(s)RNA 3'-end processing","blastocyst formation"],["4.9e-46","3.3e-44","2.0e-42","6.9e-42","1.2e-41","9.7e-40","1.7e-36","3.8e-36","1.7e-35","4.6e-32","1.2e-31","2.7e-30","4.3e-28","1.3e-27","2.7e-27","1.8e-26","3.3e-26","3.3e-26","3.3e-26","6.9e-25","2.7e-24","4.7e-24","1.8e-23","1.9e-22","5.5e-22","3.3e-20","1.2e-18","1.6e-18","3.9e-18","6.7e-17","1.0e-16","1.6e-16","4.8e-16","1.5e-15","2.1e-14","2.4e-14","4.0e-14","4.1e-13","6.2e-13","2.9e-12","3.4e-12","1.8e-11","2.1e-11","7.5e-11","8.9e-11","9.5e-11","2.2e-10","3.7e-10","7.1e-10","1.5e-09","2.2e-09","3.1e-09","3.1e-09","1.1e-08","1.2e-08","4.6e-08","5.0e-08","6.3e-08","6.5e-08","9.1e-08","1.3e-07","1.3e-07","1.5e-07","3.5e-07","5.3e-07","1.4e-06","1.7e-06","3.2e-06","8.4e-06","1.0e-05","1.1e-05","1.2e-05","2.0e-05","2.4e-05","4.2e-05","5.8e-05","7.8e-05","1.4e-04","1.7e-04","2.2e-04","3.0e-04","4.8e-04","5.1e-04","6.2e-04","8.9e-04","9.8e-04","1.5e-03","1.7e-03","1.8e-03","1.8e-03","2.0e-03","3.1e-03","3.2e-03","4.0e-03","4.6e-03","4.6e-03","5.0e-03","8.1e-03","8.9e-03","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.4e-02","1.4e-02","1.5e-02","1.6e-02","1.6e-02","1.8e-02","1.8e-02","1.9e-02","2.1e-02","2.5e-02","2.6e-02","2.6e-02","2.8e-02","2.9e-02","3.0e-02","3.1e-02","3.3e-02","3.6e-02","3.7e-02","3.7e-02","4.5e-02","4.7e-02","4.7e-02","4.7e-02","4.9e-02"],[1010,1951,265,335,275,473,79,439,694,535,60,955,4453,39,193,203,5579,5579,5579,4252,183,6210,6291,4112,7607,662,5752,5973,5929,6227,6576,4714,28,5234,5094,5102,12046,50,32,6192,74,532,880,13117,72,13201,10148,3377,37,1236,10941,29,459,14808,14027,76,7,6,192,10642,360,360,363,5541,11397,6880,24,15,9867,163,24,11915,394,185,299,816,85,1795,413,23,553,15,222,132,4,93,1917,1932,1934,25,22,24,1846,1564,1577,20,2,52,1500,2,2,202,2176,2177,1270,178,239,110,123,76,37,196,200,186,2413,203,2282,36,3,4,50,2,44,17070,2516,237,353,9,2,2,12,40],[74,91,45,48,45,52,28,48,54,48,27,56,105,19,36,36,114,114,114,102,33,121,121,94,126,50,111,113,112,113,116,97,12,101,93,93,146,13,11,105,14,26,36,151,13,148,135,71,10,37,139,9,23,157,153,7,5,5,14,134,19,19,19,82,138,99,6,6,124,7,6,136,14,7,8,21,8,30,15,4,9,4,12,9,3,7,22,22,22,4,4,5,21,19,19,4,2,5,18,2,2,10,22,22,17,11,10,7,8,3,5,9,9,11,23,9,22,5,2,2,4,2,3,138,23,12,12,3,2,2,3,5]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#6e05e6\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap-1" class="section level4 tabset tabset-fade">
<h4>Treemap</h4>
<div id="bp-1" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/BP-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf-1" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/MF-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc-1" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/CC-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot-1" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/scatter-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="network-analysis" class="section level2">
<h2><strong>6. Network analysis</strong></h2>
<p>Overview from <strong>STRING Database</strong>: In STRING, each protein-protein interaction is annotated with one or more ‘scores’. Importantly, these scores do <strong>not</strong> indicate the strength or the specificity of the interaction. Instead, they are indicators of <strong>confidence</strong>, i.e. how likely STRING judges an interaction to be true, given the available evidence. All scores rank from 0 to 1, with 1 being the highest possible confidence. A score of 0.5 would indicate that roughly every second interaction might be erroneous (i.e., a false positive).</p>
<pre class="r"><code>if(leng_dep != 0){
  
  string_db &lt;- STRINGdb$new(version=&quot;11&quot;, species=9606, score_threshold=400)

  string_mapped &lt;- string_db$map(string_res, &quot;Gene.name&quot;, removeUnmappedRows = TRUE, quiet = T)

  hits &lt;- string_mapped$STRING_id[1:leng_dep]

  string_db$plot_network(hits, add_link = T)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/Template_LFQ.Rmd/stringdb-1.png" width="1440" style="display: block; margin: auto;" /></p>
<div id="cytoscape-network" class="section level3">
<h3><strong>6.1. Cytoscape Network</strong></h3>
<p>Cytoscape is an open source software platform for visualizing complex networks and integrating these with any type of attribute data. In this presentation all proteins showing at least one confidence level from the STRING database are shown. The <strong>color</strong> code is relative to <strong>fold change</strong> and the <strong>size</strong> is directly proportional to the <strong>significance (p-value)</strong>.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>if(n_edge != 0){
  knitr::include_graphics(paste0(&quot;assets/network_FC_pvalue&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_FC_pvalueSKAP2_N._vs_Halo..png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
<div id="cytoscape-enrichmentmap" class="section level3">
<h3><strong>6.2. Cytoscape EnrichmentMap</strong></h3>
<p>An <strong>enrichment map</strong> is a different sort of network. Instead of nodes representing genes, nodes represent pathways or functions. Edges between these pathways or functions represent shared genes or pathway crosstalk. An enrichment map is a way to visualize your enrichment results to help reduce redundancy and uncover main themes.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant) %&gt;% pull(name)

if(!params$Imputation){
  
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  pull(name) %&gt;% 
  unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  
  print(&quot;NOT enough data.&quot;)
  
}else{
  
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  
  if(is.null(GO)){
    
    print(&quot;NOT enough data.&quot;)

    enrich_map = F
    
  }else{
    
    enrich_map = T
    
    gem &lt;- GO$result[,c(&quot;term_id&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;intersection&quot;)]
    colnames(gem) &lt;- c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;Genes&quot;)
    gem$FDR &lt;- gem$p.Val
    gem$Phenotype = &quot;+1&quot;
    gem &lt;- gem[,c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;FDR&quot;, &quot;Phenotype&quot;, &quot;Genes&quot;)]
    head(gem)
  
    write.table(gem, here::here(&quot;output/enrichmentmap.txt&quot;),col.name=TRUE,sep=&quot;\t&quot;,row.names=FALSE,quote=FALSE)
    
  }
}</code></pre>
<pre class="r"><code>if(enrich_map){
  knitr::include_graphics(paste0(&quot;assets/network_enrichmentmap&quot;, test, &quot;.png&quot;),  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="assets/network_enrichmentmapSKAP2_N._vs_Halo..png" width="150%" height="150%" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="section" class="section level2 unnumbered">
<h2></h2>
<pre class="r"><code>saveWorkbook(excel, here::here(paste0(&quot;output/results_table_&quot;, test, &quot;.xlsx&quot;)), overwrite = T)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] forcats_0.5.1               stringr_1.4.0              
 [3] dplyr_1.0.7                 purrr_0.3.4                
 [5] readr_1.4.0                 tidyr_1.1.3                
 [7] tibble_3.1.2                tidyverse_1.3.1            
 [9] openxlsx_4.2.4              here_1.0.1                 
[11] RColorBrewer_1.1-2          RCy3_2.10.2                
[13] STRINGdb_2.2.2              naniar_0.6.1               
[15] visdat_0.5.3                ggrepel_0.9.1              
[17] ggplot2_3.3.5               UpSetR_1.4.0               
[19] ggstatsplot_0.8.0           datapasta_3.1.0            
[21] patchwork_1.1.1             DT_0.18                    
[23] rrvgo_1.2.0                 org.Hs.eg.db_3.12.0        
[25] AnnotationDbi_1.52.0        gprofiler2_0.2.0           
[27] enrichR_3.0                 preprocessCore_1.52.1      
[29] SummarizedExperiment_1.20.0 Biobase_2.50.0             
[31] GenomicRanges_1.42.0        GenomeInfoDb_1.26.2        
[33] IRanges_2.24.1              S4Vectors_0.28.1           
[35] BiocGenerics_0.36.0         MatrixGenerics_1.2.1       
[37] matrixStats_0.59.0          DEP_1.12.0                 
[39] MBQN_2.2.0                  workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.3            R.methodsS3_1.8.1        
  [3] coda_0.19-4               bit64_4.0.5              
  [5] knitr_1.33                DelayedArray_0.16.2      
  [7] R.utils_2.10.1            data.table_1.14.0        
  [9] RCurl_1.98-1.3            doParallel_1.0.16        
 [11] generics_0.1.0            RSQLite_2.2.7            
 [13] correlation_0.6.1         chron_2.3-56             
 [15] bit_4.0.4                 lubridate_1.7.10         
 [17] xml2_1.3.2                httpuv_1.6.1             
 [19] assertthat_0.2.1          WRS2_1.1-2               
 [21] xfun_0.24                 hms_1.1.0                
 [23] jquerylib_0.1.4           evaluate_0.14            
 [25] promises_1.2.0.1          fansi_0.5.0              
 [27] readxl_1.3.1              caTools_1.18.2           
 [29] dbplyr_2.1.1              igraph_1.2.6             
 [31] DBI_1.1.1                 htmlwidgets_1.5.3        
 [33] reshape_0.8.8             kSamples_1.2-9           
 [35] Rmpfr_0.8-4               paletteer_1.3.0          
 [37] hash_2.2.6.1              ellipsis_0.3.2           
 [39] crosstalk_1.1.1           backports_1.2.1          
 [41] insight_0.14.2            gridBase_0.4-7           
 [43] vctrs_0.3.8               imputeLCMD_2.0           
 [45] Cairo_1.5-12.2            cachem_1.0.5             
 [47] withr_2.4.2               treemap_2.4-2            
 [49] cluster_2.1.0             lazyeval_0.2.2           
 [51] crayon_1.4.1              labeling_0.4.2           
 [53] pkgconfig_2.0.3           SuppDists_1.1-9.5        
 [55] slam_0.1-48               statsExpressions_1.1.0   
 [57] ProtGenerics_1.22.0       wordcloud_2.6            
 [59] rlang_0.4.11              RJSONIO_1.3-1.4          
 [61] lifecycle_1.0.0           MatrixModels_0.5-0       
 [63] sandwich_3.0-1            affyio_1.60.0            
 [65] BiocFileCache_1.14.0      modelr_0.1.8             
 [67] cellranger_1.1.0          rprojroot_2.0.2          
 [69] graph_1.68.0              Matrix_1.2-18            
 [71] mc2d_0.1-21               zoo_1.8-9                
 [73] reprex_2.0.0              whisker_0.4              
 [75] GlobalOptions_0.1.2       pheatmap_1.0.12          
 [77] png_0.1-7                 viridisLite_0.4.0        
 [79] rjson_0.2.20              mzR_2.24.1               
 [81] PMCMRplus_1.9.0           parameters_0.14.0        
 [83] bitops_1.0-7              shinydashboard_0.7.1     
 [85] R.oo_1.24.0               KernSmooth_2.23-17       
 [87] blob_1.2.1                shape_1.4.6              
 [89] multcompView_0.1-8        tmvtnorm_1.4-10          
 [91] ggsignif_0.6.2            scales_1.1.1             
 [93] memoise_2.0.0             magrittr_2.0.1           
 [95] plyr_1.8.6                gplots_3.1.1             
 [97] zlibbioc_1.36.0           compiler_4.0.3           
 [99] plotrix_3.8-1             pcaMethods_1.82.0        
[101] clue_0.3-59               cli_3.0.0                
[103] affy_1.68.0               XVector_0.30.0           
[105] pbapply_1.4-3             MASS_7.3-53              
[107] tidyselect_1.1.1          vsn_3.58.0               
[109] stringi_1.6.2             highr_0.9                
[111] yaml_2.2.1                GOSemSim_2.16.1          
[113] norm_1.0-9.5              MALDIquant_1.19.3        
[115] grid_4.0.3                sass_0.4.0               
[117] tools_4.0.3               rstudioapi_0.13          
[119] circlize_0.4.13           foreach_1.5.1            
[121] git2r_0.28.0              gridExtra_2.3            
[123] ipmisc_6.0.2              pairwiseComparisons_3.1.6
[125] farver_2.1.0              mzID_1.28.0              
[127] digest_0.6.27             BiocManager_1.30.16      
[129] shiny_1.6.0               proto_1.0.0              
[131] BWStest_0.2.2             Rcpp_1.0.6               
[133] broom_0.7.8               BayesFactor_0.9.12-4.2   
[135] performance_0.7.2         later_1.2.0              
[137] ncdf4_1.17                httr_1.4.2               
[139] MSnbase_2.15.7            ComplexHeatmap_2.6.2     
[141] effectsize_0.4.5          colorspace_2.0-2         
[143] rvest_1.0.0               XML_3.99-0.6             
[145] fs_1.5.0                  rematch2_2.1.2           
[147] gmm_1.6-6                 plotly_4.9.4.1           
[149] xtable_1.8-4              gmp_0.6-2                
[151] jsonlite_1.7.2            zeallot_0.1.0            
[153] NLP_0.2-1                 R6_2.5.0                 
[155] tm_0.7-8                  gsubfn_0.7               
[157] pillar_1.6.1              htmltools_0.5.1.1        
[159] mime_0.11                 glue_1.4.2               
[161] fastmap_1.1.0             BiocParallel_1.24.1      
[163] codetools_0.2-16          mvtnorm_1.1-2            
[165] utf8_1.2.1                lattice_0.20-41          
[167] bslib_0.2.5.1             sqldf_0.4-11             
[169] curl_4.3.2                gtools_3.9.2             
[171] zip_2.2.0                 GO.db_3.12.1             
[173] limma_3.46.0              rmarkdown_2.9            
[175] munsell_0.5.0             GetoptLong_1.0.5         
[177] GenomeInfoDbData_1.2.4    iterators_1.0.13         
[179] impute_1.64.0             haven_2.4.1              
[181] gtable_0.3.0              bayestestR_0.10.0        </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
